+++
title = "Genetic determinants of"
full_title = "Genetic determinants of \"evolvability\""
date = "2007-07-09"
upstream_url = "https://www.gnxp.com/WordPress/2007/07/09/genetic-determinants-of-evolvability/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/07/09/genetic-determinants-of-evolvability/).

Genetic determinants of "evolvability"

In comments on previous posts, I’ve alluded to the “mutational target size” of a phenotype– that is, the number of locations in the genome that affect (or could, in theory, affect) the phenotype in question. If one imagines mutations as darts thrown randomly at a dartboard, the “target size” analogy is self-explanatory. There are a number of reasons why different phenotypes could have different target sizes– the number of genes underlying the trait is likely an important parameter, and it seems intuitive to suggest that some phenotypes would be selected for robustness (ie. a smaller target size).

Data in this area is hard to come by. Luckily, expression microarrays have put at our disposition the ability to assay thousands of phenotypes– gene expression profiles– in parallel. A [recent paper](http://www.sciencemag.org/cgi/content/abstract/317/5834/118) takes a look at this question through the use of mutation accumulation lines in yeast. I’ve [mentioned](https://www.gnxp.com/blog/2007/06/do-phenotypes-evolve-neutrally.php) MA lines before– essentually, they’re lines propogated with an extremely small effective population size, such that selection plays a nearly negligible role in the fate of new mutations.

The authors propogated a number of these lines for 4000 generations, then assayed gene expression to see which genes had diverged in expression, and by how much. The results are somewhat intuitive– they find that gene expression evolvability is correlated with trans-mutational target size (as judged by expression profiling of knockout strains) and the presence of a particular promoter sequence. This is a small step in out understanding of regulatory evolution, but an important one. They conclude:

> We show that not all genes are equally sensitive to the effects of > random spontaneous mutations and identify structural properties > (presence of a TATA box and trans-mutational target sizes) that > greatly influence a gene’s potential to undergo regulatory change. > These determinants provide a mechanistic basis to serve as a foundation for more-realistic models of gene expression evolution that account for levels of polymorphism and divergence in cis and trans gene regulation.

### Related Posts:

- [Was lactose tolerance
  inevitable?](https://www.gnxp.com/WordPress/2008/01/14/was-lactose-tolerance-inevitable/) - [Phenotype does not imply ancestry
  (always)](https://www.gnxp.com/WordPress/2019/09/28/phenotype-does-not-imply-admixture/) - [Genomics to complex
  traits](https://www.gnxp.com/WordPress/2009/02/18/genomics-to-complex-traits/) - [Robustness and fragility in neural
  development](https://www.gnxp.com/WordPress/2012/04/27/robustness-and-fragility-in-neural-development/) - [The different dynamics of memes vs.
  genes](https://www.gnxp.com/WordPress/2011/06/26/the-different-dynamics-of-memes-vs-genes/) - [Common disease, common
  variant](https://www.gnxp.com/WordPress/2006/12/01/common-disease-common-variant/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F07%2F09%2Fgenetic-determinants-of-evolvability%2F&linkname=Genetic%20determinants%20of%20%22evolvability%22 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F07%2F09%2Fgenetic-determinants-of-evolvability%2F&linkname=Genetic%20determinants%20of%20%22evolvability%22 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F07%2F09%2Fgenetic-determinants-of-evolvability%2F&linkname=Genetic%20determinants%20of%20%22evolvability%22 "Email")[](https://www.addtoany.com/share)
