+++
title = "What is molecular"
full_title = "What is molecular biology?"
date = "2017-10-04"
upstream_url = "https://www.gnxp.com/WordPress/2017/10/04/what-is-molecular-biology/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/10/04/what-is-molecular-biology/).

What is molecular biology?

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/10/mobiocell.jpeg?resize=192%2C263)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/10/mobiocell.jpeg?resize=192%2C263)](https://www.amazon.com/exec/obidos/ASIN/0815344325/geneexpressio-20)In [Massive Migrations? The Impact of Recent aDNA Studies on our View of Third Millennium Europe](https://www.cambridge.org/core/journals/european-journal-of-archaeology/article/massive-migrations-the-impact-of-recent-adna-studies-on-our-view-of-third-millennium-europe/B225B14D82066373B2C11295AA1D1462) there is a reference to “molecular biological work.” I regularly see scholarship in evolutionary and population genomics and genetics referred to as “molecular biology” outside of the field, because since the 1960s molecular methods have been part and parcel of the discipline.

I get where this is coming from. In the case of ancient DNA there is some serious “bench biology” going on, and the skills of a molecular biologist come in handy. **But I don’t consider a lot of evolutionary and population genomics work “molecular biology.”** The first authors mentioned in the ancient DNA papers cited in the review are not trained as molecular biologists, but rather come out of computer science, statistics, archaeology or population genetics.

To illustrate the difference between population genomic type work and molecular biology (in this case, genetics), let’s look at the *LCT* locus, which is implicated in lactase persistence (lactose intolerance).

Here is something out of the population genetic/genomic tradition, [Genetic Signatures of Strong Recent Positive Selection at the Lactase Gene](http://www.sciencedirect.com/science/article/pii/S0002929707628389):

> … To assess the population-genetics evidence for selection, we typed > 101 single-nucleotide **polymorphisms** covering 3.2 Mb around the > lactase gene. In northern European–derived populations, two alleles > that are tightly associated with lactase persistence (Enattah et al. > 2002) uniquely mark a common (∼77%) **haplotype** that extends largely > undisrupted for \>1 Mb. We provide two new lines of genetic evidence > that this long, common haplotype arose rapidly due to **recent > selection**: (1) by use of the **traditional FST** measure and a novel > test based on pexcess, we demonstrate large frequency differences > among populations for the persistence-**associated markers** and for > flanking markers throughout the **haplotype**, and (2) we show that > the haplotype is unusually long, given its high frequency—a hallmark > of **recent selection**. We estimate that **strong selection** > occurred within the past 5,000–10,000 years, consistent with an advantage to lactase persistence in the setting of dairy farming; the signals of selection we observe are among **the strongest yet seen for any gene in the genome**.

Now, compare to something which I would classify as molecular genetic, [T −13910 DNA variant associated with lactase persistence interacts with Oct-1 and stimulates lactase promoter activity in vitro](https://academic.oup.com/hmg/article/14/24/3945/2355865/T-13910-DNA-variant-associated-with-lactase):

> Two phenotypes exist in the human population with regard to expression > of lactase in adults. Lactase non-persistence (adult-type hypolactasia > and lactose intolerance) is characterized by a **decline in the > expression of lactase-phlorizin hydrolase (LPH)** after weaning. In > contrast, lactase-persistent individuals have a high LPH throughout > their lifespan. Lactase persistence and non-persistence are associated > with a T/C polymorphism at position −13 910 upstream the lactase gene. > A nuclear factor **binds more strongly** to the T −13 910 variant > associated with lactase persistence than the C −13 910 variant > associated with lactase non-persistence. Oct-1 and > glyceraldehyde-3-phosphate dehydrogenase **were co-purified** by DNA > affinity purification using the sequence of the T −13 910 variant. > Supershift analyses show that Oct-1 **binds** directly to the T > −13 910 variant, and we suggest that GAPDH is co-purified due to > **interactions** with Oct-1. Expression of Oct-1 **stimulates reporter > gene expression** from the T and the C −13 910 variant/LPH promoter > constructs only when it is **co-expressed** with HNF1 α. **Binding > sites** for other intestinal transcription factors (GATA-6, HNF4 α, > Fox and Cdx-2) were identified in the region of the −13 910 T/C > polymorphism. Three of these sites are required for the **enhancer > activity** of the −13 910 region. The data suggest that the binding of Oct-1 to the T −13 910 variant directs increased **lactase promoter activity and this might provide an explanation for the lactase persistence phenotype** in the human population.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/10/principlesofpopulationgenetics.jpeg?resize=198%2C255)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/10/principlesofpopulationgenetics.jpeg?resize=198%2C255)](https://www.amazon.com/exec/obidos/ASIN/0878933085/geneexpressio-20)I bolded the sorts of words which are good “tells” about the field that the paper is coming from. Traditional molecular biological work focuses a lot on mechanistic details. That is, **the questions being asked are in relation to mechanisms on the scale of molecular machinery**. Usually, there is talk about binding affinities, interactions, activities and references to protein products, macromolecule complexes, and often something to do with a reporter gene. Mentions of methods such as co-purification in the abstract would not be present in population genetics publications.

In contrast in the first paper, you see references to associations, selection, which is an evolutionary genetic parameter, and statistics such as Fst. Basically, the authors are focused on patterns of variation over time and space, and **genetic markers are the currency they are using to perform the book-keeping**. When scholars focus on phylogenetic questions there is even less attention to genomic functional detail than ifthey are curious about population genetic parameters (e.g., genic vs. inter-genic). The molecular data are inputs into methods of inference, not the ends in and of themselves.

Obviously, this semantical issue is not a big deal. It’s just I have friends who are molecular biologists…and what they do is very different from what I do.

### Related Posts:

- [Open Thread,
  8/31/2014](https://www.gnxp.com/WordPress/2014/08/31/open-thread-8312014/) - [Biology and the preprint
  revolution](https://www.gnxp.com/WordPress/2014/04/25/biology-and-the-preprint-revolutin/) - [A new evolution
  textbook](https://www.gnxp.com/WordPress/2007/08/10/a-new-evolution-textbook/) - [Types of
  genetics](https://www.gnxp.com/WordPress/2012/04/25/types-of-genetics/) - [Biology is what you study, not how you
  study](https://www.gnxp.com/WordPress/2007/07/10/biology-is-what-you-study-not-how-you-study/) - [A resolution to the molecular clock
  debates?](https://www.gnxp.com/WordPress/2006/10/30/a-resolution-to-the-molecular-clock-debates/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F10%2F04%2Fwhat-is-molecular-biology%2F&linkname=What%20is%20molecular%20biology%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F10%2F04%2Fwhat-is-molecular-biology%2F&linkname=What%20is%20molecular%20biology%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F10%2F04%2Fwhat-is-molecular-biology%2F&linkname=What%20is%20molecular%20biology%3F "Email")[](https://www.addtoany.com/share)
