+++
title = "Genetic distinctiveness"
full_title = "Genetic distinctiveness vs genetic diversity"
date = "2011-11-28"
upstream_url = "https://www.gnxp.com/WordPress/2011/11/28/genetic-distinctiveness-vs-genetic-diversity/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/11/28/genetic-distinctiveness-vs-genetic-diversity/).

Genetic distinctiveness vs. genetic diversity

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/220px-Agueybana.jpg?resize=220%2C165)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/220px-Agueybana.jpg?resize=220%2C165)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/220px-Agueybana.jpg)  
*Meeting the Taino*

In the comments below a few days ago someone expressed concern at the diminishing of genetic diversity due to the disappearance of indigenous populations. My response was bascally that **it depends.** The issue here is whether that disappearance is due to assimilation, or extinction. If a given population is genetically absorbed into another, obviously their genetic diversity is by and large maintained. What disappears are the specific genotypes, the combinations of gene pairs, which are distinctive to that given group. This is the same dynamic at the heart of the [‘disappearing blonde gene’](https://en.wikipedia.org/wiki/Disappearing_blonde_gene) meme. Unless there is selection at the loci which encode or predispose one to blonde hair the ‘gene’ isn’t going anywhere. Rather, the implicit issue here is that blonde people are intermarrying with non-blonde people, and if the genetic variant has a recessive expression then the frequency of the trait will decrease. Populations with a high degree of homozygosity at the ‘blonde loci’ are distinctive in a very particular manner, but they’re no more or less ‘diverse’ than other populations which don’t manifest the same tendency.

A toy example will suffice. Take two populations, A and B, and one locus, 1, with two variants, X and x. Assume that the two populations are the same size. At locus 1 population A is 100% X, and population B is 100% x. In a diploid scenario then all the individuals in population A will be XX, and in B will be xx. When you add A + B you get a frequency of X of 0.5, and of x of 0.5 (since the two populations are balanced in size).



Now imagine a scenario where all individuals in population A pair up with someone in population B (assume sex balance in both populations). In the first generation, F1, all the offspring will be heterozygote Xx (hybrids). The frequency of X and x will be 0.5 still, as in the previous generation. But no individual now reflects the genotype of the parental populations, as all individuals are heterozgyote. At the level of alleles, specific genetic variants, you’ve go the same diversity (X and x at locus 1). But at the level of genotype there’s a huge shift. Two genotypes (XX and xx) no longer exist, but a novel one is now fixed in the population (Xy).

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/401px-Lesley-AnnBrandt.jpg?resize=250%2C266)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/401px-Lesley-AnnBrandt.jpg?resize=250%2C266)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/401px-Lesley-AnnBrandt.jpg)  
*A novel combination*

Finally, in the F2 generation, the offspring of F1, [Hardy-Weinberg](https://en.wikipedia.org/wiki/Hardy%E2%80%93Weinberg_principle) will reassert itself. 25% of the genotypes will be XX, 25% xx, and 50% Xx, due to p² + 2pq + q² = 1. In this scenario some of the distinctiveness of the parental and F1 generations in terms of genotype are evident, but the diversity in the allelic sense of the parental and F1 states remains the same, X = 0.5 and x = 0.5. Observe that if you’re looking at genotypic diversity the F2 generations are actually more diverse than the parental (because Xy is a different genotype). **In other words, in some ways the aggregation of various distinct populations may increase diversity by generating novel combinations.**

This is not to deny that a very specific historically contingent form of diversity in terms of distinctness of particular groups is threatened today. That’s why it was important that the [HGDP](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project) was overloaded with threatened groups like the Bushmen, Kalash, and Pygmies. These populations may be assimilated soon, and with that assimilation it will be more difficult to extract out historically very important information which will inform us about the human past.

But another issue is extinction instead of assimilation. Wouldn’t this eliminate a lot of genetic variation? Perhaps. I actually considered this issue a few years back with the *Star Trek* reboot. If you haven’t watched the film, there’s a major spoiler next. So basically on the order of \~10,000 Vulcans survived the destruction of their planet. Culturally the preservation was rather good, because the Vulcan elders, who are the repositories of the culture, were saved. In this way a fully fleshed Vulcan culture could easily reemerge out of the genocide. On the other hand, the vast majority of Vulcans died. Isn’t ths population bottleneck a genetic catastrophe? **It depends.** If the Vulcans who survived are a relatively random assortment of the population genetically, then the disaster isn’t that bad in terms of genetic diversity.

To get some idea of why, consider the statistic of heterozygosity. This measures the extent of heterozygote states, where the two gene copies differ at a locus, across the population. It’s a proxy for genetic diversity, as more allelic diversity produces more heterozygosity.

The decay of heterozygosity over time due to random genetic drift (without mutation) can be modeled like so:

Ht = H0(1 – 1/(2N))^(t)

The variable “t” is simply the generation time, from an initial time. H0 refers to the initial heterozygosity, and Ht is simply the value at a given time out from that initial value. The N is effective population size. This formula can be used to model population bottlenecks. The Vulcan population reduction from one on the order of billions to 10,000 was basically a massive population bottleneck. The decrease in heterozygosity that you’d expect would be:

Ht = (1 – 1/(2\*10,000))¹

Ht = 0.99995 of the initial value. Basically almost nothing. Why? Because 10,000 turns out to be a relatively large population. This makes some intuitive sense. If you have a sample size of 10,000, and it’s representative, sample variance isn’t going to be that high. If you have an infinite number of coin flips so that the ratio of heads and tails is 50:50, reducing that to 10,000 flips isn’t going to result in much of a deviation from 50:50.

Let’s look at the effect of population bottlenecks of 20 generations at various values of N. The x axis shows generation time, while the y axis illustrates the proportion of the initial heterozyosity which remains.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/hetero.png?resize=497%2C622)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/hetero.png?resize=497%2C622)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/11/hetero.png)

This is not to downplay the impact of bottlenecks and demographic stochasticity. Rather, it’s to suggestion that population genetic diversity is relatively resistant to a crash in numbers. The extinction of small tribal groups is a tragedy, but genetically it may not be as much of a problem as we think. Even in groups such as the Bushmen with a great deal of genetic diversity it is likely that most of that diversity is [already found within non-Bushmen](http://blogs.discovermagazine.com/gnxp/2011/11/the-hunter-gatherers-within-us/) populations.

*Image credits: Ian Beatty and Lesley-Ann Brandt.*

### Related Posts:

- [Is humanity's genetic diversity on the
  decline?](https://www.gnxp.com/WordPress/2007/08/08/is-humanitys-genetic-diversity-on-the-decline/) - [Brownology](https://www.gnxp.com/WordPress/2006/12/22/brownology/) - [An ontology of genetic
  diversity](https://www.gnxp.com/WordPress/2012/09/13/an-ontology-of-genetic-diversity/) - [A little bit of the Others in
  Us](https://www.gnxp.com/WordPress/2010/03/17/a-little-bit-of-the-others-in-us/) - [Ashkenazi Jews are more European in
  ancestry](https://www.gnxp.com/WordPress/2009/09/04/ashkenazi-jews-are-more-european-in-ancestry/) - [Surname & common paternal descent in
  England](https://www.gnxp.com/WordPress/2009/02/10/surname-common-paternal-descent-in-england/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F11%2F28%2Fgenetic-distinctiveness-vs-genetic-diversity%2F&linkname=Genetic%20distinctiveness%20vs.%20genetic%20diversity "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F11%2F28%2Fgenetic-distinctiveness-vs-genetic-diversity%2F&linkname=Genetic%20distinctiveness%20vs.%20genetic%20diversity "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F11%2F28%2Fgenetic-distinctiveness-vs-genetic-diversity%2F&linkname=Genetic%20distinctiveness%20vs.%20genetic%20diversity "Email")[](https://www.addtoany.com/share)
