+++
title = "The Y chromosomal sons"
full_title = "The Y chromosomal sons of the conquerors?"
date = "2015-01-14"
upstream_url = "https://www.gnxp.com/WordPress/2015/01/14/the-y-chromosomal-sons-of-the-conquerors/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/01/14/the-y-chromosomal-sons-of-the-conquerors/).

The Y chromosomal sons of the conquerors?

![**Citation:** Y-chromosome descent clusters and male differential reproductive success: young lineage expansions dominate Asian pastoral nomadic populations](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/conq1.jpg?resize=600%2C289)

[![Balaresque_FiguresRevised251114 copy](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/conq2-300x264.jpg?resize=300%2C264)![Balaresque_FiguresRevised251114 copy](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/conq2-300x264.jpg?resize=300%2C264)](http://www.nature.com/ejhg/journal/vaop/ncurrent/full/ejhg2014285a.html)When it comes to human evolutionary genetics there are two broad areas of interest for me. One the one hand there are classic questions of functional biology and population genetics. Variation of traits and how that variation was selected for over time and space. Then there are the issues of demography, phylogeography, and phylogenetics. This is the domain under which “historical population genetics” tends to fall. Between 1995 and 2005 there was a significant period when the focus was on reconstructing phylogenetic trees inferred from uniparental maternal (mtDNA) and paternal (Y chromosomal) lineages. Using a coalescent framework these non-recombining regions generated intuitively appealing and computationally tractable trees, which illustrated relationships across history. These were often superimposed upon geographical maps to reconstruct patterns of the past. [![The_Journey_of_Man\_-\_A_Genetic_Odyssey](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/The_Journey_of_Man_-_A_Genetic_Odyssey.jpg?resize=197%2C299)![The_Journey_of_Man\_-\_A_Genetic_Odyssey](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/The_Journey_of_Man_-_A_Genetic_Odyssey.jpg?resize=197%2C299)](https://www.amazon.com/exec/obidos/ASIN/0812971469/geneexpressio-20) Since 2005 the emergence of dense SNP chips, where individuals could be typed on hundreds of thousands of markers, ushered in a new era and uniparental studies faded somewhat into the backdrop (and today we are moving into whole genome analyses). But sometimes the uniparental research is still useful, in particular since there is already a huge databank of samples and studies which one can leverage. A new paper in *The European Journal of Human Genetics* does just that, [Y-chromosome descent clusters and male differential reproductive success: young lineage expansions dominate Asian pastoral nomadic populations](http://www.nature.com/ejhg/journal/vaop/ncurrent/full/ejhg2014285a.html).The figure at the top of this post is a summary of the primary results, which show how extremely common Y chromosomal haplogroups in their data set can be correlated with particular historical events. **The authors used a data set of over 5,000 males across a huge range of Eurasian populations**. Surveying the genetic variation it is clear that the haplogroup counts exhibited an exponential distribution. Many of the genotypes were found in only a few individuals, **but a few were found in many individuals**.

[![510CbnsBGLL.\_SY344_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/510CbnsBGLL._SY344_BO1204203200_.jpg?resize=241%2C346)![510CbnsBGLL.\_SY344_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/510CbnsBGLL._SY344_BO1204203200_.jpg?resize=241%2C346)](https://www.amazon.com/exec/obidos/ASIN/1405189088/geneexpressio-20)The authors refer to the haplogroups as “Descent Clusters” (DC) rather than haplogroups. You can see what the DCs are in the table at the top. DC2 is the familiar haplogroup R1a1a, of which I am a member. DC1 is the [“Genghis Khan”](http://www.unz.com/gnxp/1-in-200-men-direct-descendants-of-genghis-khan/) haplogroup. Because they’re using fast mutating microsatellites the coalescence estimates have wide intervals. But, **I am nearly 100% sure that R1a1a coalesces to a period more recently than 10,000 years ago in the past.** The reason is that I saw some posters using whole genome sequences from the Y chromosome at ASHG.These should be more precise estimates because of the enormous marker set of more slowly mutating SNPs, and they too arrived at a relatively recent period for the last common ancestor of these common male lineages. In fact, if I recall correctly the divergence between R1b and R1a dates to \~10,000 years before the present in these studies, so R1a must have a much more recent coalescence. The TMRCA for the R1a1a expansion is suspiciously close to the most recent paper on the emergence of South Asians from an admixture between an indigenous group and West Eurasians to come out of the Reich lab, [Genetic Evidence for Recent Population Mixture in India](http://www.sciencedirect.com/science/article/pii/S0002929713003248). But, even in this paper there is evidence of distinct inputs of Y chromosomes from the west into South Asia, so I suspect it too supports the proportion that the admixture between West Eurasian and indigenous groups occurred between separate and diverse West Eurasians, and not just one group (i.e., the Indo-Aryans may have been the last West Eurasians who arrived in rapid succession over the period between 3000 and 1000 BC). These results also seem to support the conjecture that the ancestors of “Austro-Asiatics” ranged far and wide.

![R1a1a resplendent](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/sonofconquerors.jpg?resize=95%2C125)

In the ultimate evaluation I am less interested in the **specific stories than in the general one.** Is this patternof “super-male” lineages new? The “Altaic” DCs clearly are associated with the Turks and Mongols, and emerged in the light of history. R1a1a and its cousins are older, and live in the shadowy zone of archaeology on the precipice of history. But is this pattern primal to our lineage? My own conjecture is that on the whole this pattern was prefigured in the ancient past whenever founder events occurred. For example, in the expansion into Oceania and the New World. **But what is different about the world *after* the Neolithic is that periodically the tree of patrilineages was “pruned”, as one branch would rise to rule them all for a moment.**There would be an elimination of numerous ancient lineages as a new shining star would dominate the firmament. But the echoes of that moment reverberate downthe millennia, as one can see in the haplogroups which are prevalent across vast swaths of Eurasia, and at a frequency far out of proportion to the norm. Like a thunderbolt, demographic revolutions explode onto the human cultural landscape, and reshape the future topology of lineages on a regular basis.

### Related Posts:

- [Single markers tell you only a bit about individual
  ancestry](https://www.gnxp.com/WordPress/2015/01/22/single-markers-tell-you-only-a-bit-about-individual-ancestry/) - [On that homogeneous
  Y](https://www.gnxp.com/WordPress/2014/01/13/on-that-homogeneous-y/) - [The men of
  Africa](https://www.gnxp.com/WordPress/2011/04/06/the-men-of-africa/) - [The geography of genes tells us only so much about
  history](https://www.gnxp.com/WordPress/2011/08/24/the-geography-of-genes-tells-us-only-so-much-about-history/) - [Surfing in from the
  east](https://www.gnxp.com/WordPress/2014/06/05/surfing-in-from-the-east/) - [Patterns in the Y chromosome and Holocene
  expansion](https://www.gnxp.com/WordPress/2019/11/29/patterns-in-the-y-chromosome-and-holocene-expansion/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F01%2F14%2Fthe-y-chromosomal-sons-of-the-conquerors%2F&linkname=The%20Y%20chromosomal%20sons%20of%20the%20conquerors%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F01%2F14%2Fthe-y-chromosomal-sons-of-the-conquerors%2F&linkname=The%20Y%20chromosomal%20sons%20of%20the%20conquerors%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F01%2F14%2Fthe-y-chromosomal-sons-of-the-conquerors%2F&linkname=The%20Y%20chromosomal%20sons%20of%20the%20conquerors%3F "Email")[](https://www.addtoany.com/share)
