+++
title = "Recombination also"
full_title = "Recombination also evolves"
date = "2015-01-18"
upstream_url = "https://www.gnxp.com/WordPress/2015/01/18/recombination-also-evolves/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/01/18/recombination-also-evolves/).

Recombination also evolves

[![recomb2](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/recomb2-300x281.jpg?resize=300%2C281)![recomb2](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/01/recomb2-300x281.jpg?resize=300%2C281)](http://biorxiv.org/content/early/2015/01/13/013755)I’m someone who until a few years ago thought of recombination as a pretty boring and static evolutionary genetic parameter. Then I went to a talk by [John Novembre](http://jnpopgen.org/) which reported on variation between human *populations* in patterns of recombination (in particular, differences in “hotspots”). For a quick review, recombination is important for two primary reasons. One is molecular genetic, insofar as it seems to have structural value for meiotic process and DNA repair. No recombination is generally not good. Second, recombination maintains the [law of independent assortment](https://en.wikipedia.org/wiki/Mendelian_inheritance#Law_of_Independent_Assortment_.28the_.22Second_Law.22.29) of traits even on the same chromosome, because over time even nearby genes will be uncoupled in their inheritance due to [crossing over](https://en.wikipedia.org/wiki/Chromosomal_crossover). From an evolutionary perspective this is important because in this way “good” and “bad” alleles can be decoupled from other other. Recombination is basically a way to enhance the ability of sex to mix and match variation.

[Graham Coop](http://www.sciencemag.org/content/319/5868/1395.short) revealed patterns of variation among individuals years ago. For example, it is from Graham’s work that I came to understand to recombination is less common in sperm than in eggs, ergo, you’ll have more variance in genomic contributions from paternal than maternal grandparents. Recently at [BAPG XI](http://www.unz.com/gnxp/bay-area-population-genomics-meeting-xi-december-6th/) [Laurie Stevison](https://scholar.google.com/citations?user=JsF2bv4AAAAJ) presented work reveal patterns of recombination variation, and the role of [PRDM9](https://en.wikipedia.org/wiki/PRDM9), across great ape lineages. I tweeted some of the results out, but there were a lot of them. I found the talk interesting, but difficult to take in because there was so much. Now Stevison has put out a preprint, [The Time-Scale of Recombination Rate Evolution in Great Apes](http://biorxiv.org/content/biorxiv/early/2015/01/13/013755.full.pdf), and I feel somewhat the same about it. There’s lots of good stuff, but unless you are steeped in this domain it is somewhat difficult to parse it and tease out distinct threads coherently. But, as you can tell from the figure at the top of this post **changes in patterns of recombination vary as a linear function of genetic divergence.** Some of this stands to reason as the karyotypes of great apes differ. And yet even taking this into account it seems there are differences in patterns such as skew of recombination across the genome (e.g., \~75% of the recombination in human genomes occurs on \~20% of the sequence, with enrichment around telomeres, and very little around centromeres). Looking over Stevison’s preprint I have to wonder as to the role of quality of data in some of the results. Genetic maps are hard to get in some populations, and the ones floating around are not always good. The big takeaway of note for me is that **though there is lots of variation in fine scale recombination patterns, there are some broad constraints.** That makes sense when you note that there are structural/mechanistic reasons for recombination rooted in the nature of meiosis. It’s not a totally neutral parameter which can explore the full space of possibilities. But, in this context obviously the variation in hotspots shows that there are different ways to skin this cat.

Finally, there’s one issue that jumped out at me, and that is they found that “European human population presents the strongest hotspot usage across the genome.” This aligns with earlier work. But I wonder how much of this tendency to find uniqueness in Europeans is due to the enormous amount of genomic resources available for this population. It’s also intriguing in light of the evidence that the [European mutation spectrum is different](http://www.unz.com/gnxp/europeans-mutate-differently/).

In any case, I think everyone should read this preprint several times. I know I’m going to.

### Related Posts:

- [The hotspot
  paradox](https://www.gnxp.com/WordPress/2006/10/08/the-hotspot-paradox/) - [More on recombination & natural
  selection](https://www.gnxp.com/WordPress/2010/03/28/more-on-recombination-natural-selection/) - [Recombination week at
  Science](https://www.gnxp.com/WordPress/2008/01/31/recombination-week-at-science/) - [Resolving the hotspot
  paradox](https://www.gnxp.com/WordPress/2007/02/01/resolving-the-hotspot-paradox/) - [Alu and chimpanzees &
  humans](https://www.gnxp.com/WordPress/2007/10/18/alu-and-chimpanzees-humans/) - [Natural selection and
  recombination](https://www.gnxp.com/WordPress/2010/03/28/natural-selection-and-recombination/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F01%2F18%2Frecombination-also-evolves%2F&linkname=Recombination%20also%20evolves "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F01%2F18%2Frecombination-also-evolves%2F&linkname=Recombination%20also%20evolves "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F01%2F18%2Frecombination-also-evolves%2F&linkname=Recombination%20also%20evolves "Email")[](https://www.addtoany.com/share)
