+++
title = "Neandertal DNA, take II"
full_title = "Neandertal DNA, take II"
date = "2006-11-16"
upstream_url = "https://www.gnxp.com/WordPress/2006/11/16/neandertal-dna-take-ii/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/11/16/neandertal-dna-take-ii/).

Neandertal DNA, take II

As [promised](https://www.gnxp.com/blog/2006/11/neandertal-genome-in-two-years.php), this week’s Science has an [article](http://www.sciencemag.org/cgi/content/abstract/314/5802/1113) from Eddy Rubin’s group reporting 65 kb of Neandertal DNA sequence. It’s not the million bases reported in Nature, but it’s nothing to sneeze at, either.

In terms of admixture, they find no evidence for it, though they can’t rule out low levels:

> \[T\]he maximum likelihood estimate for the Neanderthal contribution to modern genetic diversity is zero. However, the 95% CI for this estimate ranges from 0 to 20%, so a definitive answer to the admixture question will require additional Neanderthal sequence data

This group is taking a different approach to their sequencing– since the method they use generates data more slowly, they’re not looking to get an entire genome anytime soon. Instead, they’re going to look at loci of particular interest for human evolution:

> Based on these results, we attempted to recover specific Neanderthal > sequences from library NE1. We focused on recovering sequences that we > had previously identified by shotgun sequencing because of the low > complexity of library NE1, and were able to recover 29 of 35 sequences > we targeted (table S4). The authenticity of these sequences was > confirmed by the presence of library vector sequences in the reads. > Our success in recovering both previously unknown cave bear and known > Neanderthal genomic sequences using direct genomic selection indicates > that this is a feasible strategy for purifying specific cloned > Neanderthal sequences out of a high background of Neanderthal and > contaminating microbial DNA. This raises the possibility that, > **should multiple Neanderthal metagenomic libraries be constructed from independent samples, direct selection could be used to recover Neanderthal sequences from several individuals to obtain and confirm important human-specific and Neanderthal-specific substitutions**.

So once again, not a whole lot in terms of actual biology (though perhaps the anthropologists will take issue with that statement), but the proof of principle is there– expect some big papers on “sexy” genes like ASPM, microencephalin, and FOXP2 in the coming years.

### Related Posts:

- [A Neandertal genome in two
  years?](https://www.gnxp.com/WordPress/2006/11/15/a-neandertal-genome-in-two-years/) - [Neandertals, rare big
  animals](https://www.gnxp.com/WordPress/2009/07/18/neandertals-rare-big-animals/) - [Arrays on the way
  out?](https://www.gnxp.com/WordPress/2007/05/17/arrays-on-the-way-out/) - [Are you a
  caveman?](https://www.gnxp.com/WordPress/2011/12/15/are-you-a-caveman/) - [Ancient DNA analyses are
  accurate](https://www.gnxp.com/WordPress/2015/05/18/ancient-dna-analyses-are-accurate/) - [FOXP2 & Neandertals; a
  reprise](https://www.gnxp.com/WordPress/2008/04/15/foxp2-neandertals-a-reprise/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F11%2F16%2Fneandertal-dna-take-ii%2F&linkname=Neandertal%20DNA%2C%20take%20II "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F11%2F16%2Fneandertal-dna-take-ii%2F&linkname=Neandertal%20DNA%2C%20take%20II "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F11%2F16%2Fneandertal-dna-take-ii%2F&linkname=Neandertal%20DNA%2C%20take%20II "Email")[](https://www.addtoany.com/share)
