+++
title = "2sa little interbreeding"
full_title = "2sa little interbreeding goes a long way"
date = "2006-11-10"
upstream_url = "https://www.gnxp.com/WordPress/2006/11/10/2s-a-little-interbreeding-goes-a-long-way/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/11/10/2s-a-little-interbreeding-goes-a-long-way/).

2s-a little interbreeding goes a long way

Yesterday Michael Blowhard enthusiastically [linked](http://www.2blowhards.com/archives/2006/11/interbreeding.html#003530) to the recent Neandertal introgression story, and a reader commented:

> Don’t bet on it, Michael. **Paleoarchaeology postdoc.** and regular Querencia reader Laura wrote to me off- blog:  
> “Saw your blog and the mention of the neanderthal interbreeding > article last week…it’s all fluff, published by one of the two main > proponents of the PC theory that neanderthals were just like us, blah > blah blah….I don’t exclude the possibility that they did interbreed, but so far there isn’t any convincing evidence. **And if they did, it would have been on such a small scale that we won’t see it genetically or physically (IMHO)**.”  
> She also sent a couple of PDF’s on Neanderthal DNA. I may post later….

1\) She was responding to earlier [morphology papers](https://stephenbodio.blogspot.com/2006/10/h-neanderthalensis-h-sapiens.html). I have made it pretty clear that I suspect that the papers from Trinkaus and his fellow travelers making a case for interbreeding based on anatomy and fossil evidence were timed to come right before the Lahn, and likely Paabo, papers which would add genetic data to the mix.  
2) The correspondent is obviously an intelligent person with a professional background in paleoanthropology, but, with all due respect, there is a serious problem with the appeal to authority here: **she just don’t know evolutionary population genetics**. My post yesterday, Introgression vs. gene flow was designed to make explicit the varieties of genetic admixture and their relevance for the evolutionary path of a lineage, but, I think since people are still offering “humble opinions” informed by genetic intuition, a review of a very basic model might be helpful.

  
OK. Imagine two populations, let’s assumed some simplifying parameters like fixed size & discrete generations, etc, etc….  
Pop A = 990 individuals  
Pop B = 10 individuals  
Let’s mix them together, a random mating situation  
Pop A + B = 1000 individuals  
Let’s assume that on gene Z population A was fixed for allele Z1 and population be was fixed for allele Z2. In other words, the genetic character of the populations was disjoint on that locus. In generation 1 after the mixing 1% of the alleles within the population will be Z2 and 99% will be Z1. You know that **random genetic drift tends to result in one or another allele fixing within a population, that is, going to 100% frequency**. What are the chances for Z1 and Z2 from generation 1 of A + B? One assumes intuitively that the more numerous allele has a greater chance of fixing, and that is correct.  
The chance of Z1 fixing is 99%  
The chance of Z2 fixing is 1%  
That is, the chance of an allele from generation 1 fixing in the population is simply its frequency within the population. A *de novo* mutation has a chance of fixation inversely proportional to the population size. That is, 1/(2N) in a diploid species, and 1/N in a haploid species.  
The key is that if there is a minimal level of admixture in an ancestral population, then looking at one locus may not be very informative because drift tends to eliminate most lineages, and the numbers are already stacked against rare alleles. Over time **all gene lineages will go extinct**. Each generation is descended from the subset of the previous generation, so genetic variation must always be replenished by mutation.  
Nevertheless, the scenario above was one of **selective neutrality**. That is, the main factor effecting chance of fixation was population size. The smaller population had a much higher chance of having their genetic information lost on any given locus (though over the whole genome there should remain about \~1% contribution of information). A great problem with mtDNA studies, or even Y studies, is that they are one locus of information. They are informative, but it is problematic when some confuse the history of genes with the history of populations. Genes can illuminate the shape of the history of populations, but they do not perfectly track those populations because genes and populations are subject to different dynamics, and are different orders of organization.  
So let’s introduce selection into the equation. Consider the same populations. Now, consider that allele Z2 has a 1% selection coefficient vs. Z1, that this, *ceteris paribus* an individual carrying Z2 is 1% more reproductively fit than one carrying Z1. Imagine that now instead of one generation of random mating an individual from population B, carrying Z2, is introduced into population A. Space these introductions out. **What are the chances that Z2 will replace Z1?** Remember, in the earlier scenario there is only a 1% chance, to each according to their numbers. Not so here.  
I will model not the chance of fixation, but extinction, since it makes the math simpler. The chance of fixation of a new mutation in a large population is 2*s*. The introduction of Z2 via one isolated mating is operationally the same as a mutation. **So each introduction of a new individual from popuation B into population A is associated with a 2*s* probability of fixation of allele Z2 against the genetic background of population A**. The probability of extinction is 1 – 2*s*, so, let’s use a selection coefficient of 0.01, 1%. That means:  
– 1 introduction of a Z2 allele into the population, A, which is fixed for Z1, has a 98% chance of maintaining Z1 as fixed.  
– The introduction of the new allele and its subsequent extinction can be considered an independent “event” from other introductions and extinctions  
– **Assuming independence, what the chances of 10 straight events resulting in extinction?**  
I put it this way because biologically once Z2 is fixed, subsequent Z2 introductions are irrelevant genetically. The question I’m asking is, what is the chance that even after 10 introductions Z2 disappears from the population? Since they are independent events you simply multiply out like so:  
(1 – 2\**0.01*)¹⁰ \~ 82% chance of extinction.  
Compare this to the first example with a neutral allele: there was a 99% chance of extinction! The introduction of even a moderate selection coefficient of 1% increased the chance of fixation by over an order of magnitude. If I increase the selection coefficient to 10%, there is only a 10% chance of extinction of Z2 after each and every introduction.  
Now, the model above is simplified, but I wanted to mainly get across the logic: **loci subject to selection have different evolutionary dynamics than those subject to neutral processes, and, one can not infer across these two situations as if all things are equal**. In the second situation, what if the selection coefficient of the minority allele is 10%? It is highly likely that it will “jump” and fix in the new population, but, that does not mean that **the genetic background is any different, one can not infer from one locus to the whole genome**. Of course, one assumes that there will be [linkage disequilibrium](https://en.wikipedia.org/wiki/Linkage_disequilibrium) around the locus of selection, but over time recombination will break apart genetic correlations across loci and selected locus will have had a very different evolutionary history than the other modal loci within the population. In short: **Neandertal introgression is *less* a story of evolutionary demographics than it is one of evolutionary genetics**. The demographic, sociological and cultural impact of Neandertals and other archaics might have been trivial and of little note, but their evolutionary **genetic** impact could still remain enormous because of the winds of selection.

### Related Posts:

- [Neandertals & introgression
- the past
  week....](https://www.gnxp.com/WordPress/2006/11/14/neandertals-introgression-the-past-week/) - [Neandertal introgression &
  SEED](https://www.gnxp.com/WordPress/2006/11/10/neandertal-introgression-seed/) - [Neandertal & H. sapiens sapiens
  interbreeding](https://www.gnxp.com/WordPress/2006/11/01/neandertal-h-sapiens-sapiens-interbreeding/) - [Neandertal & H. sapiens sapiens
  interbreeding](https://www.gnxp.com/WordPress/2006/10/31/neandertal-h-sapiens-sapiens-interbreeding/) - [Neandertal autosomal sequence goes
  live](https://www.gnxp.com/WordPress/2006/07/20/neandertal-autosomal-sequence-goes-live/) - [Neandertal autosomal sequence goes
  live](https://www.gnxp.com/WordPress/2006/07/20/neandertal-autosomal-sequence-goes-live/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F11%2F10%2F2s-a-little-interbreeding-goes-a-long-way%2F&linkname=2s-a%20little%20interbreeding%20goes%20a%20long%20way "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F11%2F10%2F2s-a-little-interbreeding-goes-a-long-way%2F&linkname=2s-a%20little%20interbreeding%20goes%20a%20long%20way "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F11%2F10%2F2s-a-little-interbreeding-goes-a-long-way%2F&linkname=2s-a%20little%20interbreeding%20goes%20a%20long%20way "Email")[](https://www.addtoany.com/share)
