+++
title = "Do you want your"
full_title = "Do you want your genotype in a public data set?"
date = "2013-01-16"
upstream_url = "https://www.gnxp.com/WordPress/2013/01/16/do-you-want-your-genotype-in-a-public-data-set/"

+++
Source: [here](https://www.gnxp.com/WordPress/2013/01/16/do-you-want-your-genotype-in-a-public-data-set/).

Do you want your genotype in a public data set?

In the near future one of my projects is revising and expanding the “PHYLO” pedigree file which I put up a week ago. Basically I want there to be a public data set which has a modest number of SNPs useful for phylogenetic analysis (100-200,000) with a wide population coverage. Additionally, I am going to do a few things like rename the family ids to populations, and also release it with scripts to help in running Admixture (for example, shell scripts which will automate replication and later analysis of replicates). Finally, I’m planning on running \~50 replicates of K = 2 to K = 20 with 10-fold cross-validation (yes, this is will take a while) to get a good sense of the “best” K’s. The reality is that most people probably are only interested in the “most informative” K, +/- 1, so there’s no need for everyone to run K = 2 to K = 20. The time saved should be used on running replicates, and then [CLUMPP](http://www.stanford.edu/group/rosenberglab/clumpp.html) to merge the results.

I would say that this is for ‘amateurs’ only, but I don’t think it’s betraying confidence to observe that several academic researchers at prominent institutions have ended up inquiring of me of how to get good public data sets. This sort of information still hasn’t percolated to the general public, including scientists who don’t work on population genomics. After a few trial runs with public data sets people with academic access could move to things like the POPRES data set.

But the ultimate point of this post is to ask: do you want to be in this data set? If so, I need the file (23andMe format is fine, otherwise, pedigree files only), your name, and some minimal ethnic information. **I’m not going to add everyone**. I just want to diversify the public data set a little. But I am going to put names in the sample sheet, so you won’t have anonymity. As you know I don’t particular care about this personally, but your mileage may vary. Researchers might need to contact or check that people are who they are.

Email: contactgnxp -at- gmail -dot- com

### Related Posts:

- [A note on open
  genomics](https://www.gnxp.com/WordPress/2012/08/28/a-note-on-open-genomics/) - [Lots and lots of genotype
  data!](https://www.gnxp.com/WordPress/2013/01/09/lots-and-lots-of-genotype-data/) - [Using your 23andMe data in
  Plink](https://www.gnxp.com/WordPress/2013/01/07/using-your-23andme-data-in-plink/) - [Back to crunching personal genomic
  data](https://www.gnxp.com/WordPress/2012/11/22/back-to-crunching-personal-genomic-data/) - [1000 Genomes data, 290,000
  SNPs](https://www.gnxp.com/WordPress/2015/09/30/1000-genomes-data-290000-snps/) - [Confidence in inference in phylogenetic data
  sets](https://www.gnxp.com/WordPress/2013/03/03/confidence-in-inference-in-phylogenetic-data-sets/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F01%2F16%2Fdo-you-want-your-genotype-in-a-public-data-set%2F&linkname=Do%20you%20want%20your%20genotype%20in%20a%20public%20data%20set%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F01%2F16%2Fdo-you-want-your-genotype-in-a-public-data-set%2F&linkname=Do%20you%20want%20your%20genotype%20in%20a%20public%20data%20set%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F01%2F16%2Fdo-you-want-your-genotype-in-a-public-data-set%2F&linkname=Do%20you%20want%20your%20genotype%20in%20a%20public%20data%20set%3F "Email")[](https://www.addtoany.com/share)
