+++
title = "Defining developmental"
full_title = "Defining developmental disorders through genetics"
date = "2010-08-13"
upstream_url = "https://www.gnxp.com/WordPress/2010/08/13/defining-developmental-disorders-through-genetics/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/08/13/defining-developmental-disorders-through-genetics/).

Defining developmental disorders through genetics

To many people, the term “autism” suggests a specific disorder – one with characteristic and recognizable symptoms, presumably reflecting the same underlying cause. In fact, no such disorder exists. Autism refers to a variable spectrum of symptoms – including deficits in social interaction, impaired communication (especially a delay in developing language), narrow, restricted interests and stereotyped behaviours. Any one child who is diagnosed with autism may show only some of these symptoms. There is a wide range of IQ in autism, including very high levels seen in what has been known as Asperger’s syndrome, but the average is about 70. There is also a high incidence of epilepsy (around 10%).

  
Psychiatrists have long recognized this variability and use the term “[autism spectrum disorder](http://www.nichd.nih.gov/health/topics/asd.cfm)” to encompass the entire range. Until recently, with a couple of exceptions, they have not had the means to distinguish different subtypes of autism based on their underlying cause. One of these exceptions, which has been known for some time, is the gene responsible for [Fragile X syndrome](http://www.fragilex.org/html/home.shtml). Routine screening for Fragile X mutations allows pediatric psychiatrists to define up to 5% of autism referrals as arising from this cause. We now know that the symptoms of autism can be caused by a mutation in any of a large number of different genes (or by mutations which affect several adjacent genes on a chromosome).

Screening for the latter type of mutation – deletions or duplications of several genes, collectively called copy number variants – is already being proposed as a routine step of clinical genetic testing in patients presenting with symptoms of autism. These mutations are easy to detect but will probably be responsible for no more than 10% of cases. Point mutations – changes to a single base of DNA – are likely to account for the rest. Fortunately, it is now possible to sequence the entire genome, or at least the entire “exome” – the part of the genome that codes for proteins – in an individual for about 3,000 dollars and in under a week. A far cry from the 3 billion dollars and ten years it took to sequence the first human genome!

This approach has already been used to identify mutations in genes on the X chromosome in autism or schizophrenia cases but can now be extended to the entire genome. It will not always be obvious which mutation is causative in any individual, and we should expect a good deal of complexity due to combinations of mutations, but it should at least be possible to identify a primarily responsible mutation in a large proportion of cases.

The obvious question then is whether mutations in different genes are associated with distinct profiles of symptoms. By grouping together patients with mutations in the same locus, it may be possible to recognise specific profiles of symptoms that are otherwise obscured by variability among carriers and phenotypic overlap with other patients.
This approach has been used very successfully in diagnosing cases of mental retardation, intellectual disability or other forms of developmental delay based on genetic lesion and has led to the identification and clinical characterisation of many new, distinct neurodevelopmental syndromes.

The first study to attempt this in autism spectrum disorder has been published recently by Bruining and colleagues. They looked at cases of autism due to [Klinefelter syndrome](http://www.nichd.nih.gov/health/topics/klinefelter_syndrome.cfm) (caused by an extra X chromosome in males: XXY), deletion in a region of chromosome 22 (22q11) or a group with unknown causes. By analyzing the profile of a list of clinical symptoms across these groups they were able to distinguish the Klinefelter and 22q11 cases from each other and from the cases with unknown etiology. There is still a lot of variability in each type and substantial overlap between them in “clinical symptom space”, but the carriers of specific mutations are significantly more similar in profile to each other than to the general cases.

This knowledge is tremendously useful in several ways. First, it becomes possible to make clinical predictions about an individual’s prognosis, by comparison with other carriers of the same mutation.
Second, it allows prediction of genetic risk to relatives – this can be hugely important to parents of an autistic child who are considering having additional children. Third, identification of the mutated gene is the first step to elucidating the underlying defect at a neurobiological level. Ultimately, this may suggest routes to therapies which are rationally designed and personalised.

The promise of new therapeutics is illustrated by progress in understanding the biology underlying Fragile X syndrome. The fragile X gene encodes a protein, [FMR1](https://en.wikipedia.org/wiki/FMR1), which functions in nerve terminals to hold a set of RNA molecules in a state where they are ready to be translated into protein when the synapse is active. The new proteins are needed when a synapse has to be strengthened after use (a core mechanism of learning). Another protein, a glutamate channel called mGluR1, performs the opposing function – when activated it signals for these mRNAs to be translated. If the FMR1 protein is mutated then the RNA molecules get translated too early.
This effect can be counter-balanced by turning down the activity of the mGluR1 protein. Remarkably, this results in very significant amelioration of the “symptoms” of FMR1 deletion in a mouse model of Fragile X syndrome. These results are so impressive that drugs to block mGluR proteins are now in small-scale clinical trials of human Fragile X patients.

This example illustrates how discovery of the responsible gene and elucidation of its functions at a molecular level can suggest highly specific ways to correct or compensate for the effect of the mutation, specifically in those patients with that lesion. We can expect this kind of approach to be similarly successful in discriminating patients with other disorders such as schizophrenia or epilepsy into genetically distinct subgroups. This promises to radically transform how patients with these diverse symptoms are diagnosed and treated – no longer lumped together into categories of questionable validity and usefulness, but based on their individual genetic profile.

Mirrored from [Wiring the Brain](https://wiringthebrain.blogspot.com/).

Shen, Y., Dies, K., Holm, I., Bridgemohan, C., Sobeih, M., Caronna, E., Miller, K., Frazier, J., Silverstein, I., Picker, J., Weissman, L., Raffalli, P., Jeste, S., Demmer, L., Peters, H., Brewster, S., Kowalczyk, S., Rosen-Sheidley, B., McGowan, C., Duda, A., Lincoln, S., Lowe, K., Schonwald, A., Robbins, M., Hisama, F., Wolff, R., Becker, R., Nasir, R., Urion, D., Milunsky, J., Rappaport, L., Gusella, J., Walsh, C., Wu, B., Miller, D., & , . (2010). Clinical Genetic Testing for Patients With Autism Spectrum Disorders *PEDIATRICS, 125* (4) DOI: [10.1542/peds.2009-1684](”http://dx.doi.org/10.1542/peds.2009-1684″)

Piton, A., Gauthier, J., Hamdan, F., Lafrenière, R., Yang, Y., Henrion, E., Laurent, S., Noreau, A., Thibodeau, P., Karemera, L., Spiegelman, D., Kuku, F., Duguay, J., Destroismaisons, L., Jolivet, P., Côté, M., Lachapelle, K., Diallo, O., Raymond, A., Marineau, C., Champagne, N., Xiong, L., Gaspar, C., Rivière, J., Tarabeux, J., Cossette, P., Krebs, M., Rapoport, J., Addington, A., DeLisi, L., Mottron, L., Joober, R., Fombonne, E., Drapeau, P., & Rouleau, G. (2010). Systematic resequencing of X-chromosome synaptic genes in autism spectrum disorder and schizophrenia *Molecular Psychiatry* DOI: [10.1038/mp.2010.54](”http://dx.doi.org/10.1038/mp.2010.54″)

Bruining H, de Sonneville L, Swaab H, de Jonge M, Kas M, van Engeland H, & Vorstman J (2010). Dissecting the clinical heterogeneity of autism spectrum disorders through defined genotypes. *PloS one, 5* (5) PMID: [20526357](”http://www.ncbi.nlm.nih.gov/pubmed/20526357″)

### Related Posts:

- [Deletions and
  autism](https://www.gnxp.com/WordPress/2008/02/13/deletions-and-autism/) - [Pausing research on autism (for
  now)](https://www.gnxp.com/WordPress/2021/09/29/pausing-research-on-autism-for-now/) - [Increased de novo mutations for older
  fathers](https://www.gnxp.com/WordPress/2015/01/27/increased-de-novo-mutations-for-older-fathers/) - [Autism &
  yawning](https://www.gnxp.com/WordPress/2007/12/24/autism-yawning/) - [Baron-Cohen on
  Autism](https://www.gnxp.com/WordPress/2007/07/12/baron-cohen-on-autism/) - [What’s in a name? Genetic overlap between major
  psychiatric…](https://www.gnxp.com/WordPress/2010/05/26/whats-in-a-name-genetic-overlap-between-major-psychiatric-disorders/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F13%2Fdefining-developmental-disorders-through-genetics%2F&linkname=Defining%20developmental%20disorders%20through%20genetics "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F13%2Fdefining-developmental-disorders-through-genetics%2F&linkname=Defining%20developmental%20disorders%20through%20genetics "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F13%2Fdefining-developmental-disorders-through-genetics%2F&linkname=Defining%20developmental%20disorders%20through%20genetics "Email")[](https://www.addtoany.com/share)
