+++
title = "Natural selection and"
full_title = "Natural selection and recombination"
date = "2010-03-28"
upstream_url = "https://www.gnxp.com/WordPress/2010/03/28/natural-selection-and-recombination/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/03/28/natural-selection-and-recombination/).

Natural selection and recombination

Razib has a [nice discussion](http://blogs.discovermagazine.com/gnxp/2010/03/26/natural-selection-recombination-in-the-human-genome/) of an interesting observation [just published in PLoS Genetics](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1000886)— that there is a negative correlation between recombination rate in the human genome and population differentiation. This observation, along with the complementary observations of correlations [between nucleotide diversity and recombination](http://www.ncbi.nlm.nih.gov/pubmed/19148272?dopt=Abstract) and [between nucleotide diversity and density of functional elements](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1000471), form part of a growing body of literature establishing that the signatures of natural selection–positive and negative–have influenced overall patterns of genetic diversity in humans.

It’s important to emphasize again that these observations are influenced by both positive selection (the removal of genetic diversity at sites linked to advantageous alleles) and background selection (the removal of genetic diversity at sites linked to deleterious alleles). One important question is the relative role of these two forces in generating these overall patterns (the implications for human evolution of extensive positive selection are somewhat different than the implications of extensive negative selection); there are a couple ways forward on addressing this discussed by the authors here.

The authors here also raise the intriguing possibility of leveraging populations which have diverged at different times to examine differences in the efficiency of natural selection over time; they don’t quite have the data to do this yet, but they certainly will in the next couple years. They do make the observation, using admittedly suboptimally ascertained data, that there does appear to be the same qualitative relationship–perhaps even stronger– between recombination rate and differentiation even between very closely related populations like the Chinese and the Japanese; though only suggestive, this raises the possibility that the signatures of selection (again, both positive and negative) are detectable even on a quite short timeframe. Overall, this is an exciting direction for the use of resequencing datasets that will be coming out soon.

Finally, since John Hawks doesn’t have comments, I’ll make a comment on [his post on this paper](http://johnhawks.net/weblog/topics/evolution/selection/acceleration/keinan-and-reich-2010-fst-recombination.html). In particular, based on the observation above (about the relationship between differentiation between closely-related populations and recombination), he writes:

> There are a lot more genes that are geographically circumscribed and > low in frequency affecting FST at a more localized level, and fewer > affecting major allele frequencies between continental regions.

Though this may be true, the correlation between FST and differentiation between closely-related populations observed here is almost certainly not due to any effect of this sort. The data used in the Chinese-Japanese comparison (for example) is from the Affymetrix and Illumina genotyping chips (ie. HapMap 3), which contain mostly common variation and no (or very few) low-frequency SNPs specific to the Japanese (or Chinese). This effect is likely due to small differences in allele frequency between the Chinese and the Japanese at relatively \*common, non-geographically circumscribed\* SNPs. That is, imagine two SNPs, one at 55% frequency in Japan and 50% frequency in the rest of the world, and one at 50% frequency everywhere. Their observation (I think) is that SNPs of the former type are more common in low recombination rate areas of the genome, not that they find a bunch of new alleles that have arisen in the last few thousand years since those populations split. One could double-check this, but based on the chips they used, I’m pretty confident this is the case.

### Related Posts:

- [More on recombination & natural
  selection](https://www.gnxp.com/WordPress/2010/03/28/more-on-recombination-natural-selection/) - [Humans have adapted on genome-wide
  level?](https://www.gnxp.com/WordPress/2009/01/16/humans-have-adapted-on-genome-wide-level/) - [Lewontin's Paradox in the 21st
  century](https://www.gnxp.com/WordPress/2021/02/04/lewontins-paradox-in-the-21st-century/) - [New theory of MHC
  diversity?](https://www.gnxp.com/WordPress/2008/11/06/new-theory-of-mhc-diversity/) - [Natural selection and dopamine receptor
  genes](https://www.gnxp.com/WordPress/2012/03/09/natural-selection-and-dopamine-receptor-genes/) - [What drives accelerated molecular
  evolution?](https://www.gnxp.com/WordPress/2006/10/02/what-drives-accelerated-molecular-evolution/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F03%2F28%2Fnatural-selection-and-recombination%2F&linkname=Natural%20selection%20and%20recombination "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F03%2F28%2Fnatural-selection-and-recombination%2F&linkname=Natural%20selection%20and%20recombination "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F03%2F28%2Fnatural-selection-and-recombination%2F&linkname=Natural%20selection%20and%20recombination "Email")[](https://www.addtoany.com/share)
