+++
title = "Natural selection &"
full_title = "Natural selection & recombination in the human genome"
date = "2010-03-26"
upstream_url = "https://www.gnxp.com/WordPress/2010/03/26/natural-selection-recombination-in-the-human-genome/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/03/26/natural-selection-recombination-in-the-human-genome/).

Natural selection & recombination in the human genome

If you are like me, and if you are reading this weblog there is a significant probability you are like me, you read [L. L. Cavalli-Sforza](https://www.gnxp.com/blog/2006/08/10-questions-for-luigi-luca-cavalli.php)‘s [History and Geography of Human Genes](https://www.amazon.com/exec/obidos/ASIN/0691029059/geneexpressio-20) in the 1990s, and in the early aughts Spencer Wells’ [A Journey of Man](https://www.amazon.com/exec/obidos/ASIN/0812971469/geneexpressio-20). Science has come very far in the last in the last 10-15 years, even Cavalli-Sforza’s magnum opus pales in comparison to the literal tsunami of data and analysis which the “post-genomic era” has ushered in. Instead of a gene here and there, or even the mtDNA and Y chromosome, researchers are now looking at hundreds of thousands of genetic variants, [SNPs](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism), across genomes. We’re rapidly approaching the era of [whole genome sequencing](https://en.wikipedia.org/wiki/Full_genome_sequencing), even if we’re not quite there yet.

But what’s the purpose of advances in technique and computation? Though the long-term project is to understand human variation and genetic function so as to have biomedical utility, in the short-term there is an enormous wealth of more abstract population genetic insight which can be extracted. Because of the biomedical focus of contemporary genomics we take a somewhat anthropocentric view, which is fine by me as I am an unregenerate speciest. The fish, fowl and crawling things of the earth can come later. And in any case, the beauty of the human focus of modern evolutionary genomics is that there are whole disciplines such as paleoanthropology which can serve as partners in interdisciplinary projects.

Humans are like any other organism, buffeted by conventional evolutionary genetic dynamics, [drift](https://en.wikipedia.org/wiki/Genetic_drifthttp://en.wikipedia.org/wiki/Genetic_drift), migration, natural selection, as well as processes which are more biophysically rooted such as [recombination](https://en.wikipedia.org/wiki/Genetic_recombination) and mutation. Each of these processes leave their tell-tale marks on the genome. Mutation replenishes variation which drift and selection often eliminate, the former by chance and the latter in the form of negative selection. Migration serves to homogenize across populations through gene flow, while diversifying within populations by introducing novel variants. Finally, recombination breaks up linear associations of genetic variants along a DNA sequence, and has been used to [explain sex](https://en.wikipedia.org/wiki/Muller's_ratchet).

  
In regards to *H. sapiens* it seems that our recent evolutionary history is dominated by a few big events. Within the last 100,000 years we underwent an extremely rapid population expansion from a small founding group within Africa, and radiated adaptively across all continents except for Antarctica. We are a then a relatively genetically homogeneous population, with much of the extant variation remaining within Africa, and the non-African groups getting progressively less diverse with distance from that continent. Basically a model whereby our species spread across the world via serial founder events. This simple model suffices in the broad sketch, but there is much more to the story. Over the past few years the older idea that current continental populations are the descendants of the first settlers, that is, the first modern humans who displaced the archaic populations which preceded them, seems unlikely to be to totally correct in all cases. It is likely wrong in [Europe](http://scienceblogs.com/gnxp/2010/01/european_man_the_farmer.php) and to some extent [India](http://scienceblogs.com/gnxp/2009/09/south_asians_as_a_hybrid_popul.php), no trivial exceptions.

There is much which can be said about details of demographic history in regards to the possibility of mass migrations, but today I want to focus on another dynamic: **the effect of natural selection on the human genome**. There are some researchers who are very skeptical of the efficacy of selection in shaping the patterns of variation we see, constraining it to a few loci such as that which confers lactase persistence or resistance to malaria. [Others](https://www.amazon.com/exec/obidos/ASIN/0465002218/geneexpressio-20/) feel that selection’s power in shaping the genome is far more pervasive. Finally, there is a [middle path](http://scienceblogs.com/gnxp/2009/06/adaptation_might_not_be_a_sphe.php), which emphasizes a diverse and complex portfolio.

A new paper explores the extent and nature of selection in human genomes through combining a rather old population genetic statistic with new expanded data sets and powerful statistical techniques, [Human Population Differentiation Is Strongly Correlated with Local Recombination Rate](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1000886#pgen-1000886-g004):

> Allele frequency differences across populations can provide valuable > information both for studying population structure and for identifying > loci that have been targets of natural selection. Here, we examine the > relationship between recombination rate and population differentiation > in humans by analyzing two uniformly-ascertained, whole-genome data > sets. We find that population differentiation as assessed by > inter-continental*F*_(ST)shows negative correlation with > recombination rate, with*F*_(ST) reduced by 10% in the tenth of the > genome with the highest recombination rate compared with the tenth of > the genome with the lowest recombination rate (P≪10⁻¹²). This pattern > cannot be explained by the mutagenic properties of recombination and > instead must reflect the impact of selection in the last 100,000 years > since human continental populations split. The correlation between > recombination rate and*F*_(ST) has a qualitatively different > relationship for*F*_(ST) between African and non-African populations > and for*F*_(ST) between European and East Asian populations, > suggesting varying levels or types of selection in different epochs of > human history.

You know of *F*_(ST), even if you don’t know what *F*_(ST) is. You have heard that 15% of the variation in human genes is between races, and 85% within races. That 15% is an *F*_(ST) of 0.15. In other words,*F*_(ST) is a population genetic statistic which partitions the variance in genes between and within populations. If you have two populations and both have allele frequencies of 0.50 for two alleles, A & B, at one locus, then the *F*_(ST) would naturally be 0 as there is no between population difference, you can swap individuals from either group interchangeably for purposes of comparison. In contrast if the frequencies were disjoint so that all individuals in one population were of allele A and all individuals in the other of B, then naturally the *F*_(ST) would be 1, as all the variance is between populations, and all the information you need is found within population substructure. A perfect opportunity for profiling!

But this varies by gene and genomic region. As you know from above most variance is within races, not between them. But for the gene [SLC24A5](https://en.wikipedia.org/wiki/SLC24A5) almost all the variance is between Europeans and Africans, not within them. Similarly, all the variance on this gene is between Europeans and East Asians. Finally, there is no variance between East Asians and Africans on this gene. Why? Because it looks like that this gene has recently increased in frequency in West Eurasia, to the point where a new mutation has replaced the ancestral variant, which is common in Africa and East Eurasia. Additionally, it is notable that this genetic variant seems to account for 30-40% of the skin tone difference between Africans and Europeans. The point is that **total genome variation is not always a good indicator of the evolutionary history of a specific gene**. This is a problem especially in the case of those which we might find of interest.

In the paper above they find that areas of high recombination are negatively correlated with *F*_(ST) on a global scale. In other words, genomic regions which recombine more often across DNA strands and so shuffle genetic variation about and break apart [linear associations](https://en.wikipedia.org/wiki/Haplotype) show lower *F*_(ST) values. Reduced between population variation.

It’s rather clear in their first figure. Before we jump to that, let me note that they’re using the Perlegen data set, which has two dozen African Americans, Chinese and Europeans, respectively, and 1 million SNPs. Each panel has the *F*_(ST) values on the Y axis, and the recombination rate on the X axis.

![reichfig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/03/reichfig1.png?resize=600%2C596)![reichfig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/03/reichfig1.png?resize=600%2C596)

It’s pretty clear what’s going on just through inspection. There’s an average decrease of 4% in *F*_(ST) for every 1 cM increase in recombination rate. The correlation estimates for *F*_(ST) and median recombination rates are for each panel:

A −0.962 (P = 8.9×10−6)  
B −0.815 (P = 0.0041)  
C −0.931 (P = 0.0001)  
D −0.361 (P = 0.306)

The correlation estimates tell you something you can see visually: **there’s a big difference in the relationships contingent upon which populations you’re using to calculate *F*_(ST).** In particular, a lot of the linear relationship between *F*_(ST) and recombination rate is actually due to the African vs. non-African difference. This is not a total surprise, Africans have a lot of genetic variation. In terms of genes one can think of non-Africans as [simply a branch of Africans](http://blogs.discovermagazine.com/gnxp/2010/02/17/what-average-genetic-variation-can-tell-us-or-not/) in many ways. Additionally, there’s been suggestive data for a decade now that when Africans left Africa they were subject to new selection pressures which seem common to Eurasians as a whole (though to be fair these signatures of selective sweeps in Eurasia may simply be false positives generated by population bottlenecks and the like).

Yes, I will work back to selection from genetic variation; the two are related, though the relation can be subtle. So why the negative correlation between *F*_(ST) and recombination? Consider an SNP, a single DNA base pair, which is subject to positive selection. It can increase rapidly in frequency so that it goes from \~0 to \~1 in proportion with the population. Fair enough, but SNPs do not exist in an abstract universe, they’re physically embedded in DNA, and so are flanked by many other bases. If an SNP is subject to powerful directional selection which drives its frequency upward, then adjacent bases also “[hitchhike](https://en.wikipedia.org/wiki/Genetic_hitchhiking)” along in frequency. In other words, powerful directional selection can reorder the variation of whole genomic regions, depending on how powerful the selection is. It can sweep away the noisy scatter of variants introduced over many generations by mutation and replace them with a long sequence of alleles from an ancestral genome which harbored the selected variant. Over time mutation can mask the homogenization as it replenishes variation, but there is also another dynamic which blocks the long march to genetic uniformity: recombination. Recombination can tear apart blocks of alleles as they sweep up in frequency, and the more recombination, the greater the counterforce to the homogenizing power of selection on the local genome as the block is chopped up evermore.

Now consider the nature of selection in different populations. Let us stipulate that the light skin of East Asians and Europeans are adaptations; we [know that they are conferred by different SNPs](http://blogs.discovermagazine.com/gnxp/2010/03/05/oca2-makes-east-asians-white-and-europeans-blue/). In other words, selection operated on different genetic variants to produce the adaptation (though by and large across the same set of genes). Therefore, *F*_(ST) on pigmentation genes is relatively high because of between population difference, and these genes tend to be surrounded by regions of homogenization as they seem to have swept up to high frequency rapidly and dragged along many nearby alleles. Assuming this effect of natural selection equal recombination would naturally tend to work against increased *F*_(ST) by reducing the number of linked alleles being dragged along by breaking apart the genomic blocks along their transient up.

A similar effect occurs in the case of background selection, which is operative around deleterious mutations. When there are alleles which are negatively selected their neighbors are also effected; consider it a sort of population genetic “property value.” Because negative selection tends to reduce effective population size, on a large geographical scale it can result in increased *F*_(ST) (consider the genetic uniqueness of isolated populations which have gone through bottlenecks). Again, recombination can blunt the impact of a deleterious allele on its neighbors. The authors do note that the particular dynamics of positive and background selection differ, the latter is a gentler affair by far, a repetitive tap as opposed to a sledgehammer, but the genomic resolution of their analysis is such that they lack power to explore these differences.

I’ll let the authors describe the peculiarities of the European-Chinese panel:

> …The weaker correlation for the*F*STbetween European and Chinese > populations is driven by a dip in differentiation at very low > recombination rate loci…which is not at all what is seen in the > comparison of African and non-African populations…**This curve shows a > qualitatively non-monotonic pattern**, which motivated us to perform a > quadratic regression fitted within the bootstrapping framework. The > regression is concave and includes very significant linear (P = > 3.0×10−4) as well as quadratic (P = 1.8×10−5) terms. Conversely, > quadratic regression gives a non-significant quadratic term > for*F*STbetween African Americans and each of the other two > populations and if anything is slightly convex. As expected, for > single SNP analysis (without binning by recombination rate), linear > regression is very significant for*F*STbetween African Americans and > either non-African population (P≪10−12). For*F*STbetween Chinese and > Europeans, however, linear regression is not significant (P = 0.81), > while a quadratic regression is very significant (P≪10−12)….

Non-monotic is just a way to say that the trend reverses direction. In other words, the linear model isn’t too good a fit on what’s going on with the variation between Europeans and Chinese, and how it relates to recombination rates. They offer two speculative possibilities for the “inverted U-shaped” nature of the relationship between *F*_(ST) and recombination in Europeans and Chinese. First, the smaller effective population sizes of non-African groups results in greater efficacy of background selection. As random genetic drift tends to increase the frequency of deleterious alleles, powerful negative selection is given opportunity work against that region of the genome. This results in more background selection as adjacent genomic regions are impacted. Because of differing population sizes the balance between positive and background selection is different for Africans and non-Africans. A second hypothesis is that gene flow between the two Eurasian groups allowed for selective sweeps to move from one group to the other. In other words, between population variance can be reduced if a favored alleles spreads across all populations from one original group ([lactase persistence](http://blogs.discovermagazine.com/gnxp/2007/08/09/lactase-persistance-in-eurasia-different-departure-same-destination/) in much of northern Eurasia may be a case of just this).

Intriguingly they found the linear relationship between *F*_(ST) and recombination to be stronger in genomic regions which are coding (i.e., they have genes which code for proteins), and in particular in the X chromosome. The second aligns with other [recent work](http://blogs.discovermagazine.com/gnxp/2010/02/07/x-chromosome-marks-the-spot-again/) which indicates that the X chromosome may be subject to stronger selective pressures than the rest of the genome because of its peculiar expression pattern (males have only one copy of the gene, and females express only one copy per cell due to X chromosome inactivation).

Finally, they replicated their results using other data sets. I’ll just show the figure from HapMap3:  
![journal.pgen.1000886.g004](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/03/journal.pgen.1000886.g004.png?resize=600%2C1128)![journal.pgen.1000886.g004](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/03/journal.pgen.1000886.g004.png?resize=600%2C1128)

I obviously rotated so you could see the labels at higher resolution. Here are the populations:

WAF (“West African”) is a combined sample of YRI (Yoruba in Ibadan, Nigeria) and LWK (Luhya in Webuye, Kenya)

EAS (“East Asia”) is a combined sample of CHB (Han Chinese in Beijing, China), CHD (Chinese in Metropolitan Denver, CO, USA), and JPT (Japanese in Tokyo, Japan)

EUR (“Europe”) is a combined sample of CEU (ancestry from Northern and Western Europe) and TSI (Toscani in Italia)

GIH is a sample of Gujarati Indians in Houston, TX, USA

MKK is a sample of Maasai in Kinyawa, Kenya; and CHI (Chinese) is a combined sample of CHB and CHD.

Their analysis of these findings are cautious:

> **A striking result is that the relationship between** > ***F***_(**ST**) **and recombination rate is stronger for** > ***F***_(**ST**) **between pairs of closely-related populations**, > whether within or outside Africa:*F*_(ST) between a West African > sample and Maasai (of mixed West African and East African > ancestry…decreases by an average of 6% for every 1 cM/Mb…*F*_(ST) > between Italians and individuals of North-Western European ancestry > decreases by 10% for every cM/Mb …and*F*_(ST) between Japanese and > individuals of Chinese ancestry decreases by 4%.**..In view of the > large effective population size in recent human history since each of > these pairs of populations have split, these observations support the > possibility that the different patterns observed between different > pairs of populations are due to natural selection operating more > efficiently in the context of larger population sizes.** We observed a > weak convex relationship with recombination rate for*F*_(ST)between > closely-related populations in a quadratic regression analysis …which > is intriguingly opposite to what was observed between Europeans and > Asians…On the other hand, these observations do not seem to support > the possibility that the different patterns are due to selective > sweeps being shared to different extent across different pairs of > populations since the level of gene flow between HapMap 3 > closely-related populations likely have had been higher than that > between continents. These results, while interesting, should be viewed > with caution due to the confounder of ascertainment bias. It will be > possible to test these observations further by analyzing data from the > 1000 Genomes Project, where whole-genome sequencing will generate data > that is largely free of ascertainment bias for many of the HapMap 3 > populations as well as additional populations

The reason that selection would be more powerful at large population sizes is that the noise of random genetic drift is less likely to interfere with its deterministic process. Additionally, one presumes there would be more extant genetic variation in large populations than small ones. But though these results are interesting, they don’t seem to put too much stock in them.

Let me finish with the author’s conclusion:

> In conclusion, we have shown that genome-wide human population > differentiation in allele frequencies is significantly correlated with > recombination rate on a megabase scale, demonstrating that natural > selection has had a profound effect on allele frequency distributions > averaged over the last hundred thousand years. While these results > likely reflect the effects of hitchhiking and background selection, > disentangling the strengths of these two forces will require extending > the analyses presented in this paper. One important direction is to > use genetic maps that have fine spatial resolution, which may shed > light on the detailed distribution of selective coefficients that have > shaped allele frequency differentiation. A second direction in which > these results can be extended is to compare more populations of > continentally diverse ancestry. This should facilitate an exploration > of the relationship between recombination rate and population > differentiation during different epochs of human evolution, and should > allow a better understanding of how demographic history has shaped the > impact of natural selection on patterns of human genetic variation.

**Note:** I left a lot out in this treatment. It’s Open Access so you can read the whole thing!

**Citation:** Keinan A, Reich D, 2010 Human Population Differentiation Is Strongly Correlated with Local Recombination Rate. PLoS Genet 6(3): e1000886. doi:10.1371/journal.pgen.1000886

### Related Posts:

- [The History and Geography of Genes...not the
  book](https://www.gnxp.com/WordPress/2008/01/14/the-history-and-geography-of-genes-not-the-book/) - [Luca Cavalli-Sforza
  biography](https://www.gnxp.com/WordPress/2005/11/21/luca-cavalli-sforza-biography/) - [Interview with Luigi Luca Cavalli-Sforza in
  Nature](https://www.gnxp.com/WordPress/2007/10/17/interview-with-luigi-luca-cavalli-sforza-in-nature/) - [The geography of genes tells us only so much about
  history](https://www.gnxp.com/WordPress/2011/08/24/the-geography-of-genes-tells-us-only-so-much-about-history/) - [What L. L. Cavalli-Sforza got
  wrong?](https://www.gnxp.com/WordPress/2008/04/21/what-l-l-cavalli-sforza-got-wrong/) - [A picture is worth a thousand words, part
  n](https://www.gnxp.com/WordPress/2008/04/21/a-picture-is-worth-a-thousand-words-part-n/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F03%2F26%2Fnatural-selection-recombination-in-the-human-genome%2F&linkname=Natural%20selection%20%26%20recombination%20in%20the%20human%20genome "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F03%2F26%2Fnatural-selection-recombination-in-the-human-genome%2F&linkname=Natural%20selection%20%26%20recombination%20in%20the%20human%20genome "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F03%2F26%2Fnatural-selection-recombination-in-the-human-genome%2F&linkname=Natural%20selection%20%26%20recombination%20in%20the%20human%20genome "Email")[](https://www.addtoany.com/share)
