+++
title = "Massive genomic sample"
full_title = "Massive genomic sample sizes = detecting evolution in real time"
date = "2017-09-17"
upstream_url = "https://www.gnxp.com/WordPress/2017/09/17/massive-genomic-sample-sizes-detecting-evolution-in-real-time/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/09/17/massive-genomic-sample-sizes-detecting-evolution-in-real-time/).

Massive genomic sample sizes = detecting evolution in real time

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/journal.pbio_.2002458.g003.jpg?resize=300%2C270)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/journal.pbio_.2002458.g003.jpg?resize=300%2C270)](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2002458)The recent PLOS BIOLOGY paper, [Identifying genetic variants that affect viability in large cohorts](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2002458), seems to have triggered a feeding frenzy in the media. For example, [Big Think](http://bigthink.com/paul-ratner/scientists-find-evidence-of-human-evolution-still-actively-happening) has put up [Researchers Find Evidence That Human Evolution Is Still Actively Happening](http://bigthink.com/paul-ratner/scientists-find-evidence-of-human-evolution-still-actively-happening).

I wasn’t paying close attention **because of course human evolution is still happening actively.** From a genetic perspective, evolution is just change in allele frequencies. Populations aren’t infinite, so even if there wasn’t any selection stochastic forces would shift allele frequencies. But of course selection is probably happening. For adaptation by natural selection to occur you need heritable variation on a trait where there are fitness differences as a function of variation within the population. It seems implausible that these conditions don’t still apply. There’s plenty of fitness variation in the population, and it’s unlikely to be random as a function of heritable variation.

But the devil is in the details. And last year [Field et al.](https://www.ncbi.nlm.nih.gov/pubmed/27738015) used the modern genomic tools available to detect selection occurringover the past 2,000 years. It is not credible that it wo[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/comp-stats_genomics.jpeg?resize=153%2C234)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/comp-stats_genomics.jpeg?resize=153%2C234)](https://www.amazon.com/exec/obidos/ASIN/B000WMELKG/geneexpressio-20)uld have magically stopped a few centuries ago.

So why is this new paper such a big deal? (note that it’s in PLOSBIOLOGY, not PLOS GENETICS) **Because the method they use is ingenious and simple**. Basically, they’re looking at changes in allele frequencies as a function of age in huge populations. It’s a little more complicated than that, they used a logistic regression to control for some of the other variables. But they found some biologically plausible hits with their data set of 50,000-150,000. And, they replicated their hits from a European sample to a non-European one.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/biometry.jpeg?resize=183%2C275)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/biometry.jpeg?resize=183%2C275)](https://www.amazon.com/exec/obidos/ASIN/0716786044/geneexpressio-20)This does bring me back to a discussion I observed a while back. An evolutionary geneticist who works with Drosophila mentioned offhand that in his field there really wasn’t that much of a need for more data. They could spend all their time to doing analysis. A prominent human geneticist whose work focused on biomedicine piped up that that wasn’t true at all for their field. There are some differences in the scientific questions, but there are also differences in terms of what you can do with humans as a model organism.

In the paper they look forward to the day of increasing sample sizes an order of magnitude beyond where it is now. At some point in the near future, large fractions of entire nations will be sequenced at medical grade level (30x coverage).

Anyway, you should read[Identifying genetic variants that affect viability in large cohorts](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2002458). It’s pretty straightforward.

### Related Posts:

- [More detecting natural
  selection](https://www.gnxp.com/WordPress/2007/06/18/more-detecting-natural-selection/) - [Natural selection caught in the
  act](https://www.gnxp.com/WordPress/2021/05/19/natural-selection-caught-in-the-act/) - [Random acts of
  ill-health](https://www.gnxp.com/WordPress/2010/01/05/random-acts-of-ill-health/) - [Human driven elephant
  evolution?](https://www.gnxp.com/WordPress/2008/01/22/human-driven-elephant-evolution/) - [Composite test for positive selection in the
  genome](https://www.gnxp.com/WordPress/2010/01/07/composite-test-for-positive-selection-in-the-genome/) - [Female mate preference as a tool, not the
  hand?](https://www.gnxp.com/WordPress/2008/08/23/female-mate-preference-as-a-tool-not-the-hand/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F09%2F17%2Fmassive-genomic-sample-sizes-detecting-evolution-in-real-time%2F&linkname=Massive%20genomic%20sample%20sizes%20%3D%20detecting%20evolution%20in%20real%20time "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F09%2F17%2Fmassive-genomic-sample-sizes-detecting-evolution-in-real-time%2F&linkname=Massive%20genomic%20sample%20sizes%20%3D%20detecting%20evolution%20in%20real%20time "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F09%2F17%2Fmassive-genomic-sample-sizes-detecting-evolution-in-real-time%2F&linkname=Massive%20genomic%20sample%20sizes%20%3D%20detecting%20evolution%20in%20real%20time "Email")[](https://www.addtoany.com/share)
