+++
title = "HAR1F tangents"
full_title = "HAR1F tangents"
date = "2006-08-17"
upstream_url = "https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/).

HAR1F tangents

The [fastest-evolving region in the human genome](https://www.gnxp.com/blog/2006/08/fastest-evolving-human-gene.php) contains a non-coding RNA named HAR1F. In the study, the authors found that the HAR1F RNA doesn’t seem to have the characteristics of a microRNA. They also checked the secondary structure of the RNA (the stems and loops that form when RNA base-pairs with itself and folds up) and found no similarity to other known non-coding RNA structures. HAR1F is found in Cajal-Retzius cells. Cajal-Retzius cells are the major source of a secreted factor called Reelin. One idea about the role of Cajal-Retzius cells is that they sit in the area that will be the outermost part of the cortex (the marginal zone) and release Reelin down towards what will be the innermost part (the subplate). So a gradient of Reelin is formed, strongest at neocortical layer II (outermost cell layer) and weakest at layer VI (innermost cell layer). This is thought to be important for establishing the cortical layers in the right order (but [see here](http://dev.biologists.org/cgi/content/abstract/133/3/537)). Also, different parts of the neocortex have different amounts of each cell layer (i.e. visual cortex might have a thicker layer IV than motor cortex), and changes in Reelin expression might account for some of this.

To get an idea of what a non-coding RNA in the brain could be up to, you might check out [this Perspective by Goodrich and Kugel](http://www.nature.com/nrm/journal/v7/n8/abs/nrm1946.html). They review a number of recently characterized ncRNAs that affect [transcription](https://en.wikipedia.org/wiki/Transcription_%28genetics%29#Eukaryotic_transcription). Do we all know the central dogma? I’m never sure. Just in case, transcription is the process of producing an RNA strand by reading off the DNA template (the genome). Most RNA is made through the orchestrated efforts of a bunch of general transcription factors (TFIIB, TFIID, etc..) and RNA polymerase II (Pol II). All of these factors and Pol II itself are targets for regulation. You can slow down and speed up the machine by adding on or taking off phosphate groups on Pol II’s amino acids, for instance. Besides, the general transcription factors there are specific regulatory transcription factors. Hox genes are a well-known example of these. They have specific DNA motifs that they like to bind to near their target genes, so they can get close and screw around with the DNA or the interaction between DNA and transcription machinery.

As an example, there is this ncRNA called Neuron-Restrictive Silencer Element double-stranded RNA (NRSE dsRNA). NRSE dsRNA relieves the genes that should be expressed in neurons from repression by a protein called Neuron-Restrictive Silencing Factor (NRSF). NRSF is this blanket repressor that keeps neuron-specific genes from being made in other tissues. The cells where NRSE is expressed then will be allowed to make a whole slew of RNAs that others aren’t allowed to. So NRSE seems responsible for deciding where neurons get made, but then, of course, we don’t know what determines where NRSE gets made. Anyway, this is just an example. ncRNAs can block transcriptional repressors and activate transcriptional activators, annnd if they get the notion, they can directly interact with RNA Pol II and speed it up or slow it down. Given HAR1F’s developmental role, it might work in the higher branches of a control hierarchy and affect transcription of some neuronal genes that are themselves transcriptional regulators.

### Related Posts:

- [More on non-coding
  RNAs](https://www.gnxp.com/WordPress/2006/09/19/more-on-non-coding-rnas/) - [RNA Recoding of Brain
  Proteins](https://www.gnxp.com/WordPress/2005/03/16/rna-recoding-of-brain-proteins/) - [A Nobel Prize for gene
  expression](https://www.gnxp.com/WordPress/2006/10/02/a-nobel-prize-for-gene-expression/) - [AT --\> GC mutations, recombination and
  selection](https://www.gnxp.com/WordPress/2006/10/02/at-gc-mutations-recombination-and-selection/) - [The many faces of
  microRNAs](https://www.gnxp.com/WordPress/2007/02/12/the-many-faces-of-micrornas/) - [Technology marches
  on...](https://www.gnxp.com/WordPress/2006/10/03/technology-marches-on/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F08%2F17%2Fhar1f-tangents%2F&linkname=HAR1F%20tangents "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F08%2F17%2Fhar1f-tangents%2F&linkname=HAR1F%20tangents "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F08%2F17%2Fhar1f-tangents%2F&linkname=HAR1F%20tangents "Email")[](https://www.addtoany.com/share)
