+++
title = "Haplogroups in the"
full_title = "Haplogroups in the postgenomic era"
date = "2014-12-11"
upstream_url = "https://www.gnxp.com/WordPress/2014/12/11/haplogroups-in-the-post-genomic-era/"

+++
Source: [here](https://www.gnxp.com/WordPress/2014/12/11/haplogroups-in-the-post-genomic-era/).

Haplogroups in the post-genomic era

[![Haplogroup_R1b\_(Y-DNA)](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/12/Haplogroup_R1b_Y-DNA-e1418324851886.png?resize=550%2C355)![Haplogroup_R1b\_(Y-DNA)](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/12/Haplogroup_R1b_Y-DNA-e1418324851886.png?resize=550%2C355)](https://en.wikipedia.org/wiki/Haplogroup_R1b_%28Y-DNA%29)

Sometimes results precede an appropriate interpretative framework. I am beginning to think that that was the case with the explosion in analysis of Y and mtDNA phylogenies around the year 2000. This was a research program which took the direct male (Y) and female (mtDNA) lineages of humans, inferred a phylogenetic tree, and drew historical and demographic inferences. Part of the reason that this research program flourished is that these lineages are easy to model as a tree, because there isn’t recombination and sex on these lineages. The genealogy of these markers **is actually a straightforward tree.** And, in the case of mtDNA it is copious, so extraction was and is relatively easy.

[![The_Journey_of_Man\_-\_A_Genetic_Odyssey](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/12/The_Journey_of_Man_-_A_Genetic_Odyssey1.jpg?resize=197%2C299)![The_Journey_of_Man\_-\_A_Genetic_Odyssey](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/12/The_Journey_of_Man_-_A_Genetic_Odyssey1.jpg?resize=197%2C299)](https://www.amazon.com/exec/obidos/ASIN/0812971469/geneexpressio-20)This period not only resulted in an explosion of research, but also several seminal popularizations, in particular Spencer Wells’ [Journey of Man](https://www.amazon.com/exec/obidos/ASIN/0812971469/geneexpressio-20), Bryan Sykes’ [Seven Daughters of Eve](https://www.amazon.com/exec/obidos/ASIN/0393323145/geneexpressio-20), and Steve Olson’s [Mapping Human History](https://www.amazon.com/exec/obidos/ASIN/0618352104/geneexpressio-20). The way this method worked involved taking the distributions and diversity of particular Y and mtDNA lineages present in the world today, and inferring back to the past routes of human expansions assuming that the tree-like model of genetic fission reflected a serial founder process of human migration. On the largest scale this work did confirm and extend previous research which implied a dominant signal “Out of Africa,” as well as broad geographic racial clusters. **But on a finer scale the method likely led us astray, in large part because it seems that the [model of human expansion and diversification was just too simple](http://biorxiv.org/content/early/2014/03/21/003517).** Empirically, the rise of whole-genome analysis via SNP-chips in the middle years of the 2000s and today ancient DNA has revolutionized and clarified our understanding of the human past. Many of the inferences made from Y and mtDNA turn out to be wrong, even if the original results are broadly robust.

And that is a key point, because the original Y and mtDNA results are correct, and there has been an enormous body of research already in this area, **they can be leveraged in understanding the human past when slotted into the framework sculpted on the edges by autosomal SNP data and ancient DNA.** A serial founder model where the tree-like phylogeny was recapitulated spatially and temporally always had “anomalies.” In light of new results those anomalies may actually make much more sense. In particular, the common lineages which span Sub-Saharan Africa and western Eurasia (e.g., R1b above) are less perplexing in light of a model of periodic Eurasian back migration after the initial Out of Africa event. Admixture between long distinct lineages resolves issues relating to haplogroups with discordant geographic distributions. As long a the results are correct and methodologically sound, more data is better.

### Related Posts:

- [Neanderthal Y chromosomes, a tale of
  loss?](https://www.gnxp.com/WordPress/2016/04/15/neanderthal-y-chromosomes-a-tale-of-loss/) - [The Cro-Magnons have no descendants in Europe
  today](https://www.gnxp.com/WordPress/2015/05/15/the-cro-magnons-have-no-descendants-in-europe-today/) - [Please ignore mtDNA and Y chromosomal
  haplogroups](https://www.gnxp.com/WordPress/2013/05/20/please-ignore-mtdna-and-y-chromosomal-haplogroups/) - [Clarification on
  "roots"](https://www.gnxp.com/WordPress/2005/11/26/clarification-on-roots/) - [Sex-mediated gene flow & demographic
  history](https://www.gnxp.com/WordPress/2009/08/04/sex-mediated-gene-flow-demographic-history/) - [Older than
  Adam](https://www.gnxp.com/WordPress/2013/03/06/older-than-adam/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F12%2F11%2Fhaplogroups-in-the-post-genomic-era%2F&linkname=Haplogroups%20in%20the%20post-genomic%20era "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F12%2F11%2Fhaplogroups-in-the-post-genomic-era%2F&linkname=Haplogroups%20in%20the%20post-genomic%20era "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F12%2F11%2Fhaplogroups-in-the-post-genomic-era%2F&linkname=Haplogroups%20in%20the%20post-genomic%20era "Email")[](https://www.addtoany.com/share)
