+++
title = "Genes in space"
full_title = "Genes in space"
date = "2012-05-25"
upstream_url = "https://www.gnxp.com/WordPress/2012/05/25/genes-in-space/"

+++
Source: [here](https://www.gnxp.com/WordPress/2012/05/25/genes-in-space/).

Genes in space

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/05/754px-Lensshoe_hubble.jpg?resize=600%2C477)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/05/754px-Lensshoe_hubble.jpg?resize=600%2C477)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/05/754px-Lensshoe_hubble.jpg)

From some of the same people who brought you the [genetic map of Europe](http://www.nature.com/nature/journal/v456/n7218/abs/nature07331.html), a very important paper, [A model-based approach for analysis of spatial structure in genetic data](http://www.nature.com/ng/journal/vaop/ncurrent/abs/ng.2285.html). Here’s the abstract:

> Characterizing genetic diversity within and between populations has > broad applications in studies of human disease and evolution. We > propose a new approach, spatial ancestry analysis, for the modeling of > genotypes in two- or three-dimensional space. In spatial ancestry > analysis (SPA), **we explicitly model the spatial distribution of each > SNP by assigning an allele frequency as a continuous function in > geographic space.** We show that the explicit modeling of the allele > frequency allows individuals to be localized on the map on the basis > of their genetic information alone. We apply our SPA method to a > European and a worldwide population genetic variation data set and > identify SNPs showing large gradients in allele frequency, and we suggest these as candidate regions under selection. These regions include SNPs in the well-characterized LCT region, as well as at loci including FOXP2, OCA2 and LRP1B.

Within the guts of this paper they make an important observation: **constructing a set of populations and then generating pairwise statistics of differentiation across those populations has an element of arbitrariness.** Rather than going in that direction the authors here are evaluating variation of genes as a function of continuous space, rather than binning them into discrete populations. In this way they can use patterns of genes to back infer the likely geographic origin of an individual, and more intriguingly pinpoint genetic loci which exhibit sharp gradients across space, and so may be targets of natural selection. The adaptive story for *LCT* is straightforward. But what of *OCA2*, which is mostly well known as a pigmentation locus which has been implicated in blue vs. brown eye variation in Europeans? As I like to say, interesting times….

And of course, they have released [the software](http://genetics.cs.ucla.edu).

***Credit:** [Wikipedia](https://en.wikipedia.org/wiki/File:DNA_orbit_animated_static_thumb.png)*

### Related Posts:

- [Reprising genes &
  geography](https://www.gnxp.com/WordPress/2012/04/16/reprising-genes-geography/) - [A picture is worth a thousand words, part
  n](https://www.gnxp.com/WordPress/2008/04/21/a-picture-is-worth-a-thousand-words-part-n/) - [SpaceMix](https://www.gnxp.com/WordPress/2015/01/09/spacemix/) - [What L. L. Cavalli-Sforza got
  wrong?](https://www.gnxp.com/WordPress/2008/04/21/what-l-l-cavalli-sforza-got-wrong/) - [The genetics of
  Fenno-Scandinavia](https://www.gnxp.com/WordPress/2008/09/15/the-genetics-of-fenno-scandinavia/) - [Genetic map of Europe
  again](https://www.gnxp.com/WordPress/2008/08/31/genetic-map-of-europe-again/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F05%2F25%2Fgenes-in-space%2F&linkname=Genes%20in%20space "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F05%2F25%2Fgenes-in-space%2F&linkname=Genes%20in%20space "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F05%2F25%2Fgenes-in-space%2F&linkname=Genes%20in%20space "Email")[](https://www.addtoany.com/share)
