+++
title = "Genome architecture in"
full_title = "Genome architecture in Caenorhabditis"
date = "2015-06-29"
upstream_url = "https://www.gnxp.com/WordPress/2015/06/29/genome-architecture-in-caenorhabditis-is-a-local-story/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/06/29/genome-architecture-in-caenorhabditis-is-a-local-story/).

Genome architecture in Caenorhabditis

[![450px-Adult_Caenorhabditis_elegans](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/06/450px-Adult_Caenorhabditis_elegans-300x98.jpg?resize=300%2C98)![450px-Adult_Caenorhabditis_elegans](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/06/450px-Adult_Caenorhabditis_elegans-300x98.jpg?resize=300%2C98)](https://en.wikipedia.org/wiki/Caenorhabditis_elegans#/media/File:Adult_Caenorhabditis_elegans.jpg)[*C. elegans*](https://en.wikipedia.org/wiki/Caenorhabditis_elegans) is kind of a big deal, because it is a [model organism](https://en.wikipedia.org/wiki/Model_organism). Model organisms are a big deal because they illuminate general principles in biological processes, and allow life science to become more than “stamp collecting.” As a practical matter it is also great be part of a model organism community because they *are* communities. And these scientific cultures have come a long way. It used to be that you had the [“fly room.”](https://www.amazon.com/exec/obidos/ASIN/0813193370/geneexpressio-20/) Today you have the [“fly meetings”](http://www.genetics-gsa.org/drosophila/2015/), which boast \~1,500 scientists attending! (the fact that these meetings are sponsored by the *Genetics Society of America* should make it quite clear how central*Drosophila* are in the world of genetics)

[![51DX03NQH6L.\_SY344_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/06/51DX03NQH6L._SY344_BO1204203200_-197x300.jpg?resize=197%2C300)![51DX03NQH6L.\_SY344_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/06/51DX03NQH6L._SY344_BO1204203200_-197x300.jpg?resize=197%2C300)](https://www.amazon.com/exec/obidos/ASIN/023113147X/geneexpressio-20)For *C.elegans* Andrew Brown’s [In the Beginning Was the Worm: Finding the Secrets of Life in a Tiny Hermaphrodite](https://www.amazon.com/exec/obidos/ASIN/023113147X/geneexpressio-20) is the best treatment I know. It’s an organism whose every cell has been mapped, thanks to the sort of bench science labor that’s unlikely to be repeated in the near future. This pinpoint precision in the cytological and morphological structure isone reason my friend Armand Leroi can wax eloquently about *elegans*. **He’s a *developmental* geneticist.** But as [Patrick Phillips](https://scholar.google.com/citations?user=JbH6mVkAAAAJ&hl=en&oi=ao) admitted to me years ago a hermaphroditic [selfing](https://en.wikipedia.org/wiki/Reproduction#Autogamy) species exhibits some deficiencies from apopulation genetic perspective.*C.* e*legans* just doesn’t have that much genetic diversity. Not a big deal if you are exploring molecular genetic mechanisms, but kind of a problem for exploration population *variation*(unless you arestudying *H. sapiens*, in which case all is forgiven). Luckily, there are “outcrossing” variants of *Caenorhabditis* out there. These outcrossing lineages are more amenable to traditional population genetic theory, which was developed in the context of sexually reproducing diploid organisms such as *Drosophila* and humans.

[![originsgenomearchitecture](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/06/originsgenomearchitecture.png?resize=234%2C284)![originsgenomearchitecture](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/06/originsgenomearchitecture.png?resize=234%2C284)](https://www.amazon.com/exec/obidos/ASIN/0878934847/geneexpressio-20/)This being 2015, to do cutting edge population genetics you need to also develop genomic resources for these exotic outcrossing species of worms. That seems to be exactly what’s happening, as highlighted in a new paper, [Reproductive Mode and the Evolution of Genome Size and Structure in *Caenorhabditis* Nematodes](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005323). While *elegans* is very low in variation (many researchers like to work on isogenic lines which differ by a mutant here and there), some of the outcrossing lineages are very polymorphic. This makes assembly of the genome more difficult for various technical reasons (ergo, for *de novo* [assembly](https://en.wikipedia.org/wiki/Sequence_assembly) you usually attempt to find an extremely homozygous individual of a given species). On Twitter [Patrick offered](https://twitter.com/granex/status/614527977167020032) that they “generated more than 2,000 inbred lines for sequencing and genetic map”, and that over “90% went extinct.” Obviously they got it done, but it was tedious.

![**Citation**: Fierst JL, Willis JH, Thomas CG, Wang W, Reynolds RM, Ahearne TE, et al. (2015) Reproductive Mode and the Evolution of Genome Size and Structure in Caenorhabditis Nematodes. PLoS Genet 11(6): e1005323. doi:10.1371/journal.pgen.1005323](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/06/journal.pgen_.1005323.g001.png?resize=400%2C399)

The abovewasn’t a *population* genomic paper. They didn’t have large sample sizes, nor were they focusing on questions that applied to the microevolutionary scale (within species level lineages). Rather, they were comparing the genomes of *Caenorhabditis* lineages which diverged on the order of \~30 million years ago. The effective population size difference between selfing and outcrossing lineages is huge, with the authors reporting*N_(e)* \< 10,000 for *C. elegans* *N_(e)* \> 1,000,000 for *C. remanei*. This is a big deal because variation in effective population size has been argued by many, foremost Mike Lynch, as one of the drivers of the phenomenon of huge genome size differences. Lynch is a fertile mind with many ideas, and if you are curious about them I’d recommend a purchase and read through of[The Origins of Genome Architecture](https://www.amazon.com/exec/obidos/ASIN/0878934847/geneexpressio-20/). But the upshot from this paper seems to be that the broader thesis of Lynch and his supporters **is not favored by these *specific* results utilizing comparative genomics.** Every few years I reread Lynch’s 2005 paper, [The Origins of Eukaryotic Gene Structure](http://mbe.oxfordjournals.org/content/23/2/450.full), because genomics is a rapidly changing field, and many of the predictions and conjectures are now being tested.

**Ideally genetics is a science which produces powerful general theories.** Actually, it’s not an ideal. It’s the concrete aim. Genetics as a science was forged in the fires of Mendelism. It was a formal and abstract advance over intuitive models of blending inheritance. It worked for 50 years without any knowledge of its concrete biophysical mode of transmission, DNA. With the emergence of genomics, and the fusing of this field with classical genetics, it is expected that one would probe the bounds of generality. E.g., what population genetic parameters influence genome size, or the proportion of the genome that is intergenic, and the number of self genetic elements? The genomic is a rich multi-dimensional topography, and for many organisms we have poor singular drafts. That will change. The quest for the abstract frameworksfrom which we can deduce and predict will probably come with more data, and the computational power to analyze that data. **Almost certainly this will start with the model organisms in the wake of the maturing of human genomics.** I’m not sure that the reduction in genome size of selfing organisms says much generally about evolutionary processes, but it’s part of a broader puzzle which needs to be assembled so that genomics moves beyond its exploratory phase of description.

**Citation:**Fierst JL, Willis JH, Thomas CG, Wang W, Reynolds RM, Ahearne TE, et al. (2015) [Reproductive Mode and the Evolution of Genome Size and Structure in*Caenorhabditis*Nematodes](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005323). PLoS Genet 11(6): e1005323. doi:10.1371/journal.pgen.1005323

### Related Posts:

- [Phenotype to genotype: as easy as
  1,2,3...10](https://www.gnxp.com/WordPress/2007/10/03/phenotype-to-genotype-as-easy-as-1-2-3-10/) - [The contingent history of
  science](https://www.gnxp.com/WordPress/2009/09/19/the-contingent-history-of-science/) - [The ethics of blogging
  conferences](https://www.gnxp.com/WordPress/2009/06/03/the-ethics-of-blogging-conferences/) - [Sex ratios](https://www.gnxp.com/WordPress/2006/09/14/sex-ratios/) - [Attendance at ASHG Meetings since
  1981](https://www.gnxp.com/WordPress/2017/10/12/attendance-at-ashg-meetings-since-1981/) - [Extensive transcription in the
  fly](https://www.gnxp.com/WordPress/2006/10/01/extensive-transcription-in-the-fly/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F06%2F29%2Fgenome-architecture-in-caenorhabditis-is-a-local-story%2F&linkname=Genome%20architecture%20in%20Caenorhabditis "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F06%2F29%2Fgenome-architecture-in-caenorhabditis-is-a-local-story%2F&linkname=Genome%20architecture%20in%20Caenorhabditis "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F06%2F29%2Fgenome-architecture-in-caenorhabditis-is-a-local-story%2F&linkname=Genome%20architecture%20in%20Caenorhabditis "Email")[](https://www.addtoany.com/share)
