+++
title = "Reconstructing a"
full_title = "Reconstructing a generation unsampled"
date = "2012-01-14"
upstream_url = "https://www.gnxp.com/WordPress/2012/01/14/reconstructing-a-generation-unsampled/"

+++
Source: [here](https://www.gnxp.com/WordPress/2012/01/14/reconstructing-a-generation-unsampled/).

Reconstructing a generation unsampled

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/01/blank_pedigree_5gens-155x300.jpg?resize=155%2C300)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/01/blank_pedigree_5gens-155x300.jpg?resize=155%2C300)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/01/blank_pedigree_5gens.jpg)In the near future I will be analyzing the genotype of an individual where all four grandparents have been typed. But this got me thinking about my own situation: **is there a way I could “reconstruct” my own grandparents?** None of them are living. The easiest way to type them would be to obtain tissue samples from hospitals. This is not totally implausible, though in this case these would be Bangladeshi hospitals, so they might not have saved samples or even have a good record of hem. Another way would be to extract DNA from the burial site. This is not necessarily palatable. But assuming you did this, if you have access to a forensic lab it might be pretty easy (though I think most forensic labs using [VNTRs](https://en.wikipedia.org/wiki/Variable_number_tandem_repeat), rather than SNP chips, so I don’t know if they’d touch every chromosome), I’m not sure that the quality would be optimal for more vanilla typing operations, especially for older samples which are likely to be contaminated with a lot of bacteria.

For me the simplest option is to look at relatives. Each of my grandparents happens to have had siblings, so there are many sets of relatives related to just each of those individuals of interest. I also have many cousins, so pooling all the genotypes together and using the information of a pedigree one could ascertain which chromosomal segments are likely to derive from a particular grandparent. To give a concrete example, my mother has a maternal cousin to whom she is quite close. By typing my mother and her cousin **one could infer that the segments shared across the two individuals derive from the common maternal grandparents.** Of course there’s a problem that cousins have a coefficient of relatedness of only 1/8th, so there is going to be a lot of information missing. But, if you had lots of cousins you could presumably reconstruct the genotypes far better.



But what if you didn’t have any of this? I came up with a crazy idea, and I want to throw it out there to see how crazy it is. The issue from the perspective of you, the indivdual without grandparental information, is that for either your mother or your father you don’t know which homologous chromosomes come from which parent (your grandparents, their parents). As it happens, everyone has a male parent and a female parent. **So if you can assign a a chromosomal region as having come from the male, and another as having come from the female**, then you can reconstruct some of your grandparents’ genotypes because you know their sexes. How can you make this determination?

[Genomic imprinting](https://en.wikipedia.org/wiki/Genomic_imprinting#Genetic_mapping_of_imprinted_genes). This is a phenomenon where genes from a given parent, often of a particular sex, are expressed, while those of the other sex are repressed (often it manifests in terms of methylation or lack of methylation). Therefore, if you have a gene, A, which is usually expressed if inherited from a male parent and repressed if it is inherited from a female parent then the state of that gene within a chromosomal region can be a “tag” for the sex of the parent of origin. With enough of these imprinted genes you can create a mosaic of the genome of the individual in terms of sex of origin. Obviously genomic regions from different sexes are from different parents. If you have enough children of these two parents you should be able to infer the whole genomes of these individuals.

The big reason this probably won’t work is that there just aren’t enough imprinted genes in the human genome. But what do readers think?

### Related Posts:

- [Healthcare under an insane
  dictator](https://www.gnxp.com/WordPress/2009/07/26/healthcare-under-an-insane-dictator/) - [Predicting blue eyes from
  genes](https://www.gnxp.com/WordPress/2009/03/10/predicting-blue-eyes-from-genes/) - [A DNA database
  nation?](https://www.gnxp.com/WordPress/2006/05/20/a-dna-database-nation/) - [Genetic counseling and conflict of
  interest](https://www.gnxp.com/WordPress/2012/07/14/genetic-counseling-and-conflict-of-interest/) - [DNA databases -
  revisited](https://www.gnxp.com/WordPress/2006/05/24/dna-databases-revisited/) - [Grandparents as reality, not
  theory](https://www.gnxp.com/WordPress/2012/04/17/grandparents-as-reality-not-theory/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F01%2F14%2Freconstructing-a-generation-unsampled%2F&linkname=Reconstructing%20a%20generation%20unsampled "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F01%2F14%2Freconstructing-a-generation-unsampled%2F&linkname=Reconstructing%20a%20generation%20unsampled "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F01%2F14%2Freconstructing-a-generation-unsampled%2F&linkname=Reconstructing%20a%20generation%20unsampled "Email")[](https://www.addtoany.com/share)
