+++
title = "Denisovans,"
full_title = "Denisovans, Neanderthals, Yetis, oh my!"
date = "2018-03-19"
upstream_url = "https://www.gnxp.com/WordPress/2018/03/19/denisovans-neanderthals-yetis-oh-my/"

+++
Source: [here](https://www.gnxp.com/WordPress/2018/03/19/denisovans-neanderthals-yetis-oh-my/).

Denisovans, Neanderthals, Yetis, oh my!

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/03/browning1.jpg?resize=600%2C472&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/03/browning1.jpg?resize=600%2C472&ssl=1)](http://www.cell.com/cell/fulltext/S0092-8674(18)30175-2)

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/03/cryptozoo.jpeg?resize=183%2C275&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/03/cryptozoo.jpeg?resize=183%2C275&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/0684856026/geneexpressio-20/ref=as_at?creativeASIN=0716745305&linkCode=w61&imprToken=vYrSv9uK3G-.k.JhcKLrPA&slotNum=54)An excellent open access paper is out in *Cell* which explores the distribution of archaic hominin, and in particular Denisovan, ancestry, [Analysis of Human Sequence Data Reveals Two Pulses of Archaic Denisovan Admixture](http://www.cell.com/cell/fulltext/S0092-8674(18)30175-2):

> Anatomically modern humans interbred with Neanderthals and with a > related archaic population known as Denisovans. Genomes of several > Neanderthals and one Denisovan have been sequenced, and these > reference genomes have been used to detect introgressed genetic > material in present-day human genomes. Segments of introgression also > can be detected without use of reference genomes, and doing so can be > advantageous for finding introgressed segments that are less closely > related to the sequenced archaic genomes. We apply a new > reference-free method for detecting archaic introgression to 5,639 > whole-genome sequences from Eurasia and Oceania. We find Denisovan > ancestry in populations from East and South Asia and Papuans. > Denisovan ancestry comprises two components with differing similarity > to the sequenced Altai Denisovan individual. **This indicates that at least two distinct instances of Denisovan admixture into modern humans occurred, involving Denisovan populations that had different levels of relatedness to the sequenced Altai Denisovan.**

Before you get caught up in the results, you should [check out the methods](http://www.cell.com/action/showMethods?pii=S0092-8674%2818%2930175-2). They’re pretty ingenious. Though with novel results like this people also really need to work their way through them as well (the authors present a lot of simulation results to validate the method, so I’m sure that will convince most; it certainly sways me).

The plots at the top of this post show the different distribution of Neanderthal and Denisovan admixture, by matching regions of the genome that they’ve identified as archaically introgressed. The ultimate logic is to look for variants which aren’t found in Africans, and are found in non-Africans, and scan over segments of the genome hoping that you can pick up the haplotypes that would slowly be chopped up over time through recombination that came in from Neanderthals or Denisovans.

At the top-left of the figure, you see “Northwest Europeans.” The segments tend to concentrate at the bottom-right of the panel. That means that they match the Neanderthal reference sequence to a high degree, but not the Denisovan. This makes sense since everything we know from earlier work indicates that Northwest Europeans don’t have Denisovan ancestry.

On the bottom-right you see Papuans. They’re very out of place because they are the only population in the list where Denisovan ancestry is greater than Neanderthal ancestry. This is visible in the match patterns.

South and East Asian populations exhibit a pattern with high (relative) levels of Neanderthal matches, but also a minor amount of Denisovan matching. This aligns with earlier work, which reported low levels of Denisovan admixture among populations with eastern Eurasian ancestry broadly.

**The surprise is that the variation in matching to the Denisovan Altai genome exhibited a north-to-south cline.**In particular, Northeast Asian populations seem to have a mix of two types of Denisovan. One, which is close to the Denisovan sequence that is normally used as a reference, and one which is diverged from it. The Papuans and South Asians seem to have Denisovan ancestry which is not so much like the Altai sample. This is not very shocking of course.

Finns barely miss the p-value cut-off (Bonferroni-corrected threshold), but they clearly have some Denisovan from East Asian gene flow, and some of it looks to be similar to the Altai Denisovan. Curiously, the Vietnamese (Kinh) don’t show any Altai Denisovan, but the Dai do. The Japanese have a lower proportion of the Altai Denisovan than the two Han Chinese samples. And very strangely the 1K Genomes samples from the New World, a substantial proportion of which have Amerindian admixture, show no Denisovan.

Pontus Skoglund immediately made a very interesting observation:

> There is an undiscussed but potentially explosive implication of this > paper: the 2nd Denisovan gene flow signal in East Asia seems to be > absent from Native American ancestry–could Denisovans have survived after the isolation of these lineages \<30 kya?? <https://t.co/q348Z1iHXB> [pic.twitter.com/stWVJVObMQ](https://t.co/stWVJVObMQ) >
> — Pontus Skoglund (@pontus_skoglund) [March 15, 2018](https://twitter.com/pontus_skoglund/status/974331511972532230?ref_src=twsrc%5Etfw)

And Alexander Kim followed up:

> Browning et al. even more interesting wrt Native Americans alongside > this tentative suggestion in Lipson & Reich 2017 > (<https://t.co/0PVl9oC0i2>). So if true: Denisovan \> Mal’ta before 24 kaBP (another event?) but absent in both “ANE” & East Asian-like portions of NatAm ancestry? [pic.twitter.com/bBImVlAFkC](https://t.co/bBImVlAFkC) >
> — Alexander M. Kim (@amwkim) [March 18, 2018](https://twitter.com/amwkim/status/975185654249349120?ref_src=twsrc%5Etfw)

In the thread to Skoglund’s original comment [Africa Gomez](https://twitter.com/aBugBlog/status/974404075444948999) notes that the authors suggest that high linkage disequilibrium in New World populations, due to recent admixture between diverged groups, may reduce the power to detect the Denisovan ancestry. So perhaps that’s that?

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/01/whoweare.png?resize=180%2C271&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/01/whoweare.png?resize=180%2C271&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/110187032X/geneexpressio-20/ref=as_at?creativeASIN=0275961036&linkCode=w61&imprToken=T0XqexeBoDXvoyr0cR1T6A&slotNum=45)But for a moment, let’s set that aside. The best evidence right now is that the Denisovan admixture into Papuans, and therefore South Asians, occurred not too after the Neanderthal admixture event. That mixture is reasonably well dated because of ancient genomes which are closer to the period of admixture. But what about the second event with the Altai Denisovan? If what Skoglund says is true the date for that might be closer to the Last Glacial Maximum, and not when modern humans came to dominate the region. And I say dominate because there’s evidence that anatomically modern humans may have ventured quite far into eastern Eurasia before they finally swept aside more established lineages.

A few years back [researchers found](https://www.nature.com/articles/nature13408) that one of the mutations that allow for Tibetan high altitude adaptation seems to have come in from a Denisovan genetic background. Spencer Wells, who knows a thing or two about Central Asia, has always half-seriously suggested that the legends of the Yeti derive from populations of archaic humans who persisted in the uplands of the heart of Eurasia.

But perhaps they weren’t pure Denisovans in any case. Work out of David Reich’s lab has suggested that Denisovans themselves, or at least the Alta Denisovan, harbors a deep ancient lineage diverged from modern humans, Neanderthals, and Denisovans, in low fractions. The “Altai Denisovan” admixture may have come into Northeast Asians via a mixed population, which arose when modern humans came to dominate eastern Eurasia, but only transmitted the Altai Denisovan ancestry later.

### Related Posts:

- [Was H. luzonensis
  Denisovan?](https://www.gnxp.com/WordPress/2021/08/14/was-h-luzonensis-denisovan/) - [More and more evidence for 'complex demography' in
  archaic…](https://www.gnxp.com/WordPress/2021/07/19/more-and-more-evidence-for-complex-demography-in-archaic-ancestry/) - [We are all Denisovans
  now!](https://www.gnxp.com/WordPress/2020/04/22/we-are-all-denisovans-now/) - [There were possibly late archaic introgression events
  in…](https://www.gnxp.com/WordPress/2018/04/21/there-was-possibly-late-archaic-introgression-events-in-eurasia/) - [The Wallacean Hominin
  Wonderland](https://www.gnxp.com/WordPress/2020/07/25/the-wallacean-hominin-wonderland/) - [Would you conduct medical experiments on Neanderthals?
  If…](https://www.gnxp.com/WordPress/2016/03/18/would-you-conduct-medical-experiments-on-neanderthals-if-not-call-them-human/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F03%2F19%2Fdenisovans-neanderthals-yetis-oh-my%2F&linkname=Denisovans%2C%20Neanderthals%2C%20Yetis%2C%20oh%20my%21 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F03%2F19%2Fdenisovans-neanderthals-yetis-oh-my%2F&linkname=Denisovans%2C%20Neanderthals%2C%20Yetis%2C%20oh%20my%21 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F03%2F19%2Fdenisovans-neanderthals-yetis-oh-my%2F&linkname=Denisovans%2C%20Neanderthals%2C%20Yetis%2C%20oh%20my%21 "Email")[](https://www.addtoany.com/share)
