+++
title = "Eurogenes 500K SNP"
full_title = "Eurogenes 500K SNP BioGeographicAncestry Project"
date = "2010-11-23"
upstream_url = "https://www.gnxp.com/WordPress/2010/11/23/eurogenes-500k-snp-biogeographicancestry-project/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/11/23/eurogenes-500k-snp-biogeographicancestry-project/).

Eurogenes 500K SNP BioGeographicAncestry Project

Since I have been promoting the [Dodecad Ancestry Project](https://dodecad.blogspot.com/), it seems only fair to bring to your attention [Eurogenes 500K SNP BioGeographicAncestry Project](https://bga101.blogspot.com/). The sample populations are a bit different from [Dodecad](https://dodecad.blogspot.com/), but again ADMIXTURE is the primary tool. But the author also makes recourse to other methodologies to explore more than simply population level variation. For example, his most recent post is [Locating and visualizing minority non-European admixtures across our genomes](https://bga101.blogspot.com/2010/11/locating-and-visualizing-minority-non.html):

> Imagine, for example, a white American carrying a couple of tiny > segments of West African origin, from an ancestor who lived 250 years > ago, and an eastern Finn with no Asian ancestors in the last 4000 > years or more. If we run an inter-continental ADMIXTURE analysis with these two, it’s very likely the American will score 100% European, while the eastern Finn will probably come out around 9% North and East Asian due to really old Uralic influence. >
> That sort of thing isn’t a huge problem when comparing the genetic > structure of populations. Obviously, overall, eastern Finns rather > than white Americans are genetically closer to North Asians, and > that’s basically what ADMIXTURE picks up. However, if the focus is > also on individuals, this certainly can become an issue. Our hypothetical American might be aware of that African ancestor, with solid paperwork backing up their genealogical connection, but he’s pulling his hair out because nothing’s showing up via genetic tests. >
> So let’s take a look at a real life example of how RHHcounter can pick up segments of potentially recent Sub-Saharan African origin…

![euras1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/euras1.png?resize=250%2C144)![euras1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/euras1.png?resize=250%2C144)  
Olivia Munn & Uyghur woman

**The basic issue here is that in terms of genomic variation old admixture looks different from new admixture.** Someone who is a first generation Eurasian, with a Chinese and European parent, may be about the same ancestral mix proportionally as a Uyghur. They would resemble a Uyghur on STRUCTURE and be placed within that cluster on a PCA chart (this is what happens in 23andMe). But, the Uyghur “Eastern” and “Western” genetic heritage has been reshuffled to a great extent by recombination over the past 1,000-2,000 years. In contrast, a first generation Eurasian will have huge swaths of their genome which are Eastern or Western on alternating strands (from their respective parents). In population genetic language a group of first generations hybrids would be exhibit a lot of [linkage disequilibrium](https://en.wikipedia.org/wiki/Linkage_disequilibrium) (LD). In a [panmictic](https://en.wikipedia.org/wiki/Panmictic) hybrid population LD will decay due to recombination, which breaks apart the distinctive allelic associations inherited from the parental populations.

  
This is the key to differentiating between the old “Asian” ancestry which sometimes falls out of the genetic variation of Finns at low frequencies, and more recent Asian ancestry. For example, the paleoanthropologist [Vance Haynes](https://en.wikipedia.org/wiki/Vance_Haynes) is apparently a great-grandson of one of the original “Siamese Twins,” [Chang Bunker](https://en.wikipedia.org/wiki/Chang_and_Eng_Bunker). Chang Bunker was a Chinese Thai, so presumably Vance Haynes would come out to be \~10% Asian, and would be shifted toward the Asian cluster in relation to other Europeans. On the other hand, a closer look at his genome would indicate differences from a Turk who was \~10% Asian, because Vance Haynes’ Asian ancestry has only had three generations for recombination to break apart the original allelic associations which were passed down from Chang Bunker. After only these few generations the genome would still show many segments of clustered ancestry with distinctive sets of markers characteristic of Han Chinese.

Let’s make this more concrete. Below are two “ancestry paintings” from 23andMe. One is of a reference example, a Uyghur woman, and another is of a Eurasian individual. The difference is pretty obvious:

![eurasianman](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/eurasianman.png?resize=600%2C526)![eurasianman](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/eurasianman.png?resize=600%2C526)

![23andmeclus](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/23andmeclus.png?resize=117%2C139)![23andmeclus](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/23andmeclus.png?resize=117%2C139)For the record, 23andMe says that the Eurasian man is 50% Asian, 50% European. For the Uyghur woman, 52% European, 48% Asian. As I indicated above, Eurasian individuals who are projected onto the variation of the HGDP sample tend to cluster with the Uyghurs. In the image to the left the black mark indicates the Eurasian man. The Uyghurs are green. The purple rectangles are Hazaras.

But obviously this is a trivial example. What’s the point of sniffing around for non-European ancestry in individuals whose non-European ancestry is 1) visible, and 2) recent and immediate. No, a bigger question here are claims and suggestions by some white Americans that they have significant non-European ancestry. Usually this is Native American. But in the case of one of the European-origin samples which “Polako” (the principal behind the [BGA Project](https://bga101.blogspot.com/)) analyzed it seems there is a *suggestion* of West African ancestry.

![dandon](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/dandon.png?resize=200%2C173)![dandon](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/11/dandon.png?resize=200%2C173)This individual is Dr. Don Conrad of [Genomes Unzipped](http://www.genomesunzipped.org/). In particular, Polako found that there were [two nearby segments](http://img88.imageshack.us/img88/5421/donm.png) on two chromosomes which exhibited a pattern of population atypical heterozygosity in Dr. Don Conrad’s genome. Look at chromosomes 7 and 13. Contrast the pattern with my [distant paternal cousin](https://en.wikipedia.org/wiki/File:GlobalR1a1a.png), [Dr. Daniel MacArthur](http://img88.imageshack.us/img88/9200/danielm.png). He also exhibits points of heterozygosity, but they’re randomly distributed across the genome. It’s old admixture or just noise.

Polako doesn’t make much of Dr. Don Conrad’s results, and neither do I (presumably as Dr. Don Conrad is a member of [Genomes Unzipped](http://www.genomesunzipped.org/) it’s easy to talk about his results without any of the ethical or moral hassles about confidentiality). On the other hand, unlike Dr. Dan MacArthur, a little utilization of the powers of the interwebs indicates that Dr. Don Conrad is an American. In particular, of recent Midwestern background. Though I’m not a total creep, so I didn’t start poking around [Ancestry.com](http://www.ancestry.com/). But after the [Pickrell affair](http://www.genomesunzipped.org/2010/11/am-i-partly-jewish-an-unexpected-turn-of-events.php) I am probably just a touch more hesitant to laugh off peculiar results from these sorts of analyses as simply algorithms-gone-*meshugana*.

*Image Credit: Colegota*

### Related Posts:

- [If you've done
  23andMe....](https://www.gnxp.com/WordPress/2010/10/25/if-youve-done-23andme/) - [The Dodecad ancestry
  project](https://www.gnxp.com/WordPress/2010/10/25/the-dodecad-ancestry-project/) - [Dodecad Ancestry Project is at
  \~10,000](https://www.gnxp.com/WordPress/2011/09/26/dodecad-ancestry-project-is-at-10000/) - [Have ADMIXTURE run on your genetic
  data](https://www.gnxp.com/WordPress/2010/10/30/have-admixture-run-on-your-genetic-data/) - [My Dodecad
  results](https://www.gnxp.com/WordPress/2010/10/29/my-dodecad-results/) - [Introducing the Harappa Ancestry
  Project](https://www.gnxp.com/WordPress/2011/01/17/introducing-the-harappa-ancestry-project/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F11%2F23%2Feurogenes-500k-snp-biogeographicancestry-project%2F&linkname=Eurogenes%20500K%20SNP%20BioGeographicAncestry%20Project "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F11%2F23%2Feurogenes-500k-snp-biogeographicancestry-project%2F&linkname=Eurogenes%20500K%20SNP%20BioGeographicAncestry%20Project "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F11%2F23%2Feurogenes-500k-snp-biogeographicancestry-project%2F&linkname=Eurogenes%20500K%20SNP%20BioGeographicAncestry%20Project "Email")[](https://www.addtoany.com/share)
