+++
title = "Genomewide association"
full_title = "Genomewide association studies in the UK"
date = "2007-06-06"
upstream_url = "https://www.gnxp.com/WordPress/2007/06/06/genome-wide-association-studies-in-the-uk/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/06/06/genome-wide-association-studies-in-the-uk/).

Genome-wide association studies in the UK

Results from the most ambitious (and expensive) set of genome-wide association studies for common diseases were [published today in Nature](http://www.nature.com/nature/journal/v447/n7145/abs/nature05911.html) (**open access! You can read it for free!**). Funded by the Welcome Trust in the UK, a “dream team” of clinical geneticists and statisticians assembled a common set of 3000 controls to compare genetically to around 2000 cases each of Type I diabetes, Type II diabetes, arthritis, cardiovascular disease, Crohn’s disease, bipolar disorder, and hypertension.

This study is being trumpeted as a major success, and to some extent, it is– for all diseases except hypertension, at least one strong signal and many weaker signals were identified. As correlational studies are largely hypothesis-generating, some of these will lead to major discoveries about the pathology of disease. In Crohn’s disease, for example, the consortium has found a couple loci involved in [autophagy](https://en.wikipedia.org/wiki/Autophagy) and the elimination of intracellular bacteria. They also confirm the association of another locus involved in autophagy. It’s easier for people working on a disease to focus on pathways that are already known to be involved in the disease (for example, there’s a known autoimmune component to Crohn’s disease); it often takes this kind of top-down study to jolt people out of complacency.

The consortium has also make publicly available an impressive suite of software, along with new algorithms for genotype calling and mutlti-locus association, incorporating information from the HapMap. These tools are certainly at the cutting edge, and represent major advances in their own right.

On the other hand, one can’t help but notice that the loci identify contribute only a fraction of the known genetic component of these diseases. This is a proof of principle– the base has been laid; it’s now feasable to scale these sorts of case-control studies up to tens of thousands of individuals. But is that really the most effective way at getting at the genetic basis of these diseases? Perhaps not.

A final comment– I noted in [the comments](http://js-kit.com/api/static/pop_comments?ref=http://gnxp.com&path=/1913881579981880955?url=http://www.gnxp.com/blog/2007/06/recent-human-evolution.php) of a previous post[](https://www.gnxp.com/blog/2007/06/recent-human-evolution.php) that the big data sets used for population genetics these days are generated for medical reasons. There’s a ton of population genetic information here, which the authors are likely going to make more use of the future. They do give us a glimpse, though– they note a number of genomic regions that show marked geographic variability *within the UK* (and note they limit themselves to self-identified “white Europeans”):

> Thirteen genomic regions showing strong geographical variation are > listed in Table 1, and Supplementary Fig. 7 shows the way in which > their allele frequencies vary geographically. The predominant pattern > is variation along a NW/SE axis. The most likely cause for these > marked geographical differences is natural selection, most plausibly > in populations ancestral to those now in the UK. Variation due to > selection has previously been implicated at LCT (lactase) and major > histocompatibility complex (MHC), and within-UK differentiation at > 4p14 has been found independently, but others seem to be new findings. > All but three of the regions contain known genes. Aside from > evolutionary interest, genes showing evidence of natural selection are > particularly interesting for the biology of traits such as infectious > diseases; possible targets for selection include NADSYN1 (NAD > synthetase 1) at 11q13, which could have a role in prevention of > pellagra, as well as TLR1 (toll-like receptor 1) at 4p14, for which a > role in the biology of tuberculosis and leprosy has been suggested.

### Related Posts:

- [How predictive are known genetic factors for disease
  risk?](https://www.gnxp.com/WordPress/2008/11/19/how-predictive-are-known-genetic-factors-for-disease-risk/) - [Finding rare variants involved in
  disease](https://www.gnxp.com/WordPress/2009/03/05/finding-rare-variants-involved-in-disease/) - [Common variants are commonly
  unpromising](https://www.gnxp.com/WordPress/2010/04/01/common-variants-are-commonly-unpromising/) - [Reporting genome-wide association
  studies](https://www.gnxp.com/WordPress/2007/07/22/reporting-genome-wide-association-studies/) - [All diabetes, all the
  time](https://www.gnxp.com/WordPress/2007/02/15/all-diabetes-all-the-time/) - [Genome-wide association study for breast
  cancer…](https://www.gnxp.com/WordPress/2007/06/28/genome-wide-association-study-for-breast-cancer-susceptibility/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F06%2Fgenome-wide-association-studies-in-the-uk%2F&linkname=Genome-wide%20association%20studies%20in%20the%20UK "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F06%2Fgenome-wide-association-studies-in-the-uk%2F&linkname=Genome-wide%20association%20studies%20in%20the%20UK "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F06%2Fgenome-wide-association-studies-in-the-uk%2F&linkname=Genome-wide%20association%20studies%20in%20the%20UK "Email")[](https://www.addtoany.com/share)
