+++
title = "Jumping genes = new"
full_title = "Jumping genes = new species?"
date = "2006-09-08"
upstream_url = "https://www.gnxp.com/WordPress/2006/09/08/jumping-genes-new-species/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/09/08/jumping-genes-new-species/).

Jumping genes = new species?

As of this moment [RPM](http://scienceblogs.com/evolgen/) has not commented on a [new paper](http://www.sciencemag.org/cgi/content/full/313/5792/1448) in *Science*, [Gene Transposition as a Cause of Hybrid Sterility in *Drosophila*](http://www.sciencemag.org/cgi/content/full/313/5792/1448). Here is an [accessible summary](http://www.rochester.edu/news/show.php?id=2603). The basic idea is found in the abstract:

> …Genomic and molecular analyses show that *JYAlpha* > [transposed](https://en.wikipedia.org/wiki/Transposition_%28genetics%29) > to the third chromosome during the evolutionary history of the *D. > simulans lineage*. Because of this > [transposition](https://en.wikipedia.org/wiki/Transposition_%28genetics%29), > a fraction of hybrids completely lack *JYAlpha* and are sterile, > representing reproductive isolation without sequence evolution.

*JYAlpha* is implicated in sperm motility. In *D. melanogaster* is located on the 4^(th) chromosome. In *D. simulans* it is on the 3^(rd). A subsequent generation is produced where many males turn out to be sterile because they lack any *JYAlpha* at all!

  
What are the implications for this? What does it mean? First, I think one needs to keep in mind one of Dobzhansky’s theories about speciation, and that is the idea of [Dobznaksy-Muller incompabilities](https://www.google.com/search?hl=en&safe=off&sa=X&oi=spell&resnum=0&ct=result&cd=1&q=dobzhansky-muller+incompatibilities&spell=1).  
Here is the idea. Imagine you have two genes, loci if you will, 1 & 2.  
In the ancestral population loci 1 & 2 are fixed for alleles, the “ancestral alleles.” By fixed I mean that the frequency of the modal allele is north of 99%. Now, imagine that the ancestral population is split into two, perhaps by rising sea levels on an island. Now you have two “allopatric” populations.  
Next, introduce two ideas:  
1) Substitution of neutral alleles  
2) Epistasis  
[Epistasis](https://en.wikipedia.org/wiki/Epistasis) can be thought of as gene-to-gene interaction. The mechanical details are irrelevant in this narrative, but, imagine that locus 1 produces a protein which binds with the protein produced by locus 2 into a dimer which acts as a catalyst in an important biochemical pathway. As for substitution of neutral alleles, this derives from the insights of [Neutral Theory](https://en.wikipedia.org/wiki/Neutral_theory), which holds that the rate of substitution of neutral alleles is independent of population size and that transient polymoprhism is enough in many populations (especially large ones) to generate the appearence of diversity on many loci. We often think of neutral theory in the context of nonfunctional DNA, but, note that **if a new mutant which performs the exact same function as the old allele arises its probability of fixation is 1/2N_(e)**, just as you would expect with any neutral allele. This ignores the possibility of selective sweeps as positively favored alleles are fixed over time. In short, extinction is inevitable for almost any allele.  
Couple these two facts together with the reality of two separate demes, and you can enter into a situation where stochastic processes could build up variation across the loci **between** populations which maybe unbridgeable. In plain English the dance between locus 1 and locus 2 within the two populations may wander into distinct and alternative directions so that “partner switching” as would be possible in a readmixed population might cause serious problems.  
If in one population:  
locus 1 becames → locus 1a  
locus 2 becames → locus 2b  
And in another  
locus 1 becames → locus 1b  
locus 2 becames → locus 2a  
You may have the possibility that the descendent loci coevolved to bind with each other into the essential dimer, but no longer be able to couple with the ancestral partner. Specifically, imagine that 2b evolves in the genetic background of 1a, and 1b in the genetic background of 2a. The key here is that the the substitutions for the “a” alleles occurred so that new variants of “b” alleles emerged to couple with the peculiarities of the “a” alleles, **not** the ancestral alleles.  
Confused? Well, see the graph below. What does this have to do with the paper above? The authors show that genetic elements shifting rapidly can supercharge this process and result in powerful postzygotic barriers that might be important in speciation. In other words, these transposons are close to [speciation genes](https://www.google.com/search?hl=en&q=speciation+genes&btnG=Google+Search), though not quite (I think the paper is interesting, but more research needs to be done, and all that)….  
![dobz3.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2006/09/dobz3.jpg?w=640)![dobz3.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2006/09/dobz3.jpg?w=640)

### Related Posts:

- [Introductions to myself and my
  interests](https://www.gnxp.com/WordPress/2008/12/08/introductions-to-myself-and-my-interests/) - [What causes regulatory evolution between
  species?](https://www.gnxp.com/WordPress/2008/09/20/what-causes-regulatory-evolution-between-species/) - [Speciation
  genes](https://www.gnxp.com/WordPress/2008/12/16/speciation-genes/) - [Inter-species monkey
  lovin'](https://www.gnxp.com/WordPress/2007/08/25/inter-species-monkey-lovin/) - [Reconstructing human origins in the genomic
  era](https://www.gnxp.com/WordPress/2006/08/18/reconstructing-human-origins-in-the-genomic-era/) - [How long before the Y is incorporated into
  association…](https://www.gnxp.com/WordPress/2010/02/24/how-long-before-the-y-is-incorporated-into-association-studies/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F08%2Fjumping-genes-new-species%2F&linkname=Jumping%20genes%20%3D%20new%20species%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F08%2Fjumping-genes-new-species%2F&linkname=Jumping%20genes%20%3D%20new%20species%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F08%2Fjumping-genes-new-species%2F&linkname=Jumping%20genes%20%3D%20new%20species%3F "Email")[](https://www.addtoany.com/share)
