+++
title = "Desmond Tutu, Spaniards,"
full_title = "Desmond Tutu, Spaniards, and genetic distance"
date = "2010-08-21"
upstream_url = "https://www.gnxp.com/WordPress/2010/08/21/desmond-tutu-spaniards-and-genetic-distance/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/08/21/desmond-tutu-spaniards-and-genetic-distance/).

Desmond Tutu, Spaniards, and genetic distance

Since we’ve been talking about [Fst](https://en.wikipedia.org/wiki/Fixation_index) a fair amount, I thought it might be nice to put it in some concrete graphical perspective. First, to review Fst in the genetic context measures the proportion of genetic variation which can be attributed to between population differences. To give a “toy” example if you randomly divided the population of a large Swedish village into two groups, and calculated their Fst, it should be \~ 0, because if you randomly select from an unstructured population by definition there shouldn’t be noticeable between population differences. In contrast, if you compare a Swedish village to a Japanese village, a large fraction of the genetic variation is going to be distinct to each population. Around \~10% of the genetic variation in fact will be between the two groups. Many of the genes will be extremely informative, so that if you know the allelic state from a given individual you can predict with a high degree of certitude which population that individual was from (e.g., [*SLC24A5*](https://en.wikipedia.org/wiki/SLC24A5) and [*EDAR*](https://en.wikipedia.org/wiki/EDAR)). A small set of ancestrally informative alleles would produce a sequence of conditional probabilities of extremely high certitude (on the order of 10 genes for these two populations should suffice, perhaps three for “government work”).

But to put this in perspective, and show how genetic variation differs from locale to locale, I though I would compare continental-scale Fst values with that in a small region, southern Africa. The Fst values for the first I obtained from [Investigation of the fine structure of European populations with applications to disease association studies](http://www.nature.com/ejhg/journal/v16/n12/abs/ejhg2008210a.html), and the second, [Complete Khoisan and Bantu genomes from southern Africa](http://www.nature.com/nature/journal/v463/n7283/full/nature08795.html). The Bantu in this case is Desmond Tutu, who is from the Xhosa tribe, and has substantial admixture from the non-Bantu populations which were resident in South Africa prior to the arrival of the Bantus.

First, in tabular format:

[TABLE]

Second, two adjacent bar graphs. In the foreground I’ve simply take the Spain vs. other Eurasian population comparisons, while in the background Desmond Tutu is the reference for the four Bushmen.

![FinalscreenbushFinal](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/FinalscreenbushFinal.png?resize=561%2C566)![FinalscreenbushFinal](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/FinalscreenbushFinal.png?resize=561%2C566)

In some ways this comparison is an exaggeration of the variation in African genes. The Bushmen and Bantu populations are of very distinct origins, as the latter spread over eastern and southern Africa only in the last [2,000 years](https://en.wikipedia.org/wiki/Bantu_expansion). The Bushmen-Bantu cultural gap is one of sharp discontinuity, and despite gene flow it is still to some extent a genetic one as well. But there are other factors dampening Fst in this case. First, Tutu is himself of partial Khoisan ancestry (of whom there are other groups besides the Bushmen), so his genetic distance is likely to be smaller than someone from the Zulu tribe, which has presumably had less admixture with the indigenous populations, being a bit farther from the edge of the demographic “wave of advance.” Second, the gene chips are geared toward Eurasian populations, and presumably missed African, and particularly Bushmen, specific variants because they didn’t go looking.

My own confusion on these issues the past week illustrates I suppose the difficulty in mapping these abstruse and yet materially concrete patterns onto human categories. But quite often wrestling with the difficulties in the surest path to illumination.

### Related Posts:

- [Population Pairwise Fst on 250,000
  SNPs](https://www.gnxp.com/WordPress/2021/10/28/pairwise-fst-on-250000-snps/) - [1 migrant needed to prevent genetic
  divergence](https://www.gnxp.com/WordPress/2012/01/26/1-migrant-needed-to-prevent-genetic-divergence/) - [Ashkenazi Jews are more European in
  ancestry](https://www.gnxp.com/WordPress/2009/09/04/ashkenazi-jews-are-more-european-in-ancestry/) - [Evidence for natural selection between
  populations](https://www.gnxp.com/WordPress/2005/04/12/evidence-for-natural-selection-between-populations/) - [Visualizing caste & linguistic differences in
  India…](https://www.gnxp.com/WordPress/2009/09/27/visualizing-caste-linguistic-differences-in-india-fst/) - [Genetic Map of East
  Asia](https://www.gnxp.com/WordPress/2008/12/07/genetic-map-of-east-asia/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F21%2Fdesmond-tutu-spaniards-and-genetic-distance%2F&linkname=Desmond%20Tutu%2C%20Spaniards%2C%20and%20genetic%20distance "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F21%2Fdesmond-tutu-spaniards-and-genetic-distance%2F&linkname=Desmond%20Tutu%2C%20Spaniards%2C%20and%20genetic%20distance "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F21%2Fdesmond-tutu-spaniards-and-genetic-distance%2F&linkname=Desmond%20Tutu%2C%20Spaniards%2C%20and%20genetic%20distance "Email")[](https://www.addtoany.com/share)
