+++
title = "Chosen genes of the"
full_title = "Chosen genes of the Chosen People"
date = "2010-08-27"
upstream_url = "https://www.gnxp.com/WordPress/2010/08/27/chosen-genes-of-the-chosen-people/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/08/27/chosen-genes-of-the-chosen-people/).

Chosen genes of the Chosen People

![ashjewheadshot](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/ashjewheadshot.png?resize=200%2C226)![ashjewheadshot](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/ashjewheadshot.png?resize=200%2C226)Last spring two very thorough papers came out which surveyed the genetic landscape of the Jewish people (my posts, [Genetics & the Jews it’s still complicated](http://blogs.discovermagazine.com/gnxp/2010/06/genetics-the-jews-its-still-complicated/), [Genetics & the Jews](http://blogs.discovermagazine.com/gnxp/2010/06/genetics-the-jewish-question/)). The novelty of the results was due to the fact that the research groups actually looked across the very diverse populations of the Diaspora, from Morocco, Eastern Europe, Ethiopia, to Iran. They constructed a broader framework in which we can understand how these populations came to be, and how they relate to each other. Additionally, they allow us to have more perspective as to the generalizability of medical genetics findings in the area of “Jewish diseases,” which for various reasons usually are actually findings for Ashkenazi Jews (the overwhelming majority of Jews outside of Israel, but only about half of Israeli Jews).

Just as the two aforementioned papers were deep explorations of the genetic history of the Jewish people, and allowed for a systematic understanding of their current relationships, a new paper in *PNAS* takes a slightly different tack. First, it zooms in on Ashkenazi Jews. The Jews whose ancestors are from the broad swath of Central Europe, and later expanded into Poland-Lithuania and Russia. The descendants of Litvaks, Galicians, and the assimilated Jewish minorities such as the Germans Jews. Second, though constrained to a narrower population set, the researchers put more of an emphasis on the evolutionary parameter of natural selection. Like any population Jews have been impacted by drift, selection, migration (and its variant admixture), and mutation. Teasing apart these disparate parameters may aid in understanding the origin of Jewish diseases.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)The paper is open access, so you don’t have to take my interpretation as the last word. [Signatures of founder effects, admixture, and selection in the Ashkenazi Jewish population](http://www.pnas.org/content/early/2010/08/23/1004381107.abstract):

> The Ashkenazi Jewish (AJ) population has long been viewed as a genetic > isolate, yet it is still unclear how population bottlenecks, > admixture, or positive selection contribute to its genetic structure. > Here we analyzed a large AJ cohort and found higher linkage > disequilibrium (LD) and identity-by-descent relative to Europeans, as > expected for an isolate. However, paradoxically we also found higher > genetic diversity, a sign of an older or more admixed population but > not of a long-term isolate. Recent reports have reaffirmed that the AJ > population has a common Middle Eastern origin with other Jewish > Diaspora populations, but also suggest that the AJ population, > compared with other Jews, has had the most European admixture. Our > analysis indeed revealed higher European admixture than predicted from > previous Y-chromosome analyses. Moreover, we also show that admixture > directly correlates with high LD, suggesting that admixture has > increased both genetic diversity and LD in the AJ population. > Additionally, we applied extended haplotype tests to determine whether > positive selection can account for the level of AJ-prevalent diseases. > We identified genomic regions under selection that account for lactose > and alcohol tolerance, and although we found evidence for positive > selection at some AJ-prevalent disease loci, the higher incidence of > the majority of these diseases is likely the result of genetic drift following a bottleneck. **Thus, the AJ population shows evidence of past founding events; however, admixture and selection have also strongly influenced its current genetic makeup.**

The sample size of Ashkenazi Jews was \~400, and they looked at \~700,000 SNPs. As I said, how Jews relate to other populations really isn’t at the core of this paper as it was in the earlier ones from the spring, but there were the PCA plots (sorry [Mike](http://scienceblogs.com/mikethemadbiologist/)), a *frappe* bar plot, and a phylogenetic tree derived from Fst statistic. Again, remember that PCA is showing you the largest independent components of genetic variation within the data. The bar plot has a set of ancestral populations of which individuals are composites of. And finally, Fst measures between population component of genetic variation. The larger the Fst across two populations the bigger the genetic distance.

\[nggallery id=6\]

Using the Druze & Palestinians as the ancestral Middle Eastern reference the authors estimated that the European admixture into Ashkenazi Jews is on the order of 30-55%. This is in the same ballpark as the previous studies, so no great surprise. As I stated in earlier posts **the authors can spin the same results in very different ways.** From what I can tell these authors are inclined to emphasize the strong possibility that in terms of genetic distance Ashkenazi Jews are somewhat closer to Europeans than they are to Levantine Arabs. Of course these sorts of assertions need to be handled with care. The genetic distance between Ashkenazi Jews and Tuscans is less than half that between Ashenazi Jews and Russians, while the Jewish-Russian value is about 50% larger than the Jewish-Palestinian one. Remember that there’s a fair amount of circumstantial evidence that [Tuscans](http://blogs.discovermagazine.com/gnxp/2007/06/lydians-etruscans/) may themselves be a relatively recent hybrid population between indigenous residents of the Italian peninsula and Near Easterners.

![ashjtab1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/ashjtab1.png?resize=350%2C251)![ashjtab1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/ashjtab1.png?resize=350%2C251)One thing that this paper does do is rebut any strong assertion that Ashkenazi Jews are a genetically homogeneous population which went through a powerful bottleneck. Basically, the idea that Jewish diseases are just an outcome of the operational inbreeding that occurs when genetic variation is expunged from a population through low effective population size. The clincher seems to be comparison of heterozygosity of Ashkenazi Jews and gentile Europeans. The former are actually somewhat more heterozygous than the latter. There’s been a bit of evidence from previous research that the long term effective population size of Ashkenazi Jews was not necessarily very small, so this isn’t a total surprise. Remember that heterozygosity simply means the fraction of individuals heterozygous at a locus.

One way you can become heterozygous is naturally admixture. Remember that populations differ across many genes. As an example, there’s a pigmentation gene, *SLC24A5*, where all Europeans are at one state, and all West Africans in another. Naturally African Americans exhibit much more heterozygosity on this locus than the ancestral populations. The Ashkenazi Jewish case is less extreme because the two parental populations are genetically closer, but the principle still holds.

A consequence of recent admixture between genetically different populations are high levels of [linkage disequilibrium](https://en.wikipedia.org/wiki/Linkage_disequilibrium), non-random associations of alleles at different loci across the genome. Why? **There are many genes where two populations may be very different.** Offspring inherit half their genome from one parent, and half from the other, and the parents pass along to their offspring particular associations of alleles. There may be a set of European distinctive alleles on a chromosome, and an African distinctive set of alleles, so that in a hybrid individual the alleles are strongly correlated across loci. These associations are broken down over time by [recombination](https://en.wikipedia.org/wiki/Recombination). The regularity of this process can serve as a clock with which to measure the period since admixture. African Americans were used to calibrate the time since admixture for the Uyghur people of western China, who are mixed from West and East Eurasian populations. The authors did not do this in this paper, I assume because the ancestral populations were genetically rather close in comparison to the two above examples, so there’d be less linkage disequilibrium to break down in the first place.

In the Ashkenazi Jewish population they found more linkage disequilibrium than in Europeans as well as longer [haplotypes](https://en.wikipedia.org/wiki/Haplotype). This could be the result of a population bottleneck where drift could drive up the frequency of blocks of the genome, but as they note in the paper that should probably reduce heterozygosity. The natural inference then is that admixture between distinct populations can explain both data points.

![ashslselect](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/ashslselect.png?resize=350%2C487)![ashslselect](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/ashslselect.png?resize=350%2C487)But let’s cut to the chase. What genes exhibit signatures of natural selection in Ashkenazi Jews? More precisely, what distinctive regions of the genome exhibit signatures of natural selection? They used the standard haplotype type based methods. Basically you’re looking for regions of the genome where there are long blocks of correlated alleles, signs of a selective sweep due to a favored variant which dragged along flanking genomic regions as it rose rapidly in frequency, more rapidly than recombination could break apart the associations. Because recombination does breaks up associations over time, you need the selective sweeps to be relatively recent to detect them with these methods. Since the Jewish people, and Ashkenazi Jews more particularly, are relatively recent historically timing shouldn’t be an issue for Jewish specific sweeps. But another factor is that the two primary tests they used, EHH and iHS, are not good at picking up sweeps which are just starting. EHH is geared toward sweeps which are almost complete, so the frequency of the selected allele is near 100%. iHS is better are mid-range values. Using a combination of these two techniques they found that six genes which are implicated in diseases characteristic of Ashkenazi Jews have the hallmarks of natural selection. Natural selection is self-evident, so what seems to have been going here is that the disease was simply a side effect or byproduct of [adaptation](http://blogs.discovermagazine.com/gnxp/2010/07/disease-as-a-byproduct-of-adaptation/).

The strongest signal they found was in *ALDH2*. The strongest signal in Europeans, *LCT*, was not found in Ashkenazi Jews. But is *LCT* a strong signal in Europeans? Many Southern European populations have low frequencies of the derived *LCT* allele, indicating that they haven’t been subject to strong selection for lactase persistence. These are the same populations genetically close to the Ashkenazi Jews. The authors suggest that the Jewish-European admixture occurred before the sweep of the derived *LCT* allele, but it seems more plausible that the Ashkenazim simply admixed with a European population, such as Italians, which do not exhibit much lactase persistence. As for *ALDH2*, the association between genetic variation on this locus and alcoholism is well known, and has been used to explain the low Jewish rates of the disease. In this case, the authors posit that protection from alcoholism is a positive side effect of natural selection:

> The mechanism driving selection of the ALDH2 locus is unknown, but a > plausible target of selection also within this selected region is the > TRAFD1/FLN29 gene, which is a negative regulator of the innate immune > system, important for controlling the response to bacterial and viral > infection (49). TRAFD1/FLN29 may have conferred a selective advantage > in the immune response to a pathogen, perhaps near the time that the > Jews returned to Israel from their Babylonian captivity. Despite the unclear selective mechanism, this remains a remarkable example of a putatively selected region accounting for a known population phenotype.

Many of the other loci naturally did not show signatures of natural selection. But this sort of work is exploratory, and there are limits to the power of their techniques. As it is, it seems that we’re very far along on understanding the phylogenetic tree of the Jewish people, and we’re finally getting a grip on the exogenous parameters which might prune the branches.

**Citation:** Steven M. Bray, Jennifer G. Mulle, Anne F. Dodd, Ann E. Pulver, Stephen Wooding, & Stephen T. Warren (2010). Signatures of founder effects, admixture, and selection in the Ashkenazi Jewish population PNAS : [10.1073/pnas.1004381107](http://www.pnas.org/content/early/2010/08/23/1004381107.abstract)

**Related:** [John Hawks](http://johnhawks.net/), [New data on Ashkenazi population history](http://johnhawks.net/weblog/reviews/genomics/selection/bray-ashkenazi-2010.html).

*Image Credit: Wikimedia*

### Related Posts:

- [Jews and
  genetics](https://www.gnxp.com/WordPress/2010/06/06/jews-and-genetics/) - [More Jews, fewer
  markers](https://www.gnxp.com/WordPress/2010/07/16/more-jews-fewer-markers/) - [Jewish genes, Jewish history, Jewish
  identity](https://www.gnxp.com/WordPress/2008/09/02/jewish-genes-jewish-history-jewish-identity/) - [Ashkenazi Jews are more European in
  ancestry](https://www.gnxp.com/WordPress/2009/09/04/ashkenazi-jews-are-more-european-in-ancestry/) - [Ashkenazi Jews are Middle Eastern & European
  hybrids](https://www.gnxp.com/WordPress/2009/12/09/ashkenazi-jews-are-middle-eastern-european-hybrids/) - [The origins of Ashkenazi Jews near
  resolution](https://www.gnxp.com/WordPress/2016/07/20/the-origins-of-ashkenazi-jews-near-resolution/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F27%2Fchosen-genes-of-the-chosen-people%2F&linkname=Chosen%20genes%20of%20the%20Chosen%20People "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F27%2Fchosen-genes-of-the-chosen-people%2F&linkname=Chosen%20genes%20of%20the%20Chosen%20People "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F27%2Fchosen-genes-of-the-chosen-people%2F&linkname=Chosen%20genes%20of%20the%20Chosen%20People "Email")[](https://www.addtoany.com/share)
