+++
title = "PCA, Razib around the"
full_title = "PCA, Razib around the world"
date = "2010-08-10"
upstream_url = "https://www.gnxp.com/WordPress/2010/08/10/pca-razib-around-the-world-a-little/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/08/10/pca-razib-around-the-world-a-little/).

PCA, Razib around the world (a little)

![price_fig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/price_fig1.jpg?resize=350%2C249)![price_fig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/price_fig1.jpg?resize=350%2C249)I have put up a few [posts](http://blogs.discovermagazine.com/gnxp/2010/05/beyond-visualization-of-data-in-genetics/) warning readers to be careful of confusing PCA plots with real genetic variation. PCA plots are just ways to capture variation in large data sets and extract out the independent dimensions. Its great at detecting population substructure **because the largest components of variation often track [between population differences](https://en.wikipedia.org/wiki/Lewontin's_Fallacy), which consist of sets of correlated allele frequencies.** Remeber that PCA plots usually are constructed from the two largest dimensions of variation, so they will be drawn from just these correlated allele frequency differences between populations which emerge from historical separation and evolutionary events. Observe that African Americans are distributed along an axis between Europeans and West Africans. Since we know that these are the two parental populations this makes total sense; the between population differences (e.g., [*SLC24A5*](https://en.wikipedia.org/wiki/SLC24A5) and [Duffy](https://en.wikipedia.org/wiki/Duffy_antigen)) are the raw material from which independent dimensions can pop out. But on a finer scale one has to be cautious because the distribution of elements on the plot as a function of principal components is sensitive to the variation you input to generate the dimensions in the first place.

I can give you a concrete example: **me.** I showed you my [23andMe ancestry](http://blogs.discovermagazine.com/gnxp/files/2010/08/zdna.png) painting yesterday. I didn’t show you my position on the HGDP data set because I’ve shared genes with others and I don’t want to take the step of displaying other peoples’ genetic data, even if at a remove. But, I have reedited some “demo” screenshots and placed where I am on the plot to illustrate what I’m talking about above. The first shot is my position on the two-dimensional plot of first and second principal components of genetic variation from the HGDP data set.

  
![gsa-lillymendel-world](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/gsa-lillymendel-world.png?resize=350%2C352)![gsa-lillymendel-world](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/gsa-lillymendel-world.png?resize=350%2C352)No surprise that I’m in the Central/South Asian cluster. But what may surprise you is that I’m not in the South Asian cluster, I’m in the Central Asian cluster. In the Central Asian cluster are Uyghurs and Hazaras. These are two hybrid populations, a mixture of West and East Eurasian elements. The Uyghurs are likely the outcome of a process of admixture between the Iranian and Tocharian Indo-European populations of the cities of the Tarim basin, and later Turkic speaking settlers who arrived in the wake of the expansion and later collapse of the first Uyghur Empire (the historical connection between the current Uyghurs and ancient Uyghurs is tenuous at best, and complicated). The Hazaras are a more recent population, likely emerging as the product of intermarriages between Mongol soldiers who arrived in the 13th century, and indigenous women, Persians, Turks, and assorted Indo-Iranian groups between the Zagros and Khyber Pass. It is somewhat ironic that I’m on the edge of the Hazara cluster since they are almost certainly in part descended from Genghis Khan’s family, and my own surname is Khan. But I know that my Y chromosomal lineage is R1a1, very common across Central and Southern Eurasia, and not a Mongolian one at all.

![23andmepcazoom](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/23andmepcazoom1.png?resize=350%2C355)![23andmepcazoom](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/23andmepcazoom1.png?resize=350%2C355)**Zoom!** Now we’ve constrained the input data set to the Central/South Asian groups. First, look at the Kalash. They’re strange, which is no surprise, they’re an inbred mountain group in Pakistan who have not adopted Islam. The Pakistani Taliban looks to be [ending them as we speak](http://www.telegraph.co.uk/news/6214794/Taliban-targets-descendants-of-Alexander-the-Great.html). I really would prefer that they were just thrown out of the data set for this zoom view, because on this fine grained scale I don’t think they add much at all. They’re just an example of what long term endogamy can do to your allele frequencies. The bigger picture is the axis between the populations of Pakistan, and those of Central Asia. **Observe that I’ve changed position.** Whereas when taking world wide genetic variation into account I clustered with Central Asians, now I’m 2/3 of the way to the South Asian cluster. I will tell you that I’ve shared “genes” with around 50 South Asians now, from various parts of the subcontinent, and in the 23andMe plot they overlay the South Asians nearly perfectly. I’ve put labels at the approximate ethno-linguistic position. I’m an outlier. 23andMe tells me that I’m 43% “East Asian.” The typical South Asian is in the 10-30% range. My first assumption was that I have a lot of [ancient South Indian](http://www.nature.com/nature/journal/v461/n7263/abs/nature08365.html), which just shows up as East Asian in their algorithm. With this in mind I tried sharing with a lot of South and East Indians, and found out two interesting points. First, South Indians seem no higher than 30-35% East Asian. Bengalis on the other hand are more East Asian, with Bangladeshis more East Asian than West Bengalis. My sample size for Bengalis is small, so take that with caution. Second, the PCA plots put the South Indians firmly in the South Asian cluster, but the Bengalis trail out toward my own position. This indicates again that different methods are telling you slightly different things. The PCA is only a thin slice of variation, but it’s highly informative of between population differences. A Bengali and a South Indian with the same “East Asian” fraction in the ancestry painting nevertheless have consistently different positions on the PCA, with Bengalis closer to the East Asians. Additionally, there’s an ethnic Persian in this zoom plot that I’m describing, and they are positioned near the Balochi. But on the world wide plot they’re on the margins of the European cluster. Another illustration that position of an element is sensitive to the input data because of how the dimensions are generated.

[Blaine Bettinger](http://www.thegeneticgenealogist.com/2010/08/02/explore-your-genetic-heritage/), who inspired me to post this, told a story with his ancestry painting which was plausible. What can I say? First, I have less than 1% African ancestry. This could be noise. But, I do observe that the South Asians with Muslim names are enriched in the set of those who I’ve shared genes with and who have less than 1%, but not 0%, African ancestry. Just as Muslim South Asians have non-trivial West Asian ancestry, I suspect that many of us have Sub-Saharan African ancestry through the same dynamic. Sub-Saharan African soldiers were prominent across South Asia with the arrival of Muslims. Bengal even has a period of rule by [Abyssinian rulers](https://en.wikipedia.org/wiki/History_of_Bengal#Abyssinian_rulers). But the bigger issue for me is the East Asian component. Here is a figure from a [paper](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.0020215) published 4 years ago:

![journal.pgen.0020215.g005](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/journal.pgen.0020215.g005.png?resize=600%2C504)![journal.pgen.0020215.g005](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/journal.pgen.0020215.g005.png?resize=600%2C504)

The figure is showing Fst value comparing Indian Americans with Europeans and East Asians. Fst measures between population differences in allele frequency, in this case the alleles being 207 [indels](https://en.wikipedia.org/wiki/Indel). Take a look at the Bengalis. These are *West* Bengalis, who I believe have a lesser East Asian component, but even there the allele frequency difference to East Asians is near that of Europeans. The Assamese, who speak a language very close to Bengali, are similar. Assam was ruled by a [Tibeto-Burman people](https://en.wikipedia.org/wiki/Ahoms) for nearly 600 years. The Oriya speakers, from the southwest of Bengal, are more distant from East Asians. As one goes south and east, and west and north, the distance from East Asians increases. This shouldn’t be that surprising, but nice to confirm. The fact that the genetic distance increases as one goes south means that for northeast South Asia you need to complexify the model from a two-way admixture with “ancient North Indians” and “ancient South Indians.” Set next to these two is an East Asian element, which is also clear in the Indo-Aryan peoples of Nepal.

![Sheikh Hasina, Khaleda Zia](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/Sheikh-Hasina-Khaleda-Zia.jpg?resize=300%2C216)![Sheikh Hasina, Khaleda Zia](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/Sheikh-Hasina-Khaleda-Zia.jpg?resize=300%2C216)Of course anyone who knows Bengalis won’t be totally surprised by an East Asian component to their ancestry. To the left are head shots of the two women who have dominated Bangladeshi politics for the past two decades, Khaleda Zia and Sheik Hasina. They’re both Bengalis, but they do look different, and I know many people who look like one or the other (or a combination). My family is from one of most easternmost districts of Bengali, next to [Tripura](https://en.wikipedia.org/wiki/Tripura). In fact my late maternal grandmother lived in Tripura for some of her childhood (she was almost trampled to death by the Maharani of Tripura’s insane elephant as a young girl!). When I was a young child I once saw a black and white photo from my father’s college days, and I was curious who the Asiatic looking young man in the middle of the photograph was. Turns out it was my father! Sometimes our expectations affect how we perceive people. I have never perceived my father to have an Asian cast to his features as a more mature man, but others have told me that he does still exhibit them.

There is still the question of how Bengalis came to have this particular admixture. I think the most plausible scenario probably synthesizes conventional village-to-village intermarriage and isolation-by-distance, along with some component of migrationism. Tribes such as the Chakma have left Burma in historical time. The [Chakma](https://en.wikipedia.org/wiki/Chakma_people) of Bangladesh now speak a dialect of Bengali, not their ancestral Sino-Tibetan tongue. I believe that a non-trivial portion of Bengalis have ancestors who were tribal people who shifted their religious identity to that of Hinduism or Islam (from Theravada Buddhism in the case of the Chakma, or animism in the case of the [Garos](https://en.wikipedia.org/wiki/Garo_people) before their Christianization). But eastern South Asia is adjacent to mainland Southeast Asia, and it stands to reason that continuous gene flow would over time would also have introduced East Asian alleles into the Bengali gene pool.

*Image Credit: TopNews.in*

### Related Posts:

- [PCA plots and
  trees](https://www.gnxp.com/WordPress/2010/06/01/pca-plots-and-trees/) - [The genetic world in
  3-D](https://www.gnxp.com/WordPress/2011/03/24/the-genetic-world-in-3-d/) - [What L. L. Cavalli-Sforza got
  wrong?](https://www.gnxp.com/WordPress/2008/04/21/what-l-l-cavalli-sforza-got-wrong/) - [Just pushing
  buttons](https://www.gnxp.com/WordPress/2010/08/24/just-pushing-buttons/) - [Population structure using haplotype
  data](https://www.gnxp.com/WordPress/2012/01/28/population-structure-using-haplotype-data/) - [A picture is worth a thousand words, part
  n](https://www.gnxp.com/WordPress/2008/04/21/a-picture-is-worth-a-thousand-words-part-n/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F10%2Fpca-razib-around-the-world-a-little%2F&linkname=PCA%2C%20Razib%20around%20the%20world%20%28a%20little%29 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F10%2Fpca-razib-around-the-world-a-little%2F&linkname=PCA%2C%20Razib%20around%20the%20world%20%28a%20little%29 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F10%2Fpca-razib-around-the-world-a-little%2F&linkname=PCA%2C%20Razib%20around%20the%20world%20%28a%20little%29 "Email")[](https://www.addtoany.com/share)
