+++
title = "People, not pots, in"
full_title = "People, not pots, in Africa"
date = "2010-08-29"
upstream_url = "https://www.gnxp.com/WordPress/2010/08/29/people-not-pots-in-africa/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/08/29/people-not-pots-in-africa/).

People, not pots, in Africa

![324_1035_F5](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/324_1035_F5.png?resize=300%2C210)![324_1035_F5](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/324_1035_F5.png?resize=300%2C210)[Last weekend](http://blogs.discovermagazine.com/gnxp/2010/08/genetic-variation-within-africa-and-the-world/) I mentioned a paper, [The Genetic Structure and History of Africans and African Americans](http://www.sciencemag.org/cgi/content/full/324/5930/1035), which had the best coverage of disparate African populations we’ve seen so far. The map to the left shows the various ancestral population clusters inferred from the samples they had. Really the only failing is that they didn’t have samples from Angola, Zambia, Zimbabwe and Mozambique. Unfortunately, that’s not totally trivial. These are regions which were effected by the [Bantu Expansion](https://en.wikipedia.org/wiki/Bantu_expansion), with southern Angola in particular still having remnants of [Khoisan language](https://en.wikipedia.org/wiki/Khoisan_languages) speakers which likely attest to the pre-Bantu populations. Luckily for us innovation and scientific ingenuity are such that minor questions can quickly be answered because of how cheap the basic methods have become. A new paper in *The European Journal of Human Genetics* tackles Mozambique in particular, and discerns a heretofore unknown possible population cluster. [A genomic analysis identifies a novel component in the genetic structure of sub-Saharan African populations](http://www.nature.com/ejhg/journal/vaop/ncurrent/abs/ejhg2010141a.html):

> Studies of large sets of single nucleotide polymorphism (SNP) data > have proven to be a powerful tool in the analysis of the genetic > structure of human populations. In this work, we analyze genotyping > data for 2841 SNPs in 12 sub-Saharan African populations, including a > previously unsampled region of southeastern Africa (Mozambique). We > show that robust results in a world-wide perspective can be obtained > when analyzing only 1000 SNPs. Our main results both confirm the > results of previous studies, and show new and interesting features in > sub-Saharan African genetic complexity. There is a strong > differentiation of Nilo-Saharans, much beyond what would be expected > by geography. Hunter-gatherer populations (Khoisan and Pygmies) show a > clear distinctiveness with very intrinsic Pygmy (and not only Khoisan) > genetic features. Populations of the West Africa present an unexpected > similarity among them, possibly the result of a population expansion. Finally, we find a strong differentiation of the southeastern Bantu population from Mozambique, which suggests an assimilation of a pre-Bantu substrate by Bantu speakers in the region.

The main value-add of the research were the 279 individuals from Mozambique, who they plugged into previous data sets (e.g., HGDP, HapMap3). It must also be noted that they limited their genetic survey to \~2800 SNPs.This is sufficient for their purposes. Below are the figures of interest from the paper. Note immediately how Mozambique separates out at K = 4 in the first image. The subsequent figures are from PCA. The axes represent components of variation. The last panel shows a PCA plot transposed onto a map. In this case, PC 1 & PC3.t

\[nggallery id=7\]

The first figure is important **because it suggests population structure we hadn’t known of in the Bantu Expansion.** This doesn’t mean that it should be surprising. With Africa’s current level of genetic variation it seems implausible that the carriers of the Bantu culture would not have assimilated other groups along the wave of advance. In fact, as a cultural movement gains steam through positive feedback loops different societies may become co-opted into them, and spread the culture in their own turn. As an American example, I will give the Irish American Catholic hierarchy’s campaigns against German language parochial school instruction in the 19th century. Old English aside the language of the Irish was originally not English, but by the early 19th century apparently English had already become [dominant](https://en.wikipedia.org/wiki/History_of_the_Irish_language#Nineteenth_and_twentieth_centuries) among the Roman Catholic peasantry of Ireland. So they brought English, not Gaelic, to the United States. Similarly, the spread of Islam in India occurred predominantly under the ageis of Turks and Afghans, not Arabs, while the spread of Islam in Southeast Asia was promoted by South Asian Muslim merchants in their turn. So you have Arab cultural forms in eastern Indonesia thanks to cultural expansions at two removes from the original Arab source (in fact, it could be argued that the Turks and Afghans were Islamicized through a Persian intermediate as well).

But it is the PCA plots which are of more curiosity for me. They note that **it is the third component of variation which maps well onto geographic distance.** In the paper they say:

> This is the PC that is mostly correlated with geography…and the fact > that it is the third rather than the first component, as would be > expected if isolation by distance was the predominant force shaping > genetic diversity…implies that directional population movements (such > as the Bantu expansion) and barriers to gene flow (such as that between food producers and hunter gatherers) are more relevant than geographic distance to understand the genetic landscape of sub-Saharan Africa….

There were folk migrations in Africa. They might simply not have been the ones we are aware of, at least in our sparest conceptions. Those folk migrations were very recent, within the last \~2,000 years or so. Which is why the distinctive correlations between language and genes persist, especially on the outer edge of the wave of advance in southern Africa (in contrast, the Pygmies of the Congo have lost their native language, and the western Pygmies are highly admixed with their neighbors).

**Citation:** [A genomic analysis identifies a novel component in the genetic structure of sub-Saharan African populations](https://dx.doi.org/10.1038/ejhg.2010.141)

**Addendum:** The life of [Shaka](https://en.wikipedia.org/wiki/Shaka) may give us a clue as the disturbances which pushed the Bantu ever outward.

### Related Posts:

- [Genetic variation within Africa (and the
  world)](https://www.gnxp.com/WordPress/2010/08/22/genetic-variation-within-africa-and-the-world/) - [Africans Americans mostly West African, but some
  mostly…](https://www.gnxp.com/WordPress/2009/12/23/africans-americans-mostly-west-african-but-some-mostly-european/) - [African genetic
  structure](https://www.gnxp.com/WordPress/2009/04/30/african-genetic-structure/) - [The Bushmen & the
  Bantu](https://www.gnxp.com/WordPress/2010/02/17/the-bushmen-the-bantu/) - [The secret history of
  Africa](https://www.gnxp.com/WordPress/2009/04/17/the-secret-history-of-africa/) - [Ancient human population
  sizes](https://www.gnxp.com/WordPress/2007/12/12/ancient-human-population-sizes/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F29%2Fpeople-not-pots-in-africa%2F&linkname=People%2C%20not%20pots%2C%20in%20Africa "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F29%2Fpeople-not-pots-in-africa%2F&linkname=People%2C%20not%20pots%2C%20in%20Africa "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F29%2Fpeople-not-pots-in-africa%2F&linkname=People%2C%20not%20pots%2C%20in%20Africa "Email")[](https://www.addtoany.com/share)
