+++
title = "Monophyletic Australian"
full_title = "Monophyletic Australian Marsupials"
date = "2010-08-04"
upstream_url = "https://www.gnxp.com/WordPress/2010/08/04/monophyletic-australian-marsupials/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/08/04/monophyletic-australian-marsupials/).

Monophyletic Australian Marsupials

![551px-Monito_del_Monte_ps6](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/551px-Monito_del_Monte_ps6.jpg?resize=200%2C218)![551px-Monito_del_Monte_ps6](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/551px-Monito_del_Monte_ps6.jpg?resize=200%2C218)Though I don’t blog about the topic with the breadth and depth of individuals such as [Brian Switek](http://brianswitek.com/) or [Darren Naish](http://scienceblogs.com/tetrapodzoology/) I do take some interest in [natural history](https://en.wikipedia.org/wiki/Natural_history). This is the domain which was my original focus as a child when it came to science, and I continue to observe it from afar with great fondness. General questions, such as the role of contingency and necessity in the arc of evolution, are obviously the sort of issue which natural history can be brought to bear upon. But I also have a fascination with specific, often anomalous details. For example, the [Monito del Monte](https://en.wikipedia.org/wiki/Monito_del_Monte) of Chile is generally held to be more closely related to the marsupials of Australia than those of the New World. It is the only extant member of the order [Microbiotheria](https://en.wikipedia.org/wiki/Microbiotheria), and its connection to Australian marsupials is one of those surprises which go to show you why science is done in the field, and not just theorized from your *a priori* beliefs. It’s why you play the game, and don’t simply allow the handicapping professionals to decide wins and losses.

A new paper in *PLoS Biology* explores the phylogenetic relationship of Australian and New World marsupials through a more robust genomically focused technique. Though the method has a “in silico” spin, the basics seem to be grounded in [cladistics](https://en.wikipedia.org/wiki/Cladistics). **Look for derived characters which can indicate [monophyly](https://en.wikipedia.org/wiki/Monophyly).** Monophyly simply means that all of a set of organisms descend from one common ancestor. So, famously, the class of [reptiles](https://en.wikipedia.org/wiki/File:Phylogenetic-Groups.svg.png) is *not* monophyletic. Some of the descendants of the common ancestors of all reptiles are not included within the class, birds. Earlier generations of taxonomists tended to classify organisms based on their characters, and the set of characters which they chose for reptiles included groups, such as crocodiles and tortoises, which were genetically very distant (when compared to crocodiles and birds). Though anatomically informative, these sorts of taxonomic classifications misled one as to evolutionary history. Not a minor matter. Ergo, the rise of cladistic techniques which replaced intuition with a more formal hypothetico-deductive framework. Because of its generality as a method naturally you can substitute genetic loci for morphological character traits, and so you get papers such as the one below.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)[Tracking Marsupial Evolution Using Archaic Genomic Retroposon Insertions](http://www.plosbiology.org/article/info:doi/10.1371/journal.pbio.1000436):

> Ever since the first Europeans reached the Australian shores and were > fascinated by the curious marsupials they found, the evolutionary > relationships between the living Australian and South American > marsupial orders have been intensively investigated. However, neither > the morphological nor the more recent molecular methods produced an > evolutionary consensus. Most problematic of the seven marsupial groups > is the South American species Dromiciops gliroides, the only survivor > of the order Microbiotheria. Several studies suggest that Dromiciops, > although living in South America, is more closely related to > Australian than to South American marsupials. This relationship would > have required a complex migration scenario whereby several groups of > ancestral South American marsupials migrated across Antarctica to > Australia. We screened the genomes of the South American opossum and > the Australian tammar wallaby for retroposons, unambiguous > phylogenetic markers that occupy more than half of the marsupial > genome. From analyses of nearly 217,000 retroposon-containing loci, we > identified 53 retroposons that resolve most branches of the marsupial > evolutionary tree. **Dromiciops is clearly only distantly related to > Australian marsupials, supporting a single Gondwanan migration of marsupials from South America to Australia.** The new phylogeny offers a novel perspective in understanding the morphological and molecular transitions between the South American and Australian marsupials.

[Retroposons](https://en.wikipedia.org/wiki/Retroposon) are genetic elements which insert randomly throughout the genome, and rarely in the same location in across lineages. This avoids “false positives” where you observe genetic features across taxa which you incorrectly infer to indicate a phylogenetic relationship. The pattern of variation of randomly distributed distinctive retroposons can theoretically be used to map the sequence of relatedness of the same genes ([orthologous](https://en.wikipedia.org/wiki/Homology_(biology)#Orthology)) across species. Retroposon insertions copious within the marsupial genome, so naturally they’re a good candidate for markers which might exhibit the distinctiveness necessary to explore deep time evolutionary relationships. Additionally retroposons can nest within each other, within newer insertion events overlain over older ones, so that they create a sort of genetic palimpsest. These researchers filtered the loci harboring retroposons down to 53 which were particularly informative for relationships across the marsupial species for which they had genomic data, two species per order excluding orders without more than one species. The two species within each order were selected from lineages which were presumed to exhibit the deepest evolutionary split within the clade.

Granted, it isn’t as if taxonomists haven’t been interested in the relationships of marsupial mammals. As noted in the paper the nature of the phylogenetic tree frames plausible hypotheses which explain the current biogeographic pattern we see. Where there are two sets of marsupial mammals separated by the Pacific, but where the spatial pattern does not perfectly correspondent to the phylogenetic relationship. Here is a figure from a 2004 [paper](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6T39-4D98XVW-7&_user=10&_coverDate=10/13/2004&_rdoc=1&_fmt=high&_orig=search&_sort=d&_docanchor=&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=76c0cfc900b11c4116ac4dc61bce5ab5):

![0](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/0.jpeg?resize=600%2C359)![0](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/0.jpeg?resize=600%2C359)

Australian and South American marsupials are color coded. As you can see, *Dromiciops*, Monito del Monte, is nested within the monophyletic clade which includes all the Australian mammals. But, the aforementioned paper was based on mitochondrial DNA. The DNA passed along the maternal lineage, easy to extract and amplify, as well as analyze (because of the lack of recombination). But for the purposes of exposing such deep time relationships mtDNA may not be optimal, and should not be the last word.

Much of the “guts” of the paper was obviously computational, and wasn’t explored in detail within the text. So let’s jump to the outcome, the new branch of the tree of life for marsupials:

![journal.pbio.1000436.g002](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/journal.pbio.1000436.g002.png?resize=449%2C600)![journal.pbio.1000436.g002](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/journal.pbio.1000436.g002.png?resize=449%2C600)

Ah, **now you see that Australian marsupials are a monophyletic clade!** The Monito del Monte is no longer nested within their own lineage, but is now an outgroup. It would be peculiar if it was not the closest of the outgroups, so its positioning is reasonable in terms of what we’d expect. From the discussion:

> Given the limitations just mentioned, the retroposon marker system > identified a clear separation between the South American and > Australasian marsupials. **Thus, the current findings support a simple > paleobiogeographic hypothesis, indicating only a single effective migration from South America to Australia**, which is remarkable given that South America, Antarctica, and Australia were connected in the South Gondwanan continent for a considerable time. >
> The search for diagnostic South American or Australidelphian marsupial > morphological characters has been so far confounded by the lack of a resolved marsupial phylogeny…The newly established marsupial tree can now be applied not only to morphological and paleontological studies but also to clearly distinguish genomic changes.

Life is not always parsimonious, but when more powerful techniques which can resolve issues to a greater degree of precision produce more parsimony, then the world is as it should be in science. The main curiosity I have is to wonder if the outcome isn’t a little too convenient for the generation of more elegant paleontological models. I’m not casting doubt on the integrity of the researchers, but with methods which require such heavy cognitive lifting, and operationally are a touch opaque because of the technical component, one would be assuaged by replication. I believe we will be in the future. If we have \$1,000 genomes for human beings in a few years, NSF grants for taxonomists who lean on genomics may go a lot further in 2020.

*Image Credit: José Luis Bartheld from Valdivia, Chile*

**Citation:** Nilsson MA, Churakov G, Sommer M, Tran NV, Zemann A, Brosius J, & Schmitz J (2010). Tracking marsupial evolution using archaic genomic retroposon insertions. PLoS biology, 8 (7) PMID: [20668664](http://www.ncbi.nlm.nih.gov/pubmed/20668664)

### Related Posts:

- [Brian Switek in Times Online on
  Ida](https://www.gnxp.com/WordPress/2009/05/25/brian-switek-in-times-online-on-ida/) - [Rebuttal to The Natural History of Ashkenazi
  Intelligence](https://www.gnxp.com/WordPress/2008/09/06/rebuttal-to-the-natural-history-of-ashkenazi-intelligence/) - [Contingency &
  evolution](https://www.gnxp.com/WordPress/2007/10/23/contingency-evolution/) - [Brian Ferguson and Ashkenazi
  IQ](https://www.gnxp.com/WordPress/2008/09/25/brian-ferguson-and-ashkenazi-iq/) - [Crazy genetic mutant,
  alien....](https://www.gnxp.com/WordPress/2009/09/28/crazy-genetic-mutant-alien/) - [Evolution: What the Fossils Say and Why It
  Matters](https://www.gnxp.com/WordPress/2007/11/25/evolution-what-the-fossils-say-and-why-it-matters/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F04%2Fmonophyletic-australian-marsupials%2F&linkname=Monophyletic%20Australian%20Marsupials "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F04%2Fmonophyletic-australian-marsupials%2F&linkname=Monophyletic%20Australian%20Marsupials "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F04%2Fmonophyletic-australian-marsupials%2F&linkname=Monophyletic%20Australian%20Marsupials "Email")[](https://www.addtoany.com/share)
