+++
title = "50,000 years of"
full_title = "50,000 years of Dreamtime"
date = "2010-08-08"
upstream_url = "https://www.gnxp.com/WordPress/2010/08/08/50000-years-of-dreamtime/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/08/08/50000-years-of-dreamtime/).

50,000 years of Dreamtime

![auzcricket](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/auzcricket.jpg?resize=400%2C196)![auzcricket](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/auzcricket.jpg?resize=400%2C196)

*“…the occupation of Australia/New Guinea is momentous in that it demanded watercraft and provides by far the earliest evidence of their use in history. Not until about 30,000 years later (13,000 years ago) is there strong evidence of watercraft anyway else in the world, from the Mediterranean.*

*Initially, archaeologists considered the possibility that the colonization of Australia/New Guinea was achieved accidentally by just a few people swept to sea while fishing on a raft near an Indonesian island. In an extreme scenario the first settlers are picture as having consisted of a single pregnant young woman carrying a male fetus. But believers in the fluke-colonization theory have been surprised by recent discovers that still other islands, lying to the east of New Guinea, were colonized soon after New Guinea itself, by around 35,000 years ago….”*

– page 42 of [Guns, Germs and Steel](https://www.amazon.com/exec/obidos/ASIN/0393317552/geneexpressio-20)

The settlement of Australia is a breakthrough in the “human story.” Very soon after anatomically modern humans began to replace (and to some extent [assimilate](http://blogs.discovermagazine.com/gnxp/2010/05/the-three-layers-of-the-neandertal-cake/)) other lineages of our genus in Eurasia we pushed beyond the previous outer limits of the domains of humankind. The ancestors of Australian Aboriginals swept past the [Wallace Line](https://en.wikipedia.org/wiki/Wallace_line), and quickly settled the Ice Age continent of [Sahul](https://en.wikipedia.org/wiki/Sahul_Shelf), consisting of Australia and Papua New Guinea. The biogeography of Australia is well known. Aside from [bats and some endemic rodents](https://en.wikipedia.org/wiki/Mammals_of_Australia#Placental_mammals) the continent was free of placental mammals before modern humans arrived.

  
As for when these humans made landfall, there is some debate as to that particular issue. The oldest remains from Australia, [Mungo Man](https://en.wikipedia.org/wiki/Mungo_man), has been dated to anywhere between 70,000, and 30,000, years before the present. If we took the older date then Australia would have been settled almost immediately after the expansion of non-African modern humanity. If we accepted the younger date, then the settlement of Australia would have been concurrent with the final replacement of Neandertals by modern humans in Europe. The current consensus seems to be that Mungo Man dates to approximately 46,000 years before the present. As the first dating of a particular individual from a species in a region is liable to miss earlier individuals who were not fossilized it seems likely that Australia was settled by anatomically modern humans on the order of 46,000 years before the present, but somewhat earlier than that date. That would imply that Australia was populated by anatomically modern humans at least [10,000 years before Europe](https://en.wikipedia.org/wiki/Neanderthal#Coexistence_with_H._sapiens). One should probably not be too surprised by this. Out-of-Africa humans were probably initially tropically adapted so lateral migration would have been easier, but also, there were no hominin competitors in Australia.

But how do these archaeological insights relate to the current Aboriginal population of Australia? Such questions are fraught with politics, but let’s put that to the side. We know that Australia was not totally isolated from the rest of the world. The [dingo](https://en.wikipedia.org/wiki/Australian_Dingo#Distribution_in_the_past) arrived from Southeast Asia within the last 4,000 years. The Aboriginals of northern Australia were certainly familiar with the idea of agriculture, because they traded with the [Torres Straits Islanders](https://en.wikipedia.org/wiki/Torres_Strait_Islanders), who were farmers and seafarers, and who had contacts with New Guinea (see [After the Ice](https://www.amazon.com/exec/obidos/ASIN/0674019997/geneexpressio-20/)). Some anthropologists, such as [Joseph Birdsell](https://www.amazon.com/exec/obidos/ASIN/0195063619/geneexpressio-20/), proposed that modern Aboriginals were a compound of multiple migration events, and had undergone a great deal of evolution *in situ*. Additionally, classically trained physical anthropologists in the early 20th century noted morphological parallels between Australian Aboriginals and the peoples of India, giving rise to the construct of the [Australoid race](https://en.wikipedia.org/wiki/Australoid_race) (a term still used by Indian anthropologists). As I noted earlier the connection between South Asia and Australia genetically seems [likely to be distant and tenuous at best](http://blogs.discovermagazine.com/gnxp/2010/08/and-they-came-from-central-asia/), inferring from what we know of uniparental markers (genetic variants passed only through the mother or father, the mtDNA and Y). The genetic data tentatively seem to reject Birdsell’s model, and favor a more parsimonious one of a single original settlement on Sahul, and subsequent diversification and isolation (Australia, Tasmania and Papua New Guinea were separated only \~10,000 years ago with rising sea levels).

But there’s only so much that uniparental lineages can tell us. There are limits to the information one can glean from relatively short sequences of mtDNA and Y, and, these gene lineages are subject to their own particular dynamics. Not only do human mating patterns exhibit sex-specific biases, but the neutrality of these lineages from an evolutionary perspective [has been questioned](http://johnhawks.net/weblog/reviews/genetics/mtdna_migrations/selection-mtdna-dowling-tree-review-2009.html). And,the haploid nature of these loci also mean that the effective population size is small (i.e., only copy of each per person, instead of two as in the case of most genes) and stochastic fluctuations may be more extreme than in the rest of the genome. On the one hand more random variation could allow for the emergence of greater between population differences which might be informative, but on the other hand it can also swamp out the past history too quickly and result in convergences which tell us nothing about phylogenetic connections.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)All this is why a new paper looking at the broader genomic patterns of variation in Australian Aboriginals is important for clarifying and adding more precision to our evolutionary historical assumptions, which would frame more specific inferences about this population. There are, and were, difficulties in obtaining the data for historical and political reasons. But now that the barrier has been breached, I assume that we’ll be seeing more in the near future. [Whole-Genome Genetic Diversity in a Sample of Australians with Deep Aboriginal Ancestry](http://www.cell.com/AJHG/abstract/S0002-9297(10)00364-2):

> Australia was probably settled soon after modern humans left Africa, > but details of this ancient migration are not well understood. Debate > centers on whether the Pleistocene Sahul continent (composed of New > Guinea, Australia, and Tasmania) was first settled by a single wave > followed by regional divergence into Aboriginal Australian and New > Guinean populations (common origin) or whether different parts of the > continent were initially populated independently. Australia has been > the subject of relatively few DNA studies even though understanding > regional variation in genomic structure and diversity will be > important if disease-association mapping methods are to be > successfully evaluated and applied across populations. We report on a > genome-wide investigation of Australian Aboriginal SNP diversity in a > sample of participants from the Riverine region**. The phylogenetic > relationship of these Aboriginal Australians to a range of other > global populations demonstrates a deep common origin with Papuan New > Guineans and Melanesians, with little evidence of substantial later > migration until the very recent arrival of European colonists.** The > study provides valuable and robust insights into an early and > important phase of human colonization of the globe. A broader survey > of Australia, including diverse geographic sample populations, will be required to fully appreciate the continent’s unique population history and consequent genetic heritage, as well as the importance of both to the understanding of health issues.

The sample consisted of 38 individuals, 30 females and 8 males, from the [Riverina region](https://en.wikipedia.org/wiki/File:Riverina_bioregion.png) of New South Wales. The sample size may be small, but for the broad-brush and relatively coarse questions being asked in this paper they’re sufficient. Consider that genomic sequencing of one Native American and one Bushman could tell you that the latter is likely to come from a far more genetically diverse population than the former. If you constructed a phylogenetic tree with half a dozen individuals of each of the populations you’d see that the Native Americans are a subset of the Bushman genetically, so to speak. If you’re trying to distinguish between questions such as, “did the last common ancestor of Australian Aboriginals and Javanese live 5,000, or 50,000, years before the present”, then this is a sufficient sample. A bigger issue is that the sample has substantial European admixture through the paternal lineages. From what I have heard attempts were made to get a more “pure” Aboriginal group, but the logistics were too difficult in the end. Science is the art of the possible.

They used an Affymetrix chip with nearly 1 million [SNPs](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism) (out of 3 billion base pairs), but filtered it down even further for this analysis. Most of the work used a data set of \~160,000 SNPs, arrived via quality controls, as well as the intersection with HapMap3 and HGDP SNP sets. Again, in light of the coarse questions asked 160,000, let along 16,000, should probably suffice. Remember they’re trying to move beyond what we can infer from classical autosomal markers and uniparental lineages. This is a big step in that direction.

![abofig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig1.png?resize=400%2C504)![abofig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig1.png?resize=400%2C504)The figure to the left shows two phylogenetic trees (**note:** I may reedit these figures for ease of display or clarity). The utility of the trees is obvious: **they’re showing you how populations relate to each other.** So you throw all the individuals in each given population into a pot, average out their genetic character, and perform pairwise calculations on them. The other groups are from the [HGDP](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project) data set. The statistic they’re using is F_(st); basically a measure of between population genetic variation. Alleles, genetic variants, vary in frequency from population to population, as well as the fact that different individuals within populations have different genotypes, and this is just capturing the component which is varying across populations. So as an example, if the allele *x* in two populations is at frequency 0.5 for both, then the F_(st) is 0. There’s no difference. If *x* is at 1 in one population and 0 in the other, then F_(st) = 1. All the variation is between the populations, since there is none within the populations.

The trees illustrate visually the relationships in an F_(st) matrix of pairwise population comparisons. Populations which are genetically close are not very distant from each other along the length of the tree, while those which are genetically very different are farther from each other terminus-to-terminus. But remember that these visualizations don’t tell us anything necessarily in a concrete manner before we interpret them through the filter of what we already know. For example, the Mozabites, “MOZ”, are outside of the main clusters. Why? Without knowing anything about their history we might assume that they were isolated from the original African population at an early point in time (though observe the minimal distance from the trunk, peculiar). But we know their history, and the topology in that region of the network is an outcome of admixture. The Mozabites have a substantial amount of recent Sub-Saharan African ancestry. Similarly, two of the groups near the root of the East Eurasian cluster are actually relatively recent admixtures between West and East Eurasian populations, the [Uyghurs](http://blogs.discovermagazine.com/gnxp/2008/03/uyghurs-are-hybrids/) and Hazaras.

The Australian Aboriginals are similar to the Mozabites, Uyghurs and Hazaras. Their position in the first panel is near the root of the Oceanian cluster. This is due to their substantial European admixture, which we know is present through their oral history, recorded history, and, the physically composite nature of many modern Australian Aboriginals. To generate the second tree the authors reconstructed the allele frequencies of the Australian Aboriginals by subtracting the European component of admixture. They did this by noting that they had Western European populations in HapMap3, and the offspring population between these groups which they knew, and the unknown Aboriginal parent population. Using the Structure program they simply performed the algebra, whereby Aboriginal = Admixed Aboriginal – European (OK, not “simply”). And as you can see, by using the reconstructed Aboriginal allele frequencies the tree now places this group, AuR\*, firmly within the Oceanian cluster.

![abofig2](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig2.png?resize=300%2C441)![abofig2](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig2.png?resize=300%2C441)The clustering of the Oceanian groups itself alone gives us strong evidence that the settlement of Sahul was by one population which later diversified, rather than separate independent groups. But let’s back up a bit, and look at the admixture aspect again. To the left you see the PCA plots of the HGDP data set which you should be familiar with. Each axis represents and independent dimension of genetic variation. For the first panel the x-axis, PC1, is the separation between Africans and non-Africans. This is the biggest dimension of variation, and points to the Out-of-Africa event. The second dimension seems to map well onto the east-west axis, more or less. Remember that each PC is rank ordered in terms of the proportion of the total genetic variation which it can explain independently. Interestingly PC3 and PC4 allow for the separation of Oceanians and Amerindians from other groups. In isolation-by-distance and serial bottleneck models it shouldn’t be too surprising that these two groups on the geographic margins of the traditional human range would exhibit some genetic peculiarities due to their history after separation from Eurasian groups. This is why the [Kalash](https://en.wikipedia.org/wiki/Kalash) of Pakistan are also outliers, this non-Muslim tribe remained isolated in their mountain valley and so accumulated their own genetic distinctiveness.

And yet note the position of Amerindians and Oceanians in the first panel, they’re somewhat closer to West Eurasians than East Asians. In the case of Amerindians there has long been the model whereby the ancient Beringian population which expanded into the New World had a component of ancestry which was closer to West Eurasia. This is true today among Siberian groups such as Yakuts, but differentiating the more recent introgression of Russian ancestry with an ancient West Eurasian substratum is difficult.

But admixture is surely part of the puzzle too. Compare the linear topology of Aboriginals and Amerindians with [African Americans](http://blogs.discovermagazine.com/gnxp/2010/07/one-principal-component-to-rule-them-all/). The PCA plot is putting the focus on between population differences, so these sorts of distributions, so cleanly linear, are indicative of possible recent admixture between two distinct populations. The populations of the New World and Australia were relatively small and thin in terms of distribution, so it should not be too surprising that a substantial uptake of European ancestry has occurred in both cases. Isolated cases of individuals “going native” probably illustrate a bigger trend. The 2002 film [Rabbit-Proof Fench](https://en.wikipedia.org/wiki/Rabbit-Proof_Fence_(film)) was a dramatization of the reality that quite often children of mixed heritage will identify with one culture and parent. This is not an atypical disjunction between genes and culture in terms of their mode of inheritance. Both parents contribute equally genetics to the autosome, but cultural contribution is more of a contingent matter.

![abofig3](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig3.png?resize=250%2C245)![abofig3](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig3.png?resize=250%2C245)To further explore the admixture within Aboriginals the authors performed a *frappe* and Structure analysis. These two methods differ in the details but perform basically the same operation; they take individuals and assign components of their genome to K putative ancestral groups. So K = 2 would indicate 2 ancestral groups, while K = 10 would indicate 10. Here’s we’re looking at K = 5. Again, proper caution is warranted with these methods because without context we may not be able to interpret the results. But in this case the ends are clear and distinct: **what is the extent and range of European ancestry in this Australian Aboriginal sample?** Both the *frappe* and Structure programs paralleled each other in outcome; the Aboriginal sample varied quite a bit in ancestral quanta. The Papuans and Melanesians serve as appropriate Oceanian references. The Melanesians have a residual component (shaded orange and yellow top to bottom) which is similar to East Asians. This is a marker of the [Austronesian](https://en.wikipedia.org/wiki/Austronesian_languages) expansion into the Pacific. The Papuans and Aboriginals generally lack this, which stands to reason considering their greater cultural isolation from the Austronesians.

The *frappe* and Structure results dovetail perfectly with the PCA plots. Both suggest that the Aboriginal population is admixed, with the parental populations being West Eurasian (European) and Oceanian, and, that that admixture varies from individual to individual. The F_(st) also suggested this, though at a coarser population wide scale, and only with prior knowledge of the possibility of admixture. Not only that, but the mtDNA and Y chromosomal results on these individuals also comes out to the same inferred proportion. Recall that there were only 8 males, so the Y sample is small. But they calculate that \~40% of the Y lineages are not Aboriginal, while nearly \~100% of the female ones are. This sort of disjunction is common in the New World among Mestizo and African American populations, as well as the mixed [Cape Coloured](http://blogs.discovermagazine.com/gnxp/2010/04/cape-coloureds-an-instance-of-a-generality/) population of South Africa. The proportional of autosomal ancestry inferred from these uniparental markers is about what was calculated with their SNP-chip, suggesting the persistence of this sex-biased admixture pattern over the past two centuries. Remember that if something more complex demographically had occurred we may not have been able to infer admixture from uniparental lineages. Imagine if the Aboriginal tribes in New South Wales which were admixed were decimated by an unadmixed group, so that only the females from the admixed group survived. After that event both mtDNA and Y chromosomal lineages would have been Aboriginal, but the European ancestry would persist in the autosome.

![abofig4](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig4.png?resize=350%2C236)![abofig4](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/abofig4.png?resize=350%2C236)To the left is an estimate of individual-to-individual ancestral quanta. The mode, the most frequent value, is near the total genome estimate of the whole population, around \~2/3 Aboriginal. There is clearly a wide range of variation in admixture. It looks like that within this Australian Aboriginal community 20% of this sample are 50% or more European in ancestry. Interestingly one man is \~100% Aboriginal. The authors do remark that their estimates are probably low balling the Aboriginal ancestral quantum; the SNP-chip was constructed with European genetic variation as a baseline, so it is missing Australian variation on loci where Europeans are monomorphic. But even with that taken into account the Aboriginal group here is substantially admixed. This prompts me to ask: is it possible that there is more distinctive indigenous genetic material in the \~20 million white citizens of Australia than within the indigenous groups themselves? I’ve already suggested that this [dynamic](http://blogs.discovermagazine.com/gnxp/2010/06/amerindians-of-brazil-more-numerous-than-you-think/) is exactly what is operative in Brazil, but the analogy is only rough a best. Many Australian whites derive from recent waves of migration and may not have any ancestors with roots back to the 19th century (i.e., all their grandparents and/or parents may have been born in Europe or the British Isles). But that must be balanced against the fact that Australian Aboriginals are much more European in ancestry than Brazilian Amerindians likely are. The [rapid growth of indigenous](https://en.wikipedia.org/wiki/Indigenous_Australians#Demographics) Australians can’t be a function purely of high fertility. Rather, many people of mixed heritage are identifying as indigenous. The outmarriage rates for urban Aboriginals in some of the literature is estimated to be in the 70-90% range.

So far we’ve covered aspects of Australian Aboriginal genetics relevant to paleoanthropology and historical population genetics. But as I have observed many a time, **one of the primary reasons for this sort of population analysis is to clarify background parameters for medical genetics**. The life expectancy gap between indigenous and non-indigenous Australians is [on the order of 10 years](https://en.wikipedia.org/wiki/Indigenous_Australians#Life_Expectancy). It also seems plausible that the same disease-driven population crash which occurred in the New World after contact with Europeans was also driving population changes in Australia around the time of European settlement. Since Aboriginals were generally hunter-gatherer groups many infectious diseases which required higher densities could only be incubated among Europeans. Aboriginals near European settlements, or those who settled within them, would naturally be exposed to these infections and suffer greater morbidity and mortality. One wonders if some of the diseases which Aboriginals suffer from are due to genetic differences between the populations in regards to immunity, as well as the “diseases of civilization” (e.g., type 2 diabetes).

As noted in the paper admixed populations present both pitfalls and opportunities when it comes to elucidating risk alleles:

> Whereas the admixture present in the AuR \[the Riverina Aboriginals > -Razib\] sample presents a potential challenge in conducting > traditional association methods for disease gene discovery…it opens > the possibility of using admixture mapping…Admixture mapping is most > suitable for traits, like CKD, that differ in frequency between the > two parental populations of an admixture group. The approach > essentially looks for genomic regions with an excess of higher-risk > population ancestry relative to other regions or controls..A set of markers, spread across the genome, that are highly informative as to ancestry (ancestry informative markers or AIMs) is an essential requirement for admixture mapping.

![figabo6](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/figabo6-300x216.png?resize=300%2C216)![figabo6](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/08/figabo6-300x216.png?resize=300%2C216)CKD above refers to chronic kidney disease, which Aboriginals suffer at \~10 times greater rates than non-Aboriginals. To the left is a figure which shows the distribution of SNP rs12458349, a derived allele which has the highest F_(st) value between AuR\* and HapMap3 [populations](https://www.broadinstitute.org/~debakker/p3.html). The genomic region which rs12458349 is embedded within has been implicated in diabetic nephropathy, a major cause of CKD. Derived here means that the SNP is evolutionarily novel in relation to the ancestral state, which all other human populations exhibit.

So why is this derived variant at a high frequency in Aboriginals (and other Oceanians), but not other human populations? It could be random genetic drift. As populations migrated out of Africa they may have gone through bottlenecks and isolations in a step-wise fashion and each group down the spatial and temporal sequence would accumulate their own unique variants. Or, it may have been adaptation, which drive up frequencies through positive selection around that genomic region. But this is where limitations of sample size and representativeness crop up, as these are not quite the coarse questions which we were focusing on earlier. From the the text:

> Genetic drift, or random changes in allele frequencies, is expected to > be a major force in a population, like Aboriginal Australia, that has > been relatively small and/or isolated for a long period of time. > Differentiation could also be explained by natural selection. The > presence of several highly differentiated SNPs in the region, spanning > nearly 0.5 Mb in length, hints at the presence of a long common > haplotype that might be indicative of genetic hitch-hiking and recent > positive selection. However, it is difficult to distinguish between possible explanations because the sample is small, with extensive admixture hampering phasing and direct investigation of linkage-disequilibrium-based selection signals.

Note however the presence of the derived allele in Papuans and Aboriginals, but not Melanesians. In the phylogenetic network the Aboriginals are the outgroup, and yet on this character the Melanesians are. This is a locus which will no doubt be explored in the future, because the patterns here will be fascinating to tease apart, and, of possible medical relevance.

Overall this paper has confirmed much of what we know, or at least solidified our background assumptions. The contemporary peoples of Melanesia, New Guinea and Australia have a common ancestral heritage. Coalescence times back to the last common ancestor between these populations and non-Oceanian groups suggest that their residence in their current locations is antique, and *possibly* back to the first settlement. One should be duly cautious about extrapolating from contemporary patterns to variation to the past, but I think on this scale we’re on more solid ground. Parts of southern Australia and Tasmania may have an equitable climate where ancient DNA samples may have been preserved, so that could resolve the issues with more certainty in the future.

But aside from phylogeny, a closer study of Australian Aboriginal genetics may also give us insights into the impact which agriculture and higher population densities had upon our species’ genomes. The Australian Aboriginals were aware of agriculture because of contacts with the peoples of the Torres Straits, but they never seem to have adopted it. In contrast to Australia the [highlands of New Guinea](https://en.wikipedia.org/wiki/New_Guinea#History) developed a relatively high population density with the spread of a gardening mode of production. So here you have two populations which were in contact \~10,000 years ago, and have diverged in mode of production subsequent to that period. A comparison in [allele frequencies](http://blogs.discovermagazine.com/gnxp/2010/07/why-tibetans-breath-so-easy-up-high/) between these two populations would then be instructive as to the power of drift and selection to drive evolutionary change over 10,000 years.

Obviously there needs to be more work done, and extrapolating from one sample will not do. The north coast Aboriginals were certainly in contact with sailors from Southeast Asia before Europeans arrived, and they speak a different [group of languages](https://en.wikipedia.org/wiki/Indigenous_Australian_languages#Classification) from those in the rest of the continent. The existence of tribes with non-trivial numbers of blonde individuals in the western deserts despite no other apparent European admixture also demands to be explored. From what [Joseph Birdsell documented](https://www.gnxp.com/blog/2005/08/blonde-australian-aboriginals.php) about the Mendelian inheritance patterns of blondism among these tribes it seems likely that the genetic architecture is very different from that in Europeans.

Over the past generation we’ve begun to really understand how the human tree of life branched out and flourished. Now it’s time to fill in the gaps, and with whole genome sequencing around the horizon many of the technical limitations will be removed. But what about the social and political ones? The consent given by this Aboriginal group has now opened a window into the evolutionary genetic history of all Australian Aboriginals, imperfect as that is. But what will happen when many more people in developed nations get sequenced, and so know their own genetic history with great detail? If a non-trivial proportion of Aboriginal ancestry is found across the old stock white population of Australia could a collaborative project just “reconstruct” the Aboriginal genome from these individuals, and so do an end-around the socio-political minefields? I suppose we’ll see soon enough.

*Image credit: Aboriginal cricket team, 1868, Wikimedia Commons*

**Citation:** McEvoy, Brian P., Lind, Joanne M., Wang, Eric T., Moyzis, Robert K., Visscher, Peter M., van Holst Pellekaan, Sheila M., & Wilton, Alan N. (2010). Whole-Genome Genetic Diversity in a Sample of Australians with Deep Aboriginal Ancestry The American Journal of Human Genetics : [10.1016/j.ajhg.2010.07.008](10.1016/j.ajhg.2010.07.008)

### Related Posts:

- [Animal Sea Crossings, Hippo
  Bleg](https://www.gnxp.com/WordPress/2011/03/20/animal-sea-crossings-hippo-bleg/) - [The Others could travel by
  sea](https://www.gnxp.com/WordPress/2010/02/16/the-others-could-travel-by-sea/) - [Pegging the human expansion of out of
  Africa](https://www.gnxp.com/WordPress/2017/05/19/pegging-the-human-expansion-of-out-africa/) - [The Ancient Neanderthal
  Mariner](https://www.gnxp.com/WordPress/2018/04/26/191783/) - [Australian Aborignals, a people that dwelt
  alone?](https://www.gnxp.com/WordPress/2007/05/07/australian-aborignals-a-people-that-dwelt-alone/) - [Humans like us in Sri Lanka 48,000 years
  ago](https://www.gnxp.com/WordPress/2020/06/14/humans-like-us-in-sri-lanka-48000-years-ago/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F08%2F50000-years-of-dreamtime%2F&linkname=50%2C000%20years%20of%20Dreamtime "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F08%2F50000-years-of-dreamtime%2F&linkname=50%2C000%20years%20of%20Dreamtime "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F08%2F08%2F50000-years-of-dreamtime%2F&linkname=50%2C000%20years%20of%20Dreamtime "Email")[](https://www.addtoany.com/share)
