+++
title = "Levels of human"
full_title = "Levels of human variation"
date = "2014-05-13"
upstream_url = "https://www.gnxp.com/WordPress/2014/05/13/levels-of-human-variation/"

+++
Source: [here](https://www.gnxp.com/WordPress/2014/05/13/levels-of-human-variation/).

Levels of human variation

[![51qciM4cBhL.\_SS400\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/05/51qciM4cBhL._SS400_.jpg?resize=200%2C258)![51qciM4cBhL.\_SS400\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/05/51qciM4cBhL._SS400_.jpg?resize=200%2C258)](https://www.amazon.com/exec/obidos/ASIN/0691029059/geneexpressio-20)A quick post to clarify things. When we talk about human variation and history we’re talking about phenomena which we need to decompose into different levels of analysis, because there are major differences in terms of methodology and questions we’re asking. Too often public presentation tends to melt them together and confuse separate strands.

First, there is the level of phylogeny. The question here relates to the history of human genes and populations. **With the rise of genomic technologies it is trivially easy to identify human clusters, and not that much harder to infer historical-demographic events.** This is important, because due to Richard C. Lewontin’s popularization of the correct point that most human genetic variation is partitioned within, not across, populations, the public is under the impression that population structure is trivial. It’s not. Some extant modern human populations may have diverged from the rest of our species as early as \~100,000 years ago (the Khoisan peoples of southern Africa). Gene flow between even Eurasian populations can be rather attenuated over the over of 50,000 years (e.g., Northeast Asians seem to have had minimal gene flow from other Eurasian populations over the past \~30,000 years).

Second, there is the question of **variation in phenotype**. Do biological traits vary by population? Yes. Is this variation heritable? Some of it is. This is where an understanding of phylogeny is important, because it turns out that some phenotype distributions are strongly divergent from phylogeny. For example, as early as 2007 it was clear that [human pigmentation](http://mbe.oxfordjournals.org/content/24/3/710.abstract) had been subject to regional selection events which decoupled ancestry from variation on those traits. Melanesians who have been separate from Africans as long (or longer in the case of West Eurasians due to admixture) as any other human population exhibit phenotypic and genotypic similarities to Africans, likely because of functional constraint at low latitudes. But this is not always the case. The peoples of Southeast Asia tend to be rather light skinned for their environmental conditions. But now we know that they are likely the product of recent migration due to human phylogenomic and archaeological results (i.e., in Southeast Asian for \<4,000 years).

Third, there is the question of **between group differences in complex traits.** This where the first point, the triviality of between group differences as asserted in [Lewontin’s Fallacy](https://en.wikipedia.org/wiki/Human_Genetic_Diversity:_Lewontin%27s_Fallacy) comes into play. If between group differences are not viable phylogenetically, i.e., human genetic history is not structured by divergences and fusions which are evident in the genome, then the whole argument collapses. But I think point one and two illustrate that the move into question three is not logically unsound, but must be **gripped one empirical trait at a time.** Complex traits are difficult to tackle, and we have to be cautious and not let prior conceptions cloud our judgment, but here’s a paper from a few years ago which illustrates where we might be going, [Evidence of widespread selection on standing variation in Europe at height-associated SNPs](http://www.nature.com/ng/journal/v44/n9/abs/ng.2368.html):

> By studying height, a classic polygenic trait, we demonstrate the > first human signature of widespread selection on standing variation. > We show that frequencies of alleles associated with increased height, > both at known loci and genome wide, are systematically elevated in > Northern Europeans compared with Southern Europeans…This pattern > mirrors intra-European height differences and is not confounded by > ancestry or other ascertainment biases. The systematic frequency differences are consistent with the presence of widespread weak selection (selection coefficients \~10−3–10−5 per allele) rather than genetic drift alone….

### Related Posts:

- [PCA plots and
  trees](https://www.gnxp.com/WordPress/2010/06/01/pca-plots-and-trees/) - [Correction on
  MC1R....](https://www.gnxp.com/WordPress/2005/07/15/correction-on-mc1r/) - [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/) - [Complex traits &
  evolution](https://www.gnxp.com/WordPress/2008/12/06/complex-traits-evolution/) - [10 questions for Luigi Luca
  Cavalli-Sforza](https://www.gnxp.com/WordPress/2006/08/24/10-questions-for-luigi-luca-cavalli-sforza/) - [Gene expression, African Americans &
  Europeans](https://www.gnxp.com/WordPress/2008/12/08/gene-expression-african-americans-europeans/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F05%2F13%2Flevels-of-human-variation%2F&linkname=Levels%20of%20human%20variation "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F05%2F13%2Flevels-of-human-variation%2F&linkname=Levels%20of%20human%20variation "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F05%2F13%2Flevels-of-human-variation%2F&linkname=Levels%20of%20human%20variation "Email")[](https://www.addtoany.com/share)
