+++
title = "Multiregionalism in"
full_title = "Multiregionalism in natural selection"
date = "2015-12-11"
upstream_url = "https://www.gnxp.com/WordPress/2015/12/11/multi-regionalism-in-natural-selection/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/12/11/multi-regionalism-in-natural-selection/).

Multi-regionalism in natural selection

[![Screenshot - 12102015 - 03:55:03 PM](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/12/Screenshot-12102015-035503-PM.png?resize=625%2C509)![Screenshot - 12102015 - 03:55:03 PM](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/12/Screenshot-12102015-035503-PM.png?resize=625%2C509)](http://biorxiv.org/content/early/2015/12/08/033928)

[![Screenshot - 12102015 - 04:26:46 PM](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/12/Screenshot-12102015-042646-PM.png?resize=300%2C244)![Screenshot - 12102015 - 04:26:46 PM](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/12/Screenshot-12102015-042646-PM.png?resize=300%2C244)](http://biorxiv.org/content/early/2015/12/08/033928)The [human genome](http://www.nature.com/nature/journal/v507/n7492/full/nature12961.html) is littered with many genes from diverged lineages. That is, any given human has segments from lineages which are deeply diverged from the dominant demographic element in our ancestry, which diverged from an African population which flourished on \~200,000 years ago, and among non-Africans a population derived from Northeast Africa \~50,000 years ago. The Neanderthal ancestry of non-Africans, which is in the range of \~2 percent, diverged from the rest of the genome on the order of \~500,000 years ago from the main stem of our heritage. A similar time span divides us from the Denisovan ancestry in Oceanians, and to a lesser extent island Southeast Asians, and even less among East and South Asians. Due to the lack of ancient genomes such definitive inferences are more difficult to make for African populations, but there are [suggestive clues](http://www.pnas.org/content/108/37/15123.full) that diverged lineages also contributed to the Khoe-San and Pygmy, and therefore other African, peoples.

Though the initial Neanderthal admixture results tended to focus on their implications for ancestry, and not function, a recent spate of work has suggested that archaic admixture in modern lineages [may have been adaptive](https://westhunt.wordpress.com/2014/01/30/adaptive-neanderthal-admixture/). But at some point one needs to go beyond genome-wide assessments, **and look at specific genes**. In that vein, a preprint out of Rasmus Nielsen’s group, [Archaic adaptive introgression in *TBX15*/*WARS2*](http://biorxiv.org/content/early/2015/12/08/033928). This related to their paper about [adaptation of Greenland Inuit](http://www.sciencemag.org/content/349/6254/1343) to their environment. What they report here is that **Greenlanders, and Eurasians more broadly as well, exhibit evidence of carryingan introgressed haplotype which derives from Denisovans or a Denisovan-related population.**\* The map above shows distributions of an allele which is strongly associated with the introgressed haplotype. You can see that it is absent in Sub-Saharan Africa, fixed in Greenland, present in high frequencies in the New World (near fixation in Amerindians), at moderate frequencies in East Asia, and lower frequencies elsewhere in Eurasia. The haplotype harbors two genes, *[TBX15](http://www.genecards.org/cgi-bin/carddisp.pl?gene=TBX15)* and *[WARS2](http://www.genecards.org/cgi-bin/carddisp.pl?gene=WARS2)*. What do these genes do? Lots of things:

> The *TBX15*/*WARS2* region is highly pleiotropic: it has been found to > be associated with a variety of traits. These include the > differentiation of adipose tissue5, body fat distribution…facial > morphology…stature…ear morphology…hair pigmentation…and skeletal > development…Interestingly, for several of body fat distribution > studies, the introgressed SNPs have significant genome-wide > associations. **The Denisovan alleles tend to increase waist > circumference and waist-hip ratio, after correcting for BMI.**

They went to great lengths to ascertain whether this was an introgressed haplotype, and where it came from in relation to the genomes we have on hand (Neanderthals, modern humans, and Denisovans). Broadly they are convincing that it is introgressed, and not deep structure from Africa. And, they make a good case that the population fromwhich this haplotype derived is closer to Denisovans than to Neanderthals. The summary is really in the haplotype network above. To the bottom right you see a cluster of common human haplotypes. Then you see the Neanderthal haplotype, and then further along the Denisovan haplotype. Finally, you see a cluster of introgressed haplotypes. First, remember it’s not established that the donor population was Denisovan. Second, there have been derived mutations since the allele moved between populations.

The functional reason why this swept to fixation in Greenlanders seems obvious. It’s related to the nature of fat deposition, and GWAS correlate it with effects on BMI and body shape, and, it is jointly in high frequency in Amerindians and the populations of the Arctic. In all likelihood the sieve was in Beringia, where climatic conditions were extreme, and all sorts of adaptations were necessitated. And the authors state that plainly. In relation to mechanism there are suggestions that there regulatory and epigenetic dynamics at play. The expression differences seem clear, in terms of how the Denisovan variant effects the magnitude of the genetic consequence. But the epigenetic aspect is very confused to me. Part of it seems to be that the authors themselves are trying to make sense of the results (jump to “DMR”), but I wish they would at least expand, because there is some lack of clarity as to the details (I had a friend whoseresearchis in epigenetics read that section and they found it a bit unclear, so they didn’t want to evaluate whether it made sense, so I felt better as to my confusion).

With all that put out there, let’s get to the crux of the issue that we can agree on. Using simulations the authors did establish that this is unlikely to have increased in frequency in all these populations simply due to drift. That is, chance. The alternative then is that selection is increasing the frequency of this haplotype, and assorted functional alleles. In the case of Amerindians you see see that it is close to fixation. There is a recessive aspect to the nature of methylation patterns, which are associated with gene expression, so that may give us a clue why this variant is fixed in Greenland, and nearly so in unadmixed Amerindians. If the expression of a favored trait is recessive, then it makes sense to make the final step from \~90% (where nearly 20% of the population would have a disfavored morph) to \~99% (where only \~2% would).

**But what’s going on in Old World populations?**To my knowledge there is no evidence of Denisovan admixture in western Eurasian populations, but you can find these “Denisovan” alleles in them. The simplest explanation **is that the haplotype is derived from another archaic population**, within the same clade as Denisovans with Neanderthals as an outgroup, which was resident further west. In fact, look at the structure the introgressed haplotypes. The Amerindians have the most diverged branch, while the western and southern Eurasian groups are represented within the haplotype closest to Denisovans. To me this is suggestive of an early admixture event closer to the point-of-departure from Africa. As modern humans moved east the serial bottleneck effect occurred with this introgressed haplotype.

A second possibility is that this allele may be from Denisovans, and that it is so favored that even if small levels of eastern Eurasian admixture don’t manifest themselves in total genome-wide admixture estimates a few copies were sufficient for this to become common outside that zone. This gets to the title of the post: one can posit a multi-regional system of selected variants sweeping across interconnected demes, which transcend the fact that migration between the demes is too low to make significant contributions to total genome content. This may explain the [presence of East Asian](http://biorxiv.org/content/biorxiv/early/2015/03/13/016477.full.pdf) *[EDAR](http://biorxiv.org/content/biorxiv/early/2015/03/13/016477.full.pdf)*in the Motala samples, for example.

Finally, I think one has to consider the high probability that the target of selection on this locus has varied over time and space. The introgressionof this archaic allele into non-Africans was an ancient event, but it does not seem to have fixed into any populations outside of the Beringian Diaspora. Why? It may be that there are balancing effects going on, perhaps frequency dependence, or, in even over-dominance (I tend to discount the last in most cases, but the fractions in East Asia are so close to 50%). Along the East Asian littoral the frequency is in an intermediate range, while in western and southern Eurasia they’re present at lower frequencies, though lower in South Asia than in some of the European groups. This is intriguing because when it comes to alleles which are*not*subject to selection South Asian share more ancestry with East Eursaians, and you generally see a pattern where they occupy a position in between (e.g., this is the case when it comes to Neanderthal admixture, with East Asians having the most, and Europeans the least, with South Asians between).

There’s a lot that’s going to be researched and published between now and 2025. These authors have established that they found an introgressed variant, but it’s too widely distributed to have the neat solution that*[EPAS1](http://www.ncbi.nlm.nih.gov/pubmed/25043035)*in Tibetans has.

\* My father and brother carry one copy of the introgressed haplotype.

### Related Posts:

- [What did modern humans look like during the "Out of
  Africa"…](https://www.gnxp.com/WordPress/2018/04/16/what-did-modern-humans-look-like-during-the-out-of-africa-event/) - [Reevaluating "multiple origins" for modern
  humans](https://www.gnxp.com/WordPress/2019/02/21/reevaluating-multiple-origins-for-modern-humans/) - [Basal Eurasians and pre-Out of Africa population
  structure](https://www.gnxp.com/WordPress/2014/10/23/basal-eurasians-and-pre-out-of-africa-population-structure/) - [The dam of ancient DNA starts to
  break](https://www.gnxp.com/WordPress/2013/01/21/the-dam-of-ancient-dna-starts-to-break/) - [Older than
  Adam](https://www.gnxp.com/WordPress/2013/03/06/older-than-adam/) - [The telos of modern
  humans](https://www.gnxp.com/WordPress/2019/04/19/the-teleos-of-modern-humans/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F12%2F11%2Fmulti-regionalism-in-natural-selection%2F&linkname=Multi-regionalism%20in%20natural%20selection "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F12%2F11%2Fmulti-regionalism-in-natural-selection%2F&linkname=Multi-regionalism%20in%20natural%20selection "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F12%2F11%2Fmulti-regionalism-in-natural-selection%2F&linkname=Multi-regionalism%20in%20natural%20selection "Email")[](https://www.addtoany.com/share)
