+++
title = "Lions, and tigers, and"
full_title = "Lions, and tigers, and snow leopards! Oh, my!"
date = "2010-02-15"
upstream_url = "https://www.gnxp.com/WordPress/2010/02/15/lions-and-tigers-and-snow-leopards-oh-my/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/02/15/lions-and-tigers-and-snow-leopards-oh-my/).

Lions, and tigers, and snow leopards! Oh, my!

![A_tiger_in_Pilibhit_Tiger_Reserve.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/A_tiger_in_Pilibhit_Tiger_Reserve.jpg?resize=333%2C358)![A_tiger_in_Pilibhit_Tiger_Reserve.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/A_tiger_in_Pilibhit_Tiger_Reserve.jpg?resize=333%2C358)One of the interesting aspects ensuing from the rise of molecular phylogenetics is that the trees are generally concordant in broad strokes with older research which was based on morphology. This is not too surprising, as nature does tend to show some intelligible patterns which can be cross-checked with different methods (e.g., the fact of evolutionary change in the tree of life is evident in both the paleontological record and analysis of molecular genetic variation). Additionally, the [cladistic revolution](https://en.wikipedia.org/wiki/Cladistics) has produced a general uniformity of underlying logic when it comes to systematic analysis. Genes can complement analyses utilizing classical morphological traits.  
But obviously sometimes what surprises and exhibits discordance can also inform. The controversy about the relationship of [Archaea](https://en.wikipedia.org/wiki/Archaea#A_new_domain) to other branches of the tree of life illustrate how molecular techniques can overturn old orthodoxies on a grand scale, though it is also probably informative that this sort of taxonomic reorganization would occur among relatively simple organisms which less salient “physical traits” (i.e., biologists have been studying mammals for a much longer period of time and with greater depth than unicellular organisms). Molecular techniques can more regularly add wrinkles to the shape of our knowledge on a more granular scale when it comes to the branches of the tree of life. The most well known example of this is our own species. Jared Diamond’s [The Third Chimpanzee](https://www.amazon.com/exec/obidos/ASIN/0060845503/geneexpressio-20) alludes to the fact that humans, chimpanzees and bonobos are a [monohyletic clade](https://en.wikipedia.org/wiki/Monophyly); we three are more closely related to each other than we are to gorillas. The category “apes” resembles the category “reptiles,” as some descendants of the most recent common ancestor of all of the terminal branches of the class are not included (for apes, humans, and for reptiles, birds).  
Because humans control funding, our own evolutionary genetic history, and its relationship to other extant primates, has been relatively well elucidated. By contrast, examination of other charismatic species of note has been sketchier. With that in mind, a recent paper outlining the possible relationships of “Big Cats” to each other is of particular interest, [Supermatrix and species tree methods resolve phylogenetic relationships within the big cats, Panthera (Carnivora: Felidae)](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WNH-4Y9SVTC-2&_user=10&_coverDate=02/04/2010&_rdoc=1&_fmt=high&_orig=search&_sort=d&_docanchor=&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=b9062acb8ab5c28c952671c0c9fe378a):

> The pantherine lineage of cats diverged from the remainder of modern > Felidae less than 11 million years ago and consists of the five big > cats of the genus Panthera, the lion, tiger, jaguar, leopard, and snow > leopard, as well as the closely related clouded leopard. A significant > problem exists with respect to the precise phylogeny of these highly > threatened great cats. Despite multiple publications on the subject, > no two molecular studies have reconstructed Panthera with the same > topology. These evolutionary relationships remain unresolved partially > due to the recent and rapid radiation of pantherines in the Pliocene, > individual speciation events occurring within less than 1 million > years, and probable introgression between lineages following their > divergence. We provide an alternative, highly supported interpretation > of the evolutionary history of the pantherine lineage using novel and > published DNA sequence data from the autosomes, both sex chromosomes > and the mitochondrial genome. New sequences were generated for 39 > single-copy regions of the felid Y chromosome, as well as four > mitochondrial and four autosomal gene segments, totaling 28.7 kb. > Phylogenetic analysis of these new data, combined with all published > data in GenBank, highlighted the prevalence of phylogenetic > disparities stemming either from the amplification of a mitochondrial > to nuclear translocation event (numt), or errors in species > identification. Our 47.6 kb combined dataset was analyzed as a > supermatrix and with respect to individual partitions using maximum > likelihood and Bayesian phylogenetic inference, in conjunction with > Bayesian estimation of species trees (BEST) which accounts for > heterogeneous gene histories. **Our results yield a robust consensus > topology supporting the monophyly of lion and leopard, with jaguar > sister to these species, as well as a sister species relationship of > tiger and snow leopard. These results highlight new avenues for the > study of speciation genomics and understanding the historical events > surrounding the origin of the members of this lineage.**

Many phylogenetic studies of animals rely on mtDNA for ease of extraction & analysis. This paper looks at loci on the autosome, as well as the Y chromosome. Compared to the HapMap or HGDP data set with a rich bounty of SNPs this is weak tea, but you take what you an get. I don’t even know if cats have their own specialized caste of biologists who focus on them. Where are the cat equivalent of entomologists and and herpetologists? Primatology even exists as a subfield of [mammalogy](https://en.wikipedia.org/wiki/Mammalogy)! The family is good enough to be “pets” but not good enough to warrant specialized scientific interest?  
But enough belly-aching. One point noted in this paper is that previous work using a narrower set of markers tended to give conflicting results. The big cats speciated relatively recently, and can still interbreed, so it isn’t trivial analyzing their relationships with only a few genes. Here is the first figure, which shows earlier trees:  
![cats1.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/cats1.png?resize=500%2C603)![cats1.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/cats1.png?resize=500%2C603)  
*Fig. 1. Prior phylogenetic hypotheses of the genus Panthera. (A and B) based solely on morphological characters, (C-M) from biochemical or molecular studies. (A) Hemmer  
(1978), Herrington (1986), Christiansen (2008). (B) Salles (1992), (C) RFLP of complete mtDNA genomes using 28 restriction endonucleases (Johnson et al., 1996), (D)  
2 mtDNA genes \[647 bp\] (Janczewski et al., 1995), (E) 2 mtDNA genes \[697 bp\] (Johnson and O’Brien, 1997), (F) 4 mtDNA genes \[1435 bp\] (Mattern and McLennan, 2000), (G) 40 source trees, 282 elements (Bininda-Emonds et al., 1999), (H) 1316 chemical characters (Bininda-Emonds et al., 2001), (I) Variation within the hypervariable mtDNA CR Q3 and RS2 (Kim et al. 2001; Jae-Heup et al., 2001), (J) 3 Y-linked \[3604 bp\] (Pecon-Slattery et al., 2004), (K) 6 mtDNA and 3 Autosomal genes \[6500 bp\] (Yu and Zhang, 2005), (L) 7 mtDNA genes \[3816 bp\] (Wei et al., 2009), (M) 19 Autosomal, 5 X, 4 Y, 6 mtDNA genes \[23,920 bp\] (Johnson et al., 2006).*  
Here are the trees they generated:  
![cats2.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/cats2.png?resize=500%2C577)![cats2.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/cats2.png?resize=500%2C577)  
In general, they did not find that the topology of the tree was very sensitive to any particular loci, but there were exceptions. They observe in the discussion:

> One of the more intriguing results from this study was the unique > topology produced from the Transthyretin (TTR) gene segment, which > produced relationships and branch lengths not reproduced by any of the > other gene segments or partitions in the supermatrix, regardless of > size or genomic location…The jacknifing analysis showed that when the > TTR segment was excluded from the autosomal partition, the bipartition > support for relationships changed significantly (Fig. 7). TTR was the > only gene segment that had such a profound effect on topological > rearrangement when removed from the autosomal partition (without the > much longer 730 CES7 sequence)…It seems probable that TTR serves as a > carrier molecule in the urine of big cats, and may be involved in > their territorial markings. **The strong phylogenetic signal tracking > the early divergence between lion-leopard-jaguar and tiger-snow > leopard could be explained if this locus, or one nearby, was a > *speciation gene* that contributed to the initial divergence of these > two clades,** which are independently supported by the Y chromosome, > mtDNA, and supermatrix partitions.

From what I recall “speciation genes” are pretty rare, but in any case, the specific case here illustrates the importance of using many markers so the phylogenetic tree is robust and an accurate reflection of the general pattern of variation within the genome of a species. The phylogenies of genes have a relationship to the phylogenies of species, since the latter are reducible to the former, but different genes may have different phylogenies. Which is why it has been practice to rely on “neutral markers” which have not been subject to natural selection. But of course if you use mtDNA or Y chromosomes you may also see particular sex-specific dynamics, which may not be reflected in the rest of the genome.  
As for why big cat phylogenetics is important, they’re cool. And, it looks like they’re all under threat of being extirpated from the wild. So analyze the non-captive lineages while you can! They might be gone in the blink of an eye. Oh my!  
**Citation:** Davis, B.W., et al. Supermatrix and species tree methods resolve phylogenetic relationships within the big cats, Panthera (Carnivora: Felidae). Mol. Phylogenet. Evol. (2010), doi:10.1016/j.ympev.2010.01.036

### Related Posts:

- [Friday Fluff – December 24th,
  2010](https://www.gnxp.com/WordPress/2010/12/24/friday-fluff-december-24th-2010/) - [The end of phylogenetic
  controversies](https://www.gnxp.com/WordPress/2012/10/07/the-end-of-phylogenetic-controversies/) - [Paleoanthro
  Primer](https://www.gnxp.com/WordPress/2005/03/12/paleoanthro-primer/) - [Genetics for the 21st
  century](https://www.gnxp.com/WordPress/2012/07/03/genetics-for-the-21st-century/) - [Here be snow
  leopards!](https://www.gnxp.com/WordPress/2011/07/14/here-be-snow-leopards/) - [A resolution to the molecular clock
  debates?](https://www.gnxp.com/WordPress/2006/10/30/a-resolution-to-the-molecular-clock-debates/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F02%2F15%2Flions-and-tigers-and-snow-leopards-oh-my%2F&linkname=Lions%2C%20and%20tigers%2C%20and%20snow%20leopards%21%20Oh%2C%20my%21 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F02%2F15%2Flions-and-tigers-and-snow-leopards-oh-my%2F&linkname=Lions%2C%20and%20tigers%2C%20and%20snow%20leopards%21%20Oh%2C%20my%21 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F02%2F15%2Flions-and-tigers-and-snow-leopards-oh-my%2F&linkname=Lions%2C%20and%20tigers%2C%20and%20snow%20leopards%21%20Oh%2C%20my%21 "Email")[](https://www.addtoany.com/share)
