+++
title = "Body odor, Asians, and"
full_title = "Body odor, Asians, and earwax"
date = "2010-10-25"
upstream_url = "https://www.gnxp.com/WordPress/2010/10/25/east-asians-dry-earwax-and-adaptation/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/10/25/east-asians-dry-earwax-and-adaptation/).

Body odor, Asians, and earwax

![Ear](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/Ear.jpg?resize=120%2C188)![Ear](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/Ear.jpg?resize=120%2C188)When I was in college I would sometimes have late night conversations with the guys in my dorm, and the discussion would random-walk in very strange directions. During one of these quasi-salons a friend whose parents were from Korea expressed some surprise and disgust at the idea of wet earwax. It turns out he had not been aware of the fact that the majority of the people in the world have wet, sticky, [earwax](https://en.wikipedia.org/wiki/Earwax). I’d stumbled onto that datum in the course of my reading, and had to explain to most of the discussants that East Asians generally have dry earwax, while convincing my Korean American friend that wet earwax was not something that was totally abnormal. Earwax isn’t something we explore in polite conversation, so it makes sense that most people would be ignorant of the fact that there was inter-population variation on this phenotype.

But it doesn’t end there. Over the past five years the genetics of earwax has[come](http://blogs.discovermagazine.com/gnxp/2006/01/wet-or-dry-ear-wax/) back into the spotlight, because of its variation and what it can tell us about the history and evolution of humans since the Out of Africa event. Not only that, it seems the variation in earwax has some other [phenotypic](https://www.gnxp.com/blog/2007/04/earwax-and-breast-milk.php) [correlates](https://www.gnxp.com/blog/2009/06/ear-wax-to-breast-cancer-risk.php). The SNPs in and around [*ABCC11*](https://en.wikipedia.org/wiki/ABCC11) are a set where East Asians in particular show signs of being different from other world populations. The variants which are nearly fixed in East Asia around this locus are nearly disjoint in frequency with those in Africa. Here are the frequencies of the alleles of [rs17822931](http://www.snpedia.com/index.php/Rs17822931) on *ABCC11* from [ALFRED](http://alfred.med.yale.edu/):  
![abcc11A](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abcc11A.png?resize=600%2C288)![abcc11A](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abcc11A.png?resize=600%2C288)

  
[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)The expression of the dry earwax phenotype is contingent on an AA genotype, it has recessive expression. So in a population where the allele frequency of A \~0.50, the dry earwax phenotype would have a \~0.25 frequency. In a population where the A allele has a \~0.20 frequency, the dry earwax phenotype would be at \~0.04 frequency. Among people of European descent the dry earwax phenotype is present at proportions of less than \~5%. Because of recessive expression a larger minority of Japanese and Chinese should manifest wet earwax, though interestingly the ALFRED database indicates that Koreans are fixed for the A allele. In Africa conversely the G allele seems to be fixed.

**So the question is: why?** A new paper in *Molecular Biology and Evolution* argues that the allele frequency differences are a function of positive directional selection since humans left Africa \~100,000 years ago.[The impact of natural selection on an *ABCC11* SNP determining earwax type](http://www.ncbi.nlm.nih.gov/pubmed/20937735):

> A nonsynonymous single nucleotide polymorphism (SNP), rs17822931-G/A > (538G\>A; Gly180Arg), in the*ABCC11* gene determines human earwax type > (i.e., wet or dry) and is one of most differentiated nonsynonymous > SNPs between East Asian and African populations. A recent genome-wide > scan for positive selection revealed that a genomic region > spanning*ABCC11*,*LONP2*, and*SIAH1* genes has been subjected to a > selective sweep in East Asians. Considering the potential functional > significance as well as the population differentiation of SNPs located > in that region, rs17822931 is the most plausible candidate > polymorphism to have undergone geographically restricted positive > selection. In this study, we estimated the selection intensity or > selection coefficient of rs17822931-A in East Asians by analyzing two > microsatellite loci flanking rs17822931 in the African (HapMap-YRI) > and East Asian (HapMap-JPT and HapMap-CHB) populations. **Assuming a > recessive selection model, a coalescent-based simulation approach > suggested that the selection coefficient of rs17822931-A had been > approximately 0.01 in the East Asian population**, and a simulation > experiment using a pseudo-sampling variable revealed that the mutation > of rs17822931-A occurred 2006 generations (95% credible interval, 1023 > to 3901 generations) ago. **In addition, we show that absolute > latitude is significantly associated with the allele frequency of > rs17822931-A in Asian, Native American, and European populations, > implying that the selective advantage of rs17822931-A is related to an adaptation to a cold climate.** Our results provide a striking example of how local adaptation has played a significant role in the diversification of human traits.

The region around *ABCC11* has come under scrutiny with the emergence of tests of natural selection predicated on inspecting patterns of [linkage disequilibrium](https://en.wikipedia.org/wiki/Linkage_disequilibrium#Linkage_disequilibrium_measure.2C_D) (LD). LD is basically measuring the association of genetic variants within the genome shifted away from expectation. A [selective sweep](https://en.wikipedia.org/wiki/Selective_sweep) tends to generate a lot of LD around the target of natural selection because as the allele in question rises in frequency its neighbors also [hitchhike](https://en.wikipedia.org/wiki/Genetic_hitchhiking) along. The hitchhiking process means that within a population you may see regions of the genome which exhibit long sequences of correlated [single-nucelotide polymorphisms](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism) (SNPs), haplotypes. An initial selective event will presumably generate a very long homogenized block, which over time will break apart through recombination and mutation, as variation is injected back into the genome. The extent and decay of LD then can help us gauge the time and strength of selection events.

But LD can emerge via other processes besides natural selection. Imagine for example that a population of Africans and Europeans mix in a given generation. Europeans and Africans have different genetic makeups, on average, so the initial generations will have more LD than expectation because recombination will only slowly break apart the physical connection between genomic regions from European and African ancestors. The decay of LD then can give one a sense of the [time since admixture](http://blogs.discovermagazine.com/gnxp/2008/03/uyghurs-are-hybrids/) as well as selection. Not only that, stochastic demographic events and processes are also important and may drive the emergence of LD. Consider a bottleneck where the frequency of a particular haplotype is driven up by random genetic drift alone. The details of these alternative scenarios are explored in the 2009 paper [The role of geography in human adaptation](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1000500).

All this is preamble to the fact that **there’s a lot of LD around *ABCC11***. Here’s a visualization from the HapMap populations:

![abcc11B](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abcc11B.png?resize=588%2C330)![abcc11B](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abcc11B.png?resize=588%2C330)

![abc11](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abc11-.png?resize=300%2C280)![abc11](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abc11-.png?resize=300%2C280)From left to right you have Chinese & Japanese, Utah whites, and the Yoruba from Nigeria. An absolute value of *D’* \~0 means that there’s linkage equilibrium; the default or null state where there are no atypical excessive correlations of alleles across the genome. The axes here are pairwise combinations of SNPs around *ABCC11*, with a focus around rs17822931, a nonsynonymousSNP which seems to be the likely functional source of the variance in earwax and other phenotypes. In terms of LD rank order the results are not surprising, across the genome East Asians tend to exhibit more LD than Europeans, and Europeans exhibit more LD than the Yoruba. Part of this is probably a function of population history, a serial bottleneck model Out of Africa would posit that drift and other stochastic forces would have a stronger impact on the genomes of East Asians than Europeans. But this seems like it can’t be the whole picture here; note the variance in allele frequency in the New World as well as in Oceania. Some of the Amerindian populations seem to have a higher frequency of the ancestral G allele on rs17822931. The figure above is easier to understand, the Y-axis is showing you the extent of heterozygosity at a given location. GA is heterozygous, GG is homozygous. Africans again tend to exhibit more heterozygosity than non-Africans, but note the sharply diminished heterozygosity for the East Asian sample around rs17822931 in *ABCC11*. Remember that heterozygosity tends not to go above 0.50 in a random mating population in a diallelic model (though in selective breeding it may go above 0.50 for F1 generations).

The major findings of this paper beyond what was known before seem to be a) an explicit model of how East Asians could have arrived at a high frequency of the AA genotype at rs17822931, and, b) the correlation between climate and the frequency of A. I’ll get to the second point in a bit, but what about the first? Using the nature of variation in two [microsatellites](https://en.wikipedia.org/wiki/Microsatellite) flanking the SNP of interest in East Asians, and assuming a recessive selection model, the authors posit that the A allele began to rise in frequency \~50,000 years ago, and, that the selection coefficient was \~1% per generation. This a significant value for the selection parameter, and the timing is possible in light of the separation of non-Africans into a western and eastern group around that period.

But honestly I’m pretty skeptical of this. The confidence intervals don’t inspire confidence, and from what little I know selection for recessive traits should exhibit less linkage disequilibrium. At low frequencies there is very little affect of natural selection on the allele because it is mostly “masked” in heterozygotes, and therefore there will be a long period before its proportion begins to rise more rapidly. During this time recombination will have time to chop up the haplotypes around the SNP, reducing the length of the statistically associated haplotype block. Also, the authors themselves don’t seem to believe that the phenotype of earwax itself was the target of selection, so its recessive expression pattern should be less important from where I stand.

![abcc11d](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abcc11d.png?resize=300%2C268)![abcc11d](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/10/abcc11d.png?resize=300%2C268)The idea that the genes around *ABCC11* might have something to do with adaptation to cold is suggestive, but almost every East Asian trait of distinction has been hypothesized to have something to do with cold at some point by physical anthropologists. You’d figure that the Cantonese lived in igloos going by all the myriad adaptations to frigid conditions which they exhibit. The reality is that much of China, Korea and Japan are subtropical today. In any case the last figure shows the correlation across several lineages. Earlier they found that by comparing variation around this region in humans with other primates that Africans seem to be subject to purifying selection. This means that there’s constraint so that neutral forces don’t change the frequencies of functionally significant regions. It is well known that on average Africans are more diverse than non-Africans, probably because the latter are a sampling of the former, but, on a small minority of genes the reverse is true. This is likely due to the relaxation of functional constraint as humans left the ancestral African environment. And this is clearly true for rs17822931; most non-African populations exhibit some heterozygosity. East Asians here are an exception, not the rule, at having derived allele frequencies nearly fixed. The regression lines in this last figure are all statistically significant. It is interest that there are particularly strong correlations between latitude and and frequency of the derived A allele among Europeans and Native Americans. In contrast the relationship within Asian populations is weaker. Only 17% of the allele frequency variance can be explained by latitude variance among the Asian ALFRED sample.

But we shouldn’t allow the hypothesis to rise and fall just on this evidence. After all there have likely been substantial movements of populations within the last 10,000. Perhaps *especially* in East Asia, where the expansion of the Han south may have triggered the movement of both the Thai and Vietnamese people out of South China and into mainland Southeast Asia. The best evidence of adaptation would be among admixed populations; presumably those at higher latitudes would have higher frequencies of the AA genotype than those at lower latitudes. Instead of categorizing the populations into three coarse classes probably a more sophisticated treatment using ancestral quanta derived from STRUCTURE or ADMIXTURE as independent variables would be informative. Remember, adaptation should show evidence of decoupling ancestry from phenotype.

Finally, I have to point to this section of the discussion:

> What is the cause of the selective advantage of rs17822931-A? Although > the physiological function of earwax is poorly understood (Matsunaga > 1962), dry earwax itself is unlikely to have provided a substantial > advantage. The rs17822931-GG and GA genotypes (wet earwax) are also **strongly associated with axillary osmidrosis**, suggesting that the ABCC11 protein has an excretory function in the axillary apocrine gland (Nakano et al. 2009)…,

I really didn’t know what this meant. So I looked it up. Here’s what I found, [A strong association of axillary osmidrosis with the wet earwax type determined by genotyping of the *ABCC11* gene](http://www.biomedcentral.com/1471-2156/10/42):

> Apocrine and/or eccrine glands in the human body cause odor, > especially from the axillary and pubic apocrine glands. As in other > mammals, the odor may have a pheromone-like effect on the opposite > sex. Although the odor does not affect health, axillary osmidrosis > (AO) is a condition in which an individual feels uncomfortable with > their axillary odor, regardless of its strength, and may visit a > hospital. Surgery to remove the axillary gland may be performed on > demand. AO is likely an oligogenic trait with rs17822931 accounting > for most of the phenotypic variation and other unidentified functional > variants accounting for the remainder. However, no definite diagnostic > criteria or objective measuring methods have been developed to > characterize the odor, and whether an individual suffers from AO depends mainly on their assessment and/or on examiner’s judgment. Human body odor may result from the breakdown of precursors into a pungent odorant by skin bacteria….

Perhaps the paper should have been titled “why barbarians smell bad”? In any case, an idea for a book title on Korean genetics: “the least smelly race.”\*

**Citation:** Ohashi J, Naka I, & Tsuchiya N (2010). The impact of natural selection on an ABCC11 SNP determining earwax type. Molecular biology and evolution PMID: [20937735](http://www.ncbi.nlm.nih.gov/pubmed/20937735)

\* I’m referencing [The Cleanest Race](https://www.amazon.com/exec/obidos/ASIN/1933633913/geneexpressio-20/).

### Related Posts:

- [The future shall belong to the
  odorless!](https://www.gnxp.com/WordPress/2010/10/27/the-future-shall-belong-to-the-odorless/) - [Earwax and breast
  milk](https://www.gnxp.com/WordPress/2007/04/26/earwax-and-breast-milk/) - [Why the Japanese think Westerners smell bad (well,
  one…](https://www.gnxp.com/WordPress/2014/05/15/why-the-japanese-think-westerners-smell-bad-well-one-reason/) - [Wet or dry ear
  wax?](https://www.gnxp.com/WordPress/2006/01/30/wet-or-dry-ear-wax/) - [Earwax and breast
  cancer](https://www.gnxp.com/WordPress/2009/06/02/earwax-and-breast-cancer/) - [Why the Chinese don't buy
  deodorant](https://www.gnxp.com/WordPress/2018/02/03/why-the-chinese-dont-buy-deodorant/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F10%2F25%2Feast-asians-dry-earwax-and-adaptation%2F&linkname=Body%20odor%2C%20Asians%2C%20and%20earwax "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F10%2F25%2Feast-asians-dry-earwax-and-adaptation%2F&linkname=Body%20odor%2C%20Asians%2C%20and%20earwax "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F10%2F25%2Feast-asians-dry-earwax-and-adaptation%2F&linkname=Body%20odor%2C%20Asians%2C%20and%20earwax "Email")[](https://www.addtoany.com/share)
