+++
title = "The future shall, and"
full_title = "The future shall, and should, be sequenced"
date = "2017-04-02"
upstream_url = "https://www.gnxp.com/WordPress/2017/04/02/the-future-shall-and-should-be-sequenced/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/04/02/the-future-shall-and-should-be-sequenced/).

The future shall, and should, be sequenced

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/gr4.jpg?resize=505%2C504)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/gr4.jpg?resize=505%2C504)](http://www.cell.com/ajhg/fulltext/S0002-9297(17)30107-6)

Last fall I talked about a preprint, [Human demographic history impacts genetic risk prediction across diverse populations](http://biorxiv.org/content/early/2016/11/24/070797). It’s now published in *AJHG*, with the same informative title,[Human Demographic History Impacts Genetic Risk Prediction across Diverse Populations](http://www.cell.com/ajhg/fulltext/S0002-9297(17)30107-6). Even though talked about this before, I thought it would be useful to highlight again.

To recap, GWAS is a pretty big deal, but only in the last 15 years or so. With genome-wide data researchers began to explore associations between diseases and population genetic variation. In some cases they discovered strong associations between characteristics and genetic variants, but in many casese it turned out that though a trait is highly heritable (e.g., schizophrenia) the causal variants are either not common or do not explain much of the variation in the poplation (or both).

But as the second decade of GWAS proceeds the sample sizes are getting larger, and researchers are moving from SNP-chips, with their various biases, to high quality whole-genome sequences. One of the major sorts of low hanging fruit in the minds of many people are **rare variants.** Basically SNP-chips are geared toward finding common variations within large populations, since they have a finite number of markers they are going to interrogate. Sequencing though is a comprehensive catalog of the genome in a relative sense. If you have high coverage (so you sample the site many times) you can easily discover rare mutations within an individual genome that makes them distinctive from almost the rest of the human race (these may be *de novo* mutations, or, they could be mutations private to their extended pedigree).

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/gr1.jpg?resize=300%2C498)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/gr1.jpg?resize=300%2C498)](http://www.cell.com/ajhg/fulltext/S0002-9297(17)30107-6)But context matters. Martin et al. find that confirmed GWAS hits in Europeans tend to exhibit decreased portability as a function of genetic distance. This isn’t entirely surprising, especially if rarer variants are part of the explanation. Rare variants usually emerged later in history, after the differentiation between geographic races.

A solution would be to have a diverse panel of populations in your studies. For many reasons this was not to be. Northwest Europeans are enormously enriched in current data sets. Martin et al. observe that recent this has diminished somewhat, from 95% European to less than 80%. But they observe that this is mostly due to the inclusion of “Asian” samples, as opposed to African and Native Americans, who remain as undererpresented as they did several years ago.

The African and Native American samples present somewhat different problems. The Native American groups are quite drifted due to bottlenecks. Likely they have their own variants due to the combined affects of mutation and selection through 15 to 20,000 years of isolation from other human populations. In contrast, the African groups have lots of diversity with a high time depth due to their ancestral histories, which are less subject to bottleneck effects. The prediction ability into Africans of current GWAS looks to be rather pathetic. This is reasonable because their diversity is poorly captured in Eurocentric study designs, and, they are more genetically diverged from Europeans than Asians are.

Ultimatley I think, and hope, this portability question will be of short term utility. As sequencing gets cheap, and studies become more numerous, we’ll fill in the gaps of understudied populations. Finally, ethics is above my paygrade, but I do hope those who demand a strenuous bar on consent keep in mind that that will result in slower growth of these study populations. Academics want to do a good job, but they also want to stay on the good side of IRB.

**Citation:** Martin, Alicia R., et al. “Human demographic history impacts genetic risk prediction across diverse populations.” *bioRxiv* (2016): 070797.

### Related Posts:

- [Problems in
  PRS?](https://www.gnxp.com/WordPress/2019/05/21/problems-in-prs/) - [Sibling design or bust for polygenic risk
  scores](https://www.gnxp.com/WordPress/2020/07/28/sibling-design-or-bust-for-polygenic-risk-scores/) - [How predictive are known genetic factors for disease
  risk?](https://www.gnxp.com/WordPress/2008/11/19/how-predictive-are-known-genetic-factors-for-disease-risk/) - [Genotype them all, and let GWAS sort it
  out](https://www.gnxp.com/WordPress/2016/08/28/genotype-them-all-and-gwas-sort-it-out/) - [Genome-Wide Associations in
  Asians](https://www.gnxp.com/WordPress/2010/02/12/genome-wide-associations-in-asians/) - [Yes Virginia, trans-ethnic inferences from GWAS are
  kosher](https://www.gnxp.com/WordPress/2013/06/24/yes-virginia-trans-ethnic-inferences-from-gwas-are-kosher/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F02%2Fthe-future-shall-and-should-be-sequenced%2F&linkname=The%20future%20shall%2C%20and%20should%2C%20be%20sequenced "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F02%2Fthe-future-shall-and-should-be-sequenced%2F&linkname=The%20future%20shall%2C%20and%20should%2C%20be%20sequenced "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F02%2Fthe-future-shall-and-should-be-sequenced%2F&linkname=The%20future%20shall%2C%20and%20should%2C%20be%20sequenced "Email")[](https://www.addtoany.com/share)
