+++
title = "How Tibetans can"
full_title = "How Tibetans can function at high altitudes"
date = "2017-04-04"
upstream_url = "https://www.gnxp.com/WordPress/2017/04/04/how-tibetans-can-function-at-high-altitudes/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/04/04/how-tibetans-can-function-at-high-altitudes/).

How Tibetans can function at high altitudes

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/F2.medium.gif?resize=440%2C193)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/F2.medium.gif?resize=440%2C193)](http://www.pnas.org/content/early/2017/03/28/1617042114.full)  
About seven years ago I wrote[two](http://blogs.discovermagazine.com/gnxp/2010/07/why-tibetans-breath-so-easy-up-high/#.WOPoUPnytD8) [posts](http://blogs.discovermagazine.com/gnxp/2010/05/breathing-like-buddha-altitude-tibet/#.WOPoUPnytD8)about how Tibetans manage to function at very high altitudes. And it’s not just physiological functioning, that is, fitness straightforwardly understood. High altitudes can cause a sharp reduction in reproductive fitness because women can not carry pregnancies to term. In other words, **high altitude is a very strong selection pressure.** You adapt, or you die off.

![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/1st_Dalai_Lama.jpg?resize=250%2C312)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/1st_Dalai_Lama.jpg?resize=250%2C312)For me there have been two things of note since those original papers came out. First, one of those loci seem to have been [introgressed from a Denisovan genetic background](http://www.nature.com/nature/journal/v512/n7513/abs/nature13408.html). I want to be careful here, because the initial admixture event may not have been into the Tibetans proper, but earlier hunter-gatherers who descend from Out of Africa groups, who were assimilated into the Tibetans as they expanded 5-10,000 years ago. Second, it turns out that [dogs have been targeted for selection](http://genome.cshlp.org/content/early/2014/04/10/gr.171876.113) on *EPAS1*as well (the “Denisovan” introgression) for altitude adaptation as well.

This shows that in mammals at least there’s a few genes which show up again and again. The fact that *EPAS1* and *EGLN1* were hits on relatively small sample sizes also reinforces their powerful effect. When the *EPAS1* results initiallycame out they were highlighted as the strongest and fastest instance of natural selection in human evolutionary history. One can quibble about the details about whether this was literally true, but that it was a powerful selective event no one could deny.

A new paper in *PNAS*, [Genetic signatures of high-altitude adaptation in Tibetans](http://www.pnas.org/content/early/2017/03/28/1617042114.full), revisits the earlier results with a much larger sample size (the research group is in China) comparing Han Chinese and Tibetans. They confirm the earlier results, but, they also find other loci which seem likely targets of selection in Tibetans. Below is the list:

|                                                                                                                                                                                                                                                                                                                                                                               |        |        |                     |               |             |         |                  | |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|--------|---------------------|---------------|-------------|---------|------------------| | **SNP**                                                                                                                                                                                                                                                                                                                                                                       | **A1** | **A2** | **Frequency of A1** |               | **Pvalue** | **FST** | **Nearest gene** | |                                                                                                                                                                                                                                                                                                                                                                               |        |        | **Tibetan**         | **EAS (Han)** |             |         |                  | | rs1801133                                                                                                                                                                                                                                                                                                                                                                     | A      | G      | 0.238               | 0.333         | 6.30E-09    | 0.021   | MTHFR            | | rs71673426                                                                                                                                                                                                                                                                                                                                                                    | C      | T      | 0.102               | 0.013         | 1.50E-08    | 0.1     | RAP1A            | | rs78720557                                                                                                                                                                                                                                                                                                                                                                    | A      | T      | 0.498               | 0.201         | 4.70E-08    | 0.191   | NEK7             | | rs78561501                                                                                                                                                                                                                                                                                                                                                                    | A      | G      | 0.599               | 0.135         | 6.10E-15    | 0.414   | EGLN1            | | rs116611511                                                                                                                                                                                                                                                                                                                                                                   | G      | A      | 0.447               | 0.003         | 3.60E-19    | 0.57    | EPAS1            | | rs2584462                                                                                                                                                                                                                                                                                                                                                                     | G      | A      | 0.211               | 0.549         | 3.90E-09    | 0.203   | ADH7             | | rs4498258                                                                                                                                                                                                                                                                                                                                                                     | T      | A      | 0.586               | 0.287         | 1.70E-08    | 0.171   | FGF10            | | rs9275281                                                                                                                                                                                                                                                                                                                                                                     | G      | A      | 0.095               | 0.365         | 1.10E-10    | 0.162   | HLA-DQB1         | | rs139129572                                                                                                                                                                                                                                                                                                                                                                   | GA     | G      | 0.316               | 0.449         | 5.80E-09    | 0.036   | HCAR2            | | Pvalue indicates thePvalue from the MLMA-LOCO analysis.FSTis theFSTvalue between Tibetans and EASs. Nearest gene indicates the nearest annotated gene to the top differentiated SNP at each locus exceptEGLN1, which is known to be associated with high-altitude adaptation; rs139129572 is an insertion SNP with two alleles: GA and G. A1, allele 1; A2, allele 2. |        |        |                     |               |             |         |                  |

Many of these genes are familiar. **Observe the allele frequency differences between the Tibetans and other East Asians (mostly Han).** The sample sizes are on the order of thousands, and the SNP-chip had nearly 300,000 markers. What they found was that the between population Fst of Han to Tibetan was \~0.01. So only 1% of the SNP variance in their data was partitioned between the two groups. These alleles are **huge outliers.**

The authors used some sophisticated statistical methods to correct for exigencies of population structure, drift, admixture, etc., to converge upon these hits, but even through inspection the deviation on these alleles is clear. And as they note in the paper it isn’t clear all of these genes are selected simply for hypoxia adaptation. *MTFHR*, which is quite often a signal of selection, may have something to due to folate production (higher altitudes have more UV).*ADH7*is part of a set of genes which always seem to be under selection, and*HLA*is never a surprise.

Rather than get caught up in the details it is important to note here that expansion into novel habitats results in lots of changes in populations, so that two groups can diverge quite fast on functional characteristics. The PCA makes it clear that Tibetans and Hans have very little West Eurasian admixture, and the Fst based analysis puts their divergence on the order of 5,000 years before the present. The authors admit honestly that this is probably a lower bound value, but I also think it is quite likely that Tibetans, and probably Han too, are compound populations, and a simple bifurcation model froma common ancestral population is probably shaving away too many realistic edges. In plainer language, there has been gene flow between Han and Tibetans probably \<5,000 years ago, and Tibetans themselves probably assimilated more deeply diverged populations in the highlands as they expanded as agriculturalists. An estimate of a single divergence fits a complex history to too simple of a model quite often.

The take home: **understanding population history is probably important to get a better sense of the dynamics of adaptation**.

**Citation:** Jian Yang, Zi-Bing Jin, Jie Chen, Xiu-Feng Huang, Xiao-Man Li, Yuan-Bo Liang, Jian-Yang Mao, Xin Chen, Zhili Zheng, Andrew Bakshi, Dong-Dong Zheng, Mei-Qin Zheng, Naomi R. Wray, Peter M. Visscher, Fan Lu, and Jia Qu, Genetic signatures of high-altitude adaptation in Tibetans, PNAS 2017 ; published ahead of print April 3, 2017, doi:10.1073/pnas.1617042114

### Related Posts:

- [Getting better with
  age!](https://www.gnxp.com/WordPress/2007/09/10/getting-better-with-age/) - [We don't know why Ethiopians breathe
  easy](https://www.gnxp.com/WordPress/2012/12/11/we-dont-know-why-ethiopians-breathe-easy/) - [Evolution on
  high](https://www.gnxp.com/WordPress/2007/04/19/evolution-on-high/) - [Deterministic within the stochastic...selection that
  is](https://www.gnxp.com/WordPress/2006/06/16/deterministic-within-the-stochastic-selection-that-is/) - [Breathing like Buddha: altitude &
  Tibet](https://www.gnxp.com/WordPress/2010/05/14/breathing-like-buddha-altitude-tibet/) - [Why the indigenous still dominate the Andean
  region](https://www.gnxp.com/WordPress/2009/05/19/why-the-indigenous-still-dominate-the-andean-region/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F04%2Fhow-tibetans-can-function-at-high-altitudes%2F&linkname=How%20Tibetans%20can%20function%20at%20high%20altitudes "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F04%2Fhow-tibetans-can-function-at-high-altitudes%2F&linkname=How%20Tibetans%20can%20function%20at%20high%20altitudes "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F04%2Fhow-tibetans-can-function-at-high-altitudes%2F&linkname=How%20Tibetans%20can%20function%20at%20high%20altitudes "Email")[](https://www.addtoany.com/share)
