+++
title = "SNP, SV, CNV…does the"
full_title = "SNP, SV, CNV…does the substrate matter?"
date = "2007-10-03"
upstream_url = "https://www.gnxp.com/WordPress/2007/10/03/snp-sv-cnv-does-the-substrate-matter/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/10/03/snp-sv-cnv-does-the-substrate-matter/).

SNP, SV, CNV…does the substrate matter?

[Paired-End Mapping Reveals Extensive Structural Variation in the Human Genome](http://www.sciencemag.org/cgi/content/abstract/1149504):

> Structural variation of the genome involves kilobase- to > megabase-sized deletions, duplications, insertions, inversions, and > complex combinations of rearrangements. We introduce high-throughput > and massive paired-end mapping (PEM), a large-scale genome sequencing > method to identify structural variants (SVs) \~3 kb or larger that > combines the rescue and capture of paired-ends of 3 kb fragments, > massive 454 Sequencing, and a computational approach to map DNA reads > onto a reference genome. PEM was used to map SVs in an African and > putatively European individual and identified shared and divergent SVs > relative to the reference genome. **Overall, we fine-mapped more than > 1000 SVs and documented that the number of SVs among humans is much > larger than initially hypothesized; many of the SVs potentially affect > gene function.** The breakpoint junction sequences of more than 200 > SVs were determined with a novel pooling strategy and computational > analysis. Our analysis provided insights into the mechanisms of SV > formation in humans.

  
[ScienceDaily](http://www.sciencedaily.com/releases/2007/09/070927154931.htm) has a [much more intelligible](http://www.sciencedaily.com/releases/2007/09/070927154931.htm) lede, [Individual Differences Caused By Shuffled Chunks Of DNA In The Human Genome](http://www.sciencedaily.com/releases/2007/09/070927154931.htm):

> “The focus for identifying genetic differences has traditionally been > on point mutations or SNPs — changes in single bases in individual > genes,” said Michael Snyder, the Cullman Professor of Molecular, > Cellular & Developmental Biology and senior author of the study, which > was published in Science Express. “**Our study shows that a > considerably greater amount of variation between individuals is due to > rearrangement of big chunks of DNA**.”

SNP, that’s [single nucleotide polymoprhism](https://en.wikipedia.org/wiki/Single_nucleotide_polymorphism), variation of base pairs on the DNA sequence which are the discrete units of the genome. These have been the traditional “gold standard” in terms of identifying how genetic variation may (or may not!) result in functional variation or modeling the dynamics of molecular evolution. But now there are plenty of papers trying to trumpet [copy number variation](https://www.google.com/search?q=copy+number+variation&btnG=Search%21&domains=scienceblogs.com%2Fgnxp&sitesearch=scienceblogs.com%2Fgnxp), and then there are the [internicine conflicts over evo-devo & *cis* regulatory elements](http://scienceblogs.com/evolgen/2007/07/evodevo_revisited_1.php).  
To all this I say: **a pox on all your houses! There is so much more in nature’s toolkit than are imagined in your philosophies!** There is a specific issue which I think needs to be addressed: perhaps the rise of evo-devo and its cohorts are a necessary correction to the hegemony of the infinite small effect models in population genetics and the focus on SNPs from the molecular angle. But at the end of the day **evolution is substrate neutral**. Variation is variation, and as long as it is heritable then that’s all you need. I tend to find the sequence level variation vs. regulation type of arguments to be epiphenomenal; the sort of scientific debates necessary for the process of science to work itself out, but not consequential in the long run in relation to their genuine impact on evolutionary biology. In short, they’re like the arguments about the term “gene,” they tell you more about what the specific scientists are interested in and the focus of their research project than they do about what a gene might or might not be (I’m pretty instrumentalist about what a “gene” is).

### Related Posts:

- [Beyond SNPs - Structural variation in the human
  genome](https://www.gnxp.com/WordPress/2006/01/23/beyond-snps-structural-variation-in-the-human-genome/) - [Common structural variants are like
  SNPs](https://www.gnxp.com/WordPress/2006/05/11/common-structural-variants-are-like-snps/) - [Macaque
  genome](https://www.gnxp.com/WordPress/2007/04/13/macaque-genome/) - [Structural polymorphisms and
  SNPs](https://www.gnxp.com/WordPress/2006/06/30/structural-polymorphisms-and-snps/) - [Chimp vs. human
  genomes](https://www.gnxp.com/WordPress/2005/09/21/chimp-vs-human-genomes/) - [The alpha of the pack once
  again!](https://www.gnxp.com/WordPress/2006/02/23/the-alpha-of-the-pack-once-again/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F10%2F03%2Fsnp-sv-cnv-does-the-substrate-matter%2F&linkname=SNP%2C%20SV%2C%20CNV%E2%80%A6does%20the%20substrate%20matter%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F10%2F03%2Fsnp-sv-cnv-does-the-substrate-matter%2F&linkname=SNP%2C%20SV%2C%20CNV%E2%80%A6does%20the%20substrate%20matter%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F10%2F03%2Fsnp-sv-cnv-does-the-substrate-matter%2F&linkname=SNP%2C%20SV%2C%20CNV%E2%80%A6does%20the%20substrate%20matter%3F "Email")[](https://www.addtoany.com/share)
