+++
title = "Detecting selection up"
full_title = "Detecting selection up and down the tree of life"
date = "2015-04-07"
upstream_url = "https://www.gnxp.com/WordPress/2015/04/07/detecting-selection-up-and-down-the-tree-of-life/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/04/07/detecting-selection-up-and-down-the-tree-of-life/).

Detecting selection up and down the tree of life

![Is *ADSL* the locus of human genius?](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/04/SELECTION.jpg?resize=600%2C704)

God knows I would sleep more if it weren’t for [bioRxiv](http://biorxiv.org/). A new single author preprint debuts a new method, 3P-CLR, which extends [XP-CLR](http://genome.cshlp.org/content/20/3/393.full), as a method to detect natural selection. The key is that it uses an explicit three-population tree to pick up selection events after the most recent, and second most recent, divergence events. So in the tree of ((Eurasians , Africans)Archaic Humans), this method can pick up perturbations which suggest selection *after* the emergence of a coherent anatomically modern population, but before it differentiated into its gorgeous mosaic.

In any case, the most recent version of the preprint, [Testing for ancient selection using cross-population allele frequency differentiation](http://biorxiv.org/content/early/2015/04/06/017566.1):

> A powerful way to detect selection in a population is by modeling > local allele frequency changes in a particular region of the genome > under scenarios of selection and neutrality, and finding which model > is most compatible with the data. Chen et al. (2010) developed a > composite likelihood method called XP-CLR that uses an outgroup > population to detect departures from neutrality which could be > compatible with hard or soft sweeps, at linked sites near a beneficial > allele. However, this method is most sensitive to recent selection and > may miss selective events that happened a long time ago. To overcome > this, we developed an extension of XP-CLR that jointly models the > behavior of a selected allele in a three-population tree. Our method – > called 3P-CLR – outperforms XP-CLR when testing for selection that > occurred before two populations split from each other, and can > distinguish between those events and events that occurred specifically > in each of the populations after the split. We applied our new test to > population genomic data from the 1000 Genomes Project, to search for > selective sweeps that occurred before the split of Africans and > Eurasians, but after their split from Neanderthals, and that could > have presumably led to the fixation of modern-human-specific > phenotypes. We also searched for sweep events that occurred in East > Asians, Europeans and the ancestors of both populations, after their > split from Africans.

The software will be posted on the author’s [github](https://github.com/ferracimo) when the manuscript is accepted somewhere.

A minor note is that the data set used was from the [1000 Genomes](ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/20131219.populations.tsv). The Sub-Saharan Africans then are *not* from the hunter-gatherer populations, the Khoisan and the Pygmy, who seem to have the largest reservoir of genetic variation. The figure above is from a major signal of selection which is specific to modern humans, but excluded from the Neandertal populations. That is, fixed in us for a derived mutation, fixed in our cousins for the ancestral type (ancestral as judged by reference to the chimpanzee outgroup). My main curiosity is to push the three-population model so that it is ((Khoisan, non-Khoisan)Archaic Humans). I know from ASHG that there are now a fair amount of good quality whole genomes from African hunter-gatherers, so no doubt people are looking for these signatures.

**The holy grail here for some geneticists (e.g., Svante Paabo) is to find that gene or genes which changed in us to make us*sui generis***. I no longer believe that this will ever be found. Assuming tens of millions of polymorphisms floating around in the genome no doubt candidate genes will emerge, just like*FOXP2*did all those years ago. But I no longer believe that there is a necessary or sufficient genetic variant for our humanity. It’s a quantitative trait, and many of the hominin lineages were actually [stumbling](http://blogs.discovermagazine.com/gnxp/2010/04/increased-rate-of-encephalization/) in the same direction.

On a more optimistic note, those of us who work on non-human genomes will also have data sets to rival those who are savants of humanics in the near future, so these methods are generally useful.

**Citation:** Testing for ancient selection using cross-population allele frequency differentiation, Fernando Racimo, bioRxiv doi: http://dx.doi.org/10.1101/017566

### Related Posts:

- [More detecting natural
  selection](https://www.gnxp.com/WordPress/2007/06/18/more-detecting-natural-selection/) - [Local ancestry deconvolution made simpler
  (?)](https://www.gnxp.com/WordPress/2018/07/26/local-ancestry-deconvolution-made-simpler/) - [It's raining selective
  sweeps](https://www.gnxp.com/WordPress/2018/11/03/its-raining-selective-sweeps/) - [The future belongs to morning
  people](https://www.gnxp.com/WordPress/2019/08/16/the-future-belongs-to-morning-people/) - [Complex history of archaic
  ancestry](https://www.gnxp.com/WordPress/2021/07/19/complex-history-of-archaic-ancestry/) - [Composite test for positive selection in the
  genome](https://www.gnxp.com/WordPress/2010/01/07/composite-test-for-positive-selection-in-the-genome/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F04%2F07%2Fdetecting-selection-up-and-down-the-tree-of-life%2F&linkname=Detecting%20selection%20up%20and%20down%20the%20tree%20of%20life "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F04%2F07%2Fdetecting-selection-up-and-down-the-tree-of-life%2F&linkname=Detecting%20selection%20up%20and%20down%20the%20tree%20of%20life "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F04%2F07%2Fdetecting-selection-up-and-down-the-tree-of-life%2F&linkname=Detecting%20selection%20up%20and%20down%20the%20tree%20of%20life "Email")[](https://www.addtoany.com/share)
