+++
title = "Genetic Convection"
full_title = "Genetic Convection"
date = "2015-02-22"
upstream_url = "https://westhunt.wordpress.com/2015/02/22/genetic-convection/"

+++
Source: [here](https://westhunt.wordpress.com/2015/02/22/genetic-convection/).

Genetic Convection

Advantageous gene variants certainly spread by people marrying someone
from the next village, but they can also be spread by population
movements – settling virgin lands, farmers pushing aside foragers, and
of course conquest, our old friend. If a Fisher wave is diffusion,
genetic transport because of those population movements could be
compared to convection, as our commenter RCB has suggested.

People are sometimes interested in estimating the point of origin of a
sweeping allele: this is probably effectively impossible even if
diffusion were the only spread mechanism, since the selective advantage
might well vary in both time and space. But that’s ok, since population
movements – genetic convection – are real and very important. This means
that the difficulties in estimating the origin of a Fisher wave are
totally insignificant, compared to the difficulties of estimating the
effects of past colonizations, conquests and Völkerwanderungs. So when
Yuval Itan and Mark Thomas
[estimated](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000491)
that 13,910 T LCT allele originated in central Europe, in the early
Neolithic, they didn’t just go wrong because of failing to notice that
the same allele is fairly common in northern India: no, their whole
notion was unsound in the first place. We’re talking turbulence on
steroids. Hari Seldon couldn’t figure this one out from the existing
geographic distribution.

Same thing with the EDAR allele: Kamberov et al
[estimate](http://ac.els-cdn.com/S0092867413000676/1-s2.0-S0092867413000676-main.pdf?_tid=6f894fb8-ba63-11e4-9104-00000aab0f27&acdnat=1424589876_7ccba0e400aee50d318e70606f6f7ce5)
that it originated in central China, but there’s no reason to think that
they’re right. Not least because we don’t even understand what the
advantage is.

It looks as if the EDAR mutation was fairly common among the original
Amerindians. If so, how rapidly did it spread in the Americas? Just as
fast the the place was colonized : deep into South America in a thousand
years.

On the other hand, the mere fact that an allele has spread very far is a
strong hint that it was carried by population movements, not just a
Fisher wave. As we gradually figure out ancient population movements, to
a large extent using ancient DNA, we will have a better idea of the
origin and trajectory of such mutations. Seeing LCT 13910 in both Europe
(especially northern Europe) and India suggests that was it was carried
along by the Indo-European expansion. The delf508 Cystic fibrosis
mutation (also found in India) probably was as well, and perhaps the
common 35delG deafness mutation. On the other hand, SLC24A5 and Factor V
Leiden probably came with the Middle Eastern farmers.

Many of these allele look like defenses against infectious disease, or
some kind of adjustment to an agricultural diet: you’d expect that they
originated among either the EEF farmers or the Indo-Europeans, not the
old Mesolithic foragers of Europe.

