+++
title = "1 million SNPs to bind"
full_title = "1 million SNPs to bind us all"
date = "2009-11-23"
upstream_url = "https://www.gnxp.com/WordPress/2009/11/23/1-million-snps-to-bind-us-all/"

+++
Source: [here](https://www.gnxp.com/WordPress/2009/11/23/1-million-snps-to-bind-us-all/).

1 million SNPs to bind us all

A a new paper in [PLoS ONE](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0007888), [Genetic Variation and Recent Positive Selection in Worldwide Human Populations: Evidence from Nearly 1 Million SNPs](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0007888):

> Our analyses both confirm and extend previous studies; in particular, > we highlight the impact of various dispersals, and the role of > substructure in Africa, on human genetic diversity. We also identified > several novel candidate regions for recent positive selection, and a > gene ontology (GO) analysis identified several GO groups that were > significantly enriched for such candidate genes, including immunity > and defense related genes, sensory perception genes, membrane > proteins, signal receptors, lipid binding/metabolism genes, and genes > involved in the nervous system. Among the novel candidate genes > identified are two genes involved in the thyroid hormone pathway that > show signals of selection in African Pygmies that may be related to > their short stature.

They seem to have looked at about twice as many SNPs by combining the sets of Illumina and Affymetrix chips as the norm. But they looked at only around 1/4 the number of individuals as [other studies which used the HGDP panel](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2675971/?tool=pubmed). To a first approximation the Affy and Illumina chips are really close in the patterns of variation which they detect, but, the Illumina chip had a significantly higher heterozygosity (this is evident in some of the supplementals just by inspection).

[](https://www.gnxp.com/blog/uploaded_images/mapselecmap-777117.png)I reformatted a figure which shows ancestral contributions to the individuals in their sample at K = 6 (6 hypothetical populations which contribute to genetic variation). In the paper they discuss the fact that the Uyghur and Hazara resemble each other, and that the Uyghur seem to have a non-trivial Central/South Asian component, and finally that the Russian and Adygei have East Asian and Central/South Asian ancestry. None of this is surprising, [all this was evident in other papers](http://scienceblogs.com/gnxp/2008/02/shadows_of_the_past_in_genes.php) which used the same sample.

First, in regards to Russians, analysis of genetic variation among East European populations sometimes show a “long tail” of variation which leads toward East Asia among Russians. That is, Russians tend to cluster with other Europeans, but a minority of individuals are deviated in the direction of East Asians, that minority shrinking in proportion to distance from Europeans. The historical reason for this presents itself plainly: **a significant minority of ethnic Russians have Tatar antecedents in the recent past, and of those who do not such ancestry may be derived from Slavicized Finno-Ugric populations who may have ancient connections to the populations of Siberia.** The [Russian Orthodox priest who was murdered](http://www.nytimes.com/2009/11/20/world/europe/20moscow.html) last week known for preaching to Muslims was himself an ethnic Tatar by origin.

Second, one should expect the Uyghur and Hazara to resemble each other. The Hazara likely emerged during the period of Mongol rule of Iran and Afghanistan, and are descendants in part of Mongols and Turks from greater Mongolia who settled down in Afghanistan. The Uyghurs are a Turkic-speaking people, but historically the Tarim Basin was inhabited by Europoid populations. The emergence of the Uyghur and Hazara mimic each other almost perfectly. In particular, the East Asian component of their ancestry is from the same region. The non-East Asian aspect differs a bit, but not too much when set next to the East Asian component. Interestingly, the Uyghur speak a Turkic language, while the Hazara speaking Dari, the Persian dialect. One can probably chalk that up to distance from the Turco-Mongol ur-heimat.

Third, the Central/South Asian component among the Uyghur should not be too surprising, there is significant evidence that the Tarim basin was influenced by Indo-Iranians, as well as the Tocharians. Buddhism arrived in East Asia via the Tarim Basin after all, and there have always been trade routes from the southern edge of the Tarim down into northern India. But what about the Russians and the Adygei? I think that this signal has something to do with what we’ve termed elsewhere as [“Ancestral North Indians”](http://scienceblogs.com/gnxp/2009/09/south_asians_as_a_hybrid_popul.php) (ANI), who were closely related to European populations, and probably emerged from somewhere in Eastern Europe to Central Asia. I’ve been told that the Fst number for ANI-Northern European populations is on the order of the distance between Baltic peoples and southern Italians. So this group may have emerged on the margins of Europe, and expanded mostly within Asia.

[](https://www.gnxp.com/blog/uploaded_images/poselec-774968.png)There’s also an interesting chart showing patterns of selection, or at least what they detected, across geographies. Even if most of the signals are false positives one may hold that the real signals within this subset will still recapitulate the geographic relationships shown to the left. The patterns of selection mirror overall phylogenetic relationships. Note the overlap patters of Central/South Asians with Europeans and East Asians, some of both, but dominated by the former.

**Citation:** Lopez Herraez D, Bauchet M, Tang K, Theunert C, Pugach I, et al. 2009 Genetic Variation and Recent Positive Selection in Worldwide Human Populations: Evidence from Nearly 1 Million SNPs. PLoS ONE 4(11): e7888. doi:10.1371/journal.pone.0007888

### Related Posts:

- [Signals of recent positive selection in a worldwide
  sample…](https://www.gnxp.com/WordPress/2009/03/24/signals-of-recent-positive-selection-in-a-worldwide-sample-of-human-populations-again-sort-of/) - [Selectives sweeps and the alleles that love
  them](https://www.gnxp.com/WordPress/2008/09/29/selectives-sweeps-and-the-alleles-that-love-them/) - [Composite test for positive selection in the
  genome](https://www.gnxp.com/WordPress/2010/01/07/composite-test-for-positive-selection-in-the-genome/) - [Natural selection and dopamine receptor
  genes](https://www.gnxp.com/WordPress/2012/03/09/natural-selection-and-dopamine-receptor-genes/) - [What makes human populations
  different?](https://www.gnxp.com/WordPress/2008/02/04/what-makes-human-populations-different/) - [Selection or demography in differences between
  human…](https://www.gnxp.com/WordPress/2009/01/10/selection-or-demography-in-differences-between-human-populations/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F11%2F23%2F1-million-snps-to-bind-us-all%2F&linkname=1%20million%20SNPs%20to%20bind%20us%20all "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F11%2F23%2F1-million-snps-to-bind-us-all%2F&linkname=1%20million%20SNPs%20to%20bind%20us%20all "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F11%2F23%2F1-million-snps-to-bind-us-all%2F&linkname=1%20million%20SNPs%20to%20bind%20us%20all "Email")[](https://www.addtoany.com/share)
