+++
title = "Technology marches on…"
full_title = "Technology marches on…"
date = "2006-10-03"
upstream_url = "https://www.gnxp.com/WordPress/2006/10/03/technology-marches-on/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/10/03/technology-marches-on/).

Technology marches on…

The generation of data on biological functions has taken off exponentially in the past decade or so, and much of that is due to microarray technologies that allow thousands of experiments to be run in parallel. I love data, and the only thing that could possible entice me to get my hands wet in a laboratory is the possibility of generating a lot of it. So microarrays kick ass, but there’s always been a sort of voodoo around it– it’s difficult to measure abolute levels of the RNA you’re measuring (I get the impression things are more reliable when doing things like CGH with DNA), and things aren’t always totally replicable from platform to platform or from lab to lab. Things have gotten better, of course, and they will continue to do so. From a [perspective article](http://www.genome.org/cgi/content/full/16/10/1195?etoc) in the most recent Genome Research:

> \[T\]here is agreement that \[mircoarray technology\] ought to be able > to detect RNA from small amounts of sample material, even single > cells, in a way that faithfully represents RNA abundances. In > addition, there would be advantages to describing abundance levels in > absolute terms-numbers or molar amounts-rather than relative values, > so that comparisons between genes and across many experiments can be > undertaken. Furthermore, the dynamic range of microarrays should match > the range of expression levels found in cells. Indeed, if the > sensitivity, dynamic range, and quantitative nature of measurements > could be improved, the current need for cross-validation with > real-time PCR would become redundant.  
> …  
> The range of mRNA abundance levels in biological systems can approach > 6 orders of magnitude, which clearly cannot be addressed by the \~10^3 > dynamic range of current microarray experiments. In contrast, the > Hesse investigators found the dynamic range of their single molecule > detection system to be 4.7 orders of magnitude. In particular, the > range at the lower end, at which regulatory molecules may be > expressed, can be extended with greater confidence. It is possible > that by using an appropriate single molecule microarray system, the > full 6 orders of magnitude of biological expression levels can be > addressed in a single readout.

Just something to whet your appetite…

### Related Posts:

- [Innovation!](https://www.gnxp.com/WordPress/2009/08/23/innovation/) - [Arrays on the way
  out?](https://www.gnxp.com/WordPress/2007/05/17/arrays-on-the-way-out/) - [Reanalysing gene expression differences between
  populations](https://www.gnxp.com/WordPress/2007/06/28/reanalysing-gene-expression-differences-between-populations/) - [Tools for a better
  genomics](https://www.gnxp.com/WordPress/2006/12/14/tools-for-a-better-genomics/) - [Fitness and
  disease](https://www.gnxp.com/WordPress/2006/05/19/fitness-and-disease/) - [Highly parallel genomic
  assays](https://www.gnxp.com/WordPress/2006/07/20/highly-parallel-genomic-assays/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F03%2Ftechnology-marches-on%2F&linkname=Technology%20marches%20on%E2%80%A6 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F03%2Ftechnology-marches-on%2F&linkname=Technology%20marches%20on%E2%80%A6 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F03%2Ftechnology-marches-on%2F&linkname=Technology%20marches%20on%E2%80%A6 "Email")[](https://www.addtoany.com/share)
