+++
title = "What drives accelerated"
full_title = "What drives accelerated molecular evolution?"
date = "2006-10-02"
upstream_url = "https://www.gnxp.com/WordPress/2006/10/02/what-drives-accelerated-molecular-evolution/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/10/02/what-drives-accelerated-molecular-evolution/).

What drives accelerated molecular evolution?

In the paper describing the discovery of [Human Accelerated Region 1](https://www.gnxp.com/blog/2006/08/fastest-evolving-human-gene.php) (HAR1), an RNA gene expressed in the brain and apparently under selection along the human lineage, the authors allude to the method they used to identify this region of the genome. The [paper describing what they did in more detail](http://genetics.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pgen.0020168.eor) is in revision at PLoS Genetics and is thus freely available.

Their basic approach was to find regions of the genome that were highly conserved throughout most of vertebrate evolution, but particulaly divergent in humans. To do this, they first aligned the chimp, mouse, and rat genomes to find highly conserved regions. They then looked at the genomes of 17 other species (including humans) and compare two phylogenetic models for each of the conserved blocks– a model where the substutution rate is held constant to a model where an additional parameter for the human substitution rate is added. If the two models are significantly different, then *voila*, humans have experienced faster molecular evolution in that area.

They eventually come up with 202 interesting regions of the genome this way. The most impressive one they follow up a bit, as [described previously](https://www.gnxp.com/blog/2006/08/fastest-evolving-human-gene.php). The regions they find are primarily non-coding, and located near genes involved in transcription and DNA binding. As is warranted, they give a nod to [King and Wilson](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=retrieve&db=pubmed&list_uids=1090005&dopt=Abstract).

But have these areas really been under natural selection? One thing they note is that the substitutions they find are disproportionately AT–\>GC mutations, and are located predominantly in areas with high recombination. Could the these substitutions simply be a [neutral byproduct of the amount of recombination](http://mbe.oxfordjournals.org/cgi/content/abstract/21/6/984) in these areas?

It’s certainly possible, but one possibility, raised by a professor of mine, is that recombination hotspots aid the efficacy of natural selection. That is, in order for a number of benefecial mutations (which have arisen on different backgrounds) to sweep to selection with any efficiency, they must be on the same haplotype. The only way for two benefical mutations on different haplotypes to end up together? Recombination. So perhaps the presence of HARs in regions of high recombination is not an argument for their neutrality, but rather a hint as to the mechanisms that allow for strong positive selection on a region.

### Related Posts:

- [Accelerated human evolution in non-coding regions?
  (reprise)](https://www.gnxp.com/WordPress/2006/11/04/accelerated-human-evolution-in-non-coding-regions-reprise/) - [Accelerated human evolution in non-coding
  regions?](https://www.gnxp.com/WordPress/2006/11/02/accelerated-human-evolution-in-non-coding-regions/) - [AT --\> GC mutations, recombination and
  selection](https://www.gnxp.com/WordPress/2006/10/02/at-gc-mutations-recombination-and-selection/) - [Conservation of expression in human and mouse
  brains](https://www.gnxp.com/WordPress/2007/04/23/conservation-of-expression-in-human-and-mouse-brains/) - [New Pritchard paper, Adaptive evolution of
  conserved…](https://www.gnxp.com/WordPress/2007/07/18/new-pritchard-paper-adaptive-evolution-of-conserved-non-coding-elements-in-mammals/) - [Imprinted gene undergoing accelerated evolution in
  humans?](https://www.gnxp.com/WordPress/2007/03/14/imprinted-gene-undergoing-accelerated-evolution-in-humans/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F02%2Fwhat-drives-accelerated-molecular-evolution%2F&linkname=What%20drives%20accelerated%20molecular%20evolution%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F02%2Fwhat-drives-accelerated-molecular-evolution%2F&linkname=What%20drives%20accelerated%20molecular%20evolution%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F02%2Fwhat-drives-accelerated-molecular-evolution%2F&linkname=What%20drives%20accelerated%20molecular%20evolution%3F "Email")[](https://www.addtoany.com/share)
