+++
title = "More neuronal microRNAs"
full_title = "More neuronal microRNAs on the way"
date = "2006-10-23"
upstream_url = "https://www.gnxp.com/WordPress/2006/10/23/more-neuronal-micrornas-on-the-way/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/10/23/more-neuronal-micrornas-on-the-way/).

More neuronal microRNAs on the way

One poster at the SFN conference last week described a microRNA (miR132) discovered using a novel screening technique for learning related genes that controls dendrite growth and production of new synapses. The method is called Serial Analysis of Chromatin Occupancy (SACO). The team that first produced SACO in 2004 focused on a transcription factor called CREB (CyclicAMP Response Element Binding Protein, this is NOT CPEB). Part of CREB’s popularity stems from the emphasis it has received as a sort of final common path for long-term memory processes in the scheme presented by Eric Kandel in his Nobel work. In that view, CREB is activated by signaling pathways initiated during learning, such as the cyclic-AMP dependent protein kinase (PKA). Activated CREB then goes to the nucleus and sits next to parts of the genome that it wants to regulate up or down. Soren Impey and Daniel Storm showed a few years ago that mice carrying a reporter gene (lacZ) with 6 places for CREB to sit (CREs) in front of it showed more reporter expression in the hippocampus following learning than following simple stimulus exposure. There are many questions as to why you would want to bother to regulate genes at the transcriptional level in response to learning. It seems slow and clunky compared to translating a pre-existing RNA near the affected synapses, but the level of many transcripts does change in the minutes to hours following learning or induction of synaptic plasticity. One assumes it is not for nothing.

When a transcription factor like CREB settles down on the DNA you can play a dirty trick and glue it to its seat using formaldehyde. In the initial SACO study, they activated CREB using a drug that activates PKA and then glued it to all the different parts of the DNA that it lighted upon. They then pulled CREB and whatever DNA would come with it out of solution and sequenced 21 nucleotide strings of it. 21 nucleotides turns out to be just enough to uniquely identify a region in a large genome. These segments represent candidate CREB binding sites, especially if the same segment comes up multiple time. So they sorted through all these potential CREB binding sites and found that most of them were either in genes or near them, so that was nice. Some of the SACO-identified sites were near microRNAs. One of those microRNAs is miR132 and (I guess because it has such a pretty name) Impey and co decided to follow it up.

In the SFN poster, they showed an increase in miR132 with neuronal activity. Activity in cell cultures (where most of this work was done) can lead to dendritic growth and production of new synapses, but not if miR132 isn’t around. The straightforward story then is that activity activates CREB which goes to the nucleus and causes transcription of miR132 which then must come out and inhibit the translation of some protein? They used several predictoin algorithms to try and pick out what neuronal gene miR132 might antagonize and came up with a protein called p250GAP. We’d have to get involved in a whole other signaling pathway to explain what p250GAP does, but let’s just say that it is in prime position to regulate the cytoskeleton and therefore dendritic and synaptic morphology. Note that miR132 works in the opposite direction from miR134 in terms of synapse growth (although they could be active at different times and in different areas). Reports in Drosophila that RNA interference associated machinery is degraded in response to activity raised the possibility that RNAs that promote synaptic growth and strength could be regulated as a group by RNAi and released from inhibition in concert. But alas, with microRNAs having opposite functional effects the appealing idea of coordinated regulation seems less plausible now.

### Related Posts:

- [APOE4: a blessing and a
  curse](https://www.gnxp.com/WordPress/2007/01/13/apoe4-a-blessing-and-a-curse/) - [Associative neurons in the
  amygdala](https://www.gnxp.com/WordPress/2006/10/19/associative-neurons-in-the-amygdala/) - [BDNF and Arc regulation: NMDARs vs
  AMPARs](https://www.gnxp.com/WordPress/2006/06/16/bdnf-and-arc-regulation-nmdars-vs-ampars/) - [HAR1F
  tangents](https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/) - [MicroRNA binding sites under selection in
  humans](https://www.gnxp.com/WordPress/2006/11/01/microrna-binding-sites-under-selection-in-humans/) - [What causes regulatory evolution between
  species?](https://www.gnxp.com/WordPress/2008/09/20/what-causes-regulatory-evolution-between-species/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F23%2Fmore-neuronal-micrornas-on-the-way%2F&linkname=More%20neuronal%20microRNAs%20on%20the%20way "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F23%2Fmore-neuronal-micrornas-on-the-way%2F&linkname=More%20neuronal%20microRNAs%20on%20the%20way "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F10%2F23%2Fmore-neuronal-micrornas-on-the-way%2F&linkname=More%20neuronal%20microRNAs%20on%20the%20way "Email")[](https://www.addtoany.com/share)
