+++
title = "Visualizing"
full_title = "Visualizing intraEuropean phylogenetic distances"
date = "2017-12-10"
upstream_url = "https://www.gnxp.com/WordPress/2017/12/10/visualizing-intra-european-phylogenetic-distances/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/12/10/visualizing-intra-european-phylogenetic-distances/).

Visualizing intra-European phylogenetic distances

![Neighbor-joining tree of genetic distances between populations](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/12/European.jpg?resize=625%2C427)



[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/12/historygeo.jpeg?resize=195%2C258)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/12/historygeo.jpeg?resize=195%2C258)](https://www.amazon.com/exec/obidos/ASIN/0691087504/geneexpressio-20)In L. L. Cavalli-Sforza’s [The History and Geography of Human Genes](https://www.amazon.com/exec/obidos/ASIN/0691087504/geneexpressio-20) he used between population group genetic distances, as measured in F_(ST) values, to generate a series of visualizations, which then allowed him to infer historical processes. Basically the way it works is that you look at genetic variation, and see how much of it can be allocated to **between** groups. If none of it can be allocated to **between** groups, then in a population genetic sense it doesn’t make much sense to speak of distinctive groups, they’re basically one breeding population. The higher the F_(ST) statistic is, the more of the variation is partitioned between the groups.

Roughly this is used to correlate with genetic distance as well as evolutionary divergence. The longer two populations have been separated, the more and more genetic differences they’ll accumulate, inflating the F_(ST) value. There are a lot of subtleties that I’m eliding here (see [Estimating and interpreting F_(ST): the impact of rare variants](https://www.ncbi.nlm.nih.gov/pubmed/23861382) for a survey of the recent literature on the topic and pathways forward), **but for a long time, F_(ST) was the go-to statistic for making phylogenetic inferences on a within-species scale.**

Today we have other techniques, Structure, Treemix, fineStructure, and various local ancestry packages.

But F_(ST) is still useful to give one a *Gestalt* sense of population genetic differences. Cavalli-Sforza admits in [The History and Geography of Human Genes](https://www.amazon.com/exec/obidos/ASIN/0691087504/geneexpressio-20) that European populations had very low pairwise F_(ST), but because of the importance of Europe for sociocultural reasons a detailed analysis of the region was still provided in the text. Additionally, they had lots of European samples (non-European Caucasoids were thrown into one category for macro-group comparisons because there wasn’t that many samples).

Using results from the 2015 paper [Massive migration from the steppe was a source for Indo-European languages in Europe](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5048219/), **I visualized pairwise genetic distances for European populations, ancient and modern** **(Han Chinese as an outgroup), on a tree.** What the results illustrate is that

1.  Ancient populations were very distinct in Europe from modern ones.
2.  Many modern groups are clustered close together.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/12/pursuitofglory.jpeg?resize=181%2C279)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/12/pursuitofglory.jpeg?resize=181%2C279)](https://www.amazon.com/exec/obidos/ASIN/B000WW0LFA/geneexpressio-20)The bulk of the population genetic structure in modern Europe seems to have been established in the period between 3000 BCE and 2000 BCE. This is not that much time for a lot of distinctiveness to develop, especially on the geographically open North European plain. I suspect with more and more Mesolithic and early to middle Neolithic DNA we’ll see that some of the modern population structure is a ghost of ancient substrate absorption.

Many of the ethno-national categories that are very significant in recent history, and impact the cultural memories of modern people and their genealogies, have very shallow roots. This does not mean they are not “real” (I don’t know what that’s supposed to mean at all), **just that many of the identities which seem so salient to us today may be relatively recent in terms of their significance to large groups of humans**….

### Related Posts:

- [1 migrant needed to prevent genetic
  divergence](https://www.gnxp.com/WordPress/2012/01/26/1-migrant-needed-to-prevent-genetic-divergence/) - [Ashkenazi Jews are more European in
  ancestry](https://www.gnxp.com/WordPress/2009/09/04/ashkenazi-jews-are-more-european-in-ancestry/) - [Visualizing caste & linguistic differences in
  India…](https://www.gnxp.com/WordPress/2009/09/27/visualizing-caste-linguistic-differences-in-india-fst/) - [Reprising genes &
  geography](https://www.gnxp.com/WordPress/2012/04/16/reprising-genes-geography/) - [Luca Cavalli-Sforza
  biography](https://www.gnxp.com/WordPress/2005/11/21/luca-cavalli-sforza-biography/) - [A picture is worth a thousand words, part
  n](https://www.gnxp.com/WordPress/2008/04/21/a-picture-is-worth-a-thousand-words-part-n/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F12%2F10%2Fvisualizing-intra-european-phylogenetic-distances%2F&linkname=Visualizing%20intra-European%20phylogenetic%20distances "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F12%2F10%2Fvisualizing-intra-european-phylogenetic-distances%2F&linkname=Visualizing%20intra-European%20phylogenetic%20distances "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F12%2F10%2Fvisualizing-intra-european-phylogenetic-distances%2F&linkname=Visualizing%20intra-European%20phylogenetic%20distances "Email")[](https://www.addtoany.com/share)
