+++
title = "Cattle genetic variation"
full_title = "Cattle genetic variation & evolution"
date = "2009-04-23"
upstream_url = "https://www.gnxp.com/WordPress/2009/04/23/cattle-genetic-variation-evolution/"

+++
Source: [here](https://www.gnxp.com/WordPress/2009/04/23/cattle-genetic-variation-evolution/).

Cattle genetic variation & evolution

There are some papers out on the genome of the domestic cow out right now. [ScienceNews](http://www.sciencenews.org/view/generic/id/43190/title/Cattle_genome_sequenced) has an overview:

> Two competing research teams have cataloged the “essence of bovinity” > found in the DNA of cattle, but not without disagreement on some > essential points.  
> Reporting online April 23 in Science and April 24 in Genome Biology, > the two groups compiled drafts of the bovine genome, identifying genes > important for fighting disease, digesting food and producing milk.

I don’t see the *Genome Biology* paper on the site yet, but there are two in *Science*. First, [The Genome Sequence of Taurine Cattle: A Window to Ruminant Biology and Evolution](http://www.sciencemag.org/cgi/content/full/324/5926/522):

> To understand the biology and evolution of ruminants, the cattle > genome was sequenced to about sevenfold coverage. The cattle genome > contains a minimum of 22,000 genes, with a core set of 14,345 > orthologs shared among seven mammalian species of which 1217 are > absent or undetected in noneutherian (marsupial or monotreme) genomes. > Cattle-specific evolutionary breakpoint regions in chromosomes have a > higher density of segmental duplications, enrichment of repetitive > elements, and species-specific variations in genes associated with > lactation and immune responsiveness. Genes involved in metabolism are > generally highly conserved, although five metabolic genes are deleted > or extensively diverged from their human orthologs. **The cattle > genome sequence thus provides a resource for understanding mammalian > evolution and accelerating livestock genetic improvement for milk and > meat production.**

This paper focuses on the big picture taxonomic context of bovines within the bigger family tree of mammals, and how their genome is similar and/or different. [Orthologs](https://en.wikipedia.org/wiki/Homology_(biology)#Orthology) are just genes in different species which had a common ancestor before speciation, and often retain similar functions (though not always). They look at which genes have reorganized or seem to have been subject to selection in cattle, and not unsurprisingly note:

> **There has been substantial reorganization of gene families encoding > proteins present in milk.** One such rearrangement affecting milk > composition involves the histatherin (HSTN) gene within the casein > gene cluster on BTA6 (fig. S21). In the cattle genome, HSTN is > juxtaposed to a regulatory element (BCE) important (23) for β-casein > (CSN2) expression, and as a probable consequence, HSTN is regulated > like the casein genes during the lactation cycle. This rearrangement > that led to the juxtaposition of HSTN next to the BCE is also the > probable cause of deletion of one of the two copies of -S2-like casein > genes (CSN1S2A) present in other mammalian genomes (24). The > biological implications of this change in casein gene copy number are > not yet clear.

The whole paper is an alphabet soup of genes, so probably you’ll want to do a few rereads. The second paper, [Genome-Wide Survey of SNP Variation Uncovers the Genetic Structure of Cattle Breeds](http://www.sciencemag.org/cgi/content/full/324/5926/528), zooms in on the evolutionary history of cattle and the phylogenetic relationship of various breeds:

> The imprints of domestication and breed development on the genomes of > livestock likely differ from those of companion animals. A deep draft > sequence assembly of shotgun reads from a single Hereford female and > comparative sequences sampled from six additional breeds were used to > develop probes to interrogate 37,470 single-nucleotide polymorphisms > (SNPs) in 497 cattle from 19 geographically and biologically diverse > breeds. These data show that cattle have undergone a rapid recent > decrease in effective population size from a very large ancestral > population, possibly due to bottlenecks associated with domestication, > selection, and breed formation. **Domestication and artificial > selection appear to have left detectable signatures of selection > within the cattle genome, yet the current levels of diversity within > breeds are at least as great as exists within humans.**

Here are the figures of note:  
![cattlestruc.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/cattlestruc.png?resize=500%2C659)![cattlestruc.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/cattlestruc.png?resize=500%2C659)  
Remember that each *K* refers to assigning the genes to *K* populations, and seeing how the individual’s genomes sort out as you increase the number of *K*‘s. The PC charts are rather self-evident; populations of common descent cluster with each other and so forth.  
I read through the papers quickly, but I didn’t see anything that jumped out at me as too surprising. Cattle have been subject to evolution due to disease and selection for milk production. The Indian derived variants are in a different clade from the other cattle. A large ancestral population seems to have gone through a bottleneck event, probably due to domestication. Nevertheless, even if there isn’t surprising I assume that plowing through the genomes of organism after organism will start yielding some confounds of expectation.  
Of course, knowing the nature of cattle genetic variation could lead to a
[‘supercow’](http://www.abc.net.au/news/stories/2009/04/24/2551319.htm) if you believe press reports….  
(as it happens I wonder if a supercow is going to be impossible because of the [genetic correlation](https://www.google.com/cse?cx=017254414699180528062:uyrcvn__yd0&q=g+matrix+site:http://scienceblogs.com/gnxp/&sa=Search))

### Related Posts:

- [Building better
  beef](https://www.gnxp.com/WordPress/2007/01/12/building-better-beef/) - [Got milk, in
  Turkey?](https://www.gnxp.com/WordPress/2008/08/07/got-milk-in-turkey/) - [Mobile genetic elements in the wooly
  mammoth](https://www.gnxp.com/WordPress/2009/06/08/mobile-genetic-elements-in-the-wooly-mammoth/) - [Genome-wide association for heart
  disease](https://www.gnxp.com/WordPress/2007/06/08/genome-wide-association-for-heart-disease/) - [Tasmanian Tiger mtDNA
  sequenced?](https://www.gnxp.com/WordPress/2009/01/12/tasmanian-tiger-mtdna-sequenced/) - [Purring in the post-genomic
  era](https://www.gnxp.com/WordPress/2014/11/10/purring-in-the-post-genomics-era/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F04%2F23%2Fcattle-genetic-variation-evolution%2F&linkname=Cattle%20genetic%20variation%20%26%20evolution "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F04%2F23%2Fcattle-genetic-variation-evolution%2F&linkname=Cattle%20genetic%20variation%20%26%20evolution "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F04%2F23%2Fcattle-genetic-variation-evolution%2F&linkname=Cattle%20genetic%20variation%20%26%20evolution "Email")[](https://www.addtoany.com/share)
