+++
title = "When a trait isn't a"
full_title = "When a trait isn't a trait isn't a trait"
date = "2010-04-12"
upstream_url = "https://www.gnxp.com/WordPress/2010/04/12/when-a-trait-isnt-a-trait-isnt-a-trait/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/04/12/when-a-trait-isnt-a-trait-isnt-a-trait/).

When a trait isn't a trait isn't a trait

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)One of the great things about evolutionary theory is that it is a formal abstraction of specific concrete aspects of reality and dynamics. It allows us to squeeze inferential juice from incomplete prior knowledge of the state of nature. In other words, you can make predictions and models instead of having to observe every last detail of the natural world. But abstractions, models and formalisms often leave out extraneous details. Sometimes those details turn out not to be so extraneous. Charles Darwin’s original theory of evolution had no coherent or plausible mechanism of inheritance. R. A. Fisher and others imported the empirical reality of Mendelism into the logic of evolutionary theory, [to produce the framework of 20th century population genetics](https://www.amazon.com/exec/obidos/ASIN/0226684644/geneexpressio-20/). Though accepting the genetic inheritance process of Mendelism this is original synthesis was not informed by molecular biology, because it pre-dated molecular biology. After James Watson and Francis Crick uncovered the biophysical basis for Mendelism molecular evolution came to the fore, and [neutral theory](https://en.wikipedia.org/wiki/Neutral_theory_of_molecular_evolution) emerged as a response to the particular patterns of genetic variation which new molecular techniques were uncovering. And yet through this much of R. A. Fisher’s image of an abstract genetic variant floating against a statistical soup of background noise variation persisted, sometimes dismissed as [“bean bag genetics”](http://www.doubletongued.org/index.php/citations/bean_bag_genetics_1/).

We’ve come a long way from the first initial wave of discussions which were prompted by the molecular genetic revolution. We have [epigenetics](https://en.wikipedia.org/wiki/Epigenetics), [evo-devo](https://en.wikipedia.org/wiki/Evolutionary_developmental_biology) and variation in gene regulation. **None of these processes “overthrow” evolutionary biology, though in some ways they may revolutionize aspects of it.** Science is over the long haul after all an eternal revolution, as the boundaries of comprehension keep getting pushed outward. A few days ago[I pointed to Sean Carroll’s recent work](http://blogs.discovermagazine.com/gnxp/2010/04/sean-carrolls-the-biologist-double-duty-day/), which emphasizes that one must think beyond the sequence level, and focus on particular features such as [*cis*-regulartory elements](https://en.wikipedia.org/wiki/Cis-regulatory_element). Here we’ve been tunneling down to the level of the gene, but what about the traits, the phenotypes, which are affected by genetic variation?

It is well known that the sparest abstraction of genotypic-phenotypic relationship can be illustrated like so:

genetic variation → phenetic variation

But each element of this relation has to be examined greater detail. What type of genetic variation? Sequence level variation? Epigenetic variation? The second component is perhaps the most fraught, with the arrow waving away the myriad details and interactions which no doubt lurk between genotype and phenotype. And finally you have the phenotype itself. Are they all created alike in quality so that we can ascribe to them dichotomous values and quantities?

A new paper in *PNAS* examines the particulars of morphological phenotypes and physiological phenotypes, and their genetic control, as well as rates of evolution. [Contrasting genetic paths to morphological and physiological evolution](http://www.pnas.org/content/early/2010/03/25/0910339107.abstract):  

> The relative importance of protein function change and gene expression > change in phenotypic evolution is a contentious, yet central topic in > evolutionary biology. Analyzing 5,199 mouse genes with recorded mutant > phenotypes, we find that genes exclusively affecting morphological > traits when mutated (dubbed “morphogenes”) are grossly enriched with > transcriptional regulators, whereas those exclusively affecting > physiological traits (dubbed “physiogenes”) are enriched with > channels, transporters, receptors, and enzymes. Compared to > physiogenes, morphogenes are more likely to be essential and > pleiotropic and less likely to be tissue specific. Morphogenes evolve > faster in expression profile, but slower in protein sequence and gene gain/loss than physiogenes. **Thus, morphological and physiological changes have a differential molecular basis; separating them helps discern the genetic mechanisms of phenotypic evolution.**

Morphology here refers to gross anatomical features. The sort of traits and characteristics which a paleontologist or anatomist might take interest in. Physiology is more about function, and the physical structures which enable that function. It is naturally closer to the scale of molecular biology as physiology melts into biochemistry. Of course at the other end physiology also merges with anatomy as physiology occurs within features of interest to the anatomist. By way of generalization perhaps physiology may be considered more granular, while morphology more gross, in the context of this paper.

They used the mouse because it’s a species which has long served as a model organism, and there are a host of well known and characterized mutations for both physiology and morphology. Utilization of mice in these fields in the context of evolutionary research dates back to the early 20th century. So systems biologists have a lot of research that’s already been done to work with. They found 5199 mouse genes with known phenotypes in the Mouse Genome Informatics database. 821 affected only morphological traits and 912 affected only physiological traits.

Figure 1 shows the breakdown by [Gene Ontology](https://en.wikipedia.org/wiki/Gene_Ontology):

![morphfig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig1.png?resize=600%2C277)![morphfig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig1.png?resize=600%2C277)

Going by what little I know about these topics the second to the fourth panels aren’t surprising. Morphological traits are built from molecular structures, while the transporter activity classes are a more cellular scale, and so would seem to be below the threshold of salience for morphological traits. The first panel is not something I’d expected, but it makes sense after the fact. Figure 2 clarifies. The right panels have proportions, the left counts.

![morphfig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig11.png?resize=600%2C277)![morphfig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig11.png?resize=600%2C277)

The primary point is this: morphogenes seem to affect more traits than physiogenes, and, their affect is less tissue specific when it comes to a particular trait. When this pattern is highlighted the enrichment toward transcriptional regulation makes more sense to me it is transcriptional regulation might allows for more trait by trait level control of variation. If there is a relationship of many traits to one gene that would probably impose a constraint on the sequence level to a greater extent than if the gene was implicated in variation on one trait. The gap in pleiotropy is closed somewhat when you constrain to essential genes, those whose mutation results in decrease of fitness to zero (through death or lack of ability to reproduce). Pleiotropy presumably is constraining the genetic landscape toward particular fitness peaks. Tissue specificity seems understandable when you consider the localization of many physiological processes, and their biochemical complexities (I’m thinking of the vagaries of gene expression in the liver here).

But they looked at more than how the traits and genes distribute now, they tried to sniff out if there were differences in the rate of evolution of morphogenes and physiogenes contingent upon the class of genetic variants. Remember that you have sequent level changes on exons which can alter proteins. You have *cis*-acting elements as critical cogs in gene regulation. And you have more gross genomic features such as gene duplication or deletion.

![morphfig3](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig3.png?resize=450%2C500)![morphfig3](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig3.png?resize=450%2C500)Figure 3 shows the differences between mice and humans on particular genes in relation to sequence level substitutions as well as gene expression profiles. Specifically in the case of the former you want to know the rate of nonsynonymous substitution, those substitutions at base pairs which change the amino acid translated, standardized by the overall mutation rate. So panel C is the one to focus on. Note that physiogenes seem to have evolved more since the last divergence between human and mice lineages than morphogenes. Why might this be? An immediate thought that comes to mind is that tissue-specific expressing physiological processes are liable to be modulated more often than gross morphology, which might be controlled by genes with a lot of pleiotropic effects and so constrained. Even when you control to tissue-specificity the pattern remains, as evident in panel D. The pattern seems somewhat inverted in relation to rate of evolution when it comes to gene expression profiles, as you can see in the last three panels. Evolution happens, but by somewhat different genetic means in these cases. The authors finger pleiotropy in particular as the problem for sequence level evolution in morphogenes, as changes in proteins are much more likely to be problematic if those proteins are upstream from many more traits.

In a way these results show that evolution has to be a versatile designer. When it comes to physiogenes the illustrator is in charge, creating new traits from the most basic genetic raw material, changes in a base pair here and a base pair there. But for morphogenes evolution has to use the tools and tricks of photoshopping, making recourse to extant elements and rearranging or tweaking things here and there so as not to upset the complex applecart while modulating on the margins.

What about *cis*-acting regulatory elements? In the paper they allude to the argument of [Sean Carroll](http://scienceblogs.com/evolgen/2007/08/is_sean_carroll_wrong.php) that *cis*-acting regulatory elements are critical for the evolution of morphological traits. That would imply that morphogenes should be enriched vis-a-vis physiogenes for changes on these elements. They didn’t find that in figure 4. On the contrary.

![morphfig4](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig4.png?resize=600%2C355)![morphfig4](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/04/morphfig4.png?resize=600%2C355)

But I don’t think they perceive their result as a rock-solid refutation of Carroll because it was somewhat indirect. I’ll quote from the paper:

> …Because experimentally confirmed mammalian *cis* elements are few, > are likely to have been confirmed in only one species, and are > potentially biased toward certain classes of genes,we tested the above > hypothesis by using *cis*-elements that were **predicted** exclusively > by motif sequence conservation among a set of vertebrate genome > sequences and recorded in the *cis*RED database (20). In *cis*RED, > 8,440 predicted mouse cis-elements and 7,688 **predicted** human > cis-elements were found to be in the proximity of 586 mouse morphogenes and their human orthologs, respectively. Similarly, 7,082 mouse cis-elements and 7,215 human cis-elements were predicted for 621 physiogenes….

I’m inclined to accept this result and its generalizability, but there’s a layer of analysis and modeling in this case which doesn’t exist in the others. Additionally, Carroll’s thesis is about the whole animal kingdom and a mouse-human comparison may be atypical.

Finally they wanted to look at gene duplication. They found:

> Together with the D_(fam) result, our analyses show that, whereas physiogene families expand/contract faster than morphogene families, the rate of expansion/contraction is relatively constant across lineages for a given family.

I wonder if the duplication here might have something to do with modulating dosages of various substrates in biochemical processes. This may have more direct relevance to physiological processes.

It is important to note as they did that the category “morphogene” and “physiogene” is somewhat artificial, as is the distinction between morphology and physiology. Nature is fundamentally one, and we break it apart as particular joints for ease of our own abstractions and categorizations. Additionally all genes presumably have some effect on morphology and physiology, and though this exploration looks under the hood a bit more than some of the older abstractions it too is a simplification. The key is that the argument here seems to be that these breaking apart of categories and processes gives us useful marginal return in comprehension of evolutionary dynamics. A trait is not always just a trait. Different classes of phenotypes may have different evolutionary genetic implications by their very nature. Some of this is common sense, those traits which are less functionally significant will exhibit more genic variation. But distinctions in terms of form and function themselves are at a further level of detail. And, I presume that generalizations that we make from mouse-human comparisons as here have some limitations across the tree of life.

**Citation:** Liao BY, Weng MP, & Zhang J (2010). Contrasting genetic paths to morphological and physiological evolution. Proceedings of the National Academy of Sciences of the United States of America PMID: [20368429](http://www.ncbi.nlm.nih.gov/pubmed/20368429)

### Related Posts:

- [The theory of
  evolution](https://www.gnxp.com/WordPress/2008/05/09/the-theory-of-evolution/) - [Happy Darwin Day! Now read some
  Fisher....](https://www.gnxp.com/WordPress/2016/02/12/happy-darwin-day-now-read-some-fisher/) - [The Genetical Theory of Natural Selection, preface &
  chapter…](https://www.gnxp.com/WordPress/2006/07/22/the-genetical-theory-of-natural-selection-preface-chapter-1/) - [Why evolutionary science is genetic
  science](https://www.gnxp.com/WordPress/2015/04/04/why-evolutionary-science-is-genetic-science/) - [Evolution's gears don't reverse
  easily](https://www.gnxp.com/WordPress/2011/05/12/evolutions-gears-dont-reverse-easily/) - [The once and future genomics of sexual
  selection](https://www.gnxp.com/WordPress/2015/03/25/the-once-and-future-genomics-of-sexual-selection/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F04%2F12%2Fwhen-a-trait-isnt-a-trait-isnt-a-trait%2F&linkname=When%20a%20trait%20isn%27t%20a%20trait%20isn%27t%20a%20trait "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F04%2F12%2Fwhen-a-trait-isnt-a-trait-isnt-a-trait%2F&linkname=When%20a%20trait%20isn%27t%20a%20trait%20isn%27t%20a%20trait "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F04%2F12%2Fwhen-a-trait-isnt-a-trait-isnt-a-trait%2F&linkname=When%20a%20trait%20isn%27t%20a%20trait%20isn%27t%20a%20trait "Email")[](https://www.addtoany.com/share)
