+++
title = "A success for genomewide"
full_title = "A success for genomewide assciation studies?"
date = "2008-07-02"
upstream_url = "https://www.gnxp.com/WordPress/2008/07/02/a-success-for-genome-wide-assciation-studies/"

+++
Source: [here](https://www.gnxp.com/WordPress/2008/07/02/a-success-for-genome-wide-assciation-studies/).

A success for genome-wide assciation studies?

Recent advances in genotyping have made genome-wide association studies the standard way to study the genetics of a phenotype in humans (model organisms, for various reasons, are suddenly lagging behind). In terms of simply identifying loci underlying disease, this approach has led to a number of notable successes, but there’s been a nagging concern that identifying loci of small effect doesn’t help much with diagnostics–even with [30 loci now know to impact height](https://www.gnxp.com/blog/2008/04/tall-loci-genetics-of-height.php), genetics is able to account for something like 3% of the total variance in the phenotype despite the fact that the heritability of the trait is something like 90%.

Different traits have different genetic architectures, of course, so the recent publication of 30 loci involved in Crohn’s disease has led to [vastly different results](http://www.nature.com/ng/journal/vaop/ncurrent/abs/ng.175.html):

> To advance gene discovery further, we combined data from three studies > on Crohn’s disease (a total of 3,230 cases and 4,829 controls) and > carried out replication in 3,664 independent cases with a mixture of > population-based and family-based controls. The results strongly confirm 11 previously reported loci and provide genome-wide significant evidence for 21 additional loci, including the regions containing STAT3, JAK2, ICOSLG, CDKAL1 and ITLN1.

So with the same number of loci, how much of the variance are these authors able to explain? It’s 10%, a modest number, but larger than that currently mapped for most (any?) other complex trait. And keeping in mind that the heritability of Crohn’s disease is estimated at 50%, they’ve accounted for something like a fifth of the total genetic variance. I’m not sure exactly what threshold makes genetics clinically useful, but surely this is approaching it.

### Related Posts:

- [Is the "missing heritability" right under
  our…](https://www.gnxp.com/WordPress/2010/06/21/is-the-missing-heritability-right-under-our-noses/) - [Genome-wide association study for breast
  cancer…](https://www.gnxp.com/WordPress/2007/06/28/genome-wide-association-study-for-breast-cancer-susceptibility/) - [Following up association
  studies](https://www.gnxp.com/WordPress/2008/10/06/following-up-association-studies/) - [More pigmentation
  genetics](https://www.gnxp.com/WordPress/2008/05/19/more-pigmentation-genetics/) - [Notes on the Common Disease-Common Variant debate: two
  years…](https://www.gnxp.com/WordPress/2009/04/17/notes-on-the-common-disease-common-variant-debate-two-years-later/) - [Asia finally getting in on the genome-wide association
  game?](https://www.gnxp.com/WordPress/2008/08/18/asia-finally-getting-in-on-the-genome-wide-association-game/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F07%2F02%2Fa-success-for-genome-wide-assciation-studies%2F&linkname=A%20success%20for%20genome-wide%20assciation%20studies%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F07%2F02%2Fa-success-for-genome-wide-assciation-studies%2F&linkname=A%20success%20for%20genome-wide%20assciation%20studies%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F07%2F02%2Fa-success-for-genome-wide-assciation-studies%2F&linkname=A%20success%20for%20genome-wide%20assciation%20studies%3F "Email")[](https://www.addtoany.com/share)
