+++
title = "What causes regulatory"
full_title = "What causes regulatory evolution between species?"
date = "2008-09-20"
upstream_url = "https://www.gnxp.com/WordPress/2008/09/20/what-causes-regulatory-evolution-between-species/"

+++
Source: [here](https://www.gnxp.com/WordPress/2008/09/20/what-causes-regulatory-evolution-between-species/).

What causes regulatory evolution between species?

It’s widely accepted that many of the differences between species are due to changes in gene regulation, rather than in protein sequences themselves. It’s plausible, however, that changes in the sequences of transcription factors could lead to large-scale changes in gene regulation in a relatively simple manner–just a single nucleotide substitution could alter transcription factor binding across the whole genome.

Is this how evolution has worked? A [new study](http://www.sciencemag.org/cgi/content/abstract/1160930) takes advantage of a pretty cool resource–a mouse carrying human chromosome 21–to answer no, it doesn’t appear so.

The authors begin by noting that several transcription factors bind different spots in human and mouse livers. With the mouse carrying human chromosome 21, they can pretty definitively test whether this difference in binding is due to changes in the transcription factors themselves or to changes in binding site sequence. **Remarkably, it doesn’t seem to matter whether the human chromosome is located in a mouse or a human–the same human regulatory sequences are bound**. Similarly, the locations of an epigentic mark they test seems to depend almost solely on the species the chromosome belongs to rather than the species in which it’s located.

This has enormous implications–as the authors write:

> Here we show that each layer of transcriptional regulation within the > adult hepatocyte, from the binding of liver master regulators and > chromatin remodeling complexes to the output of the transcriptional > machinery, is directed primarily by DNA sequence. Although > conservation of motifs alone cannot predict transcription factor > binding, we show that **within the genetic sequence there must be > embedded adequate instructions to direct species-specific > transcription.**

Now it’s “just” a question of deciphering those instructions.

### Related Posts:

- [Stabilized, not
  activated](https://www.gnxp.com/WordPress/2006/09/27/stabilized-not-activated/) - [Evolution of Primate
  Regulation](https://www.gnxp.com/WordPress/2008/11/21/evolution-of-primate-regulation/) - [Gene expression makes the
  man](https://www.gnxp.com/WordPress/2006/03/08/gene-expression-makes-the-man/) - [Regulatory change in
  autism](https://www.gnxp.com/WordPress/2006/10/22/regulatory-change-in-autism/) - [HAR1F
  tangents](https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/) - [Extensive transcription in the
  fly](https://www.gnxp.com/WordPress/2006/10/01/extensive-transcription-in-the-fly/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F09%2F20%2Fwhat-causes-regulatory-evolution-between-species%2F&linkname=What%20causes%20regulatory%20evolution%20between%20species%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F09%2F20%2Fwhat-causes-regulatory-evolution-between-species%2F&linkname=What%20causes%20regulatory%20evolution%20between%20species%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F09%2F20%2Fwhat-causes-regulatory-evolution-between-species%2F&linkname=What%20causes%20regulatory%20evolution%20between%20species%3F "Email")[](https://www.addtoany.com/share)
