+++
title = "Buddy, can you spare"
full_title = "Buddy, can you spare some ascertainment?"
date = "2012-12-18"
upstream_url = "https://www.gnxp.com/WordPress/2012/12/18/buddy-can-you-spare-some-ascertainment/"

+++
Source: [here](https://www.gnxp.com/WordPress/2012/12/18/buddy-can-you-spare-some-ascertainment/).

Buddy, can you spare some ascertainment?

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/12/Screenshot-from-2012-12-18-093200.png?resize=470%2C359 "Screenshot from 2012-12-18 09:32:00")![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/12/Screenshot-from-2012-12-18-093200.png?resize=470%2C359 "Screenshot from 2012-12-18 09:32:00")](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/12/Screenshot-from-2012-12-18-093200.png)

The above map shows the population coverage for the [Geno 2.0](http://shop.nationalgeographic.com/ngs/browse/productDetail.jsp?productId=2001246&gsk) SNP-chip, put out by the [Genographic Project](https://genographic.nationalgeographic.com). Their paper outlining the utility and rationale by the chip is now out on [arXiv](http://arxiv.org/abs/1212.4116). I saw this map last summer, when Spencer Wells hosted a webinar on the launch of Geno 2.0, and it was the aspect which really jumped out at me. The number of markers that they have on this chip is modest, only \>100,000 on the autosome, with a few tens of thousands more on the X, Y, and mtDNA. In contrast, the [Axiom® Genome-Wide Human Origins 1 Array Plate](http://www.affymetrix.com/browse/products.jsp?productId=prod560001&categoryId=cat20004&productName=Axiom%26%23174%3B-Genome%26%2345%3BWide-Human-Origins-1-Array-Plate#1_1) being used by [Patterson et al.](http://www.genetics.org/content/early/2012/09/06/genetics.112.145037.full.pdf) has \~600,000 SNPs. But as is clear by the map above Geno 2.0 is ascertained in many more populations that the other comparable chips (Human Origins 1 Array uses 12 populations). It’s obvious that if you are only catching variation on a few populations, all the extra million markers may not give you much bang for the buck (not to mention the biases that that may introduce in your population genetic and phylogenetic inferences).

  
[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/12/Screenshot-from-2012-12-18-094951.png?resize=320%2C338 "Screenshot from 2012-12-18 09:49:51")![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/12/Screenshot-from-2012-12-18-094951.png?resize=320%2C338 "Screenshot from 2012-12-18 09:49:51")](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/12/Screenshot-from-2012-12-18-094951.png)To the left are the list of populations against which the Human Origins 1 Array was ascertained, and they look rather comprehensive to me. In contrast, for Geno 2.0 ‘ancestrally informative markers’ were ascertained on 450 populations. The ultimate question for me is this: is all the extra ascertainment on diverse and obscure groups worth it? On first inspection Geno 2.0’s number of SNPs looks modest as I stated, **but in my experience when you quality control and merge different panels together you are often left with only a few hundred thousand SNPs in any case**. 100-200,000 SNPs is also sufficient to elucidate relationships even in genetically homogeneous regions such as Europe in my experience (it’s more than enough for model-based clustering, and seems to be overkill for MDS or PCA). One issue that jumps out at me about the Affymetrix chip is that it is ascertained toward the antipodes. In contrast, Geno 2.0 takes into account the Eurasian heartland. I suspect, for example, that Geno 2.0 would be better for population or ancestry assignment for South Asians because it would have more informative markers for those populations.

Ultimately I can’t really say much more until I use both marker sets in different ***and*** similar contexts. Since Geno 2.0 consciously excludes many functional and medically relevant SNPs its utility is primarily in the domain of demographics and history. If the populations in question are well covered by the Human Origins 1 Array, I see no reason why one shouldn’t go with it. Not only does it have more information about biological function, but the number of markers are many fold greater. On the other hand, Geno 2.0 may be more useful on the “blank zones” of the Affy chip. Hopefully the Genographic Project results paper for Geno 2.0 will come out soon and I can pull down their data set and play with it.

**Cite:** arXiv:1212.4116

### Related Posts:

- [The Genographic Project: on to the
  autosome!](https://www.gnxp.com/WordPress/2012/07/25/the-genographic-project-onto-the-autosome/) - [The Genographic Project's Scientific Grants
  Program](https://www.gnxp.com/WordPress/2012/11/11/the-genographic-projects-scientific-grants-program/) - [Spencer Wells in Vanity
  Fair](https://www.gnxp.com/WordPress/2007/07/01/spencer-wells-in-vanity-fair/) - [Spencer Wells
  interview](https://www.gnxp.com/WordPress/2007/03/31/spencer-wells-interview/) - [Geneographic project in
  PLOS](https://www.gnxp.com/WordPress/2007/06/29/geneographic-project-in-plos/) - [Arrays on the way
  out?](https://www.gnxp.com/WordPress/2007/05/17/arrays-on-the-way-out/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F12%2F18%2Fbuddy-can-you-spare-some-ascertainment%2F&linkname=Buddy%2C%20can%20you%20spare%20some%20ascertainment%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F12%2F18%2Fbuddy-can-you-spare-some-ascertainment%2F&linkname=Buddy%2C%20can%20you%20spare%20some%20ascertainment%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F12%2F18%2Fbuddy-can-you-spare-some-ascertainment%2F&linkname=Buddy%2C%20can%20you%20spare%20some%20ascertainment%3F "Email")[](https://www.addtoany.com/share)
