+++
title = "RNA regulons"
full_title = "RNA regulons"
date = "2007-08-23"
upstream_url = "https://www.gnxp.com/WordPress/2007/08/23/rna-regulons/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/08/23/rna-regulons/).

RNA regulons

One of my favorite recent ideas wondering through the literature is that of an RNA regulon or post-transcriptional operon. Operons in prokaryotes are groups of genes whose protein products all function in the same biochemical pathway. The genes are coordinated by sticking them all next to each other and transcribing all when you transcribe one. The post-transcriptional operon idea is that RNA motifs allow proteins in the same biochemical pathway to be regulated at the translation step instead. If several proteins were needed, for instance, to build some new architecture sticking off a cell at a specific location far from the nucleus, it wouldn’t do to have to coordinate them way back there. Instead, you just throw in an RNA motif, say AUUUA. Then produce an RNA binding protein that is specific for that motif. Now traffic that protein to the location of interest. All of the RNAs will be localized to the right spot.

Of course, localizaton is just one way this could work. Any process better controlled faster or farther away from the nucleus could use an RNA regulon. One notable case is that of the Pumilio family (Puf) RNA-binding proteins in yeast. Melissa J. Moore explains it [here](http://www.sciencemag.org/cgi/content/full/309/5740/1514):

> … each Puf protein exhibited a highly skewed distribution of bound > mRNAs: Puf1p and Puf2p bound mostly mRNAs encoding membrane-associated > proteins, Puf3p almost exclusively targeted messages for > nuclear-encoded mitochondrial proteins, and Puf4p and Puf5p associated > primarily with transcripts encoding proteins bound for the nucleus. In > several cases, a majority of the subunits comprising a particular > multiprotein machine, such as the mitochondrial ribosome and a number > of nuclear chromatin modification complexes, were encoded by mRNAs > “tagged” by a single Puf protein. Together with earlier data (12), > these new results (16) strongly support the idea that the expression > of proteins with common functional themes or subcellular distributions > is coordinated by large-scale regulatory networks operating at the > mRNP level.

Many other examples can be found in [this review by Jack Keene](http://www.nature.com/nrg/journal/v8/n7/full/nrg2111.html). I don’t think I’ve seen an example of this yet, but given the slight wobble in microRNA specificity, one could imagine a single microRNA regulating a whole set of genes. Also, most interesting for my neuro-tastes is [the recent report from the Moore lab](https://dx.doi.org/10.1016/j.cell.2007.05.028) showing that the immediate-early gene implicated in neuronal homeostasis, Arc, may be part of a regulon defined by introns in the 3’UTR. The mechanism is just too clever but requires an explication on the “pioneer round” of translation. Basically the cell tricks itself into thinking it made a funky RNA and destroys it after one round of synthesis. The other RNAs regulated in this path in neurons must have opposing effects to Arc though because knocking down this negative regulation pathway led to increased excitability (increased Arc reduces neuronal excitability). This raises a more general question. The idea of RNA regulons is nice, but how much can you predict knowing that your gene of interest is part of one? RNAs associate with multiple complexes throughout their lifespan, and complexes gain and lose factors dynamically. Also, how promiscuous are RNA binding proteins for cellular processes? For instance, I originally became aware of the Hu proteins as positive regulators of the pre-synaptic calcium-buffering protein GAP-43, but it turns out that they also regulate proteins involved in immune function. Maybe I am just thinking at too high a level of cellular organization. Perhaps all of those proteins respond to calcium in some way. At any rate, I’m expecting that RNA regulons will be increasingly important in understanding the translational regulation that must take place in dendrites to produce persistent memories. Looking forward to more on that in the next year or so.

### Related Posts:

- [A mechanism for miRNA-mediated
  repression](https://www.gnxp.com/WordPress/2007/06/19/a-mechanism-for-mirna-mediated-repression/) - [HAR1F
  tangents](https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/) - [A Nobel Prize for gene
  expression](https://www.gnxp.com/WordPress/2006/10/02/a-nobel-prize-for-gene-expression/) - [RNA
  activation](https://www.gnxp.com/WordPress/2006/11/02/rna-activation/) - [RNA Recoding of Brain
  Proteins](https://www.gnxp.com/WordPress/2005/03/16/rna-recoding-of-brain-proteins/) - [I'd be fired if that were my
  job](https://www.gnxp.com/WordPress/2006/11/24/i-d-be-fired-if-that-were-my-job/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F23%2Frna-regulons%2F&linkname=RNA%20regulons "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F23%2Frna-regulons%2F&linkname=RNA%20regulons "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F23%2Frna-regulons%2F&linkname=RNA%20regulons "Email")[](https://www.addtoany.com/share)
