+++
title = "Positive selection in"
full_title = "Positive selection in regulatory sequences"
date = "2007-08-18"
upstream_url = "https://www.gnxp.com/WordPress/2007/08/18/positive-selection-in-regulatory-sequences/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/08/18/positive-selection-in-regulatory-sequences/).

Positive selection in regulatory sequences

As many of our readers are aware, humans and chimpanzees are rather different, and have diverged considerably since we last shared a common ancestor a few million years ago. An interesting question in evolutionary biology is: what the hell happened? What makes us so different? Comparing the consensus genome sequences of humans and chimps shows millions and millions of base pairs that have changed; which of those are functionally relevant?

One of the hypotheses that has been “rediscovered” in the last few years is that the important changes should be involved in gene regulation, rather than protein sequence itself (this is of course from the classic [King and Wilson](http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=retrieve&db=pubmed&list_uids=1090005&dopt=AbstractPlus) paper). A [new paper](http://www.nature.com/ng/journal/vaop/ncurrent/abs/ng2104.html) takes a look at putative regulatory regions at a number of genes, and concludes that, yes, some of them have been under positive selection since the divergence of humans and chimps. Further, they run a quick and dirty analysis on the functions of the genes whose regulation appear to be under positive selection, and see some enrichment in neural and nutrition-related categories. From this, they conclude, “the present survey…suggests that human cognitive, behavioral and dietary adaptations have arisen primarily through changes in *cis*-regulatory sequences.” Meh.

That could, of course, be true. But frankly, the input of this paper didn’t really change the weight I put on that possibility at all (that is, I’d estimate the [Bayes factor](https://en.wikipedia.org/wiki/Bayes_factor) of this paper at about zero). Here’s why:

1\. I’m starting to become somewhat ill-at-ease with tests for selection that are based on estimating evolutionary substitution rates (in this paper, they compare rates at promoters with those in introns). What exactly do tests like these detect? Ultimately, this is a population genetics question– how many new positively selected alleles have to arise and become fixed for this test to be significant? And why would you expect that many are necessary for phenotypic change (as opposed to, say, one really well-placed substitution)? For example, [a recent paper](https://www.gnxp.com/blog/2007/07/evolution-by-mutations-of-large.php) showed that three regulatory substitutions completely accounted for a major morphological difference between two *Drosophila* species. Is that enough for a significant result in a test like this? If not, what kind of functional biases are being introduced into the results?

2\. I’m always ill-at-ease with results based on enrichment in [gene ontology](https://en.wikipedia.org/wiki/Gene_ontology) categories. In this case, the most significant result is in “protein folding”, which is neither a neural nor nutrition-related category. The next ones: 0.01 p-values for “other neuronal activity” and “neurogenesis”, and a 0.02 p-value for “neuronal activities”. The categories are non-disjoint (presumably “neuronal activities” and “other neuronal activity” share many of the same genes, for example), so are the enrichment results due to a few genes that fall in every neural-related category? And how can one conclude, based on this evidence, that “human cognitive, behavioral and dietary adaptations have arisen primarily through changes in *cis*-regulatory sequences” ? Given what we’re working with, that’s still pretty much speculation.

So yes, changes in regulatory sequences seem to be important in driving the divergence of humans from chimps (though not necessarily to the exclusion of other changes). But the precise changes, and the traits they involve, are certainly still up in the air.

### Related Posts:

- [Why the regulatory changes vs. coding sequence
  changes…](https://www.gnxp.com/WordPress/2006/10/15/why-the-regulatory-changes-vs-coding-sequence-changes-debate-is-inane/) - [The genome, more than
  coding](https://www.gnxp.com/WordPress/2009/08/07/the-genome-more-than-coding/) - [Neandertal autosomal sequence goes
  live](https://www.gnxp.com/WordPress/2006/07/20/neandertal-autosomal-sequence-goes-live/) - [Neandertal autosomal sequence goes
  live](https://www.gnxp.com/WordPress/2006/07/20/neandertal-autosomal-sequence-goes-live/) - [Evolution of Primate
  Regulation](https://www.gnxp.com/WordPress/2008/11/21/evolution-of-primate-regulation/) - [What makes us human
  (genetically)?](https://www.gnxp.com/WordPress/2006/10/17/what-makes-us-human-genetically/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F18%2Fpositive-selection-in-regulatory-sequences%2F&linkname=Positive%20selection%20in%20regulatory%20sequences "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F18%2Fpositive-selection-in-regulatory-sequences%2F&linkname=Positive%20selection%20in%20regulatory%20sequences "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F18%2Fpositive-selection-in-regulatory-sequences%2F&linkname=Positive%20selection%20in%20regulatory%20sequences "Email")[](https://www.addtoany.com/share)
