+++
title = "The evolutionary effect"
full_title = "The evolutionary effect of the sky gods"
date = "2011-04-24"
upstream_url = "https://www.gnxp.com/WordPress/2011/04/24/the-evolutionary-effect-of-the-sky-gods/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/04/24/the-evolutionary-effect-of-the-sky-gods/).

The evolutionary effect of the sky gods

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)Last week [I reviewed ideas](http://blogs.discovermagazine.com/gnxp/2011/04/the-court-jester-and-the-averaging-fallacy/) about the effect of “exogenous shocks” to an ecosystem of creatures, and how it might reshape their evolutionary trajectory. These sorts of issues are well known in their generality. **They have implications from the broadest macroscale systematics to microevolutionary process.** The shocks point to changes over *time* which have a general effect, but what about exogenous parameters which shift spatially and regularly? I’m talking *latitudes* here. The further you get from the equator the more the climate varies over the season, and the lower the mean temperature, and, the less the aggregate radiation the biosphere catches. [Allen’s rule](https://en.wikipedia.org/wiki/Allen%27s_rule) and [Bergmann’s rule](https://en.wikipedia.org/wiki/Bergmann's_rule) are two observational trends which biologists have long observed in relation to many organisms. The equatorial variants are slimmer in their physique, while the polar ones are stockier. Additionally, there tends to be an increase in mean mass as one moves away from the equator.

**But these rules are just general observations.** What process underlies these observations? The likely culprit would be natural selection of course. But the specific manner in which this process shakes out, on both the organismic and genetic level, still needs to be elucidated in further detail. A new paper in *PLoS Genetics* attempts to do this more rigorously and deeply than has been done before for one particular world wide mammalian species, *H. sapiens sapiens*. We have spanned the latitudes and longitudes, and so we’re a perfect test case for an exploration of the broader microevolutionary forces which shape variation.

The paper is [Adaptations to Climate-Mediated Selective Pressures in Humans](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1001375?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+plosgenetics%2FNewArticles+%28Ambra+-+Genetics+New+Articles%29). Its technical guts can be intimidating, but its initial questions and final answers are less daunting. So let’s jump straight to the last paragraph of the discussion:

> **The results of this genome scan not only increase our understanding > of the genetic landscape of adaptation across the human genome, but > they may also have a more practical value**. For example, they can be > used to select candidate genes for common disease risk and to generate > specific testable hypotheses regarding the functions of specific genes > and variants. While the results of genome-wide scans for association > with diseases and other traits are accumulating at a rapid pace, > interpretation of these results is often ambiguous because the power > to detect all common variants that are important in the etiology of > the phenotype is incomplete. This is especially true in the case of > complex traits, where variants at many loci may contribute to the > phenotype, each with a small effect. By combining the evidence from > GWAS with evidence of selection, it may be possible to separate true > causative regions from the background noise inherent in genome-wide > screens for association. To facilitate this, all of our empirical rank > statistics are publically available. Moreover, results of selection > scans that detect evidence for spatially-varying selection may be > especially relevant to diseases that show substantial differences in > prevalence across ethnic groups (e.g., sodium-sensitive hypertension, > type 2 diabetes, prostate cancer, osteoporosis). In the future, this > approach could be extended by including additional populations and > aspects of the environment to gain a more complete understanding of > how natural selection has shaped variation across the genome in > worldwide populations. Furthermore, whereas we relied on linkage disequilibrium between (potentially un-genotyped) adaptive variants and genotyped SNPs, whole genome re-sequencing data should give a more complete picture of the variation that underlies adaptation.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.jpg?resize=308%2C391)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.jpg?resize=308%2C391)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.jpg)How’d they infer this? First, they had a pretty wide coverage of populations from across the world. They pooled the HGDP and HapMap, as well as a few other populations of interest, Ethiopians, some Siberian groups, and Australian Aboriginals. I do wish that the Aboriginal data set was public, but it doesn’t seem to be! The Ethiopians are I assume the ones you can find in [Behar et al.](http://www.harappadna.org/2011/01/behar-et-al-data/) The authors had a null model which was predicated on the fact that variation in the frequencies of given genetic morphs, single nucleotide polymorphisms, should be bested predicted by population history and relationships. That is, two populations will differ on a given locus in proportion to their genetic divergence, due to random forces such as genetic drift. **Perturbations from this null model are possible targets of natural selection, which reshapes regions of the genome in a deterministic manner aiming at particular ends**. Two 21st century classic examples of this phenomenon seem to be skin pigmentation and lactase persistence. Different populations with the same phenotype, in particular, light skin and the ability to digest lactose sugar as an adult, exhibit divergent genetic architectures.

They naturally looked to see how these deviations tracked environmental parameters you see above. Keep in mind that they did take into account correlations between these variables. Additionally, correlation does not equal causation, so there could be other variables which are correlated with the ones which they explored which might be responsible for the systematic perturbations.

[](http://blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.jpg) Their method yielded a Bayes factor (BF) which measures the deviation from the null model for a given SNP. To judge off the bat whether these SNPs are plausibly the targets of adaptation you want to check to see if they’re enriched for certain classes of SNPs. They found that the SNPs which rejected the null model, where population history and demographics predicts genetic variation, tended to be much more likely to be genic or nonsynonymous. This means that the base pair is embedded in a coding gene, as opposed to much of the genome which isn’t translated into proteins. A nonsynonymous base pair is one at a location which changes the protein coded. Normally these sorts of changes are selected against because you don’t want to change the protein function, but when a population is adapting to a new environment this is obviously not so.

There are a host of results in the paper, but one pattern which seemed of interest was that different sets of SNPs can be selected in different population pools. Below are two panels which show the SNPs with significant BF, and how they vary as a function of the climatic variable depending upon the populations which are sampled. To the left you see the cluster which varied in western Eurasia, while in the left you see those which varied in eastern Eurasia. In a broad sense the target of selection was the same, but the specific SNPs which were pulled out the set of potential targets still exhibits stochasticity:

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.g002.jpg?resize=600%2C265)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.g002.jpg?resize=600%2C265)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.g002.jpg)

**Natural selection is deterministic in the broadest scale, but in its instantiations it can exhibit a great deal of randomnes.** Same phenotype. Different genotype. Similarly, the heat death of the universe may be determined, but there’s a lot of contingency of epiphemenonal detail between now and then. Modulating the range of populations analyzed often shifted the value of the statistic for a given SNP. Remember, averaging over the aggregate can remove important local information. That being said, the Venn Diagram below shows that there was a disproportionate tendency for the signals detected to be world wide. This indicates that the wheel isn’t reinvented as much as we might think. I wonder if it points to the limitations baked into the human genome in terms of the plasticity and flexibility of all its various pathways. There’s a structural engineer vetoing the elegant fancies of the architect?

The leftmost panel highlights the West Eurasian signals and the middle panel the East Eurasians.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.g004.jpg?resize=598%2C222)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.g004.jpg?resize=598%2C222)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/journal.pgen_.1001375.g004.jpg)

As noted above these sorts of studies have both evolutionary and biomedical relevance. Perhaps the most intriguing result, albeit expected from other areas of research, is the role of [antagonistic pleiotropy](https://en.wikipedia.org/wiki/Antagonistic_pleiotropy_hypothesis) in many diseases. Concretely, **it may be that a change in a particular location may increase reproductive fitness in a novel environment at the cost of later morbidity in life.** The authors suggest that pathogenic resistance and inflammatory response may have the side effect of increasing susceptibility to a range of diseases of old age. Why is this important? I think that the authors are implying in part that a plausible evolutionary mechanism of adaptation should change our prior expectation that a given genome wide association is a false positive. At least I think that. If a SNP was the target of natural selection **and** shows up on GWAS, keep an eye on it! All the better if you have a good functional understanding of what’s going on there.

But more long-term, it might change our perception of the basal risk for classes of morbidity as they vary by population. Human populations have had different evolutionary histories. Their disease risks then might vary a great deal. Between population differences may be a lot less [paradoxical](http://blogs.discovermagazine.com/gnxp/2010/10/the-hispanic-paradox-and-others/) than we think….

**Citation:** Hancock AM, Witonsky DB, Alkorta-Aranburu G, Beall CM, & Gebremedhin A (2011). Adaptations to Climate-Mediated Selective Pressures in Humans PLoS Genetics, : [10.1371/journal.pgen.1001375](10.1371/journal.pgen.1001375)

### Related Posts:

- [Beware of one
  tree!](https://www.gnxp.com/WordPress/2011/04/24/beware-of-one-tree/) - [A mental map of the
  world](https://www.gnxp.com/WordPress/2011/02/24/a-mental-map-of-the-world/) - [Evolutionary pressures, within and
  without](https://www.gnxp.com/WordPress/2011/04/20/evolutionary-pressures-within-and-without/) - [China, climate &
  history](https://www.gnxp.com/WordPress/2008/11/12/china-climate-history/) - [The Court Jester and the averaging
  fallacy](https://www.gnxp.com/WordPress/2011/04/21/the-court-jester-and-the-averaging-fallacy/) - [Are we still
  evolving....](https://www.gnxp.com/WordPress/2011/03/01/are-we-still-evolving/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F04%2F24%2Fthe-evolutionary-effect-of-the-sky-gods%2F&linkname=The%20evolutionary%20effect%20of%20the%20sky%20gods "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F04%2F24%2Fthe-evolutionary-effect-of-the-sky-gods%2F&linkname=The%20evolutionary%20effect%20of%20the%20sky%20gods "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F04%2F24%2Fthe-evolutionary-effect-of-the-sky-gods%2F&linkname=The%20evolutionary%20effect%20of%20the%20sky%20gods "Email")[](https://www.addtoany.com/share)
