+++
title = "Europeans as Middle"
full_title = "Europeans as Middle Eastern farmers"
date = "2011-04-19"
upstream_url = "https://www.gnxp.com/WordPress/2011/04/19/modern-europeans-modeled-as-middle-eastern-farmers/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/04/19/modern-europeans-modeled-as-middle-eastern-farmers/).

Europeans as Middle Eastern farmers

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth.jpg?resize=600%2C405)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth.jpg?resize=600%2C405)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth.jpg)

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)***The Pith:** Over the past 10,000 years a small coterie of farming populations expanded rapidly and replaced hunter-gatherer groups which were once dominant across the landscape. So, the vast majority of the ancestry of modern Europeans can be traced back to farming cultures of the eastern Mediterranean which swept over the west of Eurasia between 10 and 5 thousand years before the before.*

[Dienekes Pontikos](https://dienekes.blogspot.com/2011/04/global-demographic-expansions-followed.html) points me to a new paper in *PNAS* which uses a [coalescent model](https://en.wikipedia.org/wiki/Coalescent_theory) of 400+ [mitochondrial DNA](https://en.wikipedia.org/wiki/Mitochondrial_DNA) lineages to infer the pattern of expansions of populations over the past \~40,000 years. Remember that mtDNA is passed just through the maternal lineage. That means it is not subject to the confounding dynamic of [recombination](https://en.wikipedia.org/wiki/Recombination_(biology)), allowing for easier modeling as a phylogenetic tree. Unlike the autosomal genome there’s no [reticulation](http://www.thefreedictionary.com/reticulation). Additionally, mtDNA tends to be highly mutable, and many regions have been presumed to be selectively neutral. So they are the perfect [molecular clock](https://en.wikipedia.org/wiki/Molecular_clock). There straightforward drawback is that the history of one’s foremothers may not be a good representative of the history of one’s total lineage. Additionally the haploid nature of mtDNA means that genetic drift is far more powerful in buffeting gene frequencies and introduced stochastic fluctuations, which eventually obscure past mutational signals through myriad mutations. Finally, there are serious concerns as to the neutrality of mtDNA…though the authors claim to address that in the methods. I should also add that it also happens to be the case that there is less controversy and more surety as to the calibration of mutational rates of mtDNA than the Y chromosomal lineages of males. Their good for determining temporal patterns of demographic change, and not just tree structures.

Here’s the abstract, [Rapid, global demographic expansions after the origins of agriculture](http://www.pnas.org/content/108/15/6044.short):

> The invention of agriculture is widely assumed to have driven recent > human population growth. However, direct genetic evidence for > population growth after independent agricultural origins has been > elusive. We estimated population sizes through time from a set of > globally distributed whole mitochondrial genomes, after separating > lineages associated with agricultural populations from those > associated with hunter-gatherers. The coalescent-based analysis > revealed strong evidence for distinct demographic expansions in > Europe, southeastern Asia, and sub-Saharan Africa within the past > 10,000 y. **Estimates of the timing of population growth based on > genetic data correspond neatly to dates for the initial origins of > agriculture derived from archaeological evidence.** Comparisons of rates of population growth through time reveal that the invention of agriculture facilitated a fivefold increase in population growth relative to more ancient expansions of hunter-gatherers.

As Dienekes notes **until recently the orthodoxy was that the genetic variation of modern populations was well explained by the genetic variation of Paleolithic groups after the Last Glacial Maximum \~20,000 years B.P.** In this line of thought agriculture spread often by cultural diffusion, and the first local adopters in a region would then enter into a phase of demographic expansion. Bryan Sykes’ [Seven Daughters of Eve](https://www.amazon.com/exec/obidos/ASIN/0393323145/geneexpressio-20) and Stephen Oppenheimer’s [The Real Eve](https://www.amazon.com/exec/obidos/ASIN/0786713348/geneexpressio-20) are expositions of this point of view, which really was the historical genetic mainstream. This also dovetailed with the anthropological bias of “pots-not-people,” whereby cultural forms moved through transmission and not migration. There were some dissenters, such as [Peter Bellwood](https://www.amazon.com/exec/obidos/ASIN/0631205667/geneexpressio-20), but by and large the genetic evidence at least was robust enough that they could be dismissed.

So what happened? Several things. First, the sample sets of mtDNA and Y chromosomes kept getting larger. There was deeper sequencing of informative regions. Thick SNP-chip autosomal studies came to the fore, with different conclusions. Finally, ancient DNA extraction allowed scientists to compare the *real* lineages of hunter-gatherers in ancient Europe vs. what they had presumed were hunter-gatherer descendant lines in modern Europeans. The strong disjunction often found was indicative of a major failing in the prior assumptions of the theorists of the early 2000s: **that they could infer confidently past events from the palimpsest of modern genetic variation.** They couldn’t. We know that because they seem to have been wrong.

Let’s give India as an example of “what went wrong.” Here’s a paper from 2005, [Most of the extant mtDNA boundaries in South and Southwest Asia were likely shaped during the initial settlement of Eurasia by anatomically modern humans](http://www.biomedcentral.com/1471-2156/5/26):

> Since the initial peopling of South and West Asia by anatomically > modern humans, when this region may well have provided the initial > settlers who colonized much of the rest of Eurasia, the gene flow in > and out of India of the maternally transmitted mtDNA has been > surprisingly limited. Specifically, **our analysis of the mtDNA > haplogroups, which are shared between Indian and Iranian populations > and exhibit coalescence ages corresponding to around the early Upper > Paleolithic**, indicates that they are present in India largely as Indian-specific sub-lineages. In contrast, other ancient Indian-specific variants of M and R are very rare outside the sub-continent.

The [Upper Paleolithic](https://en.wikipedia.org/wiki/Upper_Paleolithic) is pre-Holocene. I generally accepted this, *until* the the studies came out from the SNP-chips which had hundreds of thousands of autosomal markers. To be short about it **Indians just seemed too close to West Eurasians if the mtDNA results were correct, *and*, representative.** In fact, if [Reconstructing Indian History](http://www.nature.com/nature/journal/v461/n7263/abs/nature08365.html) is correct, about half the South Asian genome in aggregate is *very* close to that of West Eurasians, to the point where it seems likely to have a common ancestry in the Holocene. The mistaken inference from mtDNA may be due in part to sex-biased gene flow. That is, the South Asian exogenous genome was strongly biased toward male migration, while the deep time mtDNA substrate has tended to persist underneath all these successive layers.

Moving to the paper in question, they use a “Bayesian skyline” method to reconstruct past demographic history. Specifically, the history of the direct maternal lineage. We wouldn’t really pay attention if they didn’t have interesting results. And they do indeed.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth2.jpg?resize=600%2C256)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth2.jpg?resize=600%2C256)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth2.jpg)

The table is rather straightforward. They partitioned the samples they had into putative hunter-gatherer and Neolithic lineages. Notice the difference. For some of these cases **we have very robust non-genetic evidence of expansion.** This is true especially for the African and Southeast Asian Holocene cases. Their methods here predict exactly what we already know. So the key value add is that the methods are predicting something which is more in dispute: **the demographic history of contemporary European mtDNA lineages.** The concordance of the archaeological evidence of the Neolithic transition in Europe and the inferred demographic expansion of European Neolithic mtDNA lineages is striking.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth3.png?resize=300%2C243)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth3.png?resize=300%2C243)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth3.png)The plot to the left is the curve of demographic expansion predicted from their method for Neolithic and Paleolithic lineages in Europe. The y-axis is log-scaled, so it naturally understates the explosive growth of Neolithic lineages. It comports well with what we know of how agricultural societies tend to expand and stabilize over time. During a phase of “land surplus” they enter into rapid demographic expansion, forcing the frontier of settlement out. Once the land is “filled up” we enter into the classic Malthusian “stationary state,” where the grinding misery of the peasantry becomes the lot of most. In contrast hunter-gatherer lineages didn’t experience such an explosive shift. Though pre-modern hunter-gatherer landscapes were more diversified than what we experience today, because they had access to the rich “bottom lands” and seashores now monopolized by agriculturalists, the carrying capacity of the land was generally lower for their lifestyle, and waxed and waned more gradually with shifts in ecology.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth4.jpg?resize=350%2C503)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth4.jpg?resize=350%2C503)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/04/growth4.jpg)The authors also did some neat geo-visualization, if I do say so (and I’m jealous!). The two panels illustrate the spread of agriculture as inferred from archaeology, and the rate of population growth calculated from the joint information of the time of onset of a farming lifestyle in a region and the point on the “growth curve” for the Middle Eastern lineages at that time. So above you see the spread of agriculture from the eastern Mediterranean from 8000 BC to 2500 BC. *Then*, you see a geographical illustration of the S-shaped growth curve of the farmers. Their initial colonies experienced modest growth, but there was a transition zone in the middle of rapid expansion. Why? Perhaps there was a necessary critical mass, before the superiority of numbers began to wear down the hunter-gatherers. But this itself was a transient, as the farmer societies ran up against the limits of ecology along the northern European plain (or, perhaps just as likely, they encountered dense hunter-gatherer societies which were able to temporarily withstand their aggressive expansion on the European maritime fringe). I suspect that the models are more complex than a one-two punch, in either time or space. There were likely **several pulses and distinct streams coming out of the Middle East which populated Europe.**

They conclude that “Mesolithic ancestry makes up only a fraction of contemporary European genomes. U5a, U5b1, V, and 3H combined account for ≈15% of western Europeans mtDNA haplogroups.” Note that U5a and U5b are modal among the [Finnic peoples of Europe](https://en.wikipedia.org/wiki/Haplogroup_U_(mtDNA)#Haplogroup_U5). [V](https://en.wikipedia.org/wiki/Haplogroup_V_(mtDNA)) seems widely distributed, and modal in northern Scandinavia and the western Mediterranean. I can’t seem to find easy information on 3H.

From the supplements here are the European haplgroups they selected:

> We chose haplogroups associated with an origin in Near Eastern > populations during the Holocene: T1, T2, J1a, K2a, and H4a. These > haplogroups (T1, T2, J1a, and K) all appear to have Near Eastern > founders that migrated to Europe after the Younger Dryas (2). After > inspecting the haplogroup K network in Behar et al. (4), we chose the > subgroup K2a, which appears to be present in the Near East (including > non-Ashkenazi Jews) and European populations (but not North Africa). > Haplogroup H4a is thought to have expanded throughout Europe during > the Neolithic (5). However, the location of its origin is still not > certain (6). Removing H4a from the Skyline analysis did not > substantively change the timing of Holocene period expansion (results not shown). European haplogroups U5, V, and 3H are associated with an indigenous origin in Europe (2). Haplogroups U5a, U5b1, V, and 3H have all been attributed a TMRCA during the Last Glacial Period (2, 7–9)

Readers more well versed in the literature on mtDNA haplogroups can pick these details apart.

Where does this leave us? **If this and other recent papers are correct. then the expansion of farming to Europe from the Middle East resembles the settlement of the New World far more than we may have thought!** In some regions there was likely near total replacement of the substrate, perhaps like the United States. In others there was modest uptake of the indigenous substrate, as is the case in Argentina. Finally, there were regions where the indigenous hunter-gatherer substrate may have persisted to a far greater extent. I think this may be the case mostly in Baltic Europe, which combined both the possibility of relatively high hunter-gatherer carrying capacities because of marine resources and a climatic regime rather unsuitable to the initial Middle Eastern crops.

**Citation:** Gignoux CR, Henn BM, & Mountain JL (2011). Rapid, global demographic expansions after the origins of agriculture. Proceedings of the National Academy of Sciences of the United States of America, 108 (15), 6044-9 PMID: [21444824](http://www.ncbi.nlm.nih.gov/pubmed/21444824)

### Related Posts:

- [The introduction of farming & new genes to
  Europe](https://www.gnxp.com/WordPress/2009/09/03/the-introduction-of-farming-new-genes-to-europe/) - [Phylogeography of deep European genetic
  history](https://www.gnxp.com/WordPress/2010/04/22/phylogeography-of-deep-european-genetic-history/) - [They have been practicing for 40,000
  years....](https://www.gnxp.com/WordPress/2006/04/06/they-have-been-practicing-for-40-000-years/) - [Tall, to short, to tall
  (again)](https://www.gnxp.com/WordPress/2006/09/29/tall-to-short-to-tall-again/) - [The rise of the farming
  lineages](https://www.gnxp.com/WordPress/2012/10/05/the-rise-of-the-farming-lineages/) - [Why I still lean toward a Sub-Saharan African origin
  for…](https://www.gnxp.com/WordPress/2016/02/18/why-i-still-lean-toward-a-sub-saharan-african-origin-for-modern-humanity/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F04%2F19%2Fmodern-europeans-modeled-as-middle-eastern-farmers%2F&linkname=Europeans%20as%20Middle%20Eastern%20farmers "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F04%2F19%2Fmodern-europeans-modeled-as-middle-eastern-farmers%2F&linkname=Europeans%20as%20Middle%20Eastern%20farmers "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F04%2F19%2Fmodern-europeans-modeled-as-middle-eastern-farmers%2F&linkname=Europeans%20as%20Middle%20Eastern%20farmers "Email")[](https://www.addtoany.com/share)
