+++
title = "Why not release data for"
full_title = "Why not release data for phylogenetic papers?"
date = "2013-03-01"
upstream_url = "https://www.gnxp.com/WordPress/2013/03/01/why-not-release-data-for-phylogenetic-papers/"

+++
Source: [here](https://www.gnxp.com/WordPress/2013/03/01/why-not-release-data-for-phylogenetic-papers/).

Why not release data for phylogenetic papers?

[Last month](http://blogs.discovermagazine.com/gnxp/2013/01/the-voyage-of-krishna-crusoe/#.UTEMkIc-u0w) I noted that a paper on speculative inferences as to the phylogenetic origins of Australian Aborigines was hampered in its force of conclusions by the fact that the authors didn’t release the data to the public (more accurately, peers). There are likely political reasons for this in regards to Australian Aborigine data sets, so I don’t begrudge them this (Well, at least too much. I’d probably accept the result more myself if I could test drive the data set, but I doubt they could control the fact that the data had to be private). **This is why when a new paper on a novel phylogenetic inference comes out I immediately control-f to see if they released their data.** In regards to genome-wide association studies on medical population panels I can somewhat understand the need for closed data (even though anonymization obviates much of this), but I don’t see this rationale as relevant at all for phylogenetic data (if concerned one can remove particular functional SNPs).

  
Yesterday I noticed *PLoS Genetics* published a paper on the genomics of Middle Eastern populations, [Genome-Wide Diversity in the Levant Reveals Recent Structuring by Culture](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003316). The results were moderately interesting (I’ll review the paper in detail later), but bravo to the authors for [putting their new data set online](http://bhusers.upf.edu/~mhaber/PLOS/). The reason is simple: reading the paper I wanted to see an explicit phylogenetic tree/graph to go along with their figures (e.g., with TreeMix). Now that I have their data I can do that tonight, time permitting.

One major aspect of science is reproducibility. Because of capital outlays this is not always viable, and often occurs in a haphazard fashion. But with phylogenetics done on a computer this is less of an issue. I have a desktop at home devoted 99% to running data sets, in part for my own interest, and in part because I want to check the robustness of some of the inferences I see in papers like the ones above.

### Related Posts:

- [Do you want your genotype in a public data
  set?](https://www.gnxp.com/WordPress/2013/01/16/do-you-want-your-genotype-in-a-public-data-set/) - [Lots and lots of genotype
  data!](https://www.gnxp.com/WordPress/2013/01/09/lots-and-lots-of-genotype-data/) - [1000 Genomes data, 290,000
  SNPs](https://www.gnxp.com/WordPress/2015/09/30/1000-genomes-data-290000-snps/) - [The value of "open
  genomics"](https://www.gnxp.com/WordPress/2011/04/16/the-value-of-open-genomics/) - [Survey on personal
  genomics](https://www.gnxp.com/WordPress/2011/08/31/survey-on-personal-genomics/) - [On getting gated
  papers](https://www.gnxp.com/WordPress/2012/12/03/on-getting-gated-papers/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F03%2F01%2Fwhy-not-release-data-for-phylogenetic-papers%2F&linkname=Why%20not%20release%20data%20for%20phylogenetic%20papers%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F03%2F01%2Fwhy-not-release-data-for-phylogenetic-papers%2F&linkname=Why%20not%20release%20data%20for%20phylogenetic%20papers%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F03%2F01%2Fwhy-not-release-data-for-phylogenetic-papers%2F&linkname=Why%20not%20release%20data%20for%20phylogenetic%20papers%3F "Email")[](https://www.addtoany.com/share)
