+++
title = "Input determining output"
full_title = "Input determining output in ADMIXTURE"
date = "2011-03-21"
upstream_url = "https://www.gnxp.com/WordPress/2011/03/21/input-determining-output-in-admixture/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/03/21/input-determining-output-in-admixture/).

Input determining output in ADMIXTURE

One reason I [posted about how to run ADMIXTURE](http://blogs.discovermagazine.com/gnxp/2011/03/analyzing-ancestry-with-admixture-step-by-step/) was so that the more readers themselves could become familiar with the biases of the program. That way they would get cautious about over-reading one particular set of results (the same goes for using PCA to visualize genetic relationships). [Dienekes](https://dienekes.blogspot.com/2011/03/note-of-caution-on-admixture-estimates.html) elaborates in detail on this point, [A note of caution on admixture estimates](https://dienekes.blogspot.com/2011/03/note-of-caution-on-admixture-estimates.html):

> Much more can be said on this issue, but let’s summarize a couple of lessons: >
> – The full extent of an admixture cline can be captured only if unadmixed populations on either side of the cline exist. Use as many populations as possible to capture the full extent of an admixture cline. >
> – Use of an admixed population in lieu of an unadmixed native one inflates the inferred native component. Use native populations if possible instead of admixed ones . >
> – Even in the absence of unadmixed native populations, it is sometimes possible to reconstruct the admixture proportions as per Reich et al. (2009). >
> Capturing the complexities of human prehistory from modern populations > is tricky. Nonetheless, with increased coverage of human genetic > diversity **(there are already \~9k individuals in my database)**, new analytical techniques, and, hopefully some archaeogenetic calibration genotypic, we are bound to learn much more about the distant human past in the not-so distant future.

9,000 individuals in one random guy’s database. Wow. I know Zack has more than 6,000 now. I’ve got around 4,000 myself without much expenditure of effort.

I think it is important to be very cautious about looking at ADMIXTURE results alone and in isolation. The range of possible models one could generate from a set of ADMIXTURE bar plots is enormous. A synthesis of ethnographic, historical, and paleoanthropological information, is necessary to really squeeze further analytic juice out of these powerful new tools.

### Related Posts:

- [History and Geography of Genes on a
  Desktop](https://www.gnxp.com/WordPress/2010/10/25/history-and-geography-of-genes-on-a-desktop/) - [Basque genetic distinctiveness
  (again)](https://www.gnxp.com/WordPress/2011/12/28/basque/) - [Seeing Ötzi through our
  eyes](https://www.gnxp.com/WordPress/2012/03/08/seeing-otzi-through-our-eyes/) - [Ancestry analysis
  quickstart](https://www.gnxp.com/WordPress/2020/07/17/ancestry-analysis-quickstart/) - [A note on open
  genomics](https://www.gnxp.com/WordPress/2012/08/28/a-note-on-open-genomics/) - [Eurasia, ADMIXTURE supervised &
  unsupervised](https://www.gnxp.com/WordPress/2011/03/16/eurasia-admixture-supervised-unpservised/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F03%2F21%2Finput-determining-output-in-admixture%2F&linkname=Input%20determining%20output%20in%20ADMIXTURE "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F03%2F21%2Finput-determining-output-in-admixture%2F&linkname=Input%20determining%20output%20in%20ADMIXTURE "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F03%2F21%2Finput-determining-output-in-admixture%2F&linkname=Input%20determining%20output%20in%20ADMIXTURE "Email")[](https://www.addtoany.com/share)
