+++
title = "French Canadians are"
full_title = "French Canadians are genetically special"
date = "2013-09-30"
upstream_url = "https://www.gnxp.com/WordPress/2013/09/30/exome/"

+++
Source: [here](https://www.gnxp.com/WordPress/2013/09/30/exome/).

French Canadians are genetically special (not that that's a good thing)

![Jack Kerouac, credit: [Tom Palumbo](https://en.wikipedia.org/wiki/File:Kerouac_by_Palumbo.jpg)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2013/09/220px-Kerouac_by_Palumbo.jpg?resize=220%2C220 "220px-Kerouac_by_Palumbo")

***The Pith**: Higher Mendelian disease rates among French Canadians may be due to their demographic history.*

As I have [noted before](http://blogs.discovermagazine.com/gnxp/2013/05/vulcans-through-the-eye-of-the-bottleneck/#.UkkKF4ZwqSo), demographic bottlenecks with extremely strong effects on the character of [population genetic variation](https://en.wikipedia.org/wiki/Zygosity#Homozygous) need to be very radical in their nature to be of any significance. The population pinhole has to be on the order of hundreds, rather than thousands, of individuals. **But that does not preclude more modest bottlenecks generating subtle shifts in the genetic [site frequency spectrum](http://www.stats.ox.ac.uk/~etheridg/orsay/notes3.pdf)**. Strong bottlenecks may be needed to drive wholesale extinction of once common alleles (or the fixation of those at moderate frequencies), but mild bottlenecks may nevertheless perturb the allele frequency distribution. In particular, the number of alleles which are present at very low frequencies can be strongly impacted by demographic variation and natural selection. This is the logical rationale which serves as the basis for nucleotide sequence based tests for detecting natural selection, such as [Tajima’s D](https://en.wikipedia.org/wiki/Tajima's_D). An excess of low frequency variants suggest a bottleneck and subsequent population expansion, or positive and/or purifying selection. In contrast, balanced polymorphism frequencies point to a shrinking population or [balancing selection](https://en.wikipedia.org/wiki/Balancing_selection).

![Citation: Casals F, Hodgkinson A, Hussin J, Idaghdour Y, Bruat V, et al. (2013) Whole-Exome Sequencing Reveals a Rapid Change in the Frequency of Rare Functional Variants in a Founding Population of Humans. PLoS Genet 9(9): e1003815. doi:10.1371/journal.pgen.1003815](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2013/09/journal.pgen_.1003815.g0011.png?resize=300%2C308 "journal.pgen.1003815.g001")

These basic ideas have been around for decades, **but it is with powerful genomic technologies that they are truly giving us actionable insights.** A new paper in *PLOS Genetics* lays it out simply enough, [Whole-Exome Sequencing Reveals a Rapid Change in the Frequency of Rare Functional Variants in a Founding Population of Humans](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003815). To the left you see a site frequency distribution for French and French Canadian populations. What is clear is that for the derived allele (i.e., mutations from the ancestral state) distribution in [exonic](https://en.wikipedia.org/wiki/Exon) regions of the genome French Canadians are much more skewed toward the low frequency portion of the spectrum than French proper. This skew is more noticeable for deleterious mutations, such as [nonsense](https://en.wikipedia.org/wiki/Nonsense_mutation) and [missense mutations](https://en.wikipedia.org/wiki/Missense_mutation) (nonsense mutations usually produce nonfunctional protein, while missense mutations may alter the nature of the protein in some specific detail through amino acid substitution).

The focus here is on exons, \~1% of the genome, because these are the regions that are translated into the final protein product, **and the authors seem particularly interested in the functional consequences of the site frequency spectrum of the French Canadians**. This makes sense, because the French Canadian population has long been known to have a somewhat high burden of recessive diseases. Why? As noted in the paper the French Canadian ancestry [derives overwhelmingly from a founding population](http://blogs.discovermagazine.com/gnxp/2011/01/the-genomic-heritage-of-french-canadians/#.UkknJYZwqSo) of less than 10,000. Not only that, but this expanding population exhibits geographic substructure, with demographic expansion being particularly powerful along the edge of the pale of Quebecois settlement. This results in increased genetic drift on the edge, as a smaller portion of the population contributes to descendants pushing over the frontier. The key is to note how striking it is that a few hundred years of demographic separation can result in the emergence of ‘private alleles.’ To a great extent this is intuitively obvious, as private alleles emerge *de novo* in families, and many French Canadian families have had many of generations separated from the ancestral homeland to accumulate distinctive markers specific to their lineage.

Over the long term many “x” whole-genome coverage (so on average the same base can be found in 10 or 20 or 30 reads to reduce possible false positives) is going to be ubiquitous, **and we’ll get a sense of the distribution of genetic load within and across families**. One major demographic-historical dynamic highlighted in this paper is that serial bottleneck events in human history (e.g., the “Out of Africa” migration) may endow different populations with different site frequency spectra, and so imply diverse genetic disease loads. Seeing as how genomic work tends to be focused on populations of European descent we haven’t truly explored these sorts of inter-population possibilities in great depth, but they’re in the offing. I suspect for example that Indiansubcastes will tend to have many private alleles due to bottlenecks and recent expansions. And, in the short-term this may also redound to the benefit of those who argue for the benefits of genetic diversity through random mating across populations.

**Citation:** Casals F, Hodgkinson A, Hussin J, Idaghdour Y, Bruat V, et al. (2013) Whole-Exome Sequencing Reveals a Rapid Change in the Frequency of Rare Functional Variants in a Founding Population of Humans. PLoS Genet 9(9): e1003815. doi:10.1371/journal.pgen.1003815

### Related Posts:

- [Out of Africa: by the
  skulls?](https://www.gnxp.com/WordPress/2007/07/20/out-of-africa-by-the-skulls/) - [Modeling human demographic
  history](https://www.gnxp.com/WordPress/2008/06/03/modeling-human-demographic-history/) - [Genetic distinctiveness vs. genetic
  diversity](https://www.gnxp.com/WordPress/2011/11/28/genetic-distinctiveness-vs-genetic-diversity/) - [The last glacial maximum bottlenecks and human
  phylogeny](https://www.gnxp.com/WordPress/2022/06/19/the-last-glacial-maximum-bottlenecks-and-human-phylogeny/) - [When genetic drift = more
  adaptation?](https://www.gnxp.com/WordPress/2008/05/15/when-genetic-drift-more-adaptation/) - [When less is more: founder
  flush](https://www.gnxp.com/WordPress/2006/07/02/when-less-is-more-founder-flush/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F09%2F30%2Fexome%2F&linkname=French%20Canadians%20are%20genetically%20special%20%28not%20that%20that%27s%20a%20good%20thing%29 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F09%2F30%2Fexome%2F&linkname=French%20Canadians%20are%20genetically%20special%20%28not%20that%20that%27s%20a%20good%20thing%29 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F09%2F30%2Fexome%2F&linkname=French%20Canadians%20are%20genetically%20special%20%28not%20that%20that%27s%20a%20good%20thing%29 "Email")[](https://www.addtoany.com/share)
