+++
title = "What's your s?"
full_title = "What's your s?"
date = "2005-05-01"
upstream_url = "https://www.gnxp.com/WordPress/2005/05/01/what-s-your-s/"

+++
Source: [here](https://www.gnxp.com/WordPress/2005/05/01/what-s-your-s/).

What's your s?

A few weeks ago John Hawks [posted](http://johnhawks.net/weblog/reviews/genetics/genetics_mre_aapa_2005.html) something important, he suggested that the basic paradigm which population geneticists arguing for Out-of-Africa vs. !Out-of-Africa are working with is faulty (Henry Harpending has implied the same to me in emails). As Greg Cochran [specified](http://js-kit.com/api/static/pop_comments?ref=http://gnxp.com&path=/3935/#285940), the cruxes are the issues of selection and interbreeding: the models that are used to infer homonid admixture often assume that the populations never exchanged genes except for recent and singular hybrdization events/periods. Additionally there is often an assumption of neutrality as regards the genetic locus in question (on the well known loci like mtDNA).1 In other words, it seems some geneticists who have spent years breeding rat and fly lineages in a controlled environment forget that humans are generally far more promiscuous and don’t have grad students and lab techs monitoring clandestine inter-lineage trysts.

How plausible is it that archaic populations did not interbreed so that alleles were never exchanged? Since we don’t have archaic populations around anymore (or so we think….), let’s look at possible H. sapiens analogs. Consider lactose tolerance, there is [strong evidence](http://www.springerlink.com/app/home/contribution.asp?wasp=6e57e19b6688406ba1d3353be079590c&referrer=parent&backto=issue,15,19;journal,1,111;linkingpublicationresults,1:100421,1) that the allele that confers adult lactose catabolism in many populations has a common origin in Eurasia. In short, the lactose tolerance of populations traditionally classified as “Caucasoid” derives from a common mutational event \~10,000 years ago. On the other hand, the lactose tolerance of various African populations seems to derive from alternative alleles (there is also [evidence](https://dienekes.blogspot.com/2005/04/abstracts-from-74th-meeting-of-aapa.html) that dipigmentation in Europeans and East Asians is controlled by different alleles, control-f “pigment” or “Horton”). But looking specifically at the distribution of lactose tolerance in South Asia, in northern India about 70% of the population is lactose tolerant, in southern India about 30% is ([cite](https://www.gnxp.com/MT2/archives/001681.html)). Doing a little googling for this sort of data I stumbled on to some “Aryan” sites that suggested that this was evidence that northern Indians are more like Europeans than they are like southern Indians. That is correct, on that particular locus! My own survey of the literature suggests (no surprise) that South Asians are a genetically diverse group, and depending on the locus you analyze you will come to different conclusions, though I personally believe that the balance of the evidence (evaluated over all “informative” loci) suggests that north and south Indians have far more in common genetically than they do with any other populations outside of South Asia (excepting a few groups like the Parsis). The difference between north and south India as far as lactose tolerance goes might simply be due to the feasibility of pastoralism and the density of cattle, so the selection pressures were far greater in the north than in the south.2

We obviously know a lot more about the details of population substructure among modern populations than we do about putative “archaic” groups. Unfortunately paleontology can only tell us so much (and I get the impression that a “splitting” mentality is dominant in some sectors of paleoanthropology), and a lot of what passes for background assumptions in many hypotheses are built on slim conjecture. Additionally, particular areas like Eastern Asia have a dearth of homonid remains over the past few hundred thousand years until the emergence of moderns. John points out that the reason we might be finding so few “archaic” alleles where recombination seems to not have occurred is that it may be that only those alleles with very narrowly constrained local adaptations might have remained cordoned off from the rest of the humanity and so evince an ancient [coalescense](http://www.pandasthumb.org/pt-archives/000234.html).

Which brings me to a second point-[David Boxenhorn expects](http://js-kit.com/api/static/pop_comments?ref=http://gnxp.com&path=/3935/#285939) “local populations to be better adapted to local climatic conditions, making old-new hybrids which combine the best of both the most fit.” Is this a valid assertion? Depends. Certainly I suspect that the lactose tolerance allele spread throughout Western Eurasia through such a process of deme-to-deme genetic exchange, or at least a mild form of demic diffusion where hybridization played a large role. We have talked before about the fact that sharp phenotype differences might persist even where neutral alleles (ancestrally informative) show clinal gradients due to selection pressures (Henry Harpending’s canonical examples are the San of the Kalahari and the Rh- allele among the Basques). But obviously this doesn’t always happen: when groups face each other as “collectives” there can be simple replacement of one population for another. In other words, the fitness of allele X vs. allele Y on locus A is swamped out by the reality that people carrying allele X are simply being marginalized/killed off by people carrying allele Y so that there isn’t a chance for the “fit” allele to remain present at a high frequency in the population (consider the light-skinned Australians who live in Queensland). Now, if there are a small group of survivors who get absorbed into the Y carrying population which bring in allele X, if the selection pressure is environmental the frequency of X should increase over time. If X was favored only by social/sexual selection, then it might actually be selected against!

All in all it is a big mess. I suspect that, as I have noted before, that one reason Out-of-Africa is so popular is that it avoids such messes. It also preserves our “common sense” need to square genetic phylogenies with ancestral phylogenies as well as phenotypic particularities (that is, one “human” group emerged with a particular unique suite of genes and a distinctive adaptational skillset in one location at one time). It is sometimes said that genuine oral memories don’t go back much further than a few centuries, and so it certainly seems that for most people you would observe a concordance between ancestry and appearance. Selection usually takes at least a few generations to work its magic even for traits where there is a lot of variation in fitness within the population.3 Also, many situations where selective sweeps could occur, perhaps in terms of resistance to plague, are not always susceptible to visual inspection.

But the problem isn’t just with the lay audience for popular books: someone must be producing the books. Spencer Wells is no idiot (no, really, Harvard accepted him!). Neither are a host of other scientists and scholars who promote Out-of-Africa. The scientists who pushed for an early separation between the ape and human lineage were also not idiots, and they kept believing their fossils as opposed to the biochemical evidence which suggested a recent divergence until the fossils falsified them. So I have to bring up the “p” word, paradigm. Kuhn’s model is abused and distorted to the point where science becomes just “another superstition.” But, in reality, it’s like democracy, a shitty system that happens to be better than any other out there (at least at getting an empirically based handle on the world out there). Over the past 10 years a swarm of studies have emerged that examine the histories of genes (or in particular, the relationship between various alleles). Most of the time scientists have focused on the NRY and mtDNA because the assumption of neutrality and the lack of recombination made their models far simpler, and by the time a pipeline of papers was com  
pressed into a pithy popular press articles the history of genes became the history of our species. On the foundational level of course scientists understand that human beings are simply a collection of genes, and there is no necessary reason that all these genes have tracked the exact same paths through the various individuals and subpopulations that have comprised the homonid lineage. But it would certainly be easier if they did! Over 20 years ago Richard Dawkins wrote [The Selfish Gene](https://www.amazon.com/exec/obidos/ASIN/0192860925/geneexpressio-20/102-3972032-6983341?%5Fencoding=UTF8&camp=1789&link%5Fcode=xm2) to promote the viewpoint that the body, the individual, was just a vessel for the propogation of genes, at least from the perspective of evolution. Ten years later mitochondrial Eve burst upon the scene, and the history of the small bits of rapidly mutating mitochondrial DNA was conflated with the history of the human species. With the glut of data produced by improved genomic sequencing techniques the blind spots in the empirical landscape are being illuminated. The familiar singular vistas offered by mitochondrial Eve are being overshadowed by bizarre counterintuitive conformations girdling the whole horizon. The roads in gene land might be narrow but the scenery escapes a simple adjective, and it is simple adjectives that sell books. [John Brockman](http://www.edge.org/3rd_culture/bios/brockman.html), your scientists need you!

Addendum: John states somewhat cryptically: “That would leave natural selection as apparently the only explanation, although what pattern of selection would create the excess of ancestral alleles in Africa is up for grabs. I have an idea, but I’m not sharing it just yet.” Well, we know there have been [selective sweeps in Eurasia](https://www.gnxp.com/MT2/archives/002266.html) that did not impact Africa to the same extent. Another factor, which I do find tenuous from the angle of functional relevance is that Africa seems to have a high pathogen load (this has [resulted in social adaptations](https://www.gnxp.com/MT2/archives/002628.html)) that might enforce some sort of frequency dependent selection on some loci (I don’t know if, for example, the mtDNA has anything to do with pathogens in terms of fitness). If I recall correctly the MHC loci, which are directly relevant for immune response and coordination, are more diverse and varied in Africa. Also, as noted above in the context of lactose tolerance it seems that there were reasonable (though obviously not perfect) genetic barriers across the Sahara for much of human prehistory (the Eurasian lactose tolerance mutation likely wasn’t in the genetic background in any of the numerous populations of the Sahel). If one assumes that the periodic expansions of the Eurasian ice sheet pushed back homonid populations so that there were only isolated remnants, periodic bottleneck effects (which would drive down the [long term effective population](https://www.gnxp.com/MT2/archives/002018.html) a lot) could have homogenized the Eurasian gene pool. In contrast Africa might have been relatively shielded from the worst of the environmental fluctuations and so been subject to fewer bottleneck events which would have driven down genetic diversity. I note that in [The Real Eve](https://www.amazon.com/exec/obidos/ASIN/0786711922/geneexpressio-20/102-3972032-6983341) Stephen Oppenheimer focuses on the Indian subcontinent as the source of all Eurasian lineages of modern humans based on his reading of the mtDNA, and the climatic explanation has the virtue of explaining this as well. Oppenheimer is rather explicit that even the modern human populations in Eurasia “went their separate ways” during the Last Glacial Maximum, never to meet again genetically….

Related: [The origins of phenotypic variation?](https://www.gnxp.com/MT2/archives/003353.html) [“Racial Diversity”](https://www.gnxp.com/MT2/archives/000402.html)

1 – Greg pointed out that chance of fixation for an allele is 2s, where s is the [selection coefficient](http://www.blackwellpublishing.com/ridley/a-z/Selection_coefficient.asp) vs. 1/2Ne, where Ne is the effective breeding population, for a neutral marker (often this is couched in terms of per mutational event, but in the comments Greg stated it as number of copies introduced, same diff from the angle of the population that is getting the novel allele into the system). Also, when Ns, where N is population size and s is selection coefficient, is much less than one (ie; Ns [](http://js-kit.com/api/static/pop_comments?ref=http://gnxp.com&path=/3944?url=http://www.gnxp.com/MT2/archives/003944.html&thetime=%20050105&MT=true)

### Related Posts:

- [Out of Africa model most
  likely](https://www.gnxp.com/WordPress/2007/10/31/out-of-africa-model-most-likely/) - ["Multiregionalism vs. Out of
  Africa"](https://www.gnxp.com/WordPress/2010/04/21/multiregionalism-vs-out-of-africa/) - [Beyond Out-of-Africa &
  Multiregionalism](https://www.gnxp.com/WordPress/2006/04/18/beyond-out-of-africa-multiregionalism/) - [2s-a little interbreeding goes a long
  way](https://www.gnxp.com/WordPress/2006/11/10/2s-a-little-interbreeding-goes-a-long-way/) - [Population structure within
  Africa](https://www.gnxp.com/WordPress/2011/03/16/population-structure-within-africa/) - [Neandertals...the continuing
  saga](https://www.gnxp.com/WordPress/2006/11/16/neandertals-the-continuing-saga/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F05%2F01%2Fwhat-s-your-s%2F&linkname=What%27s%20your%20s%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F05%2F01%2Fwhat-s-your-s%2F&linkname=What%27s%20your%20s%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F05%2F01%2Fwhat-s-your-s%2F&linkname=What%27s%20your%20s%3F "Email")[](https://www.addtoany.com/share)
