+++
title = "More on noncoding RNAs"
full_title = "More on noncoding RNAs"
date = "2006-09-19"
upstream_url = "https://www.gnxp.com/WordPress/2006/09/19/more-on-non-coding-rnas/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/09/19/more-on-non-coding-rnas/).

More on non-coding RNAs

If the 24-hour science news cycle hasn’t knocked it out of the banks yet, recall that there was some hub-bub about a non-coding RNA (ncRNA) being a candidate for the [fastest-evolving](https://www.gnxp.com/blog/2006/08/fastest-evolving-human-gene.php) [human gene](https://www.gnxp.com/blog/2006/08/har1f-tangents.php). There was some discussion about whether ncRNAs might be something to keep an eye on and why RNAs might be particularly good at evolving quickly. [This review from March](http://arjournals.annualreviews.org/doi/abs/10.1146/annurev.neuro.29.051605.112839) had some factoids worth mulling.

There are more ncRNAs than you thought:

- Half of the “[full-length long
  Japan](http://www.nature.com/ng/journal/v36/n1/abs/ng1285.html)”
  library of human cDNA clones appear to be non-coding. Anti-jargon:
  cDNA (complementary DNA) is sequence read off of RNA backwards. This
  group tried to take a very large scale unbiased picture of the RNAs
  floating around in human cells and did bioinformatics to guess whether
  they coded for protein or not. - The [FANTOM 3 consortium](http://fantom.gsc.riken.go.jp/) sez that 62%
  of the mouse genome is transcribed. Half of these transcripts are
  non-coding. - [MicroRNAs](https://www.gnxp.com/blog/2006/06/rnai-fundamentals.php)
  are a subset of ncRNAs that regulate other RNAs. 70% of microRNAs can
  be found in the brain. There’s a big section in the review on
  particular brain-specific miRNAs and potential roles in development. - There are seven brain-specific small nucleolar RNAs (snoRNAs). These
  are RNAs that act as enzymes to chemically modify nucleotides in other
  RNAs. There is a particularly provocative connection between snoRNAs
  and serotonin receptor subtype processing in Prader-Willi Syndrome, a
  form of mental retardation arising from deletion of a chunk of
  chromosome 15. - LINE-1 retrotransposons make up 17% of the human genome. These are the
  so-called “jumping genes”. They code for the proteins to reverse
  transcribe them back into the genome. So it’s relatively easy for them
  to proliferate. Technically they aren’t non-coding, but the RNA is
  used in a nontraditional way in that it serves as a template for
  reading the gene back into the genome at some point. There is some
  evidence for a role for active LINE-1s in neural differentiation.

The authors say ncRNAs might evolve more quickly because they can be duplicated with less probability of harm than coding RNAs. I’m not so sure about their argument. I’ll buy the first point, that ncRNAs are generally smaller and can be duplicated with greater fidelity. They then state that their small size also decreases their likelihood of disrupting existing genes. It seems to me that size wouldn’t matter. If a chunk of DNA gets copied into the middle of some gene, the gene should be disrupted. Maybe the consequences could be worse in the specific case where the duplication occurs between regulatory elements and the transcription start site, so that the gene’s regulation is more disrupted the further the promoter is pushed from the gene, but this seems like it would be uncommon.

I stated a while back that one reason for expecting a lot from RNAs over protein is that RNAs used to do the whole job back in the good ole days of the RNA World. Surprisingly, I was not the first brilliant, original thinker to arrive at this conclusion. In fact, these folks end by referencing Sean Eddy, who wrote a [whole review about the importance of ncRNAs and how the RNA World isn’t over](http://www.nature.com/nrg/journal/v2/n12/full/nrg1201-919a_fs.html). From his review:

> The discovery of RNA catalysis and the “RNA world” hypothesis for the > origin of life provide a seductive explanation for why rRNA and tRNA > are at the core of the translation machinery: perhaps they are the > frozen evolutionary relic of the invention of the ribosome by an > RNA-based ‘riboorganism’. Other known ncRNAs have also been proposed > to be ancient relics of the last riboorganisms123-125. The romantic > idea of uncovering molecular fossils of a lost RNA world has motivated > searches for new ncRNAs. However, as these searches start to succeed, > more and more ncRNAs are being found to have apparently well-adapted, > specialized biological roles. The idea that ncRNAs are a small and > ragged band of relics looks increasingly untenable. The tiny stRNAs and miRNAs, for example, seem to be highly adapted for a world in which RNAi processing and developmentally regulated mRNA targets exist. >
> Therefore, consider an alternative idea – the “modern RNA world”. Many > of the ncRNAs we see in fact have roles in which RNA is a more optimal > material than protein. Non-coding RNAs are often (though not always) > found to have roles that involve sequence- specific recognition of > another nucleic acid. (The choice of examples in Figs 1, 2 and 4 is > deliberate, showing how snoRNAs, miRNAs and E. coli riboregulatory > RNAs all function by sequence-specific base complementarity.) RNA, by > its very nature, is an ideal material for this role. Base > complementarity allows a very small RNA to be exquisitely sequence > specific. Evolution of a small, specific complementary RNA can be achieved in a single step, just by a partial duplication of a fragment of the target gene into an appropriate context for expression of the new ncRNA.

### Related Posts:

- [HAR1F
  tangents](https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/) - [More genes
  please!](https://www.gnxp.com/WordPress/2007/11/21/more-genes-please/) - [A Nobel Prize for gene
  expression](https://www.gnxp.com/WordPress/2006/10/02/a-nobel-prize-for-gene-expression/) - [Selection in humans, a
  follow-up](https://www.gnxp.com/WordPress/2006/09/21/selection-in-humans-a-follow-up/) - [RNA
  activation](https://www.gnxp.com/WordPress/2006/11/02/rna-activation/) - [Evolutionary forces in the (human)
  genome](https://www.gnxp.com/WordPress/2006/10/13/evolutionary-forces-in-the-human-genome/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F19%2Fmore-on-non-coding-rnas%2F&linkname=More%20on%20non-coding%20RNAs "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F19%2Fmore-on-non-coding-rnas%2F&linkname=More%20on%20non-coding%20RNAs "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F19%2Fmore-on-non-coding-rnas%2F&linkname=More%20on%20non-coding%20RNAs "Email")[](https://www.addtoany.com/share)
