+++
title = "Is intergenic expression"
full_title = "Is intergenic expression functional?"
date = "2006-09-21"
upstream_url = "https://www.gnxp.com/WordPress/2006/09/21/is-intergenic-expression-functional/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/09/21/is-intergenic-expression-functional/).

Is intergenic expression functional?

Both [Coffee Mug](https://www.gnxp.com/blog/2006/09/more-on-non-coding-rnas.php) and [I](https://www.gnxp.com/blog/2006/05/eghm-i-slow-death-of-one-gene-one.php) have made a big deal out of the fact that a large percentage of the genome in eukaryotes is transcribed into RNA. In the comments, however, gc has been skeptical of inferring much from this fact, noting that the transcription machinery of the cell is inherently “leaky”. A [new paper](http://genetics.plosjournals.org/perlserv/?request=get-document&doi=10.1371%2Fjournal.pgen.0020171.eor) adds some fresh data to the mix.

Here’s what they did: in five chimps and five humans, the researchers isolated four different tissues and measured the expression levels (using a [tiling array](https://contingencytable.blogspot.com/2006/03/things-that-make-you-say-wow.html)) of a number of overlapping 25-bp segments in the [ENCODE regions](https://www.gnxp.com/blog/2006/05/eghm-i-slow-death-of-one-gene-one.php). These segments could then be divided into genic (i.e. protein coding) and intergenic regions.

There are a number of ways these data can be looked at to determine whether the expression of different classes of sequence is functional. First, for a given tissue, one can determine the overlap between the probes expressed in chimpanzees and humans. If the percentage overlap in genic and intergenic regions is the same, that would support the hypothesis that the intergenic regions are acting similarly, in an evolutionary sense, to genic regions, and are possibly functional.

Further, previous studies have shown that gene expression in the brain is highly conserved between humans and chimps, implying negative selection, while gene expression in the testis is highly diverged, implying positive selection. If intergenic probes show the same pattern, it would suggest that the same forces are acting on intergenic transcription. And as natural selection is not expected to play a role in determining the expression of non-functional transcripts, this would then suggest the intergenic transcription is functional.

On doing both of these tests, the authors found that intergenic transcription does indeed behave like genic transcription, giving strong support for the functionality of these transcripts. Previously, gc has raised the point that expression is a continuous, rather then a binary, variable, and indeed, among the top 5% of expressed probes, there are many more genic regions, but apart from that, the raw number of genic and non-genic probes expressed at a given intensity seem pretty similar (leading to an overrepresentation of genic regions, but still…). In my opinion, this paper gives hope to those who think that the “dark matter” of the genome plays an important role in phenotypes and that leaky transcription isn’t going to be the dominant story coming out of the ENCODE data.

### Related Posts:

- [Extensive transcription in the
  fly](https://www.gnxp.com/WordPress/2006/10/01/extensive-transcription-in-the-fly/) - [Evolutionary forces in the (human)
  genome](https://www.gnxp.com/WordPress/2006/10/13/evolutionary-forces-in-the-human-genome/) - [Epigenetics and gene
  structures](https://www.gnxp.com/WordPress/2009/02/26/epigenetics-and-gene-structures/) - [A Nobel Prize for gene
  expression](https://www.gnxp.com/WordPress/2006/10/02/a-nobel-prize-for-gene-expression/) - [HAR1F
  tangents](https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/) - [How much of the genome is transcribed? Or, the utility of
  a…](https://www.gnxp.com/WordPress/2010/08/16/how-much-of-the-genome-is-transcribed-or-the-utility-of-a-good-genome-browser/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F21%2Fis-intergenic-expression-functional%2F&linkname=Is%20intergenic%20expression%20functional%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F21%2Fis-intergenic-expression-functional%2F&linkname=Is%20intergenic%20expression%20functional%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F21%2Fis-intergenic-expression-functional%2F&linkname=Is%20intergenic%20expression%20functional%3F "Email")[](https://www.addtoany.com/share)
