+++
title = "Humans nothin' special?"
full_title = "Humans nothin' special?"
date = "2006-09-17"
upstream_url = "https://www.gnxp.com/WordPress/2006/09/17/humans-nothin-special/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/09/17/humans-nothin-special/).

Humans: nothin' special?

The abstract for [this](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6VJ1-4KB10W5-2&_coverDate=10%2F31%2F2006&_alid=450872449&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=6081&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=61c644fc51a1688fa52274338c4fb98b) article by Adam Eyre-Walker (whose work I’ve quibbled with [before](https://www.gnxp.com/blog/2006/05/fitness-and-disease-ii.php)) caught my eye:

> The role of positive darwinian selection in evolution at the molecular > level has been keenly debated for many years, with little resolution. > However, a recent increase in DNA sequence data and the development of > new methods of analysis have finally made this question tractable. > Here, I review the current state-of-play of the field. Initial > estimates in Drosophila suggest that about 50% of all amino acid > substitutions, and a substantial fraction of substitutions in > non-coding DNA, have been fixed as a consequence of adaptive > evolution. Estimates in microorganisms are even higher. **By contrast, > there is little evidence of widespread adaptive evolution in our own > species.**

…\[double take\]…huh?

Or how about this:

> A few years ago, there was great optimism that, by looking for the > signature of selective sweeps, it might be possible to use genomic > scans of DNA diversity to identify regions of the genome that had > recently undergone adaptive evolution. Unfortunately, this program of > research has not proved as fruitful as had been hoped.

The [Voight et al.](http://biology.plosjournals.org/perlserv?request=get-document&doi=10.1371/journal.pbio.0040072) paper familiar to most readers does get a mention, but [Wang et al.](http://www.pnas.org/cgi/content/short/103/1/135) doesn’t. These two results, along with an [excess of linkage disequilibrium](https://www.gnxp.com/blog/2006/09/more-positive-selection.php) in genes in human compared to non-genic regions, certainly suggests widespread selection, though perhaps this selction is not detected using traditional methods.

Of course, it’s not the size of the selection coefficient that matters, rather how you use it, but still, it seems to me to be a bizarre claim that human have not been under widespread selection. Perhaps compared to *Drosophila*?

### Related Posts:

- [Selection in humans, a
  follow-up](https://www.gnxp.com/WordPress/2006/09/21/selection-in-humans-a-follow-up/) - [The genome, more than
  coding](https://www.gnxp.com/WordPress/2009/08/07/the-genome-more-than-coding/) - [An operational definition of the neutral
  theory?](https://www.gnxp.com/WordPress/2007/03/12/an-operational-definition-of-the-neutral-theory/) - [Human v. chimp: the evolutionary
  showdown](https://www.gnxp.com/WordPress/2007/05/07/human-v-chimp-the-evolutionary-showdown/) - [More than monkey
  brains](https://www.gnxp.com/WordPress/2006/10/11/more-than-monkey-brains/) - [More than monkey
  brains](https://www.gnxp.com/WordPress/2006/10/12/more-than-monkey-brains/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F17%2Fhumans-nothin-special%2F&linkname=Humans%3A%20nothin%27%20special%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F17%2Fhumans-nothin-special%2F&linkname=Humans%3A%20nothin%27%20special%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F17%2Fhumans-nothin-special%2F&linkname=Humans%3A%20nothin%27%20special%3F "Email")[](https://www.addtoany.com/share)
