+++
title = "Selection in humans, a"
full_title = "Selection in humans, a followup"
date = "2006-09-21"
upstream_url = "https://www.gnxp.com/WordPress/2006/09/21/selection-in-humans-a-follow-up/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/09/21/selection-in-humans-a-follow-up/).

Selection in humans, a follow-up

To follow up on a previous [post](https://www.gnxp.com/blog/2006/09/humans-nothin-special.php) on determining the fraction of the human genome under selection (as well as Coffee Mug’s [post](https://www.gnxp.com/blog/2006/09/more-on-non-coding-rnas.php) on non-coding RNAs), here’s a [paper](http://hmg.oxfordjournals.org/cgi/content/full/15/suppl_2/R170) which claims that, in raw numbers, more non-coding DNA has been under selection in humans than coding DNA. But still, using their method, based on the distribution of insertions and deletions in the genome, there’s not a whole lot of the genome under selection– something like 0.3%. However:

> For four reasons, \[this estimate\] may be a considerable > under-estimate. First, the method necessarily only exploits > orthologous regions that retain sufficient resemblance to allow their > accurate alignment. Lineage-specific or orthologous segments whose > sequences have diverged greatly as a consequence of positive selection > are thus not able to be aligned and are not counted towards the > genomic total. For example, it has been shown that sequences > unalignable between human and mouse often contain structural RNA > elements. Secondly, sequences are often not included if they have > recently gained function, owing to their sequence divergence being > intermediate between those of neutral and constrained sequences. > Thirdly, the method misses adaptive sequence within which selection > has not acted heterogeneously, but instead has driven both beneficial indels and substitutions to fixation. Finally, it also overlooks positively selected sites, or short regions, that are scattered among a majority of constrained bases.

This is also interesting, given results from Bruce Lahn’s lab showing accelerated evolution of nervous system genes in humans:

> More recently, we exploited this signature of selection upon indels to > conduct a genome-wide scan for positive selection on small functional intronic elements. We find such elements to be especially abundant in the introns of genes that are expressed in the brain (Lunter and Ponting, submitted for publication)

### Related Posts:

- [The genome, more than
  coding](https://www.gnxp.com/WordPress/2009/08/07/the-genome-more-than-coding/) - [Accelerated human evolution in non-coding
  regions?](https://www.gnxp.com/WordPress/2006/11/02/accelerated-human-evolution-in-non-coding-regions/) - [Is intergenic expression
  functional?](https://www.gnxp.com/WordPress/2006/09/21/is-intergenic-expression-functional/) - [More detecting natural
  selection](https://www.gnxp.com/WordPress/2007/06/18/more-detecting-natural-selection/) - [Extensive transcription in the
  fly](https://www.gnxp.com/WordPress/2006/10/01/extensive-transcription-in-the-fly/) - [Localizing recent positive selection in humans
  using…](https://www.gnxp.com/WordPress/2010/01/09/localizing-recent-positive-selection-in-humans-using-multiple-statistics/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F21%2Fselection-in-humans-a-follow-up%2F&linkname=Selection%20in%20humans%2C%20a%20follow-up "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F21%2Fselection-in-humans-a-follow-up%2F&linkname=Selection%20in%20humans%2C%20a%20follow-up "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F09%2F21%2Fselection-in-humans-a-follow-up%2F&linkname=Selection%20in%20humans%2C%20a%20follow-up "Email")[](https://www.addtoany.com/share)
