+++
title = "Ancient archaic"
full_title = "Ancient archaic admixture into the Andamanese"
date = "2016-07-25"
upstream_url = "https://www.gnxp.com/WordPress/2016/07/25/ancient-archaic-admixture-into-the-andamanese/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/07/25/ancient-archaic-admixture-into-the-andamanese/).

Ancient archaic admixture into the Andamanese

[![ng.3621-F3](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/ng.3621-F3.jpg?resize=600%2C400)![ng.3621-F3](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/ng.3621-F3.jpg?resize=600%2C400)](http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3621.html)  
[![Screenshot 2016-07-25 21.52.42](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/Screenshot-2016-07-25-21.52.42-151x300.png?resize=151%2C300)![Screenshot 2016-07-25 21.52.42](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/Screenshot-2016-07-25-21.52.42-151x300.png?resize=151%2C300)](http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3621.html)  
**Update:** If Pontus Skoglund fails to replicate your results it is not an optimal outcome….

> [@razibkhan](https://twitter.com/razibkhan) I tried to replicate the archaic admixture signal (using 4 other data sets with similar power) but failed [pic.twitter.com/UR16xoU5no](https://t.co/UR16xoU5no) >
> — Pontus Skoglund (@pontus_skoglund) [July 26, 2016](https://twitter.com/pontus_skoglund/status/757928539131904000)

**end update**  
A new paper on on archaic admixture in Andaman Islanders has come out. It’s in *Nature Genetics*, [Genomic analysis of Andamanese provides insights into ancient human migration into Asia and adaptation](http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3621.html). If you don’t have access, just read the [supplements](http://www.nature.com/ng/journal/vaop/ncurrent/extref/ng.3621-S1.pdf), they have the good stuff as usual.

The results here range fromintriguing to clarifying. But I want to engage in a little post-publication criticism: **I think the fact that the authors highlighted model-based admixture results is a disservice to the rest of their results**. [Daniel Falush](https://twitter.com/danielfalush) will be out with a paper on best practices (or not) in relation to the utilization of model-based admixture soon enough, but the plot that is part of figure 1 is really in my opinion misleading in the broader context of their results. The JAR and ONG are Andamanese samples, while ILA and BIR are isolated tribal group in south to central India. These four groupsare not very numerous, and, they are characterized by very small effective populations. In other words, the model-based admixture framework immediately infers these to be extremely close to idealized reference populations because they are so drifted. One can call this the “Kalash effect”, as the original STRUCTURE based analyses of the middle 2000s would routinely yield a “Kalash cluster.”

One consequence of the mix of populations that they have in this reference panel is some of the results are difficult to interpret. The Andamanese and Irula (a South Indian tribe) become nearly fixed for a major cluster respectively, and in the text the authors are interested enough to observe that these Andaman samples don’t show admixture because of this result. This is not necessarily true though. If these are the ‘purest’ of the samples that remain, then they will serve as the ‘pure’ reference. That doesn’t mean there hasn’t been recent gene flow into them. The same with the Irula. Because these two are highly drifted populations they are quite often going to arrive at these very high fractions in some particular K cluster.

Because of the large sample these rather obscure tribal groups, you have the result that Rajputs seem to share more ancestry with Chinese than the Bengalis (RAJ and BEN). **This is in contradiction to everything we know**. In reality, the Rajput samples have very little recent East Asian admixture, while the Bengalis are about \~10% East Asian on the time scale of the past one to two thousand years. I can offer a hypothesis for why the clustering algorithm converged on this result, but I’ll just suggest you read Lucy van Dorp’s [Evidence for a Common Origin of Blacksmiths and Cultivators in the Ethiopian Ari within the Last 4500 Years: Lessons for Clustering-Based Inference](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005397).

But what of the results overall? The intriguing aspect is the evidence that there is archaic admixture into the Andamanese, Australian indigenous peoples, and to some extent Indians (via their “Ancestral South Indian”) form a mystery hominin. The time scale they infer is on the order of divergence at least several hundred thousand year ago in the past. Presumably these results became clear because they had good high quality whole genome sequences (\~10x and more coverage) with millions of SNPs to work with (and well ascertained), as opposed to the few hundred thousand in earlier work.

[![212978](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/212978-188x300.jpg?resize=188%2C300)![212978](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/212978-188x300.jpg?resize=188%2C300)](https://www.amazon.com/Catholicism-American-Freedom-John-McGreevy/dp/0618197362?ie=UTF8&redirect=true&tag=geneexpressio-20)One of the models the authors were testing is the idea that there were two waves out of Africa, with South Asian and Oceanian peoples in particular exhibiting hallmarks of admixture from an earlier migration. There are smattering of population genomics papers which support this proposition, **but I will tell you most researchers who work with these data are skeptical.** There were some methodological issues with some earlier findings, and limitations on the sample sizes and quality (whole genome sequence), or marker set (the SNP studies).

By and large these results support the idea of one migration. If one assumes Africans are one branch of the modern human race, all non-Africans are approximately equally related to Africans, because there is one major bifurcation between Africans and non-Africans. In contrast, the two migration thesis often posits that there was an earlier migration, that was later overwhelmed by a later migration. In that case, you might have scenarios where the second population shared more genetic drift with modern Africans because of a later separation. Or, there could be a scenario where the South Eurasian substrate shared more drift with Africans, shifting the affinities of the Andaman Islanders in that direction.

In any case, they did not see evidence of any of this. Rather, Andamanese share no more ancestry with Africans than you’d expect of generic non-Africans…sort of. As it turns out when **using an outgroup with only ancestral allelic states as inferred from using the chimpanzee reference Andamanese actually have *fewer* African alleles that Europeans and East Asians.** The same is true of indigenous Australians, though the authors did not tackle this issue in detail. Of course this could be Neanderthal and Denisovan ancestry. But it isn’t. We have the ancient genomes of these populations, and the Andamanese seem typical for a East Eurasian population.

They make the inference of a few percent admixture from a population which diverged from modern humans on the order of \~300,000 years ago through a lot of testing of explicit models. Again, read the supplements. Unfortunately they don’t have the genome of the mysterious ghost population, so they can’t make definitive conclusions.

The fact that East Asians and Europeans both lack this ancestral element is peculiar, because all the evidence puts the Andamanese on the same branch as the East Asians. So you can have models whereby a composite East Asian/European population engages in back migration to Africa. Or, perhaps Ancient North Eurasians, who contributed to both Europeans and East Asians? In any case, **of the models they tested it does seem that archaic admixture from a ghost population is the most likely, but that does not mean that that is the correct answer.** Obviously they did not exhaust the possibilities of models and scenarios. For several years now people have been seeing strange things in the data from Australasian and isolated Southeast Eurasian groups. There’s a story here, we just don’t know much of the outlines, we’re just grasping. Both [Mait Metspalu](http://www.evolutsioon.ut.ee/MAIT/) and Luca Pagani have papers in the works….

What about the clarifying? It strikes me that they used some stringent selection tests (e.g., focusing on hard sweeps?), and still hit pay-dirt on positive results for the Andamanese. Both selection tests, and polygenic scores, suggest there’s a bias toward being small in these two groups. I’m curious if the Sentinelese would be the same. The thesis here is that islands result in dwarfism. But I wonder if the selection here is very recent indeed. The authors note that some of the sweeps are not complete. In the last few hundred years there has been a massive demographic collapse of the Andaman Islanders, and only the Sentinelese preserve their lifestyle in a pristine fashion.

This relates to the question in regards to phylogenetics. The authors indicate the the phenotype of these Austro-Melanesian populations is not necessarily due to shared descent from Africans, but adaptation to local circumstances. I don’t think they’ve proven it as such, but they’ve definitely shown that there’s a lot of selection occur in their new habitats. Curiously they don’t detect that there’s been adaptive introgression from the archaic, but I’m not sure if they have the power to detect that sort of thing anyhow. But, it might indicate that the admixture occurred further north?

Finally, I should mention that the [Andamanese genomes are now online](http://www.ebi.ac.uk/ena/data/view/ERZ126015).

### Related Posts:

- [How science is
  done](https://www.gnxp.com/WordPress/2016/08/10/how-science-is-done/) - [Bell Beaker going to crash-land and blow our minds
  soon?](https://www.gnxp.com/WordPress/2017/05/09/bell-beaker-going-to-crash-land-and-blow-our-minds-soon/) - [Derived PLK4 variant not common in
  Bushmen](https://www.gnxp.com/WordPress/2015/04/29/derived-plk4-variant-not-common-in-bushmen/) - [Denisovans, Neanderthals, Yetis, oh
  my!](https://www.gnxp.com/WordPress/2018/03/19/denisovans-neanderthals-yetis-oh-my/) - [The Jermyn
  Program](https://www.gnxp.com/WordPress/2010/05/11/the-jermyn-program/) - [The unbearable thinness of
  Denisovan](https://www.gnxp.com/WordPress/2011/10/31/the-unbearable-thinness-of-denisovan/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F07%2F25%2Fancient-archaic-admixture-into-the-andamanese%2F&linkname=Ancient%20archaic%20admixture%20into%20the%20Andamanese "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F07%2F25%2Fancient-archaic-admixture-into-the-andamanese%2F&linkname=Ancient%20archaic%20admixture%20into%20the%20Andamanese "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F07%2F25%2Fancient-archaic-admixture-into-the-andamanese%2F&linkname=Ancient%20archaic%20admixture%20into%20the%20Andamanese "Email")[](https://www.addtoany.com/share)
