+++
title = "10,000 genome at 30x &"
full_title = "10,000 genome at 30x & human variation"
date = "2016-07-03"
upstream_url = "https://www.gnxp.com/WordPress/2016/07/03/10000-genome-at-30x-human-variation/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/07/03/10000-genome-at-30x-human-variation/).

10,000 genome at 30x & human variation

[![Screenshot 2016-07-02 22.20.21](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/Screenshot-2016-07-02-22.20.21.png?resize=620%2C653)![Screenshot 2016-07-02 22.20.21](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/07/Screenshot-2016-07-02-22.20.21.png?resize=620%2C653)](http://biorxiv.org/content/biorxiv/early/2016/07/01/061663.full.pdf)  
[Deep Sequencing of 10,000 Human Genomes](http://biorxiv.org/content/early/2016/07/01/061663):

> We report on the sequencing of 10,545 human genomes at 30-40x coverage > with an emphasis on quality metrics and novel variant and sequence > discovery. We find that 84% of an individual human genome can be > sequenced confidently. This high confidence region includes 91.5% of > exon sequence and 95.2% of known pathogenic variant positions. We > present the distribution of over 150 million single nucleotide > variants in the coding and non-coding genome. Each newly sequenced > genome contributes an average of 8,579 novel variants. In addition, > each genome carries in average 0.7 Mb of sequence that is not found in > the main build of the hg38 reference genome. The density of this > catalog of variation allowed us to construct high resolution profiles that define genomic sites that are highly intolerant of genetic variation. These results indicate that the data generated by deep genome sequencing is of the quality necessary for clinical use.

The 30x means that they’re hitting each base on an average of 30 times, so they can be very confident of their call. This matters a lot for rare variants, as might be useful when it comes to idiopathic diseases. The 10,000 number is obviously to take it a step beyond the “1,000” genomes, which went well above 1,000 genomes in any case. But the coverage means that these are very confident calls for any given individual.

A distribution of variants shows that their panel of unrelated individuals (\~8,000) yields \~150,000,000 single nucleotide variants (out of a genome of 3,000,000,000 bases). **You see that half of these 150million are found at counts of one across their whole sample set**. In contrast, you have \~5 million variants present at allele frequencies of about 5% or more, and a bit more than \~10 million variants at 1% or more, and \~20 million variants at 0.1% or more. Remember that the [1000 Genomes paper](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html) reported that each individual within their data set have about \~5 million variants in comparison to the human reference genome.

I reiterate these dull numbers to give people a sense of what it means to have 100,000 to 1 million marker SNP-chips in humans. It is true that without imputation these chips aren’t capturing a lot of functional variants (though they’re typical designed to target a lot of the most important disease markers in particular). **But when comes to capturing the shape of genetic variation they’re a very good sampling indeed**. Consider, for example, the proportion and number of voters who are part of the samplefor exit polls or pre-election surveys. For standard PCA or genotypic model based clustering (e.g., ADMIXTURE/STRUCTURE) anything more than 1 million markers is pretty useful from what I’ve seen, and the 100,000 to 500,000 interval is sufficient for pretty much everything. And haplotype based methods that generally usephasing, like fineSTRUCTURE, seem to do fine in the \~250,000 marker range.

### Related Posts:

- [Beyond the
  sequence](https://www.gnxp.com/WordPress/2009/04/22/beyond-the-sequence/) - [Ask GNXP - personal genome
  sequence](https://www.gnxp.com/WordPress/2006/07/25/ask-gnxp-personal-genome-sequence/) - [Happy DNA Day (and whole genome sequencing
  yourself)](https://www.gnxp.com/WordPress/2022/04/25/happy-dna-day-and-whole-genome-sequencing-yourself/) - [How many human genomes have been
  sequenced?](https://www.gnxp.com/WordPress/2011/11/07/how-many-human-genomes-have-been-sequenced/) - [The \$2,000 "whole
  genome"](https://www.gnxp.com/WordPress/2015/01/10/the-2000-whole-genome/) - [On the personal genomics turning
  point](https://www.gnxp.com/WordPress/2010/05/18/on-the-personal-genomics-turning-point/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F07%2F03%2F10000-genome-at-30x-human-variation%2F&linkname=10%2C000%20genome%20at%2030x%20%26%20human%20variation "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F07%2F03%2F10000-genome-at-30x-human-variation%2F&linkname=10%2C000%20genome%20at%2030x%20%26%20human%20variation "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F07%2F03%2F10000-genome-at-30x-human-variation%2F&linkname=10%2C000%20genome%20at%2030x%20%26%20human%20variation "Email")[](https://www.addtoany.com/share)
