+++
title = "Fisherianism in the"
full_title = "Fisherianism in the genomic era"
date = "2017-04-12"
upstream_url = "https://www.gnxp.com/WordPress/2017/04/12/fisherianism-in-the-genomic-era/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/04/12/fisherianism-in-the-genomic-era/).

Fisherianism in the genomic era

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/41L69h9XdRL._SX331_BO1204203200_-1.jpg?resize=200%2C300)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/41L69h9XdRL._SX331_BO1204203200_-1.jpg?resize=200%2C300)](https://www.amazon.com/exec/obidos/ASIN/0198504403/geneexpressio-20)There are many things about [R. A. Fisher](https://en.wikipedia.org/wiki/Ronald_Fisher) that one could say. **Professionally he was one of the founders of evolutionary genetics and statistics**, and arguably the second greatest evolutionary biologist after Charles Darwin. With his work in the first few decades of the 20th century he reconciled the quantitative evolutionary framework of the school of biometry with mechanistic genetics, and formalized evolutionary theory in [The Genetical Theory of Natural Selection](https://www.amazon.com/exec/obidos/ASIN/0198504403/geneexpressio-20).

He was also an asshole. This is clear in the major biography of him, [R.A. Fisher: The Life of a Scientist](https://www.amazon.com/exec/obidos/ASIN/0471093009/geneexpressio-20). **It was written by his daughter.** But [The Lady Tasting Tea: How Statistics Revolutionized Science in the Twentieth Century](https://www.amazon.com/exec/obidos/ASIN/0805071342/geneexpressio-20) also seems to indicate he was a dick. And W. D. Hamilton’s [Narrow Roads of Gene Land](https://www.amazon.com/exec/obidos/ASIN/0716745305/geneexpressio-20) portrays Fisher has rather cold and distant, despite the fact that Hamilton idolized him.

Notwithstanding his unpleasant personality, R. A. Fisher seems to have been a veritable mentat in his early years. Much of his thinking crystallized in the first few decades of the 20th century, when genetics was a new science and mathematical methods were being brought to bear on a host of topics. It would be decades until DNA was understood to be the substrate of heredity. Instead of deriving from molecular first principles which were simply not known in that day, Fisher and his colleagues constructed a theoretical formal edifice which drew upon patterns of inheritance that were evident in lineages of organisms that they could observe around them (Fisher had a mouse colony which he utilized now and then to vent his anger by crushing mice with his bare hands). Uponthat observational scaffold they placed a sturdy superstructure of mathematical formality. That edifice has been surprisingly robust down to the present day.

One of Fisher’s frameworks which still gives insight is the [geometric model](https://en.wikipedia.org/wiki/Fisher%27s_geometric_model)of the distribution of fitness of mutations. If an organism is near its optimum of fitness, than large jumps in any direction will reduce its fitness. In contrast, small jumps have some probability of getting closer to the optimum of fitness. In plainer language, mutations of large effect are bad, and mutations of small effect are not as bad.

A new paper in *PNAS*loops back to this framework, [Determining the factors driving selective effects of new nonsynonymous mutations](http://www.pnas.org/content/early/2017/04/10/1619508114):

> Our study addresses two fundamental questions regarding the effect of > random mutations on fitness: First, do fitness effects differ between > species when controlling for demographic effects? Second, what are the > responsible biological factors? We show that amino acid-changing > mutations in humans are, on average, more deleterious than mutations > in *Drosophila*. **We demonstrate that the only theoretical model that > is fully consistent with our results is Fisher’s geometrical model.** > This result indicates that species complexity, as well as distance of > the population to the fitness optimum, modulated by long-term population size, are the key drivers of the fitness effects of new amino acid mutations. Other factors, like protein stability and mutational robustness, do not play a dominant role.

In the title of the paper itself is something that would have been alien to Fisher’s understanding when he formulated his geometric model: the term “nonsynonymous” to refer to mutations which change the amino acid corresponding to the triplet codon. The paper is understandably larded with terminology from the post-DNA and post-genomic era, **and yet comes to the conclusion that a nearly blind statistical geneticist from about a century ago correctly adduced the nature of mutation’s affects on fitness in organisms.**

The authors focused on two primary species which different histories, but well characterized in the evolutionary genomic literature: **humans and *Drosophila*. The models they tested are as follows:  
[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/F1.large_.jpg?resize=640%2C368)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/F1.large_.jpg?resize=640%2C368)](http://www.pnas.org/content/early/2017/04/10/1619508114.full)**



Basically they checkedthe empirical distribution of the [site frequency spectra](https://en.wikipedia.org/wiki/Allele_frequency_spectrum) (SFS) of the nonsynonymous variantsagainst expected outcomes based on particular details of demographics, which were inferred from synonymous variation.*Drosophila*have effective population sizes orders of magnitude larger than humans, so if that is not taken into account, then the results will be off. There are also a bunch of simulations in the paper to check for robustness of their results, and they also caveat the conclusion with admissions that other models besides the Fisherian one may play some role in their focal species, and more in other taxa. A lot of this strikes me as accruing through the review process, and I don’t have the time to replicate all the details to confirm their results, though I hope some of the reviewers did so (again, I suspect that the reviewers were demanding some of these checks, so they definitely should have in my opinion).

![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/129px-Drosophila_melanogaster_-_side_aka.jpg?resize=129%2C100)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/129px-Drosophila_melanogaster_-_side_aka.jpg?resize=129%2C100)In the Fisherian model more complex organisms are more fine-tuned due topleiotropy and other such dynamics. So new mutations are more likely to deviate away from the optimum. This is the major finding that they confirmed. What does “complex” mean? The*Drosophila*genome is less than 10% of the human genome’s size, but the [migratory locust](https://www.nature.com/articles/ncomms3957) has twice as large a genome as humans, while wheat has a sequence more than five times as large. But organism to organism, it does seem that*Drosophila* has less complexity than humans. And they checked with other organisms besides their two focal ones…though the genomes there are not as complete presumably.

As I indicated above, the authors believe they’ve checked for factors such as background selection, which may confound selection coefficients on specific mutations. The paper is interesting as much for the fact that it illustrates how powerful analytic techniques developed in a pre-DNA era were. Some of the models above are mechanistic, and require a certain understanding of the nature of molecular processes. **And yet they don’t seem as predictive as a more abstract framework!**

**Citation:** Christian D. Huber, Bernard Y. Kim, Clare D. Marsden, and Kirk E. Lohmueller, Determining the factors driving selective effects of new nonsynonymous mutations PNAS 2017 ; published ahead of print April 11, 2017, doi:10.1073/pnas.1619508114

### Related Posts:

- [Charles Darwin
  Online](https://www.gnxp.com/WordPress/2006/10/25/charles-darwin-online/) - [The theory of
  evolution](https://www.gnxp.com/WordPress/2008/05/09/the-theory-of-evolution/) - [Fisher and population
  size](https://www.gnxp.com/WordPress/2006/09/30/fisher-and-population-size/) - [The Genetical Theory of Natural
  Selection](https://www.gnxp.com/WordPress/2011/12/27/the-genetical-theory-of-natural-selection/) - [Evolutionary & ecological
  genetics](https://www.gnxp.com/WordPress/2006/03/03/evolutionary-ecological-genetics/) - [Happy Darwin Day! Now read some
  Fisher....](https://www.gnxp.com/WordPress/2016/02/12/happy-darwin-day-now-read-some-fisher/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F12%2Ffisherianism-in-the-genomic-era%2F&linkname=Fisherianism%20in%20the%20genomic%20era "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F12%2Ffisherianism-in-the-genomic-era%2F&linkname=Fisherianism%20in%20the%20genomic%20era "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F12%2Ffisherianism-in-the-genomic-era%2F&linkname=Fisherianism%20in%20the%20genomic%20era "Email")[](https://www.addtoany.com/share)
