+++
title = "Genetic variation in"
full_title = "Genetic variation in human populations and individuals"
date = "2017-04-15"
upstream_url = "https://www.gnxp.com/WordPress/2017/04/15/genetic-variation-in-human-populations-and-individuals/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/04/15/genetic-variation-in-human-populations-and-individuals/).

Genetic variation in human populations and individuals

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/nature15393-f1.jpg?resize=640%2C468)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/nature15393-f1.jpg?resize=640%2C468)](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html)  
[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/512CNVQ-yAL._SX331_BO1204203200_.jpg?resize=200%2C300)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/512CNVQ-yAL._SX331_BO1204203200_.jpg?resize=200%2C300)](https://www.amazon.com/exec/obidos/ASIN/0060932902/geneexpressio-20)I’m old enough to remember when we didn’t have a good sense of how many genes humans had. I vaguely recall numbers around 100,000 at first, which in hindsight seems rather like a round and large number. A guess. Then it went to 40,000 in the early 2000s and then further until it converged to some number just below 20,000.

**But perhaps more fascinating is that we have a much better catalog of the variation across the whole human genome now.** Often friends ask me questions of the form: “so DTC genomic company X has about 800,000 SNPs, is that enough to do much?” To answer such a question you need some basic numbers in your head, as well as what you want to “do.”

First, the human genome has about 3 billion base pairs (3 Gb). That’s a lot. But most of the genome famously doesn’t code for proteins. The exome, the proportion of the genome where bases directly translate into a protein accounts for 1% of the whole genome. That’s 30 million bases (30 Mb). But this small region of the genome is very important, as the vast majority of major disease mutations are found in the exome.

When it comes to a standard 800K SNP chip, which samples 800,000 positions across the 3 Gb genome, it is likely that the designers enriched the marker set for functional positions relevant to diseases. Not all marker positions are created equal. Though even outside of those functional positions there are often nearby SNPs that can “tag” them, so you can infer one from the state of the other.

But are 800,000 positions enough to make good ancestry inference? (to give one example) Yes. 800,000 is actually a substantial proportion of the polymorphism in any given genome. There have been some papers which improved on the numbers in 2015’s [A global reference for human genetic variation](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html), but it’s still a good comprehensive review to get an order-of-magnitude sense. The table below gives you a sense of **individual variation**:

**Median autosomal variant sites per genome  
**

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/var.png?resize=640%2C340)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/var.png?resize=640%2C340)](https://www.nature.com/nature/journal/v526/n7571/pdf/nature15393.pdf)

When it comes to single nucleotide polymorphisms (SNPs), what SNP chips are getting at, an 800K array should get a substantial proportion of your genome-wide variation. More than enough for ancestry inference or forensics. The singleton column shows mutations **specific to the individual**. When focusing on new mutations specific to an individual that might cause disease, singleton large deletions and nonsynonymous SNPs is really where I’d look.

![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/variants.png?resize=350%2C342)

![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/04/variants.png?resize=350%2C342)

But what about whole populations? The plot to the left shows the count of variants as a function of alternative allele frequency. When we say “SNP”, you really mean variants which exhibit polymorphism at a particular cut-off frequency for the minor allele (often 1%). It is clear that as the minor allele frequency increases in relation to the human reference genome the number of variants decreases.

From the paper:

> The majority of variants in the data set are rare: \~64 million > autosomal variants have a frequency \<0.5%, \~12 million have a > frequency between 0.5% and 5%, **and only \~8 million have a frequency > \>5%** (Extended Data Fig. 3a). Nevertheless, the majority of variants > observed in a single genome are common: just 40,000 to 200,000 of the > variants in a typical genome (1–4%) have a frequency \<0.5% (Fig. 1c > and Extended Data Fig. 3b). As such, we estimate that improved rare variant discovery by deep sequencing our entire sample would at least double the total number of variants in our sample but increase the number of variants in a typical genome by only \~20,000 to 60,000.

An 800K SNP chip will be biased toward the 8 million or so variants with a frequency of 5%. This number gives you a sense of the limited scope of variation in the human genome. 0.27% of the genome captures a lot of the polymorphism.

**Citation:**1000 Genomes Project Consortium. “A global reference for human genetic variation.” *Nature* 526.7571 (2015): 68-74.

### Related Posts:

- [10,000 genome at 30x & human
  variation](https://www.gnxp.com/WordPress/2016/07/03/10000-genome-at-30x-human-variation/) - [Genomics to complex
  traits](https://www.gnxp.com/WordPress/2009/02/18/genomics-to-complex-traits/) - [The dam of ancient DNA starts to
  break](https://www.gnxp.com/WordPress/2013/01/21/the-dam-of-ancient-dna-starts-to-break/) - [How many human genomes have been
  sequenced?](https://www.gnxp.com/WordPress/2011/11/07/how-many-human-genomes-have-been-sequenced/) - [Living in the age of
  structure](https://www.gnxp.com/WordPress/2016/09/05/living-in-the-age-of-structure/) - [Common variants are commonly
  unpromising](https://www.gnxp.com/WordPress/2010/04/01/common-variants-are-commonly-unpromising/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F15%2Fgenetic-variation-in-human-populations-and-individuals%2F&linkname=Genetic%20variation%20in%20human%20populations%20and%20individuals "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F15%2Fgenetic-variation-in-human-populations-and-individuals%2F&linkname=Genetic%20variation%20in%20human%20populations%20and%20individuals "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F04%2F15%2Fgenetic-variation-in-human-populations-and-individuals%2F&linkname=Genetic%20variation%20in%20human%20populations%20and%20individuals "Email")[](https://www.addtoany.com/share)
