+++
title = "Ancestry inference won’t"
full_title = "Ancestry inference won’t tell you things you don’t care about"
date = "2017-03-23"
upstream_url = "https://www.gnxp.com/WordPress/2017/03/23/ancestry-inference-wont-tell-you-things-you-dont-care-about-but-could/"

+++
Source: [here](https://www.gnxp.com/WordPress/2017/03/23/ancestry-inference-wont-tell-you-things-you-dont-care-about-but-could/).

Ancestry inference won’t tell you things you don’t care about (but could)

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/journal.pgen_.0010070.g002.png?resize=600%2C245)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/journal.pgen_.0010070.g002.png?resize=600%2C245)](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0010070)

The figure above is from Noah Rosenberg’s relatively famous paper, [Clines, Clusters, and the Effect of Study Design on the Inference of Human Population Structure](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0010070). The context of the publicationis that it was one of the first prominent attempts to use genome-wide data on a various of human populations (specifically, from the HGDP data set) and attempt model-based clustering. There are many details of the model, but the one that will jump out at you here is that the parameter*K*defines the number of putative ancestralpopulations you are hypothesizing. Individuals then shake out as proportions of each element, *K*. Remember, this is a model in a computer, and you select the parameters and the data. **The output is not “wrong,” it’s just the output based how you set up the program and the data you input yourself.**

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/The_Journey_of_Man_-_A_Genetic_Odyssey.jpg?resize=197%2C299)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/The_Journey_of_Man_-_A_Genetic_Odyssey.jpg?resize=197%2C299)](https://www.amazon.com/exec/obidos/ASIN/B009MYAOQU/nofe0f-20)These sorts of computational frameworks are innocent, and may give strange results if you want to engage in mischief. For example, let’s say that you put in 200 individuals, of whom 95 are Chinese, 95 are Swedish, and 10 are Nigerian. From a variety of disciplines we know to a good approximation that non-Africans form a monophyletic clade in relation to Africans (to a first approximation). In plain English, all non-Africans descend from a group of people who diverged from Africans more than 50,000 years ago. That means if you imagine two populations, the first division should be between Africans and non-Africans, to reflect this historical demography. But if you skew the sample size, as the program looks for the maximal amount of variation in the data set it may decide that dividing between Chinese and Swedes as the two ancestral populationsis the most likely model given the data.

This is not wrong as such. As the number of Africans in the data converges on zero, obviously the dividing line is between Swedes and Chinese. If you overload particular populations within the data, you maymarginalize the variation you’re trying to explore, and the history you’retrying to uncover.

I’ve written all of this before. But I’m writing this in context of the earlier post, [Ancestry Inference Is Precise And Accurate(Ish)](http://gnxp.com/WordPress/2017/03/your-ancestry-inference-is-precise-and-accurateish/). In that post I showed that **consumers drive genomics firms to provide results where the grain of resolution and inference varies a lot as a function of space.** That is, there is a demand that Northern Europe be divided very finely, while vast swaths of non-European continents are combined into one broad cluster.

![**Less than 5% Ancient North Eurasian**](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/b715abab23edfc10b3582f37016eba6a.jpg?resize=300%2C224)

Another aspect though is **time.** These model-based admixture frameworks can implicitly traverse time as one ascends up and down the number of *K*‘s. It is always important to explain to people that the number of *K*‘s may not correspond to real populations which all existed at the same time. Rather, they’re just explanatory instruments which illustrate phylogenetic distance between individuals. In a well-balanced data set for humans *K* = 2 usually separates Africans from non-Africans, and *K* = 3 then separates West Eurasians from other populations. Going across *K*‘s it is easy to imagine that is traversing successive bifurcations.

![**A racially mixed man, 15% ANE, 30% CHG, 25% WHG, 30% EEF**](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/Spencer_Wells.jpg?resize=250%2C326)

**But today we know that’s more complicated than that.** Three years ago Pickrell et al. published [Toward a new history and geography of human genes informed by ancient DNA](http://www.cell.com/trends/genetics/abstract/S0168-9525(14)00120-6), where they report the result that more powerful methods and data imply **most human populations are relatively recent admixtures between extremely diverged lineages.** What this means is that the origin of groups like Europeans and South Asians is very much like the origin of the mixed populations of the New World. Since then this insight has become only more powerful, as ancient DNA has shed light as massive population turnovers over the last 5,000 to 10,000 years.

These are to some extent revolutionary ideas, not well known even among the science press (which is too busy doing [real journalism](https://undark.org/article/race-science-razib-khan-racism/), i.e. the art of insinuation rather than illumination). As I indicated earlier direct-to-consumer genomics **use national identities in their cluster labels because these are comprehensible to people.** Similarly, they can’t very well tell Northern Europeans that they are an outcome of a successive series of admixtures between diverged lineages from the late Pleistocene down to the Bronze Age. Though Northern Europeans, like South Asians, Middle Easterners, Amerindians, and likely Sub-Saharan Africans and East Asians, are complex mixes between disparate branches of humanity, today **we view them as indivisible units of understanding, to make sense of the patters we see around us.**

Personal genomics firms therefore give results which allow for **historically comprehensible results.** As a trivial example, the genomic data makes it rather clear that Ashkenazi Jews emerged in the last few thousand years via a process of admixture between antique Near Eastern Jews, and the peoples of Western Europe. After the initial admixture this group became an endogamous population, so that most Ashkenazi Jews share many common ancestors in the recent past with other Ashkenazi Jews. **This is ideal for the clustering programs above, as Ashkenazi Jews almost always fit onto a particular *K* with ease**. Assuming there are enough Ashkenazi Jews in your data set you will always be able to find the “Jewish cluster” as you increase the*K*value.

But the selection of a *K* which satisfies this comprehensibilitycriterion is a matter of convenience, not necessity. Most people are vaguely aware that Jews emerged as a people at a particular point in history. In the case of Ashkenazi Jews they emerged rather late in history. At certain *K*‘s Ashkenazi Jews exhibit mixed ancestral profiles, placing them between Europeans and Middle Eastern peoples. **What this reflects is the earlier history of the ancestors of Ashkenazi Jews.** But for most personal genomics companies this earlier history is not something that they want to address, because it doesn’t fit into the narrative that their particular consumers want to hear. People want to know if they are part-Jewish, not that they are part antique Middle Eastern and Southwest European.

[![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/journal.pgen_.1002967.g004.png?resize=350%2C348)![](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/journal.pgen_.1002967.g004.png?resize=350%2C348)](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002967)Perplexment of course is not just for non-scientists. When Joe Pickrell’s [TreeMix](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002967) paper came out five years ago there was a strange signal of gene flow between Northern Europeans and Native Americans. There was no obvious explanation at the time…but now we know what was going on.

It turns out that Northern Europeans and Native Americans share [common ancestry](http://www.nature.com/nature/journal/v505/n7481/full/nature12736.html) from Pleistocene Siberians. The relationship between Europeans and Native Americans has long been hinted at in results from other methods, **but it took ancient DNA for us to conceptualize a model which would explain the patterns we were seeing.**

![**An American with recent Amerindian (and probably African) ancestry**](https://i0.wp.com/gnxp.com/WordPress/wp-content/uploads/2017/03/Screenshot-2017-03-23-20.55.45.png?resize=200%2C254)

But in the context of the United States **shared ancestry between Europeans and Native Americans is not particularly illuminating.** Rather, what people want to know is if they exhibit signs of **recent** gene flow between these groups, in particular, many white Americans are curious if they have Native American heritage. They do not want to hear an explanation which involves the fusion of an East Asian population with Siberians that occurred 15,000 to 20,000 years ago, and then the emergence of Northern Europeans thorough successive amalgamations between Pleistocene, Neolithic, and Bronze Age, Eurasians.

In some of the inference methods Northern Europeans, often those with Finnic ancestry or relationship to Finnic groups, may exhibit signs of ancestry from the “Native American” cluster. But this is almost always a function of circumpolar gene flow, as well as the aforementioned Pleistocene admixtures. One way to avoid this would be to simply not report proportions which are below 0.5%. That way, people with higher “Native American” fractions would receive the results, and the proportions would be high enough that it was almost certainly indicative of **recent** admixture, which is what people care about.

Why am I telling you this? Because many journalists who report on direct-to-consumer genomics don’t understand the science well enough to graspwhat’s being sold to the consumer (frankly, most biologists don’t know this field well either, even if they might use a barplot here and there).

And, the reality is that **consumers have very specific parameters of what they want in terms of geographic and temporal information.** They don’t want to be told true but trivial facts (e.g., they are Northern European). But neither they do want to know things which are so novel and at far remove from their interpretative frameworks that they simplycan’t digest them (e.g., that Northern Europeans are a recent population construction which threads together very distinct strands with divergent deep time histories). **In the parlance of cognitive anthropology consumers want their infotainment the way they want their religion, minimally counterintuitive.**Consume some surprise. But not too much.

### Related Posts:

- [PCA remains the swiss-army-knife to explore
  population…](https://www.gnxp.com/WordPress/2017/11/16/pca-remains-the-swiss-army-knife-to-explore-population-structure/) - [How much informative "structure" is in the HGDP data
  set?](https://www.gnxp.com/WordPress/2014/06/06/how-much-informative-structure-is-in-the-hgdp-data-set/) - [One K to rule them
  all?](https://www.gnxp.com/WordPress/2016/08/14/one-k-to-rule-them-all/) - [Details on the back of the
  envelope](https://www.gnxp.com/WordPress/2016/06/02/details-on-the-back-of-the-envelope/) - [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/) - [Lumpy genetic
  variation](https://www.gnxp.com/WordPress/2014/05/22/lumpy-genetic-variation/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F03%2F23%2Fancestry-inference-wont-tell-you-things-you-dont-care-about-but-could%2F&linkname=Ancestry%20inference%20won%E2%80%99t%20tell%20you%20things%20you%20don%E2%80%99t%20care%20about%20%28but%20could%29 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F03%2F23%2Fancestry-inference-wont-tell-you-things-you-dont-care-about-but-could%2F&linkname=Ancestry%20inference%20won%E2%80%99t%20tell%20you%20things%20you%20don%E2%80%99t%20care%20about%20%28but%20could%29 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2017%2F03%2F23%2Fancestry-inference-wont-tell-you-things-you-dont-care-about-but-could%2F&linkname=Ancestry%20inference%20won%E2%80%99t%20tell%20you%20things%20you%20don%E2%80%99t%20care%20about%20%28but%20could%29 "Email")[](https://www.addtoany.com/share)
