+++
title = "Not happening at genomic"
full_title = "Not happening at genomic speed diversification of GWAS panels"
date = "2019-02-18"
upstream_url = "https://www.gnxp.com/WordPress/2019/02/18/not-happening-at-genomic-speed-diversification-of-gwas-panels/"

+++
Source: [here](https://www.gnxp.com/WordPress/2019/02/18/not-happening-at-genomic-speed-diversification-of-gwas-panels/).

Not happening at genomic speed: diversification of GWAS panels

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/02/prs.jpg?resize=625%2C464&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/02/prs.jpg?resize=625%2C464&ssl=1)](https://www.biorxiv.org/content/10.1101/441261v3)  
One of the things that is evident and the norm when you are interested in genetics and genomics is that **things happen fast.** There are some sciences which proceed at a normal and conventional pace. But, because genomics is fundamentally driven by the synergy of two technologies, modern automated sequencing, and computation, the field has been moving at faster than the speech of light. A single whole genome sequence is now cheaper than \$1,000, whereas the first whole genome 20 years ago cost \$3,000,000,000!

People who point to a paper in 2010…well, in genomics that’s ancient history. Take a look at the initial HapMap papers from the mid-2000s if you want to have a laugh!

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/02/prs2.jpg?resize=300%2C268&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/02/prs2.jpg?resize=300%2C268&ssl=1)](https://www.biorxiv.org/content/10.1101/441261v3)But, there’s one area that it seems “genomic speed” hasn’t applied: and that’s the attempts to increase population diversification necessary in GWAS panels to maximize insight. The figure to the right is from a new preprint, [Current clinical use of polygenic scores will risk exacerbating health disparities](https://www.biorxiv.org/content/10.1101/441261v3). To my surprise, over the last few years, the proportion of people of European ancestry, which mostly means Northwest European ancestry, **in genome-wide association studies has actually increased.** The absolute number increases are still heartening, as a a lot of the low-hanging fruit can probably be picked at sample sizes of thousands.

The reason that this matters is illustrated in the figure at the top of this post: GWAS studies in European populations are not entirely informative in non-European populations. There are several factors underlying that patter for this. A major one in quantitative (p0lygenic) traits seems to be the phylogenetic distance (the “American” populations are mixed European and African and Amerindian). This distance reflects evolutionary history, as populations split, or minimize gene flow, and drift and selection result in different allele frequencies and characteristics.

Additionally, some alleles (mutations) are unique to specific populations. This is due to mutation after separation. Using “discovery” panels of European populations may miss these explanatory alleles. Some of these alleles may also be found at small fractions within European populations, though below the statistical threshold of detection. In other words, expanding the population pool is probably useful for Europeans as well.

There are also two other issues. First, in many cases, linkage disequilibrium (LD) patterns are such that many explanatory SNPs are tags for causal variants (i.e., SNP 1 in state A is correlated with SNP 2 in state B). That association may differ between populations. And within African populations LD as a whole is less useful because of the high genetic diversity and a small window of many of the blocks. Finally, there is the possibility of gene-gene interactions, which may differ between populations.

The “good” thing about this problem is that the solution is simple: **invest more money and time into non-European populations**. This is a solution where steps are straightforward, they just need to be taken. I’m not exactly alarmed, but a few years ago I was relatively sanguine, and here we are! There are some projects in Africa right now, and there are groups in India and the Middle East as well as East Asia, **but ultimately it’s all about execution, not proposals**.

### Related Posts:

- [On the personal genomics turning
  point](https://www.gnxp.com/WordPress/2010/05/18/on-the-personal-genomics-turning-point/) - [The genomics future is almost
  now](https://www.gnxp.com/WordPress/2010/10/23/the-genomics-future-is-almost-now/) - [The Illumina
  whole-genome-sequence](https://www.gnxp.com/WordPress/2009/07/05/the-illumina-whole-genome-sequence/) - [Population structure within
  Africa](https://www.gnxp.com/WordPress/2011/03/16/population-structure-within-africa/) - [Personal genomics gets very
  personal](https://www.gnxp.com/WordPress/2011/03/30/personal-genomics-gets-very-personal/) - [The hunt for ancient
  DNA](https://www.gnxp.com/WordPress/2011/08/09/the-hunt-for-ancient-dna/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F02%2F18%2Fnot-happening-at-genomic-speed-diversification-of-gwas-panels%2F&linkname=Not%20happening%20at%20genomic%20speed%3A%20diversification%20of%20GWAS%20panels "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F02%2F18%2Fnot-happening-at-genomic-speed-diversification-of-gwas-panels%2F&linkname=Not%20happening%20at%20genomic%20speed%3A%20diversification%20of%20GWAS%20panels "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F02%2F18%2Fnot-happening-at-genomic-speed-diversification-of-gwas-panels%2F&linkname=Not%20happening%20at%20genomic%20speed%3A%20diversification%20of%20GWAS%20panels "Email")[](https://www.addtoany.com/share)
