+++
title = "Humans have adapted on"
full_title = "Humans have adapted on genomewide level?"
date = "2009-01-16"
upstream_url = "https://www.gnxp.com/WordPress/2009/01/16/humans-have-adapted-on-genome-wide-level/"

+++
Source: [here](https://www.gnxp.com/WordPress/2009/01/16/humans-have-adapted-on-genome-wide-level/).

Humans have adapted on genome-wide level?

[Pervasive Hitchhiking at Coding and Regulatory Sites in Humans](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1000336). Here’s the author summary:

> There is much reported evidence for positive selection at specific > loci in the human genome. Additional papers based on comparisons > between the genomes of humans and chimpanzees have also suggested that > adaptive evolution may be quite common. At the same time, it has been > surprisingly hard to find unambiguous evidence that either positive or > negative (background) selection is affecting genome-wide patterns of > variation at neutral sites. Here, we evaluate the prevalence of > positive or background selection by using two genome-wide datasets of > human polymorphism. We document that levels of neutral polymorphism > are substantially lower in the regions of (i) higher density of genes > and/or regulatory regions, (ii) higher protein or regulatory > divergence, and (iii) lower recombination. **These patterns are robust > to a number of possible confounding factors and suggest that effects > of selection at linked sites cannot be ignored in the study of the > human genome.**

Here’s the critical bit from the discussion:

> Because recurrent adaptive substitutions leave local (on the order of > 0.1 s/ρ) and transient (on the order of Ne generations) dips in > neutral polymorphism, persistent adaptation should lead to lower > levels of neutral polymorphism in regions of lower recombination and > regions where selective sweeps are more frequent and/or stronger on > average. **Here we have confirmed these predictions by showing that > levels of SNP density are lower in the regions of lower recombination > and in the regions of higher functional density and functional > divergence.**

Areas subject to selection, positive or negative, naturally have lower neutral variation because selection “cleans” them either, through through [purifying selection](https://en.wikipedia.org/wiki/Stabilizing_selection) (as deleterious mutants are purged from regions of the genome with important functional significance) or due to the homogenizing effect around the locus in the population of a [selective sweep](https://en.wikipedia.org/wiki/Selective_sweep). In the later case one [haplotype](https://en.wikipedia.org/wiki/Haplotype), which can be a particular sequence of alleles derived from one individual which has come under selection, increases in frequency during the selective process. After selection has ceased due to the fixation of a haplotype, recombination and mutation begins to breakdown down the uniformity around area of homogeneity. The extent of the resultant new variation is obviously proportional to time of sweep, as well as recombination, mutation rate, etc.  
The [authors have some good quotes](http://www.sciencedaily.com/releases/2009/01/090116073211.htm) in *ScienceDaily*:

> “We detected a number of signatures that suggest adaptation is quite > pervasive and common,” Petrov said.  
> Humans have a very complex history from traveling around the globe, > and the human genome is also highly structured, making it complicated > and difficult to work with, he said.  
> To find the adaptation signal, Petrov and his colleagues looked for > regions of the genome that “hitchhiked” along with an adaptation. When > a genetic adaptation occurs and is passed on to offspring, other genes > on both sides of the adaptation typically accompany it. The result is > a whole region of the genome where all humans are unusually similar to > each other, referred to as a “selective sweep,” that researchers can > identify and trace through human genetic history.  
> “Adaptation becomes widespread in the population very quickly,” Petrov > said. “Whereas neutral random mutation doesn’t and would not have the > selective sweep signature.”  
> “We tried to see if these regions of unusual similarity among all > humans tended to be in particular places in the genome as the theory > predicts they should be, and indeed we find them there,” Petrov said. > **“The work suggests human beings have undergone rampant adaptation to > their environment in the last 200,000 years of history.”**

### Related Posts:

- [The genome, more than
  coding](https://www.gnxp.com/WordPress/2009/08/07/the-genome-more-than-coding/) - [Selection in humans, a
  follow-up](https://www.gnxp.com/WordPress/2006/09/21/selection-in-humans-a-follow-up/) - [More detecting natural
  selection](https://www.gnxp.com/WordPress/2007/06/18/more-detecting-natural-selection/) - [Natural selection and
  recombination](https://www.gnxp.com/WordPress/2010/03/28/natural-selection-and-recombination/) - [Chimp vs. human
  genomes](https://www.gnxp.com/WordPress/2005/09/21/chimp-vs-human-genomes/) - [Selection swimming against the genomic
  tide](https://www.gnxp.com/WordPress/2017/10/22/selection-swimming-against-the-genomic-tide/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F01%2F16%2Fhumans-have-adapted-on-genome-wide-level%2F&linkname=Humans%20have%20adapted%20on%20genome-wide%20level%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F01%2F16%2Fhumans-have-adapted-on-genome-wide-level%2F&linkname=Humans%20have%20adapted%20on%20genome-wide%20level%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F01%2F16%2Fhumans-have-adapted-on-genome-wide-level%2F&linkname=Humans%20have%20adapted%20on%20genome-wide%20level%3F "Email")[](https://www.addtoany.com/share)
