+++
title = "The Evolutionary Context"
full_title = "The Evolutionary Context of Genomic Imprinting"
date = "2007-02-05"
upstream_url = "https://www.gnxp.com/WordPress/2007/02/05/the-evolutionary-context-of-genomic-imprinting/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/02/05/the-evolutionary-context-of-genomic-imprinting/).

The Evolutionary Context of Genomic Imprinting

The conventional Mendelian  
model for diploid organisms assumes that the expression of an autosomal  
allele within an individual should be invariant of its sex of origin,  
that is, whether it is inherited from the father or the mother.  
This model is incorrect for a subset of alleles across many taxa, in  
particular mammals. In 1989 David Haig and Mark Westoby outlined  
the evolutionary rationale for parent specific gene expression.¹  
In 1992 paternal and maternal specific imprinting of the insulin growth  
factor 2 and insulin growth factor 2 repressor loci (*Igf2*, *  
Igf2r*) in mice confirmed their predictions.² Over the  
last 15 years as many as 200 loci have been identified as being subject  
to parental specific genomic imprinting.³ The mechanisms  
involved are clear at the most basic level, the repression of gene expression  
via epigenetic mechanisms such as methylation, but the details of the  
molecular genetic processes are still relatively obscure and mysterious.⁴  
But perhaps the larger question is why parental specific gene expression,  
operationally the transformation of a locus to a haploid or monoallelic  
expressive state, should occur on a particular subset of loci.  
Fundamentally this is an evolutionary question, and concurrently with  
the proximate exploration of the molecular mechanisms there has grown  
a theoretical body of work which explores the ultimate rationale of  
the emergence of these peculiar molecular systems.

What is the relevance of theory  
to biology? The examination of biological questions has traditionally  
occurred on various methodological and organizational levels.  
Darwin’s theory of evolution via natural selection upon heritable  
variation was proposed without a viable genetic theory. It was  
40 years between the publication of *The Origin of Species* and  
the emergence of modern genetics, and another 20 years before evolutionary  
biology integrated the Mendelian framework.⁵ Of course,  
this preceded the work of Francis Crick and James Watson on DNA, so  
for a generation the Neo-Darwinian Modern Synthesis of evolutionary  
biology and genetics existed without any understanding of the biochemical  
nature of genes. These historical precedents show that despite  
a lack of understanding of mechanistic details, an evolutionary theoretical  
program can be fruitful in laying the groundwork for future understanding  
of a whole system, and a cogent set of predictions may serve as the  
blueprint for a detailed line of work to elucidate proximate causes.

Currently there are three primary  
theoretical evolutionary explanations for the emergence of genomic imprinting,  
the “Evolvability Hypothesis,” the “Ovarian Time Bomb Hypothesis”  
and the “Kinship Theory.” As the Kinship Theory seems the  
most well supported and thoroughly developed project, I will focus on  
it through most of this paper and only introduce the general outlines  
of the first two theories. 

The Evolvability Hypothesis  
is a model which proposes that genomic imprinting is a mode of transmission  
which allows genetic variation to be maintained within the population.⁶  
The rough logic is similar to the reasoning behind the maintenance of  
sex in complex diploid organisms, the long term fitness of populations  
which maintain genetic variation by shielding alleles of suboptimal  
fitness from selection by masking their expression should be greater  
than populations which allow full biallelic expression and so purge  
less fit alleles from the population. There are problems with  
this model in the details. First, one must consider that because  
parental specific gene expression alternates between active and repressed  
states when inherited from the mother or father, there is little guarantee  
of long term masking from selection for a given allele. For example,  
if one posits that a maternal allele is silent, the probability of that  
allele being masked from selection via maternal transmission a given  
number of generations is simply 0.5 to the power of generations under  
consideration. In other words, the number of generations one may  
expect to be silenced is rather short, and though this model assumes  
environmental and other selective pressures change rapidly so as to  
render suboptimal alleles more optimal over time, such drastic variation  
seem unrealistic. Additionally, there is the reality that for  
recessive alleles, which are normally masked by a dominant allele in  
a diploid organism, genomic imprinting’s haploid expression pattern  
would increase its chances of being exposed to selection. In a  
conventional panmictic scenario the vast majority of low frequency recessives  
would be in heterozygote genotypes where they would be masked, while  
in the context of genomic imprinting one could expect that monoallelic  
expression would result in these recessives being exposed to selection  
without the “protection” of a dominant copy of the given gene.  
In fact, these recessives offer up a much more plausible way of maintaining  
genetic variation within a population without recourse to genomic imprinting.  
Additionally, there is the issue that the proponents of the Evolvability  
Hypothesis make reference to the fitness of groups, which is very difficult  
to model and justify in most evolutionary genetic systems. The  
ubiquity of sex suggests that some long term fitness implication of  
groups might be at play, but the relative paucity of loci which exhibit  
genomic imprinting seems to weigh against an appeal to such theoretically  
tendentious paradigms. Finally, in regards to the empirical data  
the Evolvability Hypothesis has no easy explanations for why such a  
general fitness advantage (i.e., increased genetic variability) would  
only manifest on a minuscule fraction of the genome, exhibit a patchy  
phylogenetic distribution (e.g., be more prominent in mammalian lineage)  
or predict why paternally inherited copies would be more likely to be  
over-expressed in relation to maternal copies. In short, such  
a general solution resting upon shaky theoretical foundations seems  
unwarranted.⁷

The Ovarian Time Bomb Hypothesis  
is a more specific model. Its rationale is that paternal specific  
expression is necessary as a failsafe against premature oocyte development  
and subsequent invasion maternal tissue by trophoblasts.⁸  
This hypothesis explains why there is a tendency for maternal copies  
to be repressed and paternal copies to be expressed, and the genes which  
are evolved in trophoblast development do seem to be under imprinting  
control. Nevertheless this hypothesis’ problem is the inverse  
of the Evolvability Hypothesis in that it is highly specific.  
Loci not involved in trophoblast development are assumed to have been  
imprinted by byproduct mechanisms, perhaps due to their proximity to  
imprinted loci. Additionally, the hypothesis assumes that ovarian tissue  
would have higher metastatic potentional when this is not assured.  
It maybe that male germ cells, sperm, are simply more malignant and  
so stronger selection works against them. Inactivation of maternal  
alleles could then have arisen for another reason, while relaxed selection  
would have allowed the emergence of relatively benign teratomas of oocytes.  
Perhaps most importantly, there seems to be little difference in imprinting  
between mammalian lineages with invasive placenta (i.e., subject to  
trophoblast invasions and tumors) and non-invasive placenta. Finally,  
the monoallelic imprinting of most somatic tissues must be assumed to  
be a byproduct of selection against ovarian tumors, but there is strong  
evidence that loss of heterozygosity is a major selective force which  
might reduce fitness of the organism with many cells which exhibit such  
monoallelic expression. In sum, though the Ovarian Time Bomb Hypothesis  
is not illogical or vague, its specific and precise predictions are  
met and *exceeded* to such an extent that it seems unable to account  
for the wider scope of genomic imprinting.⁹

The third and most well developed  
hypothesis is the Kinship Theory, also termed “Conflict Theory,”  
which is really an extension of the general paradigm of inclusive fitness  
elucidated by W.D. Hamilton in his famous series of papers in 1964 which  
outlined the general logic by which altruism could evolve through selection  
on the level of genes.¹⁰ The logic is straightforward.  
Consider an individual. This individual is diploid, and so carries  
two copies of each gene, alleles, inherited from their mother and father.  
Similarly, their mother and father also carry two copies of each gene,  
from their own mothers and fathers. The probability of a given allele,  
one of two copies, being passed to the next generation is ½.  
An individual is identical by descent on ½ of their alleles from either  
parent. Similarly, the offspring of a given individual is identical  
by descent on ½ of their alleles. As grandparents of an individual  
are at one remove, one is expected to be identical by descent on about  
¼ of their alleles from any given grandparent. This is an expectation,  
you have no guarantee which grandparent’s allele you will inherit  
and sampling variance will likely result in differences of genetic contribution  
from those in your lineage who one would expect would contribute equitable  
proportions to your genome. In other words, the probability of  
identity by descent on most lines of relationship when comparing given  
loci between two individuals is an expectation only. Nevertheless, evolution  
works over many generations and across populations, so even though siblings  
are only expected to be, not necessarily going to be on the sequence  
level, identical by descent ½ of the time, evolutionarily speaking  
one assumes this as a deterministic truth as the sampling variance tends  
to decrease inversely to the square root of the number sampled.

The probability of two alleles  
on a given locus across generations or between individuals defines the  
“coefficient of relatedness” between them, *r*. Because  
of the haplodiploid nature of hymenoptera genetics Hamilton showed in  
his 1964 paper how eusociality was a highly plausible arrangement for  
this taxon because the coefficient of relatedness between sisters, 0.75,  
was greater than that between mothers and their offspring, 0.5.  
The logic can be illustrated by a “thought experiment.” Consider  
an allele, *A*, which encodes a drive to be altruistic to your  
kin in relation to the coefficient of relatedness, *r*. If  
this allele programs one to come to the aid of someone who is near kin  
then it is “aiding” itself because its presence in said near kin  
would be directly proportional to *r*. In contrast, a selfish  
allele *S*, would encode a strategy of selfishness to one and all.  
Of course some of the times that the allele *A*  
drove one to help someone who was related to you would be aiding *  
S*, but what truly counts is the average over time. If you  
aided the other individuals a great deal, then the cost to yourself  
might be compensated by the benefit to them. Why would a benefit  
to them be relevant? They carry the allele *A*  
as well! Formally this is depicted by “Hamilton’s Rule,” a behavior  
is favored if *B x r \> C*, where *B*  
denotes the “benefit to the other,” *r*  
is the coefficient of relatedness between the other and oneself and *  
C* is the cost to yourself. If one aids someone whose *r*  
to oneself is 0.5 so as to increase that individual’s fitness by greater  
than a factor of 2, then even a reduction of one’s own fitness to  
zero is justified as (2 + *x*) *x*  
0.5 \> 1, 1 being the cost to oneself.

Of course there are many simplifying  
assumptions here. Inbreeding necessarily increases *r*, and  
of course there is extremely high sequence level identity of human genomes  
(for that matter, there is \~98% sequence level identity between chimpanzees  
and human beings) so the key is relative relatedness and a focus on  
recent identity by descent as opposed to identity by state or ancestral  
descent. Nevertheless, the basic logic seems to explain a great  
deal of the behavior in the animal, and even human, world. But how  
is this relevant to genomic imprinting? The key is to go beyond  
one of the implicit simplifying assumptions above, the invariant nature  
of the parents in the models, or, more precisely, the invariant nature  
of the father. In a situation where the father varies a great  
deal, as is common in many mammalian species, the *r*  
across maternally and paternally inherited genes in the offspring, from  
now on denoted *madumnal* and*  
padumnal* alleles, differs. While *madumnal*  
alleles still exhibit the canonical cross-sib *r*  
of 0.5, the *padumnal* alleles can be expected to be less than  
0.5. The reason is two fold, first, in many mammals mating pairs  
last only for one breeding season, so siblings are likely to have different  
fathers. Additionally, in many species which birth litters there is  
multiple-paternity of the siblings because of multiple matings in a  
season. This results in an expectation of *r*  
being lower across *padumnal*  
alleles, and so all things being equal Hamilton’s Rule now differs  
between the maternally and paternally inherited alleles in some cases.¹¹  
An extreme case is between half-siblings who are known to differ in  
paternity. On their *madumnal*  
alleles their *r* is 0.5, on their *padumnal*  
alleles their *r* is 0, averaging together for the canonical coefficient  
of relatedness of 0.25. In species where paternity certainty is  
not known the *padumnal* *r*  
will be proportional to the expectation of paternity identity across  
siblings.¹²

This difference between *  
r* results in a host of conflicts across genetic loci. The  
key for genomic imprinting is that mammals are characterized by relatively  
high obligate maternal investment and vivaparity results in a basal  
metabolic cost to the mother for any given offspring. In contrast  
there is a wide range of paternal investment and certainty. The  
initial work in genomic imprinting derived from the genetic conflicts  
that occurred between the fetus and the mother in mammalian species.¹³  
And to some extent the mother-offspring tug of war for resources is  
still the crux of the matter. In a situation where pair-bonds  
are minimal or transient the cost to the father, and ergo, the *padumnal*  
alleles, of reductions in future fitness of the mother, ergo, the *  
madumnal* alleles, is minimal. Therefore there is a strong  
incentive for *padumnal* alleles to induce aggressive acquisition  
of resources of offspring and transfer of fitness from the mother and  
to offspring. In contrast, the *madumnal*  
alleles are shared between mother and offspring at the value of *r*  
0.5, and between the offspring and future offspring to the value of  
0.5, so there is a strong disincentive for the *madumnal*  
alleles of the offspring from incurring a catastrophic fitness cost  
to the mother. In terms of how this plays out on the molecular  
level the loci which should exhibit genomic imprinting would be those  
focused on early growth of the fetus and resource allocation to the  
offspring. This is the general trend one sees in the mouse model, *  
Igf2* the first gene to be implicated in genomic imprinting in mice  
is a growth factor.¹⁴ Because of the costs of monoallelic  
expression, vulnerability to loss of heterozygosity and haploinsufficiency,  
only loci which effect such highly fitness relevant phenotypes should  
be targeted by genomic imprinting as otherwise the costs to both *  
madumnal* and *padumnal* alleles would be too great. In  
this fashion the narrow scope of imprinting’s genomic impact, as well  
as its functional biases, are accounted for.

One of the predictions the  
Kinship Theory of genomic imprinting, which is validated, is that there  
will be a strong bias toward *padumnal*  
expression in an attempt to trigger and induce growth and resource monopolization  
in its “selfish” drive toward increasing its own fitness irrelevant  
to the cost of the unrelated mother. But critics have pointed  
out that in many circumstances where genomic imprinting should not be  
operative bias still occurs. The reason is that in cases where  
the probability of paternity is very certain, operationally close to  
100%, the *madumnal* and *padumnal* alleles have the same  
interests. Additionally, in such “monandrous” circumstances  
there is often concomitant male parental investment, so excessive offspring  
demands incur a cost upon future *padumnal*  
alleles. It seems than an implication of Kinship theory would  
be that *madumnal* and *padumnal*  
alleles would “agree” upon expression levels and exhibit no asymmetry,  
but this is not so. But this is a fallacious prediction, for what the  
Kinship Theory predicts is that the total sum dosage should now be agreed  
upon between the two parental copies, the means by which the dosage  
occurs (i.e., the proportional contributed via expression from each  
copy) is irrelevant.¹⁵

To flesh out this phenomenon  
one must backtrack and consider the evolutionary trajectory via which  
imprinting occurs. The hypothesis conjectures that over time the *  
padumnal* allele increases dosage while the *madumnal*  
allele decreases dosage in a species where genetic conflict occurs.  
Over time the *madumnal* allele reaches zero expression, that is,  
it is silenced. At this point the *padumnal*  
allele would be assumed to win by default as it would increase dosage  
to the appropriate level to optimize its own fitness (one assumes to  
over-expression might be deleterious for a variety of reasons).  
One must remember that it seems likely that other repressor *madumnal*  
alleles would begin to interfere with the unchecked *padumnal*  
allele (e.g., *Igf2r* in most mammals repressing *Igf2*),  
but nevertheless, the final evolutionary end state should be alternative  
expression states for the maternal and paternal origin copies in the  
offspring. If the life history of the species alters for some  
reason from a competitive polyandrous to a stable monandrous norm, then  
the rational for the asymmetry disappears, and the two parental alleles  
would agree upon dosage. One might imagine that the *padumnal*  
allele would at this point begin to reduce its expression levels toward  
a lower state to increase its own future fitness because it is now tied  
to the *madumnal* allele, but this dose not imply that the *madumnal*  
allele would increase its own expression as the key is that the sum  
total of expression be reduced. In this way one can see that the overall  
implication is that monandrous species derived from a polyandrous state  
exhibit a host of combinations of genomic imprinting as the competitive  
tension between the *padumnal*  
and *madumnal* alleles is released. But, there is no positive  
selection for symmetry so long as the optimal dosage level is maintained.  
The take home message is that the logical implications of Kinship Theory  
maybe more circuitous than one might assume upon first examination,  
and it is important to draw out the full conclusions to test the predictivity  
of the theory.

Not only does Kinship Theory  
offer general predictions, it can also serve as a explanatory framework  
for human pathologies. An important example of this is Prader-Willi  
Syndrome, (PWS) a malfunction of the expression of the *padumnal*  
alleles in the chromosome 15q region.¹⁶ PWS might give  
us some insights into the nature of coevolution between *padumnal*  
and *madumnal* alleles, as it shows us the development of child  
behavior purely under “maternal” genetic influence. Though  
the corpulence of PWS children is well known, PWS children are often  
underweight at birth and do not begin to gain weight until they are  
past the age of 2. Additionally, they are known to be particularly  
quiet and undemanding infants, and their eating habits in childhood  
continue this trend as their “foraging” behavior shows little discernment  
or discrimination as compared to the “pickiness” typical of human  
young. An evolutionary story can easily be generated for PWS in  
that it shows that the optimal strategy being pursued by the *madumnal*  
alleles is one where the offspring consumes minimal maternal resources,  
at its own cost (ergo, low birth weight), does not make excessive attention  
demands (ergo, quiet), and once weaning is complete is relatively single-minded  
and self-sufficient in procuring and consuming a wide variety of food  
stuffs without parental supervision, guidance or attention. One  
could object humans are a monandrous species, but it is important to  
remember that paternity uncertainty is not unknown in our species, with  
a median rate of misattributed paternity of about 3% (with wide inter-populational  
variance).¹⁷ Additionally, the present day behavioral  
norms of our species might not be typical of its evolutionary past.  
One crucial test of the difference in imprinting has been two species  
of mice, *Peromyscus manicultus*  
and *Peromyscus polionotus*. Matings between the polyandrous *  
P. maniculatus* and monandrous *P. polionotus*  
seem to result offspring which exhibit the imprinting biases of the  
former against the latter (*maniculatus*  
females give birth to small offspring by *polionotus* males, while *  
polionotus* females give birth to large offspring by *maniculatus*  
males).¹⁸

The possibilities of variation  
in behavior between closely related species and across evolutionary  
time should be reflected in the molecular coevolution between *padumnal*  
and *madumnal* alleles in an “arms race.” And they do seem  
to leave a molecular trace.¹⁹ One a cytosine of *Igf2r*  
the *madumnal* copy is methylated in the zygote cell-stage 2, then  
umethylated in stage 4, and remethylated in stage 8. The *Igf2-H19*  
gene complex shows evidence of 30 deletions, inserts and transgenes,  
suggesting a long history of evolutionary modification.²⁰  
Additionally, there is the added theoretical possibility of conflict  
between maternal *trans-acting* imprinting alleles and *madumnal*  
imprinted alleles. The genetic reasoning is that though *madumnal*  
imprinted alleles are related to the mother by definition (they are  
inherited from the mother), imprinter alleles within the mother have  
only a ½ chance of being passed with the *madumnal*  
imprinted allele (independent transmission). This sets up a conflict  
where the *trans*-acting maternal allele has a stronger bias toward  
increasing the fitness of the mother than the *madumnal*  
allele. In the language of coefficients of relatedness from the  
maternal imprinter allele is *r*  
½ to the *madumnal* allele, but 1 between the maternal imprinter  
allele and the mother, the latter requires at least twice as much benefit  
to the offspring than to the mother. In contrast, *padumnal*  
alleles and paternal imprinter alleles tend to exhibit less conflict  
because both “agree” upon the importance of maternal investment  
to increase their own fitness at minimal cost, especially in a polyandrous  
species (as is the norm in mammals).²¹ Finally, an  
alternative line of reasoning suggests that because of the variation  
in male reproductive strategies over evolutionary time there will be  
conflict between paternal and *padumnal*  
alleles in situations where males exhibit a high degree of relatedness  
to all offspring, including future ones, because of strong pair-bonds.  
In contrast, the maternal and *madumnal*  
dynamic is considered to be more stable because of the obligate nature  
of female investment. This logic implies that epigenetic imprinting  
from paternal imprinting alleles will be less stable than maternal imprinting  
alleles.²² 

Obviously a lot more empirical  
work needs to be done to match this theoretical paper trail. Though  
the mouse has been fruitful in yielding up imprinted regions of the  
genome, it is important to test the predictions of the Kinship Theory  
in non-mammalian taxa which are Ovoviviparous or engage in intensive  
“parental investment.” Two groups which would be good targets  
are the sharks, which often give “live birth” to small broods, and  
social insects, which engage in intensive rearing of their young.  
Further elucidation of within-taxa variation in imprinting needs to  
be done to confirm the evolutionarily necessary covariance between polyandry  
and asymmetrical expression. The putative genetic networks that  
exist must be fleshed out by a more detailed understanding of the basic  
molecular mechanism of imprinting, e.g., how does an allele “know”  
it is paternal or maternal in origin? Of the 27 well characterized  
regions of the mouse genome which seem subject to parental specific  
imprinting, 8 of them have neurological and behavioral implications.²³  
While in humans there have been studies which suggest a preference by  
females for the smells of males who carry their *padumnal*  
HLA profiles.²⁴ This reinforces the likelihood that  
parental specific imprinting might not be common, but it is highly targeted  
to loci of strong fitness import, with a neurobehavioral bias.  
Some workers hypothesize that many human mental pathologies, such as  
schizophrenia, are likely to show genomic imprinting effects.²⁵  
The possibility of medical research exploring this option is important  
when human lives are at stake. Though the molecular genetic trenches  
have to be dug at some point, the theoretical-evolutionary can at least  
work as scouts pointing in the general direction of new avenues of discovery  
and constraining the possibilities.

Haig et. al., American Naturalist, Vol. 134, No. 1 (Jul., 1989) , pp. 147-155  
DeCharia et. al., Cell, 64:849-859  
A Burt, R Trivers, Genes in Conflict: The Biology of Selfish Genetic Elements, 2005  
Deleval et al., Current Opinion in Genetics & Development, 2004, 14:188-195  
W Provine, The Origins of Theoretical Population Genetics, 1971  
Beaudet et. al., The American Journal of Human Genetics, 70, 1389-1397 (2002)  
Wilkins et. al., Nature Reviews Genetics, 2003 May;4(5):359-68  
Vamuza et. al., Trends in Genetics, 1994, 10: 118-123  
Wilkins et. al., Nature Reviews Genetics, 2003 May;4(5):359-68  
Hamilton, W.D., Journal of Theoretical Biology, (1964). 7, 1-16, 17-32.  
Wilkins et. al., Nature Reviews Genetics, 2003 May;4(5):359-68  
Haig D, Genetics, 2003, 117:103-110  
Haig D., Quarterly Review of Biology, 68: 495-532  
A Burt, R Trivers, Genes in Conflict: The Biology of Selfish Genetic Elements, 2005  
Haig D, Genetics, 2003, 117:103-110  
Haig D, American Journal of Human Biology, 2003, 15: 320-329  
Anderson K, Current Anthropology, 2006, 48(3)  
Wilkins et. al., Nature Reviews Genetics, 2003 May;4(5):359-68  
Moore et. al., Reproduction, 1996, 1, 73-77  
A Burt, R Trivers, Genes in Conflict: The Biology of Selfish Genetic Elements, 2005  
Burt et. al., Proceedings of the Royal Society, 1998, 265:2393-2397  
Wilkins et. al., Nature Reviews Genetics, 2003 May;4(5):359-68  
A Burt, R Trivers, Genes in Conflict: The Biology of Selfish Genetic Elements, 2005  
Jacob S et. al., Nature Genetics, 30, 175 – 179 (2002)  
A Burt, R Trivers, Genes in Conflict: The Biology of Selfish Genetic Elements, 2005

### Related Posts:

- [Kinship theory and genomic
  imprinting](https://www.gnxp.com/WordPress/2006/05/22/kinship-theory-and-genomic-imprinting/) - [New theory of genomic
  imprinting](https://www.gnxp.com/WordPress/2006/11/19/new-theory-of-genomic-imprinting/) - [Social life & genomic
  imprinting](https://www.gnxp.com/WordPress/2006/05/23/social-life-genomic-imprinting/) - [Imprinted gene undergoing accelerated evolution in
  humans?](https://www.gnxp.com/WordPress/2007/03/14/imprinted-gene-undergoing-accelerated-evolution-in-humans/) - [Imprinted gene undergoing accelerated evolution in
  humans?](https://www.gnxp.com/WordPress/2007/03/13/imprinted-gene-undergoing-accelerated-evolution-in-humans/) - [Genetic conflict in
  fish](https://www.gnxp.com/WordPress/2007/07/28/genetic-conflict-in-fish/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F05%2Fthe-evolutionary-context-of-genomic-imprinting%2F&linkname=The%20Evolutionary%20Context%20of%20Genomic%20Imprinting "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F05%2Fthe-evolutionary-context-of-genomic-imprinting%2F&linkname=The%20Evolutionary%20Context%20of%20Genomic%20Imprinting "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F05%2Fthe-evolutionary-context-of-genomic-imprinting%2F&linkname=The%20Evolutionary%20Context%20of%20Genomic%20Imprinting "Email")[](https://www.addtoany.com/share)
