+++
title = "IRES rising, raising"
full_title = "IRES rising, raising ires"
date = "2007-02-06"
upstream_url = "https://www.gnxp.com/WordPress/2007/02/06/ires-rising-raising-ires/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/02/06/ires-rising-raising-ires/).

IRES rising, raising ires

I first became interested in internal ribosomal entry sites (IRESs) upon reading [this paper by Pinkstaff and colleagues](http://www.pnas.org/cgi/content/full/98/5/2770). They used a reporter system, which I will describe shortly, to suggest that five dendritically localized mRNAs could be translated via internal ribosomal entry. This provided a novel twist on the role of protein synthesis in synaptic plasticity. Presumably, those mRNAs are trafficked to dendrites so they can be translated rapidly and locally when the time comes. An IRES allows the protein synthesis apparatus, the ribosome, to connect with an mRNA and start translating without the normal rites of initiation. Messages containing these elements could keep translating during times when global protein synthesis has been downregulated and initiation factors are scarce. At least two of the IRES-containing mRNAs reported by Pinkstaff are known to be rapidly translated upon induction of synaptic plasticity. An intriguing scenario thus emerges in which global protein synthesis is actually inhibited at the initiation step during synaptic plasticity. This leaves excess ribosomes and only a few special mRNAs with the ability to use them, so specific plasticity proteins are upregulated while most proteins are turned down. One could imagine all sorts of scenarios to flesh this story out. The conditions for synaptic plasticity induction include a large calcium influx, activation of several enzymes, and a relatively large amount of energy to put things (like calcium) back where they came from. Maybe this is interpreted as a type of cellular stress necessitating a metabolic shutdown. This would be a way of locally coordinating expression of the necessary subset of synaptic modification proteins.

To be clear, [translation](http://www.bbc.co.uk/education/asguru/biology/04genesgenetics/03proteinsynthesis/03translation/index.shtml) is the final step of the central dogma of genetics. This is the step where the triplet genetic code of nucleotides is read off into the new language of amino acids. A tRNA carrying an amino acid matches up to three nucleotides of the mRNA chain, and the ribosome attaches this amino acid to the growing chain. The dynamic mechanism by which all these factors and RNAs and ribosomes actually get to the point of starting a new protein is called ribosomal scanning. Canonical translation initiation involves a series of recruitment steps bringing together initiation factors, the initiator tRNA carrying an amino acid, the mRNA to be translated, and the ribosomal subunits. The initiation factors bind the mRNA at the very end and assemble with a ribosome carrying the initiator tRNA. Then the whole complex slides down the mRNA looking for a start codon (AUG). The AUG sequence is sticky for an initiator tRNA because it matches the tRNA’s anticodon. The machine tarries there, allowing time to bring in a second tRNA and start the chain. Imagine everyone’s surprise and delight when it was reported that, in some viruses, translation can start without any of this process. None of the initiation factors, no initiator tRNA; the ribosome just jumps on to the RNA in the middle and starts synthesizing proteins. The idea of an IRES (internal ribosomal entry site) is that the mRNA carries a complex three dimensional structure that mimics many of these factors and provides a platform for translation to start. In fact, [recent crystal structures](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6VSR-4MV1HD3-3&_user=10&_coverDate=01%2F31%2F2007&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=539fab67e955ebec0fb0ad0d64a99c89) have provided insight into the mechanisms of action for two viral IRESs. I’m not gonna play around and pretend that I understand structural papers (or even [summaries](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6VSR-4MV1HD3-3&_user=10&_coverDate=01%2F31%2F2007&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=539fab67e955ebec0fb0ad0d64a99c89) of them), but it seems to me that, unless you can just force RNA into whatever conformation you want by crystallizing it with the right co-factors, these structural papers provide a new level of evidence for the concept of internal ribosomal entry. The RNA folds up in a way that appealingly explains how ribosomes could join and synthesize without initiation factors.

[](https://www.gnxp.com/blog/uploaded_images/capvires-700288.jpg)

Why am I so nervous about whether IRESs exist? While I was searching for information on RNA structure and IRES activity so I could explain noncanonical RNA interactions and structure to you, I came across the reviews of Marilyn Kozak ([2001](http://mcb.asm.org/cgi/content/full/21/6/1899?view=long&pmid=11238926), [2001 exchange](http://mcb.asm.org/cgi/content/full/21/23/8238?view=long&pmid=117103332001), [2005](http://nar.oxfordjournals.org/cgi/content/full/33/20/6593), and [2006](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6T39-4K7N6HR-1&_user=10&_coverDate=11%2F01%2F2006&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=cc1f9285849a7c0fd00ddec48628b1c1)). Kozak has made major contributions to the understanding of translation initiation. To give an idea, the consensus sequence that determines whether an AUG start codon is really going to be the site of translation initiation is called a [Kozak sequence](https://en.wikipedia.org/wiki/Kozak_consensus_sequence). She has been arguing for a rigorous re-examination of the evidence for IRES-dependent translation for the last 15 years or so (or perhaps longer and I haven’t gotten to that review yet). Her critical 2001 review of the evidence for IRESs in cellular mRNAs drew accusations of unscholarliness, false statements, and misrepresentation of others’ data in a strident letter to the editor with 87 signatories including names I recognize as eminent in translation control studies. Nahum Sonenberg, for instance, is an editor of the recently released [Translational Control in Biology and Medicine](http://www.cshlpress.com/default.tpl?action=full&cart=116922297359831&--eqskudatarq=565) monograph from CSHL Press. I’m a sucker for a clever contrarian, so of course I think this section from her rebuttal letter is worth quoting:

> “I will save the curious reader the trouble of counting the names > appended to the Letter to the Editor: there are 87 votes in favor of > cellular IRES elements and associated phenomena. Some of the signers > (e.g., Drs. Farabaugh, Filipowicz, Goldman, and Krug) work on subjects > completely unrelated to the content of the minireview; they must have > studied hard to qualify as judges. The letter was composed and circulated by Drs. Schneider and Sonenberg. Many of the signers have close links to Dr. Sonenberg, either as coauthors or members of the same institution. >
> It is obvious that the organizers worked hard to collect all those > signatures, but to what end? A single voice suffices to present a > logical argument. I might be alone in refusing to believe a story with so many flaws, but that does not mean I am wrong. Counting the votes determines the answer in politics (Florida excepted) but not in science.”

The jury will disregard the Florida comment. We shall now proceed to her substantive arguments.

The model system for study of IRES-dependent translation has been the dicistronic vector, a DNA construct that contains two protein-coding genes (open reading frames) separated by the putative IRES. The first gene in the sequence should be translated by the normal scanning mechanism. The amount of the second protein product indicates the degree of IRES-dependent translation. The vector of choice for a large number of studies cont  
ained *Renilla* [luciferase](https://en.wikipedia.org/wiki/Luciferase) as gene 1 and Firefly luciferase as gene 2 (the pRF vector). These can be detected independently, so you can compare IRES and normal translation. The arrangement of this vector is similar to the dicistroviruses in which IRESs were first reported. These viruses mostly carry structural, coat proteins in the first open reading frame and proteins important for replication and integration in the second one. After transcription and translation the protein products are further cleaved to produce the mature pieces needed to build a new virus. Here is one of the first reasons to be suspicious of the proposed IRES mechanism. There are a number of examples of viruses that look dicistronic, but they achieve the two protein products through RNA splicing or protein cleavage. Kozak cites a string of virus families whose downstream genes are silent until they are moved to the front of the line by removal of the upstream open reading frames.

A second issue is that there is little to no conservation between proposed IRES sequences. This is not necessarily a problem, given that IRESs are supposed to work at the level of 3D structure rather than primary sequence recognition, but it is unsettling. There are some ways of inferring an important RNA structure. For instance, nucleotides that pair in stem-loop structures may not be conserved, but their ability to pair is (i.e. a G and a C are mutated to a C and a G). The recent structural studies may provide some more criteria for how an IRES should look, but these are for the simplest proposed IRES mechanism. Other reported IRESs bear little primary sequence-level resemblance.

One of Kozak’s major themes is to criticize the efficiency of expression from IRES reading frames. This is her real feat. She goes over dozens of papers claiming to demonstrate IRES-dependent translation with a fine-toothed comb and comes up with no credible example of a strong IRES. It really matters what you compare efficiency with. Insertion of an IRES may represent an impressive-sounding increase compared to a vector containing no IRES, but still look dismal compared to regular cellular translation. Viruses need to make proteins. They can’t really just screw around with some new mechanism and make a little of this and that. Any translation mechanism proposed to be important for viral life cycles or, for that matter, cellular protein synthesis better be efficient enough to support the observed efficiency in a natural setting. Instead, even the strongest putative IRES doesn’t come close to the efficiency of regular translation. It does bring synthesis up higher than if there were no IRES at all, but Kozak has answers for that as well. This is a part of her position that is vulnerable to attack though. At what point would an IRES be efficient enough to please her? She repeatedly uses phrases such as “IRES X only increased efficiency X amount.” Why the ‘only’? She is emphasizing the smallness of the number, but we don’t really know what a meaningful level of translation should be. Is it necessary for the IRES to reach efficiency as good or better than normal cellular translation? Do viruses really need that much replication machinery? She does make a strong point regarding negative controls though. The pRF vector with no IRES at all produces variable results:

> “There is always some translation of the 3′ cistron, and this > unexplained background expression is not constant. Background > translation of the 3′ firefly luciferase (Fluc) cistron varied > \>10-fold when three different negative controls were tested by RNA > transfection \[Figure 5C in Ref. (9)\]. When tested in vitro, > translation of the 3′ cistron increased 10-fold upon simply > lengthening the intercistronic domain \[Figure 9 in Ref. (64)\]. Whatever the reason for this background, its variability would seem to set a lower limit on what level of expression constitutes a credible positive result when a candidate IRES is tested.”

Not only that, but people have occasionally used negative controls in which structured RNA sequences that actually decrease efficiency are inserted between the *Renilla\>* and Firefly genes.

What if we do find a sequence that can be placed between two open reading frames and lead to a major increase in accumulation of the product form the second frame? We have a pretty good IRES candidate now, right? Nope. You need to do RNA analysis. Here’s the problem. You don’t really know what you are putting in between those two DNA sequences. There are two creatures you might be inserting instead of a real IRES. First, there is the possibility of a cryptic promoter: the increased product is due to independent transcription of the downstream reading frame, so ribosomal entry really isn’t internal at all. You just have two mRNAs; nothing mysterious. This mechanism has been detected for the reported IRESs in p27-Kip1, VEGF, and c-sis/PDGF2. I’m getting this information directly from [this table](http://nar.oxfordjournals.org/cgi/content-nw/full/33/20/6593/TBL1) in Kozak’s 2005 review ([free](http://nar.oxfordjournals.org/cgi/content/full/33/20/6593)). Also, the first open reading frame in the fully transcribed dicistronic mRNA could simply be removed during RNA splicing (the step in mRNA maturation in which introns are cut out from between the protein-coding exons). This would leave a fully intact second open reading frame mRNA that needs no IRES to be translated. This mechanism has also been detected in three reported IRESs. The cryptic promoter idea can be circumvented by injecting RNA directly into the cells (or in vitro translation system) rather than a DNA construct. The splice site problem is less easily removed. You have to use the most sensitive possible assay and look for the spliced RNA. The problem is that the only citation given for a sensitive-enough assay reports spurious splicing caused by the candidate IRES. What constitutes proper RNA analysis is left uncertain.

Now let’s look back at the exciting findings of Pinkstaff et al. 2001. They found IRES-dependent translation associated with the mRNAs of Arc, CaMKII, neurogranin, MAP2 and dendrin. Did they run these experiments well enough for Kozak’s standards? First, they used the pRF construct. We know that baseline firefly luciferase expression in this dicistronic system is variable. Unfortunately, we can’t tell how variable it is in these experiments because all results are expressed as a ratio of *Renilla* to firefly levels normalized to the negative control baseline vector. Kozak defines as ‘weak’ any increase of less than 5-fold compared to the control vector. CaMKII and dendrin never cross this threshold, so it would be crucial to make sure these aren’t varying at the level explained by baseline fluctuations. The initial RNA analyses for neurogranin turned up two small RNAs, indicating that a splice site had been inserted. After removing this issue, neurogranin still had some of the highest IRES activity. In a further test, when a construct containing the neurogranin sequence controlling both reading frames (this time yellow and cyan fluorescent proteins) was injected into neurons, IRES activity could be up to 15% of the level of normal translation. It’s not super strong, but the story goes that this type of translation should be reserved for special times. It remains to be explained why the IRES activity is 15-fold better than control in a rat neuronal tumor cell line but only 3-4 fold better when injected into hippocampal neurons. I’d have a hard time strongly rejecting the idea of IRES-dependent translation for these mRNAs. Yes, CaMKII and dendrin are weak, but the error bars are shown on a beta-globin control and they are well out of that range. The difficult part is the RNA analysis. It was sensitive enough to catch one inserted splice site. Does that mean it is sensitive enough to catch any and all? It is just a Northern blot. On one hand, this goalpost is one of the most easily moved. On the other  
, it would be nice to see some a more exhaustive search. Also, the efficiency of the second gene can be enhanced in some cases by simply increasing the space between open reading frames, so an appropriate control vector needs to equate this parameter between control and experimental constructs.

The debate rages on although I’m not sure everyone is listening. The chapter on cellular IRES activity in Translational Control in Biology and Medicine by Elroy-Stein and Merrick cites Kozak (2005) but also cites several cellular IRESs that Kozak doesn’t accept (including XIAP, which has been shown to undergo splicing!) and argues that these protein products are “biologically effective at rather small concentrations” in an attempt to get around her ‘ineffeciency’ arguments. Even if cellular IRES activity turns out to be true, researchers need to apply more rigor and transparently report their data in the initial characterization of these mRNA elements. IRESs are certainly intriguing if they work, but it is currently too early to say. Here’s one more dead-on Kozak zinger for the road – “Is the “scholarly” review the one that repeats the premature claim or the one that explains why it is premature?”

### Related Posts:

- [Global inhibition, specific
  activation](https://www.gnxp.com/WordPress/2006/10/10/global-inhibition-specific-activation/) - [A mechanism for miRNA-mediated
  repression](https://www.gnxp.com/WordPress/2007/06/19/a-mechanism-for-mirna-mediated-repression/) - [Erin Schuman: PubMed the name and syndicate
  it](https://www.gnxp.com/WordPress/2006/07/08/erin-schuman-pubmed-the-name-and-syndicate-it/) - [Nice 'N Slow: how to make more
  GCN4](https://www.gnxp.com/WordPress/2007/05/08/nice-n-slow-how-to-make-more-gcn4/) - [Dendritic translation and
  memory](https://www.gnxp.com/WordPress/2006/10/09/dendritic-translation-and-memory/) - [Just Science Wrap
  Up](https://www.gnxp.com/WordPress/2007/02/12/just-science-wrap-up/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F06%2Fires-rising-raising-ires%2F&linkname=IRES%20rising%2C%20raising%20ires "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F06%2Fires-rising-raising-ires%2F&linkname=IRES%20rising%2C%20raising%20ires "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F06%2Fires-rising-raising-ires%2F&linkname=IRES%20rising%2C%20raising%20ires "Email")[](https://www.addtoany.com/share)
