+++
title = "Revenge of the drift"
full_title = "Revenge of the drift"
date = "2016-03-15"
upstream_url = "https://www.gnxp.com/WordPress/2016/03/15/revenge-of-the-drift/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/03/15/revenge-of-the-drift/).

Revenge of the drift

[![coop](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/coop.png?resize=608%2C659)![coop](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/coop.png?resize=608%2C659)](http://biorxiv.org/content/early/2016/03/07/042598)

As longtime readers know the role of selection and drift in shaping evolutionary processes have long been at issue within the field. Even as early as Charles Darwin’s time there were some, including his famous bulldog Thomas H. Huxely, who were skeptical that natural selection was a primary engine of evolutionary change (Darwin had convince him about the reality of common descent with modification though). The the decades around 1900 saw what Peter J. Bowler has described as the “eclipse of Darwinism.” All intellectuals understood and accepted evolution, but many were skeptical of the framework and arguments of Darwin. This eclipse receded with the integration of genetics into evolutionary theory, which gave rise to population genetics, and birthed the Neo-Darwinian Synthesis by \~1950.

[![51tQuYMf7gL.\_SY344_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/51tQuYMf7gL._SY344_BO1204203200_-186x300.jpg?resize=186%2C300)![51tQuYMf7gL.\_SY344_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/51tQuYMf7gL._SY344_BO1204203200_-186x300.jpg?resize=186%2C300)](https://www.amazon.com/exec/obidos/ASIN/080184391X/geneexpressio-20)One might argue that this marked the high-tide of adaptationism and acceptance for the role of selection in shaping all the picayune details of biological phenomena. But even then there were those who were more cautious (there are arguments over whether Sewall Wright, one of the fathers of population genetics, did or did not argue for a strong role for stochasticity in his metaphor for evolutionary process, the shifting balance across the adaptive landscape).

In the 1960s the dialogue between empirical results which reported high degrees of realized polymorphism in the field of molecular evolution and the formal models promoted by thinkers such as Motoo Kimura which eventually came under the heading “neutral theory” induceda revolution in our thinking about evolution. Though many might argue for the primacy of selection constraining and shaping morphological variation (or phenotypic traits on a coarser scale more generally), a null hypothesis on the molecular scale was that most variation was the outcome of neutral process. That is, even if most new mutations were deleterious (this may not even be the case in most of the genome of some organisms), the ones which attained high frequency to generate polymorphism did so usually by chance, not because they were favored.

This debate was surprisingly vociferous for several decades. When I first encountered it in the mid-1990s it had cooled off, but there did not seem to be a final resolution (though my impression is that among evolutionary geneticists a form of neutralism seemed to be widely accepted as a default model in any sort of hypothesis testing).

[![41qS+5MyBmL.\_SX331_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/41qS+5MyBmL._SX331_BO1204203200_-200x300.jpg?resize=200%2C300)![41qS+5MyBmL.\_SX331_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/41qS+5MyBmL._SX331_BO1204203200_-200x300.jpg?resize=200%2C300)](https://www.amazon.com/exec/obidos/ASIN/0226684644/geneexpressio-20)In general I am not a believer that genomics has “changed everything” when it comes to evolutionary biology. Rather, evolutionary genomics literally stands on the shoulders of giants (well, mostly dead white men). But **I do think genomics offers up the possibility to obtain greater empirical clarity on the relative role of neutral stochastic forces and selection in shaping variation on the molecular level.** A full genome sequence (or enough to gain an appreciable sense of patterns in the genome) is invaluable information. It is in relation to the patterns of DNA arguably all the information (at some point in the future long reads will capture structural variants and epigenomics will also advance).

Last spring I wrote about a fascinating new paper, [Natural Selection Constrains Neutral Diversity across A Wide Range of Species](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002112), in the post [Selectionism Strikes Back!](http://www.unz.com/gnxp/selectionism-strikes-back/) The title says it all. Using a wide range of genomes the authors argue that two forms of natural selection, the background selection which constrains the emergence of deleterious alleles at high frequency, and selection of positive alleles which allow for linked regions of the genome to sweep at high frequency and eliminate variation (and so generate haplotype blocks), can help resolve what has been termed “Lewontin’s paradox”. The paradox is simple: neutral theory predicts that population size will dictate the amount of variation one sees within a species. Large populations have many mutations traversing the frequency range from \~0 to 1, while small populations will have far less diversity because of the power of drift in fixing mutants rather quickly. The manner in which some have resolved this paradox is that large populations are subject to powerful selection dynamics which constrain the neutral variation; in particular, positive sweeps (producing “genetic draft”) and negative constraints homogenize large regions of the genome. Since the above paper is open access **I recommend you read it.**They found that selection did seem to impact species with different population sizes in the direction in which the selectionist resolution would imply (those species which have large population sizes should have more polymorphism, but selection constraints variation much more).

But Graham Coop has posted a note on bioRxiv, [Does linked selection explain the narrow range of genetic diversity across species?](http://biorxiv.org/content/early/2016/03/07/042598), which suggests that though the **qualitative results match the selectionist narrative, the magnitude of the effect is simply not what one might expect if selection was dominant over stochastic forces driven by variation in demographics.** That is, just because *Drosophila* has a huge census size today does not mean that it had a huge census size over the course of its history, and genetic diversity is strongly sensitive to the smallest population size over a temporal window (this is when most of the diversity can get expunged by drift forces). The figure above shows Coop’s reanalysis of the results in the above paper using their model. He suggests that quantitatively the magnitude of the effect of linked selection seems far more modest. From the preprint:

> To understand the contributions of the two explanations to levels of > diversity, it is helpful to distinguish between the average observed > level of neutral polymorphism in the genome (π) and that expected in > the absence of linked selection (π0 ). Our idealized neutral level of > variation in which π0 ≈ 4Ne μ, reflects the effective size of the > population (Ne ) in the absence of linked selection ( here Ne is not > estimated simply from putatively neutral diversity levels > genome-wide). To illustrate this point, take the extreme scenario in > which linked selection explains nearly all of the deficit in variation > in species with large census sizes, with fluctuations in population > size playing a minor role. In these species, π should be orders of > magnitude smaller than π0 , and Ne should be roughly the same order of > magnitude as the census size. In contrast, if fluctuations in > population size explain most of the deficit, then π should be close to > π0 for all species, while Ne would be many orders of magnitude lower > than census population sizes for species with large population sizes.

Coop does observe that patterns of variation **within** the genome may be strongly shaped by linked selection, and, that a thorough understanding of linked selection is essential to generating a proper model which captures natural dynamics. But at the end of the day he seems to reject the thesis that it’s “all selection, all the time.” His argument is broadly persuasive to me, but I think the authors above have work that will follow up the original paper.

**Both the original paper and the preprint that responds to it should be read closely.** I do not believe this will be a decades long debate. Yes, there are many badly assembled draft genomes out there, but in the next ten years we’ll have the data to actually test robustly these competing theses as to the power of different evolutionary forces in shaping variation. At least on the scale of the sequence….

### Related Posts:

- [Darwin, still
  relevant](https://www.gnxp.com/WordPress/2009/02/10/darwin-still-relevant/) - [My Darwin, right or
  wrong?](https://www.gnxp.com/WordPress/2009/06/07/my-darwin-right-or-wrong/) - [The theory of
  evolution](https://www.gnxp.com/WordPress/2008/05/09/the-theory-of-evolution/) - [The Structure of Evolutionary Theory blogging, chapter
  2](https://www.gnxp.com/WordPress/2008/01/29/the-structure-of-evolutionary-theory-blogging-chapter-2/) - [Post-Darwinian
  evolution?](https://www.gnxp.com/WordPress/2009/02/10/post-darwinian-evolution/) - [Jerry Fodor, Charles Darwin and Natural
  Selection](https://www.gnxp.com/WordPress/2009/07/22/jerry-fodor-charles-darwin-and-natural-selection/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F03%2F15%2Frevenge-of-the-drift%2F&linkname=Revenge%20of%20the%20drift "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F03%2F15%2Frevenge-of-the-drift%2F&linkname=Revenge%20of%20the%20drift "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F03%2F15%2Frevenge-of-the-drift%2F&linkname=Revenge%20of%20the%20drift "Email")[](https://www.addtoany.com/share)
