+++
title = "Genetic distance between"
full_title = "Genetic distance between populations"
date = "2009-05-05"
upstream_url = "https://www.gnxp.com/WordPress/2009/05/05/genetic-distance-between-populations/"

+++
Source: [here](https://www.gnxp.com/WordPress/2009/05/05/genetic-distance-between-populations/).

Genetic distance between populations

In the comments below on the post on human population structure there was some request for a bigger global perspective. Below the fold I’ve placed a table with [F_(ST)](https://en.wikipedia.org/wiki/Fixation_index) values which compare each population to the other. This an older population genetic statistic derived from the work of [Sewall Wright](https://en.wikipedia.org/wiki/Sewall_Wright), but you are almost certainly familiar with the talking point that “85% of variation is within races, and only 15% between.” That is an F_(ST) insight. The higher the F_(ST) the greater the proportion of genetic variation which can be attributed to between population differences, so it serves as a rough measure of genetic distance. But it needs to be used carefully; population genetics has many numbers, but that shouldn’t lead to accept the illusion that it is like measuring the velocity of a falling ball. Please see [Measuring Genetic Diversity: Lewontin’s Other Fallacy](https://www.gnxp.com/MT2/archives/003951.html), [Measuring Genetic Diversity: Part 2](https://www.gnxp.com/MT2/archives/003955.html), and [Human genetic diversity: Lewontin’s fallacy](http://www.ncbi.nlm.nih.gov/pubmed/12879450), for details on why statistics like F_(ST) should be interpreted cautiously.  
I’ve removed some of the more obscure populations because of limited width on the weblog. See [here](http://genome.cshlp.org/content/suppl/2009/05/01/19.5.815.DC1/Supp_Table_1.xls) for the original table.

------------------------------------------------------------------------

|            |          |       |      |       |      |      |      |      |      |          |       |
|-----------:|---------:|------:|-----:|------:|-----:|-----:|-----:|-----:|-----:|---------:|------:|
| A. Brahmin |       \* |    \* |   \* |    \* |   \* |   \* |   \* |   \* |   \* |       \* |    \* |
|    Chinese |     .071 |    \* |   \* |    \* |   \* |   \* |   \* |   \* |   \* |       \* |    \* |
|       Iban |     .075 |  .020 |   \* |    \* |   \* |   \* |   \* |   \* |   \* |       \* |    \* |
|      Irula |     .030 |  .089 | .092 |    \* |   \* |   \* |   \* |   \* |   \* |       \* |    \* |
|   Japanese |     .072 |  .009 | .031 |  .089 |   \* |   \* |   \* |   \* |   \* |       \* |    \* |
|      Khmer |     .055 |  .009 | .013 |  .075 | .018 |   \* |   \* |   \* |   \* |       \* |    \* |
|     Madiga |     .005 |  .069 | .073 |  .027 | .070 | .052 |   \* |   \* |   \* |       \* |    \* |
|       Mala |     .005 |  .068 | .072 |  .026 | .069 | .051 | .001 |   \* |   \* |       \* |    \* |
|      Nguni |     .146 |  .191 | .197 |  .172 | .190 | .174 | .150 | .150 |   \* |       \* |    \* |
| T. Brahmin |     .001 |  .074 | .080 |  .032 | .075 | .058 | .006 | .006 | .146 |       \* |    \* |
|     Tuscan |     .032 |  .109 | .113 |  .072 | .109 | .094 | .048 | .051 | .159 |     .031 |    \* |
| Vietnamese |     .066 |  .002 | .014 |  .086 | .015 | .003 | .064 | .064 | .186 |     .070 |  .105 |
|           | A. Brah. | Chin. | Iban | Irula | Jap. | Khm. | Mad. | Mala |  Ng. | T. Brah. | Tusc. |

------------------------------------------------------------------------

Notice that the distance between the Chinese and Vietnamese is smaller than between the Chinese and Japanese (that is, more of the genetic variance occurs between the populations in the latter case than the former). I’m pretty sure this is strongly conditional on the weighting of which regions of China the genetic samples come from. In other words, a sample from Manchuria would probably be closer to Japan than Vietnam, while one from Fujian may be closer to Vietnam than Japan. In other words, F_(ST) values can sometimes tell you more about the representativeness of a sample for any given class than a novel insight. Also, look at the distance between Tamil Brahmins and the tribal Irula and the Dalit populations. All these samples are from Tamil Nadu & Andhara Pradesh in the Southeast of the Indian subcontinent. Tamil Brahmins exhibit as much difference from the Irula as from the Tuscans of Italy. What’s going on? One the one hand an admixture model might explain it, but, I want to caution that I think the genetic closeness of Dalits to the Brahmins might indicate rather that the Irula tribal might have some peculiar demographic history which has increased its own distinctiveness, similar to what happened to the Icelanders. After all the outlier status of Sardinians in relation to other Europeans tells you something (the power of an island to prevent gene flow and allow for a random walk of variation), but chances are it is hard to pick a Sardinian out from a crowd of Italians unless you’re a local.  
On the other hand the relatively small proportion of between group variation between the two South Indian Brahmin groups and the two Dalit groups (they’re almost indistinguishable) is suggestive that there is something very wrong with the model that Hindu caste identity is a social construction of European colonialists (I think this hypothesis can be falsified without recourse to genetics, but biological science is a nice supplement to those who refuse to bow down to other facts). The main caveat here is that the selection of Brahmins and Dalits excludes 80-90% Hindu caste groups, though these two segments are important and informative in that they are two which are extant all across India (i.e., every region of India has a group of Untouchables and Brahmins). It may be more plausible that the ethnographic record of caste mobility is more a feature of the 80-90% who are neither Untouchables nor Brahmins, and whose caste identity and position are more fluid. For example, the [Marathas](https://en.wikipedia.org/wiki/Maratha) and [Kayasthas](https://en.wikipedia.org/wiki/Kayastha) both crystallized their current form during the Muslim period, the former as antagonists to the Muslim warrior class, and the latter as literate adjuncts.

### Related Posts:

- [Population Pairwise Fst on 250,000
  SNPs](https://www.gnxp.com/WordPress/2021/10/28/pairwise-fst-on-250000-snps/) - [1 migrant needed to prevent genetic
  divergence](https://www.gnxp.com/WordPress/2012/01/26/1-migrant-needed-to-prevent-genetic-divergence/) - [Evidence for natural selection between
  populations](https://www.gnxp.com/WordPress/2005/04/12/evidence-for-natural-selection-between-populations/) - [Desmond Tutu, Spaniards, and genetic
  distance](https://www.gnxp.com/WordPress/2010/08/21/desmond-tutu-spaniards-and-genetic-distance/) - [Ashkenazi Jews are more European in
  ancestry](https://www.gnxp.com/WordPress/2009/09/04/ashkenazi-jews-are-more-european-in-ancestry/) - [Which population is most genetically distant from
  Africans?…](https://www.gnxp.com/WordPress/2010/08/19/which-population-is-most-genetically-distant-from-africans-amerindians/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F05%2F05%2Fgenetic-distance-between-populations%2F&linkname=Genetic%20distance%20between%20populations "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F05%2F05%2Fgenetic-distance-between-populations%2F&linkname=Genetic%20distance%20between%20populations "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F05%2F05%2Fgenetic-distance-between-populations%2F&linkname=Genetic%20distance%20between%20populations "Email")[](https://www.addtoany.com/share)
