+++
title = "Selective sweeps on the"
full_title = "Selective sweeps on the X chromosome of homonoids"
date = "2015-08-20"
upstream_url = "https://www.gnxp.com/WordPress/2015/08/20/selective-sweeps-on-the-x-chromosome-of-homonoids/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/08/20/selective-sweeps-on-the-x-chromosome-of-homonoids/).

Selective sweeps on the X chromosome of homonoids

[![Screenshot from 2015-08-19 23:27:08](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/08/Screenshot-from-2015-08-19-232708.png?resize=238%2C276)![Screenshot from 2015-08-19 23:27:08](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/08/Screenshot-from-2015-08-19-232708.png?resize=238%2C276)](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005451)About ten years ago David Reich and Nick Patterson [were involved in a paper](http://www.nature.com/nature/journal/v441/n7097/abs/nature04789.html) which posited “complex speciation” in the lineage that led to humans and chimpanzees. What that means is that there was some hybridization between the proto-chimp/bonobo lineage, and that leading to hominins. As the authors state: “These unexpected features \[of the genome\] would be explained if the human and chimpanzee lineages initially diverged, then later exchanged genes before separating permanently.” The primary result happens to be a disjunction between the patterns you see in the broader genome, the autosome, and the X chromosome. The divergence from the X chromosome is far less than it should be if you would set your expectation from the autosome, suggesting that it harbors signatures of recent gene flow across the two lineages.

A new paper in *PLOS GENETICS*, [Strong Selective Sweeps on the X Chromosome in the Human-Chimpanzee Ancestor Explain Its Low Divergence](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005451), offers up a different set of possibilities. One of the authors, [Thomas Mailund](http://www.mailund.dk/index.php/2015/08/19/strong-selective-sweeps-on-the-x-chromosome-of-the-human-chimpanzee-ancestor/), has a [write-up](//www.mailund.dk/index.php/2015/08/19/strong-selective-sweeps-on-the-x-chromosome-of-the-human-chimpanzee-ancestor/%22) of where they were going with this paper and how they got there. Definitely read what he has to say.

The crux of the issue seems to be that **the diversity on the X chromosome varies in a peculiar manner.** In particular, incomplete lineage sorting, basically the overlap of variation across two species due to common ancestral alleles, seems to exhibit a bimodal distribution on the X chromosome (the bottom panel above). Going beyond just a chromosome-wide summary oraverage, the authors found that there were huge deserts where variation was gone, in contrast with broad swaths of the X chromosome genome where the variation is totally in light with roughly neutral assumptions (i.e., the effective population of the X chromosome is \~3/4 of the autosome, so that increases the power of drift, etc.).

Why this pattern? One explanation could be background selection. This is basically the removal of deleterious alleles as they arise, often resulting in reduced variation across a genomic region because of linkage. The X chromosome has a peculiar dynamic because in males normally recessive alleles, whether favored or disfavored, are subject to the full force of selection (since most recessive mutations are deleterious, they’d be purged more effectively). But background selection is a relatively gentle and continuous process. The width of the flanking regions impacted by selection against a focal mutant should be modest. What they found was that there werehuge genomic blocks without any segments of incomplete lineage sorting in humans and chimpanzees. That is, variation was removed in some portions of the genome and not others. One process that can cause this are positive selective sweeps. The authors posit there were many of these to explain how many regions of the X chromosome seem to have been affected.

[![51x-WwY-sAL.\_SX387_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/08/51x-WwY-sAL._SX387_BO1204203200_-233x300.jpg?resize=233%2C300)![51x-WwY-sAL.\_SX387_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/08/51x-WwY-sAL._SX387_BO1204203200_-233x300.jpg?resize=233%2C300)](https://www.amazon.com/exec/obidos/ASIN/0878933085/geneexpressio-20)What was driving these sweeps? At this point they’re really tentative. But they suggest **meiotic drive.** Meiotic drive is pretty famous from the deleterious *t* haplotype in [mice](https://en.wikipedia.org/wiki/Intragenomic_conflict#Killer_and_target), but there might be a major bias in when we see drive, because if it doesn’t have a deleterious drag it might result in such rapid sweeps to fixation that we won’t ever catch it in the act. It could be pervasive as a phenomenon, but we might have a skewed perspective of its basic nature.

Finally, they also report that these regions of reduced ILS correlate with regions of the X chromosome where there is very little Neanderthal admixture. So this might be part of a broader evolutionary dynamic among apes. Mailund promises more, and I’ll be waiting….

### Related Posts:

- [Chimpanzee & human
  speciation](https://www.gnxp.com/WordPress/2009/08/21/chimpanzee-human-speciation/) - [Chimp vs. human
  genomes](https://www.gnxp.com/WordPress/2005/09/21/chimp-vs-human-genomes/) - [Alu and chimpanzees &
  humans](https://www.gnxp.com/WordPress/2007/10/18/alu-and-chimpanzees-humans/) - [Chimps, humans, and
  allopatry](https://www.gnxp.com/WordPress/2009/08/31/chimps-humans-and-allopatry/) - [Drivers of selection for ghosts in the
  genome](https://www.gnxp.com/WordPress/2018/12/23/drivers-of-the-selection-for-ghosts-in-the-genome/) - [Localizing recent adaptive evolution in the human
  genome](https://www.gnxp.com/WordPress/2007/04/20/localizing-recent-adaptive-evolution-in-the-human-genome/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F08%2F20%2Fselective-sweeps-on-the-x-chromosome-of-homonoids%2F&linkname=Selective%20sweeps%20on%20the%20X%20chromosome%20of%20homonoids "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F08%2F20%2Fselective-sweeps-on-the-x-chromosome-of-homonoids%2F&linkname=Selective%20sweeps%20on%20the%20X%20chromosome%20of%20homonoids "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F08%2F20%2Fselective-sweeps-on-the-x-chromosome-of-homonoids%2F&linkname=Selective%20sweeps%20on%20the%20X%20chromosome%20of%20homonoids "Email")[](https://www.addtoany.com/share)
