+++
title = "Steak, it’s what’s for"
full_title = "Steak, it’s what’s for publication"
date = "2014-04-16"
upstream_url = "https://www.gnxp.com/WordPress/2014/04/16/steak-its-whats-for-publication/"

+++
Source: [here](https://www.gnxp.com/WordPress/2014/04/16/steak-its-whats-for-publication/).

Steak, it’s what’s for publication

[![Citation: Decker, Jared E., et al. "Worldwide Patterns of Ancestry, Divergence, and Admixture in Domesticated Cattle." arXiv preprint arXiv:1309.5118 (2013).](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/04/cattle_tree_of_life.jpg?resize=600%2C804)![Citation: Decker, Jared E., et al. "Worldwide Patterns of Ancestry, Divergence, and Admixture in Domesticated Cattle." arXiv preprint arXiv:1309.5118 (2013).](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/04/cattle_tree_of_life.jpg?resize=600%2C804)](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004254)

**Citation:**Decker JE, McKay SD, Rolf MM, Kim J, Molina Alcalá A, et al. (2014) [Worldwide Patterns of Ancestry, Divergence, and Admixture in Domesticated Cattle](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004254). PLoS Genet 10(3): e1004254. doi:10.1371/journal.pgen.1004254



![440px-Steak_03_bg_040306](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/04/440px-Steak_03_bg_040306-300x224.jpg?resize=300%2C224)![440px-Steak_03_bg_040306](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/04/440px-Steak_03_bg_040306-300x224.jpg?resize=300%2C224)I am a man of a particular age, old enough to remember when the idea of **thousands** of what were then quaintly termed ‘molecular markers’ would have left one aghast at the surfeit of data. Today the term “post-genomic” almost strikes me as as anachronistic as the “information superhighway.” This is not the post-genomic era, it just ***is***, the wildest dreams that were, ***are***. But the glorious present of data abundance is not without its limitations and pitfalls. As a friend explained once, bioinformaticians just “do stuff,” sometimes without understanding why they do stuff. Somewhere along the way the *bio* part seems to have been forgotten in the hurry to assemble the next organism as the machine demands more and more for its hungry maw. But the mechanical monster slurping through the fire hose of data with a hacked together chimera of a regular expression isn’t without some purpose. Many biologists with an interest in evolution have a dream of dense marker painting vast swaths of the tree of life, an empire of phyolgenetic information to be conquered.

But these vistas need some context, a horizon of information about the **organism**. This came to mind when I read [Jared Decker’s](https://twitter.com/pop_gen_JED) new paper on the phylogenetics of domestic cattle, [Worldwide Patterns of Ancestry, Divergence, and Admixture in Domesticated Cattle](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004254). In many ways it is a straightforward paper. You can see discussions on the earlier iterations over at [Haldane’s](http://haldanessieve.org/2013/10/22/thoughts-on-worldwide-patterns-of-ancestry-divergence-and-admixture-in-domesticated-cattle/) [Sieve](http://haldanessieve.org/2014/01/15/author-post-worldwide-patterns-of-ancestry-divergence-and-admixture-in-domesticated-cattle/) (the preprint process seems to have worked to make it a more robust and clear publication from what I can tell!). Decker utilizes some straightforward methods (at least straightforward in 2014) on a very large SNP marker data set with expansive geographic coverage. In particular, [TreeMix](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002967), [Admixture](http://dalexander.bol.ucla.edu/preprints/admixture-preprint.pdf), and [PCA](http://www.hsph.harvard.edu/alkes-price/software/). With about \~40,000 SNPs these packages should blast through the data rather quickly (I’ve used all of them with this marker density, and sample sizes of approximately the size of the one Decker has).

You can read the whole paper yourself since it is open access. **To me it seems to reiterate that cattle truly are cattle, to be pulled and prodded and traded at the whim of human beings**. The fact that many East African cattle have predominantly Indian heritage (one of the two major clades) illustrates the fact that domestic animals exhibit the protean tendencies of human culture, rather than biological organisms which are governed by standard geographical and morphological diversification through conventional population genetic pressures. But I have to still admit that much of the narrative force of this paper escapes me because I lack understanding of the cattle at a level beyond the plainly statistical genetic. In other words, **the organism matters**. Cattle geneticists who may “hum through” the plots may still be able to grasp the force of argument with a greater clarity because their understanding of the topic is fundamentally thicker than that of outsiders. Many of the paper’s inferences from genetic data clearly draw their plausibility from elements of natural history which bovine biologists would take for granted.

And this is just the beginning. Over the next decade it seems inevitable that the clusters at the heart of “genomics cores” across the world will be gorging on whole sequences of thousands of individuals for many organisms. It will be a “flood the zone” era for attempting to understand the tree of life. An army of bioinformaticists will be thrown at the data in human waves, absorbing shock after shock, slowly transforming the*ad hoc*kludge pipelines of the pre-Model T era of genomics into simpler turnkey solutions. And then the biology will come back to the fore, and the deep wellspring of knowledge by those who focus on specific organisms and is going to be the essence of the enterprise once more.

### Related Posts:

- [Important papers on recent human
  evolution](https://www.gnxp.com/WordPress/2007/09/29/important-papers-on-recent-human-evolution/) - [A very recent speciation time of
  human-chimp](https://www.gnxp.com/WordPress/2007/02/24/a-very-recent-speciation-time-of-human-chimp/) - [How many SNPs to distinguish Japanese &
  Chinese?](https://www.gnxp.com/WordPress/2010/03/05/how-many-snps-to-distinguish-japanese-chinese/) - [Population structure using haplotype
  data](https://www.gnxp.com/WordPress/2012/01/28/population-structure-using-haplotype-data/) - [The arcane art of ancient
  admixture](https://www.gnxp.com/WordPress/2012/10/21/the-arcane-art-of-ancient-admixture/) - [Look, it's a
  tree!](https://www.gnxp.com/WordPress/2007/04/07/look-it-s-a-tree/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F04%2F16%2Fsteak-its-whats-for-publication%2F&linkname=Steak%2C%20it%E2%80%99s%20what%E2%80%99s%20for%20publication "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F04%2F16%2Fsteak-its-whats-for-publication%2F&linkname=Steak%2C%20it%E2%80%99s%20what%E2%80%99s%20for%20publication "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F04%2F16%2Fsteak-its-whats-for-publication%2F&linkname=Steak%2C%20it%E2%80%99s%20what%E2%80%99s%20for%20publication "Email")[](https://www.addtoany.com/share)
