+++
title = "Disease as a byproduct"
full_title = "Disease as a byproduct of adaptation"
date = "2010-07-22"
upstream_url = "https://www.gnxp.com/WordPress/2010/07/22/disease-as-a-byproduct-of-adaptation/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/07/22/disease-as-a-byproduct-of-adaptation/).

Disease as a byproduct of adaptation

How we perceive nature and describe its shape are a matter of values and preferences. Nature does not take notice of our distinctions; they exist only as instruments which aid in our comprehension. I’ve brought this up in relation to issues such as categorization of [recessive vs. dominant traits](https://www.gnxp.com/MT2/archives/003868.html). The offspring of people of Sub-Saharan African and non-African ancestry where the non-African parent has straight or wavy hair tend to have [very curly hair](http://www.blackcelebkids.com/2007/11/24/thandie-newton-and-daughters-a-family-affair/). Therefore, one may say that the tightly curled hair form is dominant to straight or wavy hair. But, it is also the case that there is some modification in relation to the African parent in the offspring, so the dominance is not complete. When examining the morphology of the follicle, which determines the extent of the hair’s curl, the offspring may in fact exhibit some differences from both parents. **In other words our perception of the outcomes of inheritance are contingent to some extent on our categorization of the traits as well as our specific focus along the developmental pathway.**

Or consider the division between “traits” and “diseases.” The quotations are necessary. Lactose intolerance is probably one of the best cases to illustrate the gnarly normative obstructions which warp our perceptions. As a point of fact lactose intolerance is the ancestral human state, and numerically predominant. It is the “wild type.” Lactose tolerance is a relatively recent adaptation, found among a variety of West Eurasian and African populations. A more politically correct term, lactase persistence, probably better encapsulates the evolutionary history of the trait, which has shifted from the class of disease to that of genetic trait when we evaluate the bigger picture (obviously diseases are simply “bad” traits”).

  
Sometimes though the issues are more cut & dried. No one would doubt that [sickle-cell anemia](https://en.wikipedia.org/wiki/Sickle-cell_disease) is a disease. It has a major fitness impact in a colloquial sense, as well as evolutionarily. It kills you, and it kills your potential genetic lineage. But, it is also a byproduct of [adaptation](https://en.wikipedia.org/wiki/Sickle-cell_disease#Heterozygotes) to endemic malaria. Sickle-cell disease one of the classical illustrations of heterozygote advantage, whereby those who carry one copy of the mutation on the gene have increased fitness vis-a-vis those who carry two normal copies of the gene. The increase in frequency of the mutant gene though is balanced by the fact that mutant homozygotes have decreased fitness.

We can then construct a narrative of the long term evolutionary dynamics from this initial condition. When a new exogenous stress hits a population mean fitness drops immediately (take a look at the biographies of the Popes, and observe how many died of malaria in the Dark Ages when that disease was new to Italy). Natural selection quickly increases in frequency any alleles which confer protection against the exogenous stress. But, baked into the cake of how genetics in complex organisms usually works, one allele may often have multiple downstream consequences. This is [pleiotropy](https://en.wikipedia.org/wiki/Pleiotropy). This means that if a change at a locus increases aggregate fitness, it may nevertheless destabilize long established biochemical pathways. In the short term evolution simply takes the net fitness impact into account. Over the long term one assumes that “better solutions” will emerge which do not have so high a fitness drag, perhaps through the evolution of modifier genes which mask the deleterious outcomes of the initial mutant. This sort of *ad hoc* trial and error and “duct-taping” of kludges is part and parcel of how adaption works in situations where shocks out of equilibrium states are common.

In many cases the byproducts of a genetic change may be benign. To my knowledge no one knows major negative consequences of carrying the alleles which confer lactase persistence (excepting some studies indicating higher obesity, but this seems a marginal fitness impact which has only come to the fore in the past century in all likelihood). But in other cases the outcomes may not be as serious as that of sickle-cell anemia, but may rise above the level of significance where one must note the existence of a disease which is a secondary consequence of adaptation to meet a new challenge.

Yesterday I pointed to a paper which illustrates just this phenomenon, [Association of Trypanolytic ApoL1 Variants with Kidney Disease in African-Americans](http://www.sciencemag.org/cgi/content/abstract/science.1193032):

> African-Americans have higher rates of kidney disease than > European-Americans. Here, we show that in African-Americans, focal > **segmental glomerulosclerosis (FSGS) and hypertension-attributed > end-stage kidney disease (H-ESKD) are associated with two independent > sequence variants in the APOL1 gene on chromosome 22 {FSGS odds ratio > = 10.5 \[95% confidence interval (CI) 6.0 to 18.4\]; H-ESKD odds ratio > = 7.3 (95% CI 5.6 to 9.5)}.** The two APOL1 variants are common in > African chromosomes but absent from European chromosomes, and both > reside within haplotypes that harbor signatures of positive selection. > Apolipoprotein L-1 (ApoL1) is a serum factor that lyses trypanosomes. > In vitro assays revealed that only the kidney disease-associated ApoL1 > variants lysed Trypanosoma brucei rhodesiense. We speculate that > evolution of a critical survival factor in Africa may have contributed > to the high rates of renal disease in African-Americans.

In its implementation the paper has a lot of moving parts, but the outcome is straightforward. If you haven’t, you might read [Genomes Unzipped](http://www.genomesunzipped.org/2010/07/how-to-read-a-genome-wide-association-study.php) and its post [How to read a genome-wide association study](http://www.genomesunzipped.org/2010/07/how-to-read-a-genome-wide-association-study.php). This is a case where the original association studies were not reporting false results, but, it seems that one had to take a further step to really understand the likely molecular genetic and evolutionary underpinnings of what was going on. These results suggest that the original signals of association for variants within the *MYH9* gene were actually signals from within *APOL1*, which happened to be next to *MYH9*. The region around *MYH9* had already showed up in tests to detect natural selection through patterns of [linkage disequilibrium](https://en.wikipedia.org/wiki/Linkage_disequilibrium) (non-random associations of alleles at different loci within the genome, in this case the relevant consideration are adjacent loci across continuous regions of the genome which come together to form haplotype blocks). Since the footprint of natural selection on the genome is often wide that did not imply that *MYH9* was the target of natural selection *per se*, opening the likely possibility for other causal associations. A convenience in light of the difficulty of establishing a plausible functional relationship between renal failure and *MYH9*.

To explore the possibility of nearby functional candidates the researchers focused on a number of alleles within this genomic region which exhibited maximal European-African frequency differences in the [1000 Genomes Project](http://www.1000genomes.org/page.php). Once they ascertained the between population differences they then looked at differences in allele frequencies in cases and controls within the African American population for the two diseases in question (those with the trait/disease vs. those without). Table 1 has the top line raw results:

![apo1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo1.png?resize=600%2C324)![apo1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo1.png?resize=600%2C324)

WT = “Wild Type,” the ancestral allelic variant found in most populations. G1 and G2 are two haplotypes, associated alleles across the locus of the *APOL1* gene. G1 consists of the two derived non-synonymous coding variants rs73885319 (S342G) and rs60910145 (I384M) within an exonic region of *APOL1*. Non-synonymous simply means that a change at that base pair alters the amino acid coded, and exons are the genomics regions whose information is eventually translated into proteins. In other words, these are non-neutral functionally significant genomic regions which do something. G2 is a 6 base pair deletion, rs71785313, close to G1 in *APOL1*.

![apo12](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo12.png?resize=350%2C1366)![apo12](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo12.png?resize=350%2C1366)To more formally model the relationship between the alleles which are found to differ between cases and controls they performed a [logistic regression](https://en.wikipedia.org/wiki/Logistic_regression). The alleles serve as independent variables which can predict the probable outcome of the dependent variable, the probability of FSGS or H-ESKD in this case (renal failure). Figure 1 to the left has a summary of some of the results of the regression in graphical form for FSGS. I’ve rotated it so it can fit on the screen. Basically the strong signals are to the right of the chart (from your perspective). The y-axis displays (horizontal from your perspective) negative-log of p-values for a signal at a particular marker, which is defied by the x-axis (vertical for you). The labels show the particular gene at that genomic position. The smaller the p-value, the more probable that the signal is real and not random. This produces huge spikes in the negative-log values (in the body of the paper they present p-values on the order of 10⁻³⁵).

You can see that it is in *APOL1* that the biggest signals reside. The first panel, A, throws all the SNPs into the mix. On *MYH9* they highlight a few SNPs which combine to form the E-1 haplotype, which is strongly associated with cases (this is where the association between disease and genetic variants on *MYH9* are coming from). This haplotype is found in conjunction with G1 and G2 on *APOL1*. E-1 is present in 89% of haplotypes carrying G1 and in 76% of haplotypes carrying G2. **A classic illustration of likely correlation but not causation.** The second panel controls for the effect of G1. In other words, this is showing you the variation in the dependent variable that remains after you take the largest independent variable, G1, into account. The G2 haplotype is the largest effect independent variable after G1 is taken into account; in other words, it explains most of the residual variation in FSGS probability. Finally, the last panel controls for both G1 and G2. As you can see there aren’t any major signals left; the distribution is relatively flat. Logically once you account for the variables which produce change in an outcome you shouldn’t see any impact of other variables. And that’s what happens here. They also performed controls where *MYH9* was held constant, and that does not eliminate the signals in *APOL1*. *MYH9* is conditional on its correlation with *APOL1*. This was the correlation which showed up on the original association studies. The exact same pattern of signals within the logistic regression model was replicated for H-ESKD. G1 had the strongest signal, then G2. The markers within *MYH9* was not significant once one controlled for the variants in G1 and G2.

It is important to remember though that these markers are segregating within a human population where individuals have three potential genotypes. Ancestral homozygote, homozygote for the mutants, and heterozygote. They found that a recessive model of expression of disease is most appropriate in the case of these risk alleles. That is, most of the increased risk is accounted for by the change from one risk allele, the heterozygote state, to two risk alleles, the homozygote state. One risk allele increased odds of renal failure by 1.26, but two by 7.3. The odds ratio of two risk alleles compared to a base rate of one risk allele was 5.8. They report that the results for FSGS were broadly similar. This matters because the frequency of the trait/disease in a random mating population is conditional on the homozygotes if it has a recessive expression pattern. G1 was present in 40% of Yoruba HapMap data set, but in none of the two Eurasian groups, Europeans and East Asians. G2 was found in three Yoruba, but in none of the Eurasian groups. Assuming [Hardy-Weinberg equilibrium](https://en.wikipedia.org/wiki/Hardy–Weinberg_principle) the Yoruba should have 16% of the population at sharply elevated risk for FSGS and H-ESKD because they’d be homozygotes for the G1 allele.

Once they established which markers seem to implicated in this phenotypic variation, they wanted to focus on how the frequencies of those markers came to be. Specifically, G1 and G2 seem to be derived haplotypes which arose out of the ancestral background. In plain English 20,000 years ago Africans should have looked like all non-Africans genomically, at least on the functionally relevant segments, but within the last 10,000 years it looks like new variants rose in frequency driven by natural selection to new environmental stresses. The region has already broadly been surveyed by linkage disequilibrium based tests, which basically look for regions of long haplotypes, homogenized zones of the genome where many individuals have the variation removed because one gene rose so rapidly in frequency that huge adjacent sections hitchhiked up in frequency. Presumably this may have happened with the *MYH9* haplotype correlated with the traits under consideration here; G1 and G2 dragged up the E-1 haplotype as a secondary consequence of their own rise to prominence among some Sub-Saharan African populations.

So next authors turned to tried & tested techniques and focused on the risk markers which they had discovered earlier in their research, G1 and G2. Specifically, EHH, which is best at detecting selection where sweeps have nearly completed (e.g., the derived variant is at frequency 0.95 within the population), iHS, which is best at detecting sweeps which have not completed (e.g., the derived variant is at frequency 0.6), as well as ΔiHH, which I am less familiar with but is reputedly similar to iHS but uses absolute haplotype length as opposed to relative haplotype length. Figure 2 show the results of these tests:

![apol13](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apol13.png?resize=600%2C153)![apol13](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apol13.png?resize=600%2C153)

The resolution isn’t the best, **but G1 and G2 seem to be outliers on all three tests to detect natural selection by using patterns of linkage disequilibrium.** The first panel is EHH, the second and third show iHS and ΔiHH respectively, with the position of the markers being outliers among the distribution of values for the genome within the Yoruba. This is not proof of adaptation, but it changes our weights of possibilities. Additionally, they note that Europeans exhibit no such patterns on these markers. Visually the position of the markers in the latter two panels would be closer to the mode of the distribution in Europeans.

To review, first they confirmed a causal relationship between a particular set of markers, haplotypes, and the traits of interest. Second, they confirmed that said markers seem to bear the hallmarks of genomic regions subject to natural selection. We know that focal segmental glomerulosclerosis (FSGS) end-stage kidney disease (H-ESKD), the traits whose relationship to the G1 and G2 haplotypes seem confirmed, are unlikely to be targets of positive natural selection. To get a better sense of that we need to look at Apol1, the protein product of *APOL1*, and what it does. At this point I’ll quote the paper:

> ApoL1 is the trypanolytic factor of human serum that confers > resistance to the Trypanosoma brucei brucei (T. brucei brucei) > parasite…T. brucei brucei has evolved into two additional subspecies, > Trypanosoma brucei rhodesiense and Trypanosoma brucei gambiense, which > have both acquired the ability to infect humans…T. brucei rhodesiense > is predominantly found in Eastern and Southeastern Africa, while T. > brucei gambiense is typically found in Western Africa, though some > overlap exists…Since these parasites exist only in sub-Saharan Africa, > we hypothesized that the APOL1 gene may have undergone natural > selective pressure to counteract these trypanosoma adaptations. As an > initial test of this hypothesis, we performed in vitro assays to > compare the trypanolytic potential of the variant, disease-associated > forms of ApoL1 proteins with that of the “wild-type” form of ApoL1 > protein that is not associated with renal disease.

We’re talking about [sleeping sickness](https://en.wikipedia.org/wiki/African_trypanosomiasis). Here’s a [description](http://allafrica.com/stories/200805120552.html):

> It starts with a headache, joint pains and fever. It is the kind you > would expect to get over quickly. But after a while, things get worse. > You fall asleep most of the time, are confused and get intense pains > and convulsions. >
> If you do not get treatment, your body begins to waste away. > Eventually, you slip into coma and die. This is human African > trypanosommiasis, better known as sleeping sickness. If untreated, it > kills 100% of its victims in a very short time.

Cheery. I think we have a plausible reason for natural selection to kick into overdrive! Or more specifically, we have a plausible external selection pressure which will drive fitness differentials which correlate with genetic variation. Increased probability of kidney disease seems preferable to this. In terms of the molecular genetics it looks like a factor, serum resistance-associated protein (SRA), produced by *T. brucei rhodesiense* binds to a specific location of Apol1, and that mutations at G1 and G2 change exactly that location within the protein. So these mutants may block the ability of *T. brucei rhodesiense* to turn off the body’s defenses against trypanosomes.

To test this they examined the *in vitro* lytic potential of serum produced by individuals carrying the G1 and G2 haplotypes against the three subspecies of of *Trypanosoma*. *T. brucei brucei*, which normal Apol1 can lyse, and *T. brucei rhodesiense* and *T. brucei gambiense* which can infect humans (endemic to eastern and western Africa respectively, though the former extends into west Africa as well).

– All 75 samples lysed *brucie brucie*

– None lysed *brucie gambiense*

– 46 samples lysed SRA-positive *brucie rhodesiense*, all 46 samples were from G1 or G2 carrying individuals

– The potency of G2 seemed higher than G1 against SRA-positive samples of *brucie rhodesiense*, though not SRA-negative samples, where G1 seemed as potent

– Recombinants of Apol1 which had only one of the two SNPs of the G1 haplotype were less effective against *brucie rhodesiense* than those which had both (G1 haplotype)

– Recombinants with G1 and G2 were not more effective against *brucie rhodesiense* than those with G2 alone

– Recombinants with G1 alone were more potent against SRA-negative *brucie rhodesiense* than those with G2 alone

– G2 was necessary and sufficient to block SRA binding to Apol1 and allow lysing of *brucie rhodesiense*. G1 did not block SRA binding to Apol1, but was still sufficient to lyse *brucie rhodesiense*, but far less potent against SRA-positive *brucie rhodesiense* than G2

It seems that the G1 and G2 haplotypes utilize different mechanisms to enable the lysing of invasive pathogens, and so prevent the development of sleeping sickness. Their means differ, but the ends are the same. The authors note that even minimal amounts of plasma serum produced by G2 individuals seems potent enough to block the binding of SRA to Apol1 and so enable lysis. And introduction of such plasma into the bloodstreams of individuals who do not have resistance may then be highly efficacious as a preventative treatment against sleeping sickness. They do note that they did not explore in detail the mechanism by which the G1 and G2 variants result in suscepbility to kidney failure, but that’s presumably for the future.

Finally, the second to last paragraph where they bring it all together:

> It will be interesting to determine the distribution of these > mutations throughout sub-Saharan Africa. In present-day Africa, T. > brucei rhodesiense is found in the Eastern part of the continent, > while we noted high frequency of the trypanolytic variants and the > signal of positive selection in a West African population. Changes in > trypanosome biology and distribution and/or human migration may > explain this discrepancy, or resistance to T. brucei rhodesiense could > have favored the spreading of T. brucei gambiense in West Africa. > Alternatively, ApoL1 variants may provide immunity to a broader array > of pathogens beyond just T. brucei rhodesiense, as a recent report > linking ApoL1 with anti-Leishmania activity may suggest…Thus, > resistance to T. brucei rhodesiense may not be the only factor causing > these variants to be selected.

This is a very long review already. But, while I have your attention, I think I need to point to another paper on the same topic which has a slightly different twist. I won’t dig into the details with the same thoroughness as above, but rather I’ll highlight the value-add of this group’s contribution. It’s an Open Access paper, unlike the one above, so you can review it in depth yourself.[Missense mutations in the *APOL1* gene are highly associated with end stage kidney disease risk previously attributed to the *MYH9* gene](http://www.springerlink.com/content/a8856j4v67396416/):

> MYH9 has been proposed as a major genetic risk locus for a spectrum of > nondiabetic end stage kidney disease (ESKD). We use recently released > sequences from the 1000 Genomes Project to identify two western > African-specific missense mutations (S342G and I384M) in the > neighboring APOL1 gene, and demonstrate that these are more strongly > associated with ESKD than previously reported MYH9 variants. The APOL1 > gene product, apolipoprotein L-1, has been studied for its roles in > trypanosomal lysis, autophagic cell death, lipid metabolism, as well > as vascular and other biological activities. We also show that the > distribution of these newly identified APOL1 risk variants in African > populations is consistent with the pattern of African ancestry ESKD > risk previously attributed to MYH9. Mapping by admixture linkage > disequilibrium (MALD) localized an interval on chromosome 22, in a > region that includes the MYH9 gene, which was shown to contain African > ancestry risk variants associated with certain forms of ESKD…MYH9 > encodes nonmuscle myosin heavy chain IIa, a major cytoskeletal > nanomotor protein expressed in many cell types, including podocyte > cells of the renal glomerulus. Moreover, 39 different coding region > mutations in MYH9 have been identified in patients with a group of > rare syndromes, collectively termed the Giant Platelet Syndromes, with > clear autosomal dominant inheritance, and various clinical > manifestations, sometimes also including glomerular pathology and > chronic kidney disease…Accordingly, MYH9 was further explored in these > studies as the leading candidate gene responsible for the MALD signal. > Dense mapping of MYH9 identified individual single nucleotide > polymorphisms (SNPs) and sets of such SNPs grouped as haplotypes that > were found to be highly associated with a large and important group of > ESKD risk phenotypes, which as a consequence were designated as > MYH9-associated nephropathies…These included HIV-associated > nephropathy (HIVAN), primary nonmonogenic forms of focal segmental > glomerulosclerosis, and hypertension affiliated chronic kidney disease > not attributed to other etiologies…The MYH9 SNP and haplotype > associations observed with these forms of ESKD yielded the largest > odds ratios (OR) reported to date for the association of common > variants with common disease risk…Two specific MYH9 variants > (rs5750250 of S-haplotype and rs11912763 of F-haplotype) were > designated as most strongly predictive on the basis of Receiver > Operating Characteristic analysis…These MYH9 association studies were > then also extended to earlier stage and related kidney disease > phenotypes and to population groups with varying degrees of recent > African ancestry admixture…and led to the expectation of finding a > functional African ancestry causative variant within MYH9. However, > despite intensive efforts including re-sequencing of the MYH9 gene no > suggested functional mutation has been identified…**This led us to > re-examine the interval surrounding MYH9 and to the detection of novel > missense mutations with predicted functional effects in the > neighboring APOL1 gene, which are significantly more associated with > ESKD than all previously reported SNPs in MYH9.**

Table one has the top line results. Focus on the first two rows, they’re “G1” from the earlier study (that is, the two SNPs which combine to form the G1 haplotype).

![apo14](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo14.png?resize=600%2C442)![apo14](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo14.png?resize=600%2C442)

Here’s a difference between the previous paper and this one: **the table above uses cases and controls from African Americans *and* Hispanic Americans.** The [original paper](http://hmg.oxfordjournals.org/cgi/content/full/19/9/1816?view=long&pmid=20144966) which the genomic data on this sample is drawn from calculates the average ancestry of African, European and Native American in the two groups is as follows (I did some rounding to keep the values round):

African American – 85%, 10%, 5%  
Hispanic American – 30%, 55%, 15%

Not surprisingly the Hispanic American sample here is mostly Puerto Rican and Dominican, explaining the greater African than Native American ancestry. Nevertheless, it is a sufficiently different genetic background to test the effects of the same marker against different genes. They confirmed the association of the markers of large effect in African Americans within the Hispanic cohort. The risk allele frequency in the African American control group is 21% vs. 37% in the cases. For Hispanic Americans are 6% and 23% for the same categories.

OK, now to the most interesting point in this short paper:

> HIVAN has been considered as the most prominent of the nondiabetic > forms of kidney disease within what has been termed the > MYH9-associated nephropathies…We have reported absence of HIVAN in HIV > infected Ethiopians, and attributed this to host genomic factors > (Behar et al. 2006). Therefore, we examined the allele frequencies of > the APOL1 missense mutations in a sample set of 676 individuals from > 12 African populations, including 304 individuals from four Ethiopian > populations…We coupled this with the corresponding distributions for > the African ancestry leading MYH9 S-1 and F-1 risk alleles. A pattern > of reduced frequency of the APOL1 missense mutations and also of the > MYH9 risk variants was noted in northeastern African in contrast to > most central, western, and southern African populations > examined…**Especially striking was the complete absence of the APOL1 > missense mutations in Ethiopia.** This combination of the reported > lack of HIVAN and observed absence of the APOL1 missense mutations is > consistent with APOL1 being the functionally relevant gene for HIVAN > risk and likely the other forms of kidney disease previously > associated with MYH9.

![apo16](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo16.png?resize=299%2C314)![apo16](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/apo16.png?resize=299%2C314)Bingo. The previous paper focused on African Americans (along with the HapMap Yoruba). But the pattern of variation *within* Africa is interesting as well. Ethiopians are not quite like other Africans, having a great deal of admixture with populations from Arabia (many of the languages of highland Ethiopia are Semitic). But the majority of their ancestry remains similar to that of other Sub-Saharan Africans. As a point of contrast the ecology of Ethiopia differs a great deal from the rest of Sub-Saharan Africa because of its elevation, and concomitant frigidity. The mean monthly low in Addis Ababa is around 10 (50 for Americans) degrees and mean high 20-25 (high 60s to mid 70s for Americans). There isn’t much variation from month to month because of the low latitude, but the high elevation keeps the temperatures relatively moderate. **Different environments result in different selection pressures, and Ethiopia has a very unique environment within Africa.** The [tsetse fly](https://en.wikipedia.org/wiki/Tsetse_fly) which serves as a vector forTtrypanosomes does not seem to be present in the Ethiopian highlands. The map above shows the distribution within Africa of one the markers which defines the G1 haplotype in the previous paper. Note that the modal frequency is in the west of Africa, and the frequency drops off to the east (though the geographic coverage leaves a bit to be desired if you look at the raw data which went into generating this map, which smooths over huge discontinuities).

One of the points I want to reemphasize from the tests of natural selection in the first paper is that **these genetic adaptations are likely to be new, otherwise recombination would have broken up the long haplotypes and reduced linkage disequilibrium.** New as in the last 10,000 years. It is interesting that a particular subspecies of Trypanosome which is immune to these genetic adaptations is endemic to *west* Africa. We may be seeing evolution in action here, or at least the arms race between man and pathogen where man is always one step behind. In contrast, the subspecies which is effectively diffused by the genetic adaptations reviewed here is present in higher numbers precisely in the regions where the resistance mutations are extant at lower proportions. Perhaps there are different mutations in these regions of Africa, not yet properly identified. Or perhaps the we’re seeing humans in this region at an earlier stage of the dance, so to speak.

**Citation:** Giulio Genovese, David J. Friedman, Michael D. Ross, Laurence Lecordier, Pierrick Uzureau, Barry I. Freedman, Donald W. Bowden, Carl D. Langefeld, Taras K. Oleksyk, Andrea Uscinski Knob, Andrea J. Bernhardy, Pamela J. Hicks, George W. Nelson, Benoit Vanhollebeke, Cheryl A. Winkler, Jeffrey B. Kopp, Etienne Pays, & Martin R. Pollak (2010). Association of Trypanolytic ApoL1 Variants with Kidney Disease in African-Americans Science : [10.1126/science.1193032](10.1126/science.1193032)

**Citation:** Tzur S, Rosset S, Shemer R, Yudkovsky G, Selig S, Tarekegn A, Bekele E, Bradman N, Wasser WG, Behar DM, & Skorecki K (2010). Missense mutations in the APOL1 gene are highly associated with end stage kidney disease risk previously attributed to the MYH9 gene. Human genetics PMID: [20635188](http://www.ncbi.nlm.nih.gov/pubmed/20635188)

### Related Posts:

- [TCHH & curly hair in
  Europeans](https://www.gnxp.com/WordPress/2009/11/08/tchh-curly-hair-in-europeans/) - [Dominance & recessive, is it
  worthwhile?](https://www.gnxp.com/WordPress/2006/10/01/dominance-recessive-is-it-worthwhile/) - [Amongst the
  halfings](https://www.gnxp.com/WordPress/2006/11/05/amongst-the-halfings/) - [The genetics of hair texture: a
  mystery](https://www.gnxp.com/WordPress/2011/10/02/the-genetics-of-hair-texture-a-mystery/) - [Ancestral proto-Eurasians may have had wavy
  hair](https://www.gnxp.com/WordPress/2019/02/24/ancestral-proto-eurasians-may-have-had-wavy-hair/) - [Could it be hair
  form?](https://www.gnxp.com/WordPress/2007/10/17/could-it-be-hair-form/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F22%2Fdisease-as-a-byproduct-of-adaptation%2F&linkname=Disease%20as%20a%20byproduct%20of%20adaptation "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F22%2Fdisease-as-a-byproduct-of-adaptation%2F&linkname=Disease%20as%20a%20byproduct%20of%20adaptation "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F22%2Fdisease-as-a-byproduct-of-adaptation%2F&linkname=Disease%20as%20a%20byproduct%20of%20adaptation "Email")[](https://www.addtoany.com/share)
