+++
title = "The Asian human species"
full_title = "The Asian human species that we lost"
date = "2019-12-04"
upstream_url = "https://www.gnxp.com/WordPress/2019/12/04/the-asian-human-species-that-we-are-lost/"

+++
Source: [here](https://www.gnxp.com/WordPress/2019/12/04/the-asian-human-species-that-we-are-lost/).

The Asian human species that we lost

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/lone_survivor-1.jpeg?resize=183%2C275&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/09/lone_survivor-1.jpeg?resize=183%2C275&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B005XMK898/geneexpressio-20)A new paper, [The GenomeAsia 100K Project enables genetic discoveries across Asia](https://www.nature.com/articles/s41586-019-1793-z):

> The underrepresentation of non-Europeans in human genetic studies so > far has limited the diversity of individuals in genomic datasets and > led to reduced medical relevance for a large proportion of the world’s > population. Population-specific reference genome datasets as well as > genome-wide association studies in diverse populations are needed to > address this issue. Here we describe the pilot phase of the GenomeAsia > 100K Project. This includes a whole-genome sequencing reference > dataset from 1,739 individuals of 219 population groups and 64 > countries across Asia. We catalogue genetic variation, population structure, disease associations and founder effects. We also explore the use of this dataset in imputation, to facilitate genetic studies in populations across Asia and worldwide.

This is a paper where you need to jump to the [supplementary note](https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-019-1793-z/MediaObjects/41586_2019_1793_MOESM1_ESM.pdf). The really interesting thing about this paper is the Denisovan aspect. They had a lot of whole genomes of Asians. Large sample sizes and good coverage genome-wide.Here’s the relevant selection:

> The high levels of Denisovan ancestry in Melanesians and the Aeta are > consistent with an admixture event into a population that is ancestral > to both…**however, two lines of evidence suggest that the ancestors of > the Aeta experienced a second Denisovan admixture event.** First, > multiple analyses found that the Aeta are genetically more similar to > populations without appreciable Denisovan ancestry (for example, > Igorot, Malay and Malay Negrito groups) than they are to Melanesians > \[Aeta are **at least** 25% Austronesian -Razib\].his can be explained > by more recent gene flow from other populations without Denisovan > ancestry. However, such gene flow would reduce the levels of Denisovan > admixture below that found in Melanesians. More directly, **we find > that putative Denisovan haplotypes that are unique to the Aeta (n = > 962) are *significantly longer* than putative Denisovan haplotypes > shared between Aeta and Papuans** (n = 596, mean = 16.1 kb compared > with mean = 14.1 kb, Mann–Whitney U-test, P ≪ 10−10), or putative > Denisovan haplotypes unique to Papuans (n = 727, mean = 16.1 kb > compared with mean = 14.9 kb, Mann–Whitney U-test, P ≪ > 10−1,000)…supporting a scenario in which a second admixture event > between the Aeta and Denisovans happened after the separation of the > Aeta and Melanesians. Two distinct Denisovan admixture events are most consistent with Homo sapiens and Denisovans interacting within southeast Asia…making it likely that admixture occurred within Sundaland…or even farther east….

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/11/neanderthal.jpeg?resize=182%2C277&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/11/neanderthal.jpeg?resize=182%2C277&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B00F226E74/geneexpressio-20)Longer haplotypes in the Aeta indicates that the Denisovan admixture unique to them happened later than the one that might be common or contemporaneous with the Papuans.

But that’s not the only interesting pattern. In the supplements, they have an estimate of Denisovan ancestry in various Asian populations, and some patterns jump out. As found in other work, Northeast Asians have Denisovan ancestry. In the 0.20-0.25% range. South Asians have a wider range, from 0.10%-0.30%. The Andamanese are above 0.3%, though not much over. Some of the Malaysian Negritos are on the 0.35%-0.40% range. West Asians, like Europeans, are basically at \~0%.

It seems likely that [Northeast Asians](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5866234/) obtain their Denisovan admixture from a population closely related to the Altai sample. Populations from southern Eurasia and Oceania, less so. But, now there is good evidence that the Aeta, and Phillippine Paleolithic populations in general, mixed with a Denisovan group later than the event that led to the high fraction in Oceanians. Finally, there is evidence that [Papuans](https://www.genomicus.com/files/publications/Jacobs_2019_Cell_v177_p1010.pdf) carry two mixture of events. One west of the Wallace Line, and another east of it. The eastern admixture is recent, possibly less than 20,000 years ago (though the authors in the linked paper suggest these are unadmixed, I think a modern population with a high load of Denisovan could be another option).

There is suggestive evidence across these papers of admixture with Denisovans in Northeast Asia, the Phillippines, and New Guinea (or at least Wallacea). And, evidence of admixture more generally across southern Eurasians.

Whereas a single Neanderthal admixture [still a viable explanation](https://www.pnas.org/content/116/5/1639) for that hominin’s ancestry in modern humans, it seems very unlikely for the deeply diverged lineages which are termed “Denisovan.”

### Related Posts:

- [Population structure &
  Singapore](https://www.gnxp.com/WordPress/2009/09/05/population-structure-singapore/) - [Ashkenazi Jews are more European in
  ancestry](https://www.gnxp.com/WordPress/2009/09/04/ashkenazi-jews-are-more-european-in-ancestry/) - [Brownology](https://www.gnxp.com/WordPress/2006/12/22/brownology/) - [Peeling the population genetic Indian
  onion](https://www.gnxp.com/WordPress/2011/12/08/peeling-the-population-genetic-indian-onion/) - [Genomic diversity & historical
  inference](https://www.gnxp.com/WordPress/2009/02/16/genomic-diversity-historical-inference/) - [Genetic Map of East
  Asia](https://www.gnxp.com/WordPress/2008/12/07/genetic-map-of-east-asia/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F12%2F04%2Fthe-asian-human-species-that-we-are-lost%2F&linkname=The%20Asian%20human%20species%20that%20we%20lost "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F12%2F04%2Fthe-asian-human-species-that-we-are-lost%2F&linkname=The%20Asian%20human%20species%20that%20we%20lost "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F12%2F04%2Fthe-asian-human-species-that-we-are-lost%2F&linkname=The%20Asian%20human%20species%20that%20we%20lost "Email")[](https://www.addtoany.com/share)
