+++
title = "Whole genome sequencing"
full_title = "Whole genome sequencing comes to CavalliSforza’s samples"
date = "2019-06-27"
upstream_url = "https://www.gnxp.com/WordPress/2019/06/27/whole-genome-sequencing-comes-to-cavalli-sforzas-data/"

+++
Source: [here](https://www.gnxp.com/WordPress/2019/06/27/whole-genome-sequencing-comes-to-cavalli-sforzas-data/).

Whole genome sequencing comes to Cavalli-Sforza’s samples

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/12/historygeo.jpeg?resize=195%2C258&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/12/historygeo.jpeg?resize=195%2C258&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B07DMWH73Z/geneexpressio-20/ref=as_at?imprToken=3SzvErsqk0uegISbc6B6.g&slotNum=0&imprToken=T99HvL7y.cNmVg.TW4Q7EQ&slotNum=0&imprToken=67TxncJrNOO11LKRkLbXog&slotNum=23&creativeASIN=0878933085&linkCode=w61&imprToken=QiG2bf7fc5-czG6VLZ9cSg&slotNum=164)More than twenty years ago L. L. Cavalli-Sforza published [The History and Geography of Human Genes](https://www.amazon.com/exec/obidos/ASIN/B07DMWH73Z/geneexpressio-20/ref=as_at?imprToken=3SzvErsqk0uegISbc6B6.g&slotNum=0&imprToken=T99HvL7y.cNmVg.TW4Q7EQ&slotNum=0&imprToken=67TxncJrNOO11LKRkLbXog&slotNum=23&creativeASIN=0878933085&linkCode=w61&imprToken=QiG2bf7fc5-czG6VLZ9cSg&slotNum=164). Based on decades of analysis of ‘classical’ markers, this work lays out results of statistical genetic analyses based on a few hundred genes, as well as displaying Cavalli-Sforza’s encyclopedic ethnographic knowledge. A close look at this book will yield some familiar population groups to readers of this weblog. The reason for this is simple: **the cell lines continued onward to contribute to the [HGDP](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project) data set.**

In 2002 Rosenberg et al. used these populations in [Genetic Structure of Human Populations](https://science.sciencemag.org/content/298/5602/2381) by looking at “377 autosomal microsatellite loci.” Microsatellites are highly variable genetic regions. They pack a lot of diversity per locus. With more input variation Rosenberg et al. advanced beyond Cavalli-Sforza’s earlier work (instead of pairwise comparisons between populations, one could infer individual relatedness as displayed in a bar plot).

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/11/whoweare.jpeg?resize=183%2C276&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/11/whoweare.jpeg?resize=183%2C276&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B073NP8WT3/geneexpressio-20/ref=as_at?imprToken=3SzvErsqk0uegISbc6B6.g&slotNum=0&imprToken=T99HvL7y.cNmVg.TW4Q7EQ&slotNum=0&imprToken=67TxncJrNOO11LKRkLbXog&slotNum=23&creativeASIN=0878933085&linkCode=w61&imprToken=QiG2bf7fc5-czG6VLZ9cSg&slotNum=164)But times change, and in 2008 the same data set was used in [Worldwide human relationships inferred from genome-wide patterns of variation](https://www.ncbi.nlm.nih.gov/pubmed/18292342), which utilized a 650,000 marker SNP-array. Though Rosenberg et al.’s work advanced the ball considerably, the move to genome-wide analysis was even bigger. For many years this data set has been a widely used benchmark and reference (these markers and populations were part of the early basis of 23andMe’s analyses in terms of population genetic inference). As the [1000 Genomes Project](http://www.internationalgenome.org/) moved us beyond the SNP-array period, looking the **whole genome**, as opposed to a specific set of SNPs, the HGDP populations were still an important complement.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/03/briefhistory.jpeg?resize=183%2C276&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/03/briefhistory.jpeg?resize=183%2C276&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B06XP9Z5TS/geneexpressio-20/ref=as_at?imprToken=3SzvErsqk0uegISbc6B6.g&slotNum=0&imprToken=T99HvL7y.cNmVg.TW4Q7EQ&slotNum=0&imprToken=67TxncJrNOO11LKRkLbXog&slotNum=23&creativeASIN=0878933085&linkCode=w61&imprToken=QiG2bf7fc5-czG6VLZ9cSg&slotNum=164)The reason was simple: **Cavalli-Sforza was an ethnographic genius in comparison to most geneticists and had selected very interesting and informative populations.** In some ways, the original motivation given for selecting these groups, that they may have preserved phylogenetic patterns obscured in cosmopolitan populations, has only been partially justified.

Ancient DNA has shed light on the reality that almost all populations, indigenous and cosmopolitan, come out of periods of admixture between lineages which had heretofore been distinct and separate. But some of Cavalli-Sforza’s populations have been inordinately important in informing us about branches of the human family tree less well represented in the cosmopolitan samples accessible in the 1000 Genomes Project (or earlier, the HapMap). I’m thinking here of the Kalash (a relatively good proxy for “Ancestral North Indians”), Sardinians (the best representatives in the modern world of “Early European Farmers”), and African hunter-gatherers (who carry the deepest diverging lineage within the modern human clade).

With all that, finally, **the** **HGDP whole genome preprint is out.** Anders Bergstrom superstar!

[Insights into human genetic variation and population history from 929 diverse genomes](https://www.biorxiv.org/content/10.1101/674986v1):

> Genome sequences from diverse human groups are needed to understand > the structure of genetic variation in our species and the history of, > and relationships between, different populations. We present 929 > high-coverage genome sequences from 54 diverse human populations, **26 > of which are physically phased using linked-read sequencing**. > Analyses of these genomes reveal an excess of previously undocumented > private genetic variation in southern and central Africa and in > Oceania and the Americas, but an absence of fixed, private variants > between major geographical regions. **We also find deep and gradual > population separations within Africa**, contrasting population size > histories between hunter-gatherer and agriculturalist groups in the > last 10,000 years, a potentially major population growth episode after > the peopling of the Americas, **and a contrast between single > Neanderthal but multiple Denisovan source populations contributing to > present-day human populations**. We also demonstrate benefits to the > study of population relationships of genome sequences over ascertained > array genotypes. These genome sequences are freely available as a > resource with no access or analysis restrictions.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/06/humanerrors.jpeg?resize=183%2C275&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/06/humanerrors.jpeg?resize=183%2C275&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B07432D5GB/geneexpressio-20/ref=as_at?imprToken=3SzvErsqk0uegISbc6B6.g&slotNum=0&imprToken=T99HvL7y.cNmVg.TW4Q7EQ&slotNum=0&imprToken=67TxncJrNOO11LKRkLbXog&slotNum=23&creativeASIN=0878933085&linkCode=w61&imprToken=QiG2bf7fc5-czG6VLZ9cSg&slotNum=164)The authors were able to make recourse to many more subtle analytic methods with their phasing, which seems to have been considerably superior to population phasing (the HGDP does have some closely related individuals due to endogamy, but no traditional trios). Because their population set included some undersampled groups with a lot of diversity (e.g., San Bushmen), they detected about as many variants with \~1,000 individuals at good coverage as the 1000 Genome Project with \~2,500 individuals at variable coverage.

And there are major lacunae within the 1000 Genome Project data set even after taking into account ethnographically and historically important groups such as the San Bushmen. There are no Middle Eastern populations in the 1000 Genome Project. The HGDP has the Druze, Bedouin, Palestinians, and Mozabites.

**The preprint requires a lot of deep reading.** There is much in there that one can mull over (frankly, I’m excited about the supplementary text, but that’s just me). One thing that came to mind is that ancient DNA and other more narrow studies laid the groundwork for the interpretations that naturally fall out of this extremely rich potential set of analyses. For example, by looking at shared variants across western and central Africa the authors confirm the likely result that there is a basal human population of some sort mixed into peoples of far western Africa. And, they also confirm that the Yoruba are about \~5-10% Eurasian.

These sequences generate so much data that there are lots of potential models that might conform to them. Earlier work eliminates some possibilities and highlights others.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/11/neanderthal.jpeg?resize=182%2C277&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/11/neanderthal.jpeg?resize=182%2C277&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B00F226E74/geneexpressio-20/ref=as_at?imprToken=3SzvErsqk0uegISbc6B6.g&slotNum=0&imprToken=T99HvL7y.cNmVg.TW4Q7EQ&slotNum=0&imprToken=67TxncJrNOO11LKRkLbXog&slotNum=23&creativeASIN=0878933085&linkCode=w61&imprToken=QiG2bf7fc5-czG6VLZ9cSg&slotNum=164)Ancient DNA has confirmed for many that non-Africans have Neanderthal ancestry. But there have been several debates about whether there are issues with the assumption that Africans have no Neanderthal ancestry, and how it skews statistics (e.g., if Africans have some Neanderthal one will underestimate the Neanderthal fraction). Though there are still details to be hashed out, **looking at coalescence patterns of haplotypes the authors seem to be able to infer the presence of deeply diverged lineages in various populations without positing a prior model of which populations did not have the introgression as a baseline**. Basically, Neanderthal and Denisovan ancestry is going to result in some “long branches” in the phylogenies of the genes within non-African populations which are lacking in Africans, and that is what they see.

These researchers also confirm the model presented by others that Neanderthal contribution seems to have been from a single admixture event (I do wonder if perhaps Neanderthals were not simply extremely homogeneous, so multiple close admixture events may not be differentiable). They also find that the “Denisovan” population structure was more complex, and there were several admixture events into eastern Eurasian and Oceanian populations.

Finally, there are attempts to adduce the nature of population differentiation, and times of separation. As noted in the text all of these sorts of analyses are sensitive to assumptions within models. They used a variety of methods which came to different results, but, one thing that seems clear is that **Africa had a lot of deep structure for a long time, but gene flow between regional populations meant that genetic differentiation emerged gradually,** rather than in a rapid fashion due to geographic separation. Over five years ago [Iain Mathieson](https://www.med.upenn.edu/apps/faculty/index.php/g306/c404/p8942206) casually told me that he viewed much of the past 200,000 years as the collapse of deep population structure, and that does not seem to have been a crazy prediction if you read through this preprint (though the collapse may be increased rates of gene flow, rather than massive pulse admixtures).

But the separation and differentiation outside of Africa, and between the archaic lineages and Africans, seems to have exhibited more punctuation. For the [past twenty years](https://www.ncbi.nlm.nih.gov/pubmed/?term=Population+Bottlenecks+and+Pleistocene+Human+Evolution+hawks) John Hawks has been emphasizing that we need to remember that during Pleistocene Africa likely had a much larger population than the rest of the world for hominins (with perhaps a caveat for lower latitude Asia). The relatively “clean” separation between the proto-modern African lineage and the Eurasian hominins, and then the quick separation between Neanderthals and the eastern group which became Denisovans, emphasizes perhaps the importance of particular geographic barriers (deserts in the Near East), as well as the lower carrying capacity in much of Eurasia. With lower population densities and patchy occupation patterns, gene flow would be sharply reduced. This would result in drift and sharply different lineages.

There are arguments out there about whether humans are a clinal species or not. These verbal descriptions really don’t tell us much. The combination of ancient DNA and whole genome data will allow us to specific at specific times and places the nature of population dynamics. If human population relationships can be thought of as a graph, a set of interconnected edges, in some areas the connections will be thicker (ergo, lots of continuous gene flow), and in other areas, the graphs will be easier to represent as diverging trees.

**I think the last 10,000 years of the Holocene has brought to Eurasia a more African pattern, as deep structure comes crashing down due to rapid population expansion and mixing….**

### Related Posts:

- [Luca Cavalli-Sforza
  biography](https://www.gnxp.com/WordPress/2005/11/21/luca-cavalli-sforza-biography/) - [What L. L. Cavalli-Sforza got
  wrong?](https://www.gnxp.com/WordPress/2008/04/21/what-l-l-cavalli-sforza-got-wrong/) - [Interview with Luigi Luca Cavalli-Sforza in
  Nature](https://www.gnxp.com/WordPress/2007/10/17/interview-with-luigi-luca-cavalli-sforza-in-nature/) - [10 questions for Luigi Luca
  Cavalli-Sforza](https://www.gnxp.com/WordPress/2006/08/24/10-questions-for-luigi-luca-cavalli-sforza/) - [A picture is worth a thousand words, part
  n](https://www.gnxp.com/WordPress/2008/04/21/a-picture-is-worth-a-thousand-words-part-n/) - [Must read
  book!](https://www.gnxp.com/WordPress/2008/01/22/must-read-book/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F06%2F27%2Fwhole-genome-sequencing-comes-to-cavalli-sforzas-data%2F&linkname=Whole%20genome%20sequencing%20%20comes%20to%20Cavalli-Sforza%E2%80%99s%20samples "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F06%2F27%2Fwhole-genome-sequencing-comes-to-cavalli-sforzas-data%2F&linkname=Whole%20genome%20sequencing%20%20comes%20to%20Cavalli-Sforza%E2%80%99s%20samples "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F06%2F27%2Fwhole-genome-sequencing-comes-to-cavalli-sforzas-data%2F&linkname=Whole%20genome%20sequencing%20%20comes%20to%20Cavalli-Sforza%E2%80%99s%20samples "Email")[](https://www.addtoany.com/share)
