+++
title = "Common disease, single"
full_title = "Common disease, single gene?"
date = "2007-08-16"
upstream_url = "https://www.gnxp.com/WordPress/2007/08/16/common-disease-single-gene/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/08/16/common-disease-single-gene/).

Common disease, single gene?

The prevailing consensus on “common” or “complex” diseases is that they result from the interplay of multiple genetic and environmental inputs. This is assumed largely because results from [linkage studies](https://www.gnxp.com/blog/2006/07/linkage-versus-association-mini-primer.php) were discouraging, which rules out a single, highly-penetrant gene as a possibility. There are many disease models where linkage studies could fail, however; a model involving multiple factors just seems to make more sense than the alternatives (and has been confirmed for a growing number of diseases). But consider [this paper](http://www.sciencemag.org/cgi/content/abstract/1146554) on a certain type of glaucoma:

> Glaucoma is a leading cause of irreversible blindness. A genome-wide > search yielded multiple SNPs in the 15q24.1 region associated to > glaucoma. Further investigation revealed that the association is > confined to exfoliation glaucoma (XFG). Two non-synonymous SNPs in > exon 1 of the gene LOXL1 explain the association and the data suggest > that they confer risk to XFG mainly through exfoliation syndrome > (XFS). **Approximately 25% of the general population is homozygous for > the highest risk haplotype and their risk of suffering XFG is over 100 > times that of those only carrying low-risk haplotypes.**

Crunching the numbers on the table in the publication suggests that, in the population over 60, people carrying the allele have a 22% chance of developing the disease, while people without it have about a 1.5% chance (the difference between my calculations and those in the abstract is that I’m considering presence or absence of the allele, rather than the genotype, and our numbers on the prevalence of the disease might be slightly different–I used 15%). That’s a huge effect, and the authors claim that the [population attributable risk](http://www.statsdirect.com/help/miscellaneous/attributable_risk.htm) of the risk haplotype is almost 100%– that is, if the haplotype were to disappear from the population (which is obviously not a real possibility), the disease would as well.

### Related Posts:

- [Common disease, common
  variant](https://www.gnxp.com/WordPress/2006/12/01/common-disease-common-variant/) - [Defining "synthetic associations"
  down](https://www.gnxp.com/WordPress/2010/02/22/defining-synthetic-associations-down/) - [Linkage versus association: a
  mini-primer](https://www.gnxp.com/WordPress/2006/07/06/linkage-versus-association-a-mini-primer/) - [How predictive are known genetic factors for disease
  risk?](https://www.gnxp.com/WordPress/2008/11/19/how-predictive-are-known-genetic-factors-for-disease-risk/) - [A note on the Common Disease-Common Variant
  debate](https://www.gnxp.com/WordPress/2007/02/27/a-note-on-the-common-disease-common-variant-debate/) - [Mitochondrial
  genetics](https://www.gnxp.com/WordPress/2007/06/11/mitochondrial-genetics/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F16%2Fcommon-disease-single-gene%2F&linkname=Common%20disease%2C%20single%20gene%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F16%2Fcommon-disease-single-gene%2F&linkname=Common%20disease%2C%20single%20gene%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F08%2F16%2Fcommon-disease-single-gene%2F&linkname=Common%20disease%2C%20single%20gene%3F "Email")[](https://www.addtoany.com/share)
