+++
title = "A mechanism for"
full_title = "A mechanism for miRNAmediated repression"
date = "2007-06-19"
upstream_url = "https://www.gnxp.com/WordPress/2007/06/19/a-mechanism-for-mirna-mediated-repression/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/06/19/a-mechanism-for-mirna-mediated-repression/).

A mechanism for miRNA-mediated repression

RNA interference is a process by which small (20-22 nt) RNAs bind to a fully or partially complementary messenger RNA and reduce the amount of protein product from that mRNA. The general rule is that if the match is perfect (full complementarity) then the target mRNA is cut into two pieces and destroyed forthwith. If the match is imperfect such that there are bulges in the double stranded RNA that forms between the interfering RNA and the target, then the target is sequestered to a newly discovered cellular entity called a Processing Body (P Bodies, PBs). There are enzymes in PBs capable of degrading mRNAs, but sometimes the mRNAs can be released and become translationally competent again.

New research from [Kiriakidou et al](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WSN-4NT9TB9-1&_user=702304&_coverDate=06%2F15%2F2007&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000038878&_version=1&_urlVersion=0&_userid=702304&md5=ebee4fee2eec1ab4b8f1bc8b63819049) in Cell provides a mechanism for this translational repression sans degradation. The effects of small interfering RNAs (siRNAs) are mediated by the Argonaute family of proteins (Ago1, Ago2, etc). This family can be subdivided depending on the proteins’ ability to cleave RNA and thus carry out the “perfect-match” type of translational repression, but even non-cleaving Agos can do the sequestration route for repression. The latest news is that this can be achieved by blocking interactions between the cap-binding translation initiation factor eIF4E and the 5′ cap of mRNAs.

Let me unpack. For efficient initiation of protein synthesis from an mRNA, several proteins must assemble into complexes centered around the mRNA. There are several proteins that bind near the other end of the mRNA where there is a cap. A cap is a modified guanine nucleotide flipped around backward and stuck on the head-end of the mRNA early in its life. One protein in particular, eIF4E recognizes the cap structure and binds to it, recruiting other initiation factors and eventually the small ribosomal subunit. This is an important and highly regulated step in protein synthesis. For instance, there is a family of proteins (4E-BPs) whose sole function is to bind eIF4E and get in the way of cap-binding. If they become highly phosphorylated because of this signaling pathway or that, they let go and translation proceeds. Ago proteins can do the same thing, but on the cap side and without the phosphorylation business.

They showed the effect by first purifying an Ago protein with and without important amino acids for cap-interaction and testing for binding with caps immobilized on a column. Only Ago proteins with the two important (phenylalanine) amino acids could bind. Further assays in vivo showed that the mutant Agos couldn’t mediate translational repression.

There are a couple predictions to make based on these findings.

1\) Organisms with Agos that lack this domain should be bad at this process.

> This domain is not found in Ago proteins of plants, archaea, or fission yeast, in Drosophila AGO2 and in most members of the C. elegans Ago protein family, with the exception of ALG-1 and ALG-2. In addition, the MC domain is absent from proteins of the PIWI family.

I can’t recall if any of there is anything already contradictory in that list. I think there is definitely something weird about the way plants handle siRNAs, but the details escape me.

2\) RNAs that are capable of cap-independent translation should not be regulated by this process. There is debate about the degree to which mRNAs can undergo cap-independent translation, but the field is moving along as though internal ribosomal entry sites are an important cellular tool, so these RNAs should escape translational repression via this process.

### Related Posts:

- [A Nobel Prize for gene
  expression](https://www.gnxp.com/WordPress/2006/10/02/a-nobel-prize-for-gene-expression/) - [Slicer structure, piRNAs, and so much
  more](https://www.gnxp.com/WordPress/2007/01/16/slicer-structure-pirnas-and-so-much-more/) - [RNA
  activation](https://www.gnxp.com/WordPress/2006/11/02/rna-activation/) - [RNAi
  fundamentals](https://www.gnxp.com/WordPress/2006/06/08/rnai-fundamentals/) - [More on non-coding
  RNAs](https://www.gnxp.com/WordPress/2006/09/19/more-on-non-coding-rnas/) - [HAR1F
  tangents](https://www.gnxp.com/WordPress/2006/08/17/har1f-tangents/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F19%2Fa-mechanism-for-mirna-mediated-repression%2F&linkname=A%20mechanism%20for%20miRNA-mediated%20repression "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F19%2Fa-mechanism-for-mirna-mediated-repression%2F&linkname=A%20mechanism%20for%20miRNA-mediated%20repression "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F19%2Fa-mechanism-for-mirna-mediated-repression%2F&linkname=A%20mechanism%20for%20miRNA-mediated%20repression "Email")[](https://www.addtoany.com/share)
