+++
title = "Recent human evolution"
full_title = "Recent human evolution"
date = "2007-06-04"
upstream_url = "https://www.gnxp.com/WordPress/2007/06/04/recent-human-evolution/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/06/04/recent-human-evolution/).

Recent human evolution

The major genome scans for selection in humans (e.g. [Voight et al. 2006](http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16494531&query_hl=4&itool=pubmed_docsum) and [Wang et al. 2006](http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Retrieve&db=PubMed&list_uids=16371466&dopt=Abstract)) have all been focused on incomplete selective sweeps– alleles that are on their way to fixation in a given population, but haven’t quite made it. A [new paper in PLoS Genetics](http://genetics.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pgen.0030090) takes a look at those alleles that have already made it in one population or another, developing a robust test to find fixed population differences.

This is a magnificent paper, and should be required reading for anyone interested in the field of human evolutionary genetics. There’s a ton of information condensed into this article; I’ll run down some of the most important points:

1\. The major problem people have with some genome scans is that, while they generate a number of candidate loci for selection, there’s no real way to separate out those likely most fruitful for follow-up. These authors use a statistic that is robust to many demographic situations, recombination rate heterogeneity, and just about anything else you can throw at it. The loci they identify are not just enriched for true selective events; the majority are worth following up.

2\. They find selected loci in pathways for skin color (including [SLC24A5](https://www.gnxp.com/blog/2005/12/race-is-skin-deep.php)) and hair morphology in Europeans. **It’s always seemed somewhat obvious that the major visible differences between population groups should be selected for, but this provides important evidence in favor of that**. Various genes involved in nervous system development (including genes known to be involved in Alzheimer’s) have also been under selection in various populations. Finally, they note an abundance of heat shock proteins under selection, which seems surprising. I really have no explanation for that.

3\. Centromeric regions of many chromosomes have been under selection. I find this fascinating, as the most likely hypothesis for the reason why is [meiotic drive](https://en.wikipedia.org/wiki/Meiotic_drive#Meiotic_drive). This force could turn out to be a major player in evolution.

4\. We’ve mentioned hemochromatosis here before, essentially it’s a hereditary genetic disorder that appears at a particularly high frequency in Europeans, which seems to be the result of selection. The authors here do indeed find the signature of selection at this locus, **but the signal is centered at a cluster of histone genes 150kb away**. So maybe hemochromatosis has “hitch-hiked” its way to high frequency.

5\. They find fewer signals of selection in the African-American sample than in the other samples. I’m generally skepical of claims like this, and they note that they indeed had less power to detect sweeps in the African-Americans (since they’re looking at fixed differences, European admixture into the sample should reduce power). It’s nice that they look at this; I recall [another paper](https://www.gnxp.com/blog/2007/05/human-v-chimp-evolutionary-showdown.php) that made claims about absolute frequencies of selection along different lineages without considering power at all.

6\. There seems to be much evidence for selection making populations different, less so for selection affecting all populations equally. Human evolution is continuing, and making us genetically different.

For those who want to browse, here’s a [link to a table](http://genetics.plosjournals.org/perlserv/?request=slideshow&type=table&amp;doi=10.1371/journal.pgen.0030090&id=11555) with the 101 regions with the strongest evidence for selective sweeps.

### Related Posts:

- [Localizing recent adaptive evolution in the human
  genome](https://www.gnxp.com/WordPress/2007/04/20/localizing-recent-adaptive-evolution-in-the-human-genome/) - [Humans: nothin'
  special?](https://www.gnxp.com/WordPress/2006/09/17/humans-nothin-special/) - [Quantitative genetics strikes back!
  (?)](https://www.gnxp.com/WordPress/2010/02/27/quantitative-genetics-strikes-back/) - [Important papers on recent human
  evolution](https://www.gnxp.com/WordPress/2007/09/29/important-papers-on-recent-human-evolution/) - [More detecting natural
  selection](https://www.gnxp.com/WordPress/2007/06/18/more-detecting-natural-selection/) - [ASPM, Microcephalin, and
  intelligence](https://www.gnxp.com/WordPress/2007/01/16/aspm-microcephalin-and-intelligence/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F04%2Frecent-human-evolution%2F&linkname=Recent%20human%20evolution "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F04%2Frecent-human-evolution%2F&linkname=Recent%20human%20evolution "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F06%2F04%2Frecent-human-evolution%2F&linkname=Recent%20human%20evolution "Email")[](https://www.addtoany.com/share)
