+++
title = "“Dominant” & “recessive”"
full_title = "“Dominant” & “recessive”"
date = "2006-07-15"
upstream_url = "https://www.gnxp.com/WordPress/2006/07/15/dominant-recessive/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/07/15/dominant-recessive/).

“Dominant” & “recessive”

In my post below on the phenotypic difference in sensitivity to PTC (and ergo, bitterness), I allude to the fact that until recently the inheritance mode of this simple mongenic Mendelian trait (that is, controlled from one locus in the genome) was assumed to express via a dominance-recessive relationship on the phenotypic level. In other words, if you were heterozygous, you were a taster, and only homozygous individuals for non-taster alleles (two copies with lack of function) would express the “recessive” phenotype. Well guess what? Turns out that there are *three phenotypes*, and they seem to express in a fashion that is additive, so that heterozygous individuals express a phenotype *between* the two homozygotes.

  
Now, the reality is that if you discarded all traits from the dominant-recessive paradigm where heterozygotes deviated from the median expression of the dominant homozygote, **almost no traits would follow this pattern strictly** (i.e., dominant homozygote & heterozygote exhibit the exact same level of phenotypic expression). The reason is that most heteozygotes suffer some deviation from perfect dominance as their functioning copy does not perfectly compensate. For example, if the recessive phenotype is 0, and the dominant (homozygote) set to 1, the dominantly expressed heterozygote is often 0.9. Operationally this is close enough to 1 so that you can lump the two together, but in the long term evolutionary context this is not trivial, especially if the phenotypic expression has a correlation with fitness. Additionally, [haploinsufficiency](http://ghr.nlm.nih.gov/ghr/glossary/haploinsufficiency) over life history can be important in some cases. By this, I mean that over time mutational rates via somatic duplication may eventually result in many diploid cells expressing only one “good copy” because of errors which get past DNA repair mechanisms. In the case of individuals who begin with only functioning copy of a gene there is a risk that as they age a large proportion of their cease functioning properly because of mutations.^(a)  
My overall point is to offer that the narratives most of us are taught about dominance and recessive hide the fact that these traits often are highly vulnerable to deconstruction because their existence is a matter of classification. Further investigation often yields up cryptic phenotypic variation which belies the dual nature initially assayed (as in the case of taste), while in other cases the nature of classification itself may generate the dual scheme of dominance and recessivity as an artifact.  
Below is a repost from the archives on dominance & recessiveness, as I summarize a few points in a paper criticizing the concept. If this was Greek to you, you should go read Lei [on heterozygous and homozygous](http://www.geneticsandhealth.com/2006/07/15/weekly-genetics-quiz-48-homozygous-and-heterozygous/).  
Note:  
a – Consider an individual who is homozygous “wild type” vs. one who is heterozygous dominant. Let is us assume that their cell functioning is identical, that is, the “good copy” perfectly compensates for the “bad copy.” If at 40 10% of the alleles in a given individual have mutated in somatic cells, a fully “wild type” individual will only exhibit 1% loss of function across their cells because 90% of the mutants will reside in cells that are heterozygous. In contrast, the dominant heterozygote individual will lose 10% of function because there is no “back up” copy in their cells.  
**\[being repost\]**  
Over the past few years one problem I have had when discussing issues of genetics with people are confusions issuing out of high school recollections of the dominant-recessive concept. Because this concept is introduced rather early on and emphasized through the ubiquitous [Punnet Square](http://www.borg.com/~lubehawk/psquare.htm) there is often confusion about how polygenic additive traits can exist in light of the perceived centrality of dominance and recessive. Nearly a century ago R.A. Fisher theoretically [reconciled](http://www.library.adelaide.edu.au/digitised/fisher/9.pdf) (PDF) polygenic continuous variation (quantitative genetics) with a Mendelian paradigm, but obviously most people aren’t going to be reading decades old population genetics papers, and I am not sure that such a mathematical exposition is easily internalized by most.  
Recently I stumbled on to this [philosophical diatribe](http://my.pclink.com/~allchin/papers/dissolve.htm) against the concept of dominance in biology. The author basically seems to suggest that we should ditch the emphasis that we place on dominance and recessive and start from first principles and focus on the idea that in diploid organisms two copies of a given gene are expressed on any locus. To me this argument is made particularly powerful by the molecular revolution, for now we can illustrate how dominance might actually work in the context of the [Central Dogma](http://www.accessexcellence.org/RC/VL/GG/central.html) and a few basic concepts that are already partially digested from the cultural *zeitgeist*.  
Consider that you have an organism which processes nutrient X via enzyme *z*. Enzyme *z* is synthesized like so:  
Gene *z* – \> transcription of mRNA -\> translation of mRNA -\> Enzyme *z*  
Now, imagine that there is a “null” copy of Gene *z*, a mutation on the promoter prevents the initiation of transcription. In this case null-Gene-*z* results in lack of production of enzyme *z*, ergo, the inability to process nutrient X.  
But in diploid organisms you have two copies of a gene. If you have a normal copy and a null copy there will be production of the enzyme, but only half the normal amount of enzyme will be generated because only half the normal amount of DNA is transcribed.¹ If 1/2 of the normal concentration of enzyme is sufficient to break down enough of nutrient X so that the organism seems healthy one might say that the trait is “dominant,” but at the molecular level the heterozygote is expressing only “half” the phenotype as a pure “wild type.”  
It might be that in a normal context there is *almost* never enough of nutrient X in the environment so that more than 50% of the enzyme *z* is necessary for optimal functioning (the abundance of nutrient X is the limiting factor). But there could be rare circumstances where nutrient X is very abundant and those organisms who have both copies of the functioning gene can take advantage of their enzymatic advantage and store the end product of nutrient X for future use when there is a deficit. In this case there is a clear fitness advantage that wild type homozygotes have over heterozygotes, but it is not normally observed over short time scales. Additionally, there is the possibility that gene *z* is prone to mutation in the somatic (body) cells that process nutrient X, so those individuals with some level of redunancy might have a long term life history advantage over heterozygotes who might harbor a multitude of sub-optimal cells which fail to process nutrient X by the end of their life because they have no “back up” plan.  
In any case, I think that this narrative does have conceptual advantages over the dominance-recessive idea, and is simple enough that it can be introduced at the secondary educational level. As the author of the paper above notes it would also render terms like “codominance,” “incomplete dominance,” “penetrance,” etc. redundant. And the idea of starting at the molecular level might also make an understanding of polygenic traits a bit more comprehensible, and in the future **polygenic traits are going to loom large in the public imagination as the low hanging single locus fruit are picked clean and scientific press releases begin to skew toward more complex phenotypes**.  
1 – This is simplistic, there are a lot of issues relating to how/why/to what extent genes get expressed. One could perhaps concoct a model where the organism up regulates gene expression by producing a factor which induces more frequent binding of the polymerase when it “detects” that nutrient X is not being metabolized efficiently. We’ll neglect such counter scenarios.

### Related Posts:

- [Dominance & recessive, is it
  worthwhile?](https://www.gnxp.com/WordPress/2006/10/01/dominance-recessive-is-it-worthwhile/) - [Taste &
  genetics](https://www.gnxp.com/WordPress/2006/12/03/taste-genetics/) - [Lactose tolerance, is it
  "dominant"?](https://www.gnxp.com/WordPress/2006/12/05/lactose-tolerance-is-it-dominant/) - [A matter of
  taste](https://www.gnxp.com/WordPress/2006/07/13/a-matter-of-taste/) - [Beyond the Punnett Square, part
  n](https://www.gnxp.com/WordPress/2005/10/09/beyond-the-punnett-square-part-n/) - [Humans and chimps & bitter taste = convergent
  evolution](https://www.gnxp.com/WordPress/2006/04/16/humans-and-chimps-bitter-taste-convergent-evolution/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F07%2F15%2Fdominant-recessive%2F&linkname=%E2%80%9CDominant%E2%80%9D%20%26%20%E2%80%9Crecessive%E2%80%9D "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F07%2F15%2Fdominant-recessive%2F&linkname=%E2%80%9CDominant%E2%80%9D%20%26%20%E2%80%9Crecessive%E2%80%9D "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F07%2F15%2Fdominant-recessive%2F&linkname=%E2%80%9CDominant%E2%80%9D%20%26%20%E2%80%9Crecessive%E2%80%9D "Email")[](https://www.addtoany.com/share)
