+++
title = "Selection, drift,"
full_title = "Selection, drift, disease and complexity, all rolled into one…"
date = "2008-06-24"
upstream_url = "https://www.gnxp.com/WordPress/2008/06/24/selection-drift-disease-and-complexity-all-rolled-into-one/"

+++
Source: [here](https://www.gnxp.com/WordPress/2008/06/24/selection-drift-disease-and-complexity-all-rolled-into-one/).

Selection, drift, disease and complexity, all rolled into one….

One of the great things about [evolutionary genetics](http://www.science.uva.nl/~seop/entries/evolutionary-genetics/) is that it is such a diverse field in terms of the cognitive toolkit which one must access as a matter of course. Since R. A. Fisher’s [The Genetical Theory of Natural Selection](https://www.amazon.com/exec/obidos/ASIN/0198504403/geneexpressio-20/) (along with the contemporaneous work of [Sewall Wright](https://en.wikipedia.org/wiki/Sewall_Wright) and [J. B. S. Haldane](https://en.wikipedia.org/wiki/J._B._S._Haldane)) we’ve been habituated toward thinking of evolutionary processes on an abstract level which might allow us to make general deductive inferences from first principles. [Genetic drift](https://en.wikipedia.org/wiki/Genetic_drift), [selection](https://en.wikipedia.org/wiki/Natural_selection), migration, etc., are parameters which are used to construct models that allow us to generate predictions and obtain deeper insight. The discovery of DNA and the elucidation of the biophysical substrate which constrains the modes of inheritance in a concrete manner opened up the startling vistas of molecular evolutionary genetics. This discipline has allowed an inspection of how the predictions of evolutionary theory are born out on a more fine grained level. And today the genomics revolution is ramping up the data sets as computational power enables more powerful extraction of the patterns and dynamics which emerge out of these discrete streams of information.  
But this is all rather philosophical and abstract. Yesterday I posted on a paper which showed how selection and drift might have operated upon the frequency of an allele which has a disease implication, and so has a pragmatic impact on quality of life. Today I’d like to bring your attention to another paper which synthesizes the big picture ideas which might entail consequences in terms of the utilitarian details of daily life, [Natural Selection on Genes that Underlie Human Disease Susceptibility](http://www.current-biology.com/content/article/abstract?uid=PIIS0960982208006015&feed=CURBIO):

> What evolutionary forces shape genes that contribute to the risk of > human disease? Do similar selective pressures act on alleles that > underlie simple versus complex disorders …Answers to these questions > will shed light onto the origin of human disorders…and help to predict > the population frequencies of alleles that contribute to disease risk, > with important implications for the efficient design of mapping > studies…As a first step toward addressing these questions, we created > a hand-curated version of the Mendelian Inheritance in Man database > (OMIM). We then examined selective pressures on Mendelian-disease > genes, genes that contribute to complex-disease risk, and genes known > to be essential in mouse by analyzing patterns of human polymorphism > and of divergence between human and rhesus macaque. **We found that > Mendelian-disease genes appear to be under widespread purifying > selection, especially when the disease mutations are dominant (rather > than recessive). In contrast, the class of genes that influence > complex-disease risk shows little signs of evolutionary conservation, > possibly because this category includes targets of both purifying and > positive selection.**

![natselfigure4.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/natselfigure4.jpg?resize=330%2C187)![natselfigure4.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/natselfigure4.jpg?resize=330%2C187)To the right you see Figure 4 from this paper. It’s a chart which shows the [cumulative distribution](http://www.mathepi.com/comp/Images/pnorm.jpg) of the ratio of the nonsynonymous substitutions divided by synonymous substitutions (comparing between humans and rhesus macaques), where the latter in the paper is defined by D_(n)/D_(s), so that D_(n) = [nonsynonymous substitution](http://helix.mcmaster.ca/721/distance/node7.html) and D_(s) = [synonymous substitution](https://en.wikipedia.org/wiki/Synonymous_substitution). A substitution is a process where one allele, genetic variant, replaces another at a locus over time. ~~Non~~synonymous refers to a change in the genome which should not change function, while a nonsynonymous may, because of the manner in which they alter the [coding of amino acids](https://en.wikipedia.org/wiki/Genetic_code#Degeneracy_of_the_genetic_code) from DNA. In this chart hOMIM refers to “hand-curated version” of [Mendelian Inheritance in Man database](http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim). Complex refers to genes implicated in complex-disease susceptibilities (e.g., “You carry allele *Scary*, that means you have a 5% chance of developing a really scary disease by the age of 60″). Cancer, Essential and Other are pretty straightforward, at least in terms of label.  
Does the results fit our expectations? First, the x-axis represents the value of D_(n)/D_(s), while the y-axis shows the proportion of genes which fall at that threshold. The y-axis converges upon 1 because obviously you can’t go over 100% by definition . The Cancer and Essential distributions are shifted furthest to the left. That means that a higher proportion of genes have low D_(n)/D_(s), while you see that the Complex line is the one that rises up the most slowly and has the greatest number of genes which exhibit a high D_(n)/D_(s) (the color isn’t clear at this resolution, but trust me, it is if you have higher res). hOMIM is somewhere in the middle. In short, neutrality is a more powerful force in the category of genes which are shifted further to the right, while forms of selection are presumably more powerful (purifying selection which removes mutants, or positive selection which fixed new mutants) for distributions shifted to the left.  
I would have to say that the distributions here are not totally surprising based on other things we know, this is an empirical confirmation to a great extent of rules-of-thumb which many hold because of the theoretical and experimental insights of a century. For example, it is well known that complex-traits which exhibit a continuous distribution and are highly heritable tend to have weak fitness implications. Conversely, Mendelian diseases are usually classified as diseases for a reason! Additionally, the authors find that diseases which are expressed dominantly, that is, one copy results in the disease, have lower values of D_(n)/D_(s), than those which express recessively so that two copies are necessary. This is what we would expect from the fact that when low frequency alleles which only express as homozygotes are segregating within the population randomly most copies are carried within heterozygotes who are not subject to selection; in other words, there is little purification of these genes unless their frequencies are very high as per [Hardy-Weinberg](https://en.wikipedia.org/wiki/Hardy-Weinberg_principle). To make the difference between complex-disease loci and Mendelian ones more concrete, think of it in a non-disease context. Height is a quantitative trait, while eye color seems quasi-Mendelian. HMGA2 is a height locus which explains 0.3% of the variation within a population for the trait in question, while the region around [OCA2](https://www.google.com/cse?cx=017254414699180528062%3Auyrcvn__yd0&q=oca2+site%3Ahttp%3A%2F%2Fscienceblogs.com%2Fgnxp%2F&sa=Search) seems to account for 75% of the variation in blue-brown eye color. In addition the region around OCA2 may have been subject to selection and this selection may explain the difference in eye color across populations. It seems unlikely that we’ll find strong signatures around height loci that explain the variation of height across populations.  
**As the human genome is mapped with greater detail, and the parameters (selection, drift, etc.) are teased apart we will gain some understanding of the genetic past of our species as well as its current trajectory.** Loci which are implicated in large effects because of their variation are the low hanging fruit. The parameters are going to be easier to examine, after all, it’s one locus (or two, or six, etc.). On the other hand, when you are finding genes which account for 0.1% of the variance of a trait, and are happy to find even those genes, as in height, the attention to detail and caution for confounds are necessary to separate the wheat from chaff. If you read the above paper closely you’ll see that the words “[ascertainment bias](https://en.wikipedia.org/wiki/Ascertainment_bias)” crops up over and over, but unfortunately it is not as often found in popular press summaries of these sorts of results and inferences. The authors note that we are just at the beginning of mapping for complex-disease susceptibilities. The bioethical conundrums which emerged due to the discovery of smaller effect but more numerous breast cancer susceptibility loci last winter after the splash which BRCA1 made is a taste of what is to come. One thing is for sure, the future is going to be more complex….  
**Cite:** Current Biology, Vol 18, 883-889, 24 June 2008, Natural Selection on Genes that Underlie Human Disease Susceptibility, Ran Blekhman, Orna Man, Leslie Herrmann, Adam R. Boyko, Amit Indap, Carolin Kosiol, Carlos D. Bustamante, Kosuke M. Teshima, and Molly Przeworski.

### Related Posts:

- [Notes on Sewall Wright: Population
  Size](https://www.gnxp.com/WordPress/2008/06/06/notes-on-sewall-wright-population-size/) - [Notes on Sewall Wright:
  Migration](https://www.gnxp.com/WordPress/2008/07/03/notes-on-sewall-wright-migration/) - [Sewall Wright & genetic
  drift](https://www.gnxp.com/WordPress/2008/05/09/sewall-wright-genetic-drift/) - [Math and
  Creation](https://www.gnxp.com/WordPress/2006/07/12/math-and-creation/) - [The Genetical Theory of Natural
  Selection](https://www.gnxp.com/WordPress/2011/12/27/the-genetical-theory-of-natural-selection/) - [Sewall Wright, a father of evolutionary
  genetics](https://www.gnxp.com/WordPress/2007/03/25/sewall-wright-a-father-of-evolutionary-genetics/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F06%2F24%2Fselection-drift-disease-and-complexity-all-rolled-into-one%2F&linkname=Selection%2C%20drift%2C%20disease%20and%20complexity%2C%20all%20rolled%20into%20one%E2%80%A6. "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F06%2F24%2Fselection-drift-disease-and-complexity-all-rolled-into-one%2F&linkname=Selection%2C%20drift%2C%20disease%20and%20complexity%2C%20all%20rolled%20into%20one%E2%80%A6. "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F06%2F24%2Fselection-drift-disease-and-complexity-all-rolled-into-one%2F&linkname=Selection%2C%20drift%2C%20disease%20and%20complexity%2C%20all%20rolled%20into%20one%E2%80%A6. "Email")[](https://www.addtoany.com/share)
