+++
title = "Nice 'N Slow how to"
full_title = "Nice 'N Slow how to make more GCN4"
date = "2007-05-08"
upstream_url = "https://www.gnxp.com/WordPress/2007/05/08/nice-n-slow-how-to-make-more-gcn4/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/05/08/nice-n-slow-how-to-make-more-gcn4/).

Nice 'N Slow: how to make more GCN4

Let me take you to a place nice and quiet. There ain’t no one there to interrupt. Ain’t gotta rush. I just wanna take it nice and slow. – US-HER RA-YM-OND

I just summarized in a [previous post](https://www.gnxp.com/blog/2007/05/intro-to-stress-induced-translation.php) how eIF2alpha kinases can reduce the amount of a crucial resource needed for initiation of protein synthesis, namely eIF2-GTP. There are four well-characterized eIF2alpha kinases: PKR, PERK, GCN2 and HRI. I was previously a little vague in my characterization of the inducing conditions for this pathway. The four kinases are activated by double-stranded RNA (representing viral infection), ER stress (i.e. protein misfolding), amino acid starvation, and low heme (iron scarcity), respectively. Regardless of the specific cause, the general effect is to suppress protein synthesis cell-wide while the cell deals with some perturbation. Still, for the cell to go about the business of handling its biz, it has to make a few key proteins. Indeed, it might even want to make more of these proteins than it was making before.

By far, the most heavily studied of these pathways is the GCN2 response to amino acid starvation. Recall that proteins are a string of amino acids. The secret decoder ring to translate the language of nucleotides into that of amino acids is the transfer RNA (tRNA). Transfer RNAs usually have a triplet of nucleotides at one end and an amino acid at the other, like an adapter. If a cell is low on amino acids, tRNAs might get made with on amino acid on the other end. if GCN2 finds out this type of tomfoolery has been going on it brings the hammer down and phosphorylates eIF2alpha, putting a large restraint on protein synthesis. That’s all well and good; it’s like putting your state under martial law for a minute during a crisis. It’s not enough though because if the authoritarian GCN2 regime just repressed every attempt to get a message out, the cell would lose its vitality. Instead, the global repression of protein synthesis has an immediate positive effect: GCN4 synthesis increases. GCN4 is a transcription factor and can thus affect exactly which types of programs the cell is putting its resources toward. GCN4 is a practical transcription factor. Given the situation of amino acid shortages, and it drives the cell to produce more genes in the amino acid biosynthesis pathway.

So how can GCN4 escape the crackdown and curry the favor of the ribosomes? The answer lies in the first \~600 nucleotides of the mRNA coding for GCN4. The sequences in this area, called the 5′ untranslated region, allow GCN4 to play it kind of coy with the ribosome. If the ribosome gets things started with GCN4 mRNA, it might get to keep going for a little bit, but it eventually hits roadblocks. The only way for a ribosome to get to the main task of really translating GCN4 is to take it nice and slow and sometimes skip opportunities to make a move. You may not know about the primary structure of mRNAs though, so let’s take a brief look.

[](https://www.gnxp.com/blog/uploaded_images/mrna-structure-722015.jpg)  
The main thing to note is that I am an artistic genius. After that, you can note that the whole RNA doesn’t code for protein. There are big chunks that hang off the 5′ and 3′ ends of the protein-coding region (AKA the open reading frame). 5′ and 3′ refer to specific features of nucleic acid structures, but all you need to know is that ribosomes read from 5′ to 3′, and since we speak American around here, we will put the 5′ end on the left. To initiate translation the ribosome assembles with several other factors including eIF2-GTP at the very 5′ end of the mRNA. There is a structure here called the 5′ cap that is recognized by the initiation complex. The assembled ribosome et alia start scanning the mRNA from 5′ to 3′. As it moves from left to right it will first encounter the 5′ untranslated region (UTR). The 5’UTR is made up of nucleotides, but the ribosome does not translate them into amino acids yet. The signal for a ribosome to start translating the genetic code and creating a new protein is a Start Codon. Start codons have the sequence AUG. The scanning ribosome carries a tRNA with it (called the initiator tRNA) that can base-pair with AUG and which carries the amino acid, methionine, on the other end. I don’t want to get involved with the mechanism for adding amino acids onto the chain. Suffice it to say that the once the ribosome “opens” a “reading frame” it reads the nucleotide code triplet by triplet and builds a protein. The final feature of the open reading frame is the Stop Codon. There are three nucleotide triplets that do not code for any amino acid. When the ribosome reaches these, it usually pauses for a while and then stops making proteins. I would’ve said that it falls off, but I am about to describe a process that depends on it continuing along in the 3′ direction. The last two features of your average mRNA are not of great importance to the current discussion. They are the 3′ UTR (more nucleotides not coding for proteins) and the polyA tail. The polyA tail promotes translation and RNA stability.

So to be very clear: an Open Reading Frame (ORF) is the part of the RNA that a ribosome actually reads between the Start and Stop codons. The role of eIF2-GTP is to bring the initiator tRNA to the ribosome, so that it can be used as soon as the ribosome finds a Start Codon. When the peptide is started, the eIF2-GTP is used up, and if that ribosome wants to start any other peptides, it has to have a new eIF2-GTP.

Now imagine that I lied to you about 5′ UTRs. You don’t have to imagine very hard. There are quite a few 5′ UTRs that contain open reading frames. They just aren’t the main course as it were. They are called upstream open reading frames (uORFs). A ribosome can start at the cap, read an uORF and synthesize a short protein, and then scan further down the mRNA to the real ORF. GCN4 mRNA, for instance, has four uORFs before the ORF that codes for the GCN4 protein. If you artificially construct a GCN4 mRNA that lacks these uORFs, you get a lot more GCN4 protein. The uORFs thus have the effect of inhibiting translation of the downstream ORF under normal conditions. Many of the experiments detailing the 5′ UTR pretty much ignore the middle uORFs and focus on uORF1 and uORF4 because these seem to be enough to get eIF2 kinase dependent expression.

The ribosome always reads uORF1 first because that’s what it encounters first during scanning. Under normal conditions, it is reloaded with eIF2-GTP and an initiator tRNA relatively rapidly and it can read ORF4. ORF 4 is a hangup. It causes the ribosome to tarry right at the end. This is partially because its last amino acid is proline, which is relatively rare and difficult to find, so the ribosome is losing momentum by the time it reaches the end of ORF4. As it tries to scan even further down to the GCN4 ORF, it encounters a little more opposition, throws up its hands, and dissociates from the mRNA. As a result, we get no GCN4. The magic of eIF2 kinases is that they reduce the availability of eIF2-GTP and thus delay re-initiation after translation of uORF1. If the ribosome takes just a little bit longer to be ready again, it an skip ORF4. See? Isn’t that cool? If it only reads ORF1 and takes its sweet time getting ready it can get all the way to the GCN4 start codon. In this way, GCN4 protein is increased while translation of the majority of other mRNAs is inhibited.

The protein of interest in [Costa-Mattioli et al](http://www.cell.com/retrieve/pii/S0092867407003248) is not GCN4, bu  
t it has a similar mechanism. Next I hope to describe some of the experiments one has to do to discover that an mRNA is controlled in this manner. I will focus on the articles that showed that the production of ATF4 protein is regulated by eIF2 kinases in mammalian cells.

### Related Posts:

- [Intro to stress-induced translation
  regulation](https://www.gnxp.com/WordPress/2007/05/06/intro-to-stress-induced-translation-regulation/) - [Global inhibition, specific
  activation](https://www.gnxp.com/WordPress/2006/10/10/global-inhibition-specific-activation/) - [Wheat from the
  chaff](https://www.gnxp.com/WordPress/2006/12/12/wheat-from-the-chaff/) - [A Nobel Prize for gene
  expression](https://www.gnxp.com/WordPress/2006/10/02/a-nobel-prize-for-gene-expression/) - [A mechanism for miRNA-mediated
  repression](https://www.gnxp.com/WordPress/2007/06/19/a-mechanism-for-mirna-mediated-repression/) - [RNA
  activation](https://www.gnxp.com/WordPress/2006/11/02/rna-activation/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F05%2F08%2Fnice-n-slow-how-to-make-more-gcn4%2F&linkname=Nice%20%27N%20Slow%3A%20how%20to%20make%20more%20GCN4 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F05%2F08%2Fnice-n-slow-how-to-make-more-gcn4%2F&linkname=Nice%20%27N%20Slow%3A%20how%20to%20make%20more%20GCN4 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F05%2F08%2Fnice-n-slow-how-to-make-more-gcn4%2F&linkname=Nice%20%27N%20Slow%3A%20how%20to%20make%20more%20GCN4 "Email")[](https://www.addtoany.com/share)
