+++
title = "What the substrate tells"
full_title = "What the substrate tells"
date = "2012-09-16"
upstream_url = "https://www.gnxp.com/WordPress/2012/09/16/what-the-substrate-tells/"

+++
Source: [here](https://www.gnxp.com/WordPress/2012/09/16/what-the-substrate-tells/).

What the substrate tells

One of the weird things about genetics is that it encompasses both the abstract and the concrete. The formal and physical. You can talk to a geneticist who is mostly interested in details of molecular mechanisms, and is steeped in [structural biology](https://en.wikipedia.org/wiki/Structural_biology). For these people genes are specific and material *things*. In contrast there are other geneticists who focus more on genes as units of *analysis*. In this case genes are semantic labels for the mediators within an intersection of phenomena. Recall **that genetics predates the knowledge of its concrete substrate by 50 years!** By the 1920s Mendelian genetics had been fused with evolutionary biology to create a systematic framework in which we could understand the patterns of inheritance across the generations. In the 1950s the DNA revolution was upon us, but as [W. D. Hamilton](https://en.wikipedia.org/wiki/W._D._Hamilton) recalls this had only a minimal impact on the evolutionary genetic thinkers of the era. With the [Lewontin and Hubby allozyme paper](https://books.google.com/books?id=ng85sd1UR7EC&lpg=PT337&ots=pqN93e_8kK&dq=lewontin%20hubby%20allozyme%20paper&pg=PT337#v=onepage&q&f=false) in the mid-1960s this sort of benign disciplinary evasion was no longer possible; the field of molecular evolution came into its own.\*

Today with genomics these human-imposed artificialities are fading away. Consider the concept of [genetic recombination](https://en.wikipedia.org/wiki/Genetic_recombination). Originally an abstraction in a formal Mendelian system, today it is of great interest to molecular biologists who are curious as to its exact mechanism and purpose, and genomicists who are interested in the constraints upon the phenomenon due to its physical parameters (e.g., [recombination hotspots](http://www.plosbiology.org/article/info:doi/10.1371/journal.pbio.0020190)). If we were to discover alien beings I assume that there would be some sort of genetics in an abstract sense. But would they package their genes in chromosomes? Would their complex organisms tend toward [dioecy](https://en.wikipedia.org/wiki/Dioecy)? **I wouldn’t be surprised if the genetics of alien species have their own particular kinks subject to the contingent nature of the physical scaffolding of the process.**

  
But we don’t always need to go alien species. There are interesting variations among extant earthly taxa. You are almost certainly aware that some organisms have haploid genomes, while others are diploid.
Some organisms are sexual, and others are asexual. And so forth. These structural issues are relevant to both mechanism, and the population and evolutionary phenomena. Consider for example sex determination. In many species there is a heterogametic sex (e.g., XY). This sex has a marginally functional sex-determining chromosome (e.g., Y), and is dependent on the other pair for most of the remaining functionality. Therefore, humans males are subject to “sex-linked traits.” Diseases with recessive expression in females (i.e., both copies need to be mutant) are not recessive in males, who have only one copy. One of the possible evolutionary implications of this is that the X chromosome is subject to more selection than the autosome. The literature in this area is murky for possible implications, so I’m not going to explore that issue further. But, the interesting point is that genetics, chromosome biology, and sexual dimorphism can all operate together to produce particular patterns of variation. The gene as pure abstraction wouldn’t help you here, you need to be informed by its physical expression. Or at least its organization in the domain of physical expression.

And yet as I noted above we have plenty of variation in extant taxa. In birds the females are the heterogametic sex, with a ZW karyotype. In humans the X is in females 2/3 of the time, but in birds the Z is in males 2/3 of the time. Why does this matter? Because it allows us to “check” if variation in mating patterns is reflected in patterns of genomic variation. In monogamous species there is little inter-pair variation in reproductive output (close to [poisson](https://en.wikipedia.org/wiki/Poisson_distribution)), but in very polygynous species a few males dominate the gene pool for the next generation. Because in sexual species males are one half of the ancestors, drastically reducing male breeding size results in a much smaller effective population. This in turn often results in reduced genetic diversity because of the increased power of drift (as well as the common sense observation that you’re sampling from fewer individuals). In most mammals for polygyny this would mean that the X chromosome would be *more* diverse than the autosome, because the autosome is equally impacted by male and female demographics, while the X chromosome has a lesser impact from males.

That’s the theory, and it makes logical sense. But science isn’t just theory, you want to check that theory. Can we scramble the conditions and predict different results? Yes. For birds you expect a converse result in polygynous species. The Z tends to be present in males more than females, and so would be less diverse, due to the fact that male “winner-take-all” dynamics would be a more powerful an effect.

A new paper in *Evolution* looks at what we can learn from patterns in shorebirds. [The genomic signature of sexual selection in the genetic diversity of the sex chromosomes and autosomes](http://www.ncbi.nlm.nih.gov/pubmed?term=THE%20GENOMIC%20SIGNATURE%20OF%20SEXUAL%20SELECTION%20IN%20THE%20GENETIC%20DIVERSITY%20OF%20THE%20SEX%20CHROMOSOMES%20AND%20AUTOSOMES):

> Genomic levels of variation can help reveal the selective and > demographic forces that have affected a species during its history. > The relative amount of genetic diversity observed on the sex > chromosomes as compared to the autosomes is predicted to differ among > monogamous and polygynous species. Many species show departures from > the expectation for monogamy, but it can be difficult to conclude that > this pattern results from variation in mating system because forces > other than sexual selection can act upon sex chromosome genetic > diversity. **As a critical test of the role of mating system, we > compared levels of genetic diversity on the Z chromosome and autosomes > of phylogenetically independent pairs of shorebirds that differed in > their mating systems. We found general support for sexual selection > shaping sex chromosome diversity because most polygynous species > showed relatively reduced genetic variation on their Z chromosomes as > compared to monogamous species.** Differences in levels of genetic diversity between the sex chromosomes and autosomes may therefore contribute to understanding the long-term history of sexual selection experienced by a species.



[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/09/phylopair-300x235.jpg?resize=300%2C235)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/09/phylopair-300x235.jpg?resize=300%2C235)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/09/phylopair.jpg)Bam! All is well with the world. In the figure to the left you can see that they compared phylogenetically independent pairs. This is important, because you want a genuine signal of evolutionary processes, rather than just a phylogenetic correlation. If you had two diverse clades disjoint on a given trait, then just ignoring phylogeny would be misleading. One might analogize this to population stratification in case-control studies, where you don’t want to pick up associations which are are really just genetic correlations with existent population structure which you haven’t accounted for. In this case you see that the polygynous species of a pair has a lower ratio than the monogamous. Why? Remember this is the ratio of the diversity of Z, the analog to the X, with the autosome. In polygynous species male effective population size is going down because a few males monopolize mating, an the Z is a better proxy for this dynamic than the autosome.

In terms of genetic coverage this is not human genomics; we’re talking thousands of kilobases. But that’s probably sufficient for the question being asked here. What’s more important than thickness of marker set would be an expansion of this sort of analysis across the tree of life. **I believe it will happen**. Above I alluded to the convergence and fusion between disparate domains of genetics, in the near future I assume that much of evolutionary genetics will be more thoroughly phylogenetically informed. The main constraint is probably less getting the genetic material, than cranking through it with algorithms which may be computationally intense.

**Citation:**Corl, A. and Ellegren, H. (2012), THE GENOMIC SIGNATURE OF SEXUAL SELECTION IN THE GENETIC DIVERSITY OF THE SEX CHROMOSOMES AND AUTOSOMES. Evolution, 66:2138–2149. doi:10.1111/j.1558-5646.2012.01586.x

\* I am aware that there was a tradition in “physiological genetics” which prefigured this.

### Related Posts:

- [Types of
  genetics](https://www.gnxp.com/WordPress/2012/04/25/types-of-genetics/) - [Genetics for the 21st
  century](https://www.gnxp.com/WordPress/2012/07/03/genetics-for-the-21st-century/) - [You don't need "genes" for
  genetics](https://www.gnxp.com/WordPress/2012/10/12/you-dont-need-genes-for-genetics/) - [Open Thread,
  8/31/2014](https://www.gnxp.com/WordPress/2014/08/31/open-thread-8312014/) - [Genes are not just
  rhetoric](https://www.gnxp.com/WordPress/2013/04/10/genes-are-not-just-rhetoric/) - [Genes are a concept and a
  thing](https://www.gnxp.com/WordPress/2014/11/18/genes-are-a-concept-and-a-thing/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F09%2F16%2Fwhat-the-substrate-tells%2F&linkname=What%20the%20substrate%20tells "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F09%2F16%2Fwhat-the-substrate-tells%2F&linkname=What%20the%20substrate%20tells "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F09%2F16%2Fwhat-the-substrate-tells%2F&linkname=What%20the%20substrate%20tells "Email")[](https://www.addtoany.com/share)
