+++
title = "Genes, a gene, genes…"
full_title = "Genes, a gene, genes…"
date = "2012-09-06"
upstream_url = "https://www.gnxp.com/WordPress/2012/09/06/genes-a-gene-genes/"

+++
Source: [here](https://www.gnxp.com/WordPress/2012/09/06/genes-a-gene-genes/).

Genes, a gene, genes….

I spend some time thinking about [genetic architecture](https://en.wikipedia.org/wiki/Genetic_architecture). If you read media accounts of genetics you wouldn’t know there was such a thing. Why not? Because it makes a narrative rather messy and complex. Rather, a better story is one where you have a gene-for-something. **But the gene-for-something model is probably most useful in the domain of recessive diseases.** In these cases you have a mutant variant which has a powerful effect when found in two copies in an individual. The distinction between the disease state and the “wild” state is rather straightforward. Not so with many “complex traits.”

Two papers came into my RSS feed which illustrate two ends of this dynamic. First, [A Sexual Ornament in Chickens Is Affected by Pleiotropic Alleles at HAO1 and BMP2, Selected during Domestication](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002914):

> The genetic analysis of phenotypes and the identification of the > causative underlying genes remain central to molecular and > evolutionary biology. By utilizing the domestication process, it is > possible to exploit the large differences between domesticated animals > and their wild counterparts to study both this and the mechanism of > domestication itself. Domestication has been central to the advent of > modern civilization; and yet, despite domesticated animals displaying > similar adaptations in morphology, physiology, and behaviour, the > genetic basis of these changes are unknown. In addition, though sexual > selection theory has been the subject of a vast amount of study, very > little is known about which genes are underpinning such traits. We > have generated multiple intercrosses and advanced intercrosses based > on wild-derived and domestic chickens to fine-map genomic regions > affecting a sexual ornament. These regions have been over-laid with > putative selective sweeps identified in domestic chickens and found to > be significantly associated with them. **By using expression QTL > analysis, we show that two genes in one region, *HAO1* and *BMP2*, are > controlling multiple aspects of the domestication phenotype, from a sexual ornament to multiple life history traits.** This demonstrates the importance of pleiotropy (or extremely close linkage) in controlling these genetic changes.

A nice elegant solution. Strong bout of selection, and a response on two genes, which also generates a lot of side effect phenotypes. This section in particular intrigued me:

> … These results show that the **pleiotropic core of domestication > ‘modules’** (regions of the genome controlling multiple aspects of the > domestication phenotype…contain both alleles with pleiotropic effect > and extremely close linkage between QTL. Such a pattern of linkage and domestication modules are also seen in domesticated plants…indicating such a system of loose and tight linkage is responsible for the domestication phenotype in a diverse range of taxa.

I found the second paper via [Haldane’s Sieve](http://haldanessieve.org/), [Genealogies of rapidly adapting populations](http://arxiv.org/abs/1208.3185):

> The genetic diversity of a species is shaped by its recent > evolutionary history and can be used to infer demographic events or > selective sweeps. Most inference methods are based on the null > hypothesis that natural selection is a weak evolutionary force. > However, many species, particularly pathogens, are under continuous > pressure to adapt in response to changing environments. A statistical > framework for inference from diversity data of such populations is > currently lacking. Toward this goal, we explore the properties of > genealogies that emerge from models of continual adaptation. We show > that lineages trace back to a small pool of highly fit ancestors, in > which simultaneous coalescence of more than two lineages frequently > occurs. While such multiple mergers are unlikely under the neutral > coalescent, they create a unique genetic footprint in adapting > populations. The site frequency spectrum of derived neutral alleles, > for example, is non-monotonic and has a peak at high frequencies, > whereas Tajima’s D becomes more and more negative with increasing > sample size. Since multiple merger coalescents emerge in various evolutionary scenarios characterized by sustained selection pressures, **we argue that they should be considered as *null-models* for adapting populations.**

One of the authors comments at [Haldane’s Sieve](http://haldanessieve.org/2012/09/05/our-paper-genealogies-of-rapidly-adapting-populations/):

> Why should we care? We often want to learn about past dynamics from > snapshots of populations (sequence samples). To this end, we compare > the diversity patterns in the sample to model predictions and infer > model parameters. If we use an inappropriate model, we get meaningless > answers. Furthermore, some events that are very unlikely under > Kingman’s coalescent are quite common when multiple mergers are > allowed. Consider for example a lone haplotype in a large sample that > connects to the root of the tree. This is very unlikely in neutral > coalescent models and one might take it as evidence for immigration > from a diverged population. If multiple mergers dominate coalescence, > this does not come as a surprise. Similarly, an excess of singletons is not necessarily evidence for expanding populations or deleterious mutations but might be due to draft. I wonder whether more potential pitfalls of this sort exist.

There’s still a lot we don’t know. That’s good, and it’s bad.

### Related Posts:

- [Common disease, common
  variant](https://www.gnxp.com/WordPress/2006/12/01/common-disease-common-variant/) - [Treatments for some Mendelian diseases in the
  works?](https://www.gnxp.com/WordPress/2007/05/04/treatments-for-some-mendelian-diseases-in-the-works/) - [Mitochondrial
  genetics](https://www.gnxp.com/WordPress/2007/06/11/mitochondrial-genetics/) - [Two papers in Nature Genetics on gene
  expression](https://www.gnxp.com/WordPress/2007/09/17/two-papers-in-nature-genetics-on-gene-expression/) - [Tools to analyze gene
  expression](https://www.gnxp.com/WordPress/2009/12/31/tools-to-analyze-gene-expression/) - [You're sick because of your
  genes!](https://www.gnxp.com/WordPress/2008/11/17/youre-sick-because-of-your-genes/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F09%2F06%2Fgenes-a-gene-genes%2F&linkname=Genes%2C%20a%20gene%2C%20genes%E2%80%A6. "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F09%2F06%2Fgenes-a-gene-genes%2F&linkname=Genes%2C%20a%20gene%2C%20genes%E2%80%A6. "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F09%2F06%2Fgenes-a-gene-genes%2F&linkname=Genes%2C%20a%20gene%2C%20genes%E2%80%A6. "Email")[](https://www.addtoany.com/share)
