+++
title = "The 1000 Genomes Paper"
full_title = "The 1000 Genomes Paper"
date = "2015-09-30"
upstream_url = "https://www.gnxp.com/WordPress/2015/09/30/the-1000-genomes-paper/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/09/30/the-1000-genomes-paper/).

The 1000 Genomes Paper

[![nature15393-f2](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/09/nature15393-f2.jpg?resize=640%2C741)![nature15393-f2](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/09/nature15393-f2.jpg?resize=640%2C741)](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html#corres-auth)

The 1000 Genomes paper is out, [A global reference for human genetic variation](http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html). It’s open access, **read the whole thing.** Here’s the abstract:

> The 1000 Genomes Project set out to provide a comprehensive > description of common human genetic variation by applying whole-genome > sequencing to a diverse set of individuals from multiple populations. > Here we report completion of the project, having reconstructed the > genomes of 2,504 individuals from 26 populations using a combination > of low-coverage whole-genome sequencing, deep exome sequencing, and > dense microarray genotyping. We characterized a broad spectrum of > genetic variation, in total over 88 million variants (84.7 million > single nucleotide polymorphisms (SNPs), 3.6 million short > insertions/deletions (indels), and 60,000 structural variants), all > phased onto high-quality haplotypes. This resource includes \>99% of SNP variants with a frequency of \>1% for a variety of ancestries. We describe the distribution of genetic variation across the global sample, and discuss the implications for common disease studies.

[![ft067n99v9_cover](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/09/ft067n99v9_cover.jpg?resize=187%2C288)![ft067n99v9_cover](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/09/ft067n99v9_cover.jpg?resize=187%2C288)](https://www.amazon.com/exec/obidos/ASIN/0520205073/geneexpressio-20)The PSMC above is interesting to me. It shows BEB, the Bengali population form Dhaka, starting from a small base and exploding in size. There are some issues relating to ascertainment that need to be admitted here though. The Indian Gujurati sample turns out to be about half Patel, and half other Gujuratis. In contrast, the Bengalis are relatively homogeneous in ancestry (sampled from Dhaka), and don’t seem to exhibit much population structure. What I’m saying is that when the authors talk about “Gujuratis” they are really talking about “sort of Patels”, while when they talk about Bengalis, they are talking about Bengalis as a whole. There’s an apples-to-oranges aspect to this. It also needs to be kept in mind when they note the alleles private to the Gujurati (GIH) sample; that’s almost certainly due to the large number of endogamous Patels in the original Houston data set who are going to share a lot more demographic history than you’d otherwise expect among Gujuratis.

Secondly, the bottleneck + genetic homogeneity in the admixture for Bengalis reinforces the model outlined in [The Rise of Islam and the Bengal Frontier, 1204-1760](https://www.amazon.com/exec/obidos/ASIN/0520205073/geneexpressio-20). Basically the population size change above highlights that **eastern Bengalis descend from a small group of founders relatively recently in the past, despite their \>100 million modern census size.** Genetically this has resulted in the ancestral homogeneity you see in the plot above, but culturally it also allowed for the degrading of the social institutions of Indian society which allowed Hinduism to be robust to nearly one thousand years of Islamic hegemony across the subcontinent. Additionally, the lack of structure in ancestral components reflects relatively little endogamy (I have checked the runs of homozygosity in my parents’ genotypes, and they’re lower than those of my South Asian friends from particular caste/jati backgrounds).

### Related Posts:

- [10,000 genome at 30x & human
  variation](https://www.gnxp.com/WordPress/2016/07/03/10000-genome-at-30x-human-variation/) - [Batch effects in the 1000 Genomes
  data?](https://www.gnxp.com/WordPress/2019/05/04/batch-effects-in-the-1000-genomes-data/) - [The \$50 genome in
  2014?](https://www.gnxp.com/WordPress/2011/01/11/the-50-genome-in-2014/) - [Genetic variation in human populations and
  individuals](https://www.gnxp.com/WordPress/2017/04/15/genetic-variation-in-human-populations-and-individuals/) - [Genomic
  liftoff](https://www.gnxp.com/WordPress/2010/07/13/genomic-liftoff/) - [The Human Genome Diversity Project at
  high-coverage!](https://www.gnxp.com/WordPress/2020/03/25/the-human-genome-diversity-project-at-high-coverage/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F09%2F30%2Fthe-1000-genomes-paper%2F&linkname=The%201000%20Genomes%20Paper "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F09%2F30%2Fthe-1000-genomes-paper%2F&linkname=The%201000%20Genomes%20Paper "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F09%2F30%2Fthe-1000-genomes-paper%2F&linkname=The%201000%20Genomes%20Paper "Email")[](https://www.addtoany.com/share)
