+++
title = "Of association &"
full_title = "Of association & evolution"
date = "2011-01-09"
upstream_url = "https://www.gnxp.com/WordPress/2011/01/09/of-association-amp-evolution/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/01/09/of-association-amp-evolution/).

Of association & evolution

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/800px-Bellcurve.svg_.png?resize=250%2C154)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/800px-Bellcurve.svg_.png?resize=250%2C154)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/800px-Bellcurve.svg_.png)Two of the main avenues of research which I track rather closely in this space are genome-wide association studies ([GWAS](https://en.wikipedia.org/wiki/Genome-wide_association_study)), which attempt to establish a connection between a trait/disease and particular genetic markers, and inquiries into the evolutionary parameters which shape the structure of variationwithin the human genome. Often with specific relation to a particular trait/disease. By evolutionary parameters I mean stochastic and deterministic forces; mutation, migration, random drift, and natural selection. These two angles are obviously connected. Both focus on phenomena which are proximate in relation to the broader evolutionary principle: **the ultimate raison d’être, replication.** Stochastic forces such as random genetic drift reflect the error of sampling of genes from generation to generation during the process of reproduction, while adaptation through natural selection is an outcome of the variation of reproductive fitness as a function of variation of heritable traits. Both of these forces have been implicated in diseases and traits which come under the purview of GWAS (and [linkage mapping](https://en.wikipedia.org/wiki/Genetic_linkage#Linkage_map)).

GWAS are regularly in the news because of their relevance in identifying the causal genetic factors for specific diseases. For example, [schizophrenia](http://scienceblogs.com/geneticfuture/2009/07/guest_post_neil_walker_on_the.php). But they can be useful in a non-disease context as well. Human [pigmentation](http://hmg.oxfordjournals.org/content/18/R1/R9.short?rss=1&ssource=mfc) is a character whose genetic architecture has been well elucidated thanks to a host of recent association studies. The [common disease-common variant](https://en.wikipedia.org/wiki/Common_disease-common_variant) has yielded spectacular results for pigmentation; it does seem [a few common variants](http://www.ncbi.nlm.nih.gov/pubmed/17720606) are responsible for most of the variation on this trait. But this has been the exception rather than the rule.

One reason for this disjunction between the promise of GWAS and the concrete tangible outcomes is that **many traits/diseases of interest may be polygenic and quantitative.** This implies that variation in phenotype is controlled by variation across many genes, and, that the variation itself exhibits gradual continuity (a continuity which can be modeled as a normal distribution of values). The power of GWAS to detect correlated variation across genes and traits of small marginal effect is obviously limited. In contrast, it seems that about half a dozen genes can explain most of the between population variation in pigmentation. One [SNP](https://en.wikipedia.org/wiki/SLC24A5#Effect_on_skin_color) is able to account for 25-40% of the difference in shade between Europeans and Africans. This SNP is fixed in Europeans, nearly absent in Africans and East Asians, and segregating in both ancestral and derived variants in groups such as South Asians and African Americans. In contrast, though traits such as [schizophrenia](https://en.wikipedia.org/wiki/Schizophrenia#Genetic) and [height](https://en.wikipedia.org/wiki/Human_height#Determinants_of_growth_and_height) are substantially heritable, much of the variation at the population level of the trait is explainable by variation in genes. The effect size at any given locus may be small, or the variation may be accumulated through the sum of larger effect variants of low frequency. In other words, many common variants of small effect, or numerous distinctive[rare variants of large effect](http://www.plosbiology.org/article/info:doi/10.1371/journal.pbio.1000294).

  
[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)These nuances of genetic architecture are not irrelevant to the possible evolutionary arc of the traits in question. One model of the adaptation leading to the high frequency of a trait or disease is that a novel mutation rapidly “sweeps” to fixation, or nearly to fixation. In other words, it shifts from nearly \~0% to nearly \~100% frequency in the population of alleles at that locus, driven by positive selection. This sort of rapid [“hard sweep”](http://www.ncbi.nlm.nih.gov/pubmed/20178769) would also result in “hitchhiking” of associated variants in the genomic regions adjacent to the originally favored mutant, producing regions of high [linkage disequilibrium](https://en.wikipedia.org/wiki/Linkage_disequilibrium) in the genome and [haplotype](https://en.wikipedia.org/wiki/Haplotype) blocks of associated alleles across loci. Such a model does seem possible in the case of [some of the variants](http://www.snpedia.com/index.php/Rs1426654) which are responsible for diversity of pigmentation. But this neat dovetailing between the strong association of a few variants with trait variance, and signatures of positive selection being driven by adaptation, is not so easy to come by in many instances.

There are other evolutionary possibilities in terms of what could drive a high frequency of particular alleles. Population bottlenecks and inbreeding can crank up the frequency of a variant simply through chance. This may be the origin of many traits and diseases expressed recessively or in quasi-Mendelian form which run in specific populations. Let’s set such stochastic possibilities to the side for now. The well of natural selection is not quite tapped out simply by models of positive selection drawing upon singular new mutations. Another model is that of “soft sweeps” operating upon standing genetic variation. Consider for example a trait which has a heritability of 0.50. 50% of the variance in trait value can be explained by variance in genes. Selection correlated with trait value can rapidly change the distribution of the trait within the population, as modeled by the [breeder’s equation](http://blogs.discovermagazine.com/gnxp/2007/05/breeding-the-breeders-equation/). But no new mutations are necessary in this model, rather, the frequencies of extant alleles changes over time. In fact, as the proportions shift novel combinations of alleles which were once too rare to be found together in the same individual will emerge, and so offer up the possibility that the mean trait value in generation *t + n generations* may be outside of the range of trait values at*t =* 0.

Over time such selection on a quantitative trait theoretically exhausts its own fuel, genetic variation. But quite often this is not practically operative, because such traits are subject to a background level of novel mutation and [balancing selection](https://en.wikipedia.org/wiki/Balancing_selection). Stabilizing selection around a median phenotype, as well as frequency dependence and shifting environmental pressures, may produce a circumstance where adaptation never moves beyond the transient flux toward a new equilibrium. The element of the eternal race is at the heart of the [Red Queen’s Hypothesis](https://en.wikipedia.org/wiki/Red_Queen's_Hypothesis), where pathogen and host engage in an evolutionary war, and host immune responses are subject to negative frequency dependence. As the frequency of an allele rises, its relative fitness declines. As its frequency declines, its fitness rises.

Naturally such complex evolutionary models, subject to contingency and less non-trivially powerful in their generality, only become appealing when simple hard sweep models no longer suffice. But it seems highly plausible that the genetic architecture of some traits, those which seem plagued by ‘[missing heritability](http://www.nature.com/news/2008/081105/full/456018a.html),’ are going to necessitate somewhat more baroque evolutionary models to explain their ultimate emergence & persistence. A new paper in *PLoS Genetics* tackles this complexity by looking at the patterns of variation of SNPs implicated in GWAS in the [HGDP](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project) data set. [Genome-Wide Association Study SNPs in the Human Genome Diversity Project Populations: Does Selection Affect Unlinked SNPs with Shared Trait Associations?](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001266?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed:+plosgenetics/NewArticles+(Ambra+-+Genetics+New+Articles)&utm_content=Google+Feedfetcher) First, the abstract:

> Genome-wide association studies (GWAS) have identified more than 2,000 > trait-SNP associations, and the number continues to increase. GWAS > have focused on traits with potential consequences for human fitness, > including many immunological, metabolic, cardiovascular, and > behavioral phenotypes. Given the polygenic nature of complex traits, > selection may exert its influence on them by altering allele > frequencies at many associated loci, a possibility which has yet to be > explored empirically. **Here we use 38 different measures of allele > frequency variation and 8 iHS scores to characterize over 1,300 GWAS > SNPs in 53 globally distributed human populations.** We apply these > same techniques to evaluate SNPs grouped by trait association. We find > that groups of SNPs associated with pigmentation, blood pressure, > infectious disease, and autoimmune disease traits exhibit unusual > allele frequency patterns and elevated iHS scores in certain > geographical locations. We also find that GWAS SNPs have generally > elevated scores for measures of allele frequency variation and for iHS > in Eurasia and East Asia. Overall, we believe that our results provide > evidence for selection on several complex traits that has caused > changes in allele frequencies and/or elevated iHS scores at a number > of associated loci. Since GWAS SNPs collectively exhibit elevated > allele frequency measures and iHS scores, selection on complex traits > may be quite widespread. **Our findings are most consistent with this > selection being either positive or negative, although the relative > contributions of the two are difficult to discern.** Our results also > suggest that trait-SNP associations identified in Eurasian samples may > not be present in Africa, Oceania, and the Americas, possibly due to > differences in linkage disequilibrium patterns. This observation > suggests that non-Eurasian and non-East Asian sample populations > should be included in future GWAS

And now the author summary:

> Natural selection exerts its influence by changing allele frequencies > at genomic polymorphisms. Alleles associated with harmful traits > decrease in frequency while those associated with beneficial traits > become more common. In a simple case, selection acts on a trait > controlled by a single polymorphism; a large change in allele > frequency at this polymorphism can eliminate a deleterious phenotype > from a population or fix a beneficial one. However, many phenotypes, > including diseases like Type 2 Diabetes, Crohn’s disease, and prostate > cancer, and physiological traits like height, weight, and hair color, > are controlled by multiple genomic loci. Selection may act on such > traits by influencing allele frequencies at a single associated > polymorphism or by altering allele frequencies at many associated > polymorphisms. To search for cases of the latter, we assembled groups > of genomic polymorphisms sharing a common trait association and > examined their allele frequencies across 53 globally distributed > populations looking for commonalities in allelic behavior across > geographical space. **We find that variants associated with blood > pressure tend to correlate with latitude, while those associated with > HIV/AIDS progression correlate well with longitude. We also find > evidence that selection may be acting worldwide to increase the > frequencies of alleles that elevate autoimmune disease risk.**

This is a paper where jumping to the methods might be useful. Though I’m sure that the authors did not intend it, sometimes it felt as if you were following the marble being manipulated by the carnival tender. Since I was not familiar with some of the terms for the statistics, a simple allusion to the methods without elaborating in detail did not suffice. In any case, the key here is that they focused on **the set of SNPs which have been associated with trait variance in GWAS, and compared those to the total SNPs found in the HGDP data set of 53 populations.** Note that not all SNPs in GWAS were in the HGDP SNP panel. But for the general questions being asked the intersection of SNPs sufficed. Additionally, they generated a further subset of SNPs which were highly likely to be associated with trait variance. These were SNPs where other SNPs of related function were within 1 MB, or, SNPs which were found in more than one GWAS.

There were four primary statistics within the paper: Delta, Fst, LLC, and iHS. Fst and iHS are familiar. Fst measures the extent of between population variance across a set of populations. High Fst means a great deal of population structure, while Fst \~ 0 means basically no population structure. iHS is a test to detect the probability of natural selection based on patterns of linkage disequilibrium in the genome. Basically the important thing for the purposes of this paper is that iHS tends to be good at detecting alleles at moderate frequencies still presumably going through sweeps. This is in contrast to the older EHH test, which only detects sweeps which are nearly complete. If the authors are focusing on polygenic traits and soft sweeps the likelihood of that showing up on EHH is low since that is predicated on hard, nearly complete, sweeps. LLC measures the correlation between genetic variant of a trait as a function of latitude and longitude. Presumably this would be useful for smoking out those traits driven by ecological pressures (an obvious example in a general sense are consistent changes in area-to-volume ratio across taxa as organisms proceed from warmer to colder climes). Finally, Delta measures the allele frequency difference across the set of populations. The sign of Delta is simply a function of whether the allele frequency in question is higher in the first or second population in the comparison.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas1.png?resize=350%2C441)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas1.png?resize=350%2C441)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas1.png)In doing their comparisons the authors did not simply compare across all 53 populations in a pairwise fashion. Rather, they often pooled continental or regional groups. To the left is a slice of table 1. It shows the populations used to generate the Delta values, and how they were pooled. The [HGDP populations](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project#Studied_populations) are broken down by region in a rather straightforward manner. But also note that some of the comparisons are between populations within regions, and those with different lifestyles. I assume that the comparisons highlighted within the paper were performed with the aim of squeezing maximal informative juice in such an exploratory endeavor. There are no obligate hunter-gatherers within the Eurasian populations in the HGDP data set to my knowledge, so a comparison between agriculturalists and hunter-gatherers would not be possible. There is such a comparison available in the African data set. The authors generated p-values by comparing the GWAS SNPs to random SNPs within the HGDP data set. In particular, they were looking for signatures of distinctiveness among the HGDP data set.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas2.png?resize=301%2C222)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas2.png?resize=301%2C222)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas2.png)Such distinctiveness is expected. The set of SNPs associated with diseases and traits of note are not likely to be a representative subset of the SNPs across the whole genome. Remember that a [neutral model](https://en.wikipedia.org/wiki/Neutral_theory_of_molecular_evolution) of molecular evolution means that we should expect most genetic variation within the genome is going to be due to stochastic forces. Panel A of figure 1 shows that in fact the SNPs derived from GWAS did exhibit a different pattern from the total set of SNPs in the HGDP panel. Observe that the distribution of minor allele frequency (MAF) is somewhat skewed toward higher values for the GWAS SNPs. If the logic of GWAS is geared toward “common variants” which will be frequent enough within the population to generate an effect which is powerful enough to be picked up by the studies given their sample sizes, the bias toward more common variants (higher MAF) is understandable.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas3.png?resize=300%2C472)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas3.png?resize=300%2C472)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas3.png)To the left are some SNPs and traits which had low p-values (i.e., they were deviated from expectation beyond what you’d expect from random noise). Not very surprisingly they found that pigmentation related SNPs tended to show up strongly in all the measures of population differentiation and variation. rs28777 is found in *SLC45A2*, a locus which differentiates Europeans from non-Europeans. rs1834640 is in *SLC24A5*, which differentiates Europeans + Middle Easterners + Central/South Asians from other populations.rs12913832 is a “blue eye” related variant. That is, it’s one of the markers associated with blue vs. non-blue eye color differences in Europeans.

Seeing that pigmentation has been one of the few traits which has been well elucidated by the current techniques, it should be expected that more subtle and thorough methods aimed at detecting genetic variation across and within populations should stumble upon those markers first. The authors note that “SNPs and study groups associated with **pigmentation and immunological traits made up a majority of those that reached significance in our analysis**.” There has long been a tendency toward finding signatures of selection around pigmentation and disease related loci.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas4.png?resize=275%2C207)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas4.png?resize=275%2C207)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/gwas4.png)One pattern which was also evident in terms of geography in the patterns of low p-values was the tendency for Eurasian groups to be enriched. This is illustrated in figure 2. Most of the SNPs from the GWAS studies were derived from study populations which were European. Because of this there is probably a bias in the set of SNPs being evaluated which are particular informative for Europeans and related populations. Additionally, it may also be that Eurasians were subject to different selective pressures as they left the ancestral African environment \~150-50,000 years B.P. In any case, for purposes of medical analysis the authors did find that using SNPs from East Asian populations produced somewhat different results than using those from European populations. Though some studies have shown a broad applicability of SNPs across populations, there are no doubt many variants in non-European populations which have simply not been detected because GWAS studies are not particularly focused on non-European populations. Consider:

> … However, our results indicate that SNPs associated with pigmentation > in GWAS display unusual allele frequency patterns almost exclusively > in Europe, the Middle East, and Central Asia. This suggests to us that > there may be SNPs, perhaps in or near genes other than *SLC45A2, IRF4, > TYR, SLC24A4, HERC2, MC1R,* and *ASIP*, which are associated with > pigmentation in non-Eurasian populations, but which have yet to be > identified by GWAS. GWAS for pigmentation traits carried out using > non-European subjects are needed to explore this possibility further.

There are two major other classes of trait/disease which were found to vary systematically across the HGDP populations:

– High blood pressure associated variants seemed to decrease with latitude

– Infectious and autoimmune disease SNPs had elevated scores. Specifically, there were some HIV related SNPs associated with Europeans which seem to confer resistance

The first set of traits would naturally come out of GWAS derived SNPs, since so much medical research goes into identifying risk and treating high blood pressure and other circulatory ailments. A consistent pattern where **geography and not ancestry predict variation** is an excellent tell for exogenous selective pressures. The physical nature of the earth is such that as mammals spread away from the equators their physiques will be reshaped by different sets of ecological parameters. Siberian populations have [developed adaptations to cold stress](http://groups.anthropology.northwestern.edu/lhbr/Siberia.html), and there seem to be consistent cross-taxa shifts in body form to maximize or minimize heat radiation among mammals.

In the second case you have resistance to disease cropping up again, as well as [pleiotropy](https://en.wikipedia.org/wiki/Pleiotropy), whereby genetic changes can have multiple downstream consequences. Often this is temporally simultaneous; consider the [tame silver foxes](https://en.wikipedia.org/wiki/Domesticated_silver_fox). But sometimes you have a change in the past which has a subsequent consequence later in time due to different selective pressures. It is not that surprising that immunological responses can be multi-purpose, so even though Europeans did not develop resistance to HIV as a general selective pressure, similar pressures seem to have resulted in responses with general utility and now a specific use in relation to HIV. Selection can often be a blunt instrument, interposing itself into a network of interactions with multiple consequences, reshaping many traits simultaneously in the process of maximizing local fitness. This is most clear when you have a trait such as [sicke-cell disease](https://en.wikipedia.org/wiki/Sickle-cell_disease), which emerges only because the fitness benefit of heterozygosity is so great. But no doubt when it comes to many traits the byproducts are more subtle, or may seem cryptic to us. We still do not know why [*EDAR*](https://en.wikipedia.org/wiki/EDAR) was driven to higher frequency in East Asians (less body odor and thick straight hair seem implausible targets for selection).

And just as natural selection can be blunt and rude in its impact on the covariance of genes and traits, so its relaxation may remove a suffocating vice. Consider the possibilities with blood pressure: perhaps the reason that northern Eurasians have lower blood pressure is that selection for other correlated traits associated with higher values were relaxed, allowing for fitness to be maximized in this particular dimension. Similarly, African Americans have a lower frequency of the sickle-cell disease than their \~80% West African ancestry would entail, because without the pressure of endemic malaria selection for the heterozygote was removed, allowing for the purging of the allele from the gene pool.

Nevertheless, the authors do conclude::

> Despite our broad-based approach, we found only a few examples of what > may be a polygenic response to a single selective pressure.\</b\> We > did use stringent significance criteria which might mean that > additional examples can be found among the study groups that did not > quite meet our threshold of significance. It may also be that there is > something about “GWAS” traits and their underlying genetics that > served to undermine our approach.

They have several suggestions for why this didn’t pan out:  
– The GWAS variants aren’t the primary source of the variation. It could be copy number variants, rare large effect variants (“synthetic”)

– Epistasis. Gene-gene interaction, which would mask or confound linear associations between variants and traits

– Low impact of selection on GWAS SNPs, or, balancing or negative selection

They finish:

> In summary, we have examined 1,336 trait-associated SNPs in the 53 > CEPH-HGDP populations looking for individual SNPs and groups of SNPs > with unusual allele frequency patterns and elevated iHS scores. We > identified 13 different traits with an associated SNP or study group > that produced a significantly elevated score for at least one delta, > Fst, LLC, or iHS measure, a small percentage of the total number of > traits analyzed. We believe that the limited number of positive > results could be due to our stringent significance criteria or to > features of the genetic architecture of the traits themselves. > **Specifically, the roles of rare variants, epistasis, and pleiotropy > in human complex traits are, although areas of active inquiry, still > generally not well understood.** Our measures may also not be optimal > for detecting all types of selection acting on GWAS traits. It has > been speculated that variants underlying complex traits will be > influenced primarily by negative or balancing selection, which may not > produce extreme values for our measures, particularly if these forces > are relatively uniform across populations or are acting on many > regions in the genome.

If selective pressures on polygenic traits are so common perhaps genomicists are going to be thumbing through [Introduction to Quantitative Genetics](https://www.amazon.com/exec/obidos/ASIN/0582243025/geneexpressio-20). These are traits and evolutionary processes which lack clear distinction. In many ways modeling positive selection and hard sweeps resembles the economics of equilibriums. When it comes to continuous and quantitative traits subject to the effect of many genes a different way of thinking has to come to the fore. The transient no longer becomes a punctuation between the stasis, but the thing in and of itself. There are for example [HLA](https://en.wikipedia.org/wiki/Major_histocompatibility_complex#HLA_genes) genes in humans which are found in chimpanzees, because the nature of the eternal race between host and pathogen means that all the old tricks are preserved, at least at low frequencies. Human variation in intelligence, height, and all sorts of other liabilities and characteristics, may have always been with us, being buffeted continuously by a swarm of selective pressures. The question is, can our crude statistical methods ever get a grip on this diffuse but all-powerful net?

**Citation:** Casto AM, & Feldman MW (2011). Genome-Wide Association Study SNPs in the Human Genome Diversity Project Populations: Does Selection Affect Unlinked SNPs with Shared Trait Associations? PLoS Genetics : [10.1371/journal.pgen.1001266](https://dx.doi.org/10.1371/journal.pgen.1001266)

### Related Posts:

- [Migration &
  evolution](https://www.gnxp.com/WordPress/2009/10/09/migration-evolution/) - [Random acts of
  ill-health](https://www.gnxp.com/WordPress/2010/01/05/random-acts-of-ill-health/) - [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/) - [Asia finally getting in on the genome-wide association
  game?](https://www.gnxp.com/WordPress/2008/08/18/asia-finally-getting-in-on-the-genome-wide-association-game/) - [Common disease, common
  variant](https://www.gnxp.com/WordPress/2006/12/01/common-disease-common-variant/) - [Genome-wide association study for breast
  cancer…](https://www.gnxp.com/WordPress/2007/06/28/genome-wide-association-study-for-breast-cancer-susceptibility/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F01%2F09%2Fof-association-amp-evolution%2F&linkname=Of%20association%20%26%20evolution "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F01%2F09%2Fof-association-amp-evolution%2F&linkname=Of%20association%20%26%20evolution "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F01%2F09%2Fof-association-amp-evolution%2F&linkname=Of%20association%20%26%20evolution "Email")[](https://www.addtoany.com/share)
