+++
title = "Deleterious drag as a"
full_title = "Deleterious drag as a side effect of adaptation"
date = "2011-08-26"
upstream_url = "https://www.gnxp.com/WordPress/2011/08/26/deleterious-drag-as-a-side-effect-of-adaptation/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/08/26/deleterious-drag-as-a-side-effect-of-adaptation/).

Deleterious drag as a side effect of adaptation

***The Pith:** Evolution is a sloppy artist. Upon the focal zone of creative energy adaptation can sculpt with precision, but on the margins of the genetic landscape frightening phenomena may erupt due to inattention. In other words, there are often downsides to adaptation.*

A few weeks ago I [reviewed a paper](http://blogs.discovermagazine.com/gnxp/2011/08/crohns-disease-is-about-barely-keeping-you-alive/) which suggests that [Crohn’s disease](https://en.wikipedia.org/wiki/Crohn's_disease) may be a side effect of a [selective sweep](https://en.wikipedia.org/wiki/Selective_sweep). The sweep itself was possiblydriven by adaptation to nutrient deficiencies incurred by European farmers switching to a grain based diet. The reason for this is a contingent genomic reality: **the positively selected genetic variant was flanked by a Crohn’s disease risk allele.** The increment of fitness gain of the former happens to have been greater than the decrement entailed by the latter, resulting in the simultaneous increase in the frequency of both the fit and unfit variants. You can’t always have one without the other.

But that’s just focusing on one gene, though the authors did indicate that this may be a genome-wide feature. A new paper in *PLoS Genetics* argues that that is the case, at least to some extent. [Evidence for Hitchhiking of Deleterious Mutations within the Human Genome](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002240):

> Deleterious mutations reduce fitness within natural populations and > must be continually removed by natural selection. However, some > deleterious mutations reach unexpectedly high frequencies. There are a > number of mechanisms by which this could occur, including changes in > genetic or environmental constraints. Here, we investigate the > hypothesis that some deleterious mutations have hitchhiked to high > frequency due to linkage to sites that have been under positive > selection. Using a collated set of regions likely to have been > influenced by positive selection, we find that the number of > deleterious polymorphisms in hitchhiking and non-hitchhiking regions > is similar, **but that the ratio of deleterious to neutral > polymorphism is higher in hitchhiking compared to non-hitchhiking > regions.** Both computer simulations and empirical data indicate that > while hitchhiking eliminates many deleterious mutations, some are > increased in frequency. The distribution of human disease-associated > mutations is also altered in hitchhiking compared to non-hitchhiking > regions. Together, our results provide evidence that hitchhiking has influenced the frequency of linked deleterious mutations in humans, implying that the evolutionary dynamics of advantageous and deleterious mutations may often depend on one another.

To understand what’s going on here recall that the genome consists of a sequence of base pairs. Some of these base pairs code for genes, and [some do not](https://en.wikipedia.org/wiki/Intergenic_region). In the former class you have bases which can be changed and still not alter the final protein product, and so are “synonymous,” and those whose change does alter the final protein product, and so are “nonsynonymous.” By and large synonymous alternatives are often selectively “neutral,” in other words they don’t have a positive or negative impact on fitness (though as you may know some of these nonfunctional regions turn out to have some selective relevance). Nonsynonymous changes can be neutral as well, but they may also have functional consequences which are negative or positive. More often than not the consequences are negative, and nonsynonymous mutations are going to be “purified” from the genome through [background selection](https://en.wikipedia.org/wiki/Background_selection). Imagine if you will that the genome is always bubbling with new mutations. A substantial proportion of these are deleterious, but they are quickly nipped in the bud by [purifying selection](https://en.wikipedia.org/wiki/Background_selection), which is constantly pruning and constraining critical regions of the genome. [In a few cases](https://en.wikipedia.org/wiki/Lactase#Genetic_expression_and_regulation) though a mutation may be positively selected due to its adaptive value. Instead of being purified, this variant will increase in frequency rapidly, generating a[selective sweep](https://en.wikipedia.org/wiki/Selective_sweep) which reorganizes flanking regions of the genome. Neutral variants may “hitchhike” along with their selectively favored neighbors, and because of this correlated increase in frequency of linked variants on a population wide scale the swept region of the genome will exhibit a high degree of homogenization. And more broadly framing this constant patter of purifying selection and welter of selective sweeps you always have [neutral dynamics](https://en.wikipedia.org/wiki/Neutral_theory_of_molecular_evolution), as innocuous mutants rise to prominence or fade into the background via the chance winds of fortune.

In other words, **the genome is a complex and interlocking system, which has its own logic which constrains and shapes evolutionary process**. To a first approximation it may be useful to view it as a “black box,” a substrate which exists only to mediate the action of natural selection and random genetic drift over the generations. But to properly characterize the fine-grained texture of the patterns of biological variation as they are we need to grapple with the concrete reality of the genome, and the broad features which characterize its landscape. Some of these complex parameters are exogenous to the DNA structure; population genetic abstractions. Random genetic drift is affected by population size and structure. Natural selection is highly context dependent, both as a function of space and time. But other parameters are endogenous. The number of chromosomes, whether the organism is diploid, the variations in the rate of recombination. These are all features of the genome itself.

The ratio at the heart of this paper, the result that deleterious alleles may gain more from hitchhiking on positives selective sweeps than neutral alleles, is at the intersection of exogenous and endogenous parameters. Neutral dynamics can be effected by changes in population size, while the phenomenon of hitchhiking occurs due to the biophysical structure of the genome. The question which looms large here is this: why are some deleterious alleles present at a high frequency? One of the answers may be that they are parasites upon the favor shown to their adaptively beneficial neighbors. Normally the frequency distribution of deleterious alleles is skewed toward the low end, presumably because purifying selection is constantly tamping down the upward fluctuations in frequency. But with a selective sweep the gentle constraints which keep these deleterious variants in check are pushed aside by the hammer-blow of adaptive processes, which exist not to finely tune the genetic architecture for the long run, but succeed in the short run, no matter the cost. What was once a rare loss of function deleterious variant may become a far less rare loss of function variant due to a chance association with a selectively favored allele. In contrast, neutral variants exhibit a much wider range of frequencies, from fixed, to moderate, to low frequency.

But the final conclusions of the paper are tentative and cautious. There are obviously many other population biological processes which are responsible for the preservation of deleterious alleles. [Balancing selection](https://en.wikipedia.org/wiki/Balancing_selection) of various kinds comes to mind. Instead of finding the one answer the aim of the authors here seems to be to sketch out one important possible piece of the puzzle.

**Citation:** Chun S, Fay JC (2011) Evidence for Hitchhiking of Deleterious Mutations within the Human Genome. PLoS Genet 7(8): e1002240. doi:10.1371/journal.pgen.1002240

### Related Posts:

- [Quantitative genetics strikes back!
  (?)](https://www.gnxp.com/WordPress/2010/02/27/quantitative-genetics-strikes-back/) - [Humans have adapted on genome-wide
  level?](https://www.gnxp.com/WordPress/2009/01/16/humans-have-adapted-on-genome-wide-level/) - [Crohn's disease is about barely keeping you
  alive](https://www.gnxp.com/WordPress/2011/08/10/crohns-disease-is-about-barely-keeping-you-alive/) - [Human evolutionary
  genomics](https://www.gnxp.com/WordPress/2007/10/09/human-evolutionary-genomics/) - [Mirror neurons not all
  that?](https://www.gnxp.com/WordPress/2009/06/02/mirror-neurons-not-all-that/) - [Adaptation on the human
  genome](https://www.gnxp.com/WordPress/2007/06/04/adaptation-on-the-human-genome/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F08%2F26%2Fdeleterious-drag-as-a-side-effect-of-adaptation%2F&linkname=Deleterious%20drag%20as%20a%20side%20effect%20of%20adaptation "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F08%2F26%2Fdeleterious-drag-as-a-side-effect-of-adaptation%2F&linkname=Deleterious%20drag%20as%20a%20side%20effect%20of%20adaptation "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F08%2F26%2Fdeleterious-drag-as-a-side-effect-of-adaptation%2F&linkname=Deleterious%20drag%20as%20a%20side%20effect%20of%20adaptation "Email")[](https://www.addtoany.com/share)
