+++
title = "Human genetics"
full_title = "Human genetics presentations of interest"
date = "2011-08-31"
upstream_url = "https://www.gnxp.com/WordPress/2011/08/31/human-genetics-presentations-of-interest/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/08/31/human-genetics-presentations-of-interest/).

Human genetics presentations of interest

[Dienekes](https://dienekes.blogspot.com/2011/08/ichg-2011-abstracts-are-onlineic.html) alerts me to the fact that the [International Congress of Human Genetics](http://www.ichg2011.org/) abstracts [are online](http://www.ichg2011.org/abstracts/fulltext/). I spent an hour using only a few keywords, and came up with a lot.

1\) If you have a presentation and think it might interest the readers here, feel free to drop a link in (I will look in the spam folder more today, though one link shouldn’t drive it crazy).

2\) If you are a reader and found something interesting, do the same.

Below are some abstracts that caught my eye….

  
[Estimating a date of mixture of ancestral South Asian populations](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=20758&sid=972862). ” Our analyses suggest that major ANI-ASI mixture occurred in the ancestors of both northern and southern Indians 1,200-3,500 years ago, overlapping the time when Indo-European languages first began to be spoken in the subcontinent. These results suggest that this formative period of Indian history was accompanied by mixtures between two highly diverged populations, although our results do not rule other, older ANI-ASI admixture events.” I wonder how much money these researchers put on only one admixture. I’m highly skeptical that the Indo-European demographic pulse could be so powerful that they could contribute to \~35% of the ancestry of South Indian tribal populations.

[Abstract Content  
Genetic of LCT gene in Amerindian populations from Brazil](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=10382&sid=972862). “The -13910T allele associated with LP varied from 0.5% in the Xavante to 7.6% in the Guarani-Ñandeva, and the frequencies differ significantly between Kaingang and Xavante, Xavante and Guarani-Ñandeva, and between Guarani groups. The -13910T frequency is probably of European origin and its prevalence correlates with non-native admixture proportions….” As the sample sizes increase I think it would be interesting to see if European derived LCT alleles are lower in frequency than total genome ancestry. I’m modestly skeptical that such an exceptional gain of function mutation has no deleterious side effects.

[Influence of Ethnicity in Association with UCP2 -866G/A, PGC1α (Gly 482 Ser) and mtDNA 10398G/A Polymorphisms in Orissa Population Groups](http://www.ichg2011.org/cgi-bin/ichg11s?abst=mtDNA&sort=ptimes&sbutton=Detail&absno=10090&sid=887748). ” Thus the present investigation concludes that **differential pattern of association of polymorphisms is observed for different caste groups**, suggesting the putative role of ethnicity. Thus, for risk calculation and proper medical intervention, knowledge of the ethnicity and nature of variation in risk factors need serious attention.” This sounds crazy to me, but it is sort of what [Reich et al.](http://www.nature.com/nature/journal/v461/n7263/full/nature08365.html) predicted.

[Evidence for extensive ancient admixture in different human populations](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=21675&sid=972862). “…, we did find hundreds of regions that likely introgressed in from archaic human ancestors, and we estimate the amount and the timing of these ancient admixture events. These regions were found in the genomes of both sub-Saharan African and non-African populations. **While Neandertals are a natural source population for ancient admixture into non-Africans, the source for ancient admixture into sub-Saharan African populations is less obvious.**” Some of the researchers on this abstract have been arguing for ancient population structure/admixture in Africa for years. Obviously with the findings from the Neandertal genome these findings are now going to get more attention and credence.

[Genome-wide comparison of African-ancestry populations from CARe and other cohorts reveals signals of natural selection](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=11338&sid=972862).

[Genomic Reconstruction of an Extinct Population from Next-Generation Sequence Data – Insights from the Taìno Genome Project](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=10765&sid=972862). ” Today, the ancestral legacy of the Taìnos is found in traces of their genomes still present in the inhabitants of the islands. An incredible opportunity to recover these ancestral segments came when 35 Puerto Rican trios were included in the sampling plan of the 1,000 Genomes Project (TGP).” If you can reconstruct the Taino from Puerto Ricans, surely you could do the same for Neandertals and other “archaics.”

[Dating ancient admixture: the date of gene flow from Neandertals into modern Humans](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=21481&sid=972862). ” We observe that the empirical extent of LD is a factor of 3.5-times greater in non-Africans than in Africans, consistent with Neandertal sharing more alleles with non-Africans than with West Africans. Applying our correction for map uncertainty and taking into account biases in our statistic, **we conclude that the true date of gene flow was 37000-86000 years BP**, suggesting that it occurred when modern Humans encountered Neandertals, presumably in western Eurasia.”

[Abstract Content  
Genome-wide patterns of admixture among US Hispanic/Latino populations of Caribbean-descent](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=21754&sid=972862). “We observed patterns consistent with continuous influx of African ancestry across the islands and a pulse migration across populations in contact with the continental landmass. By contrast, we find less evidence for continuous flow of Native American ancestry (characterized by shorter ancestry tracts), compatible with a single migration event between 15-20 generations ago and no further contribution of Native Americans into the Caribbean.” All in perfect alignment with the dominant historical consensus. The main value I see in many of these historical population genetic results is a further clampdown on the tendency for fashionable and opportunistic revisionist scholarship which pops up every generation or so. I don’t have a problem with revisionism as such, there’s a lot of error, but in some fields it seems to flow with fashion, and not where the truth pools.

[Using ancestry information and tests of natural selection to prioritize craniofacial candidate genes for admixture mapping](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=21563&sid=972862). “fter testing, 30 out of 58 genes containing AIMs were found to exhibit signatures of natural selection. Nine out of 30 genes demonstrated significant evidence of natural selection in West Africans and 17 in Europeans. Additionally, four genes yielded significant results in both populations, although the proportion of significant SNPs in each window varied between the populations. These methods provide a means of prioritizing candidate genes for tests for phenotypic association via admixture linkage. Ancestry informative SNPs in these 30 selection-nominated candidate genes will be then tested for an association with normal human facial variation.” Imagine that you could get construct a prediction of your child’s face in the first trimester from genetic material retrieved! I dream of such things. (this naturally opens up the possibility of embryo selection too….)

[Population genetics of Finland revisited – looking Eastwards](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=21348&sid=972862). “Using a model-based estimation of individual ancestry, three ancestral populations provided a best fit for the combined Finnish and Saami dataset. Particularly, one of these ancestral populations was predominant in the Saami (average 78%), and higher in Northern Finland (average 14%) compared to the rest of the country (average 4%). Despite the fact that Finns are the closest relatives of the Saami of all populations included in this study, in general, our results show that language and genetics are only weakly related. The Finns are more closely related to most Indo-European speaking populations than to linguistically related populations such as the Saami.”

[Searching for adaptive introgression of favorable alleles in Yemeni populations](http://www.ichg2011.org/cgi-bin/ichg11s?abst=admixture&sort=ptimes&sbutton=Detail&absno=11272&sid=972862). I wish that the HapMap had included a Middle Eastern population. The old “Caucasoid” race is sampled from its western (European) and eastern (Gujarati) poles, with the intermediate populations having to be inferred.

[Characterization of the Function and Regulation of the Autism Susceptibility 2 (AUTS2) Gene](http://www.ichg2011.org/cgi-bin/ichg11s?abst=neanderthal&sort=ptimes&sbutton=Detail&absno=20782&sid=972862). “…Using comparative genomics and available ChIP-seq data sets we scanned the human-Neanderthal accelerated region and AUTS2 intron 4, which encompasses chromosomal deletions associated with ASD, for potential enhancer sequences. Using transgenic enhancer assays in zebrafish, we found 13 of the 36 tested sequences to be enhancers in the central nervous system (CNS) and to overlap with auts2 expression. Several selected sequences in this region were also tested and found to be positive CNS enhancers in mice. Combined, our results show that AUTS2 is an important neurodevelopment gene. In addition, they provide a regulatory map for CNS enhancers that could regulate AUTS2, revealing candidate sequences where nucleotide variation could lead to ASD susceptibility and **human specific traits.**” Do autistic people then lack “human specific traits”?

[African genome sequencing reveals diversity and origin of the “European” 17q21.31 inversion polymorphism](http://www.ichg2011.org/cgi-bin/ichg11s?abst=africa&sort=ptimes&sbutton=Detail&absno=21634&sid=972862). “In addition, the H2-specific duplication is polymorphic in Africans. SNP genotyping and copy number estimates from over 1500 unrelated individuals from worldwide populations support the hypothesis that the nonduplicated H2 represents the ancestral state and demonstrate that the H1 duplication is specific to Europe. We estimate that the evolutionary age of the H2 duplication is significantly older (850,000 years) when compared to the H1 duplication (320,000 years old). Remarkably, most Europeans (60%) carry either the H1 or H2 duplication, which is virtually absent (\<5%) in most other continental groups. Our results support the hypothesis that the H2 haplotype without extensive duplication is the ancestral haplotype, the H1- and H2-specific duplications emerged independently, and the duplications arose prior to the human migration out of Africa." If all non-Africans derive from one population this seems strange, in that Europeans and non-Europeans preserve totally different ancient African structure….

[Exceptions to the “One Drop Rule”? DNA evidence of African ancestry in European Americans](http://www.ichg2011.org/cgi-bin/ichg11s?abst=africa&sort=ptimes&sbutton=Detail&absno=22039&sid=972862). “We estimate that, overall, at least 2-3% of individuals with predominantly European ancestry have genetic patterns suggesting relatively deep ancestry tracing to Africa. This fraction is far lower than the genetic estimates of European ancestry of African Americans, consistent with the social history of the United States, but reveals that a small percentage of “mixed race” individuals were integrating into the European American community (passing for White) over 200 years ago, during the era of slavery in the United States.” The genetic data indicate that whites in North America are very different from whites in Latin America, in that the former tend to lack non-European admixture.

[Demographic histories of African hunting-gathering populations inferred from genome-wide SNP variation](http://www.ichg2011.org/cgi-bin/ichg11s?abst=africa&sort=ptimes&sbutton=Detail&absno=21450&sid=972862). ” We report, for the first time, evidence for recent common ancestry of Ethiopian hunter-gatherers and the Kenyan Sanye/Dahalo, who speak a language with remnant clicks, with click-speaking eastern African Khoesan populations. This work supports archaeological and linguistic studies that indicate that the distribution of Khoesan speaking populations may have extended as far north as Ethiopia.” There has been a lot of circumstantial evidence for this sort of thing, and you see it when you run ADMIXTURE yourself, in that there is some pan-East African element peaking in the Sandawe which reaches all the way to southern Ethiopia.

### Related Posts:

- [SMBE 2012](https://www.gnxp.com/WordPress/2012/06/24/smbe-2012/)
- [Comments getting caught in
  spam](https://www.gnxp.com/WordPress/2011/07/11/comments-getting-caught-in-spam/) - [Human Brain
  Evolving](https://www.gnxp.com/WordPress/2007/04/23/human-brain-evolving/) - [A little bit of the Others in
  Us](https://www.gnxp.com/WordPress/2010/03/17/a-little-bit-of-the-others-in-us/) - [Rise of the planet of the
  Indo-Europeans](https://www.gnxp.com/WordPress/2012/08/16/rise-of-the-planet-of-the-indo-europeans/) - [Thoughts on the \$1,000 genome, circa
  2007](https://www.gnxp.com/WordPress/2011/12/04/thoughts-on-the-1000-genome-circa-2007/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F08%2F31%2Fhuman-genetics-presentations-of-interest%2F&linkname=Human%20genetics%20presentations%20of%20interest "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F08%2F31%2Fhuman-genetics-presentations-of-interest%2F&linkname=Human%20genetics%20presentations%20of%20interest "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F08%2F31%2Fhuman-genetics-presentations-of-interest%2F&linkname=Human%20genetics%20presentations%20of%20interest "Email")[](https://www.addtoany.com/share)
