+++
title = "Convergent evolution"
full_title = "Convergent evolution happens!"
date = "2011-06-20"
upstream_url = "https://www.gnxp.com/WordPress/2011/06/20/convergent-evolution-happens/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/06/20/convergent-evolution-happens/).

Convergent evolution happens!

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/relatives1.jpg?resize=352%2C147)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/relatives1.jpg?resize=352%2C147)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/relatives1.jpg)In the image to the left you see three human males. You can generate three pairings of these individuals. When comparing these pairs which would you presume are more closely related than the other pairs? Now let me give you some more information. The rightmost image is of the president of Tanzania. The middle image is of the president of Taiwan (Republic of China). And finally, the leftmost image is of the prime minister of Papua New Guinea. With this information you should now know with certainty that the prime minister of Papua New Guinea and the president of Taiwan are much more closely related than either are to the president of Tanzania. But some of you may not have guessed that initially. Why? I suspect that physical inspection may have misled you. One of the most salient visible human characteristics is of the complexion of our largest organ, the skin. Its prominence naturally leads many to mistakenly infer relationships where they do not exist.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/0-2.jpg?resize=300%2C206)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/0-2.jpg?resize=300%2C206)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/0-2.jpg)This was certainly an issue when European explorers encountered the peoples of Melanesia. An older term for Melanesians is “Oceanic Negro,” and some sources suggest that the Spaniards who named the island New *Guinea* did so with an eye to the old Guinea on the coast of West Africa. To the left is an unrooted tree which illustrates the relationships between Papuans, Bantu from Kenya, and Han Chinese. Since the font is small I’ve underlined the focal populations in red. Africans are always the “outgroup” to any two non-African populations. This is a robust pattern whenever you look at averaged total genome phylogenies. In other words, when you don’t privilege particular genes in a phylogeny humanity can be divided into African and non-African branches.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)But, if you look at [pigmentation genes](http://mbe.oxfordjournals.org/content/24/3/710.full.pdf) you get a different picture altogether. As it happens, not only is variation in skin color a trait of great social importance, but it turns out to be one of the few phenotypes whose genetic architecture has been well elucidated by genomics. There are about half a dozen genes responsible for most of the between population variation in complexion. Populations far from the equator seem to have developed parallel means toward lighter skins, while those near the equator are more likely to exhibit similarities. **In other words, the phylogeny of these specific genes is out of sync with the average phylogeny of the genes in these populations.** It is the latter which is a good reflection of demographic history, not the former.

A new paper in *PLoS Genetics* looks at these sorts of parallel trends more broadly, not just focusing on one trait. In particular, the authors explore the possibilities of natural selection operating upon standing genetic variation across divergent lineages. This means that there need not be a novel mutation which is driven up toward fixation rapidly by positive selection in a “hard sweep.” Rather, as populations diversify they may be subject to selection pressures which take their extant genetic variation and shift the mean of the quantitative trait in a particular direction, altering the balance of underlying allele frequencies rather than substituting novel genetic variants at one or two loci. These are “soft sweeps.”

First, the author summary,[Parallel Adaptive Divergence among Geographically Diverse Human Populations](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002127#pgen.1002127-Barrett1):

> Identifying regions of the human genome that differ among populations > because of natural selection is both essential for understanding > evolutionary history and a powerful method for finding functionally > important variants that contribute to phenotypic diversity and > disease. Adaptive events on timescales corresponding to the human > diaspora may often manifest as relatively small changes in allele > frequencies at numerous loci that are difficult to distinguish from > stochastic changes due to genetic drift, rather than the more dramatic > selective sweeps described by classic models of natural selection. In > order to test whether a substantial proportion of interpopulation > genetic differences are indeed adaptive, we identify loci that have > undergone moderate allele frequency changes in multiple independent > human lineages, and we test whether these parallel divergence events > are more frequent than expected by chance. We report a significant > excess of polymorphisms showing parallel divergence, especially within > genes, a pattern that is best explained by geographically varying > natural selection. Our results indicate that local adaptation in > humans has occurred by subtle, repeated changes at particular genes > that are likely to be associated with important morphological and > physiological differences among human populations.

The statistics in this paper can be a bit daunting, but the basic logic is simple. The [HGDP](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project) data set has a lot of [SNP](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism) information on \~50 populations. These populations also exhibit variation in their phylogenetic relationships. We know, for example, that Amerindian populations are closer to East Asians than they are to Europeans. They pruned their population set down to very genetically distinctive groups. Those which don’t have too much admixture and are in ecologically unambiguous regions (so discard the Uyghur). For example, Europeans and East Asians in temperate climes, Pygmies and Papuans in the tropics. Comparing two pairs which were phylogenetically unrelated but ecologically distinctive in a similar manner they found broad evidence of parallel shifts in underlying allele frequency on a range of SNPs.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/journal.pgen_.1002127.g004.jpg?resize=300%2C302)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/journal.pgen_.1002127.g004.jpg?resize=300%2C302)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/journal.pgen_.1002127.g004.jpg)Remember, these are polymorphisms found in all populations. So natural selection is perturbing the frequencies around an average. Additionally, they focused on alleles with intermediate global frequency, so that one presumes there’s enough genetic variance for selection to be effective. Theoretically and through simulation the authors understand that a certain number of SNPs would be correlated in the manner which would imply parallel positive selection, and so possible convergence of trait values. **But the authors found that for several comparisons across groups there was an excess of detected SNPs.** And, the distribution across regions of the genomes for these detected SNPs is very suggestive. There was an excess of SNPs in coding regions of the genome. And, there was an even greater excess on base pairs where a change in state would result in a change in the protein! In other words, regions of the genome implicated in genuine function show more hints of convergence across unrelated lineages.

They also found particular patterns in the genes which were enriched for parallel selection:

> Genes overlapping parallel divergent SNPs were modestly enriched for > diverse functional categories associated with various cell types > including neurons, lymphocytes, cancer, and epithelium…Among the most > extreme parallel divergent genes (observed at a threshold of 0.5%) > were the skin keratinization gene*ABCA12*…*SH2B1*, which controls > serum letpin levels and body weight…*GRM5*, a glutamate receptor > associated with schizophrenia…and with pigmentation via the closely > linked*TYR*…*ATP2A2*, which causes a neuropsychiatric/keratinization > disorder…*F13A1*, a coagulation factor linked to numerous > cardiovascular diseases and to Alzheimer’s…and*IFIH1*, associated > with antiviral defense, type 1 diabetes, and psoriasis..**.The > pleiotropic nature of many of these genes suggests that selection on > one trait may have affected the evolution of other traits.**

On the last part: there’s a “correlation matrix” between genetic variance and trait variance. If you slam the genome with natural selection there will usually be a correlated response on a host of traits unrelated to the target of selection because of the complex contingent nature of biological pathways. Modulating gene X to shift the value of trait 1 to increase local fitness can have large consequences for trait 2, trait 3, trait 4, and so forth.

Of course **most of the SNPs detected are *not* targets of selection.** Remember, one assumes that simply due to random chance some SNPs will exhibit patterns which spuriously match those of regions which are the targets of parallel selection pressures. Rather, the importance of this paper is that it is another step to fleshing out the broader general theory of how adaptation and demographic events interplay across the arc of human history. It was always understood that convergent evolution is a force which must have shaped humans as they diversified and radiated across the world, but the genetic details were often left unspecified for various reasons. By filling in those details we may be able to stumble upon some very interesting general insights about the parameters which frame evolutionary process.

For example:

> …**South Americans may carry alleles adapted to temperate climates due > to their ancestral migration across Beringia, and they may have lacked > adequate time and/or genetic variation to completely re-adapt to a > tropical environment.** One SNP that fits this hypothesis lies > in*DDB1*, which protects the skin from solar UV exposure…and is one of > the strongest examples of this parallel divergence pattern, with one > allele fixed in South America, over 90% in Europe and East Asia, and > less than 40% in African and Oceania….

A biological anthropologist once told me that South American Indians look like Siberians in their bodily proportions in relation to other tropical people. Which makes sense **since they are probably the descendants of Siberians!** As for skin color, this is an interesting trait insofar as it looks like that our species evolved very dark skin relatively early in our history, at the point when we lost our fur. Tropical populations usually exhibit a modicum of functional constraint. They don’t deviate too far from the ancestral type. In contrast, temperate zone populations often “lose” the function on these pigment producing genes, though differently. To “break” a gene is far easier than to put it back together, and I suspect that’s what you’re seeing with Amerindians in the tropics. A small ancestral population which traversed Beringia only carried a non-functional copy, which probably accumulated many mutations. Once they reentered the tropical zone they “needed” function again, but that would take too many independent steps for 10-15,000 years to suffice.

As noted by the authors this was definitely a first pass. With thicker sequence level data, and better population coverage, presumably one could explore more fine-grained questions. But at least there are results which confirm what one always assumed in theory. Sometimes it is just good to do a check, because actually you never really know….

**Citation:** Tennessen JA, & Akey JM (2011). Parallel Adaptive Divergence among Geographically Diverse Human Populations PLoS Genetics

### Related Posts:

- [Clones seize European
  government](https://www.gnxp.com/WordPress/2006/07/10/clones-seize-european-government/) - [The candidate we
  deserve](https://www.gnxp.com/WordPress/2012/11/03/the-candidate-we-deserve/) - [A Black
  Swan?](https://www.gnxp.com/WordPress/2008/11/05/a-black-swan/) - [Is Newt Gingrich too fat to be
  president?](https://www.gnxp.com/WordPress/2011/12/07/is-new-gingrich-too-fat-to-be-president/) - [The Arabs of
  Brazil](https://www.gnxp.com/WordPress/2006/12/04/the-arabs-of-brazil/) - [Lawrence Summers
  Resigns](https://www.gnxp.com/WordPress/2006/02/21/lawrence-summers-resigns/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F20%2Fconvergent-evolution-happens%2F&linkname=Convergent%20evolution%20happens%21 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F20%2Fconvergent-evolution-happens%2F&linkname=Convergent%20evolution%20happens%21 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F20%2Fconvergent-evolution-happens%2F&linkname=Convergent%20evolution%20happens%21 "Email")[](https://www.addtoany.com/share)
