+++
title = "Complex interactions"
full_title = "Complex interactions among epilepsy genes"
date = "2011-06-28"
upstream_url = "https://www.gnxp.com/WordPress/2011/06/28/complex-interactions-among-epilepsy-genes/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/06/28/complex-interactions-among-epilepsy-genes/).

Complex interactions among epilepsy genes

[![](https://i0.wp.com/4.bp.blogspot.com/-oEjWIKR7gE8/TgmO2HhgaPI/AAAAAAAAAJk/uixy_jSxyLg/s320/epileptic%2Bbrain.png?w=640)![](https://i0.wp.com/4.bp.blogspot.com/-oEjWIKR7gE8/TgmO2HhgaPI/AAAAAAAAAJk/uixy_jSxyLg/s320/epileptic%2Bbrain.png?w=640)](https://i0.wp.com/4.bp.blogspot.com/-oEjWIKR7gE8/TgmO2HhgaPI/AAAAAAAAAJk/uixy_jSxyLg/s1600/epileptic%2Bbrain.png) A debate has been raging over the last few years over the nature of the genetic architecture of so-called “complex” disorders. These are disorders – such as schizophrenia, epilepsy, type II diabetes and many others – which are clearly heritable across the population, but which do not show simple patterns of inheritance. A new study looking at the profile of mutations in hundreds of genes in patients with epilepsy dramatically illustrates this complexity. The possible implications are far-reaching, especially for our ability to predict risk based on an individual’s genetic profile, but do these findings apply to all complex disorders?

Complex disorders are so named because, while it is clear that they are highly heritable (risk to an individual increases the more closely related they are to someone who has the disorder), their mode of inheritance is far more difficult to discern. Unlike [classical Mendelian](https://en.wikipedia.org/wiki/Monogenic_disorder#Single_gene_disorder) disorders (such as cystic fibrosis or Huntington’s disease), these disorders do not show simple patterns of segregation within families that would peg them as recessive or dominant, nor can they be linked to mutations in a single gene. This has led people to propose two very different explanations for how they are inherited.

One theory is that such disorders arise due to unfortunate combinations of large numbers of genetic variants that are common in the population. Individually, such variants would have little effect on the phenotype, but collectively, if they surpass some threshold of burden, they could tip the balance into a pathological state. This has been called the common disease/common variant (CD/CV) model.

The alternative model is that these “disorders” are not really single disorders at all – rather they are umbrella terms for collections of a large number of distinct genetic disorders, which happen to result in a similar set of symptoms. Within any individual or family, the disorder may indeed be caused by a particular mutation. Because many of the disorders in question are very severe, with high mortality and reduced numbers of offspring, these mutations will be rapidly selected against in the population. They will therefore remain very rare and many cases of the disorder may arise from new, or de novo, mutations. This has therefore been called the multiple rare variants (MRV) model.

[![](https://i0.wp.com/4.bp.blogspot.com/-mliCIjhgFZ4/TgmN-WIL6BI/AAAAAAAAAJc/Fa7ju3tnky8/s320/Oligogenic.jpg?w=640)![](https://i0.wp.com/4.bp.blogspot.com/-mliCIjhgFZ4/TgmN-WIL6BI/AAAAAAAAAJc/Fa7ju3tnky8/s320/Oligogenic.jpg?w=640)](https://i0.wp.com/4.bp.blogspot.com/-mliCIjhgFZ4/TgmN-WIL6BI/AAAAAAAAAJc/Fa7ju3tnky8/s1600/Oligogenic.jpg) Lately, a number of mixed models have been proposed by various researchers, including myself. Even classical Mendelian disorders rarely show strictly Mendelian inheritance – instead the effects of the major mutations are invariably affected by modifiers in the genetic background. (These are variants with little effect by themselves but which may have a strong effect in combination with some other mutation). If this sounds like a return to the CD/CV model, there are a couple important distinctions to keep in mind. One is the nature of the mutations involved – the mixed model would still invoke some rare mutation that has a large effect on protein function. It may not always cause the disorder by itself (i.e., not every one who carries it will be affected), but could still be called causative in the sense that if the affected individual did not carry it one would expect they would not suffer from the disorder. The other is the number of mutations or variants involved – under the CD/CV model this could number in the thousands (a polygenic architecture), while under the mixed model one could expect a handful to be meaningfully involved (an oligogenic architecture – see diagram from [review](http://www.ncbi.nlm.nih.gov/pubmed?term=mitchell%20kj%2C%20neurodevelopmental) in Current Opinion in Neurobiology).

The new study, from the lab of Jeff Noebels, aimed to test these models in the context of epilepsy. Epilepsy is caused by an imbalance in excitation and inhibition within brain circuits. This can arise due to a large number of different factors, including alterations in the structural organisation of the brain, which may be visible on magnetic resonance imaging. Many neurodevelopmental disorders are therefore associated with epilepsy as a symptom (usually one of many). But it can also arise due to more subtle changes, not in the gross structure of the brain or the physical wiring of different circuits, but in the way the electrical activity of individual neurons is controlled.

[![](https://i0.wp.com/4.bp.blogspot.com/-m7zuavWFq5k/TgmNxqyyfoI/AAAAAAAAAJU/p-qa2ndpzco/s320/ion%2Bchannels.png?w=640)![](https://i0.wp.com/4.bp.blogspot.com/-m7zuavWFq5k/TgmNxqyyfoI/AAAAAAAAAJU/p-qa2ndpzco/s320/ion%2Bchannels.png?w=640)](https://i0.wp.com/4.bp.blogspot.com/-m7zuavWFq5k/TgmNxqyyfoI/AAAAAAAAAJU/p-qa2ndpzco/s1600/ion%2Bchannels.png) The electrical properties of any neuron – how excitable it is, how long it remains active, whether it fires a burst of action potentials or single ones, what frequency it fires at and many other important parameters – are determined in large part by the particular [ion channel](https://en.wikipedia.org/wiki/Ion_channel) proteins it expresses. These proteins form a pore crossing the membrane of the cell, through which electrically charged ions can pass. Different channels are selective for sodium, potassium or calcium ions and can be activated by different types of stimuli – binding a particular neurotransmitter or a change in the cell’s voltage for example. Many channels are formed from multiple subunits, each of which may be encoded by a different gene. There are hundreds of these genes in several large families, so the resultant complexity is enormous.

Many familial cases of epilepsy have been found to be caused by mutations in ion channel genes. However, most epilepsy patients outside these families do not carry these particular mutations. Therefore, despite these findings and despite the demonstrated high heritability, the particular genetic cause of the vast majority of cases of epilepsy has remained unknown. Large genome-wide association studies have looked for common variants that are associated with risk of epilepsy but have turned up nothing of note. The interpretation has been that common variants do not play a major role in the etiology of [idiopathic epilepsy](https://en.wikipedia.org/wiki/Idiopathic_generalized_epilepsy) (epilepsy without a known cause).

The rare variants model suggests that many of these cases are caused by single mutations in any of the very large number of ion channel genes. A straightforward experiment to test that would be to sequence all these candidate genes in a large number of epilepsy patients. The hope is that it would be possible to shake out the “low hanging fruit” – obviously pathogenic mutations in some proportion of cases. The difficulty lies in recognising such a mutation as pathogenic when one finds it. This generally relies on some statistical evidence – any individual mutation, or such mutations in general, should be more frequent in epilepsy patients than in unaffected controls. The experiment must therefore involve as large a sample as possible and a control comparison group as well as patients.

Klassen and colleagues sequenced 237 ion channel genes in 152 patients with idiopathic epilepsy and 139 healthy controls. What they found was surprising in several ways. They did find lots of mutations in these genes, but they found them at almost equal frequency in controls as in patients. Even the mutations predicted to have the most severe effects on protein function were not significantly enriched in patients. Indeed, mutations in genes already known to be linked to epilepsy were found in patients and controls alike (though 96% of patients had such a mutation, so did 67% of controls). Either these specific mutations are not pathogenic or their effects can be strongly modified by the genetic background.

More interesting results emerged from looking at the occurrence of multiple mutations in these genes in individuals. 78% of patients vs 30% of controls had two or more mutations in known familial epilepsy genes. A similar trend was observed when looking at specific ion channel gene families, such as GABA receptors or sodium channels.

These data would seem to fit with the idea that an increasing mutational load pushes the system over a threshold into a pathological state. The reality seems more complicated, however, and far more nuanced. Though the average load was lower, many controls had a very high load and yet were quite healthy. It seems that the specific pattern of mutations is far more important than the overall number. This fits very well with the known biology of ion channels and previous work on genetic interactions between mutations in these genes.

Though one might expect a simple relationship between number of mutations and severity of phenotype, that is unlikely to be the case for these genes. It is well known that the effects of a mutation in one ion channel gene can be suppressed by mutation in another gene – restoring the electrical balance in the cell, at least to a degree sufficient for performance under normal conditions. The system is so complex, with so many individual components, that these interactions are extremely difficult to predict. This is complicated further by the fact that there are active processes within the system that act to normalise its function. It has been very well documented, especially by Eve Marder and colleagues, that changes to one ion channel in a neuron can be compensated for by homeostatic mechanisms within the cell that aim to readjust the electrical set-points for optimal physiological function. In fact, these mechanisms do not just happen within one cell, but across the circuit.

The upshot of the study is that, though some of the mutations they discovered are indeed likely to be the pathogenic culprits, it is very difficult to discern which ones they are. It is very clear that there is at least an oligogenic architecture for so-called “[channelopathies](https://en.wikipedia.org/wiki/Channelopathy)” – the phenotype is determined by several mutations in each individual. (Note that this is not evidence for a highly polygenic architecture involving hundreds or thousands of genetic variants with tiny individual effects). The important insight is that it is not the overall number or mutational load that matters but the pattern of specific mutations in any individual that is crucial. Unfortunately, given how complicated the system is, this means it is currently not possible to predict an individual’s risk, even with this wealth of data. This will likely require a lot more biological information on the interactions between these mutations from experimental approaches and computational modelling.

What are the implications for other complex disorders? Should we expect a similarly complicated picture for diseases like schizophrenia or autism? Perhaps, though I would argue against over-extrapolating these findings. For the reasons described above, mutations in ion channel genes will show especially complex genetic interactions – it is, for example, even possible for two mutations that are individually pathogenic to suppress each other’s effects in combination. This is far less likely to occur for classes of mutations affecting processes such as neurodevelopment, many of which have been implicated in psychiatric disorders. Though by no means unheard of, it is far less common for the effects of one neurodevelopmental mutation to be suppressed by another – it generally just makes things worse. So, while modifying effects of genetic background will no doubt be important for such mutations, there is some hope that the interactions will be more straightforward to elucidate (mostly enhancing, far fewer suppressing). Others may see it differently of course (and I would be pleased to hear from you if you do); similar sequencing efforts currently underway for these disorders may soon tell whether that prediction is correct.

Klassen T, Davis C, Goldman A, Burgess D, Chen T, Wheeler D, McPherson J, Bourquin T, Lewis L, Villasana D, Morgan M, Muzny D, Gibbs R, & Noebels J (2011). Exome sequencing of ion channel genes reveals complex profiles confounding personal risk assessment in epilepsy. Cell, 145 (7), 1036-48 PMID: [21703448](http://www.ncbi.nlm.nih.gov/pubmed/21703448)

Kasperaviciute, D., Catarino, C., Heinzen, E., Depondt, C., Cavalleri, G., Caboclo, L., Tate, S., Jamnadas-Khoda, J., Chinthapalli, K., Clayton, L., Shianna, K., Radtke, R., Mikati, M., Gallentine, W., Husain, A., Alhusaini, S., Leppert, D., Middleton, L., Gibson, R., Johnson, M., Matthews, P., Hosford, D., Heuser, K., Amos, L., Ortega, M., Zumsteg, D., Wieser, H., Steinhoff, B., Kramer, G., Hansen, J., Dorn, T., Kantanen, A., Gjerstad, L., Peuralinna, T., Hernandez, D., Eriksson, K., Kalviainen, R., Doherty, C., Wood, N., Pandolfo, M., Duncan, J., Sander, J., Delanty, N., Goldstein, D., & Sisodiya, S. (2010). Common genetic variation and susceptibility to partial epilepsies: a genome-wide association study Brain, 133 (7), 2136-2147 DOI: [10.1093/brain/awq130](https://dx.doi.org/10.1093/brain/awq130)

Mitchell KJ (2011). The genetics of neurodevelopmental disease. Current opinion in neurobiology, 21 (1), 197-203 PMID: [20832285](http://www.ncbi.nlm.nih.gov/pubmed/20832285)

Mirrored from http://wiringthebrain.blogspot.com

### Related Posts:

- [What’s in a name? Genetic overlap between major
  psychiatric…](https://www.gnxp.com/WordPress/2010/05/26/whats-in-a-name-genetic-overlap-between-major-psychiatric-disorders/) - [Searching for a needle in a
  needlestack](https://www.gnxp.com/WordPress/2010/10/24/searching-for-a-needle-in-a-needlestack/) - [There are no common disorders (just extremes of
  quantitative…](https://www.gnxp.com/WordPress/2010/03/14/there-are-no-common-disorders-just-extremes-of-quantitative-traits/) - [Nature/Nurture in Development and
  Disorder](https://www.gnxp.com/WordPress/2006/08/03/nature-nurture-in-development-and-disorder/) - [I've got your missing heritability right
  here...](https://www.gnxp.com/WordPress/2012/02/07/ive-got-your-missing-heritability-right-here/) - [Defining developmental disorders through
  genetics](https://www.gnxp.com/WordPress/2010/08/13/defining-developmental-disorders-through-genetics/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F28%2Fcomplex-interactions-among-epilepsy-genes%2F&linkname=Complex%20interactions%20among%20epilepsy%20genes "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F28%2Fcomplex-interactions-among-epilepsy-genes%2F&linkname=Complex%20interactions%20among%20epilepsy%20genes "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F28%2Fcomplex-interactions-among-epilepsy-genes%2F&linkname=Complex%20interactions%20among%20epilepsy%20genes "Email")[](https://www.addtoany.com/share)
