+++
title = "\"What if you're wrong\" –"
full_title = "\"What if you're wrong\" – haplogroup J"
date = "2011-06-28"
upstream_url = "https://www.gnxp.com/WordPress/2011/06/28/what-if-youre-wrong-haplogroup-j/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/06/28/what-if-youre-wrong-haplogroup-j/).

"What if you're wrong" – haplogroup J

Back when this sort of thing was cutting edge [mtDNA haplogroup J](https://en.wikipedia.org/wiki/Haplogroup_J_(mtDNA)#Distribution) was a pretty big deal. This was the haplogroup often associated with the demic diffusion of Middle Eastern farmers into Europe. This was the [“Jasmine”](https://en.wikipedia.org/wiki/The_Seven_Daughters_of_Eve#Mitochondrial_haplogroups_in_The_Seven_Daughters_of_Eve) clade in [Seven Daughters of Eve](https://www.amazon.com/exec/obidos/ASIN/0393323145/geneexpressio-20). A new paper in *PLoS ONE* makes an audacious claim: **that J is not a lineage which underwent recent demographic expansion, but rather one which has been subject to a specific set of evolutionary dynamics which have skewed the interpretations due to a false “molecular clock” assumption.** By this assumption, I mean that mtDNA, which is passed down in an unbroken chain from mother to daughter, is by and large neutral to forces like natural selection and subject to a constant mutational rate which can serve as a calibration clock to the last common ancestor between two different lineages. Additionally, mtDNA has a high mutational rate, so it accumulates lots of variation to sample, and, it is copious, so easy to extract. What’s not to like?

First, the paper, [Mutation Rate Switch inside Eurasian Mitochondrial Haplogroups: Impact of Selection and Consequences for Dating Settlement in Europe](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0021543?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+plosone%2FEvolutionaryBiology+%28PLoS+ONE+Alerts%3A+Evolutionary+Biology%29):

> R-lineage mitochondrial DNA represents over 90% of the European > population and is significantly present all around the planet (North > Africa, Asia, Oceania, and America). This lineage played a major role > in migration “out of Africa” and colonization in Europe. In order to > determine an accurate dating of the R lineage and its sublineages, we > analyzed 1173 individuals and complete mtDNA sequences from Mitomap. > This analysis revealed a new coalescence age for R at 54.500 years, as > well as several limitations of standard dating methods, likely to lead > to false interpretations. These findings highlight the association of > a striking under-accumulation of synonymous mutations, an > over-accumulation of non-synonymous mutations, and the phenotypic > effect on haplogroup J. Consequently, **haplogroup J is apparently not > a Neolithic group but an older haplogroup (Paleolithic) that was > subjected to an underestimated selective force.** These findings also > indicated an under-accumulation of synonymous and non-synonymous > mutations localized on coding and non-coding (HVS1) sequences for > haplogroup R0, which contains the major haplogroups H and V. These new > dates are likely to impact the present colonization model for Europe > and confirm the late glacial resettlement scenario.

John Hawks has [written at length](http://johnhawks.net/taxonomy/term/236) of the possible distortions that selection might produce in our understanding of the history of mtDNA lineages, and therefore our understanding of the history of the population groups which these genealogies are used as proxies for. So I won’t review that much. I find the dynamics that they’re detecting possible, even plausible. But I don’t see why the authors having introduced skepticism start to conjure up positive visions of what is the true nature of the demographics which underpin these mtDNA phylogenies, now that they’ve “corrected” for variation in the power of the molecular clock to let use look through the glass clearly.

Readers with more fluency in the mtDNA literature can probably pick it apart. At the end of the day I’m always wondering **what do the subfossils tell us?** In other words, ancient DNA. Inferences from contemporary populations have been a total hash at a finer grain than that of continents, so you probably shouldn’t rest on that leg alone.

Finally, I thought this paper was of interest because it’s an inversion of [R1b1b2](http://blogs.discovermagazine.com/gnxp/2011/06/first-farmers-facing-the-ocean/). That’s a Y chromosomal haplogroup which was once presumed to be Paleolithic but now seems likely to be Neolithic. These authors are claiming that a mtDNA haplogroup which was once presumed to be Neolithic is actually Paleolithic. All this I think indicates that we should be modulating outward our error bars whenever we make assertions based on uniparental data with any time depth and below a very coarse level of spatial granularity.

### Related Posts:

- [Where are the "Paleolithic
  Europeans"?](https://www.gnxp.com/WordPress/2010/01/19/where-are-the-paleolithic-europeans/) - [European man perhaps not a Middle Eastern
  farmer](https://www.gnxp.com/WordPress/2010/04/30/european-man-perhaps-not-a-middle-eastern-farmer/) - [Phylogeography of deep European genetic
  history](https://www.gnxp.com/WordPress/2010/04/22/phylogeography-of-deep-european-genetic-history/) - [Surfing in from the
  east](https://www.gnxp.com/WordPress/2014/06/05/surfing-in-from-the-east/) - [Single markers tell you only a bit about individual
  ancestry](https://www.gnxp.com/WordPress/2015/01/22/single-markers-tell-you-only-a-bit-about-individual-ancestry/) - [European man perhaps a Middle Eastern
  farmer](https://www.gnxp.com/WordPress/2010/01/19/european-man-perhaps-a-middle-eastern-farmer/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F28%2Fwhat-if-youre-wrong-haplogroup-j%2F&linkname=%22What%20if%20you%27re%20wrong%22%20%E2%80%93%20haplogroup%20J "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F28%2Fwhat-if-youre-wrong-haplogroup-j%2F&linkname=%22What%20if%20you%27re%20wrong%22%20%E2%80%93%20haplogroup%20J "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F28%2Fwhat-if-youre-wrong-haplogroup-j%2F&linkname=%22What%20if%20you%27re%20wrong%22%20%E2%80%93%20haplogroup%20J "Email")[](https://www.addtoany.com/share)
