+++
title = "New developments in HBD"
full_title = "New developments in HBD research"
date = "2005-02-19"
upstream_url = "https://www.gnxp.com/WordPress/2005/02/19/new-developments-in-hbd-research/"

+++
Source: [here](https://www.gnxp.com/WordPress/2005/02/19/new-developments-in-hbd-research/).

New developments in HBD research

[The Mercury News ran an interesting article](http://www.mercurynews.com/mld/mercurynews/10932272.htm) (registration required) yesterday titled, “Gene researchers find variations by ancestry.” It describes a new paper published in Science by David Hinds, et al, describing DNA variation between three human populations. And \[surprise!\] the paper is astonished to see that the researchers believe that race (the term “human populations” is used in the paper) may be significant. Interestingly enough, the term “race” is never used in the actual published paper, so you can see that the newspaper is playing with semantics and creating a controversy on this issue by using a word that quickly incites knee-jerk reactions. Here’s some snippets from the actual article, since I know that nobody wants to go and register…

The first comprehensive map of genetic variation among several ethnic groups, published in today’s issue of the journal Science, shows patterns of genetic variation that could explain differences in health, disease and response to medication. This is a key step toward the possibility of personalized medicine based on genetic variations.

…

Although it is not their goal, the Perlegen scientists have found differences that suggest “race” has biological significance.

Critics fear that the identification of biological differences among races could bolster cranks and demagogues, allowing scientists to play into the hands of racists. Many anthropologists, sociologists, geneticists and population biologists consider race a social construct.

But scientists and doctors say the idea of race-based medicine has new respectability — and that identifying tiny differences could help reduce health disparities among the races.

It is known, for instance, that hypertension affects black Americans at a higher rate than white Americans. And white Americans sometimes take longer to clear certain drugs from the liver than East Asians.

Life’s genetic blueprint is 99.9 percent similar from person to person.

The remaining 0.1 percent consists of single-letter DNA variations called SNPs (pronounced SNIPS), or single nucleotide polymorphisms.

Most patterns of genetic variation are common and found in all populations.

But others are less common — and are likely to determine a person’s vulnerability to disease and response to medications, as well as other traits, such as eye or hair color, height and body type.

Uncommon SNPs occur in different frequencies in different populations. For instance, 70 percent of African-Americans might have a nucleotide represented by the letter A; 30 percent might have a letter T. At the same spot, the percentage may be flipped in European-Americans. The difference in frequency of a specific letter could make a population more susceptible to disease.

Finding those differences, and identifying whether they have any clinical significance, is the goal, said Paul Cuzenza, who oversees research collaborations at Perlegen.

The Perlegen researchers analyzed nearly 1.6 million SNPs across 71 unrelated individuals.

They found that most of the SNPs were common to the three human populations in the study. But 94 percent of the study’s SNPs were found in African-Americans, 81 percent in European-Americans and 74 percent in Chinese-Americans.

The presence of these patterns allowed the scientists to create the first picture of the structure of human genetic variation.

“Our paper in no way makes any sort of a scientific statement or definition of race,” said David Cox, Perlegen’s chief scientific officer. “When you look at any group of individuals, you’ll see differences in their DNA.”

Perlegen is now working to generate an even better map, describing variation across individuals of Japanese, Chinese, Nigerian and European ancestry.

The company hopes to include 4 million SNPs in 270 individuals by the end of the year.

Here’s an extract from the paper:

Individual differences in DNA sequence are the genetic basis of human variability. We have characterized whole-genome patterns of common human DNA variation by genotyping 1,586,383 single-nucleotide polymorphisms (SNPs) in 71 Americans of European, African, and Asian ancestry. Our results indicate that these SNPs capture most common genetic variation as a result of linkage disequilibrium, the correlation among common SNP alleles. We observe a strong correlation between extended regions of linkage disequilibrium and functional genomic elements. Our data provide a tool for exploring many questions that remain regarding the causal role of common human DNA variation in complex human traits and for investigating the nature of genetic variation within and between human populations.

Since I’m such a nice guy, I’ve uploaded the actual paper in the [gnxpfiles](http://groups.yahoo.com/group/gnxpfiles/) group. The title of the paper, Whole-Genome Patterns of Common DNA Variation in Three Human Populations, is called “1072.pdf” in the Files section of gnxpfiles:

I’ve also uploaded an interesting article from the “Perspectives” section of the journal about medical applications of this new research, written by a member of the International HapMap Project. The title of the paper, Harvesting Medical Information from the Human Family Tree, is called “1052.pdf” in the Files section of gnxpfiles:

Addendum from Razib: Because we put so many PDFs up early this month GNXP will come close to our very generous bandwidth allotments. So I’ve created a “[gnxpfiles](http://groups.yahoo.com/group/gnxpfiles/)” yahoo group to place PDFs for now.

Posted by Arcane at [03:05 PM](https://www.gnxp.com/MT2/archives/003618.html) [](http://js-kit.com/api/static/pop_comments?ref=http://gnxp.com&path=/3618?url=http://www.gnxp.com/MT2/archives/003618.html&thetime=%20021905&MT=true)

### Related Posts:

- [Ancient DNA analyses are
  accurate](https://www.gnxp.com/WordPress/2015/05/18/ancient-dna-analyses-are-accurate/) - [Fixing science, in
  part](https://www.gnxp.com/WordPress/2011/05/19/fixing-science-in-part/) - [Epigenetics News on
  'epigenetics'](https://www.gnxp.com/WordPress/2007/04/26/epigenetics-news-on-epigenetics/) - [The incentives for finding "genes
  for"....](https://www.gnxp.com/WordPress/2009/05/21/the-incentives-for-finding-genes-for/) - [David Rowe's final
  paper](https://www.gnxp.com/WordPress/2006/11/06/david-rowe-s-final-paper/) - [Same science, different
  inference](https://www.gnxp.com/WordPress/2009/10/12/same-science-different-inference/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F02%2F19%2Fnew-developments-in-hbd-research%2F&linkname=New%20developments%20in%20HBD%20research "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F02%2F19%2Fnew-developments-in-hbd-research%2F&linkname=New%20developments%20in%20HBD%20research "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F02%2F19%2Fnew-developments-in-hbd-research%2F&linkname=New%20developments%20in%20HBD%20research "Email")[](https://www.addtoany.com/share)
