+++
title = "MHC, HLA & immune"
full_title = "MHC, HLA & immune response"
date = "2005-02-09"
upstream_url = "https://www.gnxp.com/WordPress/2005/02/09/mhc-hla-immune-response/"

+++
Source: [here](https://www.gnxp.com/WordPress/2005/02/09/mhc-hla-immune-response/).

MHC, HLA & immune response

Recently, [David Boxenhorn](http://www.rishon-rishon.com/) and I had an exchange on the topic of immune response and its implications about the history of populations. In short, the relative heterozygosity of Ashkenazi Jews mitigates against an extreme population bottleneck in their history. Nevertheless, I realized that perhaps I should be more specific about immune related issues since I tend to take them as a “given” on this blog.

[Tissue rejection](http://www.nlm.nih.gov/medlineplus/ency/article/000815.htm) is a major problem when it comes to transplants. Because of the advances in medicine transplantation of organs can save the lives of those who would in the past have simply died. But, [nearly 100,000 people](http://www.unos.org/) in the United States are currently on waitlists. The problem is that the body’s immune system tags the transplanted tissue as “foreign,” resulting in a defense response which damages the organ and therefore the health of the patient. Having a “match” is no panacea, but [one study](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/T/Transplants.html#typing) (control-f “Number of HLA mismatches”) showed that 68% of kidney transplant patients were alive after five years when they had a perfect match (a 6 out of 6 antigen match) as opposed to 56% of those had zero matches. There are differences between organs, but I suspect most people would prefer 68% over 56% if it meant waiting a little longer and spending more money searching for a more appropriate donor (some organs, like the brain and testicles are relatively shielded from immune responses, though I know of no cases where these have been transplanted!).

The culprit is the [“Major Histocompability Complex”](http://www.cehs.siu.edu/fix/medmicro/mhc.htm) (MHC), which serves in a fashion as the body’s “anti-viral software,” scanning for and tagging foreign pathogens. [This site](http://www.cehs.siu.edu/fix/medmicro/mhc.htm) has a rather brief, but graphically oriented (including animation), synopsis of the physiology of the MHC.

For our purposes, we are concerned with the [Human Leukocyte Antigen](http://www.hon.ch/Library/Theme/Allergy/Glossary/hla.html) (HLA), in particular, class I and class II molecules as these do the heavy lifting when it comes to snitching out pathogens and other foreign tissue to our white cells. In the case of tissue typing three genes for class I and one or two for class II control the expression of the antigens which are usually being sought out as matches. The HLA coding region is scattered across a long section of [chromosome 6](http://alfred.med.yale.edu/alfred/entityresults.asp?condition=loci&chromo=6). Since humans are diploid, we have two copies of each gene, and HLA expression tends to be codominant, you have many variables in terms of the characterization of the immune profile.

Now take a look at this [list](http://www.anthonynolan.org.uk/HIG/lists/class1list.html) of alleles for class I (loci A, B and C are the ones normally checked for tissue typing) and another [list](http://www.anthonynolan.org.uk/HIG/lists/class2list.html) for class II (the DR and DQ loci are the important ones). As you can see, there are many alleles. Why is this so?

That is a topic I broach on this blog frequently, and the two candidates seem to be natural selection in the form of [balancing selection](https://www.gnxp.com/MT2/archives/003542.html) or [negative frequency dependent selection](http://www.blackwellpublishing.com/ridley/a-z/Frequency-dependent_selection.asp). In short, the former refers to “overdominance” or “heterozygote advantage,” which occurs on a loci if you have two different copies (alleles) of a gene and you are more “fit” than someone who is homozygous (duplicate copies). This preserves a mix of polymorphisms in the population. Frequency dependent selection works by giving an advantage to rare phenotypes, perhaps because pathogens have not had time to evolve a defense because it is so rare in the first place (this seems less popular at this time, perhaps because so many of the polymorphisms seem rather ancient and persistant).

Diversity is very advantageous if this level of polymorphism is a judge, as [heterozygosity](http://www.everythingbio.com/glos/definition.php?word=heterozygosity) on HLA A & B loci hovers around \~90% in most populations, in other words 9 out of 10 individuals carry different forms of the gene. I have mentioned before that the variation of the HLA loci seems to be very ancient, and some alleles coalesce to before the split between humans and chimpanzees. Well, I picked up a molecular evolution text I own to double check a few facts in this post and I note that some researchers see a possible coalescence on some alleles before the separation of prosimians and other primates over 65 million years ago. On the molecular level much of the genome is now known to be “neutral” in that random genetic drift is far more important than (possibly to the exclusion of) selection in shaping the character of the sequences, but on portions of the MHC coding region there is strong evidence of positive selection in maintaining diversity via greater substitutions on nonsynonomous (base pair substitutions which impact the type of amino acid coded) than synonomous substitutions (mutations which are silent because the change does not effect the amino acid that is translated).

So, I think I’ve made the point that extreme polymorphism is likely the result of evolutionary forces. In the case of tissue typing nature has presented us a problem. Consider that the phenotypes generated by five loci are proportional to the combinations of the alleles at these loci. So, if n = number of possible alleles, then n1\*n2\*n3n4\*n5 results in a enormous number of combinations. Add to this that we are diploid, and the loci are codominant, so if you imagine the alleles at each loci as a gene-complex with a specific genotype, and consider that there is always an analogous copy, then the numbers start to multiply very quickly (I believe (\[number of gene-complexes\] – 1)!/((\[number of gene-complexes\] – 1)!\*2, which results is some staggering back of the envelope calculations).

Now it isn’t as scary as all that, each allele is not exhibited at an equal frequency, and a perfect tissue match is not always necessary (see morbidity rates above for kidney transplants), but, a fraction of an enormous number is still rather large. If you are part of a large nuclear family you might be in luck, because these genes are all located on chromosome 6, and [crossing over](http://www.everythingbio.com/glos/definition.php?word=crossing+over) is not common, you are actually receiving one of your parent’s two gene-complexes, so your chance of matching up with a sibling is 1/4 (1/2 chance from you mother, and 1/2 chance from your father). Nevertheless, though there is some linkage disequilibrium on this portion of the genome (some alleles across the various loci are more likely to be found together because of natural selection favoring such a conformation), normally the linkage between various alleles is broken up through recombination over time on a population wide scale. Basically, if you don’t have a tissue match in your own family, you often have to look far and wide.

This is where interpopulation differences begin to loom large. As I said before, the various HLA polymorphisms are not created equal, some are far more common than others, but this comes into play on a populational level. HLA-A34, which is present in 78% of Australian Aborigines, has a frequency of less than 1% in both Australian whites and Chinese. Though the HLA loci have been shaped by natural selection, because of sharp differences between populations in the frequency of various alleles, they are still used to compare them (HLA alleles are also used in paternity tests because of their polygenic &  
polymorphic nature). You can [compare differences between populations yourself](http://www.ncbi.nlm.nih.gov/mhc/ihwg.cgi?ID=9&cmd=PRJOV) if you are really curious. If you are a member of a not-so-numerous-minority, you might be a little concerned after all I’ve said about tissue typing. This is, in my opinion, one of the more damaging aspects of the “race & ethnicity do not exist on a biological level” mantra, since mortality rates are real issues that must be grappled with, and a lack of awareness of tissue type differences might perpetuate a sanguine attitude amongst those who think that they have a healthy heart or kidney and don’t give much thought to the possibility that one day they too might need to get a hold of a spare organ.

Posted by razib at [12:24 AM](https://www.gnxp.com/MT2/archives/003580.html) [](http://js-kit.com/api/static/pop_comments?ref=http://gnxp.com&path=/3580?url=http://www.gnxp.com/MT2/archives/003580.html&thetime=%20020905&MT=true)

### Related Posts:

- [New theory of MHC
  diversity?](https://www.gnxp.com/WordPress/2008/11/06/new-theory-of-mhc-diversity/) - [Different immune systems...not attenuated
  ones](https://www.gnxp.com/WordPress/2006/07/15/different-immune-systems-not-attenuated-ones/) - [I am an unbeliever...in
  overdominance](https://www.gnxp.com/WordPress/2007/08/24/i-am-an-unbeliever-in-overdominance/) - [Clones are
  Cool](https://www.gnxp.com/WordPress/2006/07/12/clones-are-cool/) - [Non-adaptive immune
  adaptation?](https://www.gnxp.com/WordPress/2005/09/03/non-adaptive-immune-adaptation/) - [Starving because of
  plague](https://www.gnxp.com/WordPress/2008/03/10/starving-because-of-plague/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F02%2F09%2Fmhc-hla-immune-response%2F&linkname=MHC%2C%20HLA%20%26%20immune%20response "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F02%2F09%2Fmhc-hla-immune-response%2F&linkname=MHC%2C%20HLA%20%26%20immune%20response "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F02%2F09%2Fmhc-hla-immune-response%2F&linkname=MHC%2C%20HLA%20%26%20immune%20response "Email")[](https://www.addtoany.com/share)
