+++
title = "Patterns of genetic"
full_title = "Patterns of genetic diversity within Africa"
date = "2018-11-16"
upstream_url = "https://www.gnxp.com/WordPress/2018/11/16/patterns-of-genetic-diversity-within-africa/"

+++
Source: [here](https://www.gnxp.com/WordPress/2018/11/16/patterns-of-genetic-diversity-within-africa/).

Patterns of genetic diversity within Africa

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/11/violinPlotAfrica.jpg?resize=625%2C517&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/11/violinPlotAfrica.jpg?resize=625%2C517&ssl=1)](https://www.biorxiv.org/content/early/2018/11/14/470583?rss=1)

The violin-plot above is from a new preprint, [Runs of Homozygosity in sub-Saharan African populations provide insights into a complex demographic and health history](https://www.biorxiv.org/content/early/2018/11/14/470583?rss=1). Hereâ€™s the abstract:

> The study of runs of homozygosity (ROH), contiguous regions in the > genome where an individual is homozygous across all sites, can shed > light on the demographic history and cultural practices. We present a > fine-scale ROH analysis of 1679 individuals from 28 sub-Saharan > African (SSA) populations along with 1384 individuals from 17 > world-wide populations. Using high-density SNP coverage, we could > accurately obtain ROH as low as 300Kb using PLINK software. The > analyses showed a heterogeneous distribution of autozygosity across > SSA, revealing a complex demographic history. They highlight > differences between African groups and can differentiate between the > impact of consanguineous practices (e.g. among the Somali) and > endogamy (e.g. among several Khoe-San groups). The genomic > distribution of ROH was analysed through the identification of ROH > islands and regions of heterozygosity (RHZ). These homozygosity cold > and hotspots harbour multiple protein coding genes. Studying ROH therefore not only sheds light on population history, but can also be used to study genetic variation related to the health of extant populations.

This sort of run-of-homozygosity analysis is enabled by high-density genotyping or whole-genome sequencing. After quality control, the authors had 1 to 1.5 million SNPs for all populations.

The interesting thing about this preprint is that by looking at the violin-plots can you can see exactly all the things that population geneticists have learned about the demography, structure, and history of humans in the past generation or so.

- The rightmost panel shows the average total length of short ROH.
  Partly [the pattern](http://www.pnas.org/content/102/44/15942) fits
  into the older serial bottleneck model of the settlement of the world.
  The pattern of Amerindian \> East Asian \> European \> African. But
  what about the lower fractions for mixed Latin Americans and
  Gujuratis? This is a consequence of admixture, as these populations
  are mixtures in a sense of other groups. - The length of the long ROH segments, the second to last panel on the
  right, is indicative of recent patterns of marriage. Within Africa,
  you see some groups have many individuals with lots of long ROH
  segments. This is because ofconsanguinity. As the authors observe,
  the Oromo and Somali are both Cushitic speaking groups from the Horn
  of Africa, but the latter are universally Muslim, while only a
  minority of the former are. Islamic cultures have traditionally
  encouragedconsanguineousmarriages, and you can see the difference
  between these groups (whose total length of short segments is
  similar). - The pattern of ROH here can be predicted by simple genetic models: the
  extent of random mating within populations, recombination rates across
  the genome, and total population size. What modern genomic technology
  does is provide data to test the models.



### Related Posts:

- [Not all homozygosity is created the same
  (way)](https://www.gnxp.com/WordPress/2012/08/15/not-all-homozygosity-is-created-the-same-way/) - [Genetic diversity and intellectual
  disability](https://www.gnxp.com/WordPress/2013/07/05/genetic-diversity-and-intellectual-disability/) - [Runs of homozygosity are not good for your
  functioning](https://www.gnxp.com/WordPress/2018/01/17/runs-of-homozygosity-are-not-good-for-your-functioning/) - [African genetic
  structure](https://www.gnxp.com/WordPress/2009/04/30/african-genetic-structure/) - [Ancient human population
  sizes](https://www.gnxp.com/WordPress/2007/12/12/ancient-human-population-sizes/) - [Genomic diversity & historical
  inference](https://www.gnxp.com/WordPress/2009/02/16/genomic-diversity-historical-inference/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F11%2F16%2Fpatterns-of-genetic-diversity-within-africa%2F&linkname=Patterns%20of%20genetic%20diversity%20within%20Africa "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F11%2F16%2Fpatterns-of-genetic-diversity-within-africa%2F&linkname=Patterns%20of%20genetic%20diversity%20within%20Africa "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F11%2F16%2Fpatterns-of-genetic-diversity-within-africa%2F&linkname=Patterns%20of%20genetic%20diversity%20within%20Africa "Email")[](https://www.addtoany.com/share)
