+++
title = "A genetic map of"
full_title = "A genetic map of fireworks in time"
date = "2014-03-22"
upstream_url = "https://www.gnxp.com/WordPress/2014/03/22/a-genetic-map-of-fireworks-in-time/"

+++
Source: [here](https://www.gnxp.com/WordPress/2014/03/22/a-genetic-map-of-fireworks-in-time/).

A genetic map of fireworks in time

![**Citation:** [Towards a new history and geography of human genes informed by ancient DNA](http://biorxiv.org/content/early/2014/03/21/003517), Joseph Pickrell, David Reich, doi: 10.1101/003517](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/03/DiffModels.jpg?resize=600%2C457)

  
*And when the Lord thy God shall deliver them before thee; thou shalt smite them, and utterly destroy them; thou shalt make no covenant with them, nor show mercy unto them.*

*Neither shalt thou make marriages with them; thy daughter thou shalt not give unto his son, nor his daughter shalt thou take unto thy son.*

– Deuteronomy 7:2, 7:3

In my post [below](http://www.unz.com/gnxp/the-holocene-lattice/) I did not elaborate in detail my personal model for *how* the genetic variation we see around us in modern humans came about. Part of the reason is that what I have in mind is not necessarily parsimonious. Rather, it’s a conception that has developed organically over the years reading papers, looking at the data on humans myself, and finally feedback from people in the comments of this weblog. **One thing that Joe Pickrell and David Reich have shown with simulations though is that elegant parsimonious models can *fit* data with low enough granularity.** To be concrete the “serial founder bottleneck” framework wasn’t constructed out of thin air. Rather, it actually was rooted in empirical patterns which were evident in the genotypic data. But all models are approximations and miss details. For example, Sohini Ramachandran’s [2005 paper](http://www.pnas.org/content/102/44/15942.long) was anchored around Addis Ababa. But this was even at that time obviously a simplification, as the authors surely knew that there was plenty of circumstantial evidence that Ethiopia has been subject to recent admixture, so that particular population could not be the source of the Out of Africa expansion (see [Pagani et al.](https://www.cell.com/AJHG/abstract/S0002-9297(12)00271-6)for confirmation with dense marker data sets).

As another example, consider India. Between Western and Eastern Eurasia, this region is inhabited by populations which are at approximately symmetrical genetic distance in either direction, with perhaps a mild bias toward Western Eurasians (this may be an artifact of sampling too many high caste populations though). Taken at face value it is a perfect illustration of the maxim that geography predicts genetic variation. But what [Moorjani et al.](https://www.cell.com/AJHG/abstract/S0002-9297(13)00324-8) most recently have confirmed is that raw genetic distance measures which summarize allelic differences on the population wide level missed patterns of recent admixture evident on the genomic scale. That is, when you look across segments of the genome of South Eurasians it is obvious that they are the outcome of an admixture event between a West Eurasian population and another group which has closer affinities to Eastern Eurasians (and closest to Andaman Islanders). When you average these ancestral components out it does place South Asians between the two antipodes of Eurasia, but that elides an essential component of historical information about how that average came about.

![Cann, Rebecca L. “DNA and human origins.” Annual Review of Anthropology (1988): 127-143.](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/03/eve.jpg?resize=300%2C526)

To outline more fully what I have in mind it is essential to reiterate my understanding of the default model which coalesced about \~10 years ago, though its origins are deeper. The modern orthodoxy was solidified by [mtDNA Eve](https://en.wikipedia.org/wiki/Mitochondrial_Eve). What these results from the 1980s showed is that the phylogenetic tree of the mitochondrial lineage of humans is rooted in a diversification from an Africa population, with a series of branching-offs successively of anatomically modern groups which swept into Western Eurasia, Eastern Eurasia, Australasia and the New World, successively. In the 1990s microsatellites and other autosomal markers confirmed the broad outlines of this narrative. So somewhere on the order of \~50-100,000 years ago something happened, and the anatomically modern human beings which had been resident within Africa for a \~100,000 years left. After they left they swept aside all the other archaic hominins which were long resident across Eurasia, and charted new territory beyond in Australasia and the New World. As the branches of this human tree took root in particular localities they established the broad patterns of genetic variation we see around us today, with the obvious exceptions due to European colonialism. That is, you could take the phylogenetic tree and transpose it after a fashion on a geographic map, because the general framework of human variation was established \~15,000 years ago, after the [Last Glacial Maximum](https://en.wikipedia.org/wiki/Last_Glacial_Maximum). Here is a 2003 paper from*The American Journal of Human Genetics,*, [The genetic heritage of the earliest settlers persists both in Indian tribal and caste populations](http://www.ncbi.nlm.nih.gov/pubmed/12536373). In the abstract the authors state: “Taken together, these results show that Indian tribal and caste populations derive largely from the same genetic heritage of Pleistocene southern and western Asians and have received **limited gene flow from external regions since the Holocene.**” This last section seems likely to be wrong. Around half of the genetic heritage of South Asians is probably derived from a component whose ancestors were not within the confines of the Indian subcontinent before the Holocene.\* Similar issues crop up in Europe. Again, from *The American Journal of Human Genetics*, 2000, [Tracing European Founder Lineages in the Near Eastern mtDNA Pool](http://www.sciencedirect.com/science/article/pii/S0002929707629541). The last sentence of the abstract states that “the immigrant Neolithic component is likely to comprise less than one-quarter of the mtDNA pool of modern Europeans.” Though the authors were focusing on mtDNA, it led to a spate of articles which presumed that Europeans are predominantly descended from the Pleistocene populations of the continent. The current consensus is more muddled, though the [most recent work](http://biorxiv.org/content/early/2013/12/23/001552) suggests a greater role for farmers, and an [exogenous non-European element](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature12736.html?WT.ec_id=NATURE-20131121).

![**Credit:** Kwamikagami, Indigenous Australian language families](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/03/Australian_language_families.jpg?resize=300%2C238)

As Pickrell and Reich emphasize **there are simply too many exceptions to the old model of diversification Out of Africa and subsequent stasis and gene flow via isolation by distance dynamics.** It is fashionable to say that human genetic variation is clinal, and it is in a rough fashion. But this description does I think mislead us as to how these clines came about. The old model had a simple elegant attraction, as it consisted of a rapidly bifurcating phylogenetic tree, which was modified on the edges by subsequent gene flow in the new equilibrium environment. Where did it go wrong? First, I think the revision begins **before the Holocene.** The Holocene is not particularly special, there have been Interglacials before. What is different is that modern humans developed agriculture, and then what we call civilization. But not everywhere. In Australia the native populations remained hunter-gatherers up to the European discovery. But it is not feasible to assume that they were subject to cultural or genetic stasis. The expansion of the [Pama–Nyungan languages](https://en.wikipedia.org/wiki/Pama%E2%80%93Nyungan_languages) likely occurred within the last 4,000 years. This shows that **hunter-gatherer populations can enter into phases of rapid cultural change and expansion.** Does this reflect a genetic change across Australia? We don’t have extensive samples to test this hypothesis to my knowledge, though surely it will be checked within the next 10 years. I would predict that this change did perturb the genetic landscape across Australia. And it seems implausible that Australia was exceptional. When the first dense marker analyses or whole genomes come back from [Cro-Magnons](https://en.wikipedia.org/wiki/Cro-Magnon) I predict they will be radically different from the hunter-gatherers which were resident on the continent during the early Holocene. At least different enough to warrant the hypothesis of Pleistocene population turnover. I could be wrong, but the conjecture is not crazy.

Obviously agriculture changed things, by turbo-charging possible winner-take-all dynamics. We have historical cases of this. [French Canadians](http://blogs.discovermagazine.com/gnxp/2011/01/the-genomic-heritage-of-french-canadians/#.Uy4OYh_ztTM) and [Slavs](http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1001555) are both cases of populations which were once relatively modest and began in a narrow delimited region, but now are quite expansive and numerous. In the case of the East Slavs the demographic expansion also entailed the absorption of numerous Uralic tribes, as well as later Turks. And this illustrates one of the major details which I think has characterized the genetic turnover of human populations: **phase shifts from a relatively static one defined by isolation by distance gene flow across clinal gradiants, to a rapid expansion of a small subset, and the overlay of this component as a palimpsest over the underlying variation.** In some cases the replacement is nearly total, as in the modern United States. In other cases, as among Great Russians, the Slavic affinities of this population, and its association with Poles and other groups are clear, but there was a non-trivial uptake of exogenous segments which might allow for a reconstruction of the prior genetic landscape. These changes occur over short periods, and are bright fireworks against the comparatively static firmament. Explosions in the dark, which rapidly requilibrates to a stationary state.

**In a way what I am describing is Out of Africa writ large**. For decades scholars have been perplexed by the rapid expansion of anatomically modern humans out of Africa over a short period, and their subsequent sweep across the Old World and into Australia. But perhaps this is not *sui generis*, but the typical pattern of humans. For example the replacement of the Melanesian-like population of Southeast Asia over the last 4,000 years by people from the valleys and coastal zones of southern China could be said to resemble an Out of Africa event, perturbing a genetic landscape that had been relatively static for on the order of 10,000 years. And if humans, why not other medium sized mammals? The most [recent results](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004016) from dog genomics are intriguing, and the perplexing aspects are awful familiar to me.

\* I do not believe that the “Ancestral North Indians” were resident within the confines of the Indian subcontinent before the Holocene.

### Related Posts:

- [Bro do you even
  supplement?](https://www.gnxp.com/WordPress/2018/03/28/do-you-supplement/) - [Scientism,
  huh?](https://www.gnxp.com/WordPress/2006/11/26/scientism-huh/) - [Open Thread,
  4/2/2018](https://www.gnxp.com/WordPress/2018/04/01/open-thread-4-2-2018/) - [The State May Have Your Genome Sooner Than You
  Think](https://www.gnxp.com/WordPress/2010/06/14/the-state-may-have-your-genome-sooner-than-you-think/) - [ScienceBlogs has good
  blogs](https://www.gnxp.com/WordPress/2010/07/20/scienceblogs-has-good-blogs/) - [Kat](https://www.gnxp.com/WordPress/2006/11/24/kat-5/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F03%2F22%2Fa-genetic-map-of-fireworks-in-time%2F&linkname=A%20genetic%20map%20of%20fireworks%20in%20time "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F03%2F22%2Fa-genetic-map-of-fireworks-in-time%2F&linkname=A%20genetic%20map%20of%20fireworks%20in%20time "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F03%2F22%2Fa-genetic-map-of-fireworks-in-time%2F&linkname=A%20genetic%20map%20of%20fireworks%20in%20time "Email")[](https://www.addtoany.com/share)
