+++
title = "Living in the age of"
full_title = "Living in the age of structure"
date = "2016-09-05"
upstream_url = "https://www.gnxp.com/WordPress/2016/09/05/living-in-the-age-of-structure/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/09/05/living-in-the-age-of-structure/).

Living in the age of structure

[![Screenshot 2016-09-05 04.14.05](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/09/Screenshot-2016-09-05-04.14.05.png?resize=620%2C314)![Screenshot 2016-09-05 04.14.05](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/09/Screenshot-2016-09-05-04.14.05.png?resize=620%2C314)](http://biorxiv.org/content/biorxiv/early/2016/09/04/073221.full.pdf)

Jonathan Novembre and Benjamin Peter have posted a preprint of a review, [Recent advances in the study of fine-scale population structure in humans](http://biorxiv.org/content/biorxiv/early/2016/09/04/073221.full.pdf), which readers will find useful. In particular, the citations are a gold-mine for anyone attempting to navigate this literature.

The figure above from their preprint illustrates the number of markers needed to differentiate populations in Europe. Recall that genetic variation within Europe, especially Northern Europe, is rather low. It’s pretty clear that if you sample 100 SNPs from the human genome you can’t differentiate much. At 1,000 SNPs structure begins to appear, and this is starting to be well resolved by 10,000 SNPs. By 100,000 SNPs you are pretty much going to hit diminishing returns for regional diversity on Europe level scales. The pattern differs by method. PCA for example does much better with 10,000 SNPs in Europe than the model-based clustering (e.g., ADMIXTURE) in my experience, but the two are comparable as you near 100,000 SNPs. Beyond 100,000 SNPs there is not that much increase in resolution for genome-wide methods that rely on genotypes at this level of genetic diversity.

[![51qciM4cBhL.\_AC_UL320_SR242,320\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/09/51qciM4cBhL._AC_UL320_SR242320_.jpg?resize=242%2C320)![51qciM4cBhL.\_AC_UL320_SR242,320\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/09/51qciM4cBhL._AC_UL320_SR242320_.jpg?resize=242%2C320)](https://www.amazon.com/exec/obidos/ASIN/0691087504/geneexpressio-20)Of course, if you want really fine-scale differences, between villages for examples, more markers, and perhaps whole-genome sequencing that can pick up rare variants, are useful. In other words, there are cases one can imagine where more data than is normally available on SNP-chips ps useful. But these are definite boundary conditions. Once you get to the point of distinguishing branches of extended families you really can’t collapse the genealogies any further.

Another instance where more marker density, or the power of high coverage whole-genome sequencing, might be useful is for local ancestry deconvolution. If you’re assigning ancestry to windows of the genome then your marker density is going to be a limiting factor, as you might be slicing the 100,000 SNPs into 1,000 subunits.

Finally, there’s the issue of the models being tested. Novembre and Peter allude to the fact that many of these models posit stylized discrete pulse admixtures. As it turns out in some cases ancient DNA seems to have confirmed that something like this went on. That is, long periods of local stability and panmixia, followed by genetic turnover and admixture. But they note that there isn’t a good simulation framework where demographic scenarios are allowed to generate*in silico*data for testing new models. In other words, biologists are currently having to rely on “natural experiments.”

### Related Posts:

- [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/) - [Buddy, can you spare an
  SNP?](https://www.gnxp.com/WordPress/2007/09/20/buddy-can-you-spare-an-snp/) - [Finns as European
  outliers](https://www.gnxp.com/WordPress/2008/10/24/finns-as-european-outliers/) - [More markers, or more
  populations?](https://www.gnxp.com/WordPress/2010/07/02/more-markers-or-more-populations/) - [The genes in Spain fall rather
  evenly](https://www.gnxp.com/WordPress/2010/05/27/the-genes-in-spain-fall-rather-evenly/) - [GWAS, population structure and the Han
  Chinese](https://www.gnxp.com/WordPress/2009/11/25/gwas-population-structure-and-the-han-chinese/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F09%2F05%2Fliving-in-the-age-of-structure%2F&linkname=Living%20in%20the%20age%20of%20structure "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F09%2F05%2Fliving-in-the-age-of-structure%2F&linkname=Living%20in%20the%20age%20of%20structure "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F09%2F05%2Fliving-in-the-age-of-structure%2F&linkname=Living%20in%20the%20age%20of%20structure "Email")[](https://www.addtoany.com/share)
