+++
title = "Escape from the PBodies"
full_title = "Escape from the PBodies"
date = "2006-06-27"
upstream_url = "https://www.gnxp.com/WordPress/2006/06/27/escape-from-the-p-bodies/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/06/27/escape-from-the-p-bodies/).

Escape from the P-Bodies

When I’m writin’ I’m trapped in between the lines. I escape when I finish the rhyme. – Rakim

I wanted to go a little deeper into one of the [new papers](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=abstractplus&list_uids=16777601&query_hl=80&itool=pubmed_docsum) in the [recent Cell](https://www.gnxp.com/blog/2006/06/brightsurfing.php). It is a set of findings in [miRNA](http://js-kit.com/api/static/pop_comments?ref=http://gnxp.com&path=/114979659580594655?url=http://www.gnxp.com/blog/2006/06/rnai-fundamentals.php) research that can be tangentially related to synaptic plasticity. For the field of miRNA research, this paper is important because it is one of the clearest examples of miRNA derepression. To date, in most examples of miRNA-mediated gene silencing the target RNA is moved to P bodies where it is eventually degraded. Why make all this RNA just to destroy it? Not to mention that the system would be a lot more flexible if new RNAs didn’t have to be transcribed each time new proteins were needed. A couple papers have come out recently that show RNAs being released from repression in response to environmental signals, but this one proposes a distinct mechanism and examines the relationship with P bodies. Interestingly, the other two papers were both concerned with synaptic plasticity.

The authors were able to find modular elements in the 3′ UTR (untranslated region, a common area for regulatory elements in mRNA) of the cationic amino acid transporter 1 (CAT-1) mRNA that confer susceptibility to repression by a specific miRNA (miR-122) and derepression by interaction with a protein called HuR. These elements can act independently of the rest of the CAT-1 message. For instance, certain chunks of the 3′ UTR can be attached to a glowy protein stolen from fireflies (luciferase) to make it a target for miR-122. They do a decent job mimicking the behavior of CAT-1 in a human hepatoma cell line in ideal growth conditions.

Under less than ideal growth conditions, such as amino acid starvation, CAT-1 protein is upregulated within an hour while CAT-1 mRNA comes up in a range around 4 hours. So CAT-1 protein is regulated independently of the mRNA levels. The luciferase reporter constructs that only contain miR-122 binding sites are not affected in the starvation condition. Translation remains low. A further 1-kb of the CAT-1 3′ UTR is needed for a protein increase in response to cellular stress. This chunk of UTR contains three adenine (A) and uracil (U) rich regions. These, like other AU-rich elements (AREs), interact with an RNA-binding protein called HuR. People have generally thought that Hu binding to AREs stabilized the RNA by blocking exonucleolytic degradation. The authors carried out binding assays and also measured the effect of each of these AREs independent of the others. They found that the strength of HuR binding to a particular ARE predicts the amount of stress-inducibility of a reporter with that ARE.

Thus, the story so far is that miR-122 represses CAT-1 constitutively, but in response to certain environmental conditions, HuR becomes active and binds to elements downstream from the miRNA sites and derepresses CAT-1. Now, often when a miRNA binds its target they will both localize to cytoplasmic processing bodies (PBs). This is the case for CAT-1 and miR-122. Under normal conditions CAT-1 is found in PBs, but in response to starvation CAT-1 moves out of PBs and associates with polysomes (protein synthesis machinery). HuR binding to the AREs is necessary for this relocalization. The authors mention a preliminary finding that Ago2 (of Slicer fame) remains associated with CAT-1 when it relocalizes, but they show data that miR-122 abundance in PBs doesn’t change. I find it a little difficult to square these two, as one would expect that the interaction of Ago2 and CAT-1 RNA would be mediated by the miRNA, but perhaps miR-122 is just so abundant that some of it can wander off with CAT-1 without making a dent.

So HuR is little like a molecular Lee Marvin rescuing CAT from certain destruction (yes, I went there). The ability of HuR to do this even when the AREs are on a different RNA being repressed by a different miRNA makes the mechanism seem more general. HuR is part of a family of proteins called ELAV-like RNA-binding or Hu proteins, some of which (HuB, HuC, and HuD) are neuron-specific. Research into regulation of neuronal proteins by these Hu proteins goes back several years now and has focused on regulation of a protein called GAP-43.

GAP-43 is associated with axonal growth during development and is phosphorylated by protein kinase C (PKC) in response to LTP induction. It is not entirely clear what GAP-43 does at a mechanistic level yet, but the vague impression is that when GAP-43 is involved axons are growing and establishing new connections. I find it interesting that PKC may be regulating GAP-43 at more than one level. PKC can directly phosphorylate GAP-43 which leads to a number of changes in intracellular and membrane signaling. PKC activation also leads to [upregulation of all three neuron-specific Hu family proteins](http://www.pnas.org/cgi/content/full/102/34/12065). These proteins in turn stabilize GAP-43 mRNA and lead to GAP-43 protein increase.

Hu proteins and GAP-43 are [regulated in response to hippocampus-dependent learning tasks](http://www.pnas.org/cgi/content/full/101/5/1217) too.

Abstract:

> Increase of the RNA-binding protein HuD and posttranscriptional up-regulation of the GAP-43 gene during spatial memory >
> Alessia Pascale, Pavel A. Gusev, Marialaura Amadio, Tania Dottorini, Stefano Govoni, Daniel L. Alkon, and Alessandro Quattrone >
> Neuronal ELAV-like proteins (HuB, HuC, and HuD) are highly conserved > RNA-binding proteins able to selectively associate with the 3′ UTR of > a subset of target mRNAs and increase their cytoplasmic stability and > rate of translation. We previously demonstrated the involvement of > these proteins in learning, reporting that they undergo a sustained > up-regulation in the hippocampus of mice trained in a spatial > discrimination task. Here, we extend this finding, showing that a > similar up-regulation occurs in the hippocampus of rats trained in > another spatial learning paradigm, the Morris water maze. HuD, a > strictly neuron-specific ELAV-like protein, is shown to increase after > learning, with a preferential binding to the cytoskeletal fraction. > HuD up-regulation is associated with an enhancement of GAP-43 mRNA and > protein levels, with an apparently increased HuD colocalization with > the GAP-43 mRNA and an increased association of neuronal ELAV-like > proteins with the GAP-43 mRNA. These learning-dependent biochemical > events appear to be spatiotemporally controlled, because they do not > occur in another brain region involved in learning, the retrosplenial > cortex, and at the level of protein expression they show extinction 1 month after training despite memory retention. By contrast, HuD mRNA levels still remain increased after 1 month in the CA1 region. This persistence may have implications for long-term memory recall.

The Morris water maze is one of the most common behavioral tasks used to assess hippocampal function in rodents. It is assumed that the animals are learning to navigate in a two-dimensional space using distal cues to create a cognitive map. All of this should make synaptic changes in the hippocampus. Others have shown directly using zinc staining that over-training on the water maze task can actually lead to axonal growth into new layers of the hippocampus, which is a major structural change that may well contribute to permanent memory storage.

I guess what I’m saying is that GAP-43 is probably a good candidate for a memory-related molecule that is regulated by miRNAs. If I had the bioinformatics kn  
ow-how I would go look up the potential miRNA binding sites in GAP-43’s 3′ UTR. Then I would look to see if they had similar distance from the AREs in GAP-43’s 3′ UTR as they do in CAT-1. Then I would stretch and see if other proteins that have similar function to GAP-43 and share conserved sequences such as MARCKS and Neurogranin also have these sorts of motifs. These guys are also regulated in response to synaptic plasticity, and I think it would be really cool if they were regulated in coordination by Hu proteins derepressing the same miRNA. [MARCKS has a CU-rich element that is recognized by HuD and HuR](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=search&DB=pubmed). I’m really thinking that miRNAs, since they don’t have to be specific, could regulate protein modules, so GAP-43 and MARCKS could share the same miRNA.

Here are a couple other interesting things that I haven’t really integrated. Recently, a SNP in a human neuron-specific Hu protein (ELAVL4) was [associated with age-at-onset of Parkinson’s disease](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=search&DB=pubmed). Also, HuD and HuR appear to [regulate acetylcholinesterase](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=search&DB=pubmed) through an ARE in its 3′ UTR. [1 in 20 human genes have AU-rich elements](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=retrieve&db=pubmed&list_uids=8578590&dopt=Abstract), so just tying things together with AREs isn’t going to be enough, but if things had AREs and similar miRNA-binding sites I could see room for coordination.

### Related Posts:

- [Somatodendritic
  microRNAs](https://www.gnxp.com/WordPress/2007/07/24/somatodendritic-micrornas/) - [A mechanism for miRNA-mediated
  repression](https://www.gnxp.com/WordPress/2007/06/19/a-mechanism-for-mirna-mediated-repression/) - [Universal
  Granules](https://www.gnxp.com/WordPress/2006/12/25/universal-granules/) - [RNAi
  fundamentals](https://www.gnxp.com/WordPress/2006/06/08/rnai-fundamentals/) - [Slicer structure, piRNAs, and so much
  more](https://www.gnxp.com/WordPress/2007/01/16/slicer-structure-pirnas-and-so-much-more/) - [I'd be fired if that were my
  job](https://www.gnxp.com/WordPress/2006/11/24/i-d-be-fired-if-that-were-my-job/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F06%2F27%2Fescape-from-the-p-bodies%2F&linkname=Escape%20from%20the%20P-Bodies "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F06%2F27%2Fescape-from-the-p-bodies%2F&linkname=Escape%20from%20the%20P-Bodies "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F06%2F27%2Fescape-from-the-p-bodies%2F&linkname=Escape%20from%20the%20P-Bodies "Email")[](https://www.addtoany.com/share)
