+++
title = "Reflections on the"
full_title = "Reflections on the evolution at ASHG 2012"
date = "2012-11-11"
upstream_url = "https://www.gnxp.com/WordPress/2012/11/11/reflections-on-the-evolution-at-ashg-2012/"

+++
Source: [here](https://www.gnxp.com/WordPress/2012/11/11/reflections-on-the-evolution-at-ashg-2012/).

Reflections on the evolution at ASHG 2012

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/11/2012-11-09-12.171.jpg?resize=300%2C266)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/11/2012-11-09-12.171.jpg?resize=300%2C266)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2012/11/2012-11-09-12.171.jpg)As most readers know I was at ASHG 2012. I’m going to divide this post in half. First, the generalities of the meeting. And second, specific posters, etc.

**Generalities:**

– [Life Technologies/Ion Torrent](http://www.iontorrent.com/) apparently hires d-bag bros to represent them at conferences. The poster people were fine, but the guys manning the [Ion Torrent Bus](http://www.iontorrent.com/ionbus/) were total jackasses if they thought it would be funny/amusing/etc. Human resources acumen is not always a reflection of technological chops, but I sure don’t expect organizational competence if they (HR) thought it was smart to hire guys who thought (the d-bags)it would be amusing to alienate a selection of conference goers at ASHG. Go Affy & Illumina!

– Speaking of sequencing, there were some young companies trying to pitch technologies which will solve the problem of lack of long reads. I’m hopeful, but after the Pacific Biosciences fiasco of the late 2000s, I don’t think there’s a point in putting hopes on any given firm.

– I walked the poster hall, read the titles, and at least skimmed all 3,000+ posters’ abstracts. No surprise that genomics was all over the place. But perhaps a moderate surprise was how big exomes are getting for medically oriented people.

– Speaking of medical/clinical people, I noticed that in their presentations they used the word ‘[Caucasian](http://blogs.discovermagazine.com/gnxp/2011/01/stop-using-the-word-caucasian-to-mean-white/)‘ a lot. This was not evident in the pop-gen folks. It shows the influence of bureaucratic nomenclaturein modern medicine, as they have taken to using somewhat nonsensical US Census Bureau categories.

– Twitter was [a pretty big deal](http://www.genomesunzipped.org/2012/11/ten-guidelines-for-tweeting-at-conferences.php). There were so many interesting sessions that I found myself checking my feed constantly for the [\#ASHG2012](https://twitter.com/search?q=%23ashg2012) hashtag. It was also an easy way to figure out who else was at the same session (e.g., in my case, very often[Luke Jostins](https://twitter.com/lukejostins)).

– If you could track the patterns of movements of smartphones at the conference it would be interesting to see a network of clustering of individuals. For example, the evolutionary and population genomics posters were bounded by more straight-up informatics (e.g., software to clean your raw sequence data), from which there was bleed over. But right next to the evolution and population genomics sections (and I say genomics rather than genetics, because the latter has been totally subsumed by the former) you had some type of pediatric disease genetics aisles. I wasn’t the only one to have a freak out when I mistakenly kept on moving (i.e., you go from abstruse discussions of the population structure of Ethiopia, to concrete ones about the likely probability of death of a newborn with an autosomal dominant disorder, with photos of said newborn!).

– It was obvious which sessions were more multidisciplinary: just note the “churn” between speakers. People were switching sessions speaker-by-speaker, so if there was a stretch not to their liking, they would opt out.

– Number of questions per talk seemed to follow [a power law](https://en.wikipedia.org/wiki/Power_law). Many, many, talks had to have the moderator ask a token question. But there were a few panels where people rushed to the mics, and the moderators had to turn them away (this happened to me a couple of times, though I had the habit of sitting in the middle of aisles so that people wouldn’t have to edge past me, which disadvantaged me).

– [23andMe](http://23andme.com/) will supposedly have the new ancestry painting, with many more populations, up by the end of the year at the latest. I’ll believe it when I see it, but the person who was telling me this seemed totally sincere, and I’m hopeful.

– I drank a fair amount some of the nights, and have a lot of business cards from people I don’t remember. But one thing that seems to be emerging is a proliferation of intermediation and b2b services. With the diversity of choices it stands to reason that some firms are stepping into the clutter and attempting to make a profit by matching the two parties at the ends of the transaction. One person who I *do* recall Michael Heltzen of [BlueSEQ](http://blueseq.com/), which he pitches as the “Hotels.com of sequencing.”

– Overall this was a well run conference in terms of logistics. I’ll definitely be at Boston next year!

– Lots of stuff on archaic hominin admixture and selection. Lots.

– Friends don’t let friends use *structure* when they could use *admixture*. It seems that most people have switched to the latter, which is fast. But a few groups are still using the former. And they shouldn’t be, because their burn-in and replication parameters are set way low (I use *structure* for microsatellites) so that it won’t take a thousand years to converge. If you are doing this, why go for the power of Bayesian phylogenetics in the first place?

– Luke Jostins suggested that I looked different in real life from the head shot. I suspect that perhaps Luke has lower powers of perception in this domain. A very drunk member of a well respected lab decided to start yelling my name at a dim after party,\* so I can’t look that different (and it isn’t as if there aren’t a lot of brown dudes walking around at these things).

– [Konrad Karczewski](https://github.com/konradjk) gave me a “free the data” button which I wore, but there were mixed results when I asked if people were going to release their data sets. Some presenters offered to email me the data, but since I wasn’t flashing my badge I’m curious why they’d offer to even do this, as opposed to releasing it to millions of *other* strangers!

**Specifics:**

This section will mostly cover the talks and posters in the evolutionary and population genomics category. I can comment on the talks because I went to them, and on the posters because I looked at them multiple times. One thing to note is that many of the posters and some of the talks were on papers which are already in circulation (preprint or already published). I’m not going to touch on that much. I’ve reviewed/linked to most of those.

– One of the guys involved in fetal whole genome sequencing from last spring stated that the primary cost here is going to be in the sequencing. He’s also confident that they can move the sequencing much further back from 18 weeks (i.e., in terms of sample collection and analysis turnaround).

– There is a lot of talk about structural variants, etc., but for high-throughput sequencing methods we’re still not ‘there.’ I actually went to a CNV talk where the presenter presented some RFLP results! He stated that the reality was that for clinical purposes high-throughput isn’t feasible or accurate enough to distinguish 3 vs. 4 vs. 5 copies.

– I don’t get a lot of the CNV stuff which repeats SNP-results with CNVs. For example, the posters which recapitulated geographical fine-structure with CNV. This was OK for the first pub, but doing it over and over again seems gratuitous.

– Simon Gravel has some very awesome [software](http://www.stanford.edu/~gravel/software.html).

– [Luca Pagani](http://www.cell.com/AJHG/abstract/S0002-9297(12)00271-6) is confident about *[rolloff](http://genetics.med.harvard.edu/reich/Reich_Lab/Software.html)*‘s admixture estimate for Ethiopia. He’s moving to Ethiopian whole genomes now, and plans on doing follow ups on this question (his own methods are in line with *rolloff*).

– The rumored paper (i.e., I’ve heard about this paper for a few years) which connects Northeast African populations with the Khoisan of southern Africa will finally be published soon. At least that was what I was told…as I noted, this result has been around for a long time, but someone hasn’t been published. Basically the group has some Cushitic speaking samples from Kenya, and it looks like that these are the Ethiopian analogy to Andaman Islanders (or as close as you can get).

– We’ll see something on Afrikaner genomics soon enough. I wasn’t told explicitly, but it was pretty obvious.

– The [Nielsen Group](http://cteg.berkeley.edu/~nielsen/) is still working on high altitude adaptations. They don’t see hard sweeps. Of course I didn’t get confirmation of whether these were old variants, but it looks as if a lot of preliminary stuff did not have the power to detect anything in the first group. As usual they are up to something.

– Speaking of the Nielsen Group, [Melissa Wilson Sayres’s](http://cteg.berkeley.edu/members/sayres.html) work on purifying selection on Y chromosomal lineages was persuasive to me. Basically, effective population differences (e.g, polygyny) just can not explain the lower diversity of the Y lineages (they ran simulations). Luckily for the phylogeographers this won’t impact the utility of Y trees (positive selection would, but that’s not what she’s talking about). I’m a little confused whether it was Sayres’ talk or not, but these results may explain the discordance in coalescence between mtDNA and Y lineages (the former has a deeper coalescence).

– Also, Amy Goldberg from [Noah Rosenberg](http://www.stanford.edu/group/rosenberglab/people.html)‘s lab presented some theoretical work that showed that complex demographic history has an impact on the variance, as opposed to the mean, effective population size you might infer for a given sex. Someone from Michael Hammer’s lab started asking me if I liked their research while I was looking at Amy’s poster, and I said sure (I’d blogged it), but her theoretical results also explain some of the weird stuff I’d see out of their lab.

– Sriram Sankararaman had a poster on Neandertal admixture in modern human lineages. In the broad outlines the Reich lab and the Wall lab seem to agree (along with others, such as Melinda Yang in the Slatkin lab). We’re seeing the convergence of a new orthodoxy/paradigm. And they seem to agree broadly with [Graham Coop’s conjecture](http://blogs.discovermagazine.com/gnxp/tag/coops-conjecture/).

– There was a lot of stuff on East Asian genetics, but nothing too cutting edge. I was kind of disappointed. A massive Y and mtDNA study did suggest two waves of admixture in the Tibetan highland, which *a priori* seems plausible to me. But the rule-of-thumb I have is not to bet against the Nielsen Group, which remains skeptical. Another paper suggested deep lineages of haplogroup M among the Burmans. This is interesting because the Burmans are presumably culturally somewhat intrusive, supplanting the Mon populations.

– The guy from the Peopling of the British Isles presented. Two points. First, \~40 percent of the ancestry in England proper seems Anglo-Saxon. Second, their clustering method seemed to find many more ‘micro-populations’ along the “Celtic Fringe” and in Scotland. Why? My hunch is that the Anglo-Saxon expansion wasn’t a diffusion process. Rather, the hordes of Hengist and Horsa probably admixed with the local Brythonic Celtic population on the East Anglian shore, and the rapidly expanded. There is a high probability of some later assimilation (there is some suggestion that Alfred the Great’s line were Brythonic nobles who were absorbed into the Anglo-Saxon power structure), but the emergence of a huge Anglo-Saxon/England proper cluster was very evident in the figure displayed. The main opposition to this thesis I can think of is that isolation-by-distance gene flow is very efficacious in the topography of England, but less so in the more rugged borderlands.

– Speaking of isolation-by-distance, an Estonian geneticist claimed to me that the distinction between Estonians and Finns probably has to do with the arrival of the original Finnic populations from the east, and their subsequent separation. While the Estonians engaged in gene flow with the Latvians, they diverged from the Finns across the water, who were more isolated until the Swedes arrived.

– There was a poster (didn’t talk to the presenters) which did whole genome analysis of a South Indian man or two, and indicated that there is evidence that these individuals are basal to all other non-Africans. This is another attempt to reaffirm the possibility of an ancient “southern route” out of Africa. I wasn’t convinced because there wasn’t much detailing of their methods (they pointed to a diversity estimate, but that’s not enough these days).

– Another Indian group confirmed a lot of stuff that [Zack](http://www.harappadna.org/) has found already, but supplemented it with lots of low caste/tribal samples, which most people lacked. They assert (rightly) that *within* South Asia there are genetic distances across populations/castes which are analogous to inter-continental differences.

– I am excited by the synthesis of spatial and genetic variation data…but am beginning to realize that this has limitations, because we can’t transpose genetic variation representation onto tesseracts (because we can’t visualize tesseracts). In short, two or three dimensional representations remove important information at the finer-grain. And it’s at the finer-grain that we’re focusing now.

– Apparently Mexicans and Chileans overestimate their European ancestry. The presenters found that 40-45% of the ancestry of their Chilean sample was Amerindian. I asked about sampling, and they admitted this might be an issue. The same applied to their other results. We need thicker data sets here. Basically if it’s a heterogeneous country, you can’t have a pie-graph labelled with that country.

– There was a poster on associating OCA2-HERC2 in Brazilians with hair, eye, and skin color. The association of OCA2-HERC2 with skin color is unadmixed Europeans is mixed, but seems to show up in this population. Assuming stratification is not a problem (I believe they looked at that genomically), it seems that the effect on skin only shows up when you have a particular pigmentation genetic architecture. It’s a matter of statistics, not biology.

– Speaking of pigment, Mark Shriver had a poster which correlated perceived, apparent, and genomic racial ancestry. Perceived means how you’re perceived by others. Apparent is taking physical traits and averaging them quantitatively (facial features, skin color, etc.). And genomic ancestry is what you know about. Estimating ancestry quanta. The surprising thing is that people seemed to underestimate African ancestry from apparent physical features (looking at the scatter of apparent to genomic ancestry). This goes against folk wisdom, which asserts that “African features are dominant.”

– Lots of corrections of naive usages of Fst in the literature. A poster out of the Price lab suggested using likelihood ratios, and if not possible, Hudson’s Fst. This showed up multiple times in various forms. Fst will not die, but will be reborn!

– Saw a poster which claimed first cousin marriage decreases expected value of offspring by 3 cm! (this was not in the evolution and pop genomics sections, and I probably should have spent more time looking over complex traits, etc., but there’s only so much you can do)

– More evidence of multiple migrations into New World. Lots of New World genomics. I didn’t talk to these presenters because they were always busy.

– Spencer Wells told me that they’d finally be publishing their paper using their Geno 2.0 results soon. They had really good population coverage, though I wish they’d had the bar plot rotated 90 degrees. I couldn’t read labels too well.

Finally, **there was A LOT of software, and A LOT of methods.** This is one of the things where I assume over the next decade it will shake out into a few big players. Right now labs are pumping out software to infer ancestry, phase data, etc., and playing up their advantages. This is all good, but at some point the focus will go back to biology, and the software will be the wind beneath its wings. I’m trying to free up time to play with some of the software, though much of it isn’t online yet (the presenters always assured it would be up soon, but I know how that goes.).

\* This was *not* a pleasant experience.

### Related Posts:

- [ASHG 2012
- which
  sessions](https://www.gnxp.com/WordPress/2012/10/02/ashg-2012-which-sessions/) - [Mutants documentary
  torrent](https://www.gnxp.com/WordPress/2006/10/16/mutants-documentary-torrent/) - [Joseph Smith prophesied ASHG
  2012](https://www.gnxp.com/WordPress/2012/09/06/joseph-smith-prophesied-asgh-2012/) - [Open Thread,
  11-08-2012](https://www.gnxp.com/WordPress/2012/11/08/open-thread-11-08-2012/) - [The Family that Walks on All
  Fours](https://www.gnxp.com/WordPress/2006/03/18/the-family-that-walks-on-all-fours/) - [ASHG 2014, Razib's fun times
  (hopefully)](https://www.gnxp.com/WordPress/2014/09/21/ashg-2014-razibs-fun-times-hopefully/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F11%2F11%2Freflections-on-the-evolution-at-ashg-2012%2F&linkname=Reflections%20on%20the%20evolution%20at%20ASHG%202012 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F11%2F11%2Freflections-on-the-evolution-at-ashg-2012%2F&linkname=Reflections%20on%20the%20evolution%20at%20ASHG%202012 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F11%2F11%2Freflections-on-the-evolution-at-ashg-2012%2F&linkname=Reflections%20on%20the%20evolution%20at%20ASHG%202012 "Email")[](https://www.addtoany.com/share)
