+++
title = "Human ancestry is"
full_title = "Human ancestry is complex with ancient DNA"
date = "2016-03-28"
upstream_url = "https://www.gnxp.com/WordPress/2016/03/28/human-ancestry-is-complex-with-ancient-dna/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/03/28/human-ancestry-is-complex-with-ancient-dna/).

Human ancestry is complex with ancient DNA

[![1-s2.0-S0960982216302470-gr2](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/1-s2.0-S0960982216302470-gr2.jpg?resize=620%2C561)![1-s2.0-S0960982216302470-gr2](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/1-s2.0-S0960982216302470-gr2.jpg?resize=620%2C561)](http://www.cell.com/current-biology/abstract/S0960-9822(16)30247-0)

[The Combined Landscape of Denisovan and Neanderthal Ancestry in Present-Day Humans](http://www.cell.com/current-biology/abstract/S0960-9822(16)30247-0):

> Some present-day humans derive up to ∼5% …oftheir ancestry from > archaic Denisovans, an evenlarger proportion than the ∼2% from > Neanderthals…We developed methods that can disambiguate the locations > of segments of Denisovan and Neanderthal ancestry in present-day > humans and applied them to 257 high-coverage genomes from 120 diverse > populations, among which were 20 individual Oceanians with high > Denisovan ancestry…In Oceanians, the average size of Denisovan > fragments is larger than Neanderthal fragments, **implying a more > recent average date of Denisovan admixture in the history of these > populations** (p= 0.00004). **We document more Denisovan ancestry in > South Asia than is expected based on existing models ofhistory, > reflecting a previously undocumented mixture related to archaic > humans** (p= 0.0013). Denisovan ancestry, just like Neanderthal > ancestry, has been deleterious on a modern human genetic background, > as reflected by its depletion near genes. Finally, the reduction of > both archaic ancestries is especially pronounced on chromosome X and > near genes more highly expressed in testes than other tissues (p= > 1.2× 10−7to 3.2× 10−7for Denisovan and 2.2× 10−3to 2.9× > 10−3for Neanderthal ancestry even after controlling for differences in level of selective constraint across gene classes). **This suggests that reduced male fertility may be a general feature of mixtures of human populations diverged by \>500,000 years.**

Take a [look at the supplements](http://www.cell.com/cms/attachment/2051794959/2059460386/mmc1.pdf) for the functional stuff. I am not going to address that. Much of those results have been circulation or in other papers over the years. Rather, I want to highlight the variation in patterns of Denisovan admixture in non-Oceanian groups. Here is an important section:

> Taken together, the evidence of Denisovan admixture in modern humans > could in theory be explained by a single Denisovan introgression into > modern humans, followed by dilution to different extents in Oceanians, > South Asians, and East Asians by people with less Denisovan ancestry. If dilution does not explain these patterns, however, a minimum of three distinct Denisovan introgressions into the ancestors of modern humans must have occurred.

You see it on the figure above. The South Asian groups consistently jump well above the trend line for inferred Denisovan as a function of shared ancestry with ~~Australians~~ non-West Eurasian ancestry. Also, if you look at the admixture patterns for **Denisovan ancestry in South Asia you see they follow the ANI-ASI cline**. That is, it seems to come into the South Asian populations through the “Ancestral South Indians.” Interestingly, the Onge sample of Andaman Islanders has less Denisovan than low caste South Asian groups, reminding us that though the Onge and their kin are the closest modern populations to the ASI, they are not descended from the ASI. The highest fraction of inferred Denisovan is in the Sherpa people of Nepal.

[![sherpa](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/sherpa-300x233.png?resize=300%2C233)![sherpa](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/03/sherpa-300x233.png?resize=300%2C233)](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643256/)The figure to the right is from [Admixture facilitates genetic adaptations to high altitude in Tibet](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643256/), and the authors find that **the Sherpa are at one extreme in an ancestry cline in comparison to other East Asians.** The figure is hard to make out, so I will tell you that many of the Sherpa are fixed for the red component, while other Tibetans are in positions in the middle, and most East Asians have low fractions of the red, with the Dai having none. The Gujarati sample form the HapMap have low fractions of both East Asian components. This is almost certainly an artifact of the shared ancestry of all eastern Eurasians (and perhaps Oceanians), of which the ASI were one descendant group. The proportion of Denisovan in low caste South Asians indicates that the fraction in ASI was about at the same level as the Sherpa. I suspect that ASI and the Tibetan groups got their Denisovan via different paths, but it doesn’t seem like we know yet.

Overall I do marvel at what ancient DNA can tell us. Without it we wouldn’t be talking about any of these admixture events; they’d be signals too weak to have left an obvious mark in the genome.

### Related Posts:

- [We are all Denisovans
  now!](https://www.gnxp.com/WordPress/2020/04/22/we-are-all-denisovans-now/) - [More and more evidence for 'complex demography' in
  archaic…](https://www.gnxp.com/WordPress/2021/07/19/more-and-more-evidence-for-complex-demography-in-archaic-ancestry/) - [Was H. luzonensis
  Denisovan?](https://www.gnxp.com/WordPress/2021/08/14/was-h-luzonensis-denisovan/) - [Biogeography, Denisovans, and
  Siberia](https://www.gnxp.com/WordPress/2020/06/05/biogeography-denisovans-and-siberia/) - [Complex history of archaic
  ancestry](https://www.gnxp.com/WordPress/2021/07/19/complex-history-of-archaic-ancestry/) - [High-coverage Neanderthal genome and gene flow
  with…](https://www.gnxp.com/WordPress/2020/03/16/high-coverage-neanderthal-genome-and-gene-flow-with-denisovans/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F03%2F28%2Fhuman-ancestry-is-complex-with-ancient-dna%2F&linkname=Human%20ancestry%20is%20complex%20with%20ancient%20DNA "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F03%2F28%2Fhuman-ancestry-is-complex-with-ancient-dna%2F&linkname=Human%20ancestry%20is%20complex%20with%20ancient%20DNA "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F03%2F28%2Fhuman-ancestry-is-complex-with-ancient-dna%2F&linkname=Human%20ancestry%20is%20complex%20with%20ancient%20DNA "Email")[](https://www.addtoany.com/share)
