+++
title = "Evolution has a simple"
full_title = "Evolution has a simple genetic skeleton"
date = "2013-09-17"
upstream_url = "https://www.gnxp.com/WordPress/2013/09/17/evolution-has-a-simple-genetic-skeleton/"

+++
Source: [here](https://www.gnxp.com/WordPress/2013/09/17/evolution-has-a-simple-genetic-skeleton/).

Evolution has a simple genetic skeleton

![**Citation:** Keinan, Alon, and Andrew G. Clark. “Recent explosive human population growth has resulted in an excess of rare genetic variants.” science 336.6082 (2012): 740-743.  
APA](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2013/09/spectra.png?resize=515%2C419 "spectra")

Some of the topics that I discuss in this space may seem abstruse, but really they’re often elaborations upon rather elementary basic models of the world. **When it comes to a subject like evolutionary genetics deep thinking extending from a few simple conceptual anchors yields great insight**. Those anchors trace back to the foundations of [Mendelian genetics](https://en.wikipedia.org/wiki/Mendelian_inheritance). For diploid organisms [the law of equal segregation](https://en.wikipedia.org/wiki/Mendelian_inheritance#Law_of_Segregation_.28The_.22First_Law.22.29) states that of the two gene copies organisms have there is an equal probability of contribution of either to their offspring. This explains the simple power of [Punnett squares](https://en.wikipedia.org/wiki/Punnett_square) and the inheritance patterns of recessive traits. The law of [independent assortment](https://en.wikipedia.org/wiki/Mendelian_inheritance#Law_of_Independent_Assortment_.28The_.22Second_Law.22.29) states that genes (and therefore implicitly Mendelian traits) are passed independently from each other from parents to offspring. These abstractions are concretized on the cytological and molecular genetic scale during [meiosis](https://en.wikipedia.org/wiki/Meosis), as [homlogous chromosomes](https://en.wikipedia.org/wiki/Homologs) which are composed of packed sequences of genes partition themselves into separate haploid gametes (sperm and egg\*). Early in meiosis, during [prophase 1](https://en.wikipedia.org/wiki/Prophase), [crossing-over](https://en.wikipedia.org/wiki/Chromosomal_crossover) between homologs results in [genetic recombination](https://en.wikipedia.org/wiki/Genetic_recombination), which preserves the law of independent assortment even when genes are on the same chromosome by breaking apart associations between specific physical genetic regions which might exhibit co-inherited distinctiveness (if the genes are very close [they are linked](https://en.wikipedia.org/wiki/Genetic_linkage)).

These patterns of classical Mendelian genetics served as the basis for analysis and abstraction which preceded the molecular understanding of these patterns by decades. To move to the stage where I often put the spotlight in this blog, population genetics, one must imagine repeated meioses across sets of individuals. In some ways Mendelian dynamics can be recapitulated at the population genetic scale. If two parents are heterozygote carriers of a gene for a recessive trait there is an expectation that 1/4 of their offspring will exhibit the trait in question. But if the parents have only three offspring, obviously the expectation will not be met. Across the population there will be deviation from the expectation because of sampling noise within the meiotic processes and subsequent fertilization, but by aggregating across the population you can replicate the 3:1 ratio at the heart of recessive traits when heterozygotes are paired. This also falls out of the [Hardy-Weinberg Equilibrium](https://en.wikipedia.org/wiki/Hardy%E2%80%93Weinberg_principle), *p*² + 2*pq* + *q*² = 1, where *p* = 0.5 and *q* = 1 – *p*, and *q*² defines the fraction with recessive expression.

Population genetics can be thought of as an attempt to understand the flux and flow of the fractions of *p* and *q*. More traditionally it is the study of the change in [allele](https://en.wikipedia.org/wiki/Allele) frequencies modified by parameters such as mutation, migration, selection, and drift. An allele is a genetic variant, which arises through mutation. Today many people think of a mutation as a change in a DNA base, such as A → G. This is a [single-nucleotide polymoprhism](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism). But the idea of a mutation goes back decades before the understanding that DNA was the physical basis for genetic inheritance. It was [initially](https://en.wikipedia.org/wiki/Hugo_De_Vries#Mutation_theory)observed via radical phenotypic transformations, albeit assumed to be coded by discrete genetic elements which were variable. Physically, mutations can be due to genetic duplication, or deletions of chromosomal regions, not just single base changes. One could argue that rearrangements of [karyotype](https://en.wikipedia.org/wiki/Karyotype), the number and nature of chromosomes, are mutations as well. All of these have physical explanations. SNPs can emerge via faulty DNA repair. Duplication and deletion can occur during crossing-over and such. Meiosis is a complex and fraught process, and molecular geneticists have documented many ways by which it can produce copies of the genome with errors (often resulting in pathology). **But for the purposes of population genetics what you need to remember is that *mutations are changes, variation injected into the gene pool*.** Migration, like mutation, often introduces variation as well (though one can imagine situations where variation is reduced, such as the case of a population which is swamped out by an [isogenic lineage](http://www.merriam-webster.com/dictionary/isogenic)).

Selection and drift tend to work in the other direction. They usually remove variation. Positive selection can sweep one allele to fixation, from zero to 100 percent. As a side effect due to genetic linkage flanking regions of the favored allele are also swept to fixation, resulting in a long homogeneous region of the genome identical to the copy carried by the individual who carried the first mutant. Negative selection favors homogeneity by putting a tamp down on the variation which mutation introduces, purifying the gene pool. Random genetic drift may seem an odd candidate for a parameter removing variation, but recall that allele frequencies which reaches zero or 100 percent result in loci which are not polymorphic and will remain in that state barring new variation such as mutation or migration. Over time without new variation all polymorphic sites will become monomorphic as one variant comes to dominate each locus. Drift is just a consequence of the sampling variance which you can see within meiosis itself. The law of equal segregation implies you have an equal probability of contributing either of your copies of a gene to your offspring, but with finite offspring you are likely to be somewhat biased toward one or the other allele by chance. Working over a whole population this sampling eventually results in consistent inequalities of transmission generation to generation. This is information lost through imperfect fidelity of replications of the gene pool of one generation to the next. But whereas this imperfection on a finer genetic scale introduces variation (mutation), pooling across the population you are losing variation as only a subset of individuals defining particular allele frequencies in a given generation give rise to the whole of the next generation.

Working up the ladder of abstraction I want to introduce now the concept of a [site frequency spectrum](http://www.stats.ox.ac.uk/~etheridg/orsay/notes3.pdf). It is truly a simple idea, and is illustrated in the figure at the top of this post. If you imagine a population where individuals are reproducing, there are some common mutations, derived alleles different from the ancestral state, which are nearly fixed in the population. Others are at middling frequency, perhaps being maintained by some sort of [balancing selecting](https://en.wikipedia.org/wiki/Balancing_selection). But an enormous number of mutations will be singletons, which help define an individual’s distinct mutational load due to*de novo*variants introduced during the process of meiosis. Many of these will immediately be lost due to chance, or perhaps removed through negative selection. What you see above is that the spectrum of derived mutations, by taking the frequency with specific counts (i.e., the first bar indicates the frequency of derived mutants which can be classed as singletons, the last the class where there are 15 copies segregating within the population). **Demography and selection will perturb this distribution**. Positive selection will probably skew it more toward the right, as there are more derived variants with many copies. Negative selection should skew it to the left, as mutants deviating from the ancestral state are selected against.

Genetics can seem complex, especially theoretical evolutionary models. But the reality is that these models are ultimately grounded in a finite number of relatively simple elements. The difficulty comes from the fact that these interlocking foundational units are numerous, and can be difficult to keep track of. One doesn’t have to know that crossing-over during genetic recombination occurs during [synapsis](https://en.wikipedia.org/wiki/Synapsis), but at the end of the day the different dimensions of genetics are complementary, as they cross-link and construct a seamless whole.

\* I am aware that only one of the four may develop into a gamete for the egg.

### Related Posts:

- [Genetics
  reference](https://www.gnxp.com/WordPress/2006/09/28/genetics-reference/) - [Doing evolution's
  sums](https://www.gnxp.com/WordPress/2010/05/27/doing-evolutions-sums/) - ["Dominant" &
  "recessive"](https://www.gnxp.com/WordPress/2006/07/15/dominant-recessive/) - [Will an infection become an epidemic? A simple
  model](https://www.gnxp.com/WordPress/2007/03/28/will-an-infection-become-an-epidemic-a-simple-model/) - [Dissolving the dominance of
  dominance](https://www.gnxp.com/WordPress/2005/04/04/dissolving-the-dominance-of-dominance/) - [The once and future genomics of sexual
  selection](https://www.gnxp.com/WordPress/2015/03/25/the-once-and-future-genomics-of-sexual-selection/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F09%2F17%2Fevolution-has-a-simple-genetic-skeleton%2F&linkname=Evolution%20has%20a%20simple%20genetic%20skeleton "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F09%2F17%2Fevolution-has-a-simple-genetic-skeleton%2F&linkname=Evolution%20has%20a%20simple%20genetic%20skeleton "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F09%2F17%2Fevolution-has-a-simple-genetic-skeleton%2F&linkname=Evolution%20has%20a%20simple%20genetic%20skeleton "Email")[](https://www.addtoany.com/share)
