+++
title = "The ancient scramble for"
full_title = "The ancient scramble for Eurasia"
date = "2016-05-19"
upstream_url = "https://www.gnxp.com/WordPress/2016/05/19/the-ancient-scramble-for-eurasia/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/05/19/the-ancient-scramble-for-eurasia/).

The ancient scramble for Eurasia

[![TestSEAsia.gz..1.out](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/05/TestSEAsia.gz..1.out_-e1463632287739.png?resize=620%2C465)![TestSEAsia.gz..1.out](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/05/TestSEAsia.gz..1.out_-e1463632287739.png?resize=620%2C465)](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/05/TestSEAsia.gz..1.out_.png)

When I wrote the [Pleistocene was humanity’s Hyborian age](http://www.unz.com/gnxp/the-pleistocene-as-humanitys-hyborian-age/), I meant **humanity.** For contingent reasons the new genetic sciences of ancient DNA have elucidated the history of northwest Eurasia first. But prior to the [Great Divergence](https://en.wikipedia.org/wiki/Great_Divergence) Europe was not quite so exceptional. In fact the historian Victor Lieberman wrote [Strange Parallels](https://www.amazon.com/exec/obidos/ASIN/0521804965/geneexpressio-20), his macrohistory of Eurasia, to highlight just how similar the trajectories of Western Europe and mainland Southeast Asia were up until the early modern era, when the West distanced itself from the rest. In short, **European prehistory updates our priors for the prehistory of us all.**

For various reasons having to do with professional responsibilities I look at [TreeMix](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002967) plots quite often. Like [PCA](https://en.wikipedia.org/wiki/Principal_component_analysis) TreeMix is great for exploratory data analysis. You throw a bunch of populations in there, and it searches a bunch of parameters which can fit the model. But often the results are weird.

They’re not weird because they’re “wrong.” They’re weird because we’ll forcing data to give us answers, and the model pops out something which is reasonable with the conditions imposed on it. And often we just don’t have the big picture. Statistical inference was indicating strange connections between Native Americans and Europeans for the past decade…but it took [ancient DNA from Siberia](http://www.bbc.com/news/science-environment-25020958) to resolve the mystery. Europeans and Amerindians exhibit ancestry from a shared common population. In Europe this ancestry is relatively recent, on the order of the past \~4 to 5 thousand years. Statistical genetic inference can tell us our model is missing something, but it can not always specify clearly just exactly what we’re missing.

The image above from a TreeMix plot is hard to make out; [click it](http://www.unzcloud.com/wp-content/uploads/2016/05/TestSEAsia.gz..1.out_.png). But what it will show you are two things which are strange:

1\) Gene flow from betweenthe East African (mostly HapMap Masai for what it’s worth) node and Mbuti (HGDP) to the Papuans (HGDP).

2\) Gene flow from near the East African node to the point which defines the whole East Eurasian, Amerindian, and Oceanian, nodes.

I would laugh this off, but I see it all the time in TreeMix. I know I’m not the only one. I have no explanation for it. It’s obviously not recent admixture. Rather, there are affinities between populations which we just don’t have a good model for. Knowing what we know about ancient Europe it is mostly likely that these gene flow edges which seem inexplicable reflect prehistoric events which make sense only in the context of population patterns which have been totally obscured over the last 10,000 years. Ancient DNA from China will probably shed a great deal of light on these topics. I predict that the Chinese will exhibit the same discontinuity with their Paleolithic ancestors that modern Europeans do, and the affinities between East Eurasians and some Africans in these TreeMix plots probably is a shadow of a “ghost population” which has been absorbed in Eurasia, and may have contributed to some of the ancestry of a group which migrated back to Africa.

**Notes**I set TreeMix to check for covariance across blocks of 1000 SNPs. I had 215,000 total markers in the data set (very high quality ones). I rooted it with Mbuti,set 5 migration edges, and ran it 10 times. They all looked the same. Most of the populations are pooled from public sources.

### Related Posts:

- [Different Neandertal
  strokes](https://www.gnxp.com/WordPress/2011/03/21/different-neandertal-strokes/) - [The modern human back-sweep from Southeast
  Asia](https://www.gnxp.com/WordPress/2019/12/21/the-modern-human-back-sweep-from-southeast-asia/) - [The last glacial maximum bottlenecks and human
  phylogeny](https://www.gnxp.com/WordPress/2022/06/19/the-last-glacial-maximum-bottlenecks-and-human-phylogeny/) - [The pruned
  tree](https://www.gnxp.com/WordPress/2012/09/03/the-pruned-tree/) - [The brambly bush of
  humanity](https://www.gnxp.com/WordPress/2012/09/18/the-brambly-bush-of-humanity/) - [Beyond trees and European
  trees](https://www.gnxp.com/WordPress/2012/03/07/beyond-trees-and-european-trees/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F05%2F19%2Fthe-ancient-scramble-for-eurasia%2F&linkname=The%20ancient%20scramble%20for%20Eurasia "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F05%2F19%2Fthe-ancient-scramble-for-eurasia%2F&linkname=The%20ancient%20scramble%20for%20Eurasia "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F05%2F19%2Fthe-ancient-scramble-for-eurasia%2F&linkname=The%20ancient%20scramble%20for%20Eurasia "Email")[](https://www.addtoany.com/share)
