+++
title = "Disease global &"
full_title = "Disease global & historical context"
date = "2013-05-29"
upstream_url = "https://www.gnxp.com/WordPress/2013/05/29/disease-in-a-global-and-historical-context/"

+++
Source: [here](https://www.gnxp.com/WordPress/2013/05/29/disease-in-a-global-and-historical-context/).

Disease: global & historical context

![**Citation:** Corona, Erik, et al. “Analysis of the Genetic Basis of Disease in the Context of Worldwide Human Relationships and Migration.” PLoS Genetics 9.5 (2013): e1003447.](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2013/05/journal.pgen_.1003447.g002.png?resize=495%2C497 "journal.pgen.1003447.g002")

The above figure is from a paper in *PLoS GENETICS*, [Analysis of the Genetic Basis of Disease in the Context of Worldwide Human Relationships and Migration](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003447). The authors synthesize two diverse domains of human genomics. First, there are biomedically focused [genome-wide association studies](https://en.wikipedia.org/wiki/Genome-wide_association_study) and their like which attempt to identify risk [alleles](https://en.wikipedia.org/wiki/Allele) for particular diseases. In some cases these risk alleles are very[penetrant](https://en.wikipedia.org/wiki/Penetrance), in that a particular state predicts with high likelihood a disease phenotype. But in most cases the yield is elevated or decreased risks for highly complex traits such as type 2 diabetes. Second, there is the domain of evolutionary genomics which attempts to reconstruct a phylogenetic and population genetic history so as to frame contemporary patterns of variation in their proper context. How this might be important or of interest is obvious in the case of [malaria resistance genes](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003509). Alleles conferring resistance have arisen in multiple populations due to parallel environmental pressures. Phylogenetic relationships between these populations should inform your predictions as to the likely similarities of the mutations between the populations. Meanwhile, population genetic theory can give you clues as to the likelihood of [multiple adaptations](http://www.ncbi.nlm.nih.gov/pubmed/20660645).

**The goal here is to increase understanding of the nature of the emergence of disease, and perhaps target individual risk more effectively**. Above in the figure you see two interesting patterns: risk for type 2 diabetes alleles as a function of descent, and risk as a function of*de novo*mutation or independent selective event. The phylogenetic tree represents real relationships as inferred from the \>600,0000 SNPs in the HGDP [data set](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project). The risk alleles were culled from the literature, and were computed for individuals and populations. The real population risks were then compared to a model of risks which might occur in a scenario with this particular phylogenetic tree and the normal process of random genetic drift (see [methods](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003447#s4) for the gory details!). What you see are phylogenetic relationships (African populations shifted toward higher risk) and independent events (Pima Indians shifted toward higher risk) where there is a higher risk toward diabetes (red shifted).

There are all sorts of shortcomings to this analysis. The authors are limited by the risk alleles in their study, which is certainly far less than thorough or exhaustive. Additionally, their population coverage was thin in some regions, resulting in reduced ability to even squeeze power from their model in particular cases. But one thing that jumps out at you is that the patterns here inferred from risk alleles in a highly polygenic disease like type 2 diabetes don’t even track what you see in the real world. Many South Asian groups have*very*high risks of type 2 diabetes. It just so happens that these groups are not in the HGDP sample. There is actually a rather informative critique from two epidemiologists [in the comments of this paper](http://www.plosgenetics.org/annotation/listThread.action?root=66481). They make many points that came to mind in the specifics. But they ended in a fashion which raised my eyebrows:

> Finally, **the need to avoid stigmatizing populations based on genetic > risk has been much discussed.**It is not difficult to imagine a media > announcement based on this publication – “Genetic risk of diabetes > found in African populations”. Similar claims were made for > intelligence not very long ago. Not all speculation is neutral.

As it happens I come from a population with very high risk for metabolic disease. I have no idea if I’m stigmatized by this fact, but I am very glad that medical professionals are becoming aware of differential risks, and moving beyond coarse one-size-fits-all understandings of human health. The BMI values developed for European Americans are probably rather inappropriate to South Asians because of the way we distribute fat (in short, we need to be thinner to exhibit the same risk profile all things equal). Again , I have no idea if this is stigmatizing, **but it is real.**

So despite all the real concerns I have with the methodology in the paper above, **I believe that these sorts of analyses are essential parts of the broader answer.** We now live in the age of the antiobiotic revolution and an understanding of germ theory. Those were the big returns on investment for public health. For the short term gains in human well being and life expectancy are going to be on the margin, through increments. Despite all the skepticism I have with initial attempts to work out the relationship between population history and disease, one must begin somewhere.

**Citation:** Corona, Erik, et al. “Analysis of the Genetic Basis of Disease in the Context of Worldwide Human Relationships and Migration.” PLoS Genetics 9.5 (2013): e1003447

### Related Posts:

- [How predictive are known genetic factors for disease
  risk?](https://www.gnxp.com/WordPress/2008/11/19/how-predictive-are-known-genetic-factors-for-disease-risk/) - [Finding rare variants involved in
  disease](https://www.gnxp.com/WordPress/2009/03/05/finding-rare-variants-involved-in-disease/) - [Genome-wide association study for breast
  cancer…](https://www.gnxp.com/WordPress/2007/06/28/genome-wide-association-study-for-breast-cancer-susceptibility/) - [Common disease, common
  variant](https://www.gnxp.com/WordPress/2006/12/01/common-disease-common-variant/) - [A note on the Common Disease-Common Variant
  debate](https://www.gnxp.com/WordPress/2007/02/27/a-note-on-the-common-disease-common-variant-debate/) - [Common disease, single
  gene?](https://www.gnxp.com/WordPress/2007/08/16/common-disease-single-gene/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F05%2F29%2Fdisease-in-a-global-and-historical-context%2F&linkname=Disease%3A%20global%20%26%20historical%20context "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F05%2F29%2Fdisease-in-a-global-and-historical-context%2F&linkname=Disease%3A%20global%20%26%20historical%20context "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2013%2F05%2F29%2Fdisease-in-a-global-and-historical-context%2F&linkname=Disease%3A%20global%20%26%20historical%20context "Email")[](https://www.addtoany.com/share)
