+++
title = "Painting the human tree"
full_title = "Painting the human tree of life"
date = "2010-12-10"
upstream_url = "https://www.gnxp.com/WordPress/2010/12/10/painting-the-human-tree-of-life/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/12/10/painting-the-human-tree-of-life/).

Painting the human tree of life

![tisk](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/tisk.png?resize=311%2C240)![tisk](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/tisk.png?resize=311%2C240)  
Tishkoff et al.

Reading [Peter Bellwood’s](https://www.amazon.com/exec/obidos/ASIN/0631205667/geneexpressio-20/) [First Farmers: The Origins of Agricultural Societies](https://www.amazon.com/exec/obidos/ASIN/0631205667/geneexpressio-20/), I’m struck by how much of a difference five years has made. When Bellwood was writing the ‘orthodoxy’ of the nature of the expansion of farming into Europe leaned toward cultural diffusion. Today the paradigm is in [flux](http://blogs.discovermagazine.com/gnxp/2010/11/european-man-of-many-faces-cain-vs-abel/), as a new generation of genomic studies using ancient DNA, wider sets of markers, and a broader sampling of populations, makes untenable solid old truths. I’m reading Bellwood’s work in part because from what I have read elsewhere it seems as if his model seems less and less ridiculous in light of the new information bubbling out of human genomics. The swell of data in this field is such that it’s hard to keep up. You never know what you’re going to [wake up to](http://blogs.discovermagazine.com/gnxp/2010/05/the-three-layers-of-the-neandertal-cake/) in the morning.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)The assertions of archaeologists and pre-historians such as Bellwood have clear implications and offer up specific predictions about the shape of the tree of human phylogenetics. Now the results are getting robust enough that the models can be tested, and alternatives refuted or accepted. But sometimes you need to take stock. Many of my posts make the assumption that you have a lot of the background information in hand, but I know that’s not always possible. With that, I’d like to bring your attention to a paper in *Human Molecular Genetics*, [Fine-scale population structure and the era of next-generation sequencing](http://www.ncbi.nlm.nih.gov/pubmed/20876616):

> Fine-scale population structure characterizes most continents and is > especially pronounced in non-cosmopolitan populations. Roughly half of > the world’s population remains non-cosmopolitan and even populations > within cities often assort along ethnic and linguistic categories. > Barriers to random mating can be ecologically extreme, such as the > Sahara Desert, or cultural, such as the Indian caste system. In either > case, subpopulations accumulate genetic differences if the barrier is > maintained over multiple generations. Genome-wide polymorphism data, > initially with only a few hundred autosomal microsatellites, have > clearly established differences in allele frequency not only among > continental regions, but also within continents and within countries. > We review recent evidence from the analysis of genome-wide > polymorphism data for genetic boundaries delineating human population > structure and the main demographic and genomic processes shaping > variation, and discuss the implications of population structure for the distribution and discovery of disease-causing genetic variants, in the light of the imminent availability of sequencing data for a multitude of diverse human genomes.

![cousins](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/cousins.png?resize=300%2C179)![cousins](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/cousins.png?resize=300%2C179)  
E(x) = 4th cousins

The paper reviews all the different ways in which human populations are related, the evolutionary forces which they’re shaped by, and, nested layers of population structure which we’re now just starting to explore. A few months ago [I blogged](http://blogs.discovermagazine.com/gnxp/2010/07/really-fine-grained-genetic-maps-of-europe/) that geneticists have found that they could differentiate population clusters on the scale of **nearby villages** in Europe! This is incredible, as recently as 15 years ago scientists would have struggled for a dozen markers to differentiate populations separated by continents. In the paper on [Sami genomics](http://blogs.discovermagazine.com/gnxp/2010/12/the-men-of-the-north-the-sami/) which I covered earlier in the week I didn’t even bother to mention that the Sami apparently exhibit internal population structure. Not too surprising given the fragmented nature of low density marginal ecologies, but nevertheless a reality check on the fact that we tend to perceive such people as a homogeneous whole. The [Bushmen](http://scienceblogs.com/notrocketscience/2010/02/sequencing_the_bishop_-_scientists_present_full_genomes_of_a.php) of South Africa are among the most diverse people in the whole world. SNP-chips fine-tuned with European genetic variation are likely missing many variants peculiar to Bushmen, so our estimates are surely low bounds.

![colored](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/colored.png?resize=300%2C270)![colored](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/colored.png?resize=300%2C270)In the paper above they suggest that a next step in the exploration of human genomic variation will be on a finer scale more broadly. Baden-Baden vs. Baden-Württemberg. Patterns of similarities across chromosomal segments which indicate near-familial relations because of clear identical-by-descent DNA. For example, they note that the average Ashkenazi Jew has a genetic distance from another random Ashkenazi Jew on the order of 4th cousins. Most human variation is found within populations, but there are still thousands of markers which exhibit a great deal of inter-population variance, and serve as a distinctive record of the evolutionary history of a given group. They also suggest that generally the focus as been on broad-scale population differences which have a deep time depth, on the order of tens of thousands of years. In contrast, the origin of the Ashkenazi Jews likely goes back no earlier than 1,000 years. Over the past 1,000 years they’ve coalesced into a culturally and genetically coherent people.

But why stop with small endogamous groups? 23andMe has [ancestry paintings](http://www.thegeneticgenealogist.com/2009/11/23/hacking-admixed-23andme-ancestry-paintings) which show you your “Asian,” “European” and “African” ancestry along the chromosomes, using the three ancestral reference populations. But in the figure above from the paper you see a Cape Coloured whose ancestry has been broken down *between* the two African populations who are dominant in the ancestral makeup of that ethnic group. [Dodecad](http://blogs.discovermagazine.com/gnxp/2010/10/my-dodecad-results/) tells me I’m about \~15% East Asian, while 23andMe tells me I’m \~45% Asian. The difference is that 23andMe doesn’t break out the indigenous South Asian from exogenous East Asian. At some point I assume I’ll be able to get an ancestry painting which shows the two ancestral categories separately.

A major reason that they seem to think the focus will be on fine-scale ancestry is that it will smoke out recessive diseases found in cryptically endogamous groups. From this I conclude that they lean to the side of those who have asserted that “Jewish diseases” are well known and prominent in part because of the focus given to that group. [South Asians](http://www.nature.com/nature/journal/v461/n7263/abs/nature08365.html) would be a good target of such new focus; according to some researchers
there are clear patterns of genetic endogamy in this set of populations (for which we have anthropological cause). They also note that these recent rare variants are naturally going to break down along population lines, because they haven’t had time to spread.

Speaking of lacking time to spread, the evolutionary parameter of natural selection may also exhibit a lot of between population difference. Strong “sweeps” due to positive selection tend to be partitioned across continental races. This is partly a function of time. By the time a sweep goes from one end of a continent to another, it may run into another sweep which renders its selective effective irrelevant. It has famously been found that light pigmentation in western and eastern Eurasia have different genetic architectures. In other words, the phenotype is arrived at by different genotypic means in the two groups. Additionally, even within broader groups selection because of local adaptation can differentiate demes. For example, the Tibetans and the Han, or differences in lactase persistence in Spain.

Finally, they note that there will be continued advances in understanding of admixed ancestry, as well as a special focus on exonic regions (coding). The latter should be especially interesting, as it might give us more insight into functional differences and similarities. The tentative possibilities of the 1000 Genomes, HapMap, and HGDP, are also outlined. Here is their conclusion:

> New sequencing technology enables genetic studies with larger and > larger sample sizes, increasing our power to detect associations > between genetic variants and medically relevant traits, especially in > the case of rare variants. Understanding patterns of admixture and > population structure is an important part of maximizing this detection > power and reducing confounding factors in genome > sequencing-to-phenotype association studies (50-54). As we have seen > in the analysis of dense array-genotype data, the same sequencing > technology that enables sequencing/phenotype mapping will also enable > us to improve our knowledge of population structure at a fine scale. > This improved knowledge should be of assistance not only in identifying structure in association studies, but also in the description of human history and genetic adaptation, and in the development of personalized medicine tools.

Oh, and the bar plot is nice & crisp. I’ve reedited it a bit, but check out (left to right, K = 9 to K = 5):  
![barplot](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/barplot.png?resize=600%2C882)![barplot](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/12/barplot.png?resize=600%2C882)

**Citation:** Brenna M. Henn, Simon Gravel, Andres Moreno-Estrada, Suehelay Acevedo-Acevedo, & Carlos D. Bustamante (2010). Fine-scale population structure and the era of next-generation sequencing Hum. Mol. Genet. : [10.1093/hmg/ddq403](https://dx.doi.org/10.1093/hmg/ddq403)

### Related Posts:

- [Admixture mapping in Northern
  Europeans?](https://www.gnxp.com/WordPress/2013/09/19/admixture-mapping-in-northern-europeans/) - [Whole genomes of ancient farmers and
  hunter-gatherers](https://www.gnxp.com/WordPress/2020/11/24/whole-genomes-of-ancient-farmers-and-hunter-gatherers/) - [The day of the
  farmer](https://www.gnxp.com/WordPress/2011/03/29/the-day-of-the-farmer/) - [Nomads as
  post-hunter-gatherers](https://www.gnxp.com/WordPress/2012/07/12/nomads-as-post-hunter-gatherers/) - [Patterns in the Y chromosome and Holocene
  expansion](https://www.gnxp.com/WordPress/2019/11/29/patterns-in-the-y-chromosome-and-holocene-expansion/) - [Excavating the Neolithic genetic
  strata](https://www.gnxp.com/WordPress/2010/12/11/excavating-the-neolithic-genetic-strata/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F12%2F10%2Fpainting-the-human-tree-of-life%2F&linkname=Painting%20the%20human%20tree%20of%20life "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F12%2F10%2Fpainting-the-human-tree-of-life%2F&linkname=Painting%20the%20human%20tree%20of%20life "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F12%2F10%2Fpainting-the-human-tree-of-life%2F&linkname=Painting%20the%20human%20tree%20of%20life "Email")[](https://www.addtoany.com/share)
