+++
title = "23andMe v3 chip & me"
full_title = "23andMe v3 chip & me"
date = "2011-01-26"
upstream_url = "https://www.gnxp.com/WordPress/2011/01/26/23andme-v3-chip-me/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/01/26/23andme-v3-chip-me/).

23andMe v3 chip & me

Yesterday the first batch of results from 23andMe’s v3 chip came online. Instead of 550,000 SNPs you get \~1 million. The difference is pretty clear when you look at the raw SNPs. Under Account → Browse Raw Data, I can enter *[LCT](https://www.23andme.com/you/explorer/gene/?gene_name=lct)*, and this is what I see:  
[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/lct23andme.png?resize=500%2C373)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/lct23andme.png?resize=500%2C373)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/lct23andme.png)

I’m line \#2. A sibling is line \#1. Looking at this sort of stuff makes it really likely I’ll upgrade. My main rationale for not upgrading is that there’s diminishing marginal returns for ancestry related stuff. Speaking of ancestry, let’s compare my sibling’s [ancestry painting](https://www.23andme.com/you/ancestry/paint/) to my own.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/23mev3B.png?resize=600%2C473)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/23mev3B.png?resize=600%2C473)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/23mev3B.png)

In a bit of a surprise, while I’m 43% “Asian,” my sibling is 40% “Asian.” This is somewhat reflected in [global similarity](https://www.23andme.com/you/globalsim/).

|                    |           |             |                |
|--------------------|-----------|-------------|----------------|
|                    | **Razib** | **Sibling** | **Difference** |
| Northern Europe    | 66.56     | 66.53       | 0.03           |
| Southern Europe    | 66.47     | 66.43       | 0.04           |
| Middle Eastern     | 66.38     | 66.35       | 0.03           |
| Northern Africa    | 65.78     | 65.68       | 0.1            |
| Central/South Asia | 67.18     | 67.14       | 0.04           |
| East Asia          | 67.66     | 67.56       | 0.1            |
| Siberia            | 67.47     | 67.36       | 0.11           |
| Oceania            | 67.39     | 67.32       | 0.07           |
| North America      | 67.2      | 67.15       | 0.05           |
| South America      | 67.33     | 67.18       | 0.15           |
| West African       | 63.99     | 63.87       | 0.12           |
| Central African    | 64        | 63.89       | 0.11           |
| Eastern African    | 64.15     | 64.06       | 0.09           |
| South African      | 64.04     | 63.93       | 0.11           |

I was curious if the differences were v3 vs. v2 chip, so I compared two individuals of Northern European ancestry:

|                    |                        |                        |                |
|--------------------|------------------------|------------------------|----------------|
|                    | **Northern Europe v2** | **Northern Europe v3** | **Difference** |
| Northern Europe    | 67.82                  | 67.88                  | -0.06          |
| Southern Europe    | 67.74                  | 67.73                  | 0.01           |
| Middle Eastern     | 67.14                  | 67.12                  | 0.02           |
| Northern Africa    | 66.42                  | 66.37                  | 0.05           |
| Central/South Asia | 66.88                  | 66.94                  | -0.06          |
| East Asia          | 65.73                  | 65.77                  | -0.04          |
| Siberia            | 66.05                  | 66.07                  | -0.02          |
| Oceania            | 65.62                  | 65.74                  | -0.12          |
| North America      | 66.11                  | 66.15                  | -0.04          |
| South America      | 66.01                  | 66.02                  | -0.01          |
| West African       | 63.29                  | 63.21                  | 0.08           |
| Central African    | 63.35                  | 64.24                  | -0.89          |
| Eastern African    | 63.48                  | 63.43                  | 0.05           |
| South African      | 63.37                  | 63.27                  | 0.1            |

There’s less of a consistent difference here. So I really don’t know what to think.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/razibandsib.png?resize=300%2C257)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/razibandsib.png?resize=300%2C257)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/razibandsib.png)On the other hand, there’s no difference when it comes to the two dimensional scatter plot which maps your position on the HGDP sample populations, [global similarity advanced](https://www.23andme.com/you/globalsim/advanced/). Instead of looking at the total genome, remember that this is taking your genetic variation, and placing it within a position along a set of independent dimensions which emerge from the HGDP population variance. Here you’re looking at the two largest components of variance, which easily shake out into discernible clusters. The scale here is not fine enough to distinguish myself from my sibling; we’re basically on the same spot (the black positions are a bunch of South Asians I’ve shared genes with in the attempt to elucidate whether I am a genuine genetic outlier in South Asia).

Overall I’m obviously of two minds here: the differences between between myself & my sibling are genuine, or, they’re artifacts of the fact that I’m on v2 and they’re on v3. A major reason I have to suspect that the difference of chip is important is that my sibling is genetically very close to two individuals of Northern European ancestry…who happen to be two other people who I’m sharing with who are v3. Additionally, these two people seem to be suspiciously genetically close as well. v3’s cluster with v3’s far more than random expectation.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/23mev3C.png?resize=350%2C479)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/23mev3C.png?resize=350%2C479)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/01/23mev3C.png)But then I went to [family inheritance](https://www.23andme.com/you/inheritance/). To the left you can see which regions where my sibling & I exhibit [identity by descent](https://en.wikipedia.org/wiki/Identity_by_descent). Since I do not seem to be inbred (confirmed by checking for runs-of-homozygosity in my raw genotype), my parents contribute different [homologs](https://en.wikipedia.org/wiki/Homologous_chromosome), the distinct genesinherited from their own parents, to their offspring. Imagine that the paternal genotype is Pp and and maternal is Mm, where upper case is the copy inherited from the mother, and the lower case inherited from the father (our common grandparents). The offspring could be: PM, Pm, pM, pm. Siblings have an *expectation* of 0.50 relatedness, but because of the variance there’s some wiggle room. If the [Southeast Asian](http://blogs.discovermagazine.com/gnxp/2011/01/me-myself-and-myanmar/) ancestry which we seem to have is recent enough, then recombination may not have broken up the ancestrally informative regions of the genome which are concentrated on particular homologs inherited from each parent (in particular, I have a suspicion that my father’s genome is a mosaic of conventional Bengali along with recent Southeast Asian admixture; we’ll know in a few weeks). Variance of inheritance of these regions of the genomes may then explain the fact that I am “more Asian” than my sibling. All that being said, I do find it of interest that while I have “wet earwax” and the [associated genotype](http://blogs.discovermagazine.com/gnxp/2010/10/east-asians-dry-earwax-and-adaptation/) (in heterozygote form), my sibling has the genotype for dry earwax, which is typically found in East Asia (though the minor allele with recessive expression is found at non-trivial proportions across South Asia). Such issues of personal hygiene are not ones which most have inquired of, so this was news to me, though not surprising given my heterozygosity on this locus.

**Addendum**: For Western Europeans who are on v3, are you finding trace amounts of Asian? Previous gene sharing suggests that \~1% Asian is not uncommon among Finns (and also Russians, but this can be attributable to recent Tatar admixture), but I haven’t seen this among other Europeans without recent non-European admixture. I ask because a friend on v3 who is adopted, but presumably of Western European ancestry (he was told the putative ethnicities of his biological parents), has 1% Asian.

Also, [HAP](http://www.harappadna.org/2011/01/23andme-v3-data/) is accepting v3 for South Asians, Iranians, Burmese, and Tibetans!

### Related Posts:

- [The future of personal genomics is bright.
  23andMe....](https://www.gnxp.com/WordPress/2013/12/07/the-future-of-personal-genomics-is-bright-23andme/) - [Neandertal (haplotype) in the
  family!](https://www.gnxp.com/WordPress/2011/01/27/neandertal-haplotype-in-the-family/) - [The importance of open data in genomics (and in
  everything!)](https://www.gnxp.com/WordPress/2013/01/27/the-importance-of-open-data-in-genomics-and-in-everything/) - [23andMe discount code
  (again)](https://www.gnxp.com/WordPress/2012/08/09/23andme-discount-code-again/) - [South Asian Genotype
  Project](https://www.gnxp.com/WordPress/2017/12/02/south-asian-genotype-project/) - [23andMe ancestry only is \$49.99 for Prime
  Day](https://www.gnxp.com/WordPress/2017/07/11/23andme-ancestry-only-is-49-99-for-prime-day/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F01%2F26%2F23andme-v3-chip-me%2F&linkname=23andMe%20v3%20chip%20%26%20me "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F01%2F26%2F23andme-v3-chip-me%2F&linkname=23andMe%20v3%20chip%20%26%20me "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F01%2F26%2F23andme-v3-chip-me%2F&linkname=23andMe%20v3%20chip%20%26%20me "Email")[](https://www.addtoany.com/share)
