+++
title = "Drivers of selection for"
full_title = "Drivers of selection for ghosts in the genome"
date = "2018-12-23"
upstream_url = "https://www.gnxp.com/WordPress/2018/12/23/drivers-of-the-selection-for-ghosts-in-the-genome/"

+++
Source: [here](https://www.gnxp.com/WordPress/2018/12/23/drivers-of-the-selection-for-ghosts-in-the-genome/).

Drivers of selection for ghosts in the genome

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/12/redqueen.jpeg?resize=183%2C275&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/12/redqueen.jpeg?resize=183%2C275&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/0060556579/geneexpressio-20?creativeASIN=0060556579&linkCode=w61&imprToken=8GdZQXFnf2mivgZdLDQPZw&slotNum=233)A new preprint on bioRxiv, [Strong selective sweeps before 45,000BP displaced archaic admixture across the human X chromosome](https://www.biorxiv.org/content/early/2018/12/21/503995), is suggestive of an exciting new phase in human evolutionary genomics. Basically, leveraging **whole-genomes in diverse populations to explore selection dynamics**.

The authors looked at X chromosomes in males for reasons of technical tractability. Human males carry a single X chromosome, so it’s easy to determine the sequence of genetic variants across a physical stretch of DNA (females, with two X chromosomes, require phasing). The X is interesting for two other reasons: it is present in females about 2/3 of the time (because they have two copies), and, is subject to really strong selection in males 1/3 of the time. Basically, males exhibit no recessive dynamics on the X chromosome because we carry only one. This means that genetic variants which are “recessive” in their expression to selection in females are expressed in males.

The fact that the X is disproportionatelyfound in females also means that all sorts of intra-genomic conflict driven by sex occur on this chromosome. You will know this if you read Matt Ridley’s excellent[The Red Queen](https://www.amazon.com/exec/obidos/ASIN/0060556579/geneexpressio-20?creativeASIN=0060556579&linkCode=w61&imprToken=8GdZQXFnf2mivgZdLDQPZw&slotNum=233).

The specific result here is that the authors found a common family of haplotypes in males on the X chromosome outside of Africa whose homogeneity is indicative of a very strong sweep. From the text:

> The identified selective sweeps are as strong or even stronger than > the most dramatic sweeps previously found in humans. Ten sweeps span > between 500kb and 1.8Mb in more than 50% of non-Africans (Table S2). > The strongest sweep span 900kb in 91% of non-Africans and affects 53% > of non-Africans across a 1.8Mb region. For comparison, the strongest > sweep previously reported surrounds the lactase gene and spans 800kb > in 77% of European Americans (24). **The selection coefficient on the > genetic variant driving this sweep was estimated to 0.15** (24) > suggesting even stronger selection for several of the X chromosome > sweeps we have identified. >
> The swept regions we identify here may be recurrent targets of strong > selection during human evolution. To investigate this possibility, we > intersect our findings with our previously reported evidence of > selective sweeps in the human-chimpanzee ancestor (16).**We find a > strong overlap between the sweeps reported here and regions swept > during the 2-4 my that separated the human-chimpanzee and > human-gorilla speciation events** (17, 25) shown as grey regions in > Figure 2 (Jaccard stat.: 0.17, p-value: \<1e-5) (Materials and > Methods). This suggests that the identified regions of the X > chromosome are continually subjected to extreme positive selection.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/09/genesinconflict.jpeg?resize=185%2C272&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/09/genesinconflict.jpeg?resize=185%2C272&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/0674027221/geneexpressio-20?imprToken=bitu5sxBmgRcgMbWgT-MPA&slotNum=4&imprToken=o4lBKRM-6r8CGZ8QoeDwGQ&slotNum=0&imprToken=k5WOr907WQi4eK-RrMm9HQ&slotNum=0&imprToken=MlnEsfDq0ZISF3ucK8awZw&slotNum=0&imprToken=Q1mry5m2z173HEA7UGeZtg&slotNum=0&creativeASIN=0195149300&linkCode=w61&imprToken=rji8sv2CD1zc4Tl3VlqU5Q&slotNum=124)A selection coefficient of 0.15 is eye-popping. Selection coefficients of 0.01 are reasonable. For humans, anything in the 0.10 range is more like a weird artifact than a true result. But here it is. The fact that the regions overlap with earlier targets of selection during the speciation event that led up to our lineage is clearly of interest.

Looking more closely at the regions of the X which was subject to the selection, **they found almost no archaic ancestry.** That is, the \~1% Neanderthal ancestry that is expected across the X chromosome is almost absent in these segments derived from the selection event. The inference is made that perhaps then these sweeps occurred due to introgression from a sister modern human lineage, perhaps an earlier wave out of Africa which never mixed with Neanderthals. The archaeology is compelling now that these people existed, and there are tentative suggestions from genomics which attests to their presence as well (e.g., modern human admixture into the Altai Neanderthals).

Looking at the 45,000-year-old Siberian genome the authors found the same signatures that they see in other non-Africans. This means the event had to happen between 55 and 45 thousand years ago, after the Neanderthal admixture (which is found all around these zones in the genome), but before geographical diversification and expansion of the modern human lineage.

The authors conclude:

> We hypothesize that our observations are due to meiotic drive in the > form of an inter-chromosomal conflict between the X and the Y > chromosomes for transmission to the next generation. If an averagely > even transmission in meiosis is maintained by a dynamic equilibrium of > antagonizing drivers on X and Y, it is possible that the main > bottlenecked out-of-Africa population was invaded by drivers retained > in earlier out-of-Africa populations. **If this hypothesis is true, > the swept regions represent the only remaining haplotypes from such > early populations not admixed with Neanderthals.**

Meiotic drive is a segregation distorter. A form of intra-genomic selection which is potentially very powerful. Some hypothesize that alleles normally subject to meiotic drive sweep through the population so fast that researchers underestimate the phenomenon’s ubiquity because they haven’t caught sweeps in action.

This strange evolutionary genetic process then may have preserved a genetic relic within the human genome. But on Twitter [Iosif Lazaridis](https://twitter.com/iosif_lazaridis/status/1076558671713390592) suggests that perhaps the donor population were “Basal Eurasians,” and all non-Africans may have some Basal Eurasian ancestry, with Near Easterners exhibiting more than baseline ancestry (presumably through later admixture).

### Related Posts:

- [Genetic
  conundrum](https://www.gnxp.com/WordPress/2007/04/20/genetic-conundrum/) - [Neanderthal Y chromosomes, a tale of
  loss?](https://www.gnxp.com/WordPress/2016/04/15/neanderthal-y-chromosomes-a-tale-of-loss/) - [Hard sweeps and natural selection obscured by Bronze
  Age…](https://www.gnxp.com/WordPress/2020/04/04/hard-sweeps-and-natural-selection-obscured-by-bronze-age-admixture/) - [The inevitability of sex chromosome
  degeneration](https://www.gnxp.com/WordPress/2014/02/09/the-inevitability-of-sex-chromosome-degeneration/) - [Story of X](https://www.gnxp.com/WordPress/2010/09/27/story-of-x/) - [What the substrate
  tells](https://www.gnxp.com/WordPress/2012/09/16/what-the-substrate-tells/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F12%2F23%2Fdrivers-of-the-selection-for-ghosts-in-the-genome%2F&linkname=Drivers%20of%20selection%20for%20ghosts%20in%20the%20genome "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F12%2F23%2Fdrivers-of-the-selection-for-ghosts-in-the-genome%2F&linkname=Drivers%20of%20selection%20for%20ghosts%20in%20the%20genome "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F12%2F23%2Fdrivers-of-the-selection-for-ghosts-in-the-genome%2F&linkname=Drivers%20of%20selection%20for%20ghosts%20in%20the%20genome "Email")[](https://www.addtoany.com/share)
