+++
title = "Ancient structure and"
full_title = "Ancient structure and interbreeding in Africa"
date = "2016-02-18"
upstream_url = "https://www.gnxp.com/WordPress/2016/02/18/ancient-structure-and-interbreeding-in-africa/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/02/18/ancient-structure-and-interbreeding-in-africa/).

Ancient structure and interbreeding in Africa

The figure in Ewen Callaway’s piece in *Nature*, [Evidence mounts for interbreeding bonanza in ancient human species](http://www.nature.com/news/evidence-mounts-for-interbreeding-bonanza-in-ancient-human-species-1.19394), does a good job at relaying what we know about admixture between different human lineages informed by **ancient DNA**. But is that all there is? Before ancient DNA in large quantities came online there were attempts to infer admixture based on the available data (mtDNA, modern populations) and model building and simulations. There were two natural conclusions.

First, admixture happened. Second, it did not. Here’s a paper from 2004, [Modern Humans Did Not Admix with Neanderthals during Their Range Expansion into Europe](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0020421), in *PLOS BIOLOGY*. And from 2006, [Possible Ancestral Structure in Human Populations](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020105), in *PLOS GENETICS* (the first paper has 100 more citations, \~250 vs. \~150, as was in the more mainstream journal). The relatively tentative title of the second paper as opposed to the bold aspect of the first publication does I think reflect the strength of the two positions across academia as a whole at the time (I grant that many population geneticists in particular were skeptical at “Out of Africa” with-total-replacement triumphalism, which was evident in Richard Dawkins’ [The Ancestor’s Tale](https://www.amazon.com/exec/obidos/ASIN/061861916X/geneexpressio-20), published at about that time).

[![Screenshot from 2016-02-18 09-16-07](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/02/Screenshot-from-2016-02-18-09-16-07-300x145.png?resize=300%2C145)![Screenshot from 2016-02-18 09-16-07](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/02/Screenshot-from-2016-02-18-09-16-07-300x145.png?resize=300%2C145)](http://genome.cshlp.org/content/early/2016/02/08/gr.196634.115.abstract)Ancient DNA has changed things. But it has not changed everything, because ancient DNA has been retrieved predominantly from northern Eurasia, for various reasons. I was careful to state above that Callway’s piece was informed by ancient DNA, because I think it omits the likelihood of **archaic admixture within Africa.** A group associated with [Jeff Wall](http://humangenetics.ucsf.edu/wall-jeff/) and [Michael Hammer](http://hammerlab.biosci.arizona.edu/) have been arguing for gene flow between highly diverged lineages within Africa for many years (and archaic admixture more generally going back to the mid-2000s), and yesterday they came out with two papers in *Genome Research*.

First, [Model-based analyses of whole-genome data reveal a complex evolutionary history involving archaic introgression in Central African Pygmies](http://genome.cshlp.org/content/early/2016/02/08/gr.196634.115.abstract):

> Comparisons of whole-genome sequences from ancient and contemporary > samples have pointed to several instances of archaic admixture through > interbreeding between the ancestors of modern non-Africans and now > extinct hominids such as Neanderthals and Denisovans. One implication > of these findings is that some adaptive features in contemporary > humans may have entered the population via gene flow with archaic > forms in Eurasia. Within Africa, fossil evidence suggests that > anatomically modern humans (AMH) and various archaic forms coexisted > for much of the last 200,000 yr; however, the absence of ancient DNA > in Africa has limited our ability to make a direct comparison between > archaic and modern human genomes. Here, we use statistical inference > based on high coverage whole-genome data (greater than 60×) from > contemporary African Pygmy hunter-gatherers as an alternative means to > study the evolutionary history of the genus Homo. Using whole-genome > simulations that consider demographic histories that include both > isolation and gene flow with neighboring farming populations, our > inference method rejects the hypothesis that the ancestors of AMH were > genetically isolated in Africa, thus providing the first whole > genome-level evidence of African archaic admixture. **Our inferences > also suggest a complex human evolutionary history in Africa, which > involves at least a single admixture event from an unknown archaic > population into the ancestors of AMH, likely within the last 30,000 > yr.**

And, [Whole-genome sequence analyses of Western Central African Pygmy hunter-gatherers reveal a complex demographic history and identify candidate genes under positive natural selection](http://genome.cshlp.org/content/early/2016/02/08/gr.192971.115.abstract):

> African Pygmies practicing a mobile hunter-gatherer lifestyle are > phenotypically and genetically diverged from other anatomically modern > humans, and they likely experienced strong selective pressures due to > their unique lifestyle in the Central African rainforest. To identify > genomic targets of adaptation, we sequenced the genomes of four Biaka > Pygmies from the Central African Republic and jointly analyzed these > data with the genome sequences of three Baka Pygmies from Cameroon and > nine Yoruba farmers. To account for the complex demographic history of > these populations that includes both isolation and gene flow, we fit > models using the joint allele frequency spectrum and validated them > using independent approaches. **Our two best-fit models both suggest > ancient divergence between the ancestors of the farmers and Pygmies, > 90,000 or 150,000 yr ago.** We also find that bidirectional asymmetric > gene flow is statistically better supported than a single pulse of > unidirectional gene flow from farmers to Pygmies, as previously > suggested. We then applied complementary statistics to scan the genome > for evidence of selective sweeps and polygenic selection. We found > that conventional statistical outlier approaches were biased toward > identifying candidates in regions of high mutation or low > recombination rate. To avoid this bias, we assigned P-values for > candidates using whole-genome simulations incorporating demography and > variation in both recombination and mutation rates. **We found that > genes and gene sets involved in muscle development, bone synthesis, > immunity, reproduction, cell signaling and development, and energy > metabolism are likely to be targets of positive natural selection in > Western African Pygmies or their recent ancestors.**

I have to say that sometimes I think that selection scans in population genomics are a bit little neuroscience. Neuroscience tells us that stuff happens in the brain. Selection scans tell us that adaptation targets a bunch of functional regions of the genome. Though I’m sure you would feel different in “your gene” shows up on the laundry list.

In any case, two points that I want to emphasize. Haplotypes which seem introgressed from an archaic lineage are underrepresented in genic regions. The same sort of purifying selection you see in archaic admixture in Eurasians (and now in the Altai Neanderthal) seem to be at work here. Second, **the divergence between western Pygmies and African farmer populations is nearly double the time of the “Out of Africa” event.** And, the results from this group seem consistent that admixture continued to occur *after* it had ceased to occur in Eurasians because the archaics outside of Africa had been absorbed by then (at least to our knowledge, I would not be surprised if there was later in some groups detectable at very low levels). This reinforces the idea that we need to update and complexify our idea of how modern humans came to be.

### Related Posts:

- [The hunt for ancient
  DNA](https://www.gnxp.com/WordPress/2011/08/09/the-hunt-for-ancient-dna/) - [The brambly bush of
  humanity](https://www.gnxp.com/WordPress/2012/09/18/the-brambly-bush-of-humanity/) - [Neandertal introgression and
  admixture](https://www.gnxp.com/WordPress/2011/09/12/neandertal-introgression-and-admixture/) - [Ancient DNA analyses are
  accurate](https://www.gnxp.com/WordPress/2015/05/18/ancient-dna-analyses-are-accurate/) - [The replacement of the Neandersovan Y and
  mtDNA?](https://www.gnxp.com/WordPress/2022/07/17/the-replacement-of-the-neandersovan-y-and-mtdna/) - [The blood of the first men runs thin in our
  kind](https://www.gnxp.com/WordPress/2016/02/17/the-blood-of-the-first-men-runs-thin-in-our-kind/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F02%2F18%2Fancient-structure-and-interbreeding-in-africa%2F&linkname=Ancient%20structure%20and%20interbreeding%20in%20Africa "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F02%2F18%2Fancient-structure-and-interbreeding-in-africa%2F&linkname=Ancient%20structure%20and%20interbreeding%20in%20Africa "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F02%2F18%2Fancient-structure-and-interbreeding-in-africa%2F&linkname=Ancient%20structure%20and%20interbreeding%20in%20Africa "Email")[](https://www.addtoany.com/share)
