+++
title = "Y chromosomal"
full_title = "Y chromosomal starphylogenies as intergroup competition between paternal lineages"
date = "2018-05-26"
upstream_url = "https://www.gnxp.com/WordPress/2018/05/26/y-chromosomal-star-phylogenies-as-inter-group-competition-between-paternal-lineages/"

+++
Source: [here](https://www.gnxp.com/WordPress/2018/05/26/y-chromosomal-star-phylogenies-as-inter-group-competition-between-paternal-lineages/).

Y chromosomal star-phylogenies as inter-group competition between paternal lineages

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/star.jpg?resize=300%2C577&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/star.jpg?resize=300%2C577&ssl=1)](https://genome.cshlp.org/content/early/2015/03/13/gr.186684.114.abstract)The figure to the left should be familiar to readers of this weblog. It is taken from [A recent bottleneck of Y chromosome diversity coincides with a global change in culture](https://genome.cshlp.org/content/early/2015/03/13/gr.186684.114.abstract) (Kamin et al.). Over the past few years a peculiar fact long suspected or inferred has come into sharp focus: **some of the Y chromosome haplogroups very common today were not so common in the past, and their frequency changed very rapidly over a short time period.**

What Kamin et al. did was look at sequence data across the Y chromosome to make deeper inferences. The issue is that the Y chromosome is not genetically very diverse. Earlier generations of researchers focused on highly mutable microsatellite regions for identification. While microsatellites are good for identification and classification because of their genetic diversity, they are not as good when it comes to making evolutionary inferences about parameters such as time since last common ancestor. They have very high and variable mutation rates.

Single nucleotide polymorphisms (SNPs) are probably better for a lot of evolutionary inference, but the Y chromosome doesn’t have too many of these. SNP-chip era technology which focuses on a select subset of polymorphisms at specific locations didn’t have much to choose from and likely missed rare variants.

This is where whole-genome sequence of the Y comes in. It retrieves maximal information, and with that, the authors of Kamin et al. **could definitely confirm that some Y chromosomal lineages under explosive expansion \~4,000 years ago after a bottleneck.**

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/01/whoweare.png?resize=183%2C276&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/01/whoweare.png?resize=183%2C276&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/110187032X/geneexpressio-20/ref=as_at/?imprToken=1h1z5ssUdqtqQH0YhQ-4Yw&slotNum=25&creativeASIN=110187032X&linkCode=w61&imprToken=OgM73TDinq7-x8ezy5iRxg&slotNum=17)By and large ancient DNA take a different angle, focusing on genome-wide autosomal ancestry, and lacking in high-coverage whole-genome sequences. **But they have confirmed the inferences from whole-genomes that some of these lineages exhibit explosive growth in the last \~4,000 years.** One moment they were rare, and the next moment ubiquitous.

But geneticists are geneticists. They’re interested in genetical questions, methods, and dynamics. To be frank cultural models for how those genetic patterns might have come about are either exceedingly simple and probably true (e.g., gene-culture coevolution with lactase persistence), or vague and handwavy. With the surfeit of genomic data to analyze it isn’t surprising that this happens.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/ultrasociety.jpeg?resize=178%2C284&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/ultrasociety.jpeg?resize=178%2C284&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B0185P69LU/geneexpressio-20/ref=as_at?creativeASIN=0192862154&linkCode=w61&imprToken=M0UUbk5RhMPgr3urM-pLyw&slotNum=77)This is why researchers in the field of **cultural evolution need to get involved.** They’re model-builders and should see which models predict the copious empirical results we have now when it comes to genetic change over time.

For several years now I have been asserting that inter-group competition of paternal lineages best explains the pattern of Y chromosome expansions \~4,000 years ago. A new paper brings forth a formal model which explores this hypothesis, [Cultural hitchhiking and competition between patrilineal kin groups explain the post-Neolithic Y-chromosome bottleneck](https://www.nature.com/articles/s41467-018-04375-6):

> In human populations, changes in genetic variation are driven not only > by genetic processes, but can also arise from cultural or social > changes. An abrupt population bottleneck specific to human males has > been inferred across several Old World (Africa, Europe, Asia) > populations 5000–7000 BP. Here, bringing together anthropological > theory, recent population genomic studies and mathematical models, we > propose a sociocultural hypothesis, involving the formation of > patrilineal kin groups and intergroup competition among these groups. > **Our analysis shows that this sociocultural hypothesis can explain > the inference of a population bottleneck.** We also show that our hypothesis is consistent with current findings from the archaeogenetics of Old World Eurasia, and is important for conceptions of cultural and social evolution in prehistory.

Their model is interesting because inter-group competition between paternal lineages can result in a loss of haplogroup diversity **without huge reproductive skew.** That is, instead of a highly polygynous society, one can simply posit that group dynamics of expansion and extinction produce expansions of Y chromosomal lineages.

A formal model synthesized with genomic results is a major step forward, though I haven’t dug into the methods (computational or analytic). Presumably, this is a first step.

But the discussion does review a lot of anthropological literature about the nature of human conflict and social interaction. **Basically, it seems that between nomadic hunter-gatherers and before chiefdoms, biologically defined paternal clans were often the organizing principle of society.** To some extent this makes total sense since the meta-ethnic religious and social identities explicitly appeal to fictive relationships of blood even after blood was no longer paramount. Ancient Near Eastern kings addressed each other in familial terms (e.g., “brother” and “son”), while universal religions deploy the construct of brotherhood.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/08/empiresofsilkroad.jpeg?resize=150%2C233&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2017/08/empiresofsilkroad.jpeg?resize=150%2C233&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B004UGKKBE/geneexpressio-20/ref=as_at/?imprToken=7vRanc6KnEJnT3TPIDYuDA&slotNum=2&creativeASIN=0691135894&linkCode=w61&imprToken=QQrq..S3nYSeJ5rRpa9Ffw&slotNum=131)In [Empires of the Silk Road](https://www.amazon.com/exec/obidos/ASIN/0691135894/geneexpressio-20/ref=as_at?creativeASIN=0691135894&linkCode=w61&imprToken=QQrq..S3nYSeJ5rRpa9Ffw&slotNum=131) the author makes the case that these bands of brothers were more influential in shaping history than we realize today. Not surprisingly, the authors of the above paper suggest that the Inner Asian nomad zone is where star-phylogenies have been most pervasive and persist down to historical time. As in Steven Pinker’s [The Better Angels of Our Nature](https://www.amazon.com/exec/obidos/ASIN/B0052REUW0/geneexpressio-20/ref=as_at?creativeASIN=0192862154&linkCode=w61&imprToken=M0UUbk5RhMPgr3urM-pLyw&slotNum=77) it seems that the rise of the state suppressed the viciousness of the paternal kin group. How do we know this? Because the period of the maximal explosion of star-phylogenies seem to be a transient between the early Neolithic and the historical age.

The Y chromosomal literature is just the low hanging fruit. I suspect in the next decade cultural evolutionary models will be brought to bear on the huge mountain of genomic data….

**Citation**:Cultural hitchhiking and competition between patrilineal kin groups explain the post-Neolithic Y-chromosome bottleneck Tian Chen Zeng, Alan J. Aw & Marcus W. Feldman.

### Related Posts:

- [Perhaps the 'Genghis Khan haplotype' is not Genghis
  Khan's?](https://www.gnxp.com/WordPress/2018/02/03/perhaps-the-genghis-khan-modal-haplotype-is-not-genghis-khans/) - [Adam was African, but perhaps
  barely](https://www.gnxp.com/WordPress/2011/05/20/adam-was-african-but-perhaps-barely/) - [Surname & common paternal descent in
  England](https://www.gnxp.com/WordPress/2009/02/10/surname-common-paternal-descent-in-england/) - [The Y chromosomal sons of the
  conquerors?](https://www.gnxp.com/WordPress/2015/01/14/the-y-chromosomal-sons-of-the-conquerors/) - [Y chromosomes around the
  Baltic](https://www.gnxp.com/WordPress/2021/05/08/y-chromosomes-around-the-baltic/) - [Backwards in
  Time](https://www.gnxp.com/WordPress/2008/03/18/backwards-in-time/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F05%2F26%2Fy-chromosomal-star-phylogenies-as-inter-group-competition-between-paternal-lineages%2F&linkname=Y%20chromosomal%20star-phylogenies%20as%20inter-group%20competition%20between%20paternal%20lineages "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F05%2F26%2Fy-chromosomal-star-phylogenies-as-inter-group-competition-between-paternal-lineages%2F&linkname=Y%20chromosomal%20star-phylogenies%20as%20inter-group%20competition%20between%20paternal%20lineages "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F05%2F26%2Fy-chromosomal-star-phylogenies-as-inter-group-competition-between-paternal-lineages%2F&linkname=Y%20chromosomal%20star-phylogenies%20as%20inter-group%20competition%20between%20paternal%20lineages "Email")[](https://www.addtoany.com/share)
