+++
title = "Selection is going on"
full_title = "Selection is going on with SLC24A5…"
date = "2018-05-24"
upstream_url = "https://www.gnxp.com/WordPress/2018/05/24/selection-is-going-on-with-slc24a5/"

+++
Source: [here](https://www.gnxp.com/WordPress/2018/05/24/selection-is-going-on-with-slc24a5/).

Selection is going on with SLC24A5….

![**The ancestral allele for rs1426654 at SLC24A5**](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/slc24a5-1.jpg?resize=625%2C568&ssl=1)

[](https://www.snpedia.com/index.php/Rs1426654)  
[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/04/molecularpopulationgenetics.jpeg?resize=180%2C238&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/04/molecularpopulationgenetics.jpeg?resize=180%2C238&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/0878939652/geneexpressio-20/ref=as_at/?imprToken=PSlMMhhqZsYJQZivHyij.A&slotNum=10&creativeASIN=B00RTY0LPO&linkCode=w61&imprToken=TlKC3Z.2TonJfIX35XEb3g&slotNum=96)On this week’s episode of [The Insight](http://insitome.libsyn.com/website/molecular-population-genetics), I talked to [Matt Hahn](https://twitter.com/3rdreviewer) about why he [wrote his new book](%3Ca%20href=), his opinions on “Neutral Theory”, and what he thought about David Reich’s [op-ed](https://www.nytimes.com/2018/03/23/opinion/sunday/genetics-race.html). Without Spencer’s supervision, I have to admit that I think I lost control and just went [“full nerd”](https://twitter.com/razibkhan/status/999477699180662785). Next week we’re dropping Carl Zimmer’s podcast, so rest assured that the world will come back into balance, and[The Insight](http://insitome.libsyn.com/website)will be more welcoming to civilians!

At a certain point, Matt and I were discussing allele frequency differences between populations and he came close to saying*all* such differences between human populations were of modest frequency in relation topairwise comparisons (e.g., 40% vs. 49%). Obviously, this is not true, because there is always the huge difference in*SLC24A5*at SNP[rs1426654](https://www.snpedia.com/index.php/Rs1426654)(at Duffy and a few other loci). A substitution of a G for an A converts the codon from alanine to threonine.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/mutants.jpeg?resize=180%2C280&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/mutants.jpeg?resize=180%2C280&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/0670031100/geneexpressio-20/ref=as_at?creativeASIN=0670031100&linkCode=w61&imprToken=M4gkRRJX1Cx-I3nqWlJIQA&slotNum=193)You have heard of this locus because of a paper in 2005, [SLC24A5, a putative cation exchanger, affects pigmentation in zebrafish and humans](https://www.ncbi.nlm.nih.gov/pubmed/16357253). This paper came out in December of 2005, a few years after Armand Leroi wrote in [Mutants](https://www.amazon.com/exec/obidos/ASIN/0670031100/geneexpressio-20/ref=as_at?creativeASIN=0670031100&linkCode=w61&imprToken=M4gkRRJX1Cx-I3nqWlJIQA&slotNum=193) that geneticists still hadn’t come to grips with normal variation in pigmentation in humans. The above publication was the first step in solving this question in the years between 2005 to 2010, at least to a good first approximation.

In the sample in the paper they explain 25-40% of the variation in melanin index between Africans and Europeans with this single genetic change (for various technical reasons it’s probably not that big an effect, though it is still big, and probably the largest effect quantitative trait locus for pigmentation in the human genome).

It turns out that this mutation, the derived variant, is almost disjoint is frequency between Europeans and Africans. That is, about \~100% of Africans carry the ancestry G base at while \~0% of Europeans carry the G base (as opposed to the A base). Interestingly, East Asians carry the G base at \~100% frequency as well. If you genotype an anonymous individual and their genotype is AG or GG on atrs1426654 **then it is highly likely that that individual is not a European**.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/04/10Kexplosion.jpeg?resize=181%2C278&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/04/10Kexplosion.jpeg?resize=181%2C278&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/B0042FZRPC/geneexpressio-20/ref=as_at?creativeASIN=B0042FZRPC&linkCode=w61&imprToken=AU7P4KH8IkSPLjAk4i5eLQ&slotNum=57)To give an example of how this works, in 2013 I stumbled onto a [paper](https://www.ncbi.nlm.nih.gov/pubmed/?term=Population+differences+of+two+coding+SNPs+in+pigmentation-related+genes+SLC24A5+and+SLC45A2) which genotyped 101 Europeans from Cape Town in South Africa. That means there are 202 alleles (two per person) atrs1426654. Of these, 5 of the alleles were ancestral (G). From this, I immediately concluded that it was highly likely that the Afrikaaner people of South Africa have non-European ancestry. I came to this conclusion because of 5 copies of the ancestral allele, \~2.5%, is shockingly high for a European population, and it was long surmised that the Afrikaaner people had some non-European heritage (Khoisan, Bantu, South and Southeast Asian) ancestry. The major of the whites sampled in Cape Town could have been Afrikaaners (I’ve [confirmed](https://www.gnxp.com/WordPress/2017/09/22/the-non-european-ancestry-of-afrikaners/) this with genome-wide data).

To get a sense of where my intuitions come from you need **to look at allele counts within populations**. Using [1000 Genomes](http://useast.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=15:48133787-48134787;v=rs1426654;vdb=variation;vf=989832), [Yale’s Alfred](https://alfred.med.yale.edu/alfred/SiteTable1A_working.asp?siteuid=SI007419V), and [Gnomad](https://gnomad.broadinstitute.org/)I assembled a representative list to give you a sense of what’s going on. Using 126,548 counted alleles in Gnomad for individuals of European (non-Finnish) descent you see that 0.38% out of the total, 486, areancestral.

|                              |                       |                   |          | |------------------------------|-----------------------|-------------------|----------| | **Population**               | **Ancestral alleles** | **Total alleles** | **Freq** | | Samaritan                    | 0                     | 74                | 0%       | | Basque                       | 0                     | 216               | 0%       | | Greeks (Thrace, Athens)      | 0                     | 184               | 0%       | | Burusho                      | 0                     | 50                | 0%       | | Pandit Brahmin, Kashmir      | 0                     | 40                | 0%       | | European (Non-Finnish)       | 486                   | 126548            | 0%       | | Ashkenazi Jewish             | 47                    | 10148             | 0%       | | European (Finnish)           | 329                   | 25790             | 1%       | | Iraq Kurds                   | 1                     | 68                | 2%       | | Yemenite Jews                | 2                     | 78                | 3%       | | Havyaka Brahmin, Karnataka   | 2                     | 62                | 3%       | | Palestinian                  | 4                     | 122               | 3%       | | Gujarati                     | 10                    | 206               | 5%       | | Tunisian Berber              | 6                     | 110               | 5%       | | Andalusian                   | 14                    | 252               | 6%       | | Iranian                      | 6                     | 84                | 7%       | | Pashtun                      | 21                    | 190               | 11%      | | Uttar Pradesh Brahmin        | 4                     | 34                | 12%      | | Pandit Brahmin, Haryana      | 13                    | 78                | 17%      | | Punjabi                      | 42                    | 192               | 22%      | | South Asian                  | 6921                  | 30774             | 22%      | | Kalash                       | 14                    | 48                | 29%      | | Telugu                       | 71                    | 204               | 35%      | | Bangladeshi                  | 80                    | 172               | 47%      | | Sri Lanka Tamil              | 105                   | 204               | 51%      | | Adi-Dravida, Karnataka       | 21                    | 34                | 62%      | | Masai Kenya                  | 192                   | 286               | 67%      | | Austro-Asiatic tribe, Odisha | 43                    | 56                | 77%      | | Luhya Kenya                  | 155                   | 188               | 82%      | | Hausa                        | 68                    | 76                | 90%      | | Mende Sierra Leone           | 155                   | 170               | 91%      | | Gambian                      | 209                   | 226               | 92%      | | Ibo                          | 90                    | 94                | 96%      | | Austro-Asiatic tribe, Odisha | 92                    | 96                | 96%      | | Esan Nigeria                 | 193                   | 198               | 97%      | | Yoruba Nigeria               | 213                   | 216               | 99%      | | Biaka                        | 135                   | 136               | 99%      | | East Asian                   | 18728                 | 18856             | 99%      | | Ghana                        | 140                   | 140               | 100%     | | Mbuti                        | 74                    | 74                | 100%     |

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/01/whoweare.png?resize=183%2C276&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/01/whoweare.png?resize=183%2C276&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/110187032X/geneexpressio-20/ref=as_at/?imprToken=pZKmcNNvVeCrk0MGBsM1nQ&slotNum=12&imprToken=mpDM3JSs1wgKS4uL.99NHw&slotNum=0&creativeASIN=110187032X&linkCode=w61&imprToken=OgM73TDinq7-x8ezy5iRxg&slotNum=17)Last fall [Crawford et al.](http://science.sciencemag.org/content/early/2017/10/11/science.aan8433.full) reported thatrs1426654 is embedded in a haplotype that’s about \~30,000 years ago. Additionally, they contend that its presence within Africa is probably no earlier than the Holocene, the last \~12,000 years. [Martin et al.](https://www.cell.com/cell/pdf/S0092-8674(17)31324-7.pdf) report that KhoeSan exhibit higher frequencies of the derived allele because of Eurasian back-migration and then*in situ*natural selection. Of course, not all Eurasians. Most East Asians have the ancestral variant ofrs1426654.

This leaves us with West Eurasians, North Africans, and South Asians. I’ve put a few South Asian populations in the list to show you that there is a wide range of variation in allele frequencies. TheSouth Asians in Gnomad, probably mostly Diaspora, have the ancestral variant at only 22%. In contrast, Austro-Asiatic speaking South Asian groups from northeast India have very high frequencies of the ancestral variant. There has clearly been*in situ*[selection in some South Asian populations](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003912) for the derived variant atrs1426654. Ancestral North Indian groups (ANI) probably brought the derived allele, and Ancient Ancestral South Indians (AASI) probably tended to carry the ancestral allele, like East Eurasians and Oceanians. Additionally, South Asian populations often have high drift. Some of the differences in the Alfred data seem to be impacted by this.

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/skinnatural.jpeg?resize=192%2C262&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/05/skinnatural.jpeg?resize=192%2C262&ssl=1)](https://www.amazon.com/exec/obidos/ASIN/0520275896/geneexpressio-20/ref=as_at/?imprToken=PSlMMhhqZsYJQZivHyij.A&slotNum=10&creativeASIN=B00RTY0LPO&linkCode=w61&imprToken=TlKC3Z.2TonJfIX35XEb3g&slotNum=96)The situation in the Middle East, North Africa, and Europe is different. In the Middle East and North Africa, the ancestral variant is present at frequencies around 1-10%. Some of this can probably be attributed to admixture from Africa and in some cases South and East Asian populations. Ancient DNA from the Middle East and North Africa presents a mixed picture. The farmers who brought the Neolithic to Europe carried the derived variant atrs1426654, and some of the ancient Middle Eastern samples carry it. But not all. The recent Iberiomauserian samples which date to \~15,000 years ago don’t seem to have had the derived variant.

Though the hunter-gatherers of Western Europe only seem to have carried the ancestral variant atrs1426654, the hunter-gatherers of Scandinavia and Eastern Europe did exhibit the derived variant in some frequency, though lower than modern Europeans.

**My own hunch is that the original genetic background against which the A mutation atrs1426654 emerged will be found increasing in frequency first somewhere in the Near East after the Last Glacial Maximum**. But no ancient population shows the frequencies of the derived variant we see in modern Europeans. In isolated populations subject to drift it wouldn’t be surprising if the ancestral variant decreased to \~0%, But in European populations today in the vast majority of cases the ancestral variant is far lower than 1%, even though we know that within the last 10,000 years the ancestral populations streams had several groups with very high frequencies of that ancestral variant. The low frequency is not due to a freakish bottleneck all across Europe. It has to be selection

One thing I have pointed out is that this very low frequency of the ancestral variant indicates that the advantage atrs1426654 for the A allele in Europe is additive. In Northern Europe, the frequency of the derived variant that confers lactase persistence tops out at around \~90 percent. We know this region of the genome has been targeted by natural selection, but lactase persistence also happens to express dominantly genetically. That is, one variant of the mutant allele confers the phenotype. Once you hit \~90 percent of the derived variant only \~1 percent of the population would be lactose intolerant homozygotes (two copies of the ancestral variant). In the Gnomad sample of 60,000+ Europeans, they count three homozygote genotypes rs1426654. That’s 0.005%.

Something is happening atrs1426654. Selection. But why? No one really has any explanation beyond the obvious.

### Related Posts:

- [My new podcast with Spencer
  Wells](https://www.gnxp.com/WordPress/2017/12/13/my-new-podcast-with-spencer-wells/) - [A Kimura Age to the Kern-Hahn Era: neutrality &
  selection](https://www.gnxp.com/WordPress/2018/11/09/a-kimura-age-to-the-kern-hahn-era-neutrality-selection/) - [Reflecting on Journey of Man 15 years
  later](https://www.gnxp.com/WordPress/2018/02/08/reflecting-on-journey-of-man-15-years-later/) - [The Insight, Episode 19: Roberta Estes, the Golden
  State…](https://www.gnxp.com/WordPress/2018/05/02/the-insight-episode-19-roberta-estes-the-golden-state-killer-and-forensic-genetics/) - [Selection or demography in differences between
  human…](https://www.gnxp.com/WordPress/2009/01/10/selection-or-demography-in-differences-between-human-populations/) - [Open Thread,
  4/17/2018](https://www.gnxp.com/WordPress/2018/04/17/open-thread-4-17-2018/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F05%2F24%2Fselection-is-going-on-with-slc24a5%2F&linkname=Selection%20is%20going%20on%20with%20SLC24A5%E2%80%A6. "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F05%2F24%2Fselection-is-going-on-with-slc24a5%2F&linkname=Selection%20is%20going%20on%20with%20SLC24A5%E2%80%A6. "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F05%2F24%2Fselection-is-going-on-with-slc24a5%2F&linkname=Selection%20is%20going%20on%20with%20SLC24A5%E2%80%A6. "Email")[](https://www.addtoany.com/share)
