+++
title = "PRDM9 and the evolution"
full_title = "PRDM9 and the evolution of recombination hotspots"
date = "2010-01-02"
upstream_url = "https://www.gnxp.com/WordPress/2010/01/02/prdm9-and-the-evolution-of-recombination-hotspots/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/01/02/prdm9-and-the-evolution-of-recombination-hotspots/).

PRDM9 and the evolution of recombination hotspots

This week in Science, [three](http://www.sciencemag.org/cgi/content/abstract/science.1183439v1?sa_campaign=Email/pap/31-December-2009/10.1126/science.1183439) [papers](http://www.sciencemag.org/cgi/content/abstract/science.1181495v1?sa_campaign=Email/pap/31-December-2009/10.1126/science.1181495) [report](http://www.sciencemag.org/cgi/content/abstract/science.1182363v1?sa_campaign=Email/pap/31-December-2009/10.1126/science.1182363) that the product of the gene PRDM9 is an important determinant of where recombination occurs in the genome during meiosis. Though this may sound like something of an esoteric discovery, it’s actually pretty remarkable, and brings together a number of lines of research in evolutionary genetics. How so?

*A bit of background.*

A few somewhat related facts:

1\. A major goal in the study of speciation is the identification of the genes that underlie reproductive barriers between species. [In 2008](http://www.sciencemag.org/cgi/content/abstract/sci;323/5912/373), the first such gene in mammals was found–**in a cross between two subspecies of mouse where the male offspring are sterile (note that this follows [Haldane’s rule](https://en.wikipedia.org/wiki/Haldane%27s_rule)), a introduction of the “right” version of a single gene was sufficient to restore fertility. This gene? PRDM9**, which encodes a histone methyltranferase expressed in the mouse germline. This gene has [evolved rapidly](http://www.plosgenetics.org/article/info:doi%2F10.1371%2Fjournal.pgen.1000753#pgen.1000753-Good1) across animals, especially in the part of the protein that binds DNA. This suggests it is binding a sequence that is changing particularly rapidly over evolutionary time.

2\. The positions in the genome at which recombination during meiosis are not scattered randomly, but rather cluster together in what are called “[recombination hotspots](https://en.wikipedia.org/wiki/Recombination_hotspot)“. Enriched within these hotspots in humans is [a particular sequence motif](http://www.nature.com/ng/journal/v40/n9/abs/ng.213.html), presumably an important binding site for whatever factor is controlling recombination. As this fact was becoming clear, a group compared the positions of these recombination hotspots between humans and chimpanzees. The [result](http://www.ncbi.nlm.nih.gov/pubmed/15705809)? The positions of these hotspots are remarkably different between these species. In fact, **the positions of recombination hotspots in humans and chimpanzees are nearly non-overlapping, a fairly impressive fact given that the genomes themselves are 99.X% identical.**

3\. But perhaps \#2 isn’t all that surprising. If there are two alleles at a hotspot, one of which is “hot” and the other of which is “cold” (ie. doesn’t initiate recombination), the mechanism of recombination results in gene conversion of the “hot” allele to the “cold” allele (for details, see [here](http://www.ncbi.nlm.nih.gov/pubmed/9223314)). This should result in the relatively rapid loss over evolutionary time of recombination hotspots, which in turn results in what has been called “the [hotspot conversion paradox](http://www.ncbi.nlm.nih.gov/pubmed/9223314)“–if hotspots should trend over time to be more “cold”, how is it that they exist? One plausible resolution of this paradox–**a sequence or gene that doesn’t contain a hotspot itself might control the positioning of recombination elsewhere in the genome.**

4\. Indeed, such genes exist. In mice, [two](http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000036) [groups](http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000035) last year identified regions of the genome (though they didn’t at the time narrow it down to a gene) controlling the usage of individual hotspots. Importantly, one such region was located distantly to the hotspot, indicating an important regulator of recombination positioning. In humans, [a group last year](http://www.sciencemag.org/cgi/content/abstract/1151851) showed that **these is extensive variability between humans in how often previously identified hotspots are used, and that this variation is heritable**.

*PRDM9 brings all of these observations together*

These three papers all report that item \#1 and items \#2-4 above are all related. What do they show?

1\. [Two](http://www.sciencemag.org/cgi/content/abstract/science.1183439v1?sa_campaign=Email/pap/31-December-2009/10.1126/science.1183439) [groups](http://www.sciencemag.org/cgi/content/abstract/science.1181495v1?sa_campaign=Email/pap/31-December-2009/10.1126/science.1181495) followed up on the observation in \#4 above that there was a particular region in mouse controlling hotspot usage, and identified the relevant gene as PRDM9. [One group](http://www.sciencemag.org/cgi/content/abstract/science.1183439v1?sa_campaign=Email/pap/31-December-2009/10.1126/science.1183439) went further, testing whether variation in this gene also influenced hotspot usage in humans. Remarkably, it did, showing that **variation in PRDM9 in both mice and humans leads to variation in hotspot usage**. This variation changes the binding specificity of the gene, leading to changed hotspots and a resolution of the “hotspot conversion paradox” mentioned in \#3 above.

2\. [Another group](http://www.sciencemag.org/cgi/content/abstract/science.1182363v1?sa_campaign=Email/pap/31-December-2009/10.1126/science.1182363) took a different route to a similar conclusion. They followed up the sequence motif mentioned in \#2 above as being enriched in recombination hotspots in humans. The “hotspot paradox” predicts that, if this motif is “hot”, it should be in the process of being removed from the human genome. Similarly, if it’s not “hot” in chimpanzees, it should *not* be in the process of being removed from the chimp genome. **Indeed, this motif has been preferentially lost along the human lineage as compared to the chimp lineage**. They then asked, what is binding this motif? They had two criteria–a protein with a predicted binding site similar to their motif, and lack of conservation of this protein between humans and chimpanzees. Only one gene fit these criteria–PRDM9. Thus, **the rapid evolution of PRDM9 is responsible for the puzzling observation that recombination hotspots are entirely unconserved between humans and chimps**.

*A brief conclusion*

I’ll reiterate that this is a pretty remarkable discovery, opening up the possibility of a direct link between the evolution of recombination and speciation. Is the effect of PRDM9 on recombination responsible for the conformation to Haldane’s rule in the mouse cross described in \#1? Or is there some additional effect of this gene? Is the evolution of PRDM9 sufficient to describe the evolution of recombination hotspots in all animals? One can imagine a whole host of additional questions. Certainly, this is a story to be continued.

### Related Posts:

- [Recombination also
  evolves](https://www.gnxp.com/WordPress/2015/01/18/recombination-also-evolves/) - [Resolving the hotspot
  paradox](https://www.gnxp.com/WordPress/2007/02/01/resolving-the-hotspot-paradox/) - [What drives accelerated molecular
  evolution?](https://www.gnxp.com/WordPress/2006/10/02/what-drives-accelerated-molecular-evolution/) - [The hotspot
  paradox](https://www.gnxp.com/WordPress/2006/10/08/the-hotspot-paradox/) - [Recombination week at
  Science](https://www.gnxp.com/WordPress/2008/01/31/recombination-week-at-science/) - [Population genomics & the
  coalescent](https://www.gnxp.com/WordPress/2009/07/08/population-genomics-the-coalescent/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F01%2F02%2Fprdm9-and-the-evolution-of-recombination-hotspots%2F&linkname=PRDM9%20and%20the%20evolution%20of%20recombination%20hotspots "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F01%2F02%2Fprdm9-and-the-evolution-of-recombination-hotspots%2F&linkname=PRDM9%20and%20the%20evolution%20of%20recombination%20hotspots "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F01%2F02%2Fprdm9-and-the-evolution-of-recombination-hotspots%2F&linkname=PRDM9%20and%20the%20evolution%20of%20recombination%20hotspots "Email")[](https://www.addtoany.com/share)
