+++
title = "How Columbus was not a"
full_title = "How Columbus was not a seer"
date = "2010-01-13"
upstream_url = "https://www.gnxp.com/WordPress/2010/01/13/how-columbus-was-not-a-seer/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/01/13/how-columbus-was-not-a-seer/).

How Columbus was not a seer

A week ago I pointed out that in some visualizations of world wide population variation South Asians & mestizos seem to overlap which each other to a great extent. The reason for this is that both populations can be modeled as admixtures between two separate, but related, populations. Mestizos are the products of pairings between Europeans and indigenous America populations, while South Asians seem to be a stabilized hybrid population which emerged from the fusion of a West Eurasian (closely related to European) and East Eurasian (distantly related to East Asians) populations. The East Eurasian ancestors of South Asians may be distantly related to indigenous American populations, but in a world wide scale the relationship is relatively close (i.e., compared to Europeans vs. indigenous Americans). So when mapped onto a plot of genetic variation incorporating world wide populations South Asians and mestizos naturally resemble each other. That said, a commenter observes:

> Great example of how two dimensions lose information.  
> Given how different the two populations are genetically, guarantee > that the third component separates them pretty cleanly.

Correct. A new paper illustrates this. [Magnitude of Stratification in Human Populations and Impacts on Genome Wide Association Studies](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0008695):

> Genome-wide association studies (GWAS) may be biased by population > stratification (PS). We conducted empirical quantification of the > magnitude of PS among human populations and its impact on GWAS. Liver > tissues were collected from 979, 59 and 49 Caucasian Americans (CA), > African Americans (AA) and Hispanic Americans (HA), respectively, and > genotyped using Illumina650Y (Ilmn650Y) arrays. RNA was also isolated > and hybridized to Agilent whole-genome gene expression arrays. We > propose a new method (i.e., hgdp-eigen) for detecting PS by projecting > genotype vectors for each sample to the eigenvector space defined by > the Human Genetic Diversity Panel (HGDP). Further, we conducted GWAS > to map expression quantitative trait loci (eQTL) for the \~40,000 > liver gene expression traits monitored by the Agilent arrays. > HGDP-eigen performed similarly to the conventional self-eigen methods > in capturing PS. However, leveraging the HGDP offered a significant > advantage in revealing the origins, directions and magnitude of PS. > Adjusting for eigenvectors had minor impacts on eQTL detection rates > in CA. In contrast, for AA and HA, adjustment dramatically reduced > association findings. At an FDR = 10%, we identified 65 eQTLs in AA > with the unadjusted analysis, but only 18 eQTLs after the eigenvector > adjustment. Strikingly, 55 out of the 65 unadjusted AA eQTLs were > validated in CA, indicating that the adjustment procedure > significantly reduced GWAS power. A number of the 55 AA eQTLs > validated in CA overlapped with published disease associated SNPs. For > example, rs646776 and rs10903129 have previously been associated with > lipid levels and coronary heart disease risk, however, the rs10903129 > eQTL was missed in the eigenvector adjusted analysis.

  
The main point of the paper is to smoke out population substructure which might generate spurious false positives in health-related [genome-wide association](https://en.wikipedia.org/wiki/Genome-wide_association_study) studies. The problem is pretty obvious. Imagine you have a medical study with a lot of blacks and whites, and you just assume they’re all genetically basically the same. Then you look for associations of particular genetic variants within the population which has disease X. Of course, it could be that blacks or whites tend to have more of disease X than the other population, and, it turns out hat **blacks and whites also tend to differ on a whole lot of genes.** Modern human population genetics might have “disproved race,” but it sure is very interested in “population substructure.”  
Patterns of between population variation can be visualized by extracting out the independent dimensions of variance, and plotting them against each other. Generally the charts I post on this illustrate the two dimensions which can explain the most variance in the data set (the alleles frequencies across all the [SNPs](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism) in this case), principal components 1 and 2. But the comment above highlights that there are many other dimensions, though they explain less of the variance.  
One issue that the authors of the above paper pinpoint is that the nature of these dimensions are sensitive to the populations which you include in your original data set to generate them. They distinguish here between the dimensions generated from the full [HGDP](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project) data set, which includes \~50 world populations, and visualizations which rely only on one population. In this study they project their own samples of European, African and Hispanic Americans on the dimensions extracted out of the HGDP data set, and also onto dimensions generated from the populations themselves. As an example, consider Hispanic American projected upon the dimensions of variation constructed from Asians, Africans and Europeans, or, Hispanic Americans projected upon the dimensions of variation extracted from only the variance extant within their own population. From what I could tell they actually didn’t find that correcting for total genome variation using these two methods was particularly helpful in generating greater clarity as to the role of population substructure in producing false positives. So let’s focus on on the visualizations, which go back to the title of the post.  
The first chart has PC 1 & PC 2 from the HGDP populations, with their sample of about 50 African Americans projected onto it:  
![col1.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col1.png?resize=500%2C490)![col1.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col1.png?resize=500%2C490)  
Pretty much zero surprise here. I would be willing to assume that the self-identified African American who clusters with Europeans is an error of some sort (e.g., a sample mix-up), but other studies show the same tendency quite frequently. I conclude then that there are actually people who are inadvertently “passing” as black, at least culturally (on the outside they probably look whiter than [G. K. Butterfield](https://en.wikipedia.org/wiki/G._K._Butterfield)).  
The second chart now has PC 1 & PC 3. So the dimension of variation which explains the second largest proportion of variance has now been replaced by the dimension which explains the third largest proportion.  
![col2.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col2.png?resize=500%2C477)![col2.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col2.png?resize=500%2C477)  
Now Native Americans are distinct from East Asians in the HGDP sample. This is because of PC 3. This goes to the commenter’s point that looking at more dimensions of variation gives us a better sense of real population differences.  
Jumping back to PC 1 and 2, but with Hispanics projected onto the HGDP generated space:  
![col3.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col3.png?resize=500%2C503)![col3.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col3.png?resize=500%2C503)  
I don’t know the provenance of the Hispanics, but it looks to me that they’re likely to include many Puerto Ricans, seeing as there’s a large amount of African admixture here. Nevertheless, you still see the overlap between Hispanics and South Asians that you did with the Gujarati-Mexican comparison, though attenuated. So let’s look at PC 1 & PC 3.  
![col4.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col4.png?resize=500%2C490)![col4.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col4.png?resize=500%2C490)  
And yes, all of a sudden mestizos and South Asians do not overlap, and in fact South Asians are further from mestizos than Europeans or Middle Easterners. One could have predicted this from the previous chart.  
Finally, I want to round out the inspection by looking at two charts which project European Americans onto PC 1, PC 2 and PC 3. The European Americans are black points.  
![col5.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col5.png?resize=500%2C486)![col5.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col5.png?resize=500%2C486)  
![col6.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col6.png?resize=500%2C484)![col6.png](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/col6.png?resize=500%2C484)  
Note that European American outliers seem to have a bias toward drifting in the direction of the Native Americans and African Americans. I don’t discount the possibility of errors here, but it is important to note that deviations away from the HGDP European cluster in the last chart are toward the two groups which European Americans have historically been in contact with in North America.  
**Note:** The subjects specific to this study seem to have been resident in the eastern half of the United States. This would tend to support my supposition that they are less likely to be Mexican Americans, and more likely to be Puerto Rican or Cuban Americans, if they were Hispanic.  
**Citation:** Hao K, Chudin E, Greenawalt D, Schadt EE (2010) Magnitude of Stratification in Human Populations and Impacts on Genome Wide Association Studies. PLoS ONE 5(1): e8695. doi:10.1371/journal.pone.0008695

### Related Posts:

- [Columbus was a
  seer](https://www.gnxp.com/WordPress/2010/01/06/columbus-was-a-seer/) - [Brown people are all the same
  (perhaps)](https://www.gnxp.com/WordPress/2006/12/22/brown-people-are-all-the-same-perhaps/) - [Complex history of archaic
  ancestry](https://www.gnxp.com/WordPress/2021/07/19/complex-history-of-archaic-ancestry/) - [Biogeography, Denisovans, and
  Siberia](https://www.gnxp.com/WordPress/2020/06/05/biogeography-denisovans-and-siberia/) - [Personal genomics: more than fun &
  games](https://www.gnxp.com/WordPress/2011/11/10/personal-genomics-more-than-fun-games/) - [Genomic diversity & historical
  inference](https://www.gnxp.com/WordPress/2009/02/16/genomic-diversity-historical-inference/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F01%2F13%2Fhow-columbus-was-not-a-seer%2F&linkname=How%20Columbus%20was%20not%20a%20seer "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F01%2F13%2Fhow-columbus-was-not-a-seer%2F&linkname=How%20Columbus%20was%20not%20a%20seer "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F01%2F13%2Fhow-columbus-was-not-a-seer%2F&linkname=How%20Columbus%20was%20not%20a%20seer "Email")[](https://www.addtoany.com/share)
