+++
title = "A feline genome in full"
full_title = "A feline genome in full"
date = "2014-11-11"
upstream_url = "https://www.gnxp.com/WordPress/2014/11/11/a-feline-genome-in-full/"

+++
Source: [here](https://www.gnxp.com/WordPress/2014/11/11/a-feline-genome-in-full/).

A feline genome in full

![Best friends forever](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/11/10527267_10152223425537984_6323791029112781898_n-300x275.jpg?resize=300%2C275)

As I mentioned yesterday I’m a contributor to a paper which made a big splash yesterday in *PNAS*, [Comparative analysis of the domestic cat genome reveals genetic signatures underlying feline biology and domestication](http://www.pnas.org/content/early/2014/11/05/1410083111.abstract). It’s been pretty widely [covered in the media](http://www.wired.com/2014/11/genes-cat-domestication/). One thing that hasn’t gotten that much play because most people don’t work with whole genomes is that **the feline reference genome needed some work, and the group at Washington Unviersity’s Genome Institute really pushed it much further along the way to being useful.** Much respect to [Wes Warren](http://genome.wustl.edu/people/individual/wes-warren/) and his team. This is not an uncommon issue. We may live in the “post-genomic era,” but that really applies to humans and a few particular model organisms for now. For many lineagesthere is the requisite genome-of-the-week paper, a hastily assembled reference, and then the group goes onto greener pastures. To get a sense, the original [“cat genome” paper](http://www.ncbi.nlm.nih.gov/pubmed/17975172) had 1.9-fold coverage. That means you expect that each SNP will be sequenced \~2 times. The problem with this is that that’s an average, and with variation there will be lots of gaps (leaving aside repetitive regions which are hard to span normally). And, with a \~1% error rate it will be hard to be confident about whether the variation you see is “real” or just error. To get a sense of how much better this paper’s data is they got 58-fold coverage out of pooled samples (n=22) from a wide range of domestic cats from different lineages (as opposed to just Cinnamon the Abyssinian). They also got 7-fold coverage of the wildcat samples, essential for comparative purposes.

To get you some quick background,*F. silvestris catus* diverged from its wildcat ancestors [5,000 to 10,000 years ago](http://www.sciencemag.org/content/304/5668/259.citation). This is in contrast to the dog, which seems to have been domesticated*at least*[15,000 years ago](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004016). The mitochondrial profile of Egyptian cats \~2,500 years ago [was already similar](http://www.ncbi.nlm.nih.gov/pubmed/22923880) to what you see in Egypt today. Over the past few thousand years domestic cats have expanded across a wide range in [Eurasia](http://www.ncbi.nlm.nih.gov/pubmed/18060738). Breeds are relatively new for domestic cats, and tend to be relatively inbred lineages developed over the past few hundred years at most. In contrast the feral cats exhibit population genetic diversity in the [same range as humans](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0053537).

![**Citation:** Comparative analysis of the domestic cat genome reveals genetic signatures underlying feline biology and domestication](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/11/genes.png?resize=400%2C425)

So what did this paper find? First, I think the biggest aspect, which has been picked up by the media, is that [cats are subject to the “domestication syndrome”](http://www.genetics.org/content/197/3/795.full) due to selection on development of neural crest cells. This is not entirely surprising. Domestic cats have a reputation as being marginally tame and lacking in the [servile sycophantic affect](https://en.wikipedia.org/wiki/Odie) of the dog. But in comparison to the wildcats *F. silvestris catus* is actually very tolerant of coexistence with humans. In addition, they exhibit behavioral patterns which are not found in wildcats, such as residing in colonies. The practical reason for this is pretty obvious, as cats residing within Neolithic villages would be living cheek-by-jowl in comparison with their ancestors.

In regards to selection, because there were numerous samples, comparisons could be made across lineages using a sliding window method. Areas with high F_(st) and sharply reduced heterozygosity are tells for selection events. Everyone has their particular genes of interest. What always makes a mark for me is how often I recognize genes which are targets of selection in domestic mammals, considering that that there are \~20,000 genes (granted, some of these selection events sweep across many genes, and the ones listed are often selected based on functional considerations). Evolutionary processes are substrate-neutral, but across a particular phylogenetic depth they tend to rework the same ‘raw material’ over and over again. As we expand the post-genomic empire outward it seems likely that animals and plants closely associated with humans will get the earliest treatment. And I think that will yield some very definite insights into the nature of genomic constraint and convergence conditional on being wrapped up in the same ‘ecosystem’.

### Related Posts:

- [Purring in the post-genomic
  era](https://www.gnxp.com/WordPress/2014/11/10/purring-in-the-post-genomics-era/) - [Pieces of interest for new
  readers....](https://www.gnxp.com/WordPress/2014/11/25/pieces-of-interest-for-new-readers/) - [Through the Wormhole, Are We Here for a Reason?
  (premier,…](https://www.gnxp.com/WordPress/2015/05/13/through-the-wormhole-are-we-here-for-a-reason-premier-may-13th/) - [The cats came, they conquered, and they
  purred](https://www.gnxp.com/WordPress/2016/09/20/the-cats-came-they-conquered-and-they-purred/) - [Open Thread,
  09/01/2013](https://www.gnxp.com/WordPress/2013/09/01/open-thread-09012013/) - [My article in The New York Times is
  out](https://www.gnxp.com/WordPress/2014/11/24/my-article-in-the-new-york-times-is-out/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F11%2F11%2Fa-feline-genome-in-full%2F&linkname=A%20feline%20genome%20in%20full "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F11%2F11%2Fa-feline-genome-in-full%2F&linkname=A%20feline%20genome%20in%20full "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F11%2F11%2Fa-feline-genome-in-full%2F&linkname=A%20feline%20genome%20in%20full "Email")[](https://www.addtoany.com/share)
