+++
title = "Sibling design or bust"
full_title = "Sibling design or bust for polygenic risk scores"
date = "2020-07-28"
upstream_url = "https://www.gnxp.com/WordPress/2020/07/28/sibling-design-or-bust-for-polygenic-risk-scores/"

+++
Source: [here](https://www.gnxp.com/WordPress/2020/07/28/sibling-design-or-bust-for-polygenic-risk-scores/).

Sibling design or bust for polygenic risk scores

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2020/07/Screenshot-from-2020-07-28-17-10-30.jpg?resize=625%2C575&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2020/07/Screenshot-from-2020-07-28-17-10-30.jpg?resize=625%2C575&ssl=1)](https://www.biorxiv.org/content/10.1101/2020.07.20.212530v1)

Unless you have been sleeping under a rock you are aware that there are lots of constant discussions about “polygenic risk scores” (PRS) and residual “population stratification.” The basic issue here is that when you have a risk profile generated by lots of smaller genetic effects that you sum up together the inference of those genetic effects can be skewed and shaped by correlations between the environment and genome-wide ancestry. To “correct” for this researchers traditionally look at the relatedness between individuals, and their population identity, and “account” for that in the model (an older and more primitive method is simply to remove outliers from homogeneous populations and assume there is no stratification).

A new preprint, [Demographic history impacts stratification in polygenic scores](https://www.biorxiv.org/content/10.1101/2020.07.20.212530v1), suggests limitations in current methods:

> Large genome-wide association studies (GWAS) have identified many loci > exhibiting small but statistically significant associations with > complex traits and disease risk. However, control of population > stratification continues to be a limiting factor, particularly when > calculating polygenic scores where subtle biases can cumulatively lead > to large errors. We simulated GWAS under realistic models of > demographic history to study the effect of residual stratification in > large GWAS. We show that when population structure is recent, it > cannot be fully corrected using principal components based on common > variants—the standard approach—because common variants are > uninformative about recent demographic history. Consequently, > polygenic scores calculated from such GWAS results are biased in that > they recapitulate non-genetic environmental structure. Principal > components calculated from rare variants or identity-by-descent > segments largely correct for this structure if environmental effects > are smooth. However, even these corrections are not effective for > local or batch effects. While sibling-based association tests are > immune to stratification, the hybrid approach of ascertaining variants > in a standard GWAS and then re-estimating effect sizes in siblings > reduces but does not eliminate bias. Finally, we show that rare > variant burden tests are relatively robust to stratification. Our > results demonstrate that the effect of population stratification on GWAS and polygenic scores depends not only on the frequencies of tested variants and the distribution of environmental effects but also on the demographic history of the population.

The authors simulated the level of population stratification within the British…which is a very genetically homogeneous group on a worldwide scale. But even hear looking at common variants, as you’d find on most SNP-arrays, there are serious problems with uncorrected stratification. The logic that rare variants capture recent structure, so those demographic scenarios are not accounted for, totally makes sense.

Really the only way to fix this situation is **sibling-design.** Researchers who are curious about quantitative traits need to assemble as many sibling cohorts as they can, and look there. Even using SNPs identified elsewhere may cause issues.

### Related Posts:

- [Problems in
  PRS?](https://www.gnxp.com/WordPress/2019/05/21/problems-in-prs/) - [Let the genomic die
  fly!](https://www.gnxp.com/WordPress/2019/04/19/let-the-genetic-die-fly/) - [Ambry Genetics and PRS
  "equity"](https://www.gnxp.com/WordPress/2021/05/14/ambry-genetis-an-prs-equity/) - [Genetic correlation between
  friends](https://www.gnxp.com/WordPress/2019/05/21/genetic-correlation-between-friends/) - [Risk, personality and
  testosterone](https://www.gnxp.com/WordPress/2009/08/24/risk-personality-and-testosterone/) - [Ambiguity in self-classification of ancestry and its
  problem…](https://www.gnxp.com/WordPress/2020/06/29/ambiguity-in-self-classification-of-ancestry-and-its-problem-with-disease-risk/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2020%2F07%2F28%2Fsibling-design-or-bust-for-polygenic-risk-scores%2F&linkname=Sibling%20design%20or%20bust%20for%20polygenic%20risk%20scores "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2020%2F07%2F28%2Fsibling-design-or-bust-for-polygenic-risk-scores%2F&linkname=Sibling%20design%20or%20bust%20for%20polygenic%20risk%20scores "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2020%2F07%2F28%2Fsibling-design-or-bust-for-polygenic-risk-scores%2F&linkname=Sibling%20design%20or%20bust%20for%20polygenic%20risk%20scores "Email")[](https://www.addtoany.com/share)
