+++
title = "All your GWAS belong to"
full_title = "All your GWAS belong to us"
date = "2018-12-21"
upstream_url = "https://www.gnxp.com/WordPress/2018/12/21/all-your-gwas-belong-to-us/"

+++
Source: [here](https://www.gnxp.com/WordPress/2018/12/21/all-your-gwas-belong-to-us/).

All your GWAS belong to us

[![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/12/gwas-1.jpg?resize=625%2C273&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2018/12/gwas-1.jpg?resize=625%2C273&ssl=1)](https://www.biorxiv.org/content/early/2018/12/19/500090)  
Very interesting preprint, [A global view of pleiotropy and genetic architecture in complex traits](https://www.biorxiv.org/content/early/2018/12/19/500090). Nothing too surprising, but worth a read.

> After a decade of genome-wide association studies (GWASs), fundamental > questions in human genetics are still unanswered, such as the extent > of pleiotropy across the genome, the nature of trait-associated > genetic variants and the disparate genetic architecture across human > traits. The current availability of hundreds of GWAS results provide > the unique opportunity to gain insight into these questions. In this > study, we harmonized and systematically analysed 4,155 publicly > available GWASs. For a subset of well-powered GWAS on 558 unique > traits, we provide an extensive overview of pleiotropy and genetic > architecture. We show that trait associated loci cover more than half > of the genome, and 90% of those loci are associated with multiple > trait domains. We further show that potential causal genetic variants > are enriched in coding and flanking regions, as well as in regulatory > elements, and how trait-polygenicity is related to an estimate of the > required sample size to detect 90% of causal genetic variants. Our results provide novel insights into how genetic variation contributes to trait variation. All GWAS results can be queried and visualized at the GWAS ATLAS resource (http://atlas.ctglab.nl).

### Related Posts:

- [Finding the missing
  heritability](https://www.gnxp.com/WordPress/2009/12/06/finding-the-missing-heritability/) - [There are no NFL genes
  (?)](https://www.gnxp.com/WordPress/2009/10/13/there-are-no-nfl-genes/) - [Is the "missing heritability" right under
  ourâ€¦](https://www.gnxp.com/WordPress/2010/06/21/is-the-missing-heritability-right-under-our-noses/) - [Sibling design or bust for polygenic risk
  scores](https://www.gnxp.com/WordPress/2020/07/28/sibling-design-or-bust-for-polygenic-risk-scores/) - [Genome-wide association studies
  work](https://www.gnxp.com/WordPress/2010/08/05/genome-wide-association-studies-work/) - [GWAS, population structure and the Han
  Chinese](https://www.gnxp.com/WordPress/2009/11/25/gwas-population-structure-and-the-han-chinese/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F12%2F21%2Fall-your-gwas-belong-to-us%2F&linkname=All%20your%20GWAS%20belong%20to%20us "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F12%2F21%2Fall-your-gwas-belong-to-us%2F&linkname=All%20your%20GWAS%20belong%20to%20us "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2018%2F12%2F21%2Fall-your-gwas-belong-to-us%2F&linkname=All%20your%20GWAS%20belong%20to%20us "Email")[](https://www.addtoany.com/share)
