+++
title = "The chimeric robot"
full_title = "The chimeric robot"
date = "2006-04-19"
upstream_url = "https://www.gnxp.com/WordPress/2006/04/19/the-chimeric-robot/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/04/19/the-chimeric-robot/).

The chimeric robot

Richard Dawkins pioneered the popularization of the “selfish gene” concept in the [book of the same name](https://www.amazon.com/exec/obidos/ASIN/0192860925/geneexpressio-20/) in the 1970s, and yet it is clear that most people haven’t really internalized this idea. Otherwise, how to explain the success of books like [The Journey of Man](https://www.amazon.com/exec/obidos/ASIN/0812971469/geneexpressio-20/102-9239820-5488135) and [The Seven Daughters of Eve](https://www.amazon.com/exec/obidos/ASIN/0393323145/geneexpressio-20/102-9239820-5488135) which explicitly conflate population history and gene history to the point where one would assume a 1:1 covariance.

  
In the post below I pointed to a paper, [Local extinction and recolonization, species effective population size, and modern human origins](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=15757241&dopt=Abstract), which shows how more non-trivially simple models make it obvious that “Out-of-Africa” *replacement* models based on uniparental lineages (mtDNA and Y) are foundationally weak. But not everyone has academic access or a deep interest in human population genetics (heathens!). I’ll try to point to some general and very simple points which I think are in the general direction of what the authors were trying to get at (I won’t try and pretend that I can capture their metapopulation analysis and model).  
Genes and people are two different categories, and though each is ineluctably tied to the other **their long term historical dynamics might differ**. The further you go back in time the more irrelevant contemporaneous historical questions become.  
Consider this simple model. Let’s say you have two populations:  
1) **Large**  
2) **Small**  
Let’s assume that large & small derive from an ancestral population, but have been separated for a long period of time. Random genetic drift has fixed, that is driven to 100% frequency, two alternative alleles on a particular locus. On gene A population Large exhibits allele A1 at 100% frequency and population Small exhibits allele A2 at frequency 100%. A1 and A2 eventually coalesce back deep in time of course, but that is neither here nor there, reproductive barriers have been strong enough that the two population genetic structures have diverged.  
Now, let us give Large a population of 10,000, and Small a population of 1,000. Let’s assume that 10 individuals who carry the alternative allele join the population of the other group per generation (while 10 individuals of that previous generation within the destination population leave to keep the size fixed).  
10/1,000 = .01, or 1% introgression rate for Small and 10/10,000 is 0.001 or a .1% introgression rate for Large. At generation 75 only 37% of the alleles on locus A in the small population are ancestral, that is, A1. In contrast, at 75 generations 92% of the alleles at locus A are in the ancestral state in population Large. This is common sense, a smaller population is more easily swamped by an equivalent inflow of foreign alleles.  
Now, let’s take a baby step toward concreteness, assume that it is the Small & Large populations which are transferring 10 individuals per generation between each other. Since the Large population is so, well, large, let’s just say that its allele frequencies don’t change in 75 generations, that is, .92 \~ 1. This makes it easy in terms of calculating the inflow of the A2 allele into the small population, we just use the numbers we found above. In other words, the sheer size of the Large population vs. the Small population has resulted in genetic replacement at the locus we are focusing on in the Small population from the Large one.  
But does this mean that the Small population is just an arm of the Large population? That depends. Consider that the two populations live on islands, and also assume that the demic exchange is facilitated by the fact that ocean going vessels have become durable enough to facilitate minor but non-trivial population exchagnes. Now, assume that one of the islands is nearly subtropical desert (assume that irrigation is made possible by mountains which catch rain) and the other island is marine temperate. Let’s assume that the Large island people are the marine temperate.  
Now, it seems plausible that the Small island folk are dark skinned to protect against the sun, while the Large island people are light. After 75 generations the information we gather from allele A2 would suggest that most of the Small island people are preponderantly Large island in ancestry, ergo, they should look like Large islanders, right? Not necessarily. Imagine a sailor from the Large island and his Small island wife. Assume that they are fixed for alternative skin color alleles, which are different from locus A (no linkage issues). The children might be of mixed coloration, but the grandchildren will exhibit a wide range in appearance. If one assumes this is at the start of the genetic exchanges between the two islands, the Large islanders would be a tiny minority and one assumes that grandchildren of a Large island sailor would be 3/4 Small island ancestrally. Some of them would resemble him in coloration, but most would not, and there would be a range. Nevertheless, **on their overall genome they would be about 1/4 descended from him**. Unless skin color genes are closely linked to locus A then it seems probable that there would be no assocation between the character of locus A and the character of skin color genes. If dark skin is advantagenous the darker grandchildren of the Large island sailor might be far more fecund than their lighter cousins, even though genomically they are *about* the same in terms of ancestral proportions. The point here is that **selection might operate on the population to stabilize it at a particular phenotype**, even if it is not strong enough to prevent the introgression of neutral alleles (that is, perfect assortative mating). Over the long term you might see a **decoupling between genes which effect local adaptations, which would retain their ancestral character, and genes which are relatively neutral and so would be subject to replacement** (even if it isn’t migration, replacement will happen because of substitution!).  
How does this relate to Out-of-Africa? If I read them right, the researchers posit that a large long-term effective population in Africa might have resulted in the swamping out of neutral ancestrally informative alleles in many other populations. What could be the cause of large African long term [effective population](https://en.wikipedia.org/wiki/Effective_population_size)? If you read the neurobiologist WIlliam Calvin you can guess: **the Ice Ages**. The key here is that we need to keep in mind the **long-term effective population**, which is the harmonic mean evaluated over time.  
Consider two sets of numbers in two populations at 10 different times (generations):  
A is 9, 7, 6, 7, 9, 12, 8, 8, 12, 10 over 10 generations.  
B is 20, 24, 15, 10, 2, 2, 20, 21, 22, 19 over 10 generations.  
Your innate numeracy should clue you in on the fact that the arithmetic mean for A is 8.8 while B is 15.5. In other words, the average population size over 10 generations is a little less than twice as large as A in B. **But not so with the harmonic mean.** Here the numbers are 8.4 for A and 6.9 for B. In other words, the long term effective population of A is larger than B!  
This makes founder effect and population bottlenecks understandable. Consider that when a population goes through a bottleneck its genetic variation is purged and it takes quite some time for mutation to replenish it. It doesn’t matter if its census size bounces back quickly, **genetic information has been lost**. An analogy might be a blow up of a small pixelated image.  
In any case, consider the Europe-Africa population relationship. One could imagine that for most of history prior to agriculture they were within the same order of magnitude, **but during the peaks of the Ice Ages Europe’s population declined to a far greater extent than Africa’s**. During this period if the deme-to-deme migration did not let up, African alleles could swamp the less numerous Europeans. When Europe “bounced back” it would still bear the mark of this demographic imbalance. But, as I note above, this does not imply that European-specific alleles would be purged from the genetic pool if selection pressures maintained them.  
Long enough for now. The only thing I ask readers is to offer a pithy 2-sentence summation of what I’m trying to get at.

### Related Posts:

- [Genetic thugs in the
  house!](https://www.gnxp.com/WordPress/2006/04/19/genetic-thugs-in-the-house/) - [Richard Dawkins, The Economist &
  bylines](https://www.gnxp.com/WordPress/2009/07/23/richard-dawkins-the-economist-bylines/) - [Archaic-modern
  hybridization](https://www.gnxp.com/WordPress/2006/08/18/archaic-modern-hybridization/) - [At an inflection point of archaeology and
  genetics](https://www.gnxp.com/WordPress/2017/05/25/at-an-inflection-point-of-archaeology-and-genetics/) - [The few and the
  many](https://www.gnxp.com/WordPress/2010/01/18/the-few-and-the-many/) - [Readings....](https://www.gnxp.com/WordPress/2006/01/06/readings-2/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F04%2F19%2Fthe-chimeric-robot%2F&linkname=The%20chimeric%20robot "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F04%2F19%2Fthe-chimeric-robot%2F&linkname=The%20chimeric%20robot "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F04%2F19%2Fthe-chimeric-robot%2F&linkname=The%20chimeric%20robot "Email")[](https://www.addtoany.com/share)
