+++
title = "Nonadaptive immune"
full_title = "Nonadaptive immune adaptation?"
date = "2005-09-03"
upstream_url = "https://www.gnxp.com/WordPress/2005/09/03/non-adaptive-immune-adaptation/"

+++
Source: [here](https://www.gnxp.com/WordPress/2005/09/03/non-adaptive-immune-adaptation/).

Non-adaptive immune adaptation?

I have talked a fair amount about the [MHC](https://www.gnxp.com/MT2/archives/003580.html) loci on this blog. The MHC are essential cogs in our [adaptive immune system](https://en.wikipedia.org/wiki/Immune_system#Adaptive_immune_system). Part of the reason is the relevance to organ transplantation (self vs. non-self recognition), but another big picture evolutionary factor is that the MHC loci are extremely polymorphmic. When you calculate coalescence, infer the date at which variants diverged, for the alleles some of them pre-date the human-chimp split (\~6 million years ago). In practical terms some chimps and humans match on an MHC locus as opposed to near family members because the alleles transcend species. Directional selection eliminates the variation that characterizes the MHC loci while neutral evolution would result in the persistent extinction of ancient alleles. So [balancing selection](https://en.wikipedia.org/wiki/Balancing_selection) is usually the main way one explains the diversity in combination with ancient character of the some of the alleles.

But the adaptive immune system isn’t the whole story, the [innate immune system](https://en.wikipedia.org/wiki/Immune_system#Innate_immune_system) has very deep evolutionary roots throughout the animal kingdom and is often both the first and only line of defense for many organisms. Nevertheless it doesn’t seem as “sexy” to study from what I gather, so I was surprised to see a paper titled [The heritage of pathogen pressures and ancient demography in the human innate immunity CD209/CD209L region](http://www.journals.uchicago.edu/AJHG/journal/preprints/AJHG42789.preprint.pdf). This is a preprint paper, so not in its final form, but, the basic outline is this:

The CD209 and CD209L loci are essential to the innate immune system, while the former being expressed in macrophages and the latter being expressed in liver and epithelial cells. In additional to their expression differences the two exhibit sharply different signatures of evolutionary history. To a first approximation CD209 is under functional constraint, that is, mutations which alter the amino acid sequence tend to be selected against, resulting in low heterozygosity on the locus. CD209L on the other hand exhibits worldwide signatures of balancing selection, extremely high rates of polymorphism and a rather great deal of within-population vs. between-population variation. But here’s the kicker, on the CD209 locus there are two clades of alleles, a dominant one found in Africans and non-Africans, a minority clade found in Africa. When they did the coalescence for the two clades it was in excess of 2.5 million years. So the authors conclude that this is evidence of ancient African substructure which has left a mark on the human population via admixture (presumably between the expanding proto-moderns and archaics). They point to another paper, [Deep haplotype divergence and long-range linkage disequilibrium at Xp21.1 provide evidence that humans descend from a structured ancestral population](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15937130&query_hl=5), as presenting an “isolation-admixture” model which is the best fit for this deep time coalescence. I haven’t read the second paper yet, but we know what John Hawks [thinks of such activities](http://johnhawks.net/weblog/reviews/genetics/genetics_mre_aapa_2005.html)….

Update: I’ve put the [preprint in the files](http://groups.yahoo.com/group/gnxpforum/files/).

### Related Posts:

- [An ancient
  story](https://www.gnxp.com/WordPress/2006/01/29/an-ancient-story/) - [New theory of MHC
  diversity?](https://www.gnxp.com/WordPress/2008/11/06/new-theory-of-mhc-diversity/) - [A tour of
  MHC](https://www.gnxp.com/WordPress/2007/09/20/a-tour-of-mhc/) - [I am an unbeliever...in
  overdominance](https://www.gnxp.com/WordPress/2007/08/24/i-am-an-unbeliever-in-overdominance/) - [MHC & mating](https://www.gnxp.com/WordPress/2009/05/25/mhc-mating/) - [Selection swimming against the genomic
  tide](https://www.gnxp.com/WordPress/2017/10/22/selection-swimming-against-the-genomic-tide/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F09%2F03%2Fnon-adaptive-immune-adaptation%2F&linkname=Non-adaptive%20immune%20adaptation%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F09%2F03%2Fnon-adaptive-immune-adaptation%2F&linkname=Non-adaptive%20immune%20adaptation%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F09%2F03%2Fnon-adaptive-immune-adaptation%2F&linkname=Non-adaptive%20immune%20adaptation%3F "Email")[](https://www.addtoany.com/share)
