+++
title = "Endangered species act"
full_title = "Endangered species act 2016, for want of a locus"
date = "2016-06-06"
upstream_url = "https://www.gnxp.com/WordPress/2016/06/06/for-want-of-a-locus/"

+++
Source: [here](https://www.gnxp.com/WordPress/2016/06/06/for-want-of-a-locus/).

Endangered species act 2016, for want of a locus

[![Lake_Washington_Ship_Canal_Fish_Ladder_pamphlet\_-\_ocean_phase_Chinook](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/Lake_Washington_Ship_Canal_Fish_Ladder_pamphlet_-_ocean_phase_Chinook.jpg?resize=600%2C203)![Lake_Washington_Ship_Canal_Fish_Ladder_pamphlet\_-\_ocean_phase_Chinook](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/Lake_Washington_Ship_Canal_Fish_Ladder_pamphlet_-_ocean_phase_Chinook.jpg?resize=600%2C203)](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/Lake_Washington_Ship_Canal_Fish_Ladder_pamphlet_-_ocean_phase_Chinook.jpg)  
[![440px-07-03-23RedWolfAlbanyGAChehaw](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/440px-07-03-23RedWolfAlbanyGAChehaw-300x250.jpg?resize=300%2C250)![440px-07-03-23RedWolfAlbanyGAChehaw](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/440px-07-03-23RedWolfAlbanyGAChehaw-300x250.jpg?resize=300%2C250)](https://en.wikipedia.org/wiki/Red_wolf)Years ago I had a long phone conversation with a journalist about the origins and natural history of the [red wolf](https://en.wikipedia.org/wiki/Red_wolf). The reason was that I had casually mentioned that there was genetic evidence suggesting that the red wolf is a stabilized hybrid between gray wolfs and coyotes. That was 2007, if I recall correctly. It’s gotten more definitive since then, as canid [genomicists](http://genome.cshlp.org/content/21/8/1294.short) are starting to be convinced that red wolves are a species which emerged as a synthesis between gray wolves and coyotes. Nevertheless, it also seems true that the red wolf has ancient morphological roots. That is, you can find fossils which resemble red wolves very far back in prehistory.

[![hqdefault](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/hqdefault-300x225.jpg?resize=300%2C225)![hqdefault](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/hqdefault-300x225.jpg?resize=300%2C225)](http://science.sciencemag.org/content/323/5919/1339)And it’s not just *red* wolves. There’s a melanic form of the gray wolf in North America, the visually arresting black wolf. It turns out that this morph is due to introgression from [domestic dogs](http://science.sciencemag.org/content/323/5919/1339), and more specifically, the dogs brought with Amerindians \~10 to \~15,000 years ago. If you look at on a **genome-wide** scale these melanic wolves are not admixed. It’s just that there is a small portion of their genome with a novel variant that increased in frequency and moved from one “species” to the next.

This is just prologue. I’m a geneticist, and these are just interesting facts to me. **But these*genetic* facts translate into policy implications.** E.g., is the red wolf a species or not? If the gray wolf is protected, should there be culling of the melanic variant, which exhibits a phenotype introgressed from dogs? [![51D2RoDDkXL.\_SX329_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/51D2RoDDkXL._SX329_BO1204203200_-198x300.jpg?resize=198%2C300)![51D2RoDDkXL.\_SX329_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/51D2RoDDkXL._SX329_BO1204203200_-198x300.jpg?resize=198%2C300)](https://www.amazon.com/Species-History-Idea-Systematics-ebook/dp/B002WB0YW4?ie=UTF8&redirect=true&tag=geneexpressio-20)In the United States this all goes back to the [Endangered *Species* Act](https://en.wikipedia.org/wiki/Endangered_Species_Act). And the problem at the heart of this piece of legislation is that **species are not a clear and distinct concept.** As taxonomic ranks go species makes the most coherent sense, but there are many different species concepts, and different groups of scientists have different preferences (e.g., evolutionary geneticists tend to prefer the biological species concepts because it works well for their model organisms and is useful in testing particular hypotheses related to evolutionary genetics, while phylogeneticists unsurprisingly prefer the phylogenetic species concept!).

Population geneticists tend to give rather nuanced characterizations of a given species when asked. This is not a useful trait in the eyes of lawyers or politicians. Population *genomicists* often give an even more gnarly picture! (see: red wolf) **There are more things in heaven and earth than are dreamt of in our pre-genomic philosophies.**

[![sushi-621415_960_720](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/sushi-621415_960_720-300x200.jpg?resize=300%2C200)![sushi-621415_960_720](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/sushi-621415_960_720-300x200.jpg?resize=300%2C200)](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/sushi-621415_960_720.jpg)Several months ago I saw a draft of a preprint which would be posted soon speaking to many of these issues. The species in question was Pacific salmon (which, to be clear, is a very distinct lineage from Atlantic salmon!). Using genome-wide methods the results implied that populations of research interest are genetically similar even if they exhibited some distinct phenotypic differences (or, more precisely, these are cases where geography dictated genetic variation, as is a good null hypothesis). In relation to “conservation” if the goal was conservation of genetic variation, then focusing on specific phenotypes may not be relevant because the whole population range presumably had the viable extant variation.

But the devil is in the details. Here’s the preprint, [The evolutionary basis of premature migration in Pacific salmon highlights the utility of genomics for informing conservation](http://biorxiv.org/content/early/2016/06/03/056853):

> …Here we use genomic methods to investigate the evolutionary basis of > premature migration in Pacific salmon, a complex behavioral and > physiological adaptation that exists within highly-connected > populations and has experienced severe declines. Strikingly, we find > that premature migration is associated with the same single locus > across multiple populations in each of two different species. Patterns > of variation at this locus suggest that the premature migration > alleles arose from a single evolutionary event within each species and > were subsequently spread to distant populations through straying and > positive selection. **Our results reveal that complex adaptive > variation can depend on rare mutational events at a single locus, > demonstrate that CUs reflecting overall genetic differentiation can > fail to protect evolutionarily significant variation that has > substantial ecological and societal benefits, and suggest that a > supplemental framework for protecting specific adaptive variation will > sometimes be necessary to prevent the loss of significant biodiversity > and ecosystem services.**

The method here was using [Rad-seq](http://genome.cshlp.org/content/17/2/240.short), which is pretty cost-effective for non-model organisms. Depending on the confidence you want for the genotype calls (this was done with [ANGSD](http://popgen.dk/wiki/index.php/ANGSD)) you get hundreds of thousands of variants back. This is a huge improvement over publications where the fate of a species (or “species”) comes down to a few dozen microsatellites. The form of genomic analysis that that is prominent in this paper is looking at variation in the site frequency spectrum (sfs) and genetic diversity statistics (a lot of other statistics fall straight out of sfs).

[![Screenshot 2016-06-05 02.41.04](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/Screenshot-2016-06-05-02.41.04-300x291.png?resize=300%2C291)![Screenshot 2016-06-05 02.41.04](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/Screenshot-2016-06-05-02.41.04-300x291.png?resize=300%2C291)](https://i0.wp.com/www.unz.com/wp-content/uploads/2016/06/Screenshot-2016-06-05-02.41.04.png)The phenotype that was being investigated was migration timing. Remember, salmon tend to migrate at around the same time so that runs can form the huge masses which allow for mating between males and females. You’ve seen the same documentaries as I have.

Premature migration occurs in some regions of the Pacific salmon distribution, and it is important for various reasons outlined in the introduction (I’ll spare you the ecological details and leave it to your interest in that sort of thing). So comparing premature vs. normal migration timing there was a real obvious hit for likely selection around *GREB1L*. Apparently this gene is well known to be implicated in behavior shifts (for humans it seems to be related to breast cancer, but there’s an ascertainment bias for medical traits in the literature). One thing really obvious is that for salmon who migrate upstream early **the region around *GREB1L* is cleaned up so that there isn’t much genetic diversity.** This is a hallmark of a recent positive selection sweep.

More specifically, they found that there seem to be selection events around *GREB1L* in both steelhead and chinook salmon (two species of Pac salmon that diverged 10-15 million years ago), and, that these occurred independently **once** in each species. Because in the steelhead the site frequency spectrum differed across populations with the premature migration it seems likely that the sweep is at differences stages (or occurred earlier or later). The likelihood then is that a single mutational event has been moving between populations across the coastal groups within a species.

As someone interested in evolutionary dynamics of connected populations this is fascinating to me on an abstract level. Instead of a welter of numerous segregating variants it seems that the mutational events are rare enough in concert with strong selection for this particular morph, so that one variant becomes dominant quickly. But what does it mean for conservation?

Here’s the beginning of the abstract, which I chopped off earlier:

> **The delineation of conservation units (CUs) is a challenging issue > that has profound implications for minimizing the loss of biodiversity > and ecosystem services**. CU delineation typically seeks to prioritize > evolutionary significance and genetic methods play a pivotal role in > the delineation process by quantifying overall differentiation between > populations. While CUs that primarily reflect overall genetic > differentiation do protect adaptive differences between distant > populations, they do not necessarily protect adaptive variation within > highly connected populations. Advances in genomic methodology > facilitate the characterization of adaptive genetic variation, but the > potential utility of this information for CU delineation is unclear….

And now, the last sentence of the discussion:

> Taken together, **our results demonstrate that conservation units > reflecting overall genetic differentiation can fail to protect > evolutionarily significant variation that has substantial ecological > and societal benefits**, and suggest that a supplemental framework for > protecting specific adaptive variation will sometimes be necessary to > prevent the loss of significant biodiversity and ecosystem services.

Let’s go back to the red wolf for a moment, [Should Red Wolves Be Allowed to Mate With Coyotes?](http://6thfloor.blogs.nytimes.com/2014/08/19/should-red-wolves-be-allowed-to-mate-with-coyotes/):

> … Robert Wayne, a geneticist at U.C.L.A., thinks the red wolf is > itself a hybrid. When officials captured what individuals they could > find in the 1970s, the beleaguered animal already had coyotes among > its ancestors, Wayne says. He argues that red wolves should be allowed > to mate with coyotes — **that natural selection should sort out the > question of what animal is best for that landscape.**

The case here with salmonids and wolves are not analogous. It strikes me that many of the features of the red wolf are the outcome of **genome-wide admixture.** E.g., the morphometric properties of the red wolf are between those of the gray wolf and the coyote. The ancient evidence in the paleontological record of the red wolf is probably a function of the fact that red wolves are easily recapitulated out of the genetic reservoir of gray wolves and coyotes. Let me be honest: we could create the “red wolf” *de novo* if the current populations went extinct.

In the case of the salmonids the dynamics are somewhat more complex **and** simple at the same time. It is simple because the genetic architecture for a complex behavior is very elegant. One locus, with presumably asingle allele. If there were many numerous segregating variants than one would have to think that the mutational target is large (e.g., lactase persistence in humans exhibits this). But there aren’t. The authors suggest then that **mutation itself may be a rate limiting step in diversification**, so that if the endangered premature migrating salmon are extirpated they may not ‘naturally’ reemerge out of the genetic background of the more numerous conventional phenotype (at least within the normal scope of a human lifetime). Though there are heterozygotes their data may be selected against. It seems that there are probably two stable states in terms of behavior, encoded by homozygote genotypes (this is a preprint, so to me it begs the question as a critique: **how are the initial heterozygotes eventually reproducing enough to produce homozygotes?**).

The ultimate moral of the story is that **to make *informed* decisions in the area of conservation biology in the future one needs to sequence individuals of the organism of interest with reasonable good geographic coverage.** A single summary statistic, whether it be of genetic diversity, or population structure, is not going to be the end of anything. Rather, one has to frame the ends of the policy (e.g., are there ecological morphs which are deemed worthy of protection?), and then laboriously decompose the data as population geneticists would, focusing on structure and parameters of allele frequency change like selection and drift.

**Citation:** The evolutionary basis of premature migration in Pacific salmon highlights the utility of genomics for informing conservation, Daniel J Prince, Sean M O’Rourke, Tasha Q Thompson, Omar A Ali, Martha Arciniega, Hannah S Lyman, Ismail K Saglam, Anthony J Clemento, Thomas J Hotaling, Andrew P Kinziger, Adrian P Spidle, John Carlos Garza, Devon E Pearse, Michael R Miller, bioRxiv 056853; doi: [10.1101/056853](http://biorxiv.org/content/early/2016/06/03/056853)

### Related Posts:

- [Don't hate the
  hybrid!](https://www.gnxp.com/WordPress/2011/11/21/dont-hate-the-hybrid/) - [Coywolves; hybrid wolf-coyotes in New
  England?](https://www.gnxp.com/WordPress/2009/11/30/coywolves-hybrid-wolf-coyotes-in-new-england/) - [Wolf
  paleogenomics](https://www.gnxp.com/WordPress/2022/06/29/wolf-paleogenomics/) - [The rise of the black
  wolf](https://www.gnxp.com/WordPress/2009/02/05/the-rise-of-the-black-wolf/) - [A map of charismatic canid genomic
  variation](https://www.gnxp.com/WordPress/2011/05/18/a-map-of-charismatic-canid-genomic-variation/) - [The western wolves are back, did you
  notice?](https://www.gnxp.com/WordPress/2018/03/19/the-western-wolves-are-back-did-you-notice/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F06%2F06%2Ffor-want-of-a-locus%2F&linkname=Endangered%20species%20act%202016%2C%20for%20want%20of%20a%20locus "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F06%2F06%2Ffor-want-of-a-locus%2F&linkname=Endangered%20species%20act%202016%2C%20for%20want%20of%20a%20locus "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2016%2F06%2F06%2Ffor-want-of-a-locus%2F&linkname=Endangered%20species%20act%202016%2C%20for%20want%20of%20a%20locus "Email")[](https://www.addtoany.com/share)
