+++
title = "One New World population"
full_title = "One New World population expansion"
date = "2009-04-29"
upstream_url = "https://www.gnxp.com/WordPress/2009/04/29/one-new-world-population-expansion/"

+++
Source: [here](https://www.gnxp.com/WordPress/2009/04/29/one-new-world-population-expansion/).

One New World population expansion

[Haplotypic Background of a Private Allele at High Frequency in the Americas](http://mbe.oxfordjournals.org/cgi/content/abstract/26/5/995):

> Recently, the observation of a high-frequency private allele, the > 9-repeat allele at microsatellite D9S1120, in all sampled Native > American and Western Beringian populations has been interpreted as > evidence that all modern Native Americans descend primarily from a > single founding population. However, this inference assumed that all > copies of the 9-repeat allele were identical by descent and that the > geographic distribution of this allele had not been influenced by > natural selection. To investigate whether these assumptions are > satisfied, we genotyped 34 single nucleotide polymorphisms across 500 > kilobases (kb) around D9S1120 in 21 Native American and Western > Beringian populations and 54 other worldwide populations. All > chromosomes with the 9-repeat allele share the same haplotypic > background in the vicinity of D9S1120, suggesting that all sampled > copies of the 9-repeat allele are identical by descent. Ninety-one > percent of these chromosomes share the same 76.26 kb haplotype, which > we call the “American Modal Haplotype” (AMH). Three observations lead > us to conclude that the high frequency and widespread distribution of > the 9-repeat allele are unlikely to be the result of positive > selection: 1) aside from its association with the 9-repeat allele, the > AMH does not have a high frequency in the Americas, 2) the AMH is not > unusually long for its frequency compared with other haplotypes in the > Americas, and 3) in Latin American mestizo populations, the proportion > of Native American ancestry at D9S1120 is not unusual compared with > that observed at other genomewide microsatellites. Using a new method > for estimating the time to the most recent common ancestor (MRCA) of > all sampled copies of an allele on the basis of an estimate of the > length of the genealogy descended from the MRCA, we calculate the mean > time to the MRCA of the 9-repeat allele to be between 7,325 and 39,900 > years, depending on the demographic model used. The results support > the hypothesis that all modern Native Americans and Western Beringians > trace a large portion of their ancestry to a single founding > population that may have been isolated from other Asian populations > prior to expanding into the Americas.

Some elaboration in [ScienceDaily](http://www.sciencedaily.com/releases/2009/04/090428223836.htm) of the logic:

> In these earlier studies, the researchers concluded that the most > straightforward explanation for the distribution of the 9-repeat > allele was that all modern Native Americans, Greenlanders and western > Beringians descend from a common founding population. Furthermore, the > fact that the allele was absent in other Asian populations most likely > meant that America’s ancestral founders had been isolated from the > rest of Asia for thousands of years before they moved into the New > World: that is, for a period of time that was long enough to allow the > allele to originate in, and spread throughout, the isolated > population.  
> As strong as this evidence was, however, it was not foolproof. There > were two other plausible explanations for the widespread distribution > of the allele in the Americas.  
> If the 9-repeat allele had arisen as a mutation multiple times, its > presence throughout the Americas would not indicate shared ancestry. > Alternatively, if there had been two or more different ancestral > founding groups and only one of them had carried the 9-repeat allele, > certain circumstances could have prompted it to cross into the other > groups and become widespread. Say that there was a second allele — one > situated very close to the 9-repeat allele on the DNA strand — that > conferred a strong advantage to humans who carried it. Natural > selection would carry this allele into new populations and because of > the mechanics of inheritance, long stretches of DNA surrounding it, > including the functionless 9-repeat allele, would be carried along > with the beneficial allele.  
> To rule out these possibilities, the research team, which was headed > by Noah Rosenberg at the University of Michigan, scrutinized DNA > samples of people from 31 modern-day Asian populations, 19 Native > American, one Greenlandic and two western Beringian populations.  
> **They found that in each sample that contained the 9-repeat allele, > short stretches of DNA on either side of it were characterized by a > distinct pattern of base pairs, a pattern they seldom observed in > people without the allele. “If natural selection had promoted the > spread of a neighboring advantageous allele, we would expect to see > longer stretches of DNA than this with a similarly distinct > pattern,”** Schroeder said. “And we would also have expected to see > the pattern in a high frequency even among people who do not carry the > 9-repeat allele. So we can now consider the positive selection > possibility unlikely.”

I’ve addressed the logic of why positive selection would look like this before. These data aren’t particular surprising or groundbreaking, but just increase confidence in a particular model of the settlement of the New World.

### Related Posts:

- [Natural selection and dopamine receptor
  genes](https://www.gnxp.com/WordPress/2012/03/09/natural-selection-and-dopamine-receptor-genes/) - [Selection or demography in differences between
  human…](https://www.gnxp.com/WordPress/2009/01/10/selection-or-demography-in-differences-between-human-populations/) - [Native American HLAs, part
  II](https://www.gnxp.com/WordPress/2006/04/26/native-american-hlas-part-ii/) - [ADHD &
  DRD4](https://www.gnxp.com/WordPress/2007/08/07/adhd-amp-drd4/) - [Functional Variant in a Bitter-Taste Receptor
  (hTAS2R16)…](https://www.gnxp.com/WordPress/2005/11/30/functional-variant-in-a-bitter-taste-receptor-htas2r16-influences-risk-of-alcohol-dependence/) - [Strange
  drifts](https://www.gnxp.com/WordPress/2006/02/08/strange-drifts/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F04%2F29%2Fone-new-world-population-expansion%2F&linkname=One%20New%20World%20population%20expansion "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F04%2F29%2Fone-new-world-population-expansion%2F&linkname=One%20New%20World%20population%20expansion "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F04%2F29%2Fone-new-world-population-expansion%2F&linkname=One%20New%20World%20population%20expansion "Email")[](https://www.addtoany.com/share)
