+++
title = "Selection for and"
full_title = "Selection for and against pigmentation alleles in South Asia"
date = "2019-04-06"
upstream_url = "https://www.gnxp.com/WordPress/2019/04/06/selection-for-and-against-pigmentation-alleles-in-south-asia/"

+++
Source: [here](https://www.gnxp.com/WordPress/2019/04/06/selection-for-and-against-pigmentation-alleles-in-south-asia/).

Selection for and against pigmentation alleles in South Asia

![Deepika Padukone](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/04/Deepika_Padukone_at_Garnier_event.jpg?resize=300%2C404&ssl=1)

Recently some British friends were asking about what we knew about South Asian historical genetics now. I explained that it does look like there was some migration in from the Central Asian steppe **and** West Asia [into South Asia during the Holocene](https://www.biorxiv.org/content/10.1101/292581v1). To which one friend responded, “that’s obvious though, many Indians look like brown white people.” Setting aside the semantic paradox (if you are brown, you are literally not white), it is clear what he is getting at: **due to shared ancestry the facial structure of many South Asians is not that different from West Eurasians**.

The Bollywood actress Deepika Padukone is an example of someone who is rather brown-skinned (naturally), but whose facial features are such that if she went with 100% skin-bleaching she would pass as white without too much trouble. For the purposes of this post, I Googled Indian albino…and came up with [this family](https://www.reddit.com/r/interestingasfuck/comments/6g43uy/three_generations_of_an_indian_family_with/). You can make your own judgments. I don’t know what to think of that!

The reason for this post is a newly [accepted paper](https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz037/5364274), [Ancestry-specific analyses reveal differential demographic histories and opposite selective pressures in modern South Asian populations](https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz037/5364274):

> Genetic variation in contemporary South Asian populations follows a > northwest to southeast decreasing cline of shared West Eurasian > ancestry. A growing body of ancient DNA evidence is being used to > build increasingly more realistic models of demographic changes in the > last few thousand years. Through high quality modern genomes, these > models can be tested for gene and genome level deviations. Using local > ancestry deconvolution and masking, we reconstructed population-specific surrogates of the two main ancestral components for more than 500 samples from 25 South Asian populations, and showed our approach to be robust via coalescent simulations. >
> Our f3 and f4 statistics based estimates reveal that the reconstructed > haplotypes are good proxies for the source populations that admixed in > the area and point to complex inter-population relationships within > the West Eurasian component, compatible with multiple waves of > arrival, as opposed to a simpler one wave scenario. Our approach also > provides reliable local haplotypes for future downstream analyses. > **As one such example, the local ancestry deconvolution in South > Asians reveals opposite selective pressures on two pigmentation genes (SLC45A2 and SLC24A5) that are common or fixed in West Eurasians, suggesting post-admixture purifying and positive selection signals, respectively.**

What they did methodologically is both clever and straightforward. Basically, they managed to assign blocks of the genome to Ancient Ancestral South Indian (AASI) ancestry, and Ancestral North Indian (ANI) ancestry, which is mostly West Eurasian. This is something you would naturally do. The problem is: can you do this well? Recombination breaks apart blocks of ancestry every generation. So doing this with black Americans, who have admixed for \>500 years, is relatively easy (long distinct blocks). Doing it for South Asians, whose ancestry has been recombining for at least 2,000 years, and probably more in man cases, is harder (short, less distinct blocks).

![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/04/trifur.jpg?resize=300%2C580&ssl=1)![](https://i0.wp.com/www.gnxp.com/WordPress/wp-content/uploads/2019/04/trifur.jpg?resize=300%2C580&ssl=1)The phylogenetic graph that they infer after constructing synthetic ancestral populations, Mask_S (\~AASI) and MASK_N (\~ANI), is interesting in one particular way: they found that the proto-South Asians diverged before the proto-East Asians and proto-Andamanese. All this happened \>40,000 years ago.

Other scholars who have looked at this see a trifurcation where the three groups diverge quickly. To a great extent, the same applies here, but they found some divergence of proto-South Asians (AASI) of Mask_S earlier. I’m not sure if this is real. It could just be errors in their masking process. Additionally, because of geography, the Andamanese may have exchanged more gene flow with the ancestors of modern Northeast Asians. I always like to point out that the Andamanese probably came from modern Burma, and so are the descendants of ***Southeast*** Asian Paleolithic people, not South Asian ones.

The researchers also found that the Mask_N population was complex in its history. We also know this, insofar as they’re probably a mix of West Asian agro-pastoralists and later migrants from the Central Asian steppe. These two groups are close to each other compared to indigenous South Asians, but they are still quite distinctive from each other.

**The really interesting stuff is selection.** I’ve always noticed that the derived, West Eurasian variant, of *SLC24A5*, was very high in South Asians, while the West Eurasian variant of *SLC45A2* seemed to be lower than expected (based on ANI/ASI ratios). These authors find results which suggest that strong positive selection increased the former, and purifying (negative) selection decreased the latter. The haplotype of *SLC24A5*is the same in South Asians, West Asians, and Europeans. It is also the same haplotype in “Scandinavian Hunter-Gatherers” from the Mesolithic, and “Caucasian Hunter-Gatherers” from the early Holocene/Late Pleistocene. This is an old variant and an old, but long, haplotype. It is strongly associated with skin depigmentation, but its rather high frequency in South Asia and Ethiopia suggests to me that its selected effect is not known (it is also being selected in southern Africa due to an introduction from West Eurasians via Nilotic peoples).

*SLC45A2* in the derived form, unlike, *SLC24A5*, is restricted to high frequency within Europe, and [even in Europeans it is not fixed](https://alfred.med.yale.edu/alfred/mvograph.asp?siteuid=SI003963V).

And here’s the non-pigmentation stuff:

> Interestingly, **we report how loci that include genes involved with > diabetes (SETD5), diet (ZNF) and the immune response (HLA) show West > Eurasian (N) haplotypes to be significantly more represented compared > to the South Asian (S) counterparts.** This might be a stark contrast > to what is expected, given the long term history of local adaptation > of S haplotypes in local environment. We speculate that the > diet-related signal may be linked with post-Neolithic dietary shifts > that might have followed the arrival of the West Eurasian component in > the area, while the overrepresentation of West Eurasian HLA haplotypes might have some similarity, although at a different time scale, with what has happened in Native American populations after recent colonization likely caused by European borne epidemic.

This is strongly suggestive of the fact that lifestyle and mode of production shifts occurred due to introductions from West Eurasia, rather than *in situ*.

There is [some evidence](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5182071/) for lots of selection in Europe in the past 2,000 years. Ancient DNA tells us Europeans have been changing in pigmentation, and the frequency of lactase persistence alleles, very recently.

Phylogenetics will eventually be solved. But selection will the big and difficult task in the future.

### Related Posts:

- [Brown people are all the same
  (perhaps)](https://www.gnxp.com/WordPress/2006/12/22/brown-people-are-all-the-same-perhaps/) - [Peeling the population genetic Indian
  onion](https://www.gnxp.com/WordPress/2011/12/08/peeling-the-population-genetic-indian-onion/) - [How Indians are a lot like Latin
  Americans](https://www.gnxp.com/WordPress/2017/03/28/how-indians-are-a-lot-like-latin-americans/) - [David Reich interview on
  Edge](https://www.gnxp.com/WordPress/2016/02/23/david-reich-interview-on-edge/) - [Indian ancestry in Southeast Asia is older than
  statistical…](https://www.gnxp.com/WordPress/2020/12/29/indian-ancestry-in-southeast-asia-is-older-than-statistical-genetic-tests-suggest/) - [15 ancestral components to bind them
  all](https://www.gnxp.com/WordPress/2010/11/17/15-ancestral-components-to-bind-them-all/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F04%2F06%2Fselection-for-and-against-pigmentation-alleles-in-south-asia%2F&linkname=Selection%20for%20and%20against%20pigmentation%20alleles%20in%20South%20Asia "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F04%2F06%2Fselection-for-and-against-pigmentation-alleles-in-south-asia%2F&linkname=Selection%20for%20and%20against%20pigmentation%20alleles%20in%20South%20Asia "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2019%2F04%2F06%2Fselection-for-and-against-pigmentation-alleles-in-south-asia%2F&linkname=Selection%20for%20and%20against%20pigmentation%20alleles%20in%20South%20Asia "Email")[](https://www.addtoany.com/share)
