+++
title = "Standing variation,"
full_title = "Standing variation, polygenic traits, & soft selection"
date = "2014-09-21"
upstream_url = "https://www.gnxp.com/WordPress/2014/09/21/standing-variation-polygenic-traits-and-soft-selection/"

+++
Source: [here](https://www.gnxp.com/WordPress/2014/09/21/standing-variation-polygenic-traits-and-soft-selection/).

Standing variation, polygenic traits, & soft selection

![Brawand, David, et al. “The genomic substrate for adaptive radiation in African cichlid fish.” Nature (2014).](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/09/nature13726-f1.jpg?resize=600%2C571)

A [new paper](http://www.nature.com/nature/journal/v513/n7518/full/nature13726.html) on [cichlid](https://en.wikipedia.org/wiki/Cichlid) evolutionary genetics in *Nature* is pretty interesting. It’s open access, so everyone can read it, [The genomic substrate for adaptive radiation in African cichlid fish](http://www.nature.com/nature/journal/v513/n7518/full/nature13726.html):

> Cichlid fishes are famous for large, diverse and replicated adaptive > radiations in the Great Lakes of East Africa. To understand the > molecular mechanisms underlying cichlid phenotypic diversity, we > sequenced the genomes and transcriptomes of five lineages of African > cichlids: the Nile tilapia (*Oreochromis niloticus*), an ancestral > lineage with low diversity; and four members of the East African > lineage:*Neolamprologus brichardi/pulcher*(older radiation, Lake > Tanganyika),*Metriaclima zebra*(recent radiation, Lake > Malawi),*Pundamilia nyererei*(very recent radiation, Lake Victoria), > and*Astatotilapia burtoni*(riverine species around Lake Tanganyika). > We found an excess of gene duplications in the East African lineage > compared to tilapia and other teleosts, an abundance of non-coding > element divergence, accelerated coding sequence evolution, expression > divergence associated with transposable element insertions, and > regulation by novel microRNAs. In addition, we analysed sequence data > from sixty individuals representing six closely related species from > Lake Victoria, and show genome-wide diversifying selection on coding > and regulatory variants, some of which were recruited from ancient > polymorphisms. **We conclude that a number of molecular mechanisms shaped East African cichlid genomes, and that amassing of standing variation during periods of relaxed purifying selection may have been important in facilitating subsequent evolutionary diversification.**

Reading a paper like this makes it very clear to me why **organismic information** is very critical in trying to understand evolutionary processes. [R. A. Fisher](https://en.wikipedia.org/wiki/Ronald_Fisher) was a great scientist, but his attempt to create very general rules for evolutionary processes as outlined in [The Genetical Theory of Natural Selection](https://www.amazon.com/exec/obidos/ASIN/0198504403/geneexpressio-20) seems quixotic in hindsight. As a [professor of mine](https://twitter.com/wcresko) once said “biology is the science of exceptions.” This is probably one reason that the old 19th century vogue for creating “laws” went into decline; laws are useful only when the exceptions to them are very few. The diversity and specificity of evolutionary biological processes is why I think arguments in the form of dichotomies are in vain. For example, “selectionists vs. neutralists” or those who argue for the primacy of contingency in evolutionary outcomes, or those who favor convergence toward deterministic outcomes. It strikes me that the answer to all these questions must be predicated by the phrase “it depends….”

![electr10](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/09/electr10-300x199.jpg?resize=300%2C199)![electr10](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/09/electr10-300x199.jpg?resize=300%2C199)The story of African cichlids is relatively well known. You have a huge range of phenotypic outcomes which seem to be the result of relatively recent diversification. Not only that, **but cichlids *look cool*.** The question though is what can this tell us about evolution? The paper does a lot of genomic slicing and dicing of a few select lineages. Before we knew about genes the nature of transmission in evolution was something of a black-box. With DNA that changed, and now with genomics we can look at how various features differ across lineages and over time. For example copy number variation, coding regions, regulatory elements, and characters more abstruse to those outside of genomics, such as LINEs. Sequencing a bunch of cichlids gets you a publication in *Nature*, but that’s not the point of it all. The point is what does it tell you about evolution, at least in this case?

**Basically it seems that cichlid radiation has occurred due to selection on the natural genetic variation across many loci that already existed in the ancestral lineage.** It was easy enough to type, but in some ways this reality is a bit difficult, in that classical models of selection were often predicated on new mutations emerging in the genetic substrate, and rapid sweeps to fixation from the original mutant form. A human illustration of this is lactase persistence, which seems to be due to a change in gene regulation via a new mutation that arose \~5,000 years ago, and quick swept to near fixation across many diverse human lineages. This sort of phenomenon is like striking a hammer at the genome; it leaves a big mark which is easy to detect. In contrast to hard selection, soft selection on standing variation operates across many loci, and rather than a novel genotype in a singular sense produces a change in underlying allele frequencies. In some ways this is more classical Darwinian, but it also generates more work and is not as elegant as a simple model of a hard sweep from a new mutation. But that’s just how it is. At least in the cichlid lineage.

[![41SWwcvSm6L.\_SX258_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/09/41SWwcvSm6L._SX258_BO1204203200_.jpg?resize=260%2C343)![41SWwcvSm6L.\_SX258_BO1,204,203,200\_](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/09/41SWwcvSm6L._SX258_BO1204203200_.jpg?resize=260%2C343)](https://www.amazon.com/exec/obidos/ASIN/0878934812/geneexpressio-20)Actually, I suspect that’s how it is in many complex organisms. For a human illustration, see [The Genetics of Human Adaptation: Hard Sweeps, Soft Sweeps, and Polygenic Adaptation](http://www.sciencedirect.com/science/article/pii/S0960982209020703). I’m betting that it’s also the norm among our many domestic animals. For example [rabbits](http://good-lab.dbs.umt.edu/Good_Lab/Publications_files/Carniero_etal_Science_2014.pdf). A few years ago at a conference [Claire Wade](http://sydney.edu.au/vetscience/about/staff/profiles/claire.wade.php) reported that the focus on traits and genes with disjoint distributions across dog breeds had obscured the reality that much of the variation in this lineage is still shared, and there is probably a lot of soft selection on polygenic traits going on.

So the near future is in quantitative traits, and natural selection reshaping the variation which is already present in lineages. Perhaps people should start re-reading [Genetics and Analysis of Quantitative Traits](https://www.amazon.com/exec/obidos/ASIN/0878934812/geneexpressio-20). And as a weird side effect, if I was interested in such things I bet that the emphasis on selection from standing variation probably supports philosophical monism. All is one. Or something.

### Related Posts:

- [African
  mega-drought?](https://www.gnxp.com/WordPress/2007/10/09/african-mega-drought/) - [Vitamin D protective against
  radiation?](https://www.gnxp.com/WordPress/2008/11/09/vitamin-d-protective-against-radiation/) - [The end of phylogenetic
  controversies](https://www.gnxp.com/WordPress/2012/10/07/the-end-of-phylogenetic-controversies/) - [Selection for pigmentation in
  Khoisan?](https://www.gnxp.com/WordPress/2017/09/28/selection-for-pigmentation-in-khoisan/) - [Beautiful butterflies & localized
  adaptation](https://www.gnxp.com/WordPress/2010/02/06/beautiful-butterflies-localized-adaptation/) - [Out of Africa model most
  likely](https://www.gnxp.com/WordPress/2007/10/31/out-of-africa-model-most-likely/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F09%2F21%2Fstanding-variation-polygenic-traits-and-soft-selection%2F&linkname=Standing%20variation%2C%20polygenic%20traits%2C%20%26%20soft%20selection "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F09%2F21%2Fstanding-variation-polygenic-traits-and-soft-selection%2F&linkname=Standing%20variation%2C%20polygenic%20traits%2C%20%26%20soft%20selection "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F09%2F21%2Fstanding-variation-polygenic-traits-and-soft-selection%2F&linkname=Standing%20variation%2C%20polygenic%20traits%2C%20%26%20soft%20selection "Email")[](https://www.addtoany.com/share)
