+++
title = "Size matters for"
full_title = "Size matters for inbreeding"
date = "2015-11-09"
upstream_url = "https://www.gnxp.com/WordPress/2015/11/09/size-population-matters-for-inbreeding/"

+++
Source: [here](https://www.gnxp.com/WordPress/2015/11/09/size-population-matters-for-inbreeding/).

Size (population) matters for inbreeding

[![440px-Daphnia_pulex](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/11/440px-Daphnia_pulex-262x300.png?resize=172%2C197)![440px-Daphnia_pulex](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/11/440px-Daphnia_pulex-262x300.png?resize=172%2C197)](https://en.wikipedia.org/wiki/Daphnia#/media/File:Daphnia_pulex.png)Yesterday I read a paper which utilized [*Daphnia*](https://en.wikipedia.org/wiki/Daphnia) as a model to explore a very important theoretical question, which relates the role of [effective population size](http://www.nature.com/scitable/topicpage/genetic-drift-and-effective-population-size-772523) to the [genetic load](https://en.wikipedia.org/wiki/Genetic_load) (they’re inversely correlated). The theoretical aspect I am aware of, but I don’t know much about *Daphnia*. The paper is titled [Genetic load, inbreeding depression and hybrid vigor covary with population size: an empirical evaluation of theoretical predictions](http://onlinelibrary.wiley.com/doi/10.1111/evo.12802/abstract), and it’s in *Evolution*. It’s not open access, and I can’t find a preprint around, for which I apologize (you could pester the first author on ResearchGate or something). But onereason I’m interested is that they assert this:

> Our results are in clear support of theoretical models based on > recurrent mutation to unconditionally deleterious alleles on the > effects of population size on inbreeding depression, hybrid vigor, and > genetic load. **This study is the first to find such clear and > unequivocal evidence for all of the predicted effects.**

This makes me think of Richard Lewontin’s assertion back in the 1970s that theoretical population genetics was basically a machine designed to operating upon inputs which weren’t available (data). I don’t know this literature well, but it’s shocking that these ideas have only been robustly tested now! Or, perhaps these results are false positives of some sort, as it does note it’s the first to find clear and unequivocal evidence for a prediction.

The basic issue is that in small populations **genetic drift has the potential to overwhelm the power of selection in purging deleterious alleles.** How deleterious an allele is varies. Some alleles have very strong negative selection coefficients. For example, those with dominant lethal effects are going to be purged immediately for obvious reasons (if it’s dominant, it’s always expressed, and if it’s lethal, it isn’t passed on). The situation differs for those with recessive expression patterns. Even if it is lethal in homozygous form, an allele can persist at low frequency if the population is random mating, as the vast majority of copies will be in heterozygotes whose fitness is not impinged. But if the selection coefficient is low enough than even dominantly expressed alleles may not be purged. The variance in allele frequencies due to sampling is inversely proportional to the population size, so as that converges upon the selection coefficient in terms of magnitude, the efficacy of natural selection diminishes. This is at the heart of the [nearly neutral theory](https://en.wikipedia.org/wiki/Nearly_neutral_theory_of_molecular_evolution), which suggests that a lot of variation is due to the input of very weakly deleterious alleles which can’t be purged in population sizes where drift is above a particular threshold.

[![originsgenomearchitecture](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/11/originsgenomearchitecture.png?resize=234%2C284)![originsgenomearchitecture](https://i0.wp.com/www.unz.com/wp-content/uploads/2015/11/originsgenomearchitecture.png?resize=234%2C284)](https://www.amazon.com/exec/obidos/ASIN/0878934847/geneexpressio-20)Presumably, in large populations there will be many low frequency variants of weak deleterious effect and recessive expression. In contrast, in small populations the power of drift is such that even rather deleterious alleles can be fixed against the gradient of selection. At cross-purposes with this is the idea that because inbreeding populations tend to “expose” alleles which express recessively to selection they can “purge” the genetic load which drags on fitness. For example, with dog breeds there is some evidence that inbreeding needs to be conditional upon breed level variation, as some of the load may have been purged.

Apparently *Daphnia* are a species which exhibit a wide gradient of variation in genetic diversity (heterozygosity in this case), allowing one to test various hypotheses by crossing lineages sampled from wild populations in the laboratory. Their molecular assay of diversity were \~30 microsatellite loci. What they found is that in line with theoretical prediction those sampled from large populations had lots of segregating deleterious alleles, which manifested in strong inbreeding effect when individuals were purposely crossed with those genetically similar. In contrast, those from small populations did not exhibit so much inbreeding effect, indicating that a lot of the deleterious alleles were already fixed and so exposed. These individuals from small populations also exhibited lower fitness than those from large populations, reflecting in all likelihood their genetic load. Crossing individuals from different small populations resulted in immediately hybrid vigor, as the fixed variants differed across lineages.

There are a lot more details in the paper. If you have academic access, read the whole thing. If not, there’s always [\#icanhazpdf](https://twitter.com/search?q=%23icanhazpdf). I’m more interested in **general** conclusions. [Two](http://biorxiv.org/content/early/2015/10/31/030387) [preprints](http://biorxiv.org/content/early/2015/10/30/030148) just came out which addressed the reality that Neanderthals seem to have had a small effective population size. Meanwhile, the issue is very real and live in conservationgenetics, and even in the understanding of mammalian lineages more broadly, many of which have gone through bottlenecks even human intervention aside. But how much can we generalize from the [*Daphnia*](http://www.starklab.org/2011/02/04/First-crustacean-genome-sequenced%3A-The-water-flea-Daphnia-pulex/), which has a small genome (\<10% of the size of the human genome, which is around average for mammals), but \~1/3 more genes? I’d wager a lot. But I’m really going to be interested when there are whole-genome analyses of this sort of study done in *Daphnia*, and we can look at the site frequency spectrum, instead of just inferring from the fitness.

Finally, I do want to emphasize here a lot of the problems relating to inbreeding seem to be due to segregation load of partially recessive low frequency variants. This is an important foundational insight that allows us to properly conceptualize what’s happening in small populations, or in lineages that have gone through a bottleneck, and why that’s a problem.

### Related Posts:

- [The fault in our
  parameters](https://www.gnxp.com/WordPress/2018/06/21/the-fault-in-our-parameters/) - [Selection and load (through the
  bottleneck)](https://www.gnxp.com/WordPress/2014/11/03/selection-and-load-through-the-bottleneck/) - [Targaryen genetic
  load](https://www.gnxp.com/WordPress/2012/03/27/targaryen-genetic-load/) - [Inbreeding and
  homozygosity](https://www.gnxp.com/WordPress/2006/03/21/inbreeding-and-homozygosity/) - [Fitness is still a
  bugger!](https://www.gnxp.com/WordPress/2015/02/01/fitness-is-still-a-bugger/) - [I say inbreeding depression, you say
  heterosis](https://www.gnxp.com/WordPress/2007/01/25/i-say-inbreeding-depression-you-say-heterosis/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F11%2F09%2Fsize-population-matters-for-inbreeding%2F&linkname=Size%20%28population%29%20matters%20for%20inbreeding "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F11%2F09%2Fsize-population-matters-for-inbreeding%2F&linkname=Size%20%28population%29%20matters%20for%20inbreeding "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2015%2F11%2F09%2Fsize-population-matters-for-inbreeding%2F&linkname=Size%20%28population%29%20matters%20for%20inbreeding "Email")[](https://www.addtoany.com/share)
