+++
title = "Kibra up in the air"
full_title = "Kibra up in the air"
date = "2008-01-25"
upstream_url = "https://www.gnxp.com/WordPress/2008/01/25/kibra-up-in-the-air/"

+++
Source: [here](https://www.gnxp.com/WordPress/2008/01/25/kibra-up-in-the-air/).

Kibra up in the air

The first genome-wide association study on human episodic memory back in 2006 showed an association between the T allele of a gene called KIBRA and better performance on certain list-learning tasks. That study contained two replications in different populations, and the outcome was independently replicated in healthy, elderly folks. Next, another group showed an association between the T allele and very late-onset Alzheimer’s. There are some issues with interpreting this study that I certainly didn’t think of the first time I read it. Almeida et al. point out that this could be due to ‘survivorship bias’ wherein the C allele carriers that were gonna get AD got it a lot earlier and left the T allele folks to provide the ‘very late-onset’ crowd (or at least that’s how I interpret survivorship bias).

Two studies have come out in the past few months. [One replicates the effect](http://www.blackwell-synergy.com/doi/abs/10.1111/j.1582-4934.2008.00229.x?prevSearch=allfield%3A%28kibra%29) of the T allele on memory with a little smaller effect size than before. [The second fails to find any effect at all](http://www3.interscience.wiley.com/cgi-bin/fulltext/117889625/HTMLSTART). One experiment in this latter report was an exact replication of the 2006 memory study with a population of European origin (German vs. Swiss. That shouldn’t matter should it?).I don’t know how to explain the failure to replicate, but it is duly noted. Perhaps it really really matters how well you vet your cohort. For instance (from Almeida and co again):

> We did not find evidence in support of our original hypothesis that CC > carriers would be at greater risk of MCI (ed: Mild Cognitive > Impairment) (although we did observe a trend in that direction), nor > were we able to show any evidence of an effect of the gene on the > memory scores of older people with MCI. These results suggest that the > effect size of the Tâ†’C polymorphism decreases with increasing impairment of episodic memory, and that the KIBRA gene plays all but a limited role after scoresfall below a certain threshold, as is the case in MCI.

I don’t think there is any evidence that the cohort that failed to replicate had especially bad memory, but I’m not an expert in human memory assessment. A few more molecular details below:

Kibra had an especially good tie-in to memory because in yeast two-hybrid studies it binds to PKM zeta which is established as a key player in maintenance of several types of memory and synaptic plasticity in a completely separate literature. The molecular situation is foggy as well though. We don’t have any published assessment of the function or localization of endogenous Kibra protein in neurons. In fact, most of the molecular work has been done with an overexpressed GFP fusion protein. The group that discovered Kibra reports that it is a 125-kDa protein with specialized “WW” protein interaction domains at one end, while the group that reported the Kibra-memory association used a custom antibody to detect human Kibra protein and identified a 100-kDa truncated protein. One final issue is that the memory-SNP (and all SNPs in linkage disequilibrium) in human Kibra is intronic, which means we have no straightforward prediction as to how it might alter protein function. Papassotiropoulos et al.(2006) could not find a difference in the total amount of Kibra protein in human brain tissue with different alleles. Either we have to predict that the SNP produces an expression change that they couldn’t detect or that the SNP alters splicing such that the protein sequence changes but the size doesn’t.

### Related Posts:

- [The cost of
  memory?](https://www.gnxp.com/WordPress/2007/09/03/the-cost-of-memory/) - [Memory-associated
  SNP](https://www.gnxp.com/WordPress/2006/10/19/memory-associated-snp/) - [Asia finally getting in on the genome-wide association
  game?](https://www.gnxp.com/WordPress/2008/08/18/asia-finally-getting-in-on-the-genome-wide-association-game/) - [APOE4: a blessing and a
  curse](https://www.gnxp.com/WordPress/2007/01/13/apoe4-a-blessing-and-a-curse/) - [Reporting genome-wide association
  studies](https://www.gnxp.com/WordPress/2007/07/22/reporting-genome-wide-association-studies/) - [Genome-wide association for
  newbies](https://www.gnxp.com/WordPress/2010/07/19/genome-wide-association-for-newbies/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F01%2F25%2Fkibra-up-in-the-air%2F&linkname=Kibra%20up%20in%20the%20air "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F01%2F25%2Fkibra-up-in-the-air%2F&linkname=Kibra%20up%20in%20the%20air "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F01%2F25%2Fkibra-up-in-the-air%2F&linkname=Kibra%20up%20in%20the%20air "Email")[](https://www.addtoany.com/share)
