+++
title = "Selection in structured"
full_title = "Selection in structured populations"
date = "2008-01-03"
upstream_url = "https://www.gnxp.com/WordPress/2008/01/03/selection-in-structured-populations/"

+++
Source: [here](https://www.gnxp.com/WordPress/2008/01/03/selection-in-structured-populations/).

Selection in structured populations

[![Blogging on Peer-Reviewed Research](https://i0.wp.com/bpr3.org/images/rbicons/ResearchBlogging-Large-Trans.png?resize=120%2C90)![Blogging on Peer-Reviewed Research](https://i0.wp.com/bpr3.org/images/rbicons/ResearchBlogging-Large-Trans.png?resize=120%2C90)](http://bpr3.org/?p=52)Evolutionary genetics is subject to parameters; forces which pull and push and shape the nature of dynamic processes over time and space. Population size, mtuation rate, migration, selection. etc., these are all parameters we have to keep in mind when attempting to analyze the nature of evolutionary dynamics. The acceleration paper was predicated on consequences of changing on parameter, population size, upon other parameters such as selection, drift and the number of mutations. Of course I have wondered about the nature of population substructure and what it means for our species. My thinking has been influenced by these maps, which show sharp geographic discontinuities in the frequencies of recently selected alleles. Quotes like this make me wonder: **“As long as *s* \< *m*, the new allele will migrate out of its original deme before it reaches fixation there,”** where *s* = selection coefficient, and *m* is the migration rate (specifically, the probability that an individual in generation *t* in deme *x* was a member of deme *!x* during generation *t* – 1). Selection coefficients on the order of 0.1 are enormous, those on the other of 0.01 are substantial. A migration probability for an individual on the order of 0.1 within a deme also seems very high. My own intuition though is that the distribution of *m* exhibits both a greater mean and variance than *s*.

  
[Fixation Probability and Time in Subdivided Populations](http://www.genetics.org/cgi/content/abstract/164/2/767) is the first of several papers I’ve been reading to obtain a greater insight on population substructure and how it might have played out in the evolutionary history of our species. From the abstract:

> …Population structure changes the effective size of the species, often > strongly downward; smaller effective size increases the probability of > fixing deleterious alleles and decreases the probability of fixing > beneficial alleles. On the other hand, population structure causes an > increase in the homozygosity of alleles, which increases the > probability of fixing beneficial alleles but somewhat decreases the > probability of fixing deleterious alleles. **The probability of fixing > new beneficial alleles can be simply described by 2*hs*(1 – > *F*_(ST))*N*_(e)/*N*_(tot),** where *hs* is the change in fitness of > heterozygotes relative to the ancestral homozygote, *F*_(ST) is a > weighted version of Wright’s measure of population subdivision, and > *N*_(e) and *N*_(tot) are the effective and census sizes, respectively. These results are verified by simulation for a broad range of population structures, including the island model, the stepping-stone model, and a model with extinction and recolonization.

This is a technical paper, with diffusion equations and integration and simulations of stepping-stone and island models. I’ll elide over the details, but there are a few general issues that need to be noted. The work in this paper extends and adds granularity to the famous [2*s*](https://www.google.com/search?q=2s&btnG=Search!&domains=scienceblogs.com%2Fgnxp&sitesearch=scienceblogs.com%2Fgnxp) which I had spoken of before, the probability of fixation of a selectively advantageous allele within a new population. In short, if an allele confers a selection coefficience of 0.1, 10% increased fitness above the population mean, then it has a 0.2 probability of fixation. This is within a very large, operationally infinite, population. Why only 0.2 when the allele is favored? [Stochastic factors](https://www.google.com/search?hl=en&safe=off&domains=scienceblogs.com%2Fgnxp&q=gillespie+drift&btnG=Search&sitesearch=scienceblogs.com%2Fgnxp) are strong when an allele is at low frequencies, basically present in only a few copies. There is reproductive variance for any given individual (usually assumed to be Poisson distributed), and that variance is not perfectly correlated with an idealized genetic fitness. If a random “Act of God” exterminates a clutch with a very beneficial allele, then so be it.  
In terms of the formalism, *N_(e)* & *N_(tot)* are the effective and total population. Effective basically refers to the fact that not all individuals contribute to the next generation, or contribute to the same extent. Random reproductive variance will always result in a lower *N_(e)* than *N_(tot)*. *F_(ST)* is basically a measure of the [between and within population](https://en.wikipedia.org/wiki/Fixation_index) genetic variation. If most of the variation is partitioned between populations then *F_(ST)* is high, and approaches 1, but if most of it is extant within populations, it approaches 0 (*F_(ST)* across “races” is famous on the order of 0.15, so that 85% of the variance on a single locus is present within a race). *h* is measuring the extent of dominance, if the heterozygote is in between the two homozygotes it is 1/2, while if there is perfect dominant it is 1.  
There are a few general results from the analyses and simulations within this paper. A great deal of substructure can result in a reduced power of selection to drive beneficial alleles to fixation within and across demes. It can also result in the fixation of deleterious alleles due to drift. Finally, it can also favor the fixation of recessive alleles which are beneficial as well, because they are more likely to be expressed as homozygotes within a small relatively inbred population (drift kicks their frequency up just high enough so that many more copies are expressed to selection).  
![whitlock05fig2.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/whitlock05fig2.jpg?resize=300%2C327)![whitlock05fig2.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/whitlock05fig2.jpg?resize=300%2C327)Here’s a figure from the paper. By “hard selection,” the authors mean that the nature of the genotypes within the deme are relevant toward their replication in the next generation (“soft selection” seems to refer more to events which result in variant success across demes uncorrelated with their genotypes, so differences in allele frequencies are driven simply by initial differences across demes, captures by *F_(ST)*). Note that the selection coefficient, 0.001, is rather modest. You see that as the rate of migration increases the probability of fixation quickly converges upon 0.002, what one would predict from 2*s*. This is simply because migration increases the effective population size as the demes are being connected together into a larger breeding metapopulation.  
![whitlock05fig5.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/whitlock05fig5.jpg?resize=200%2C194)![whitlock05fig5.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/whitlock05fig5.jpg?resize=200%2C194)This is from figure 5, and it shows the trend for a recessive allele which is only expressed as a homozygote, with an *s* of 0.002. Note that as the migration rate increases it becomes less and less able to manifest its advantange. This is because the effective population size is increasing and within a Hard-Weinberg Equilibrium its value as *q*² is getting smaller and smaller.  
The whole paper is open access, so I encourage you to read it. At this point, the question I’m asking myself is this: effective population seems to have been increasing over time, **but how has population substructure changed, if at all, over human history?** Would small isolated groups such as the Andaman Islanders exhibit a non-trivial number of beneficial recessively expressed alleles because of their insulation from migration? As history progressed and migration increased have alleles increased in their chance of fixation in part because of the breakdown of population structure? Was the *2s* limit reached relatively early on in the course of our history so that as R.A. Fisher might contend we can ignore substructure? And how might [Shifting Balance](https://www.gnxp.com/MT2/archives/000629.html) dynamics play into this?  
**Reference:** [Fixation Probability and Time in Subdivided Populations](https://www.gnxp.com/MT2/archives/000629.html), Michael C. Whitlock, Genetics **164**: 767-779 ( June 2003)

### Related Posts:

- [10 assertions about
  evolution](https://www.gnxp.com/WordPress/2006/08/14/10-assertions-about-evolution/) - [Brain size ∝ rate of
  evolution?](https://www.gnxp.com/WordPress/2008/08/15/brain-size-rate-of-evolution/) - [Hold one hand still, move the
  other](https://www.gnxp.com/WordPress/2006/03/03/hold-one-hand-still-move-the-other/) - [Notes on Sewall Wright:
  Migration](https://www.gnxp.com/WordPress/2008/07/03/notes-on-sewall-wright-migration/) - [Stochastic training
  wheels](https://www.gnxp.com/WordPress/2007/02/06/stochastic-training-wheels/) - [Sexual selection @
  Chernobyl](https://www.gnxp.com/WordPress/2007/07/12/sexual-selection-chernobyl/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F01%2F03%2Fselection-in-structured-populations%2F&linkname=Selection%20in%20structured%20populations "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F01%2F03%2Fselection-in-structured-populations%2F&linkname=Selection%20in%20structured%20populations "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2008%2F01%2F03%2Fselection-in-structured-populations%2F&linkname=Selection%20in%20structured%20populations "Email")[](https://www.addtoany.com/share)
