+++
title = "PLoS Roundup Dec 15"
full_title = "PLoS Roundup Dec 15"
date = "2005-12-15"
upstream_url = "https://www.gnxp.com/WordPress/2005/12/15/plos-roundup-dec-15/"

+++
Source: [here](https://www.gnxp.com/WordPress/2005/12/15/plos-roundup-dec-15/).

PLoS Roundup Dec 15

Let’s work on knocking McConaughey off the main page. Here’s a quick summary of new stuff that just came out in PLoS journals. The full texts of each article are freely available.

1\. [Ablation of the Sam68 RNA Binding Protein Protects Mice from Age-Related Bone Loss](https://dx.doi.org/10.1371%2Fjournal.pgen.0010074.eor).

> Osteoporosis is a debilitating bone disease that is characterized by > reduced bone mass and microarchitectural damage, which result in > increased bone fragility and susceptibility to fracture. Peak bone > mass, which is achieved by the age of 30 in humans, has been > identified as a major determinant of resistance or susceptibility to > osteoporosis. The authors generated mice deficient for the Sam68 RNA > binding protein, a protein of unknown physiologic function. The mice > develop normally and are protected against bone loss during aging. > Age-related bone loss has long been associated with an increase in > marrow adipocytes, which are derived from the same mesenchymal lineage > as osteoblasts in bone marrow. The authors showed that Sam68 regulates > the differentiation of this mesenchymal lineage, such that in its > absence, osteoblasts continued to be generated in aging bone, leading > to preservation of bone mass. This study identifies a physiologic role for Sam68 as a modulator of the bone marrow stem cell niche and hence of bone metabolism. The data identify Sam68 as a potential therapeutic target for the prevention and treatment of age-related bone loss.

Intersting bit in the disucssion about how heterozygous mothers kill 2/3 of their homozygous knock out pups.

2\. [The Flavoring Agent Dihydrocoumarin Reverses Epigenetic Silencing and Inhibits Sirtuin Deacetylases](http://genetics.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pgen.0010077).

> Sirtuins are a family of phylogenetically conserved nicotinamide > adenine dinucleotide-dependent deacetylases that have a firmly > established role in aging. Using a simple Saccharomyces cerevisiae > yeast heterochromatic derepression assay, we tested a number of > environmental chemicals to address the possibility that humans are > exposed to sirtuin inhibitors. Here we show that dihydrocoumarin > (DHC), a compound found in Melilotus officinalis (sweet clover) that > is commonly added to food and cosmetics, disrupted heterochromatic > silencing and inhibited yeast Sir2p as well as human SIRT1 deacetylase > activity. DHC exposure in the human TK6 lymphoblastoid cell line also > caused concentration-dependent increases in p53 acetylation and > cytotoxicity. Flow cytometric analysis to detect annexin V binding to > phosphatidylserine demonstrated that DHC increased apoptosis more than > 3-fold over controls. Thus, DHC inhibits both yeast Sir2p and human > SIRT1 deacetylases and increases p53 acetylation and apoptosis, a phenotype associated with senescence and aging. These findings demonstrate that humans are potentially exposed to epigenetic toxicants that inhibit sirtuin deacetylases.

Yeah… so that sucks. We don’t know and probably can’t find out easily whether this stuff accelerates aging, but I would prefer not to eat it. My guess is that DHC will probably just be labeled “natural flavors” in most foods.

3\. [Genome-Wide Associations of Gene Expression Variation in Humans](http://genetics.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pgen.0010078).

> With the finished reference sequence of the human genome now > available, focus has shifted towards trying to identify all of the > functional elements within the sequence. Although quite a lot of > progress has been made towards identifying some classes of genomic > elements, in particular protein-coding sequences, the characterization > of regulatory elements remains a challenge. The authors describe the > genetic mapping of regions of the genome that have functional effects > on quantitative levels of gene expression. Gene expression of 630 > genes was measured in cell lines derived from 60 unrelated human > individuals, the same Utah residents of Northern and Western European > ancestry that have been genetically well-characterized by The > International HapMap Project. This paper reports significant variation > among individuals with respect to levels of gene expression, and > demonstrates that this quantitative trait has a genetic basis. For > some genes, the genetic signal was localized to specific locations in > the human genome sequence; in most cases the genomic region associated > with expression variation was physically close to the gene whose > expression it regulated. The authors demonstrate the feasibility of performing whole-genome association scans to map quantitative traits, and highlight statistical issues that are increasingly important for whole-genome disease mapping studies.

mRNA levels are a rich phenotype for doing association mapping because of microarray technologies. One problem is the question of what tissue to use. Blood is the easiest to acquire from large numbers of people, but of course that misses a lot of details.

4\. [Selective Constraint on Noncoding Regions of Hominid Genomes](https://dx.doi.org/10.1371%2Fjournal.pcbi.0010073.eor).

> A major goal of human evolutionary biology is to understand what > genetic changes make humans unique. One influential idea is that > changes in gene expression are most responsible for unique human > characteristics. Regulatory elements in noncoding DNA play a key role > in controlling gene expression, so one approach is to study > humanÃ‚Â–chimpanzee differences in these elements. Here we use > conservation in more distantly related mammals and amniotes as a way > of identifying small sequence windows that are likely to be > functional. We find that putatively functional noncoding elements defined in this manner are subject to significant selective constraint in hominids. Contrary to some previous reports, these results argue that hominid noncoding regions are not evolving free of constraint.

This is in reference to previous reports that were less carefully done that found less conservation in human and chimp promoters than other mammals. That paper looked at large blocks of DNA, which are probably not functionally relevant. This paper looks at smaller blocks.

### Related Posts:

- [Super-wolf](https://www.gnxp.com/WordPress/2009/05/03/super-wolf/)
- [Bone mutants and recent
  selection](https://www.gnxp.com/WordPress/2009/04/27/bone-mutants-and-recent-selection/) - [Human CCR5
  knockout](https://www.gnxp.com/WordPress/2009/02/11/human-ccr5-knockout/) - [Vinay's bone marrow
  match](https://www.gnxp.com/WordPress/2007/11/13/vinays-bone-marrow-match/) - [FTO, weight, and the
  mouse](https://www.gnxp.com/WordPress/2009/08/16/fto-weight-and-the-mouse/) - [Metamorphosis](https://www.gnxp.com/WordPress/2008/07/02/metamorphosis/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F12%2F15%2Fplos-roundup-dec-15%2F&linkname=PLoS%20Roundup%20Dec%2015 "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F12%2F15%2Fplos-roundup-dec-15%2F&linkname=PLoS%20Roundup%20Dec%2015 "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2005%2F12%2F15%2Fplos-roundup-dec-15%2F&linkname=PLoS%20Roundup%20Dec%2015 "Email")[](https://www.addtoany.com/share)
