+++
title = "Transgenic methods in"
full_title = "Transgenic methods in mice"
date = "2007-02-09"
upstream_url = "https://www.gnxp.com/WordPress/2007/02/09/transgenic-methods-in-mice/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/02/09/transgenic-methods-in-mice/).

Transgenic methods in mice

The use of transgenic mice in all areas of mammalian biology is now routine. We have transgenic mice that get [breast cancer](http://www3.interscience.wiley.com/cgi-bin/abstract/114046395/ABSTRACT?CRETRY=1&SRETRY=0), have [no immune system](http://www.nature.com/nature/journal/v440/n7088/abs/4401123a.html;jsessionid=EC1558D3E71EBDBD2266C23D0474385A), and have every possible molecular defect [associated with Alzheimer’s Disease](http://iospress.metapress.com/(snlbxljbz041edid4cobney4)/app/home/contribution.asp?referrer=parent&backto=issue,16,51;journal,8,49;linkingpublicationresults,1:105656,1). We even have [Laurel *and* Hardy mice](http://cancerres.aacrjournals.org/cgi/content/abstract/66/17/8897). Of course, my favorites are the memory mutants. There are mice that lack NMDA receptors only in the CA3 region of the hippocampus who can perform a memory task just fine as long as all the cues are there but can’t figure it out if they have to fill in any of the blanks. There’s the CaMKII-3’UTR mutant mouse that has an intact protein but lacks a dendritic targeting signal in the mRNA. There is the Doogie mouse which has adult expression of a subunit of the NMDA receptor (NR2B) that is predominantly found in juveniles. The adult NR2B expression was restricted to the forebrain using a now-familiar transgenic trick wherein the transgene is put under control of another gene’s regulatory sequences. The Doogie mouse performs better than normal, wild-type mice on certain memory tasks. You can’t always make a better mouse by turning up the proteins involved in synaptic plasticity though. For instance, overexpressing a hyperactive version of CamKII in the entorhinal cortex impairs spatial learning. Some of the first refinements of transgenic techniques were performed in the service of memory research. Memory is one of the few areas where you really need to be able to turn a gene off and then on again or vice versa. This is because memory can be divided into stages such as acquisition, consolidation, and retrieval, and if you’re transgenic manipulation were permanent you couldn’t distinguish an effect on one stage from another.

The tools for transgenic manipulation are slowly and steadily improving. It seems to be an incremental, heuristics-driven progress. Nudging an enhancer element over a few nucleotides ore switching out an amino acid in a fusion protein increases specificity or efficacy some X amount, so everyone starts doing it that way. Methods to target ever-more-specific cell populations in the brain and to increase temporal control over transgene expression are in constant development. Also, the list of proteins that you might want to express in a mouse brain is increasing to include fluorescent indicators of just about anything and functional proteins that can turn neural activity up and down. There are a number of [good overviews with illustrations](http://genome.wellcome.ac.uk/doc_WTD021044.html) around to give you the basics. I will hit the most salient features here and discuss some recent improvements.

The most basic way of producing a transgenic mouse is to make a DNA fragment carrying the gene you want to express and inject it into a fertilized mouse egg. The endogenous DNA repair machinery detects this broken chunk and ‘fixes’ it by sticking it in a chromosome. This egg is then implanted in a ‘pseudo-pregnant’ mouse, and if it develops properly you get baby mouse whose every cell carries your inserted DNA. There are problems with this approach even though it fairly popular, and they mainly stem from the fact that the DNA inserts randomly. Your construct could, for instance, get inserted in the middle of some important gene and disrupt its function or get inserted into a ‘silenced’ part of the genome (think epigenetics). Luckily, there are two types of DNA repair mechanism and the second one, homologous recombination, is not so random. Homologous recombination occurs during meiosis when the ‘crossing over’ occurs and chromosomes from either parent trade information. It is also utilized when one double stranded chromosome breaks. The machinery uses the other chromosome as a template to write in the missing letters. The trick with homologous recombination based transgenesis is to hijack the machinery and make it use your DNA fragment as the template. Since the machinery requires some degree of homology between the sequence of the template and the broken sequence you just need to put some homologous sequences on either side of the chunk of DNA you care to stick in the middle of the gene you are targeting. You can imagine that the conditions for all of this to go down properly are fairly rare. You would have to inject like a gajillion eggs. Instead of doing that, researchers turned to selection. They sucked some embryonic stem cells out of a mouse blastocyst and cultured them. If you put a neomycin (an antibiotic kills ES cells too) resistance gene in your DNA construct along with your other transgene, you can grow the cells on plates with neomycin, and only cells that have taken up the transgenic DNA will survive. This isn’t perfect though because the DNA could’ve been inserted randomly and we want homologous recombination (HR). So a second level of selection is necessary. One difference between HR and the random insertion is that the whole construct gets inserted in the latter, whereas in HR the ‘tails’ of the construct are removed. So you simply make a construct that looks like this: TK—Neomycin resistance—Gene of interest—TK. TK is an enzyme that produces deadly results in the presence of certain chemicals. So cells have to have neomycin resistance but lack TK to be selected. You can now grow up a population of very special cells enriched for just the genetic modification you wanted, inject some of those into a mouse blastocyst, and get chimeric babies that have your modification in some portion of their adult cells. You pray that one of your injected ES cells grows up to be a sperm or an egg (germline transmission) and breed your mice accordingly.

Now we can make transgenic mice, but we don’t have much control over where and when the gene is expressed. The simplest way to achieve more spatial control over a transgene is to attach tissue-specific DNA control sequences before the mRNA coding part. A popular tactic among memory researchers is to use the promoter/enhancer sequences from calcium/calmodulin-dependent kinase II (CaMKII) to drive expression of whatever gene they are interested in. Expression under control of the CaMKII promoter tends toward to the forebrain, so you can manipulate a gene like a ubiquitous neurotransmitter receptor without screwing up its function in, say, the spinal cord. The more we know about the function of transcription factors and DNA enhancer elements, the greater [tissue-specificity](http://nar.oxfordjournals.org/cgi/content/full/34/suppl_1/D104) we have. It is a major advance to be able to disrupt the function of a subpopulation of inhibitory interneurons in a subregion of the neocortex rather than performing experimental lesions with an electrode or a toxin. Transgenic mouse producing companies (of which Jackson Labs is probably the best-known) offer a whole array of mice with a gene called Cre recombinase under control of different tissue-specific promoters. I don’t want to go into the details of Cre recombinase. You can basically make the DNA construct carrying your favorite gene with an interrupting sequence so it is silent, and then insert it into a mouse genome. When this mouse is crossed with a tissue-specific Cre recombinase mouse, Cre will turn on the gene of interest only in the cells where it is being expressed.

Tissue-specific promoters allow some spatial and temporal control over transgene expression. For instance, if you use the promoter from a gene that is usually expressed only in adults, you can generally avoid developmental effects of your inserted gene. This degree of temporal control isn’t very fine-gra  
ined though. This is where the inducible transgenic mice become important. The oldest and still the most popular systems for inducible transgene expression are the [Tet-On and Tet-Off](http://jaxmice.jax.org/models/tet_intro.html) systems. The Tet Off system came first, but it is confusing and inconvenient because you use have to constantly feed the mice a drug to keep your transgene off and then release it from repression by not giving them the drug anymore. The Tet On system is much easier to talk about. You need two constructs in this system. One carries the gene for a man-made fusion protein called the reverse Tet Transactivator (rtTA); the other carries the transgene you need to express under the control of a tet operator (tetO) regulation sequence. The rtTA will be made all the time, but since the tetO is a bacterial regulatory sequence, mice shouldn’t express the transgene under normal conditions. Here comes the inducible part: if you feed the mice doxycycline (a tetracycline analog that works better for some reason), it binds and activates the rtTA. Activated rtTA can bind to the tet operator sequences and turn on expression of the tetO – gene of interest. Got it? Google tet off or tet on, there are lots of biotech companies who want to sell you their system and are willing to explain it with pretty diagrams. Here’s an example of two constructs you might make: 1) CaMKII Promoter – rtTA; 2) tetO – green fluorescent protein (GFP). This set would produce rtTA only in the forebrain. If you wanted to make the forebrain glow, you could feed mice doxycycline and wait a couple days. Oh, and you can quit with the doxycycline and the GFP should turn back off.

Everyone is, of course, very happy about being able to turn gene’s on and off at will in mouse brains. The rtTA system isn’t perfect yet though. It has problems with leakiness. That gene that is only supposed to be on when you add doxycycline is still on at some baseline level. There have been numerous attempts to tinker with the system through the years to increase the difference between baseline and induced expression. A recent paper by Shaikh and Nicholson attempted some improvements so they turn on their favorite gene, receptor for advanced glycation end products (RAGE). I love the idea of inducing RAGE (it would definitely be better than inducing Audioslave). In most cases, people use a cluster of tet operators to get good induction. One parameter that [Shaikh and Nicholson](http://www.ncbi.nlm.nih.gov/entrez/utils/fref.fcgi?itool=AbstractPlus-def&PrId=3051&uid=17028168&db=pubmed&url=http://jbt.abrf.org/cgi/pmidlookup?view=long&pmid=17028168) tweaked was the spacing of these operator sequences. In the system they used (called TREtight), the tet operators were packed in really close to each other. They used a tissue-specific promoter to drive rtTA expression. In this case, they used the neuronal specific enolase (NSE) promoter to express rtTA in any and all neurons. They also went against a recent trend by inserting the rtTA gene and the RAGE gene in two separate constructs instead of placing the two genes in one big chunk of DNA and injecting the whole thing. I should mention that all this refinement work is happening in cell culture, not in trangenic mice. They managed to get 142-fold induction of RAGE upon exposure of the cells to doxycycline. Not bad. They found something that works for their purposes, but this is also sort of representative of a problem with progress in the field. I can’t tell from the paper what is the factor that mattered and whether that factor would be generally helpful to apply in any further manipulations I want to do. People find something that works for them, but there aren’t a lot of papers comparing different ‘optimal’ systems. Beyond changing the nature of the DNA constructs and enhancers, there are also people tinkering with the amino acid sequence of the rtTA protein to perhaps make it more responsive or have different induction properties. For instance, one technique has been to use a version of Tet Transactivotor (TransSilencer) that actually silences tetO-controlled genes in the absence of doxycycline in conjunction with the rtTA. Now the trans-silencer is active in baseline conditions and turns down leakiness, but when you add doxycycline it both de-activates the trans-silencer and activates the trans-activator. There are also other inducible systems wherein the estrogen receptor (which can activate gene expression) has been mutated such that it is induced by tamoxifen.

One other area that isn’t necessarily specific to transgenic issues is the production of new types of proteins worth expressing. Sometimes you want to do more than just make your cells glow. [Miyoshi and Fishell](http://www.ncbi.nlm.nih.gov/entrez/utils/fref.fcgi?itool=AbstractPlus-def&PrId=3048&uid=16971113&db=pubmed&url=http://linkinghub.elsevier.com/retrieve/pii/S0959-4388(06)00118-8) have produced a table of interesting proteins worth expressing in neurons. They include proteins that glow only during synaptic transmission or during calcium influx, proteins that cross synapses and identify cell connectivity (tracers), and functional proteins that can affect neuronal physiology when expressed. As an example of a functional protein that you might want to express, there is a synaptobrevin fusion protein developed by [Alla Karpova and Karel Svoboda](http://www.ncbi.nlm.nih.gov/entrez/utils/fref.fcgi?itool=AbstractPlus-def&PrId=3048&uid=16337911&db=pubmed&url=http://linkinghub.elsevier.com/retrieve/pii/S0896-6273(05)00963-3) that shuts down synaptic transmission in the cells in which it is expressed. You could use inducible expression of this protein to really dissect the contribution of a given neuronal population. If any of you protein-refiners are out there listening, might I suggest what I think would be ideal? I don’t understand yet how you can make versions of GFP that are sensitive to calcium, pH, and voltage, but if its not too much to ask, why don’t you just make an rtTA that is sensitive to these things? I’m dreaming of a transgenic mouse with, for instance, voltage-sensitive rtTA and a tet-controlled GFP. You could insert little tiny electrodes into this mouse’s hippocampus and stimulate gene expression with minimal invasiveness but presumably greater temporal control than doxycycline going through the digestive system.

I have a couple mad scientist experiments dreamed up involving all of these inducible systems, but I think they would require making transgenic mice that carry 6 different constructs. The basic idea is to mark cells that were active during memory encoding by causing them to express rtTA. Then I could turn on any gene that I wanted just in the cells for a certain memory. Perhaps I could even reactivate the cells if I turned on the right gene (say a leaky calcium channel that greatly increased cell excitability). Would this reactivation cause the mice to perceive recollection of that particular memory? It just might and that could be used to achieve [‘Marilyn Monroe’ experiment](https://www.gnxp.com/blog/2006/08/ltp-memory.php) in which you make a mouse remember something that never happened. I’ll let you think on it for a while and post how I think it could be done next week.

### Related Posts:

- [The
  Brainbow](https://www.gnxp.com/WordPress/2007/11/09/the-brainbow/) - [The genes of
  sleeping](https://www.gnxp.com/WordPress/2009/08/15/the-genes-of-sleeping/) - [APOE4: a blessing and a
  curse](https://www.gnxp.com/WordPress/2007/01/13/apoe4-a-blessing-and-a-curse/) - [Mice that can't relate to
  people....](https://www.gnxp.com/WordPress/2006/05/04/mice-that-cant-relate-to-people/) - [Genetic Engineering Color Vision in
  Mice](https://www.gnxp.com/WordPress/2007/03/22/genetic-engineering-color-vision-in-mice/) - [Tonegawa](https://www.gnxp.com/WordPress/2006/11/27/tonegawa/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F09%2Ftransgenic-methods-in-mice%2F&linkname=Transgenic%20methods%20in%20mice "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F09%2Ftransgenic-methods-in-mice%2F&linkname=Transgenic%20methods%20in%20mice "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F09%2Ftransgenic-methods-in-mice%2F&linkname=Transgenic%20methods%20in%20mice "Email")[](https://www.addtoany.com/share)
