+++
title = "Resolving the hotspot"
full_title = "Resolving the hotspot paradox"
date = "2007-02-01"
upstream_url = "https://www.gnxp.com/WordPress/2007/02/01/resolving-the-hotspot-paradox/"

+++
Source: [here](https://www.gnxp.com/WordPress/2007/02/01/resolving-the-hotspot-paradox/).

Resolving the hotspot paradox

When, a while back, the topic of [the “hotspot paradox”](https://www.gnxp.com/blog/2006/10/hotspot-paradox.php) came up, I imagined a number of fairly straightforward analyses that would be important first steps in the resolution of said paradox. As [a new paper](http://www.journals.uchicago.edu/AJHG/journal/issues/v80n3/44272/brief/44272.abstract.html) suggests that others have very similar ideas (and I don’t have the time to get into a new project right now), I figured I’d throw my thoughts out there.

First, a simple formulation of the hotspot paradox– recombination hotspots are places in the genome the are particularly likely to be involved in recombination during meiosis. In the first steps of recombination, one allele initiates the entire process; you could imagine a particular sequence being more prone to this initiation than others. However, during recombination, the initiating sequence is replaced by the other sequence (on a very small scale, see [this figure](http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=mcb.figgrp.3272)). Thus, if there are two alleles– a “hot” allele and a “cold” allele, eventually the “cold” allele will become fixed and there will be no recombination. The paradox, then, is that recombination still occurs (read the post linked above and the links therein if this doesn’t make sense).

Personally, I don’t find this so paradoxical– it just simply means that there are factors other than local sequence variation that determine the location of a recombination event. So to resolve the paradox, one would just need to identify those factors and find out how they do whatever they do. Simple enough.

To find these factors, one could just use genetic mapping, treating hotspot intensity as a phenotype just like any other. The key for this approach is that there is variation in the population you’re using– if everyone had the same hotspots with the same recombination rate, mapping the intensity of a hotspot just wouldn’t work (mapping the variation of a phenotype only works, obviously, if there is indeed variation). The [paper I linked to above](http://www.journals.uchicago.edu/AJHG/journal/issues/v80n3/44272/brief/44272.abstract.html) demonstrates exactly that– people differ in where their recombination events occur. Note that the authors of the study are the same ones who first [characterized variation](http://www.nature.com/ng/journal/v33/n3/abs/ng1094.html) in gene expression in humans, variation then then mapped, leading to a number of [high](http://www.nature.com/nature/journal/v430/n7001/abs/nature02797.html) [profile](http://www.nature.com/nature/journal/v437/n7063/pubmed/nature04244.html) [papers](https://www.gnxp.com/blog/2007/01/differences-in-gene-expression-between.php). There’s no doubt that they would love to map this phenotype as well.

But there are also some hints that the publicly available data set they’re working with doesn’t have everything they need to do the mapping. First, each parent has “only” 8 children on average, which means there aren’t so many recombination events you can look at. They’re forced to combine events into “recombination jungles”, which are much larger (5 Mb) than recombination hotspots (a couple hundred bases) and presumably less robust.

To map the determinants of recombination in the genome, then, you’d need a pedigree with more children per family (I can think of only one off the top of my head). Alternatively, you could take a different approach and look only at a couple hotspots with high resolution using sperm typing. If you could look at the intensity of a few hotspots in a large number of men, it would be trivial (assuming you also have genome-wide genotypes for those men) to map the intensity back to causal loci in the genome. Any independently wealthy readers are encouraged to do this experiment and resolve the paradox.

### Related Posts:

- [The hotspot
  paradox](https://www.gnxp.com/WordPress/2006/10/08/the-hotspot-paradox/) - [PRDM9 and the evolution of recombination
  hotspots](https://www.gnxp.com/WordPress/2010/01/02/prdm9-and-the-evolution-of-recombination-hotspots/) - [Authenticity and the Fermi
  paradox](https://www.gnxp.com/WordPress/2010/07/05/authenticity-and-the-fermi-paradox/) - [Recombination also
  evolves](https://www.gnxp.com/WordPress/2015/01/18/recombination-also-evolves/) - [More on recombination & natural
  selection](https://www.gnxp.com/WordPress/2010/03/28/more-on-recombination-natural-selection/) - [Robert Jordan is
  sick](https://www.gnxp.com/WordPress/2006/11/21/robert-jordan-is-sick/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F01%2Fresolving-the-hotspot-paradox%2F&linkname=Resolving%20the%20hotspot%20paradox "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F01%2Fresolving-the-hotspot-paradox%2F&linkname=Resolving%20the%20hotspot%20paradox "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2007%2F02%2F01%2Fresolving-the-hotspot-paradox%2F&linkname=Resolving%20the%20hotspot%20paradox "Email")[](https://www.addtoany.com/share)
