+++
title = "Noise in gene expression"
full_title = "Noise in gene expression"
date = "2006-05-22"
upstream_url = "https://www.gnxp.com/WordPress/2006/05/22/noise-in-gene-expression/"

+++
Source: [here](https://www.gnxp.com/WordPress/2006/05/22/noise-in-gene-expression/).

Noise in gene expression

An old friend of mine (handle: deadsmith) dropped me this discussion of selection and noise in biological systems. I figured with the [various](https://www.gnxp.com/blog/2005/07/not-genes-and-not-environment.php) [discussions](https://www.gnxp.com/blog/2005/10/noise.php) [of noise](https://www.gnxp.com/blog/2006/04/no-two-alike-role-of-noise.php) [around here](https://www.gnxp.com/blog/2006/04/judith-rich-harris-responds.php), you folks might enjoy it. The words below are his, the errors in formatting are mine:

I’d love to tell you that this entry is going to be a comprehensive, cogent, and readily accessible one-stop shopping spot for all the engaging literature on noise in gene expression that has been burgeoning in the background of your copies of Nature and Science for the last ten years. But, [that’s already been written](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16179466&query_hl=1&itool=pubmed_docsum), by a far more credible source in Jonathon Rasner and Erin O’Shea. So, I’m going to recap some of that review here, and try to wrap it gently in the cushy context of why I was interested in the first place.

If you get interested in evolution on your own, it’s probably because you read a pop-science book by someone like Gould or Dawkins or Ridley; or maybe a monkey person like one of the Leakeys or Marc Hauser or some such. And if you do, then you’ll get told about the forces of evolution, namely: Selection, Drift, Flow, and noise. The last one doesn’t get a capital letter here, because it almost certainly didn’t get one in the books I’m talking about. And that’s the point: noise is last in the list, because selection is the most important force, followed by drift and flow, then noise is a distant factor. Right?

And what is “noise” anyway? When reading the intro-to-evo books, it sounds a lot like “unexplained variance in the object of study.” And I think lots of people are still using it this way, even when they mean more specific concepts, such as “stochastic,” which is not the same thing as “unexplained variance.” And where does this noise come from? And why did it come from whence it came? Is it staying for dinner? I think we tend to think of biological organisms as elaborately crafted systems with relatively precise mechanisms, often analogous to engineered circuits, e.g. feedback loops, oscillators, etc. And if you’ve ever done any engineering, then you know that there are elaborate equations that engineers use to go about estimating and minimizing the noise in their systems, so why wouldn’t natural selection have done the same thing?

Maybe furthering the engineering analogy can pursue the answer. You’ve probably purchased an electronic component at some point that failed prematurely, and you’ve probably purchased one that lasted longer than was expected. I won’t name any brand names here, but we all know that some companies spend more in quality assurance than others. Why? On one hand, some companies pick the strategy of spending a little more and making a quality product, knowing that if it’s important to the customer, then they can charge a little higher price for their product. This might be common in the automotive industry, where people don’t want to buy new cars every week, even if they cost \$50 a pop. But there are other industries where things are more disposable, because people don’t need them everyday like many people need their cars, such as point-and-shoot cameras. Why cheap out on the camera? Because it’s expensive to pay engineers to do those equations, to machine the parts to higher specs, and to have a lower threshold for crappy components that get chunked in the dumpster out back before they ever hit the shelves. So, people won’t die if their cheap camera is taking discolored pictures nearly so much as if their wheel falls off on the interstate.

Okay, enough analogy. Selection should minimize noise, but minimizing noise in gene expression, just as in electrical component failure, is a complicated venture that has to take into account all the factors of the entropic universe and might take quite a bit of extra machinery. So, if a little noise can be tolerated, then why not just make a crappy camera? Oh, evolution, you are a shrewd craftsperson indeed.

And if you’ve made it down the lay-science best sellers list to people like Dennett, then you probably encountered the idea of an evolutionary “design space“ ‐ that is, a theoretical space for the course of new phenotypes to play in en route to possible selection, and a little noise might just broaden that space a bit, without being detrimental to the existing organism.

Okay, really, enough analogies, let’s talk molecular biology.

Single celled, monoclonal organisms are ideal for studying the noise in gene expression, because you can setup a reporter system and measure how much variation there is in the clonal population in the same environment. As all good… well, at least all… intro biology books tell us that the phenotype is the product of the genotype and the environment. A + B = phenotype. Did you see a noise term in that equation? I didn’t think so… Think again. It so happens that even with genetically identical cells, in an identical environment, we see variation in phenotype. I’m not talking twin studies here, so much as single cell organisms floating in the same pudding.

One paradigmatic method in the contemporary noise papers comes from the Elowitz lab at CalTech (at least I think it started there). Elowitz noticed that some monoclonal cells weren’t behaving identically, even when they were in experimentally identical environments. So, he took some CFP (cyan fluorescent protein), and some YFP and stuck them both behind identical promoters in one case, and different promoters in other case. This allowed him to [compare two reporters that either had the same promoter](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=12183631&query_hl=2&itool=pubmed_docsum), thus the same transcriptional regulation (or at least a big part of it), with two reporters that had different promoters. From there, you can subdivide the noise into “extrinsic” and “intrinsic.” Extrinsic noise sources will result in the expression of each reporter changing similarly, and intrinsic noise affects the reporters differentially. So, you can think of the noise source from the perspective of a single reporter.

The best example of an extrinsic noise source is probably cell cycle stage. If the clones are all in a different stage of their life cycle, then you would expect gene expression to vary considerably (this, by the way, is an example of using the word noise to mean “unexplained variance” rather than “stochastic variance”). If the cells were synchronized, then you can see if some of the expression variance from the same gene across many cells goes away. But if you’re looking at two reporters of the same gene (or at least the same promoter), in the same cell, then you can measure the intrinsic noise of that gene’s expression.

Then you can combine the two fluorescent reporter construct with manipulations of gene regulation, e.g. varied transcriptional and translational efficiency. To vary transcription, you might mutate the promoter of the reporters or maybe use an inducible system with a variation in how much inducer you supply across conditions (think lac operon); to vary translation you could mutate the ribosome binding sequence or maybe the initiation codon ([Ozbudak, et al 2002](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retriev%20e&db=pubmed&dopt=Abstract&list_uids=11967532&query_hl=4&itool=pubmed_docsum)). Then you could see the relative contributions to noise of transcription or translation, and in the case of E. coli, it appears to be translation. The story behind this is that the prokaryotic translation process occurs simultaneously with continued transcription, and there is competition between the ribosome and RNases that want to eat the mRNA for access to the ribosome. This competition results in the ribosomes blocked from the mRNA, and RNases degrading the mRNA. So, you either get translated, or you get degraded, and if you assume a constant probability of translation, and that each event is independent of the next (and you’re a statshead), then you’ll recognize this as the geometric distribution and you could model this dynamic as [McAdams and Arkin did in 1997](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=9023339&query_hl=8&itool=pubmed_docsum) and predict what would later be confirmed by the Ozbudak study.

And that’s what happens with the two-reporter system in coli, but what about eukaryotes? The lab of [J.J. Collins looked into eukaryotes](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=12687005&query_hl=9&itool=pubmed_docsum) with the two-reporter system cloned into budding yeast (S. cerevisiae). Collins notes that in eukaryotes, transcription is conducted in pulses, or for your neurobiology people out there, in quanta. In contrast to the prokaryotes, which are expected to have difficulties translating their mRNAs once transcribed (due to the RNase competition mentioned above), the eukaryotes were predicted by the Collins lab to have troubles initiating transcription, and that transcription would be the major source of noise. When the initiation failed, there would be no mRNA, and when it succeeded, then there would be a stabilized promoter, allowing multiple transcripts to be reeled off the DNA. They induced the GAL1 promoter (native mechanism of yeast to produce an enzyme for galactose when the preferred sugar glucose is unavailable). Usually, the GAL1 promoter is activated by galactose, via its UAS (upstream activating sequence) and an activator responsive to galactose. Using this context, and also an artificial one employing a tet-O construct, the experimenters were able to poke and prod the transcription initiation of the gene. And whichever ways they went, the findings were that transcription was very noisy when the gene was moderately induced, but as the induction increase, the noise went down. Meanwhile, they used codon manipulations known to affect translation efficiency (this uses codon bias and the CAI for anyone interested, but I’m not going into it here ‐ suffice it to say, there’s a better way to do this experiment, but this worked well enough to fetch a Nature paper). The results: eukaryotes had noisy transcription, and translation that was robust to noise ‐ just the opposite of the E. coli bugs that Elowitz developed the two reporter system in.

And just to make things complicated, the O’Shea lab (see above) took [their own shot at eukaryotic expression noise](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15166317&query_hl=10&itool=pubmed_docsum) using nearly the same system as Collins, but wanted to characterize intrinsic vs. extrinsic transcriptional noise. The results: it varies from promoter to promoter. It’s really a crime for me to only say this much for the [O’Shea study](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15166317&query_hl=10&itool=pubmed_docsum), but as it’s really well written, I think it’s better extracurricular material for anyone that’s really interested.

So, to recap: 1) prokaryotes have noisy translation because their transcripts are fought over by RNases and ribosomes. 2) Eukaryotes have noisier transcription because activation of promoters leads to pulses of mRNA production, but 3) this varies from promoter to promoter.

Now… Why would it vary? This is how we come full circle. There are two rationales for having a noisy gene. First, it may not have been that important to refine the noisiness of the gene. It’s expensive to build in more precise control circuitry, and if you don’t need it, then you don’t need it ‐ sell the darn camera. On the other hand, what if it were a trait that didn’t need to be precisely regulated, thus was allowed to stick around and be noisy, thus provided variation in a phenotype amongst a monoclonal population? Would that give an advantage to the noisy population of clones in a time of stress? A time where the selection pressures were a little more specific? Is noise a way to allow organisms to hedge their bets when selection isn’t currently holding their feet to the fire on a particular issue? Many think so.

But this isn’t just conjecture; there are hypotheses here. One that has been tested: Are essential genes more likely to be noisy than others? Using the stochastic models developed by the Collins lab, the Eisen lab at UC Berkeley computationally analyzed yeast genes and found that [the noisier the gene expression, the less likely the gene was to be essential](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15124029&amp;query_hl=12&itool=pubmed_docsum) ‐ thus evolution isn’t willing to play around with the bread winners of the genome. What’s more? The same experimenters point out that whereas extrinsic noise changes gene expression, it changes all gene expression in the cell without regard to specific genes. Then they posit that genes that are not part of a multi-subunit complex would better tolerate this noise. Using data from two near-proteome-wide mass spectrometry studies, they show that proteins in complexes do appear to be less noisy than others.

Hot damn. Molecular evidence that selection crafts adaptations to be more precise when it’s more important. And when you phrase it that way, maybe it doesn’t sound so revolutionary. But these researchers are showing us not just which genes are expected to be more carefully engineered by evolution, but precisely where the tradeoffs in the machine are ‐ and the tradeoff points are different in the prokaryote machine versus the eukaryote machine. What’s more? There are predictions about where to look for the next essential genes. All the genes that are already essential are tightly regulated, thus have less phenotypic variance to select from. Meanwhile, the noisy genes could be the accidental saviors of a species when the next heavy selection pressure pops up. I suspect that if you start mapping things like essentiality and multi-subunit complexes to molecular phylogenies, then you may see a trace of this in the natural history, and then you’d have a historical account of the lattermost evolutionary force that even the most selection-centric evolutionists might write about.

### Related Posts:

- [Hawks on
  Noise](https://www.gnxp.com/WordPress/2006/05/25/hawks-on-noise/) - [Noise](https://www.gnxp.com/WordPress/2005/10/03/noise/) - [The moral of
  noise](https://www.gnxp.com/WordPress/2014/09/18/the-moral-of-noise/) - [Genomic noise and individual
  variation](https://www.gnxp.com/WordPress/2008/01/26/genomic-noise-and-individual-variation/) - [Noisy genes and the limits of genetic
  determinism](https://www.gnxp.com/WordPress/2010/05/12/noisy-genes-and-the-limits-of-genetic-determinism/) - [Beyond the
  sequence](https://www.gnxp.com/WordPress/2009/04/22/beyond-the-sequence/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F05%2F22%2Fnoise-in-gene-expression%2F&linkname=Noise%20in%20gene%20expression "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F05%2F22%2Fnoise-in-gene-expression%2F&linkname=Noise%20in%20gene%20expression "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2006%2F05%2F22%2Fnoise-in-gene-expression%2F&linkname=Noise%20in%20gene%20expression "Email")[](https://www.addtoany.com/share)
