+++
title = "On that homogeneous Y"
full_title = "On that homogeneous Y"
date = "2014-01-13"
upstream_url = "https://www.gnxp.com/WordPress/2014/01/13/on-that-homogeneous-y/"

+++
Source: [here](https://www.gnxp.com/WordPress/2014/01/13/on-that-homogeneous-y/).

On that homogeneous Y

![Yes, it’s not really “Y” shaped](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/01/Y_chromosome.png?resize=155%2C183)

For various reasons the [Y chromosome](https://en.wikipedia.org/wiki/Y_chromosome) is something of a big deal. As you probably recall from high school biology if you are a male you have a Y chromosome, which lacks most functional gene copies, along with an X, which has the full complement of genes.\* If you are female you carry two X chromosomes. The redundancy of having two X chromosomes results in the phenomenon of [sex-linkage](https://en.wikipedia.org/wiki/Sex_linkage), where recessive traits which are generally masked in one sex are expressed in the other (this is often well known from pathologies, from the mild such as color blindness to the severe such as haemophilia).

But aside from the proximate functional importance, there are bigger evolutionary questions of interest when it comes to the sex chromosomes. About a decade ago [the biologist](https://www.theguardian.com/science/2003/aug/28/genetics.genderissues) Bryan Sykes floated the possibility of the extinction of human males due to the long term degeneration of the Y chromosome. The reasoning is straightforward. Because most of the Y chromosome is non-recombining it can not easily purge deleterious mutations which have been fixed within a population, since these will be linked with all the other loci. In contrast in the recombining genome favored and disfavored alleles will be mixed in match. Over time the Y chromosome tends to degenerate as most of the functionality becomes biased towards sexual phenotypes, such as the [SRY region](https://en.wikipedia.org/wiki/Testis_determining_factor). A major problem with this sort of model (which Sykes himself mentions) **is that it seems that sex chromosomes have emerged multiple times across multiple lineages**, and also disappeared as well. The chromosomal organization of a species is not a static and fixed aspect of their construction, but a function of evolutionary processes through duplications and fusions. A “Y chromosome” does not magically make a male a male, especially seeing as how there are males in lineages without the Y chromosomes (in birds the females are the heterogametic sex).

[![The_Journey_of_Man\_-\_A_Genetic_Odyssey](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/01/The_Journey_of_Man_-_A_Genetic_Odyssey.jpg?resize=197%2C299)![The_Journey_of_Man\_-\_A_Genetic_Odyssey](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/01/The_Journey_of_Man_-_A_Genetic_Odyssey.jpg?resize=197%2C299)](https://www.amazon.com/exec/obidos/ASIN/0812971469/geneexpressio-20)

But the lack of recombination is not always a problem. It is because the Y chromosome, like mitochondrial DNA, does not recombine that it is extremely amenable to phylogenetic inference using a [coalescent](https://en.wikipedia.org/wiki/Coalescent_theory) framework. Most of the genome subject to recombination is constantly sliced, diced, and amalgamated, in a fashion which makes inference of genetic lineages difficult. In contrast the mtDNA and Y are inherited as long blocks of genes with numerous variants which generate long [haplotypes](https://en.wikipedia.org/wiki/Haplotypes) that can be useful information in reconstructing trees of ancestry and descent. Though limited to the male line of descent the utility of the Y for this reason made it extremely useful for phylogeographic studies in the late 1990s and early 2000s, culminating in works such as Spencer Wells’ [The Journey of Man](https://www.amazon.com/exec/obidos/ASIN/0812971469/geneexpressio-20). Though there are still Y chromosomal studies being published, with whole-genome sequences and large sample sizes, the golden years of this program are likely past us.

One of the primary assumptions behind these genealogical trees was that variation was being generated in a neutral fashion. By this, I mean that genetic variation was being distributed through random forces, and so that variation was a reasonable proxy for the demographic history of the populations. A new paper in *PLoS GENETICS* adds some nuance into this bigger picture, [Natural Selection Reduced Diversity on Human Y Chromosomes](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004064). Various versions of this paper have been floating around for several years, and the earliest I knew about the general thesis was at *ASHG 2012*. **The basic observation is that the diversity of the Y chromosome is far lower than it should be**. Much of the early work on Y chromosomal phyogenetics was done with microsatellites, highly repetitive and mutable regions of the genome. While the mutation rate of SNPs might be 10⁻⁸, microsatellites turn over on the order of 10⁻⁴. This makes them highly informative for forensic purposes, and also allows one to differentiate very similar populations. Initial attempts to use SNPs on the Y chromosome failed because the variation was too low. Today with more powerful genomic techniques those obstacles are gone, and a reasonable number of whole-genome analyses of Y chromosomes have been performed.

![**Citation:** Wilson Sayres MA, Lohmueller KE, Nielsen R (2014) Natural Selection Reduced Diversity on Human Y Chromosomes. PLoS Genet 10(1): e1004064. doi:10.1371/journal.pgen.1004064](https://i0.wp.com/www.unz.com/wp-content/uploads/2014/01/journal.pgen_.1004064.t001-293x300.png?resize=293%2C300)

The research in the paper above focuses on 16 individuals sequenced on the Y from Africa and Europe. One conventional explanation for why the Y is so much less diverse is that there is a reproductive skew among males, so that the male effective population is lower. This means that there is less variation being sampled from for males than females every generation, so in terms of diversity your rank order should be Y \< autosomal \< X chromsome \< mtDNA. But using forward simulations and modulating the sex ratio it was clear that reproductive skew could not explain the extremely low diversity of the human Y chromosome.

An alternative explanation would be selection. In particular they found evidence of selection of the purifying form, where deleterious alleles are removed, but along with them linked regions of the genome. In the case of the Y that linked region would be the whole chromosome, perhaps explaining the low diversity as variation is constantly removed by functional constraint. Intriguingly the authors also found that mutations around nonsynonymous regions of the genome which code for protein could not account for the effect they were seeing. Rather, they suggested that ampliconic regions (repetitive sequences) are being constrained strongly. There’s a lot of molecular genetic detail at this point (e.g., intra-chromosomal interactions), and I won’t go into it, but it seems that it is correct that researchers should now focus on these genomic regions to understand what’s going on in terms of mechanistic detail. There is more to be imagined in the Y than in the wildest dreams of our philosophy!

Finally, **let me reiterate these results do not invalidate the nearly two decades of Y chromosomal phylogenetics**. Background selection has the same effect across the topology, so the shape would be maintained, though the branch lengths would shorten. In other words the coalescence of Y chromosomal lineages might be deeper than has currently be implied. But all these results are inferred from 16 genomes. There are many more whole-genomes out there, and a more thorough survey seems necessary. It seems that the authors could not reject the possibility that composite positive selection across different lineages were responsible for some of the decrease in diversity. If this was the case then our Y chromosomal phylogenies would in fact be totally misleading in terms of true demographic history.

**Citation:**Wilson Sayres MA, Lohmueller KE, Nielsen R (2014) [Natural Selection Reduced Diversity on Human Y Chromosomes](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004064). PLoS Genet 10(1): e1004064. doi:10.1371/journal.pgen.1004064

\* There are exceptions [due to problems on the Y](https://en.wikipedia.org/wiki/Androgen_insensitivity_syndrome) chromosome, which often result in a female phenotype.

### Related Posts:

- [The inevitability of sex chromosome
  degeneration](https://www.gnxp.com/WordPress/2014/02/09/the-inevitability-of-sex-chromosome-degeneration/) - [Y Chromosome IV: Recombination
  Suppression](https://www.gnxp.com/WordPress/2010/08/23/y-chromosome-iv-recombination-suppression/) - [The evolving
  male](https://www.gnxp.com/WordPress/2007/11/16/the-evolving-male/) - [The neo-X on the
  fly](https://www.gnxp.com/WordPress/2009/04/14/the-neo-x-on-the-fly/) - [Chromosomes and
  Evolution](https://www.gnxp.com/WordPress/2005/10/27/chromosomes-and-evolution/) - [Cognition and the sex chromosomes: studies in
  Turner…](https://www.gnxp.com/WordPress/2006/05/04/cognition-and-the-sex-chromosomes-studies-in-turner-syndrome/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F01%2F13%2Fon-that-homogeneous-y%2F&linkname=On%20that%20homogeneous%20Y "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F01%2F13%2Fon-that-homogeneous-y%2F&linkname=On%20that%20homogeneous%20Y "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2014%2F01%2F13%2Fon-that-homogeneous-y%2F&linkname=On%20that%20homogeneous%20Y "Email")[](https://www.addtoany.com/share)
