+++
title = "Man at Bab elMandeb"
full_title = "Man at Bab elMandeb"
date = "2011-06-10"
upstream_url = "https://www.gnxp.com/WordPress/2011/06/10/man-at-bab-el-mandeb/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/06/10/man-at-bab-el-mandeb/).

Man at Bab el-Mandeb

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)In light of [my last post](http://blogs.discovermagazine.com/gnxp/2011/06/a-genomic-sketch-of-the-horn-of-africa/) I had to take note when [Dienekes](dienekes.blogspot.com/2011/06/mtdna-haplogroup-hv1-across-red-sea.html) today pointed to this new paper in the *American Journal of Physical Anthropology*, [Population history of the Red Sea—genetic exchanges between the Arabian Peninsula and East Africa signaled in the mitochondrial DNA HV1 haplogroup](https://dx.doi.org/10.1002/ajpa.21522). The authors looked at the relationship of mitochondrial genomes, with a particular emphasis upon Yemen and the Horn of Africa. This sort of genetic data is useful because these mtDNA lineages are passed from mother to daughter to daughter to daughter, and so forth, and are not subject to the confounding effects of recombination. They present the opportunity to generate nice clear trees based on distinct mutational “steps” which define ancestral to descendant relationships. Additionally, using neutral assumptions mtDNA allows one to utilize molecular clock methods to infer the time until the last common ancestor of any two given lineages relatively easily. This is useful when you want to know when a mtDNA haplgroup underwent an expansion at some point in the past (and therefore presumably can serve as a maker for the people who carried those lineages and their past demographic dynamics).

What did they find? Here’s the abstract:

> Archaeological studies have revealed cultural connections between the > two sides of the Red Sea dating to prehistory. The issue has still not > been properly addressed, however, by archaeogenetics. We focus our > attention here on the mitochondrial haplogroup HV1 that is present in > both the Arabian Peninsula and East Africa. **The internal variation > of 38 complete mitochondrial DNA sequences (20 of them presented here > for the first time) affiliated into this haplogroup testify to its > emergence during the late glacial maximum, most probably in the Near > East, with subsequent dispersion via population expansions when > climatic conditions improved.** Detailed phylogeography of HV1 > sequences shows that more recent demographic upheavals likely > contributed to **their spread from West Arabia to East Africa**, a > finding concordant with archaeological records suggesting intensive > maritime trade in the Red Sea from the sixth millennium BC onwards. Closer genetic exchanges are apparent between the **Horn of Africa and Yemen, while Egyptian HV1 haplotypes seem to be more similar to the Near Eastern ones.**

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/hornz.png?resize=295%2C258)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/hornz.png?resize=295%2C258)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/hornz.png)Much of this is totally concordant with the results we’ve generated from the autosomal genome. Though the autosomal genome is much more difficult when it comes to implementing many of the tricks & techniques of phylogeography outlined above, it does offer up a much more robust and thorough picture of genetic relationships between contemporary populations. Instead of a a distinct and unique line of paternal or maternal ancestry, thousands of autosomal SNPs can allow one t o get a better picture of the nature of the total genome, and the full distribution of ancestors.

The map to the left shows the spatial gradients of the broader haplogroup under consideration, HV1. But what about the branches? Below is an illustration of the phylogenetic network of branches of HV1, with pie-charts denoting the regional weights of a given lineage:

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/yem2.jpg?resize=600%2C410)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/yem2.jpg?resize=600%2C410)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/yem2.jpg)

Since the shading is so difficult, let me jump to the text:

> …Curiously, the HV1 root haplotype with substitution at position > 16,067 was not observed in the Arabian Peninsula except in four Yemeni > Jews, but was observed in 11 Caucasus, four Egyptian, one European, > two Maghreb, and six Near Eastern samples, **thus supporting a possible origin in the Near East.** Haplotype 16,067–16,362, possibly defining a pre-HV1 haplogroup, has so far been observed in Dubai (one), Ethiopia (four), Maghreb (one), and Yemen (three)….

I think you have be very, very, careful to not read too much into mtDNA lineage distributions and what they may tell you about the past, at least in and of themselves. With the rise of ancient DNA and deeper analyses of mtDNA sequences as well as better geographical coverage many of the inferences of the last 10 years are being radically revised. But, combined with the autosomal results the origin of these mtDNA haplogroups in the Middle East within the last \~10 thousand years seems eminently possible.

Finally, here are their time until the most recent common ancestor estimates:

> …The **TMRCA estimate for HV1 was 22,350** (14,737–30,227) years when > taking into consideration the sequences without the polymorphism at > 15,218—a figure which closely matches the estimate of 18,695 > (13,094–24,449) years when not considering those two sequences. The > control region age estimate of HV1 also presents a similar age, dating > to 19,430 (6,840–32,023) years. Age estimates of HV1 daughter > sub-haplogroups are only slightly lower—15,178 (8,893–21,671) years > for HV1a and 17,682 (10,320–25,316) years for HV1b. The common Arabian > Peninsula and East African sub-haplogroups HV1a3 and HV1b1 share a > close age of 6,549 (2,456–10,746) years and 10,268 (4,792–15,918) > years, respectively. Sub-haplogroups HV1a1 and HV1a2, which despite > being rare seem to have a wider geographical distribution, have TMRCA > of 10,268 (3,602–17,194) years and 9,518 (3,963–15,255) years, > respectively. The ratio of the dates based on the ρ statistic for the > synonymous clock relative to the complete sequence was 1.24, closely overlapping in most branches except for HV1a1 which has a very broad age estimate based only on synonymous diversity \[23,616 (4,917–42,315) years\]….

The confidence intervals on these estimates are really large. All you can say with a high degree of certainty is that the expansion of the family of HV1 haplogroups does not predate the Last Glacial Maximum, 15 to 20 thousand years ago. Many of the daughter branches seem to have emerged in the Holocene, possibly after the rise of agriculture. But with the huge possible set of ranges these temporal estimates come close to offering up pretty much zero additional clarity on the chronology of population dynamics in this region .

Readers might also be interested this from last January, [Internal Diversification of Mitochondrial Haplogroup R0a Reveals Post-Last Glacial Maximum Demographic Expansions in South Arabia](http://mbe.oxfordjournals.org/content/28/1/71.abstract) (with some of the same authors). One aspect of these sorts of papers working with mtDNA is that they remain generally oriented toward the proposition that Pleistocene population structure is extremely important in predicting contemporary patterns of genetic variation. I’m not sure this is such a robust model. The autosomal and uniparental data from Ethiopia and Somalia strongly leans us toward the proposition of admixture of two very distinct populations, one in East Africa (“Ancestral East Africans”), and Eurasian group which are likely to have been intrusive. The genetic distance between the Eurasian inferred ancestral component, which is nearly identical to that of southern Arabia, and other Eurasian components is not so large that it seems plausible that there could have a separation during the Pleistocene. In other words, there was a lot of Holocene migration. If I had to guess I would say it had something to do with the agricultural and pastoral lifestyles brought by Arabians to the Horn of African within the last 10,000 years. Simple ecology imposed a limit upon the expansion of these peoples into more classical lush tropical Africa. Eventually a population did emerge to exploit these territories, Bantus from west-central Africa. Just like the Arabian-AEA hybrid population they encountered ecological, and also demographic, limits on the margins of the Semitic and Cushitic dominated territories in the Horn of Africa. And then of course there are the Nilotes….

**Citation:** Musilová, Eliška, Fernandes, Verónica, Silva, Nuno M., Soares, Pedro, Alshamali, Farida, Harich, Nourdin, Cherni, Lotfi, Gaaied, Amel Ben Ammar El, Al-Meeri, Ali, Pereira, Luísa, & Černý, Viktor (2011). Population history of the Red Sea—genetic exchanges between the Arabian Peninsula and East Africa signaled in the mitochondrial DNA HV1 haplogroup American Journal of Physical Anthropology : [10.1002/ajpa.21522](https://dx.doi.org/10.1002/ajpa.21522)

### Related Posts:

- [Caveman rules: prepare to be
  surprised!](https://www.gnxp.com/WordPress/2013/12/04/caveman-rules-prepare-to-surprised/) - ["What if you're wrong" - haplogroup
  J](https://www.gnxp.com/WordPress/2011/06/28/what-if-youre-wrong-haplogroup-j/) - [Out of Africa and out of
  Siberia](https://www.gnxp.com/WordPress/2012/01/27/out-of-africa-and-out-of-siberia/) - [The genetics of the descendants of the
  Garamantes](https://www.gnxp.com/WordPress/2009/07/02/the-genetics-of-the-descendants-of-the-garamantes/) - [mtDNA, fair game or
  folly?](https://www.gnxp.com/WordPress/2006/12/02/mtdna-fair-game-or-folly/) - [Molecular evolution of extinct
  lineages](https://www.gnxp.com/WordPress/2008/06/10/molecular-evolution-of-extinct-lineages/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F10%2Fman-at-bab-el-mandeb%2F&linkname=Man%20at%20%20Bab%20el-Mandeb "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F10%2Fman-at-bab-el-mandeb%2F&linkname=Man%20at%20%20Bab%20el-Mandeb "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F10%2Fman-at-bab-el-mandeb%2F&linkname=Man%20at%20%20Bab%20el-Mandeb "Email")[](https://www.addtoany.com/share)
