+++
title = "Domestication as"
full_title = "Domestication as introgression and assimilation"
date = "2011-06-13"
upstream_url = "https://www.gnxp.com/WordPress/2011/06/13/domestication-as-introgression-and-assimilation/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/06/13/domestication-as-introgression-and-assimilation/).

Domestication as introgression and assimilation

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)***The Pith:** the spread of domestic rice may be a function not of the spread of rice per se, as much as a specific narrow set of genes which confer domestication to disparate rice lineages.*

This has been a big month for rice. At least for me. Despite my [background as a rice-eater](http://www.genomesunzipped.org/2011/02/guest-post-by-razib-khan-my-personal-genome.php) I’ve generally moved away from it of late. It’s an American thing, as we’ve replaced a fear of fat with a fear of refined carbohydrates. My parents have even shifted from white rice to brown rice because of concerns with type 2 diabetes (this caused some consternation in 2004, as when we visited Bangladesh as honored guests my father was given to lecturing our hosts about the evils of the white rice on offer. Remember that in these societies brown rice is often considered the fair of the poor). But the reality is that much of the Old World of Asia still relies on rice, and will do so for the foreseeable future. So I still take an inordinate interest in the oriental staff-of-life. I already reviewed [two](http://blogs.discovermagazine.com/gnxp/2011/05/one-root-for-rice/) [papers](http://blogs.discovermagazine.com/gnxp/2011/06/why-rice-is-so-nice/) on rice genetics recently, but now it’s time for a third.

Some things are similar, some things are different. Again the stars of the show are the two cultivars of *O. ativa*, *indica* and *japonica*, and wild rice, *O. rufipogon*, from which the domestic varieties are presumably derived. The question at issue are the possible differences in the genealogy of the total genome background of rice cultivars vs. particular regions of the genome relevant for domestication. In other words, did the genes responsible for domesticate traits spread and sweep across different rice lineages? Or are different rice lineages simply derived from a common ancestor which carried the original domestication traits from a singular selection event? The first paper I reviewed suggested that there was one single domestication event, and that later differentiation between *indica* and *japonica* may simply have been a function of isolation and possible hybridization with local wild strains in the case of *indica*. The second paper focused on genes responsible for domestication seemed to imply that *indica* and *japonica* may have been shaped by different selection events (more precisely, they couldn’t detect signatures of selection in *indica* at the same loci that they did for *japonica)*. A new paper in *PLoS Genetics* seems to take a broader view, highlighting**both the phylogeny of the total genome as a whole and the bouts of natural selection which might have reshaped specific genes in a particular manner.** [Two Evolutionary Histories in the Genome of Rice: the Roles of Domestication Genes](http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1002100):

> The origin of two cultivated rice*Oryza sativa indica* and*O. sativa > japonica* has been an interesting topic in evolutionary biology. > Through whole-genome sequencing, we show that the rice genome embodies > two different evolutionary trajectories. Overall genome-wide pattern > supports a history of independent origin of two cultivars from their > wild population. **However, genomic segments bearing important > agronomic traits originated only once in one population and spread across all cultivars through introgression and human selection.** Population genetic analysis allows us to pinpoint 13 additional candidate domestication genes.

From what I can gather the paper which argued for a common origin of *indica* and *japonica* in the same domestication event covered a much smaller fraction of the genome than this paper. All things equal then I’d lean toward the assessment of the relationship between the two in this paper, though they aren’t particular unequivocal either as to the relationship between *indica* and *japonica*. As in the second paper on domestication genes and selection they found a lot more diversity on *indica* than *japonica*. In fact *indica* was only marginally less diverse than *rufipogon*. The big headline results are in figure 5. The top two panels, A and C, are just showing you θ across the respective chromosomes for each lineage. θ is just a measure of nucelotide diversity. The higher the θ, the more diverse the genomic region. R, J, and I are the three rice lineages, *rufipogon*, *japonica*, and *indica*. The second two panels show the **genetic distance between two pairwise lineages using Fst measures.** Fst just reports the amount of genetic diversity which is partitioned in group level differences (e.g., Fst across continental human races is \~0.15). As Fst approaches 0, that means all the genetic diversity can be reduced to variance within the groups. As Fst approaches 1, that means that all the diversity can be accounted for by noting the differences between groups.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/divrice.jpg?resize=350%2C715)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/divrice.jpg?resize=350%2C715)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/divrice.jpg)It’s pretty clear from these panels that *japonica* is the outlier when it comes to genetic diversity. It doesn’t have nearly as much as the other two lineages. This is in line with what the paper I blogged last week reported. In fact, there’s not much of a difference between *indicia* and *rufipogon* using this statistic. But when you move to between population differences an interesting pattern emerges. Across much of these two chromosomes (rice as 12 chromosomes by the way) the three lineages exhibit about equal genetic distance from each other, but there are segments of sharply decreased genetic distance between the two domestic cultivars! These are extended regions of reduced genetic diversity overall. **And very interestingly some genes implicated in domestication are found just here.** Of course it may be that the alleles which can drive domestication were part of the standing genetic variation of the wild lineages. This means that there’s a normal range of variation in a population, and independent selection events swept up in frequency the same alleles derived from the ancestral wild types. But this is not what it looks like when you examine the genomic regions around these alleles. They share the same exact variants, which implies a common selective sweep event in which an ancestral allele and its neighbors were the target of a powerful adaptive force.

The authors suggest that this peculiar constellations of disjunctions in the evolutionary histories of different regions of the genome can not be explained by simple demographic processes such as selfing, bottlenecks, etc. Rather, it is the outcome of a complex process whereby hybridization events allowed from the introgression of ancestral domestication genes from one lineage to another, and the sweeping to fixation of these domestication genes across two lineages with very different ancestries. In other words, **common traits don’t always emerge from common ancestors, but rather, often from common experiences**.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/distance.jpg?resize=300%2C282)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/distance.jpg?resize=300%2C282)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/distance.jpg)We know this for humans. The figure to the right is from [Genetic Evidence for the Convergent Evolution of Light Skin in Europeans and East Asians](http://mbe.oxfordjournals.org/content/24/3/710.long). A of this figure shows the total genetic distances between West Africans (WA), Europeans (EU), South Asians (SA), Island Melanesians (EA), and Native Americans (NA). Each successive panel shows genetic distances on specific loci implicated in variation in pigmentation. Notice how incongruent they are with the tree in A in many cases. Look at E, where Europeans are a clear outgroup on the locus, and all other populations cluster. This is a variant on [SLC45A2](https://en.wikipedia.org/wiki/SLC45A2) which is highly diagnostic of Europeans, and is correlated with light complexion (the minority of Europeans who carry the variant found in the rest of the world are much more likely to have dark eyes, black hair, and olive skin, than not). We don’t need to go into the possible evolutionary reasons why dark skin and light skin are distributed as they are here. **Rather, we can see in the case of this trait that the histories of the specific genes may not reflect the history as told in the total genome content, because of the importance of local adaptations and selection pressures.** The case of rice may be an inversion, insofar as instead of a location adaptation differentiating relatively closer groups (e.g., look at South Asians vs. Europeans in C, D, and E), it may be a case where very different groups exhibit similarities because of convergent evolution. That convergent pressure being the impulse toward domestication.

Speaking of which, that impulse may be related to the arrival of the [Munda populations](http://blogs.discovermagazine.com/gnxp/2010/10/sons-of-the-conquerers-the-story-of-india/) from Southeast Asia into India. If the model above is correct than *indica* was a local cultivar, perhaps at some stage of domestication, which was improved by hybridization with *japonica*. Eventually admixture swamped out most of the *japonica* genetic background, but recurrent selection favored *japonica* alleles at a large set of loci implicated in domestication. In this paper they lacked the power to ascertain whether the haplotypes jumped from *indica* to *japonica* or vice versa. But I’d be willing to bet \$250 dollars that it is going to turn out to have come from *japonica* because of other aspects of archeobotany, combined with *japonica*‘s extreme homogeneity, which may be a hallmark of a process of a very powerful set of selection events which gave rise to domestic rice.

**Citation:** Ziwen He, Weiwei Zhai, Haijun Wen, Tian Tang1, Yu Wang, Xuemei Lu, Anthony J. Greenberg, Richard R. Hudson, Chung-I Wu, & Suhua Shi (2011). Two Evolutionary Histories in the Genome of Rice: the Roles of Domestication Genes PLoS Genetics : [10.1371/journal.pgen.1002100](10.1371/journal.pgen.1002100)

### Related Posts:

- [I like introgressed
  rice](https://www.gnxp.com/WordPress/2007/08/09/i-like-introgressed-rice/) - [Introgressing toward becoming
  rice](https://www.gnxp.com/WordPress/2012/10/04/introgressing-toward-becoming-rice/) - [What is going on with plant
  domestication?](https://www.gnxp.com/WordPress/2012/10/03/what-is-going-on-with-plant-domestication/) - [One root for
  rice](https://www.gnxp.com/WordPress/2011/05/03/one-root-for-rice/) - [Open Thread,
  5/11/2014](https://www.gnxp.com/WordPress/2014/05/11/open-thread-5112014/) - [Friday Fluff - May 6th,
  2011](https://www.gnxp.com/WordPress/2011/05/06/friday-fluff-may-6th-2011/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F13%2Fdomestication-as-introgression-and-assimilation%2F&linkname=Domestication%20as%20introgression%20and%20assimilation "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F13%2Fdomestication-as-introgression-and-assimilation%2F&linkname=Domestication%20as%20introgression%20and%20assimilation "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F13%2Fdomestication-as-introgression-and-assimilation%2F&linkname=Domestication%20as%20introgression%20and%20assimilation "Email")[](https://www.addtoany.com/share)
