+++
title = "Why rice is so nice"
full_title = "Why rice is so nice"
date = "2011-06-07"
upstream_url = "https://www.gnxp.com/WordPress/2011/06/07/why-rice-is-so-nice/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/06/07/why-rice-is-so-nice/).

Why rice is so nice

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/400px-Rice_diversity.jpg?resize=600%2C202)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/400px-Rice_diversity.jpg?resize=600%2C202)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/400px-Rice_diversity.jpg)

***The Pith:** What makes rice nice in one varietal may not make it nice in another. Genetically that is….*

Rice is edible and has high yields thanks to evolution. Specifically, the artificial selection processes which lead to domestication. The “genetically modified organisms” of yore! The details of this process have long been of interest to agricultural scientists because of possible implications for the production of the major crop which feeds the world. And just as much of Charles Darwin’s original insights derived from his detailed knowledge of breeding of domesticates in Victorian England, so evolutionary biologists can learn something about the general process through the repeated instantiations which occurred during domestication during the Neolithic era.

A new paper in *PLoS ONE* puts the spotlight on the domestication of rice, and specifically the connection between particular traits which are the hallmark of domestication and regions of the genome on chromosome 3. These are obviously two different domains, the study and analysis of the variety of traits across rice strains, and the patterns in the genome of an organism. But they are nicely spanned by classical genetic techniques such as [linkage mapping](https://en.wikipedia.org/wiki/Genetic_linkage#Linkage_map) which can adduce regions of the genome of possible interesting in controlling variations in the phenotype.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)In this paper the authors used the guidelines of the older techniques to fix upon regions which might warrant further investigation, and then applied the new genomic techniques. Today we can now gain a more detailed sequence level picture of the genetic substrate which was only perceived at a remove in the past through abstractions such as the ‘genetic map.’[Levels and Patterns of Nucleotide Variation in Domestication QTL Regions on Rice Chromosome 3 Suggest Lineage-Specific Selection](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0020670?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+plosone%2FEvolutionaryBiology+%28PLoS+ONE+Alerts%3A+Evolutionary+Biology%29):

> *Oryza sativa* or Asian cultivated rice is one of the major cereal > grass species domesticated for human food use during the Neolithic. > Domestication of this species from the wild grass*Oryza rufipogon* > was accompanied by changes in several traits, including seed > shattering, percent seed set, tillering, grain weight, and flowering > time. Quantitative trait locus (QTL) mapping has identified three > genomic regions in chromosome 3 that appear to be associated with > these traits. We would like to study whether these regions show > signatures of selection and whether the same genetic basis underlies > the domestication of different rice varieties. Fragments of 88 genes > spanning these three genomic regions were sequenced from multiple > accessions of two major varietal groups in*O. sativa*—*indica* > and*tropical japonica*—as well as the ancestral wild rice species*O. > rufipogon*. In*tropical japonica*, the levels of nucleotide variation > in these three QTL regions are significantly lower compared to > genome-wide levels, and coalescent simulations based on a complex > demographic model of rice domestication indicate that these patterns > are consistent with selection. In contrast, there is no significant > reduction in nucleotide diversity in the homologous regions > in*indica* rice. **These results suggest that there are differences > in the genetic and selective basis for domestication between these two > Asian rice varietal groups.**

Here’s what seems relevant for the two domestic varieties from Wikipedia:

> *Oryza sativa* contains two major subspecies: the sticky, short > grained*japonica* or*sinica* variety, and the non-sticky, > long-grained*[indica](https://en.wikipedia.org/wiki/Indica_rice "Indica rice")* > variety.*Japonica* are usually cultivated in dry fields, in temperate > East Asia, upland areas of Southeast Asia and high elevations in South > Asia, while*indica* are mainly lowland rices, grown mostly submerged, > throughout tropical Asia….

There’s long been debate about the exact phylogenetic relationship between these two strains of domestic rice. More on that later. In regards to domestication there are three categories we need to focus on in terms of adaptation: 1) traits which are common to all domestic cereals and tend to crop up almost immediately, 2) traits which are extensions and improvements upon the initial domestic prototype, 3) traits which are regional diversifications, often adaptations to climate. Consider an analogy to horses. The original domestic horse was rather small, and was only fit for drawing chariots. Eventually the breeds became larger, and suitable for cavalry. Finally, there was a diversification by task (e.g., workhorses vs. race horses) and to some extent climate.

As noted above previous classical genetic techniques had narrowed down the genetic regions responsible for various domesticate traits when comparing *japonica* to the wild *rufipogon*. Since domestication usually entails a process of selection the authors naturally presumed that they might be able to detect signatures of selection within the genome. What are the genomic tells of selection?

There are many, just as there are different types of selection. In this case what we know suggests that due to \#1 there’s going to be an initial bout of adaptation and rapid shift from wild diversity to fixed traits suitable for a crop which is going to be controlled by humans. Just as the riotous diversity of the wild varieties become constrained to monocultures, so the diversity of the wild type often gets swept away by a few genetic variants which are responsible for the favored traits. So what they might see in the domestic varieties is a sharp reduction of variation around the quantitative trait loci (QTLs) reported earlier, because those QTLs have presumably been the target of selection. In other words, a [selective sweep](https://en.wikipedia.org/wiki/Selective_sweep).

That’s what they found. At least in one lineage.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/journal.pone_.0020670.g005.jpg?resize=600%2C194)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/journal.pone_.0020670.g005.jpg?resize=600%2C194)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/06/journal.pone_.0020670.g005.jpg)

Left to right you have *indica*, *japonica*, and *rufipogon*. Front to back in each chart you see the three QTLs, and the distribution of nucleotide diversities by genetic fragments within these QTLs. The extremely skewed distribution of the domestic varieties in relation to the wild type *rufipogon* is rather obvious. Additionally, you see a stronger skew in *japonica* in relation to *indica*. **The skew in the domestic strains is toward a greater proportion of the fragments having very low nucleotide diversity.**

What could cause this? You need a further piece of information here. The domestic varieties have long regions of the genome characterized by [linkage disequilibrium](https://en.wikipedia.org/wiki/Linkage_disequilibrium) (actually, *japonica* is so homogeneous that you barely have enough variation to calculate LD!). So particular genetic variants are associated with each other, resulting in long runs of similar sequences, haplotypes. It’s as if a chunk of some ancient chromosome just “blew up” and took over that segment of the genome in *japonica*.

Natural selection could do this. Imagine that an ancestral *rufipogon* has a genetic variant which confers a domestic trait. It would be selected. Even if crossed with other strains with other domestic characteristics its particular QTL would be transmitted down to the descendants in general. But not only would the specific genetic variant which conferred the favored trait be passed on, but many of the flanking genomic regions carrying other variants would also be transmitted! This explains the extremely low genetic diversity in *japonica*, if there’s a sweep up in frequency of a particular ancestral haplotype then what were polymorphisms in the wild type become monomorphic in the domesticate.

Another explanation though could be that demographic history produced these results. Random genetic drift due to small populations, whether via bottleneck or systematic inbreeding/selfing, can also drive up the frequency of alleles favored by lady-luck and render extinct all others. To check for this the authors constructed a model where *japonica* and *indica* went through bottlenecks enforced by the domestication (note that strong selection can drive down population size as well). Even with this model the diversity in *japonica* in these QTLs remained far too low (though *indica’*s skew did not reach statistical significance).

Since both of the domestic strains exhibit traits of domestication the lack of a selective event in *indica* at these QTLs does not allow us to infer that there are no genes which were selected for these traits in the past in *indica*. On the contrary, there certainly were and are such genes. But where are they? The authors moot the possibility that selection exists at the loci under consideration, but was simply missed because the selection was by a different dynamic which might not be picked up by their test. For various reasons they are skeptical of this on its own merits, but I think the bigger issue is that the original linkage mapping was performed with *japonica* vs. wild type strains, so naturally if the two domestic subspecies differed in their genetic architectures then the QTLs of interest of *indica* would not be discovered simultaneously.

Something which I’m rather perplexed by is how this comports or aligns with the finding by many of the same researchers that the [two domestic varietals derive from the same ancestral population](http://blogs.discovermagazine.com/gnxp/2011/05/one-root-for-rice/) which was domesticated from East Asian wild rice. It could be that the history of domestication is more serial than we know, and that the common QTLs to both *japonica* and *indica* have been rendered irrelevant by new adaptations subsequent to their separation. Or, one or the other may have experienced introgression at that locus and so diverged after domestication. Interestingly in [figure 7](http://www.plosone.org/article/slideshow.action?uri=info:doi/10.1371/journal.pone.0020670&imageURI=info:doi/10.1371/journal.pone.0020670.g007) of the paper they show that phylogenetic trees which illustrates the relationship of alleles associated with each strain. It indicates that *indica* is not monophyletic on these regions, while *japonica* is. This means that the *japonica* variants share a common ancestor, from which all are descended. In contrast, *indica* variants do not. Such a pattern is consistent with the story of strong positive selection upon a single variant at some time in the past for *japonica*.From what I can tell they may actually have sent the *PLoS ONE* paper to the reviewers before the *PNAS* paper which I reviewed earlier. Because these two papers were published so close to each other they don’t cite each other, though in some ways the first paper in *PNAS* would have fleshed out the natural history of domestic rice somewhat. As it is, they kind of leave of us hanging in relation to *indica*.

Why does all of this matter? Yes, agricultural genetics is important for agriculture. But let’s get back to people. **There is a hypothesis that man is a ‘self-domesticated’ organism.** Whatever quibbles I have with artificial terms like domestication I do think that there may be broad analogies to be drawn between our own species and the organisms associated with us.

**Citation:** Xianfa Xie1, Jeanmaire Molina, Ryan Hernandez, Andy Reynolds, Adam R. Boyko, Carlos D. Bustamante, & Michael D. Purugganan (2011). Levels and Patterns of Nucleotide Variation in Domestication QTL Regions on Rice Chromosome 3 Suggest Lineage-Specific Selection PLoS ONE : [10.1371/journal.pone.0020670](10.1371/journal.pone.0020670)

*Image Credit: [IRRI Images](http://www.flickr.com/people/86712369@N00)*

### Related Posts:

- [Artificial vs. natural
  selection](https://www.gnxp.com/WordPress/2008/07/02/artificial-vs-natural-selection/) - [Introgressing toward becoming
  rice](https://www.gnxp.com/WordPress/2012/10/04/introgressing-toward-becoming-rice/) - [What is going on with plant
  domestication?](https://www.gnxp.com/WordPress/2012/10/03/what-is-going-on-with-plant-domestication/) - [I like introgressed
  rice](https://www.gnxp.com/WordPress/2007/08/09/i-like-introgressed-rice/) - [The mists of the adaptive
  fog](https://www.gnxp.com/WordPress/2008/11/18/the-mists-of-the-adaptive-fog/) - [Domestication as introgression and
  assimilation](https://www.gnxp.com/WordPress/2011/06/13/domestication-as-introgression-and-assimilation/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F07%2Fwhy-rice-is-so-nice%2F&linkname=Why%20rice%20is%20so%20nice "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F07%2Fwhy-rice-is-so-nice%2F&linkname=Why%20rice%20is%20so%20nice "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F06%2F07%2Fwhy-rice-is-so-nice%2F&linkname=Why%20rice%20is%20so%20nice "Email")[](https://www.addtoany.com/share)
