+++
title = "Genomic diversity &"
full_title = "Genomic diversity & historical inference"
date = "2009-02-16"
upstream_url = "https://www.gnxp.com/WordPress/2009/02/16/genomic-diversity-historical-inference/"

+++
Source: [here](https://www.gnxp.com/WordPress/2009/02/16/genomic-diversity-historical-inference/).

Genomic diversity & historical inference

[p-ter](https://www.gnxp.com/blog/2009/02/from-genome-wide-data-to-insights-into.php) points me to a new paper, [Global distribution of genomic diversity underscores rich complex history of continental human populations](http://genome.cshlp.org/content/early/2009/02/12/gr.088898.108.abstract?papetoc):

> Characterizing patterns of genetic variation within and among human > populations is important for understanding human evolutionary history > and for careful design of medical genetic studies. Here, we analyze > patterns of variation across 443,434 SNPs genotyped in 3,845 > individuals from four continental regions. This unique resource allows > us to illuminate patterns of diversity in previously under studied > populations at the genome-wide scale including Latin America, South > Asia, and Southern Europe.**Key insights afforded by our analysis > include quantifying the degree of admixture in a large collection of > individuals from Guadalajara, Mexico; identifying language and > geography as key determinants of population structure within India; > and elucidating a North-South gradient in haplotype diversity within > Europe.** We also present a novel method for identifying long-range > tracts of homozygosity indicative of recent common ancestry. > Application of our approach suggests great variation within and among > populations in the extent of homozygosity suggesting both demographic > history (such as population bottlenecks) and recent ancestry events > (such as consanguinity) play an important role in patterning variation > in large modern human populations.

These authors did not start out with the HGDP sample. Rather, these data sets tended to come from urban locations, while the HGDP derive from more isolated groups. Interesting they found that the genetic clusters tended to be less discrete than in the HGDP sample; reasonable in light of the disparate sampling strategies. The first figure has STRUCTURE and PC based charts to display genetic variation. I’ve rotated it clockwise so as to increase resolution (the screen width is 500 pixels).

  
![pcbust1.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/pcbust1.jpg?resize=500%2C863)![pcbust1.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/pcbust1.jpg?resize=500%2C863)  
Here’s a figure from a [related paper](http://www.cell.com/AJHG/abstract/S0002-9297(08)00443-6):  
![bust2.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/bust2.jpg?resize=500%2C510)![bust2.jpg](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/bust2.jpg?resize=500%2C510)  
Note the similarity between Mexicans from Guadalajara and African Americans. Both are admixed populations, and their admixture is not equally distributed. This results in a wide range of genetic variation between the two source populations. South Asians overlap with Mexicans not because of shared recent ancestry, but the fact that the genetic distance between South Asians and East Asians is smaller than that between Europeans and East Asians (though South Asians are closer to Europeans than they are to East Asians). Interestingly rotating the figure shows the clear geographic variable when it comes to genetic variation in South Asia. The cluster on the PC chart replicates the rough triangular geography of South Asia. I looked in the data source and the “Dravidian” samples are all from the far south of India, with 2/3 of the South Asians being Indian Punjabis.  
Also, the paper notes that inbreeding is not equally distributed within a population. So those messed up people you see around….

### Related Posts:

- [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/) - [Complex history of archaic
  ancestry](https://www.gnxp.com/WordPress/2021/07/19/complex-history-of-archaic-ancestry/) - [Population substructure of Mexican
  Mestizos](https://www.gnxp.com/WordPress/2009/05/12/population-substructure-of-mexican-mestizos/) - [Reconstructing human origins in the genomic
  era](https://www.gnxp.com/WordPress/2006/08/18/reconstructing-human-origins-in-the-genomic-era/) - [Africans Americans mostly West African, but some
  mostly…](https://www.gnxp.com/WordPress/2009/12/23/africans-americans-mostly-west-african-but-some-mostly-european/) - [Population structure &
  Singapore](https://www.gnxp.com/WordPress/2009/09/05/population-structure-singapore/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F02%2F16%2Fgenomic-diversity-historical-inference%2F&linkname=Genomic%20diversity%20%26%20historical%20inference "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F02%2F16%2Fgenomic-diversity-historical-inference%2F&linkname=Genomic%20diversity%20%26%20historical%20inference "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F02%2F16%2Fgenomic-diversity-historical-inference%2F&linkname=Genomic%20diversity%20%26%20historical%20inference "Email")[](https://www.addtoany.com/share)
