+++
title = "Genomics to complex"
full_title = "Genomics to complex traits"
date = "2009-02-18"
upstream_url = "https://www.gnxp.com/WordPress/2009/02/18/genomics-to-complex-traits/"

+++
Source: [here](https://www.gnxp.com/WordPress/2009/02/18/genomics-to-complex-traits/).

Genomics to complex traits

My post below outlining the possible future of genomics and intelligence made me recall a paper from last fall, [Predicting Unobserved Phenotypes for Complex Traits from Whole-Genome SNP Data](http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1000231;jsessionid=76E982E3AA4A11C28F12E8FC3D543A0F):

> Results from recent genome-wide association studies indicate that for > most complex traits, there are many loci that contribute to variation > in observed phenotype and that the effect of a single variant (single > nucleotide polymorphism, SNP) on a phenotype is small. **Here, we > propose a method that combines the effects of multiple SNPs to make a > prediction of a phenotype that has not been observed.** We apply the > method to data on mice, using phenotypic and genomic data from some > individuals to predict phenotypes in other, either related or > unrelated, individuals. We find that correlations between predicted > and actual phenotypes are in the range of 0.4 to 0.9. The method also > shows that the SNPs used in the prediction appear in regions that are > known to contain genes associated with the traits studied. The prediction of unobserved phenotypes from high-density SNP data and appropriate statistical methodology is feasible and can be applied in human medicine, forensics, or artificial breeding programs.

The number of QTLs for the traits here is rather small, on the order of 15. Here’s some interesting numbers:

> For the data set on mice (\~2200 individuals and \~10,000 SNP), it > took \~15 minutes with a single CPU (\~2 GHz), which compares > favourably to a number of other computing strategies on the same data > set…Assuming that computing time increases linearly with the number of > individuals and markers, the method would run within one week even if > the data set was large (e.g. 10,000 individuals with 1,000,000 SNPs). More time may be required to adequately monitor convergence, however parallel computing strategies would be useful here…Therefore, the methods described in this study can scale up to much larger data sets.

There’s obviously a big difference between 2,200 mice and 2,200 humans. And it looks like these traits had relatively big effect loci controlling the variation, with moderate to very high heritabilities (\~0.50 to almost 1.0).

### Related Posts:

- [Following up association
  studies](https://www.gnxp.com/WordPress/2008/10/06/following-up-association-studies/) - [Complex traits & evolution - follow
  up](https://www.gnxp.com/WordPress/2008/12/18/complex-traits-evolution-follow-up/) - [Is the "missing heritability" right under
  our…](https://www.gnxp.com/WordPress/2010/06/21/is-the-missing-heritability-right-under-our-noses/) - [Epistasis and Genome-Wide Association
  Studies](https://www.gnxp.com/WordPress/2008/11/02/epistasis-and-genome-wide-association-studies/) - [A success for genome-wide assciation
  studies?](https://www.gnxp.com/WordPress/2008/07/02/a-success-for-genome-wide-assciation-studies/) - [Predicting someone's face: look at their
  parents](https://www.gnxp.com/WordPress/2012/09/14/predicting-someones-face-look-at-their-parents/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F02%2F18%2Fgenomics-to-complex-traits%2F&linkname=Genomics%20to%20complex%20traits "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F02%2F18%2Fgenomics-to-complex-traits%2F&linkname=Genomics%20to%20complex%20traits "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2009%2F02%2F18%2Fgenomics-to-complex-traits%2F&linkname=Genomics%20to%20complex%20traits "Email")[](https://www.addtoany.com/share)
