+++
title = "More markers, or more"
full_title = "More markers, or more populations?"
date = "2010-07-02"
upstream_url = "https://www.gnxp.com/WordPress/2010/07/02/more-markers-or-more-populations/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/07/02/more-markers-or-more-populations/).

More markers, or more populations?

Here’s a letter to *The American Journal of Human Genetics* worth reading, [Genetic Landscape of Eurasia and “Admixture” in Uyghurs](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2790568/):

> …In the papers…by Xu and Jin, the genetic structure of Uyghurs was > described by 8150 ancestry-informative markers (AIMs). These markers > estimated the admixture rate of the Uyghur population to be around 50% East Asian ancestry by comparing Uyghurs to East Asians and Europeans. However, we suspect that the estimate of Xu and Jin may be considerably biased by insufficient reference population coverage…. >
> **The difference between the estimate of Xu and Jin (52%) and our > estimate (31%) may stem from either the different population coverage > or the sample size.** We analyzed a different and larger sample of > Uyghur individuals (n = 48) than that analyzed by Xu and Jin….Their > small sample size may have contributed to their overestimation of the > European component to admixture (i.e., to cluster assignment). > However, the insufficient population coverage may be more responsible > for the difference than the sample size or the number of markers. > **Concerning the number of markers, it is known that a relatively > small but specifically selected number of AIMs can accurately predict > ethnicity proportion**…As the two papers of Xu and Jin have > demonstrated, the estimated admixture rates reported did not change > much regardless of whether they were using chromosome 21 data only or > the whole genome, and thus a large number of markers may not be > necessary to estimate the “admixture” rate of Uyghurs. When we > analyzed only the 12 markers with the highest F_(ST) values in our > samples…the Uyghurs had a 30.2% assignment at K = 2 to the Europe and > Western Asia cluster. This estimate was not significantly different > from the above 31.2% when using all 68 markers. We consider it unlikely that a different set of appropriately chosen SNPs would give a markedly different answer based on unpublished data on some of these same populations….

Basically the authors are arguing that you’d rather have a more diverse range of populations (to get more between population genetic variance) than just keep increasing the number of markers within individuals to really capture geographic diversity. Reference population matters. I know that 23andMe tells South Asians to expect to get back that they’re 70-90% “European,” with the balance “East Asian.” People with only Native American ancestry are going to be 75% “East Asian” and 25% “European.” These sorts of results from the reference populations are pretty misleading in my opinion. If you model the variation of all the world’s populations as the combination of variation of a few reference populations you’re getting a stylized fact which is confusing if you don’t know to interpret it correctly.

Below is figure one, where they show the difference between K = 2 and K = 6 (assume two or six ancestral populations for your data set). The map illustrates the distribution of K = 6, as the intensity of each color represents the current contribution in that region of a K ancestral group.

![gr1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/gr1.jpg?resize=600%2C1000)![gr1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/gr1.jpg?resize=600%2C1000)

### Related Posts:

- [Complex history of archaic
  ancestry](https://www.gnxp.com/WordPress/2021/07/19/complex-history-of-archaic-ancestry/) - [More and more evidence for 'complex demography' in
  archaic…](https://www.gnxp.com/WordPress/2021/07/19/more-and-more-evidence-for-complex-demography-in-archaic-ancestry/) - [Selection & African
  Americans](https://www.gnxp.com/WordPress/2009/12/23/selection-african-americans/) - [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/) - [Admixture analysis isn't wrong, it
  misleads](https://www.gnxp.com/WordPress/2016/09/19/admixture-analysis-isnt-wrong-it-misleads/) - [Why the Uyghurs as we didn't know them didn't exist
  until…](https://www.gnxp.com/WordPress/2019/05/11/why-the-uyghurs-as-we-didnt-know-them-didnt-exist-until-after-1000-ad/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F02%2Fmore-markers-or-more-populations%2F&linkname=More%20markers%2C%20or%20more%20populations%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F02%2Fmore-markers-or-more-populations%2F&linkname=More%20markers%2C%20or%20more%20populations%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F02%2Fmore-markers-or-more-populations%2F&linkname=More%20markers%2C%20or%20more%20populations%3F "Email")[](https://www.addtoany.com/share)
