+++
title = "One principal component"
full_title = "One principal component to rule them all?"
date = "2010-07-23"
upstream_url = "https://www.gnxp.com/WordPress/2010/07/23/one-principal-component-to-rule-them-all/"

+++
Source: [here](https://www.gnxp.com/WordPress/2010/07/23/one-principal-component-to-rule-them-all/).

One principal component to rule them all?

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)Despite the reality that I’ve [cautioned](http://blogs.discovermagazine.com/gnxp/2010/05/beyond-visualization-of-data-in-genetics/) against taking [PCA](https://en.wikipedia.org/wiki/Principal_component_analysis) plots too literally as Truth, unvarnished and without any interpretive juice needed, papers which rely on them are almost magnetically attractive to me. They transform complex patterns of variation which you are not privy to via your *gestalt* psychology into a two or at most three dimensional representation which can you can *grok* immediately. That is why [History and Geography of Genes](https://www.amazon.com/exec/obidos/ASIN/0691029059/geneexpressio-20) was so engrossing. You *recognize* patterns which were otherwise unrecognizable. But *how* you interpret those patterns, that’s a wholly different matter. And how those patterns arise is also not something one can ignore.

![price_fig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/price_fig1.jpg?resize=350%2C249)![price_fig1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/price_fig1.jpg?resize=350%2C249)First, let’s start with an easy case. To the left is a PCA plot with four populations. Nigerians, East Asians (Chinese + Japanese), Europeans (whites from Utah), and finally, African Americans. The x-axis is the first principal component of variation, and the y-axis the second. That means that the x-axis is the independent dimension of variation within the patterns of genetic data which explains the largest fraction of the total amount of genetic variation. The sum totality of the variation can be decomposed into an large set of independent dimensions which can be rank ordered from the largest explanatory components to the smaller ones, successively by number. In a human genetic context the first principal component invariably separates Africans from non-Africans, and the second principal component often maps onto a west-east axis from Europe to the New World. Subsequent principal components can often be useful in smoking out fine scale distinctions, or relationships which are confused by the existence of [similar but different signals](http://blogs.discovermagazine.com/gnxp/2010/01/how-columbus-was-not-a-seer/) in admixed populations.

The interpretation of this plot is rather easy. You see that African Americans lay along a continuum between Nigerians and Europeans, skewed toward Nigerians, with some outliers toward East Asians. We know from other genetic findings that \~20% of the African American ancestral quanta is European, but, that quanta is not equally distributed across the population. \~10% of the African American population is more than 50% European in ancestry, while 90% is less than 50% European. And so you have a distribution which reflects this variation. As for the outliers, I will speculate and suggest that these are indications of Native American ancestry among some African Americans.

The story I presented above is probably plausible as an explanation of the visual **because we have a wealth of historical data to corroborate the plausibility of that narrative.** The fit between the results from the technique of analysis of genetic variation and what scholars have long inferred from textual sources is relatively easy. It is far more difficult to look at a PCA plot, and generate a plausible narrative that you yourself accept with a high degree of confidence with little external support. It is with that caveat in mind that I present [Toward a more uniform sampling of human genetic diversity: A survey of worldwide populations by high-density genotyping](http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WG1-50J9GWY-1&_user=10&_coverDate=07/16/2010&_rdoc=1&_fmt=high&_orig=search&_sort=d&_docanchor=&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=dc13c2b0167c7c7b0d193afc06d86ee2):

> High-throughput genotyping data are useful for making inferences about > human evolutionary history. However, the populations sampled to date > are unevenly distributed, and some areas (e.g., South and Central > Asia) have rarely been sampled in large-scale studies. To assess human > genetic variation more evenly, we sampled 296 individuals from 13 > worldwide populations that are not covered by previous studies. By > combining these samples with a data set from our laboratory and the > HapMap II samples, we assembled a final dataset of \~ 250,000 SNPs in > 850 individuals from 40 populations. With more uniform sampling, the > estimate of global genetic differentiation (FST) substantially > decreases from \~ 16% with the HapMap II samples to \~ 11%. A panel of > copy number variations typed in the same populations shows patterns of > diversity similar to the SNP data, with highest diversity in African > populations. This unique sample collection also permits new inferences > about human evolutionary history. **The comparison of haplotype > variation among populations supports a single out-of-Africa migration > event and suggests that the founding population of Eurasia may have > been relatively large but isolated from Africans for a period of > time.** We also found a substantial affinity between populations from central Asia (Kyrgyzstani and Mongolian Buryat) and America, suggesting a central Asian contribution to New World founder populations.

The studies which came out of the original [HapMap](https://en.wikipedia.org/wiki/International_HapMap_Project) had northern Europeans, Yoruba from Nigerians, and Chinese & Japanese. These three populations can tell us a lot, but there’s something lacking in the coverage. The [HGDP](https://en.wikipedia.org/wiki/Human_Genome_Diversity_Project) sample is better. But specifically because of political considerations it was not feasible to collect Indian samples, so Pakistani ones are used in their stead. Additionally, the HGDP sample is a touch biased toward isolated and distinctive populations, such as the Kalash of Pakistan. This genetic distinctiveness is important to catalog because it is fast disappearing. But the Kalash are so unique because of their long history of isolation, so one can’t really use them as a proxy population for Pakistanis, as one could with Sindhis. The [POPRES](http://www.ncbi.nlm.nih.gov/pubmed/18760391) sample seems to complement the HGDP well, but I don’t see it being used so much. Since the next phase of the HapMap has more populations, some of the deficiencies which emerged with the utilization of just three terminal groups (in a World Island context) will soon no longer be an issue.

But until that time it’s nice when studies come out which close some of the gaps in our knowledge of world wide genetic variation. This is one such study. I’m somewhat familiar with the samples already because I’ve seen it in an analysis of Indian populations. It seems that it is somewhat skewed toward South and Southeast Asian populations, but hey, these are groups which need to draw the long straw sometimes as well.

Before I go any further I should mention that they use a SNP-chip with hundreds of thousands of markers. Additionally, they looked at copy number variation. Two rather different types of variation within the genome, probably to double check that the outcomes were the same. Population historical events which shape patterns of genomic variation would presumably have a similar large scale effect on both types of variation. In their results that checked out, or so they claimed, as the paper is a manuscript without the supplements attached.

Though there’s some interesting fine-grained analysis to be had, they draw some macro-scale and deep time inferences as well. First, you probably know the famous fact that 15% of variation in genes is between races, and 85% within races. That’s derived from the [Fst](https://en.wikipedia.org/wiki/Fixation_index) statistic, which is basically partitioning between and within population variance across two populations. Obviously the value of Fst varies by the set of populations you’re comparing. That between Mbuti Pygmies and Japanese is far higher than between Chinese and Japanese. Using the HapMap the Fst was 16%. About what you’d expect. To equalize sample sizes with the HapMap they randomly selected individuals from a pooled set grouped by continent from their populations, and calculated Fst. They found values around 11%. Why the difference? Because their data set included populations which were *between* the three clusters within the HapMap.

This is naturally not a surprising result at all, but it does reiterate one issue which sometimes crops up: Platonism in relation to race. The northern European whites in the HapMaps are the whites *par excellence*. Turks, who are perhaps more centrally located in the genetic variation of West Eurasian and North African peoples, what used to be termed “Caucasoid,” are “less white.” Similarly, Nigerians are more African than Ethiopians. Chinese and Japanese are more Asian than Burmese. And so forth. When modeling between group differences there is I think a somewhat old-fashioned tendency to consider some populations racial archetypes. That modulates the input which modifies the results somewhat. The analytical technique may be as cold as stone, but they are used by flesh and blood human beings.

There is also some funny business going on with haplotype and SNP heterozygosities which I think needs to be highlighted, and speaks to the fact that SNP-chips are not perfect. They’re tools, and human tools are impacted by arbitrary or instrumental choices humans make. Let me quote:

> We also compared the SNP and haplotype heterozygosity values in each > population (Figure 2B). These two quantities are generally highly > correlated, although there are several exceptions: First, SNP > heterozygosity is higher than haplotype heterozygosity in European and > Central Asian populations. This may reflect a SNP ascertainment bias, > since many of these polymorphisms were historically selected to > maximize heterozygosity in European populations. Second, the Pygmy > sample shows a low SNP heterozygosity despite relatively high > haplotype heterozygosity. This unusual pattern could be caused by > stronger effects of SNP ascertainment bias in this population than in > others. Indeed, a recent study of Khoisan individuals (another > hunter-gatherer group from Africa) showed a similar pattern: despite > high SNP heterozygosity (\~60%) in whole-genome sequence data, a Khoisan individual showed low heterozygosity on the SNP microarray genotypes (\~22%) . Alternatively, this difference could also reflect unique attributes of population history.

In plain English the gene chips were designed with Europeans in mind, so they don’t necessarily pick up all the variation in non-European groups, who are believe it or not genetically different. This issue cropped up (as alluded to in the above text) with the [recent paper](http://www.nature.com/nature/journal/v463/n7283/full/nature08795.html) which sequenced some Bushmen as well as Desmond Tutu. The Bushmen have a lot of variation, this is well known, but they have variation at markers where Europeans don’t, and if Europeans don’t the chips may not look for polymorphism at that locus. This sort of thing probably doesn’t affect broad population relationships, but if you want to zoom in and do analysis which is sensitive to fine distinctions and quantitative differences, then it might be problematic.

Let’s jump to the pretty charts. First, a PCA plot with all of the individuals from all of the populations:

![indo1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo1.png?resize=572%2C532)![indo1](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo1.png?resize=572%2C532)

Note that PC 1 accounts for nearly eight times as much variation as PC 2. This speaks to the African vs. non-African gap. Because their data set is relatively thick in “intermediate” groups you see a spectrum. The vertical axis is obviously mostly east-west. And here’s the accompanying bar plot derived from the ADMIXTURE program. K = putative ancestral populations.

![indo2](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo2.png?resize=560%2C563)![indo2](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo2.png?resize=560%2C563)

With this many populations at K = 12 I think you could write a fantasy novel worthy of Tolkien. K = 4 is more realistic. Among the African populations you see likely Eurasian admixture in some eastern, and it seems Bushmen, individuals. In Eurasia itself you see a clinal gradation of admixture between putative ancestral components that seems to follow longitude rather well.

Because so much of the variation in the total sample is due to Africans, removing them from the picture will allow us to focus more on the relationships of the Eurasian groups. And so that’s exactly what they did. **Note that focusing on the Eurasian groups does not mean simply magnifying or zooming in on the Eurasian section of the PCA plot, rather, the plots are regenerated with a subset of the previous genetic variation.** In other words, the dimensions will shake out a bit differently.

The first plot shows Eurasian populations as a whole. The second removes Europeans and Near Easterners.

![indo3](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo3.png?resize=520%2C840)![indo3](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo3.png?resize=520%2C840)

Notice again the scale. The vast majority of the variance seems to be east-west. But, there is a noticeable north-south split. For the South Asian population it looks like they had Pakistanis who were farmers of modest means ([Arain](https://en.wikipedia.org/wiki/Arain)), high caste South Indians, and very low caste or tribal South Indians. For this Indian sample there’s a problem, and it’s the sample problem which plagued the [Up Series](https://en.wikipedia.org/wiki/Seven_Up!), they are looking at the very top and bottom of Indian society and ignoring the middle. Presumably the middle is going to be somewhere in the middle genetically as well, but nevertheless that’s something to consider in a paper which presumes to fill in the patchiness of others. In contrast, the Nepali sample was notably ethnically diverse, including both the dominant Indo-Aryan segment as well as the Tibeto-Burman [Newar](https://en.wikipedia.org/wiki/Newa_people).

In the first panel there are some curious patterns with the Southeast Asian groups. Culturally, as in language and history, the Thai and Vietnamese have relatively recent roots in the southern regions of modern China. The [Dai](https://en.wikipedia.org/wiki/Dai_people) of Yunnan are the same people in origin as the Thai of Thailand and the Lao of Laos. Both derive from migrations from Yunnan. This is historically attested, even if somewhat fragmentarily. The heartland of the Vietnamese was in the Red River valley and north into southern China, and they spread down the coast and toward the Me kong only within the last 1,000 years. Southeast Asia was not uninhabited during this period. It was dominated by the Khmer Empire, which was slowly consumed by the expanding Thai and Vietnamese polities. Some scholars argue that French colonialism actually preserved an independent Khmer nation, which otherwise would have been divided between Thailand and Vietnam, as Poland was between Germany and Russia. So the Khmer are the indigenous people, while the Thai and Vietnamese are intrusive.

What do the PCA plots tell us? I do not know where the Vietnamese samples were collected. If they were from South Vietnam, then their close position to the Chinese suggests to me that there was substantial demographic replacement or expansion from the Red River valley. In contrast, the Thai are relatively distant from the Chinese. In fact, the Cambodians are somewhat closer to the Chinese! The samples here are small, and the sets overlap, so I wouldn’t put too much stock in that. But, Thailand is geographically closer to South Asia, so isolation by distance models would predict this pattern. It seems that the ethnogenesis of the Thai occurred through the expansion of the Thai identity, likely among Khmer peoples. And it is intriguing that the Iban, an indigenous people of Borneo, are closer to the Vietnamese than they are to the Cambodians. We know that there was substantial migration between coast Vietnam and Maritime Southeast Asia, the [Chams](https://en.wikipedia.org/wiki/Cham_people) of central Vietnam, and dominant in the southern half of the nation before the Vietnamese expansion, are a Malayan people who may have migrated from Borneo.

Shifting to the second panel there’s more here to say about the South Asians. First, geography. The two lower caste groups are actually Dalits from Andhara Pradesh, a South Indian state. Dalits used to be called outcastes, so they aren’t even lower caste, but without caste. The upper caste groups are Brahmins from Andhara Pradesh and Tamil Nadu. Finally, the Irula are tribal people from Tamil Nadu. To me the tribal samples often produce weird results, and I suspect that has to do with population bottlenecks and their demographic isolation. People leave the tribes (becoming part of the Hindu society, or converting to Islam or Christianity), but few join them. The Pakistani sample are Araina, a group of conventional Punjabi farmers who have a made up ancestry from Arabs (obviously made up because they don’t cluster with Near Easterners). Let’s compare to a chart from [Reich et al.](http://blogs.discovermagazine.com/gnxp/2009/09/indians-as-hybrids-a-k-a-aryan-invasion-in-the-house/):

![indiareich7](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indiareich7.png?resize=500%2C493)![indiareich7](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indiareich7.png?resize=500%2C493)

It seems to me that they’re in rough agreement (Reich et al. uses the same two low caste groups for Andhara Pradesh for low caste South Indians by the way). Though South Indian Brahmins speak South Indian languages, and reside amongst other South Indian groups, their genetic heritage is somewhat different. Similarly, tribal peoples are also distinct from caste Hindus. Reich et al. posit that South Asians can be modeled as a composite of two groups, Ancestral North Indians, ANI, and Ancestral South Indians, ASI. Presumably the former are intrusive to the subcontinent in relation to the latter. There seem two clear dimensions along which the ratio of ANI to ASI vary: geography and caste. The proportion of ASI seems to increase from the northwest to the southeast. And, the proportion of ANI seems to increase from tribal to low caste to upper caste. The Pakistani sample does not seem to be from an elite caste (or it does not seem they were converted from an elite caste), but they have more affinity with West Eurasian populations than South Indian Brahmins. It is likely that the latter are intrusive to the south, and have admixed with the local population.

Finally, a word on the Nepali sample. On top of the ANI-ASI mixture, the Nepali groups have varying levels of Tibeto-Burman, and so East Asian, affinity. This is not a surprise if you have met Nepalis. The Assamese, and to a lesser extent Bengalis, also exhibit this pattern of Tibeto-Burman admixture. The Brahmins of Nepal are intrusive like the Brahmins of South India, and like the South Indians they admixed with the local substrate.

Next let’s move to a ADMIXTURE plot.

![indo6](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo6.png?resize=552%2C522)![indo6](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2010/07/indo6.png?resize=552%2C522)

The selection of a particular K obviously is conditioned by the patterns which “fit” with what you know, and what you *expect*. With that caution aired, the population represented by red can easily be thought of as a Middle Eastern group which expanded with agriculture. That seems to be what the authors favor. The brown population is the modal Indian ancestral population, which has little presence outside the subcontinent (nice color coding by the way! Brown people are brown). A green color represents a population which the tribal group, the Irula, are heavily weighted on. This reminds me too much of the Kalash. I suspect that the Irula went through some bottleneck or other distinctive event, and some have assimilated to various low status groups in South India.

I’m not a fantasist intent on world-building, so I’ll stop with that in reading the tea leaves of the charts. But there’s an important section which I skipped over, and will move back to now. And that’s the deep time aspect:

> A more likely explanation for the OoA bottleneck is that Eurasia was > populated by a larger population that had been relatively isolated > from other modern human populations for tens of thousands of years > prior to the expansion. The first fossil evidence for modern humans > outside of Africa is in the Middle East at Skhul and Qafzeh between > 80,000-100,000 years ago, which is at least 20,000 years prior to the > Eurasian diaspora. If a population of modern humans remained in the > Middle East until the expansion into Eurasia, there would have been > sufficient time for genetic drift to reduce heterozygosity > dramatically before the Eurasia expansion. This “Middle East > isolation” hypothesis provides a robust explanation for the relative > homogeneity of European and Asian populations relative to African > populations (see Figures 3A-B) and is supported by a recent maximum > likelihood estimate of 140,000 years ago for the time of Eurasian-West > African population separation . Interestingly, a recent study of the > Neandertal genome suggests that the non-African individuals, but not > the Africans, contain similar amount of admixture (1-4%) with the > Neandertals . **The authors suggest that the admixture must have > happened between the Neandertals with an ancestral non-African > population before the Eurasian expansion.** Given the fossil, archaeological, and genetic evidence, the Middle East isolation hypothesis warrants rigorous evaluation as whole-genome sequence data become available.

Like the vast majority of genetic studies this work supports the Out of Africa hypothesis. Non-Africans are all branches from a specific African branch. Or more accurately, an African branch which left Africa. The reduction in heterozygosity, a measure of genetic variation, from Africa to Eurasians was large. Additionally, within Africa south of the Sahara there’s little difference in heterozygosity as a function of geography, but outside of Africa it drops off as a function of distance from Africa. A plausible model then is a radiation from a small ancestral population to the four corners of the world, going through a series of bottlenecks along the way. Or at least that’s a model supported by genomic data. But, the drop in heterozygosity is so great a quick separation from the parental African population would require an implausibly small number of founders (less than 10 in one generation). So, to explain the data, they are suggesting here that the original population was not quite so small, but was isolated from the large African population for thousands of years. They assume genetic drift reduced heterozygosity, but if the model is correct I suspect that the way it worked was that bottlenecks due to climatic fluctuations swept clean a lot of the genetic variation. But in the interregnum the isolated population may have interbred with Neandertals. In fact, perhaps they picked up genes from Neandertals when their own effective population was extremely small.

In any case, a wide ranging paper. They manage to tie their results into two [other](http://blogs.discovermagazine.com/gnxp/2009/09/indians-as-hybrids-a-k-a-aryan-invasion-in-the-house/) [blockbuster](http://blogs.discovermagazine.com/gnxp/2010/05/the-three-layers-of-the-neandertal-cake/) papers.

H/T [Dienekes](https://dienekes.blogspot.com/)

**Citation** Xing J, Watkins WS, Shlien A, Walker E, Huff CD, Witherspoon DJ, Zhang Y, Simonson TS, Weiss RB, Schiffman JD, Malkin D, Woodward SR, & Jorde LB (2010). Toward a more uniform sampling of human genetic diversity: A survey of worldwide populations by high-density genotyping. Genomics PMID: [20643205](http://www.ncbi.nlm.nih.gov/pubmed/20643205)

### Related Posts:

- [A picture is worth a thousand words, part
  n](https://www.gnxp.com/WordPress/2008/04/21/a-picture-is-worth-a-thousand-words-part-n/) - [What L. L. Cavalli-Sforza got
  wrong?](https://www.gnxp.com/WordPress/2008/04/21/what-l-l-cavalli-sforza-got-wrong/) - [Chimpanzee & human
  speciation](https://www.gnxp.com/WordPress/2009/08/21/chimpanzee-human-speciation/) - [Patterns of admixture in Latin American
  mestizos](https://www.gnxp.com/WordPress/2008/03/21/patterns-of-admixture-in-latin-american-mestizos/) - [Carbs &
  ancestry](https://www.gnxp.com/WordPress/2009/12/10/carbs-ancestry/) - [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F23%2Fone-principal-component-to-rule-them-all%2F&linkname=One%20principal%20component%20to%20rule%20them%20all%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F23%2Fone-principal-component-to-rule-them-all%2F&linkname=One%20principal%20component%20to%20rule%20them%20all%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2010%2F07%2F23%2Fone-principal-component-to-rule-them-all%2F&linkname=One%20principal%20component%20to%20rule%20them%20all%3F "Email")[](https://www.addtoany.com/share)
