+++
title = "The Iranian Genome"
full_title = "The Iranian Genome Project"
date = "2012-02-07"
upstream_url = "https://www.gnxp.com/WordPress/2012/02/07/the-iranian-genome-project/"

+++
Source: [here](https://www.gnxp.com/WordPress/2012/02/07/the-iranian-genome-project/).

The Iranian Genome Project

A friend pointed out that the [Iranian Genome Project](http://irangenes.com/) site has been around for half a year. The [research team](http://irangenes.com/about/) is reputable. You can see the [rationale](http://irangenes.com/updatesblog/) at their blog. Though there are several Middle Eastern populations in the HGDP, and you can get other data sets as well, **the total number of markers tends to be modest in comparison to the HapMap, let alone a full genome sequence.** If you’re going to look for associations with particular traits (e.g., diseases) then it is best to have as many variants as possible (even if you have enough markers so that you can localize the region of the genome where there is an association, that is less useful than having the exact functional variant of interest pegged).



On a minor note the [Iranian Genome Project](http://irangenes.com/)blog keeps using the word ‘Caucasian’ as if it is exclusive of Middle Eastern populations. This is amusing because one of the Caucasian people, the [Armenians](https://en.wikipedia.org/wiki/File:Armenian_Empire.png), are unquestionably Middle Eastern. The ethnic terminology used in the medical literature can be rather ill-suited to the task of biological taxonomy. When it comes to *social* surveys I don’t have much hesitation checking ‘Asian American,’ but it really gets on my nerves when that category pops up in*medical* contexts. Asian American as a social category which is inclusive of East, Southeast, and South Asians, has some leg to stand on, even if you think it is artificial. But as a biological category it is not defensible.

But the bigger issue which I’m wondering about is the impact of high levels ofconsanguinity on attempts to smoke out genome-wide and population-wide associations. If Middle Eastern populations are strongly structured because of recent endogamy then it may be that genetic drift may result in a situation where many of the diseases susceptibilities are highly particular to small segments of a given population. Of course this doesn’t mean that you can’t start somewhere. But it does suggest that the utility of these projects is sensitive to the particular evolutionary genetic parameters which characterize a given population (e.g., the homogeneity of the Chinese vs. the diversity of the Yoruba, the high levels of inbreeding in the Middle East vs. low levels in Europe).

### Related Posts:

- [Personal Genome
  Project](https://www.gnxp.com/WordPress/2007/12/03/personal-genome-project/) - [How big does the N need to
  be?](https://www.gnxp.com/WordPress/2009/03/31/how-big-does-the-n-need-to-be/) - [The Tuatara
  genome](https://www.gnxp.com/WordPress/2013/06/17/the-tuatara-genome/) - [The Malagasy Ancestry
  Project](https://www.gnxp.com/WordPress/2012/05/10/the-malagasy-ancestry-project/) - [Structural polymorphisms and
  SNPs](https://www.gnxp.com/WordPress/2006/06/30/structural-polymorphisms-and-snps/) - [10,000 genome at 30x & human
  variation](https://www.gnxp.com/WordPress/2016/07/03/10000-genome-at-30x-human-variation/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F02%2F07%2Fthe-iranian-genome-project%2F&linkname=The%20Iranian%20Genome%20Project "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F02%2F07%2Fthe-iranian-genome-project%2F&linkname=The%20Iranian%20Genome%20Project "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2012%2F02%2F07%2Fthe-iranian-genome-project%2F&linkname=The%20Iranian%20Genome%20Project "Email")[](https://www.addtoany.com/share)
