+++
title = "One root for rice"
full_title = "One root for rice"
date = "2011-05-03"
upstream_url = "https://www.gnxp.com/WordPress/2011/05/03/one-root-for-rice/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/05/03/one-root-for-rice/).

One root for rice

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/RiceYield.jpg?resize=600%2C264)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/RiceYield.jpg?resize=600%2C264)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/RiceYield.jpg)

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)***The Pith:** There was one rice domestication event somewhere in central China about 10,000 years ago. Probably!*

Going by the numbers rice is the real staff of life. Rice is the staple for \~50% of humans alive today. So the science of rice is of major pragmatic importance of all humans (even if a major disease which impacts rice doesn’t result in mass starvation, it will probably generate a price spike for all other staples due to Asian demand). One major issue which I’ve kept track of over the years has been the *origin* of rice agriculture: **was it a parallel multi-hearth origination or a single-hearth event?** We know that there were at least two instantiations of agricultural civilization in the world without any cultural diffusion: in the New World and the Old. More likely there were at least several independent hearths in the old world which utilized local wild crops. Wheat and barely in the west, millet and rice in the east, etc. But there is also a model that rice agriculture had two independent origins, in India and China, which gave rise to the *indica* and *japonica* strains from the welter of wild rice lineages. Some genetics has supported the model of two hearths by reporting a deep time depth to the last common ancestor of these strains, on the order of \~100,000 years. The implication from phylogenetics is that there were two adaptations of local lines, which later converged in morph due to parallel selection pressures.

The interest in this issue has application to our understanding of human history. Peter Bellwood in [First Farmers](https://www.amazon.com/exec/obidos/ASIN/0631205667/geneexpressio-20) argues that what L. L. Cavalli-Sforza termed the “great human diasporas” have their roots ultimately in distinctive domestication events. Bellwood goes on to suggest that the contemporary genetic and linguistic patterns of variation we see around us are the products of rapid population growth of these ancient agricultural nuclei (the genome blogger [Digonenes works within this framework](https://dioegenesartemis.blogspot.com/)). The implication then would be that two domestications of rice would imply two population pulses. A single domestication would imply one pulse. Therefore there is a connection here between historical human population genetics and agricultural history & genetics. This stands to reason in that our own species is so parasitic on domestic crops.

[](https://dioegenesartemis.blogspot.com/)

[  
](https://dioegenesartemis.blogspot.com/)[I recently noted](http://blogs.discovermagazine.com/gnxp/2011/04/resolutions-in-the-indian-genetic-layer-cake/) recently the issue of rice agriculture has particular relevance for the peopleing of India. Some archaelogists have hypothesized that rice agriculture was associated originally with the Austro-Asiatic Munda people, and only later spread to Indo-Aryan and Dravidian populations. As it happens the Munda turn out to have a strong imprint of East Asian ancestry, on the order of 15-30%. And most importantly, they carry a genetic variant on the *EDAR* locus which has been specifically associated with East Asian groups, and seems to have been the target of natural selection. The implication being that the last common ancestor of the Indian Munda and East Asians is unlikely to pre-date the Holocene by much. And critically non-Munda tribal populations *lack* this variant, indicating that *EDAR* is exogenous to the subcontinent, and not part of the natural variation.

But *if* the Munda are intrusive, it seems peculiar that they would reinvent rice agriculture *de novo*. Or, have to adopt it from the natives. In addition, the Munda have maintained their linguistic and cultural distinctiveness down to the present, indicating some sort of advantage over the locals allowing them to maintain the coherency of their identity before being absorbed by the substrate (the majority of their ancestry does look to be indigenous South Asian, so this is a case where cultural forms persisted to a greater degree of distinctiveness beyond the genetic aspect).

A recent paper in *PNAS* looks to resolve the issue in relation to rice. It argues that the deep coalescence times between *indica* and *japonica* lineages are just an artifact of incomplete lineage sorting. [Molecular evidence for a single evolutionary origin of domesticated rice](http://www.pnas.org/content/early/2011/04/27/1104686108.abstract):

> Asian rice,*Oryza sativa*, is one of world’s oldest and most > important crop specie**s. Rice is believed to have been domesticated > ∼9,000 y ago, although debate on its origin remains contentious.** A > single-origin model suggests that two main subspecies of Asian > rice,*indica* and*japonica*, were domesticated from the wild > rice*O. rufipogon*. In contrast, the multiple independent > domestication model proposes that these two major rice types were > domesticated separately and in different parts of the species range of > wild rice. This latter view has gained much support from the > observation of strong genetic differentiation between*indica* > and*japonica* as well as several phylogenetic studies of rice > domestication. We reexamine the evolutionary history of domesticated > rice by resequencing 630 gene fragments on chromosomes 8, 10, and 12 > from a diverse set of wild and domesticated rice accessions. Using > patterns of SNPs, we identify 20 putative selective sweeps on these > chromosomes in cultivated rice. **Demographic modeling based on these > SNP data and a diffusion-based approach provide the strongest support > for a single domestication origin of rice. Bayesian phylogenetic > analyses implementing the multispecies coalescent and using previously > published phylogenetic sequence datasets also point to a single origin > of Asian domesticated rice.** Finally, we date the origin of > domestication at ∼8,200–13,500 y ago, depending on the molecular clock estimate that is used, which is consistent with known archaeological data that suggests rice was first cultivated at around this time in the Yangtze Valley of China.

Here’s an illustration of the two alternative models:

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ricp.jpg?resize=600%2C168)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ricp.jpg?resize=600%2C168)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ricp.jpg)

So what’s with incomplete lineage sorting? If the divergence of two populations is relatively recent in time you won’t get a clean separation of genetic variants which are derived from common ancestors, and so can give you a good proxy for the population divergence time. In plain **English the general problems is that the phylogeny of genes is not identical to the phylogeny of populations.** The phylogeny of genes
can give you a good sense of the phylogeny of populations, but there are limits and pitfalls.

In this specific case the authors surmise that what happened is that genetic variants with last common ancestors far predating the divergence were used as proxies. These genes seem to have had a lot of ancestral variation. To give a human illustration there are HLA alleles, which are implicated in immune response, which are shared between some chimpanzees and some humans. The coalescence at these loci far predates the emergence of the human species, though in this case probably this is due to frequency dependent dynamics (low frequency alleles have the highest fitness and so never go extinct). When it comes to a young population a lot of the ancestral variance sorted in all sorts of ways can cause confusions, when you really want to focus in *derived* traits which allow you to differentiate populations with distinctive evolutionary histories.

One variable also at work with rice, as with most plants, is the possibility of introgression across the two subspecies. In particular, the adaptive introgression of traits which are favored in domestic lineages. This could explain recent common ancestry on a subset of adaptive alleles which were subject to selective sweeps across all rice lineages, no matter ancestry. A recent common ancestry would obviate the need for a lot of introgression to explain similarities across very divergent lineages. These support a single origin for the two domestic rice subspecies.

In this paper the authors used two general methods. First, they tested different population histories with resequenced genomes for several extant lineages of rice, the two domestic ones and a wild one which is the putative parent. For example, two separation events vs. three vs. four. Additionally, they controlled for the selective sweeps which would distort phylogenetic signals, since these are giving less ancestral informative history because of the confound of local adaptation.

Second, they did more conventional phylogenetic analysis, but using a “multispeciescoalescent” framework, which takes into account ancestral variation. This is obviously important if the last common ancestor of two populations was relativley recent, not allowing for the ancestral variation to slowly be pruned by the normal process of turnover in the genome through neutral processes. With this method they found a common origin recently in time for domestic rice lineages. It was also notable that the consensus on the single origin model was sensitive to the number of markers included in the study. This goes to show that for many non-human lineages we’re at a state of knowledge which is far inferior to where we are for our species. A Low number of genetic variants to ascertain population history can still cause confusions (as they did for humans).

If this result holds up it suggests to us how rare and overwhelming agricultural innovation really was. And perhaps we should be just a touch more worried at our species’ reliance on just a few lineages, either directly through consumption, or via what we feed our own domesticates.

**Citation:** Jeanmaire Molina, Martin Sikora, Nandita Garud, Jonathan M. Flowers, Samara Rubinstein, Andy Reynolds, Pu Huang, Scott Jackson, Barbara A. Schaal, Carlos D. Bustamante, Adam R. Boyko, & and Michael D. Purugganan (2011). Molecular evidence for a single evolutionary origin of domesticated rice PNAS : [10.1073/pnas.1104686108](https://dx.doi.org/10.1073/pnas.1104686108)

*Image credit: [AndrewMT](https://en.wikipedia.org/wiki/File:RiceYield.png)*

### Related Posts:

- [I like introgressed
  rice](https://www.gnxp.com/WordPress/2007/08/09/i-like-introgressed-rice/) - [Introgressing toward becoming
  rice](https://www.gnxp.com/WordPress/2012/10/04/introgressing-toward-becoming-rice/) - [Open Thread,
  5/11/2014](https://www.gnxp.com/WordPress/2014/05/11/open-thread-5112014/) - [Friday Fluff - May 6th,
  2011](https://www.gnxp.com/WordPress/2011/05/06/friday-fluff-may-6th-2011/) - [Rice, alcohol and
  genes](https://www.gnxp.com/WordPress/2010/01/25/rice-alcohol-and-genes/) - [What is going on with plant
  domestication?](https://www.gnxp.com/WordPress/2012/10/03/what-is-going-on-with-plant-domestication/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F05%2F03%2Fone-root-for-rice%2F&linkname=One%20root%20for%20rice "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F05%2F03%2Fone-root-for-rice%2F&linkname=One%20root%20for%20rice "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F05%2F03%2Fone-root-for-rice%2F&linkname=One%20root%20for%20rice "Email")[](https://www.addtoany.com/share)
