+++
title = "The mess that is mouse"
full_title = "The mess that is mouse"
date = "2011-05-31"
upstream_url = "https://www.gnxp.com/WordPress/2011/05/31/the-mess-that-is-mouse/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/05/31/the-mess-that-is-mouse/).

The mess that is mouse

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/800px-Apodemus_sylvaticus_bosmuis.jpg?resize=105%2C120)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/800px-Apodemus_sylvaticus_bosmuis.jpg?resize=105%2C120)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/800px-Apodemus_sylvaticus_bosmuis.jpg)Recently an evolutionary geneticist told me that his colleagues who worked with mice really didn’t have their stuff together. Actually, his language was a touch more colorful than that. But the gist of the argument seemed plausible enough to me. I tend to avoid reading papers using the mouse as a model organism in genetics because I recall getting confused by the pedigrees and various strain acronyms and abbreviations (nonstandard acronyms and abbreviations have also been a problem for me whenever I try to read developmental genetics). If I want to look at the genetics of a mammal besides a human being I often like to focus on dogs. The breeds of dogs actually mean something to me. There’s only so many skinned mouse hides I want to stare at.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)With all that said there is a huge scientific complex devoted to the mouse. If the house of mouse is a mess, then someone needs to do cleaning at some point. A new paper in *Nature Genetics* starts the process, using [SNPs](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism) and variable intensity oligonucleotides (VINOs) to assess the relationships between distinct lab strains as well as wild subspecies. Ignorant as I am of the biology of the mouse I was vaguely aware that like [*elegans*](https://en.wikipedia.org/wiki/Caenorhabditis_elegans#Laboratory_uses) much of the laboratory stock derives from a very small founding population. In psychology there’s the problem of the outlook and dispositions of Western university students getting extrapolated to the whole species of man, so I have wondered about this problem for some of the [model organisms](https://en.wikipedia.org/wiki/Model_organism) now then. If you’re studying very general biological processes this shouldn’t be that big of an issue, but for evolution obviously characterizing the nature of variation is of the essence.

The paper title & the abstract,[Subspecific origin and haplotype diversity in the laboratory mouse](http://www.nature.com/ng/journal/vaop/ncurrent/abs/ng.847.html):

> Here we provide a genome-wide, high-resolution map of the phylogenetic > origin of the genome of most extant laboratory mouse inbred strains. > Our analysis is based on the genotypes of wild-caught mice from three > subspecies of Mus musculus. We show that classical laboratory strains > are derived from a few fancy mice with limited haplotype diversity. > **Their genomes are overwhelmingly Mus musculus domesticus in origin, > and the remainder is mostly of Japanese origin.** We generated > genome-wide haplotype maps based on identity by descent from fancy > mice and show that classical inbred strains have limited and > non-randomly distributed genetic diversity. In contrast, wild-derived > laboratory strains represent a broad sampling of diversity within M. > musculus. Intersubspecific introgression is pervasive in these strains, and contamination by laboratory stocks has played a role in this process. The subspecific origin, haplotype diversity and identity by descent maps can be visualized using the Mouse Phylogeny Viewer….

There are four subspecies of *Mus musculus* described in the paper:

– *Mus musculus castaneus* (southern and southeastern Asia)  
– *Mus musculus domesticus* (western Europe, southwestern Asia, Americas, Africa, and Oceania)  
– *Mus musculus musculus* (eastern Europe and northern Asia)  
– *Mus musculus molosinus* (Japan)

There were 198 mice genotyped, including 36 wild-caught mice, 62 wild-derived laboratory strains and 100 classical laboratory mice. The SNP-chip was tuned to the variation present within*domesticus*, so there was some [ascertainment bias](https://en.wikipedia.org/wiki/Sampling_bias) in which their tests would underestimate the variation in the other lineages because their SNPs were undersampled. The VINO analysis compensated for this problem. Looking over the hundreds of thousands of markers they fixed upon loci which were highly informative of subspecies origin. In other words, those alleles which were representative of a particular subspecies (*castaneus, domesticus,* and *musculus*). The figure below illustrates their results. Observe that to the right the three wild subspecies are perfectly separated into distinct colors.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ng.847-F1.jpeg?resize=600%2C503)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ng.847-F1.jpeg?resize=600%2C503)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ng.847-F1.jpeg)

According to these results classic lab strains of mice then are predominantly derived from *domesticus*, but with a small residual proportion of *musculus* which varies by lineage, as well a trace level of *castaneus*. The wild-derived strains aren’t that surprising, they mostly resembled the wild subspecies from which they’re ostensibly descended, but there’s obviously been some admixture with other lines. The “H” lines, the leftmost wild-derived lab strains above, are hybrids.

Now some of you might be curious about the fact that the wild strains are pure, while the inbred lab strains exhibit some admixture. There is a common confusion that people sometimes have a hard time understanding that admixed populations can also be inbred. A simple thought experiment shows how this could be. Imagine a couple with a set of children. The father is white, the mother is black. Their children would be biracial. If the children mated and produced offspring the grandchildren would be inbred *and* racially admixed. There’s no contradiction here. The markers which distinguish lineages above are informative *across* the subspecies, and so are a very small proportion of the markers within the genome. Inbreeding is a reflection of the much broader genomic patterns which are the outcome of particular mating systems or events.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ng.847-F2.jpeg?resize=250%2C471)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ng.847-F2.jpeg?resize=250%2C471)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/05/ng.847-F2.jpeg)Next let’s look at how the admixture of *musculus* plays out against the dominant background of *domesticus* in the classic lab strains. To the left is a figure which shows the assignments of ancestry on chromosome 6 of all the lines above. It is rotated 90 degrees counterclockwise from figure 1. At the top you see the three wild lineages, which separate perfectly on the chromosomal level just as they did on the total genome scale. Next you see the wild derived lab strains. Now you notice admixture across the genomes. Finally you have the classic lab strains, predominantly *domesticus*, but with *musculus* and *castaneus* regions also visible. Remember that each horizontal line represents a specific lineage, while points left to right correspond to positions in the genome. Observe how the *musculus* and *castaneus* ancestry is strongly correlated across many of these lineages, showing up as vertical bands.

In their classic lab strain samples the ancestry from *musculus* and *castaneus* was \~5 and \~0.30 percent respectively. But, *musculus* admixture was only found across \~47 percent of the genome, while *castaneus* admixture only across \~3 percent of the genome. If the class lab strains derived their admixture from many different crosses in the putative “primal horde” of ur-mice there should be signatures across the whole genome of these distinct events, rather than concentrated dollops in a such localized fashion. This evidence of admixture in only a few regions of the genome is suggestive of the likelihood that the classic lab strains derive from a very small founder population. More specifically the authors believers that the *musculus* admixture of the predominantly *domesticus* classic lab strains is from the Japanese *molosinnus*, which itself is a hybrid between eastern *musculus* and *castaneus*. And sure enough when they focused on the regions which are attributable to *musculus* in the classic lab strains, these tended to cluster with eastern *musculus* lineages, rather than western ones. A further check was that the identity-by-descent (IBD) measure, which is correlated with genetic similarity (though it strictly measure descent from a common ancestor), shows that the *musculus* regions of the classic lab strains have \~98% identity with *molosinnus* derived lab strains (as opposed to \~83% IBD with generic *musculus*). As *molosinnus* is is a hybrid between *musculus* and *castaneus* this explains the presence of the latter in the classic lab strain ancestry, as well as the fact that the *castaneus* segments seem to be nested within and between the *musculus* regions.

Let’s jump to the discussion:

> …One study concluded that the genome of these strains is 68% *M. m. > domesticus*, 10% *M. m. molossinus*, 6% M. *m. musculus*, 3% *M. m. castaneus* and 13% of unknown origin6. On the other hand, we previously concluded that 92% is of *M. m. domesticus*, 6% is of *M. m. musculus* and 1% is of *M. m. castaneus* origin…. >
> … >
> …The results presented here conclusively show that classical inbred > strains are overwhelmingly derived from M. m. domesticus, that the non–M. m. domesticus contribution to their genomes is largely of M. m. molossinus origin and that **intersubspecific introgression is common in wild-derived laboratory strains**. >
> … >
> In summary, our observation of residual heterozygosity among inbred > mouse strains, the striking local differences in the level of genetic > similarity between substrains, the identification of large deletions > of different ages and prevalence of contamination emphasizes the > importance of deep, unbiased and frequent genetic characterization of > laboratory stocks. Our genome browser provides access to the trees and > links between recombination intervals, local trees and the maps for > subspecific origin and haplotype diversity. Our analysis shows that > classical inbred strains are in fact mosaics of a handful of > haplotypes present in the founder fancy mice population. The genetic > divergence among these haplotypes varies widely both locally and > across the genome. Furthermore, the contribution of subspecies other > than M. m. domesticus is limited, and its distribution highlights the > complex population structure in these strains. On the other hand, > wild-derived laboratory strains represent a deep reservoir of genetic > diversity untapped in classical strains and are in many cases > analogous to the three-way intersubspecific hybrids that classical > inbred strains were thought to be. Our previous work…**combined with the results of the deep survey of mouse resources presented here shows that the laboratory mouse is an unparalleled model for genetic studies in mammals.**

Obviously it is important to know if your “pure” wild-derived lab strain isn’t so pure, especially if you’re using it as some sort of reference. From what I can gather in this paper many researchers had to just take on faith the genetic relationship between the various strains which they were utilizing in their research, relying upon 20th century methods such as pedigrees. In the near future widespread genotyping might eliminate any concerns, but until then this sort of study should give researchers a better understanding of the confounding issues which might be introduced because of a false perception of genetic variation or uniformity in their stock. The ancestry estimates quoted above seem all over the place, so it was obviously time that someone did the survey reported in this paper.

Finally, I’m struck by the mighty-mouse roar at the end of the paper. Unparalleled indeed!

**Citation:** Yang, Hyuna, Wang, Jeremy R, Didion, John P, Buus, Ryan J, Bell, Timothy A, Welsh, Catherine E, Bonhomme, Francois, Yu, Alex Hon-Tsen, Nachman, Michael W, Pialek, Jaroslav, Tucker, Priscilla, Boursot, Pierre, McMillan, Leonard, Churchill, Gary A, & de Villena, Fernando Pardo-Manuel (2011). Subspecific origin and haplotype diversity in the laboratory mouse Nature Genetics : [10.1038/ng.847](https://dx.doi.org/10.1038/ng.847)

*Image credit: [Wikimedia](https://en.wikipedia.org/wiki/File:Apodemus_sylvaticus_bosmuis.jpg)*

### Related Posts:

- [Conservation of expression in human and mouse
  brains](https://www.gnxp.com/WordPress/2007/04/23/conservation-of-expression-in-human-and-mouse-brains/) - [Types of
  genetics](https://www.gnxp.com/WordPress/2012/04/25/types-of-genetics/) - [Phenotype to genotype: as easy as
  1,2,3...10](https://www.gnxp.com/WordPress/2007/10/03/phenotype-to-genotype-as-easy-as-1-2-3-10/) - [Mad Mice](https://www.gnxp.com/WordPress/2010/04/20/mad-mice/) - [The
  Brainbow](https://www.gnxp.com/WordPress/2007/11/09/the-brainbow/) - [A Population Geneticist's
  Apology](https://www.gnxp.com/WordPress/2015/07/19/a-population-geneticists-apology/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F05%2F31%2Fthe-mess-that-is-mouse%2F&linkname=The%20mess%20that%20is%20mouse "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F05%2F31%2Fthe-mess-that-is-mouse%2F&linkname=The%20mess%20that%20is%20mouse "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F05%2F31%2Fthe-mess-that-is-mouse%2F&linkname=The%20mess%20that%20is%20mouse "Email")[](https://www.addtoany.com/share)
