+++
title = "Tea leaves and"
full_title = "Tea leaves and population substructure"
date = "2011-02-22"
upstream_url = "https://www.gnxp.com/WordPress/2011/02/22/tea-leaves-and-population-substructure/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/02/22/tea-leaves-and-population-substructure/).

Tea leaves and population substructure

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/450px-Tea_leaves_steeping_in_a_zhong_čaj_05.jpg?resize=297%2C189)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/450px-Tea_leaves_steeping_in_a_zhong_čaj_05.jpg?resize=297%2C189)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/450px-Tea_leaves_steeping_in_a_zhong_čaj_05.jpg)  
*Image credit: Wikimol*

Over the past few months I’ve been encouraging people to pull down [ADMIXTURE](http://www.genetics.ucla.edu/software/admixture/), and push the public data sets through it. Additionally, you can also convert your
23andMe raw file into pedigree format pretty easily and integrate it into the public data sets with [PLINK](http://pngu.mgh.harvard.edu/~purcell/plink/). I’ve been following [Zack’s Harappa Ancestry Project](http://www.harappadna.org/) pretty closely, but I’ve been running the software myself and manipulating its parameters and seeing how things shake out. But the more and more I do it, the more I wonder if it isn’t like [regression analysis](https://en.wikipedia.org/wiki/Regression_analysis), a technique which is just waiting to be leveraged by human biases. I began thinking of this more deeply after a conversation with a computational biologist who outlined the structural problems with how *ad hoc* the utilization of statistics is in the life sciences.

These sorts of qualms are probably why I’m posting my results more on Facebook and passing them around friends, rather than putting them out there in the public domain. It isn’t that I think the results are going to be abused. **I just don’t know what they mean a lot of the time.** Or, perhaps more honestly I am suspicious of my own propensity to *see* what I *suspect*. A case of my priors strongly shaping the inferences which I might generate.

So I decided to do an experiment. Below are 8 runs, displayed as bar plots. Each thin sliver represents an individual. The colors again represent putative ancestral populations of which the modern populations are combinations, generated by the parameter K (so K = 2 means two ancestral populations, each corresponding to a different color). There are two data sets which I analyzed, group A and group B. I’ve also noted the K’s for each plot. But aside from that, I’ll leave you ignorant what these populations are or how many there are. Jot down some ideas as to what you can see. How many populations? How do they relate to each other? Can you perceive any real information in the higher K’s? I’ll put the “answers” below the fold. There’s no point in me saying what I think, I already know which populations these are, so I’m tainted.

\[zenphotopress album=264 sort=sort_order number=8\]

  
All the populations were from the HapMap, with 80,000 markers. They were two sets of three:

Group A: Yoruba, African Americans, and Tuscans

Group B: Gujaratis, African Americans, and Mexicans

I selected these two sets because in Group A you have a population, African Americans, which can plausibly be considered a combination of the two other populations. In Group B, this is not so. The Mexican and African American populations share similar European ancestry, but both are admixed. The Gujarati populations has no *close* relationship to any of the other populations, so ADMIXTURE is just trying to do the best with what it’s being given.

How did you fare?

### Related Posts:

- [Do you want your genotype in a public data
  set?](https://www.gnxp.com/WordPress/2013/01/16/do-you-want-your-genotype-in-a-public-data-set/) - [The Dodecad ancestry
  project](https://www.gnxp.com/WordPress/2010/10/25/the-dodecad-ancestry-project/) - [The limits of computational power - shades of
  1982](https://www.gnxp.com/WordPress/2011/03/29/the-limits-of-computational-power-shades-of-1982/) - [Two public
  genotypes](https://www.gnxp.com/WordPress/2011/06/15/two-public-genotypes/) - [A note on open
  genomics](https://www.gnxp.com/WordPress/2012/08/28/a-note-on-open-genomics/) - [1000 Genomes data, 290,000
  SNPs](https://www.gnxp.com/WordPress/2015/09/30/1000-genomes-data-290000-snps/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F02%2F22%2Ftea-leaves-and-population-substructure%2F&linkname=Tea%20leaves%20and%20population%20substructure "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F02%2F22%2Ftea-leaves-and-population-substructure%2F&linkname=Tea%20leaves%20and%20population%20substructure "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F02%2F22%2Ftea-leaves-and-population-substructure%2F&linkname=Tea%20leaves%20and%20population%20substructure "Email")[](https://www.addtoany.com/share)
