+++
title = "Swedes not so"
full_title = "Swedes not so homogeneous?"
date = "2011-02-10"
upstream_url = "https://www.gnxp.com/WordPress/2011/02/10/swedes-not-so-homogeneous/"

+++
Source: [here](https://www.gnxp.com/WordPress/2011/02/10/swedes-not-so-homogeneous/).

Swedes not so homogeneous?

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/malin1.png?resize=150%2C207)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/malin1.png?resize=150%2C207)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/malin1.png)  
*Credit: [David Shankbone](https://en.wikipedia.org/wiki/File:Malin_Akerman_at_Mercedes-Benz_Fashion_Week.jpg)*

The more and more I see fine-scale genomic analyses of population structure across the world the more and more I believe that the “stylized” models which were in vogue in the early 2000s which explained how the world was re-populated after the last Ice Age (and before) were wrong in deep ways. I’m talking about the grand narratives outlined in works such as Bryan Sykes’ [The Seven Daughters of Eve](https://www.amazon.com/exec/obidos/ASIN/0393323145/geneexpressio-20), the subtitle of which was “The Science That Reveals Our Genetic Ancestry.” If I had less faith in science to always *ultimately* right its course I’d probably become a post-modernist type who asserts that all these stories are fictions. Sykes’ model in particular seems to be very likely incorrect because of the utilization of [ancient DNA](http://blogs.discovermagazine.com/gnxp/2010/11/european-man-of-many-faces-cain-vs-abel/) to elucidate population movements past in Europe. From what we can gather it looks like coarse attempts to infer past distributions from current distributions (of specific lineages and their diversity) **resulted in a great deal of false clarity.** We’re not talking differences on the margins, but fundamental confusions. For example, Basques were always assumed to be a viable “reference” population for descendants of European hunter-gatherers. This was one of the linchpins of older historical genetics models. It turns out that this [fixed assumption may have been a false one](http://blogs.discovermagazine.com/gnxp/2010/02/the-basques-may-not-be-who-we-think-they-are/).

Not only were our past assumptions in simple models wrong, but the real explanations may also be rather complex. It turns out that ancient DNA of the “first farmers” and their “hunter-gatherer” neighbors in Central Europe reveals *a lot* of discontinuity between *both* these groups and modern Europeans. Why? It may be that in fact there were multiple migrations, and the palimpsest is going to be a tough cookie to excavate. But there’s no need to be disheartened, the old paradigms came crashing down thanks to data.

[![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)![ResearchBlogging.org](https://i0.wp.com/www.researchblogging.org/public/citation_icons/rb2_large_gray.png?w=640)](http://www.researchblogging.org)With that in mind I’ve been particularly interested in the European fringe, the far west and north. *If* any hunter-gatherer descendants survive in large numbers, it will be here. This is why I’m curious as to the genetics of the [Sami](http://blogs.discovermagazine.com/gnxp/2010/12/the-men-of-the-north-the-sami/) as well as the archaeology which tracks the spread of agriculture in [Northern Europe](http://blogs.discovermagazine.com/gnxp/2010/12/the-great-northern-culture-war/). A new paper in *PLoS ONE* focuses on [Sweden](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0016747?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed:+plosone/GeneticsandGenomics+(PLoS+ONE+Alerts:+Genetics+and+Genomics)), [Swedish Population Substructure Revealed by Genome-Wide Single Nucleotide Polymorphism Data](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0016747?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed:+plosone/GeneticsandGenomics+(PLoS+ONE+Alerts:+Genetics+and+Genomics)):

> The use of genome-wide single nucleotide polymorphism (SNP) data has > recently proven useful in the study of human population structure. We > have studied the internal genetic structure of the Swedish population > using more than 350,000 SNPs from 1525 Swedes from all over the > country genotyped on the Illumina HumanHap550 array. We have also > compared them to 3212 worldwide reference samples, including Finns, > northern Germans, British and Russians, based on the more than 29,000 > SNPs that overlap between the Illumina and Affymetrix 250K Sty arrays. > The Swedes – especially southern Swedes – were genetically close to > the Germans and British, while their genetic distance to Finns was > substantially longer. The overall structure within Sweden appeared > clinal, and the substructure in the southern and middle parts was > subtle. In contrast, the northern part of Sweden, Norrland, exhibited > pronounced genetic differences both within the area and relative to > the rest of the country. These distinctive genetic features of > Norrland probably result mainly from isolation by distance and genetic > drift caused by low population density. **The internal structure > within Sweden (FST = 0.0005 between provinces) was stronger than that > in many Central European populations, although smaller than what has > been observed for instance in Finland; importantly, it is of the > magnitude that may hamper association studies with a moderate number > of markers if cases and controls are not properly matched geographically.** Overall, our results underline the potential of genome-wide data in analyzing substructure in populations that might otherwise appear relatively homogeneous, such as the Swedes.

Playing around with ADMIXTURE I’m now happy to see 350,000 SNPs, but less assured by 29,000 SNPs. After a bunch of pruning I have a data set where individuals have 100,000 SNPs, and that seems marginal when it comes to differentiating variation in Western Europe among populations, though I suppose I didn’t do it very intelligently (i.e., I didn’t try to bias toward ancestrally informative markers).

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/sw1.png?resize=180%2C368)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/sw1.png?resize=180%2C368)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/sw1.png)A major “top line” finding of this paper is that Swedes exhibit more geographical substructure than more numerous populations inhabiting expansive Central European regions. Additionally, though not as distinctive as Finns vis-a-vis other Europeans, they are somewhat distinctive, especially those in the north. The bar plot to the left is generated by STRUCTURE, and you see set sets of populations at particular K’s, each K being a putative ancestral group.

The differentiation *within* Sweden is evident at higher K’s. That’s striking because notice that the Germans and British don’t exhibit the pattern (they state in the paper that they looked for geographical patterns). But for me what is striking is the **disjunction between Scandinavians and continental Germans, and the relative lack of one between the British and the Germans.** At K = 5 a difference does crop up. At the top you see Russians, so it looks like blue = Eastern European, while red = Western European, and the Germans are a mix of the two, with the Russians and British representing extreme “types” (again, these are very stylized facts, there are no pure “types). But the break with Swedes occurs at lower K’s. Why? The first thought is water. Water blocks gene flow a great deal, but then what about Britain? I doubt all the sampling in Britain was from the old Saxon Shore of East Anglia! I will hazard a rather general explanation: **maybe it’s agriculture!** More specifically, the switch to agriculture may have occurred via different demographic processes in the two locales. Britain has a milder climate than Sweden, and could presumably support a more dense transplanted culture more easily than Sweden.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/journal.pone_.0016747.g002.png?resize=244%2C219)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/journal.pone_.0016747.g002.png?resize=244%2C219)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/journal.pone_.0016747.g002.png)Let’s look at the data in a different way. The figure to the left shows the top two dimensions of variation in the data. The x axis explains 0.64% of the variance, and the y axis 0.24% (these are genetically close groups remember). The bottom left of the distribution consists of Germans, the top of the point the Russians, and to the far right eastern Finns. Finns are something of a European outlier, along with Basques and Sardinians, but it is interesting how much greater east-west distances correspond to less variance than north-south **at this scale**. On the broader trans-European level north-south differentiation is usually more significant than west-east. Why? I think geography explains it, the Mediterranean and the Atlantic fringe allowed for a rapid expansion of agriculturalists in Southern Europe from their point of origination in Anatolia. The move north was slower, and involved more amalgamation with hunter-gatherers. But, within Northern Europe there were local differences. Inland North European plain with its rich soil and riverine network may have allowed for a great deal of demographic expansion in the face of an extremely thin pre-Neolithic population. But, they met another point of resistance at the oceanic fringe, where maritime resources were great enough to support denser hunter-gatherer populations. This, I suspect, explains the discontinuity at the [Kattegat and Skagerak](https://en.wikipedia.org/wiki/Kattegat).

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/journal.pone_.0016747.s003-.png?resize=400%2C243)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/journal.pone_.0016747.s003-.png?resize=400%2C243)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/journal.pone_.0016747.s003-.png)Let’s take another look at genetic distance. The visualization to the left is a representation of the Fst between pairs of populations. I’ve added labels. Fst just measures the proportion of genetic variance which can be partitioned between groups. The x axis is the first dimension, and the y the second. **That geography is not always a good predictor of genetic distance.** Look at how close the sample for Orkney (off the coast of northern Scotland), the British, Germans, and the Utah whites (who are mostly British and German in origin) cluster in terms of genetic distance. In contrast, the French and French Basques differ a great deal.

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/20070502091827BlankMap-Eur.png?resize=200%2C197)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/20070502091827BlankMap-Eur.png?resize=200%2C197)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/20070502091827BlankMap-Eur.png)To illustrate the weirdness of some of the patterns, like a 5 year old I took a blank map of Europe and just drew a line from region to region based on distances on the first dimension (x axis). So you see a zig-zag in Western Europe, a sweep to the east, and finally the terminus in the east of Finland. You’d be surprised how often I want to scribble on a map nonsensically when I see some of the SNP-chip data. Yes, geography does correspond to genetic distance, roughly, but some of the deviations from expectation are really weird. Sardinians and Finns in particular seem to be the extreme points on some broad underlying pattern of genetic variance in Europe. But, obviously the Basques also represent another dimension. A simple model is bound to be wrong, but a complex one is going to be wrong in a lot of the details.

Finally, we’ve been talking about ancestry only. What about functionality? Genes sometimes after *code* for differences, some of them visible, and many of them significant. Not surprisingly ancient hunter-gatherers who were resident in Sweden were [lactose intolerant](http://blogs.discovermagazine.com/gnxp/2010/04/ancient-swedes-were-lactose-intolerant/). Why would they need to be able to digest milk as adults if they didn’t have herds of cattle?

By and large the authors didn’t find much functional significant in the sharp north-south difference in Sweden. But, there were some suggestions (there’s some issues with the statistical likelihood due to the lack of particular precautions which would mitigate against false positives):

------------------------------------------------------------------------

|                        |                 |                    |                                            |          |               |     |
|------------------------|-----------------|--------------------|--------------------------------------------|----------|---------------|-----|
|                        |                 |                    |                                            |          |               |     |
| Phenotype              | Unique GWA hits | SNPs within 200 kb |                                            |          |               |     |
|                        |                 | Total              | SNPs with regional differences( p \< 0.05) |          |               |     |
|                        |                 |                    | observed                                   | expected | chi-square p1 |     |
| Eye color              | 9               | 410                | 24                                         | 34.3     | 0.08          |     |
| Hair color             | 19              | 774                | 73                                         | 64.8     | 0.29          |     |
| Skin pigment           | 3               | 140                | 22                                         | 11.7     | **0**         |     |
| Height                 | 197             | 8683               | 680                                        | 726.5    | 0.08          |     |
| Lactase deficiency     | LCT gene        | 34                 | 0                                          | 2.8      | 0.11          |     |
| Immune system          | MHC region      | 1262               | 162                                        | 105.6    | **0**         |     |
| Blood lipids           | 92              | 2465               | 257                                        | 206.2    | **0**         |     |
| Cardiovascular disease | 32              | 1921               | 190                                        | 160.7    | 0.02          |     |
|                        |                 |                    |                                            |          |               |     |
|                        |                 |                    |                                            |          |               |     |

------------------------------------------------------------------------

[![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/350px-Saami_Family_1900.jpg?resize=226%2C198)![](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/350px-Saami_Family_1900.jpg?resize=226%2C198)](https://i0.wp.com/blogs.discovermagazine.com/gnxp/files/2011/02/350px-Saami_Family_1900.jpg)Why the differentiation? I think this is a clear case of **“maybe it’s agriculture.”** Northern Sweden was not ethnically cleansed and assimilated of its Sami until the early modern period. These were traditionally non-agricultural people, the closest Europe had to hunter-gatherers (since they herded reindeer they obviously weren’t hunter-gatherers). Some of the difference may simply be a Sami substrate in the north of Sweden, with all the functional differences entailed due to the lack of thousands of years of dense agriculture life.

**Citation:** Salmela E, Lappalainen T, Liu J, Sistonen P, & Andersen PM (2011). Swedish Population Substructure Revealed by Genome-Wide Single Nucleotide Polymorphism Data PLoS ONE : [10.1371/journal.pone.0016747](https://dx.doi.org/10.1371/journal.pone.0016747)

### Related Posts:

- [Pots, people &
  seeds](https://www.gnxp.com/WordPress/2009/08/30/pots-people-seeds/) - [More genetic structure of human
  populations](https://www.gnxp.com/WordPress/2009/05/16/more-genetic-structure-of-human-populations/) - [No one knows who Europeans are descended from
  (yet)](https://www.gnxp.com/WordPress/2009/09/04/no-one-knows-who-europeans-are-descended-from-yet/) - [Blood of the
  British](https://www.gnxp.com/WordPress/2006/09/22/blood-of-the-british/) - [Phylogeography of deep European genetic
  history](https://www.gnxp.com/WordPress/2010/04/22/phylogeography-of-deep-european-genetic-history/) - [Fine scale population
  substructure](https://www.gnxp.com/WordPress/2009/03/05/fine-scale-population-substructure/)

### *Related*

[](https://www.addtoany.com/add_to/facebook?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F02%2F10%2Fswedes-not-so-homogeneous%2F&linkname=Swedes%20not%20so%20homogeneous%3F "Facebook")[](https://www.addtoany.com/add_to/twitter?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F02%2F10%2Fswedes-not-so-homogeneous%2F&linkname=Swedes%20not%20so%20homogeneous%3F "Twitter")[](https://www.addtoany.com/add_to/email?linkurl=https%3A%2F%2Fwww.gnxp.com%2FWordPress%2F2011%2F02%2F10%2Fswedes-not-so-homogeneous%2F&linkname=Swedes%20not%20so%20homogeneous%3F "Email")[](https://www.addtoany.com/share)
